<!doctype html>
<html>
<head>
<meta charset='UTF-8'><meta name='viewport' content='width=device-width initial-scale=1'>
<title>2017_Eco_Gen_ANBE_nb.md</title><style type='text/css'>html, body {overflow-x: initial !important;}.mermaid .label { color: rgb(51, 51, 51); }
.node rect, .node circle, .node ellipse, .node polygon { fill: rgb(236, 236, 255); stroke: rgb(204, 204, 255); stroke-width: 1px; }
.edgePath .path { stroke: rgb(51, 51, 51); }
.edgeLabel { background-color: rgb(232, 232, 232); }
.cluster rect { fill: rgb(255, 255, 222) !important; stroke: rgb(170, 170, 51) !important; stroke-width: 1px !important; }
.cluster text { fill: rgb(51, 51, 51); }
.actor { stroke: rgb(204, 204, 255); fill: rgb(236, 236, 255); }
text.actor { fill: rgb(0, 0, 0); stroke: none; }
.actor-line { stroke: rgb(128, 128, 128); }
.messageLine0 { stroke-width: 1.5px; stroke: rgb(51, 51, 51); }
.messageLine1 { stroke-width: 1.5px; stroke: rgb(51, 51, 51); }
#arrowhead { fill: rgb(51, 51, 51); }
#crosshead path { fill: rgb(51, 51, 51) !important; stroke: rgb(51, 51, 51) !important; }
.messageText { fill: rgb(51, 51, 51); stroke: none; }
.labelBox { stroke: rgb(204, 204, 255); fill: rgb(236, 236, 255); }
.labelText { fill: rgb(0, 0, 0); stroke: none; }
.loopText { fill: rgb(0, 0, 0); stroke: none; }
.loopLine { stroke-width: 2px; stroke: rgb(204, 204, 255); }
.note { stroke: rgb(170, 170, 51); fill: rgb(255, 245, 173); }
.noteText { fill: rgb(0, 0, 0); stroke: none; font-family: 'trebuchet ms', verdana, arial; font-size: 14px; }
.section { stroke: none; opacity: 0.2; }
.section0 { fill: rgba(102, 102, 255, 0.490196); }
.section2 { fill: rgb(255, 244, 0); }
.section1, .section3 { fill: rgb(255, 255, 255); opacity: 0.2; }
.sectionTitle0 { fill: rgb(51, 51, 51); }
.sectionTitle1 { fill: rgb(51, 51, 51); }
.sectionTitle2 { fill: rgb(51, 51, 51); }
.sectionTitle3 { fill: rgb(51, 51, 51); }
.sectionTitle { text-anchor: start; font-size: 11px; }
.grid .tick { stroke: rgb(211, 211, 211); opacity: 0.3; shape-rendering: crispedges; }
.grid path { stroke-width: 0px; }
.today { fill: none; stroke: rgb(255, 0, 0); stroke-width: 2px; }
.task { stroke-width: 2px; }
.taskText { text-anchor: middle; font-size: 11px; }
.taskTextOutsideRight { fill: rgb(0, 0, 0); text-anchor: start; font-size: 11px; }
.taskTextOutsideLeft { fill: rgb(0, 0, 0); text-anchor: end; font-size: 11px; }
.taskText0, .taskText1, .taskText2, .taskText3 { fill: rgb(255, 255, 255); }
.task0, .task1, .task2, .task3 { fill: rgb(138, 144, 221); stroke: rgb(83, 79, 188); }
.taskTextOutside0, .taskTextOutside2 { fill: rgb(0, 0, 0); }
.taskTextOutside1, .taskTextOutside3 { fill: rgb(0, 0, 0); }
.active0, .active1, .active2, .active3 { fill: rgb(191, 199, 255); stroke: rgb(83, 79, 188); }
.activeText0, .activeText1, .activeText2, .activeText3 { fill: rgb(0, 0, 0) !important; }
.done0, .done1, .done2, .done3 { stroke: rgb(128, 128, 128); fill: rgb(211, 211, 211); stroke-width: 2px; }
.doneText0, .doneText1, .doneText2, .doneText3 { fill: rgb(0, 0, 0) !important; }
.crit0, .crit1, .crit2, .crit3 { stroke: rgb(255, 136, 136); fill: rgb(255, 0, 0); stroke-width: 2px; }
.activeCrit0, .activeCrit1, .activeCrit2, .activeCrit3 { stroke: rgb(255, 136, 136); fill: rgb(191, 199, 255); stroke-width: 2px; }
.doneCrit0, .doneCrit1, .doneCrit2, .doneCrit3 { stroke: rgb(255, 136, 136); fill: rgb(211, 211, 211); stroke-width: 2px; cursor: pointer; shape-rendering: crispedges; }
.doneCritText0, .doneCritText1, .doneCritText2, .doneCritText3 { fill: rgb(0, 0, 0) !important; }
.activeCritText0, .activeCritText1, .activeCritText2, .activeCritText3 { fill: rgb(0, 0, 0) !important; }
.titleText { text-anchor: middle; font-size: 18px; fill: rgb(0, 0, 0); }
.node text { font-family: 'trebuchet ms', verdana, arial; font-size: 14px; }
div.mermaidTooltip { position: absolute; text-align: center; max-width: 200px; padding: 2px; font-family: 'trebuchet ms', verdana, arial; font-size: 12px; background-color: rgb(255, 255, 222); border: 1px solid rgb(170, 170, 51); border-top-left-radius: 2px; border-top-right-radius: 2px; border-bottom-right-radius: 2px; border-bottom-left-radius: 2px; pointer-events: none; z-index: 100; background-position: initial initial; background-repeat: initial initial; }
#write .md-diagram-panel .md-diagram-panel-preview div { }


.CodeMirror { height: auto; }
.CodeMirror-scroll { overflow-y: hidden; overflow-x: auto; }
.CodeMirror-lines { padding: 4px 0px; }
.CodeMirror pre { }
.CodeMirror-scrollbar-filler, .CodeMirror-gutter-filler { background-color: white; }
.CodeMirror-gutters { border-right-width: 1px; border-right-style: solid; border-right-color: rgb(221, 221, 221); background-color: rgb(247, 247, 247); white-space: nowrap; }
.CodeMirror-linenumbers { }
.CodeMirror-linenumber { padding: 0px 3px 0px 5px; text-align: right; color: rgb(153, 153, 153); }
.CodeMirror div.CodeMirror-cursor { border-left-width: 1px; border-left-style: solid; border-left-color: black; z-index: 3; }
.CodeMirror div.CodeMirror-secondarycursor { border-left-width: 1px; border-left-style: solid; border-left-color: silver; }
.CodeMirror.cm-keymap-fat-cursor div.CodeMirror-cursor { width: auto; border: 0px; background-color: rgb(119, 238, 119); z-index: 1; background-position: initial initial; background-repeat: initial initial; }
.CodeMirror div.CodeMirror-cursor.CodeMirror-overwrite { }
.cm-tab { display: inline-block; }
.cm-s-typora-default .cm-header, .cm-s-typora-default .cm-property { color: rgb(217, 79, 138); }
.cm-s-typora-default pre.cm-header1:not(.cm-atom) :not(.cm-overlay) { font-size: 2rem; line-height: 2rem; }
.cm-s-typora-default pre.cm-header2:not(.cm-atom) :not(.cm-overlay) { font-size: 1.4rem; line-height: 1.4rem; }
.cm-s-typora-default .cm-atom, .cm-s-typora-default .cm-number { color: rgb(149, 132, 134); }
.cm-s-typora-default .cm-table-row, .cm-s-typora-default .cm-block-start { font-family: monospace; }
.cm-s-typora-default .cm-comment, .cm-s-typora-default .cm-code { color: rgb(74, 90, 159); font-family: monospace; }
.cm-s-typora-default .cm-tag { color: rgb(169, 68, 66); }
.cm-s-typora-default .cm-string { color: rgb(126, 134, 169); }
.cm-s-typora-default .cm-link { color: rgb(196, 122, 15); text-decoration: underline; }
.cm-s-typora-default .cm-variable-2, .cm-s-typora-default .cm-variable-1 { color: inherit; }
.cm-s-typora-default .cm-overlay { font-size: 1rem; font-family: monospace; }
.CodeMirror.cm-s-typora-default div.CodeMirror-cursor { border-left-width: 3px; border-left-style: solid; border-left-color: rgb(228, 98, 154); }
.cm-s-typora-default .CodeMirror-activeline-background { left: -60px; right: -30px; background-color: rgba(204, 204, 204, 0.2); background-position: initial initial; background-repeat: initial initial; }
.cm-s-typora-default .CodeMirror-gutters { border-right-style: none; background-color: inherit; }
.cm-s-typora-default .cm-trailing-space-new-line::after, .cm-startspace::after, .cm-starttab .cm-tab::after { content: '•'; position: absolute; left: 0px; opacity: 0; font-family: LetterGothicStd, monospace; }
.os-windows .cm-startspace::after, .os-windows .cm-starttab .cm-tab::after { left: -0.1em; }
.cm-starttab .cm-tab::after { content: ' '; }
.cm-startspace, .cm-tab, .cm-starttab, .cm-trailing-space-a, .cm-trailing-space-b, .cm-trailing-space-new-line { font-family: monospace; position: relative; }
.cm-s-typora-default .cm-trailing-space-new-line::after { content: '↓'; opacity: 0.3; }
.cm-s-inner .cm-keyword { color: rgb(119, 0, 136); }
.cm-s-inner .cm-atom, .cm-s-inner.cm-atom { color: rgb(34, 17, 153); }
.cm-s-inner .cm-number { color: rgb(17, 102, 68); }
.cm-s-inner .cm-def { color: rgb(0, 0, 255); }
.cm-s-inner .cm-variable { color: black; }
.cm-s-inner .cm-variable-2 { color: rgb(0, 85, 170); }
.cm-s-inner .cm-variable-3 { color: rgb(0, 136, 85); }
.cm-s-inner .cm-property { color: black; }
.cm-s-inner .cm-operator { color: rgb(152, 26, 26); }
.cm-s-inner .cm-comment, .cm-s-inner.cm-comment { color: rgb(170, 85, 0); }
.cm-s-inner .cm-string { color: rgb(170, 17, 17); }
.cm-s-inner .cm-string-2 { color: rgb(255, 85, 0); }
.cm-s-inner .cm-meta { color: rgb(85, 85, 85); }
.cm-s-inner .cm-qualifier { color: rgb(85, 85, 85); }
.cm-s-inner .cm-builtin { color: rgb(51, 0, 170); }
.cm-s-inner .cm-bracket { color: rgb(153, 153, 119); }
.cm-s-inner .cm-tag { color: rgb(17, 119, 0); }
.cm-s-inner .cm-attribute { color: rgb(0, 0, 204); }
.cm-s-inner .cm-header, .cm-s-inner.cm-header { color: blue; }
.cm-s-inner .cm-quote, .cm-s-inner.cm-quote { color: rgb(0, 153, 0); }
.cm-s-inner .cm-hr, .cm-s-inner.cm-hr { color: rgb(153, 153, 153); }
.cm-s-inner .cm-link, .cm-s-inner.cm-link { color: rgb(0, 0, 204); }
.cm-negative { color: rgb(221, 68, 68); }
.cm-positive { color: rgb(34, 153, 34); }
.cm-header, .cm-strong { font-weight: bold; }
.cm-del { text-decoration: line-through; }
.cm-em { font-style: italic; }
.cm-link { text-decoration: underline; }
.cm-error { color: rgb(255, 0, 0); }
.cm-invalidchar { color: rgb(255, 0, 0); }
.cm-constant { color: rgb(38, 139, 210); }
.cm-defined { color: rgb(181, 137, 0); }
div.CodeMirror span.CodeMirror-matchingbracket { color: rgb(0, 255, 0); }
div.CodeMirror span.CodeMirror-nonmatchingbracket { color: rgb(255, 34, 34); }
.cm-s-inner .CodeMirror-activeline-background { background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; background-position: inherit inherit; background-repeat: inherit inherit; }
.CodeMirror { position: relative; overflow: hidden; }
.CodeMirror-scroll { margin-bottom: -30px; margin-right: -30px; padding-bottom: 30px; padding-right: 30px; height: 100%; outline: none; position: relative; box-sizing: content-box; }
.CodeMirror-sizer { position: relative; }
.CodeMirror-vscrollbar, .CodeMirror-hscrollbar, .CodeMirror-scrollbar-filler, .CodeMirror-gutter-filler { position: absolute; z-index: 6; display: none; }
.CodeMirror-vscrollbar { right: 0px; top: 0px; overflow-x: hidden; overflow-y: scroll; }
.CodeMirror-hscrollbar { bottom: 0px; left: 0px; overflow-y: hidden; overflow-x: scroll; }
.CodeMirror-scrollbar-filler { right: 0px; bottom: 0px; }
.CodeMirror-gutter-filler { left: 0px; bottom: 0px; }
.CodeMirror-gutters { position: absolute; left: 0px; top: 0px; padding-bottom: 30px; z-index: 3; }
.CodeMirror-gutter { white-space: normal; height: 100%; box-sizing: content-box; padding-bottom: 30px; margin-bottom: -32px; display: inline-block; }
.CodeMirror-gutter-elt { position: absolute; cursor: default; z-index: 4; }
.CodeMirror-lines { cursor: text; }
.CodeMirror pre { border-top-left-radius: 0px; border-top-right-radius: 0px; border-bottom-right-radius: 0px; border-bottom-left-radius: 0px; border-width: 0px; background-color: transparent; font-family: inherit; font-size: inherit; margin: 0px; white-space: pre; word-wrap: normal; color: inherit; z-index: 2; position: relative; overflow: visible; background-position: initial initial; background-repeat: initial initial; }
.CodeMirror-wrap pre { word-wrap: break-word; white-space: pre-wrap; word-break: normal; }
.CodeMirror-code pre { border-right-width: 30px; border-right-style: solid; border-right-color: transparent; width: -webkit-fit-content; }
.CodeMirror-wrap .CodeMirror-code pre { border-right-style: none; width: auto; }
.CodeMirror-linebackground { position: absolute; left: 0px; right: 0px; top: 0px; bottom: 0px; z-index: 0; }
.CodeMirror-linewidget { position: relative; z-index: 2; overflow: auto; }
.CodeMirror-widget { }
.CodeMirror-wrap .CodeMirror-scroll { overflow-x: hidden; }
.CodeMirror-measure { position: absolute; width: 100%; height: 0px; overflow: hidden; visibility: hidden; }
.CodeMirror-measure pre { position: static; }
.CodeMirror div.CodeMirror-cursor { position: absolute; visibility: hidden; border-right-style: none; width: 0px; }
.CodeMirror div.CodeMirror-cursor { visibility: hidden; }
.CodeMirror-focused div.CodeMirror-cursor { visibility: inherit; }
.CodeMirror-selected { background-color: rgb(217, 217, 217); background-position: initial initial; background-repeat: initial initial; }
.CodeMirror-focused .CodeMirror-selected { background-color: rgb(215, 212, 240); background-position: initial initial; background-repeat: initial initial; }
.cm-searching { background-color: rgba(255, 255, 0, 0.4); background-position: initial initial; background-repeat: initial initial; }
.CodeMirror span { }
@media print { 
  .CodeMirror div.CodeMirror-cursor { visibility: hidden; }
}
.CodeMirror-lint-markers { width: 16px; }
.CodeMirror-lint-tooltip { background-color: infobackground; border: 1px solid black; border-top-left-radius: 4px; border-top-right-radius: 4px; border-bottom-right-radius: 4px; border-bottom-left-radius: 4px; color: infotext; font-family: monospace; overflow: hidden; padding: 2px 5px; position: fixed; white-space: pre-wrap; z-index: 10000; max-width: 600px; opacity: 0; -webkit-transition: opacity 0.4s; transition: opacity 0.4s; font-size: 0.8em; }
.CodeMirror-lint-mark-error, .CodeMirror-lint-mark-warning { background-position: 0% 100%; background-repeat: repeat no-repeat; }
.CodeMirror-lint-mark-error { background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAQAAAADCAYAAAC09K7GAAAAAXNSR0IArs4c6QAAAAZiS0dEAP8A/wD/oL2nkwAAAAlwSFlzAAALEwAACxMBAJqcGAAAAAd0SU1FB9sJDw4cOCW1/KIAAAAZdEVYdENvbW1lbnQAQ3JlYXRlZCB3aXRoIEdJTVBXgQ4XAAAAHElEQVQI12NggIL/DAz/GdA5/xkY/qPKMDAwAADLZwf5rvm+LQAAAABJRU5ErkJggg==); }
.CodeMirror-lint-marker-error, .CodeMirror-lint-marker-warning { cursor: pointer; display: inline-block; height: 16px; width: 16px; vertical-align: middle; position: relative; background-position: 50% 50%; background-repeat: no-repeat no-repeat; }
.CodeMirror-lint-message-error, .CodeMirror-lint-message-warning { padding-left: 18px; background-position: 0% 0%; background-repeat: no-repeat no-repeat; }
.CodeMirror-lint-marker-error, .CodeMirror-lint-message-error { background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAAHlBMVEW7AAC7AACxAAC7AAC7AAAAAAC4AAC5AAD///+7AAAUdclpAAAABnRSTlMXnORSiwCK0ZKSAAAATUlEQVR42mWPOQ7AQAgDuQLx/z8csYRmPRIFIwRGnosRrpamvkKi0FTIiMASR3hhKW+hAN6/tIWhu9PDWiTGNEkTtIOucA5Oyr9ckPgAWm0GPBog6v4AAAAASUVORK5CYII=); }
.CodeMirror-lint-marker-warning, .CodeMirror-lint-message-warning { background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAMAAAAoLQ9TAAAANlBMVEX/uwDvrwD/uwD/uwD/uwD/uwD/uwD/uwD/uwD6twD/uwAAAADurwD2tQD7uAD+ugAAAAD/uwDhmeTRAAAADHRSTlMJ8mN1EYcbmiixgACm7WbuAAAAVklEQVR42n3PUQqAIBBFUU1LLc3u/jdbOJoW1P08DA9Gba8+YWJ6gNJoNYIBzAA2chBth5kLmG9YUoG0NHAUwFXwO9LuBQL1giCQb8gC9Oro2vp5rncCIY8L8uEx5ZkAAAAASUVORK5CYII=); }
.CodeMirror-lint-marker-multiple { background-image: url(data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAcAAAAHCAMAAADzjKfhAAAACVBMVEUAAAAAAAC/v7914kyHAAAAAXRSTlMAQObYZgAAACNJREFUeNo1ioEJAAAIwmz/H90iFFSGJgFMe3gaLZ0od+9/AQZ0ADosbYraAAAAAElFTkSuQmCC); width: 100%; height: 100%; background-position: 100% 100%; background-repeat: no-repeat no-repeat; }


html { font-size: 14px; background-color: rgb(255, 255, 255); color: rgb(51, 51, 51); }
body { margin: 0px; padding: 0px; height: auto; bottom: 0px; top: 0px; left: 0px; right: 0px; font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; font-size: 1rem; line-height: 1.42857143; overflow-x: hidden; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; background-position: inherit inherit; background-repeat: inherit inherit; }
a:active, a:hover { outline: 0px; }
.in-text-selection, ::selection { background-color: rgb(181, 214, 252); text-shadow: none; background-position: initial initial; background-repeat: initial initial; }
#write { margin: 0px auto; height: auto; width: inherit; word-break: normal; word-wrap: break-word; position: relative; padding-bottom: 70px; white-space: pre-wrap; overflow-x: auto; }
.for-image #write { padding-left: 8px; padding-right: 8px; }
body.typora-export { padding-left: 30px; padding-right: 30px; }
@media screen and (max-width: 500px) { 
  body.typora-export { padding-left: 0px; padding-right: 0px; }
  .CodeMirror-sizer { margin-left: 0px !important; }
  .CodeMirror-gutters { display: none !important; }
}
.typora-export #write { margin: 0px auto; }
#write > p:first-child, #write > ul:first-child, #write > ol:first-child, #write > pre:first-child, #write > blockquote:first-child, #write > div:first-child, #write > table:first-child { margin-top: 30px; }
#write li > table:first-child { margin-top: -20px; }
img { max-width: 100%; vertical-align: middle; }
input, button, select, textarea { color: inherit; font-family: inherit; font-size: inherit; font-style: inherit; font-variant-caps: inherit; font-weight: inherit; line-height: inherit; }
input[type="checkbox"], input[type="radio"] { line-height: normal; padding: 0px; }
::before, ::after, * { box-sizing: border-box; }
#write p, #write h1, #write h2, #write h3, #write h4, #write h5, #write h6, #write div, #write pre { width: inherit; }
#write p, #write h1, #write h2, #write h3, #write h4, #write h5, #write h6 { position: relative; }
h1 { font-size: 2rem; }
h2 { font-size: 1.8rem; }
h3 { font-size: 1.6rem; }
h4 { font-size: 1.4rem; }
h5 { font-size: 1.2rem; }
h6 { font-size: 1rem; }
p { -webkit-margin-before: 1rem; -webkit-margin-after: 1rem; -webkit-margin-start: 0px; -webkit-margin-end: 0px; }
.mathjax-block { margin-top: 0px; margin-bottom: 0px; -webkit-margin-before: 0rem; -webkit-margin-after: 0rem; }
.hidden { display: none; }
.md-blockmeta { color: rgb(204, 204, 204); font-weight: bold; font-style: italic; }
a { cursor: pointer; }
#write input[type="checkbox"] { cursor: pointer; width: inherit; height: inherit; margin: 4px 0px 0px; }
tr { page-break-inside: avoid; page-break-after: auto; }
thead { display: table-header-group; }
table { border-collapse: collapse; border-spacing: 0px; width: 100%; overflow: auto; page-break-inside: auto; text-align: left; }
table.md-table td { min-width: 80px; }
.CodeMirror-gutters { border-right-width: 0px; background-color: inherit; }
.CodeMirror { text-align: left; }
.CodeMirror-placeholder { opacity: 0.3; }
.CodeMirror pre { padding: 0px 4px; }
.CodeMirror-lines { padding: 0px; }
div.hr:focus { cursor: none; }
pre { white-space: pre-wrap; }
.CodeMirror-gutters { margin-right: 4px; }
.md-fences { font-size: 0.9rem; display: block; page-break-inside: avoid; text-align: left; overflow: visible; white-space: pre; background-image: inherit; background-size: inherit; background-attachment: inherit; background-origin: inherit; background-clip: inherit; background-color: inherit; position: relative !important; background-position: inherit inherit; background-repeat: inherit inherit; }
.md-diagram-panel { width: 100%; margin-top: 10px; text-align: center; padding-top: 0px; padding-bottom: 8px; overflow-x: auto; }
.md-fences .CodeMirror.CodeMirror-wrap { top: -1.6em; margin-bottom: -1.6em; }
.md-fences.mock-cm { white-space: pre-wrap; }
.show-fences-line-number .md-fences { padding-left: 0px; }
.show-fences-line-number .md-fences.mock-cm { padding-left: 40px; }
.footnotes { opacity: 0.8; font-size: 0.9rem; padding-top: 1em; padding-bottom: 1em; }
.footnotes + .footnotes { margin-top: -1em; }
.md-reset { margin: 0px; padding: 0px; border: 0px; outline: 0px; vertical-align: top; background-color: transparent; text-decoration: none; text-shadow: none; float: none; position: static; width: auto; height: auto; white-space: nowrap; cursor: inherit; line-height: normal; font-weight: normal; text-align: left; box-sizing: content-box; direction: ltr; background-position: initial initial; background-repeat: initial initial; }
li div { padding-top: 0px; }
blockquote { margin: 1rem 0px; }
li p, li .mathjax-block { margin: 0.5rem 0px; }
li { margin: 0px; position: relative; }
blockquote > :last-child { margin-bottom: 0px; }
blockquote > :first-child { margin-top: 0px; }
.footnotes-area { color: rgb(136, 136, 136); margin-top: 0.714rem; padding-bottom: 0.143rem; }
@media print { 
  html, body { height: 100%; }
  .typora-export * { -webkit-print-color-adjust: exact; }
  h1, h2, h3, h4, h5, h6 { page-break-after: avoid; break-after: avoid-page; orphans: 2; }
  p { orphans: 4; }
  html.blink-to-pdf { font-size: 13px; }
  .typora-export #write { padding-left: 1cm; padding-right: 1cm; }
  .typora-export #write::after { height: 0px; }
  @page { margin: 20mm 0mm; }
}
.footnote-line { margin-top: 0.714em; font-size: 0.7em; }
a img, img a { cursor: pointer; }
pre.md-meta-block { font-size: 0.8rem; min-height: 2.86rem; white-space: pre-wrap; background-color: rgb(204, 204, 204); display: block; overflow-x: hidden; background-position: initial initial; background-repeat: initial initial; }
p .md-image:only-child { display: inline-block; width: 100%; text-align: center; }
#write .MathJax_Display { margin: 0.8em 0px 0px; }
.mathjax-block { white-space: pre; overflow: hidden; width: 100%; }
p + .mathjax-block { margin-top: -1.143rem; }
.mathjax-block:not(:empty)::after { display: none; }
[contenteditable="true"]:active, [contenteditable="true"]:focus { outline: none; box-shadow: none; }
.task-list { list-style-type: none; }
.task-list-item { position: relative; padding-left: 1em; }
.task-list-item input { position: absolute; top: 0px; left: 0px; }
.math { font-size: 1rem; }
.md-toc { min-height: 3.58rem; position: relative; font-size: 0.9rem; border-top-left-radius: 10px; border-top-right-radius: 10px; border-bottom-right-radius: 10px; border-bottom-left-radius: 10px; }
.md-toc-content { position: relative; margin-left: 0px; }
.md-toc::after, .md-toc-content::after { display: none; }
.md-toc-item { display: block; color: rgb(65, 131, 196); text-decoration: none; }
.md-toc-inner:hover { }
.md-toc-inner { display: inline-block; cursor: pointer; }
.md-toc-h1 .md-toc-inner { margin-left: 0px; font-weight: bold; }
.md-toc-h2 .md-toc-inner { margin-left: 2em; }
.md-toc-h3 .md-toc-inner { margin-left: 4em; }
.md-toc-h4 .md-toc-inner { margin-left: 6em; }
.md-toc-h5 .md-toc-inner { margin-left: 8em; }
.md-toc-h6 .md-toc-inner { margin-left: 10em; }
@media screen and (max-width: 48em) { 
  .md-toc-h3 .md-toc-inner { margin-left: 3.5em; }
  .md-toc-h4 .md-toc-inner { margin-left: 5em; }
  .md-toc-h5 .md-toc-inner { margin-left: 6.5em; }
  .md-toc-h6 .md-toc-inner { margin-left: 8em; }
}
a.md-toc-inner { font-size: inherit; font-style: inherit; font-weight: inherit; line-height: inherit; }
.footnote-line a:not(.reversefootnote) { color: inherit; }
.md-attr { display: none; }
.md-fn-count::after { content: '.'; }
.md-tag { opacity: 0.5; }
.md-comment { color: rgb(162, 127, 3); opacity: 0.8; font-family: monospace; }
code { text-align: left; }
h1 .md-tag, h2 .md-tag, h3 .md-tag, h4 .md-tag, h5 .md-tag, h6 .md-tag { font-weight: initial; opacity: 0.35; }
a.md-print-anchor { border: none !important; display: inline-block !important; position: absolute !important; width: 1px !important; right: 0px !important; outline: none !important; background-color: transparent !important; text-shadow: initial !important; background-position: initial initial !important; background-repeat: initial initial !important; }
.md-inline-math .MathJax_SVG .noError { display: none !important; }
.mathjax-block .MathJax_SVG_Display { text-align: center; margin: 1em 0em; position: relative; text-indent: 0px; max-width: none; max-height: none; min-height: 0px; min-width: 100%; width: auto; display: block !important; }
.MathJax_SVG_Display, .md-inline-math .MathJax_SVG_Display { width: auto; margin: inherit; display: inline-block !important; }
.MathJax_SVG .MJX-monospace { font-family: monospace; }
.MathJax_SVG .MJX-sans-serif { font-family: sans-serif; }
.MathJax_SVG { display: inline; font-style: normal; font-weight: normal; line-height: normal; zoom: 90%; text-indent: 0px; text-align: left; text-transform: none; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; direction: ltr; max-width: none; max-height: none; min-width: 0px; min-height: 0px; border: 0px; padding: 0px; margin: 0px; }
.MathJax_SVG * { -webkit-transition: none; transition: none; }


@import-when-export url(http://fonts.googleapis.com/css?family=Merriweather:900,900italic,300,300italic);
@import-when-export url(http://fonts.googleapis.com/css?family=Lato:900,300);


/**
 * forked from pixyll.com
 * MIT license
 */

@charset "UTF-8";
@font-face {
	font-family: 'Merriweather';
	font-style: normal;
	font-weight: 300;
	src: local('Merriweather Light'), local('Merriweather-Light'), url(file:///Users/andrewnguyen/Library/Application%20Support/abnerworks.Typora/themes/pixyll/Merriweather-Light.ttf);
}
@font-face {
	font-family: 'Merriweather';
	font-style: normal;
	font-weight: 900;
	src: local('Merriweather Heavy'), local('Merriweather-Heavy'), url(file:///Users/andrewnguyen/Library/Application%20Support/abnerworks.Typora/themes/pixyll/Merriweather-Bold.ttf);
}
@font-face {
	font-family: 'Merriweather';
	font-style: italic;
	font-weight: 300;
	src: local('Merriweather Light Italic'), local('Merriweather-LightItalic'), url(file:///Users/andrewnguyen/Library/Application%20Support/abnerworks.Typora/themes/pixyll/Merriweather-LightItalic.ttf);
}
@font-face {
	font-family: 'Merriweather';
	font-style: italic;
	font-weight: 900;
	src: local('Merriweather Heavy Italic'), local('Merriweather-HeavyItalic'), url(file:///Users/andrewnguyen/Library/Application%20Support/abnerworks.Typora/themes/pixyll/Merriweather-HeavyItalic.ttf);
}

@font-face {
	font-family: 'Lato';
	font-style: normal;
	font-weight: 300;
	src: local('Lato Light'), local('Lato-Light'), url(file:///Users/andrewnguyen/Library/Application%20Support/abnerworks.Typora/themes/pixyll/Lato-Light.ttf);
}
@font-face {
	font-family: 'Lato';
	font-style: normal;
	font-weight: 900;
	src: local('Lato Black'), local('Lato-Blcak'), url(file:///Users/andrewnguyen/Library/Application%20Support/abnerworks.Typora/themes/pixyll/Lato-Black.ttf);
}
@font-face {
	font-family: 'Lato';
	font-style: normal;
	font-weight: bold;
	src: local('Lato Black'), local('Lato-Blcak'), url(file:///Users/andrewnguyen/Library/Application%20Support/abnerworks.Typora/themes/pixyll/Lato-Black.ttf);
}
@font-face {
	font-family: 'Lato';
	font-style: italic;
	font-weight: 900;
	src: local('Lato BlackItalic'), local('Lato-BlackItalic'), url(file:///Users/andrewnguyen/Library/Application%20Support/abnerworks.Typora/themes/pixyll/Lato-BlackItalic.ttf);
}
@font-face {
	font-family: 'Lato';
	font-style: italic;
	font-weight: bold;
	src: local('Lato BlackItalic'), local('Lato-BlackItalic'), url(file:///Users/andrewnguyen/Library/Application%20Support/abnerworks.Typora/themes/pixyll/Lato-BlackItalic.ttf);
}

h1,
.h1,
.f1 {
	font-size: 2rem;
	line-height: 2.5rem;
}
h2,
.h2,
.f2 {
	font-size: 1.5rem;
	line-height: 2rem;
}
h3,
.h3,
.f3 {
	font-size: 1.25rem;
	line-height: 1.5rem;
}
p,
.p,
.f4,
h4,
h5,
h6,
dl,
ol,
ul,
pre[cid],
div[cid],
#typora-source {
	font-size: 1.125rem;
	line-height: 1.5rem;
}

h4 {
	font-size: 1.13rem;
}
/*
 Pixyll
 A simple, beautiful theme for Jekyll that emphasizes content rather than aesthetic fluff.
 Best served with BASSCSS (http://jxnblk.github.io/basscss)
 Crafted with <3 by John Otander (@4lpine) - ©2015 John Otander MIT License http://opensource.org/licenses/MIT

*/

body {
	font-family: "Merriweather", "PT Serif", Georgia, "Times New Roman", "STSong", Serif;
	line-height: 1.5rem;
	font-weight: 400;
}

#write {
	max-width: 914px;
	color: #333;
}

img {
	width: auto;
	max-width: 100%;
}
body {
	font-size: 1.5rem;
	box-sizing: border-box;
	-moz-box-sizing: border-box;
	-webkit-box-sizing: border-box;
}

.md-table-edit {
	background: #ededed;
    padding-top: 4px;
}

table {
	width: 100%;
	font-size: 1.125rem;
}
table > thead > tr > th,
table > thead > tr > td,
table > tbody > tr > th,
table > tbody > tr > td,
table > tfoot > tr > th,
table > tfoot > tr > td {
	padding: 12px;
	line-height: 1.2;
	vertical-align: top;
	border-top: 1px solid #333;
}
table > thead > tr > th {
	vertical-align: bottom;
	border-bottom: 2px solid #333;
}
table > caption + thead > tr:first-child > th,
table > caption + thead > tr:first-child > td,
table > colgroup + thead > tr:first-child > th,
table > colgroup + thead > tr:first-child > td,
table > thead:first-child > tr:first-child > th,
table > thead:first-child > tr:first-child > td {
	border-top: 0;
}
table > tbody + tbody {
	border-top: 2px solid #333;
}
p {
	font-weight: 300;
	line-height: 1.5;
}
abbr {
	border-bottom: 1px black dotted;
	cursor: help;
}
pre,
code {
	font-family: Menlo, Monaco, "Courier New", monospace;
}

code,
.md-fences  {
	color: #7a7a7a;
}

.md-fences .CodeMirror.CodeMirror-wrap {
	top: -1.5em;
    margin-bottom: -1.5em;
}

.md-fences {
	padding: 1.125em;
	margin-bottom: 0.88em;
	font-size: 1rem;
	border: 1px solid #7a7a7a;
}

blockquote {
	padding: 1.33em;
	font-style: italic;
	border-left: 5px solid #7a7a7a;
	color: #555;
}
blockquote em {
	color: #000;
}
blockquote footer {
	font-size: .85rem;
	font-style: normal;
	background-color: #fff;
	color: #7a7a7a;
	border-color: transparent;
}
h1,
.h1,
h2,
.h2,
h3,
.h3,
h4,
.h4,
h5,
.h5,
h6,
.h6 {
	font-family: "Lato", 'Helvetica Neue', Helvetica, sans-serif;
	font-weight: 900;
	line-height: 1.2;
	margin: 1em 0 0.5em;
}
@media screen and (min-width: 48em) {
	.h1,
	h1 {
		font-size: 3.250rem;
	}
	.h2,
	h2 {
		font-size: 2.298rem;
	}
	.h3,
	h3 {
		font-size: 1.625rem;
	}
	.h4,
	h4 {
		font-size: 1.3rem;
	}
	#write>h4.md-focus:before,
	#write>h5.md-focus:before,
	#write>h6.md-focus:before{
		top: 1px;
	}
	.p,
	p,
	li {
		font-size: 1.25rem;
		line-height: 1.8;
	}
	table {
		font-size: 1.25rem;
	}
}
@media (max-width: 48em) {
	blockquote {
		margin-left: 1rem;
		margin-right: 0;
		padding: 0.5em;
	}
	.h1,
	h1 {
		font-size: 2.827rem;
	}
	.h2,
	h2 {
		font-size: 1.999rem;
	}
	.h3,
	h3 {
		font-size: 1.413rem;
	}
	.h4,
	h4 {
		font-size: 1.3rem;
	}
}
@media screen and (min-width: 64em) {
	.h1,
	h1 {
		font-size: 4.498rem;
	}
	.h2,
	h2 {
		font-size: 2.29rem;
	}
	.h3,
	h3 {
		font-size: 1.9rem;
	}
	.h4,
	h4 {
		font-size: 1.591rem;
	}
	#write>h4.md-focus:before{
		top:4px;
	}
}
a {
	color: #463F5C;
	text-decoration: underline;
}

#write {
	padding-top: 2rem;
}

#write pre.md-meta-block {
	min-height: 35px;
	padding: 0.5em 1em;
	white-space: pre;
	border: 0px;

	border-left: 30px #f8f8f8 solid;
	border-right: 30px #f8f8f8 solid;
	width: 100vw;
	max-width: calc(100% + 60px);

	margin-left: -30px;
	margin-bottom: 2em;
	margin-top: -2010px;
	padding-top: 2000px;
	padding-bottom: 10px;
	line-height: 1.5em;
	color: #7a7a7a;
	background-color: #fafafa;
	font-family: 'Lato', 'Helvetica Neue', Helvetica, sans-serif;
	font-weight: 300;
	clear: both;
	padding-left: 0;
	font-size:1.125rem;
}
.md-image>.md-meta {
	color: #463F5C
}
.footnotes {
	font-size:1.1rem;
}
.md-tag {
	font-family: 'Lato', 'Helvetica Neue', Helvetica, sans-serif;
}
.code-tooltip {
	background: white;
}
.code-tooltip-content {
	font-size: 1.1rem;
}

.task-list{
	padding-left: 0;
}

.task-list-item {
	padding-left:34px;
}

.task-list-item input{
	width: 1.25rem;
	height: 1.25rem;
	display: block;
	-webkit-appearance: initial;
	top: -2px;
}

.task-list-item input:focus{
	outline: none;
	box-shadow: none;
}

.task-list-item input:before{
	border: 1px solid #555;
	border-radius: 1.5rem;
	width: 1.5rem;
	height: 1.5rem;
	background: #fff;
	content: ' ';
	transition: background-color 200ms ease-in-out;
	display: block;
}

.task-list-item input:checked:before,
.task-list-item input[checked]:before{
	background: #333;
	border-width: 2px;
	display:inline-block;
	transition: background-color 200ms ease-in-out;
}

.task-list-item input:checked:after,
.task-list-item input[checked]:after {
	opacity: 1;
}

.task-list-item input:after {
	opacity: 1;
	-webkit-transition: opacity 0.05s ease-in-out;
	-moz-transition: opacity 0.05s ease-in-out;
	transition: opacity 0.05s ease-in-out;
	-webkit-transform: rotate(-45deg);
	-moz-transform: rotate(-45deg);
	transform: rotate(-45deg);
	position: absolute;
	top: 0.4375rem;
	left: 0.28125rem;
	width: 0.9375rem;
	height: 0.5rem;
	border: 3px solid #fff;
	border-top: 0;
	border-right: 0;
	content: ' ';
	opacity: 0;
}

.md-tag {
	color:inherit;
}

.md-toc:focus .md-toc-content{
	margin-top: 19px;
}

#outline-dropmenu {
	background-color:white;
	font-size:1rem !important;
}

#outline-content li, #outline-content ul {
	font-size:1rem !important;
}

.outline-title {
	line-height: inherit;
	margin-top: 10px;
}

.outline-expander {
	width: 18px;
}

.outline-expander:before {
 	content: "+";
	font-family: inherit;
	color: rgb(108, 108, 108); 
  	font-size: 1.5rem;
 	top: 0.1rem;
}

.outline-expander:hover:before {
	content: "+";
}

.outline-item-open>.outline-item>.outline-expander:before{
	content: "-";
}

/** source code mode */
#typora-source {
	font-family: Courier, monospace;
    color: #6A6A6A;
}

.os-windows #typora-source {
	font-family: inherit;
}

.cm-s-typora-default .cm-header, 
.cm-s-typora-default .cm-property,
.CodeMirror.cm-s-typora-default div.CodeMirror-cursor {
	color: #428bca;
}

.cm-s-typora-default .cm-atom, .cm-s-typora-default .cm-number {
	color: #777777;
}


.md-diagram-panel {
	margin-top: 24px;
	margin-left: -1.2em;
}


</style>
</head>
<body class='typora-export' >
<div  id='write'  class = 'is-mac'><h1><a name='header-n0' class='md-header-anchor '></a>2017 Ecological Genomics Course</h1><h3><a name='header-n2' class='md-header-anchor '></a>Author: ANTdrew D. Nguyen</h3><h2><a name='header-n4' class='md-header-anchor '></a>Overall Description of notebook</h2><p>I am the teaching assistant, but I will follow the tutorials and log what I&#39;ve done here. </p><h2><a name='header-n7' class='md-header-anchor '></a>Date started: 2017-01-18</h2><h2><a name='header-n8' class='md-header-anchor '></a>Date end/last modified:   2017-04-23</h2><h2><a name='header-n10' class='md-header-anchor '></a>Philosophy</h2><p>Science should be reproducible and one of the best ways to achieve this is by logging research activities in a notebook. Because science/biology has increasingly become computational, it is easier to document computational projects in an electronic form, which can be shared online through Github.  <br/></p><p> <a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a><br/>
.<br/></p><h3><a name='header-n17' class='md-header-anchor '></a>Table of contents for 60 entries (Format is <em>Page: Date(with year-month-day). Title</em>)</h3><ul><li><a href='#id-section1'>Page 1: 2017-01-18</a>. First class; intros</li><li><a href='#id-section2'>Page 2: 2017-01-20</a>. Readings for 2017-01-23 Monday  <br/></li><li><a href='#id-section3'>Page 3: 2017-01-23</a>. Week 2, Day 2, course notes</li><li><a href='#id-section4'>Page 4: 2017-01-25</a> . Week 2, Day 3, class notes (paper discussions and student project development)</li><li><a href='#id-section5'>Page 5: 2017-01-30</a>. Week 3, Day 4, class notes , Group presentations of project ideas</li><li><a href='#id-section6'>Page 6: 2017-02-01</a>. Week 3, Day 5, command line stuff</li><li><a href='#id-section7'>Page 7: 2017-02-03</a>. Installing trinity onto UVM cluster</li><li><a href='#id-section8'>Page 8: 2017-02-06</a>. Week 4, Day 6, RNA-seq</li><li><a href='#id-section9'>Page 9: 2017-02-08</a>. Week 4, Day 7, RNA-seq cont&#39;d + paper discussion DePanis et al. 2016; MolEco</li><li><a href='#id-section10'>Page 10: 2017-02-10</a>. Prepping for leading journal club discussion: 2015-02-15; Zhao et al. 2016; <em>MBE</em></li><li><a href='#id-section11'>Page 11: 2017-02-13</a>. Week 5, Day 8, RNA-seq mapping and paper discussion: Johnston et al. 2016, <em>Molecular Ecology</em></li><li><a href='#id-section12'>Page 12: 2017-02-15</a>.  Week 5, Day 9, Sam info update, ANBE paper discussion, Zhao et al. 2016; <em>MBE</em></li><li><a href='#id-section13'>Page 13: 2017-02-22</a>. Week 6, Day 10, RNA-seq and paper discussion; Dixon et al. 2016; Genomic determinants of coral heat tolerance across latitudes.</li><li><a href='#id-section14'>Page 14: 2017-02-27</a>.  Week 7, Day 11, RNA-seq and paper discussion (Edwards et al. 2016; <em>PNAS</em>)-phylogeography; Scott  Edwards visit</li><li><a href='#id-section15'>Page 15: 2017-03-06</a>. Week 8, Day 13, Population genomics; paper discussion- Gayral et al. 2013</li><li><a href='#id-section16'>Page 16: 2017-03-08</a>. Week 8, Day 14, Population genomics part 2; Paper Discussion- <a href='http://www.nature.com/nature/journal/v515/n7526/full/nature13685.html'>Romiguier et al. 2014</a></li><li><a href='#id-section17'>Page 17: 2017-03-20</a>. Week 9, Day 15, Population genomics part 3; paper discussion Gompert et al. 2014</li><li><a href='#id-section18'>Page 18: 2017-03-22</a>. Week 9, Day 16, Population genomics part 4; paper discussion; Inferring Demographic processes with SNPs</li><li><a href='#id-section19'>Page 19: 2017-03-27</a>. Week 10, Day 17, Population genomics-detecting signatures of natural selection; selection scans; paper discussion- Laurent et al. 2016, MolEco</li><li><a href='#id-section20'>Page 20: 2017-03-29 </a>. Week 10, Day 18</li><li><a href='#id-section21'>Page 21: 2017-04-03</a>. Week 11, Day 19 Karl&#39;s guest lecture; FST and FST outlier</li><li><a href='#id-section22'>Page 22: 2017-04-05</a>. Week 11, Day 20; Annotation</li><li><a href='#id-section23'>Page 23: 2017-04-10</a>. Week 12, Day 21, paper discussion Ziegler et al. 2017</li><li><a href='#id-section24'>Page 24: 2017-04-12</a>. Week 12, Day 22, KTyler info update; microbiome; paper discussion Bolnick et al. 2014</li><li><a href='#id-section25'>Page 25: 2017-04-19</a>. Week 13, Day 24, microbiome cont&#39;d</li></ul><hr /><p> <div id='id-section1'/><br/></p><h3><a name='header-n97' class='md-header-anchor '></a>Page 1: 2016-07-18. Ecological genomics, first class</h3><h3><a name='header-n98' class='md-header-anchor '></a><strong>Steve and Melissa&#39;s intro</strong></h3><ul><li><p>Steve: It is a young field, trying to establish it&#39;s own identity  <br/></p><ul><li>Ecological genomics institute, KSU: emphasis on adaptation to environment <br/></li><li>Gordon Research Conference: Integrating different levels of biological organization on <strong>ANY SYSTEM</strong>; approach and tool focused! Field going towards new data and new analytic techniques<br/></li><li>Intro to eco genomics, oxford press; Using technology to address ecological issues such as nutrient cycling, population structure, life history vairation , trophic interaction, stress responess, and adpatation to environmental change <br/></li></ul></li><li><p>DATA driven: next gen sequencing revolutionizes biology</p><ul><li>creats a new problem--large datasets!!! how to make sense? </li><li>not data limited and potentially computationally limited <br/></li></ul></li><li><p>Where is the field headed  <br/></p><ul><li><p>Molecular Ecology Journal(flagship journal representative o the field)<br/></p><ul><li>ALL systems:  corals, protists, daphnia, coral, lemurs, dandelions, steve studies trees </li><li>model organism constraint disappearing! <br/></li></ul></li><li><p>What types of questions are asked?<br/></p><ul><li>How do genes correspond with circadian rythm?<br/></li><li>How does the microbiome influence the organism? </li><li>How does epigenetic variation influence evolutionary responses? or contribute to phenotypic variation?<br/></li><li>What are the patterns of genetic diversity that can give us insights on population dynamics?<br/></li><li>What are constraints and tradeoffs and genetic mechanisms of traits? </li></ul></li></ul></li><li><p>Methods? <br/></p><ul><li><p>De novo genome assembly; sequencing a DNA book from scratch!!  <br/></p><ul><li>RNA-seq; transcriptomic profiling   <br/></li></ul></li><li><p>16 s metagenomic sequencing    <br/></p></li><li><p>Rad-seq/GBS for estiamting population structure and genetic diversity   <br/></p></li></ul></li><li><p>Proccesses studied?  <br/></p><ul><li>All evo and eco stuff; speciation, hybridization, local adaptation, genetic basis of local adaptation, genetic architecture of complex phenotypes, genes controlling host-pathogen evolutionary dynamics, pop structure, gene flow, epigenetics   <br/></li></ul></li><li><p>Goals of the course!  <br/></p><ol start='' ><li>Learn how ecology and genomes shape each other <br/></li><li>Think creatively about major questions, and pose testable hypotheses to those questions using appropriate genomic data  <br/></li><li>Think about careful experimental design and statistical analysis---shown by reading papers <br/></li><li>Achive working knowledge and level of comfort for bioinformatics routines for ecological genomics studies <br/></li></ol><h3><a name='header-n196' class='md-header-anchor '></a>Melissa background</h3></li></ul><p><strong>Background, what drove Melissa and Steve to ecological genomics?</strong>     <br/></p><p>Melissa read a cool paper that scales from analyzing a few loci to the whole genome. <br/></p><p>One figure popped out at her, FST (developed by Sewell Wright) histogram.   FST of 1= complete differentiation, FST of 0 = no diff. FST described as <strong>Alleles in space</strong>. From this histogram, Melissa was struck by how you can separate out neutral from selective ones.<br/></p><p>Melissa has a data set with 96 sea stars and then the 16s microbiome. Would be cool to see if there is heritability in some bactera</p><h3><a name='header-n205' class='md-header-anchor '></a>Steve background</h3><ul><li><p>Inspired by Yanis Antonivics (an <strong>OG</strong>) <br/></p></li><li><p>At the time, just so stories: <strong>Adaptatationist programme</strong>  <br/></p><ul><li>Just go out and go by feeling in a natural history way and prescribe an adaptation story <br/></li><li>Janis wrote a creed to quantify the operational relationship between traits, environment, and genetics   <br/></li></ul></li><li><p>Yanis was on Steve&#39;s committee and Steve was interested in adaptation with respect to invasion biology because organisms need to respond to novel environments   <br/></p><ul><li>Phenotypes can relate to the environment, but what is the genetic basis of local adaptation (in situ)? There are other confounding issues: demographic effects, plasticity   <br/></li></ul></li><li><p>Steve thinks about environment-phenotype-genetics triangle. Basically a path diagram that feeds back on each other.  <br/></p><ul><li>Relationship between genes and phenotype ---GWAS (Genome wide association study)  <br/></li><li>Relationship between genetics and environment --- Fst, clines between allele frequencies and environment  <br/></li></ul></li><li><p>Invasion history is tough because of demographic history  <br/></p></li><li><p>He decided to focus on trees; large population size, straddle huge environmental gradients so the opportunity for selection is high <br/></p><ul><li>positive relationship between Growing season length and traits  <br/></li><li>Did a  reciprocal transplant of different populations to identify the extent of local adaptation in large established common gardens  <br/></li><li>SK does GBS (genotype by sequencing)    <br/></li><li>Problem with field: validating key gene candidates          <br/></li></ul></li></ul><hr /><p> <div id='id-section2'/><br/></p><h3><a name='header-n259' class='md-header-anchor '></a>Page 2: 2017-01-20. Readings for 2017-01-25 Monday</h3><p>First, showing how I structured the readings in the repo: </p><ol start='' ><li>In the terminal, change directory into your repo</li></ol><pre class='md-fences mock-cm' style='display:block;position:relative'>cd Teaching/2017_Ecological_Genomics/</pre><p>For fun, list the items in your repo</p><pre class='md-fences mock-cm' style='display:block;position:relative'>ls
01_data				Online_notebook.md
02_scripts			README.md
03_results_output		index.Rmd
04_tutorials			index.html
2017_Ecological_Genomics.Rproj	index.pdf
ANBE_notebook.md		papers
</pre><ol start='2' ><li>Now use tree command to extract hierarchical layout   <br/></li></ol><pre class='md-fences mock-cm' style='display:block;position:relative'>tree
.
├── 01_data
├── 02_scripts
│   └── RasterPCA_demo.Rmd
├── 03_results_output
│   └── RasterPCA_demo.html
├── 04_tutorials
├── 2017_Ecological_Genomics.Rproj
├── ANBE_notebook.md
├── Online_notebook.md
├── README.md
├── index.Rmd
├── index.html
├── index.pdf
└── papers
    └── 01_week
        └── 01_Day_Monday_2017-01-25
            ├── Ellegren_2014.pdf
            ├── Lee_gould_stinchcombe_2014_AoB_PLANTS.pdf
            └── Rockman-2012-Evolution.pdf
</pre><p>So my plan is to place my papers in the &quot;papers&quot; directory, within a particular week, and then within a particular day number of the course that is annotated with the actual day and date.<br/></p><h3><a name='header-n277' class='md-header-anchor '></a>Now paper notes:</h3><ol start='' ><li><p>Genome Sequencing and population genomics in non-model organisms ; Hans Ellegren  <br/></p><ul><li><p>3 important achievements in bio over the past century: modern synthesis(evolutionary theory), mol bio, and &quot;omics&quot; era <br/></p><ul><li>Good example of how biologists can get carried away with &quot;omics&quot; or &quot;omes&quot; <br/></li><li>peptidome, degradome,   <br/></li></ul></li><li><p>Basically,  because we can sequence everything we can collect, we can study non-model organisms like never before.<br/></p></li><li><p>In fact, many genomes are sequenced <br/></p></li><li><p>Bird example: Chickens were sequenced first, then zebra finch, then a bunch of others, highlighting the rapid ability to sequence genomes <br/></p></li><li><p>Genomic information is nice because it can tell you how loci are arranged. Example, recombination can be better studied by knowing the arrangement of chromosomes. <br/></p></li><li><p>Having genomic sequences allows for us to compare the repertoire of genes and the actual sequences themselves. (birth and death evolutionary procceses in homologues; tests for positive selection)</p></li></ul></li><li><p>Identifying the genes underlying quantitative traits: a rationale for the QTP programme; Lee et al. 2014</p><ul><li><p>QTN = quantitative trait nucleotide; associating single nucleotides with quantitative traits  <br/></p></li><li><p>Critical question: What is the molecular basis for adaptation? particularly in non-model organisms</p></li><li><p>Response to Rockman 2012 paper  <br/></p><ul><li><p>Rockman&#39;s major criticisms: Effect of nucleotide on traits can be overestimated  <br/></p><ul><li>large effect variants are rare and most complex traits are polygenic  <br/></li></ul></li></ul></li><li><p>Travisano and Shaw 2013 raise the criticism that you don&#39;t need QTN&#39;s to focus on the patterns and process of adaptation  <br/></p><ul><li>knowing molecular basis has not illuminated the proceess of adaptation!  <br/></li></ul></li><li><p>5 reasons why we should care about QTNs   <br/></p><ol start='' ><li>Vertical Integration: nucleotide to ecosystems      <br/></li><li>Parallel evolution and pleiotropy   <br/></li><li>Maintenance of standing genetic variation  <br/></li><li>Role of standing genetic variation in adaptation   <br/></li><li>Understanding the role of genomic architecture in adaptation    <br/></li></ol></li></ul></li></ol><hr /><p> <div id='id-section3'/><br/></p><h3><a name='header-n358' class='md-header-anchor '></a>Page 3: 2017-01-23. Day 2, course notes</h3><h3><a name='header-n359' class='md-header-anchor '></a>Course materials</h3><p>-Papers- info updates, and discussion papers; students need to sign up   	</p><p><strong>Add definitions as you see it</strong>  <br/></p><p><strong>Info update rubric</strong>  <br/></p><ul><li>Outline <br/></li><li>20 min <br/></li><li>Learning/engaging activity <br/></li><li>use board effectively <br/></li><li>take home messages  <br/></li><li>samples from the literature <br/></li></ul><h3><a name='header-n385' class='md-header-anchor '></a>Glossary:</h3><p><em>Reads</em>: a length of sequenced DNA    <br/></p><pre class='md-fences mock-cm' style='display:block;position:relative'>* short = 50 bps; long = 100, 150, 300, 10,000-60,000 bps     
* could be single or pair-end ( 1 strand or both strands)    </pre><h3><a name='header-n390' class='md-header-anchor '></a>Melissa info update:</h3><ol start='' ><li>Advances of sequencing technologies  <br/></li><li>Range of Applications <br/></li><li>General workflow (usually involving building libraries to be sequenced )  <br/></li><li>Sequencing by synthesis (SBS)   <br/></li><li>Other technologies of SBS  <br/></li><li>Learning activity  <br/></li></ol><h3><a name='header-n410' class='md-header-anchor '></a>1. Advances of sequencing technologies</h3><p>Good example was the human genome project: <br/></p><ul><li>finished in 2001 or 2003 or so <br/></li><li>used with sanger sequencing on ABI platform <br/></li><li>it took <strong>15 years of intense effort</strong> for 1 person&#39;s genome at the cost of $3 billion dollars</li></ul><p>Then, Illumina released Hi seq X Ten sequencer in 2014 <br/></p><ul><li><strong>In a single day, they could sequence 45 whole geomes for $,1000 for each!!!</strong> <br/></li><li>90% of global data now     <br/></li><li>Good example of how costs went down, but data went up!   <br/></li></ul><p>Illumina sequencing: <br/>
Similar to sanger but fluorescence can be observed across a whole &quot;field&quot;. <br/></p><h3><a name='header-n438' class='md-header-anchor '></a>2. Range of Applications</h3><ol start='' ><li>Whole genome sequencing  <br/></li><li>RNAseq  <br/></li><li>Chip-seq  <br/></li><li>Capture-sequencing; design probes and hybridize it with sample (it is targeted)      <br/></li></ol><h3><a name='header-n452' class='md-header-anchor '></a>3. General workflow (usually involving building libraries to be sequenced )</h3><p>Step 1: Goal-oriented; which platform to use? <br/></p><ul><li><p>where genetic variation is  <br/></p><ul><li>Phenotypes   <br/></li><li>number of samples</li><li>population  <br/></li><li>individual <br/></li><li>comparative study? closely related species?  <br/></li><li>model organism or not? <br/></li></ul></li><li><p>demographic history?   <br/></p></li><li><p>adaptive gentic variation   <br/></p></li><li><p>gene expression variation under common garden conditions? <br/></p></li></ul><p><strong>Trade-offs in doing the actual sequencing</strong> <br/></p><ul><li><p>length of reads   <br/></p><ul><li>long reads are easier to assemble(piece genomic regions together)     <br/></li></ul></li><li><h1><a name='header-n498' class='md-header-anchor '></a>of reads</h1><ul><li>10 million per individual?  target capture does not need too many reads  <br/></li></ul></li><li><p>how read are distributed along the whole genome    <br/></p></li></ul><p><strong>Actual workflow now</strong> </p><ol start='' ><li><p>Extract DNA or RNA(this needs to be converted into cDNA) <br/></p></li><li><p>Fragment sample (break it into smaller junks)  <br/></p></li><li><p>Ligate adapters on the ends </p><ul><li>Often barcoded to identify samples/individuals    <br/></li></ul></li><li><p>Add on sequencing adaptors    <br/></p></li><li><p>PCR amplification      <br/></p></li></ol><h3><a name='header-n528' class='md-header-anchor '></a>4. Sequencing by synthesis (SBS)</h3><ol start='' ><li><p>DNA with different adapters <br/></p></li><li><p>get loaded into 1 of 8 lanes in a flowcell <br/></p></li><li><p>flowcell have oligos that match the sequencing adaptors; so DNA attaches to it! (P5 and P7)  <br/></p></li><li><p>Bridge amplification: bend over and amplify, back and forth, over and over  <br/></p><ul><li>creates a clustered generation; sequence of the exact same sequence locally as a dot  <br/></li></ul></li><li><p>Incorporates labeled A T G or C, and then images are taken across the whole flowcell<br/></p></li></ol><p><strong>Pac-bio&#39;s platform: Single molecule real time sequencing (SMRT)</strong>   <br/></p><ol start='' ><li>DNA frag in a well, light penetrates in a small area to capture sequencing with long reads  <br/></li></ol><h3><a name='header-n555' class='md-header-anchor '></a>6. Learning activity</h3><ol start='' ><li>pair with neighbor <br/></li><li>share ah-ha moments <br/></li><li>discuss some useful applications  <br/></li></ol><h3><a name='header-n566' class='md-header-anchor '></a>Paper discussions  : Genome sequencing and population genomics in non-model organisms (Hans Ellegren )</h3><ul><li>Why would we use it ?  <br/></li><li>SK- we&#39;re generating tons of data, but what is the limit of space we can store it in ? (Short read archive)    <br/></li><li>NCBI, who curates the data? Wild west.    <br/></li><li>10-15 gbs for 1 file of in gzip format  <br/></li><li>Table 1; 2200 eukaryote genomes vs 600 listed in this paper   <br/></li><li>How do people choose which genome to sequence? politics <br/></li><li>Can people ID species based on WGS? <br/></li><li>Are there multiple references genomes assembled?  <br/></li><li>From a single individual, do people overlay as many omics as possible? Integrate proteome, PTMs, metabolome, phenotypes, etc  <br/></li></ul><h3><a name='header-n595' class='md-header-anchor '></a>The future! Moving forward</h3><ul><li><p>1 week from today, do a blitz on the different library preps (4 different ones)    <br/></p><ul><li>4 diff info updates  <br/></li></ul></li><li><p>2 volunteers to discuss QTN programme.  <br/></p></li><li><p>Each student, 1 discussion leader and 1 info update! <br/></p></li></ul><p> </p><hr /><p> <div id='id-section4'/><br/></p><h3><a name='header-n615' class='md-header-anchor '></a>Page 4: 2017-01-25. Week 2, Day 3, class notes (paper discussions and student project development)</h3><p><strong>Tasks</strong></p><p>One of my duties is to populate the glossary on blackboard. (I&#39;ll do it here for redundancy too.)</p><p></p><p><strong>Restructured course layout</strong>: </p><pre class='md-fences mock-cm' style='display:block;position:relative'>.
├── 01_data
├── 02_scripts
│   └── RasterPCA_demo.Rmd
├── 03_results_output
│   └── RasterPCA_demo.html
├── 04_tutorials
├── 2017_Ecological_Genomics.Rproj
├── ANBE_notebook.md
├── Online_notebook.md
├── README.md
├── index.Rmd
├── index.html
├── index.pdf
└── papers
    └── 01_week
        ├── Ellegren_2014.pdf
        ├── Lee_gould_stinchcombe_2014_AoB_PLANTS.pdf
        └── Rockman-2012-Evolution.pdf
 6 directories, 12 files</pre><p>It better reflects how the reading is done throughout the week.  <br/></p><h3><a name='header-n627' class='md-header-anchor '></a>Course outline</h3><ol start='' ><li><p>Announcements: </p><ul><li>BB acceess: add through audit</li><li>Sign-ups <br/></li><li>Glossary: Whoever is doing info update must come up with a list of key terms!    <br/></li></ul></li><li><p>Info update ~ QTN <br/></p></li><li><p>Debate-style discussion </p></li><li><p>BREAK <br/></p></li><li><p>Group project discussion! <br/></p></li></ol><h3><a name='header-n654' class='md-header-anchor '></a>1. Info update ~ QTN</h3><p><strong>Q: What are QTN&#39;s (Quantitative Trait Nucleotide)?</strong> <br/></p><ul><li><p><strong>Quantitative genetic theory of adaptive traits</strong> <br/></p><ul><li>Additive genetic variance and heritability </li></ul></li></ul><p>QTN is the most reductionist you can get. The individual SNPs that contribute to the variation in a trait. Usually traits under selection and confer adaptation.<br/></p><p>Traits!</p><pre class='md-fences mock-cm' style='display:block;position:relative'>1. flowering time
2. flower color
3. thermal tolerance 
4. venom potency
5. defense or secondary compounds in plants  
6. toxin tolerance  
7. drought tolerance 
8. altitude tolerance (hypoxia)</pre><p>They are quantitative, and are continuous. They have a mean and variance. <strong>Not descrete phenotypes</strong>. Discrete phenotypes are mendelian (major effect loci) . <br/></p><p>Modern synthesis (Fischer, Haldane, WRight).  Connect alleles with trait distribution. </p><p>ex:  allele:trait</p><p>AA = 1</p><p>Aa = 2</p><p>aa = 3</p><p>The difference between Aa and aa = average effects or alpha (1).</p><p>AA = q^2 </p><p>Aa = 2pq</p><p>aa= p^2 </p><p>Additive genetic variance (Va) = sum of alpha * pi and qi.   pi and qi </p><p>Vp = phenotypic variance and the Va is only a fraction. So the heritability is Va/Vp.<br/></p><p></p><p><strong>What are the genes that explain this heritability? Alpha is the effect size of the QTN</strong></p><p>Most populations are close to their adaptive peak. Imagine fitness as a fucntion of a trait, normal distribution.  Max fitness = intermediate trait is local adaptation. Change in environment will change the fitness peak. </p><p>A mutation is random, and it can move the fitness of the population up or down. This is usally a small effect mutation. But if you have a large effect mutation, then changes are, you will move down, maintain, or up, but if it pushes too far, it&#39;ll go down. It is must more effecient for the population to evolve many small effect loci.  How do we detect this?</p><p><strong>3 main methods</strong></p><ol start='' ><li><p><strong>QTL mapping (forward genetics)</strong></p><ul><li>Segregating sites between 2 individuals (red vs blue) and they&#39;re diploid</li><li>They&#39;re homozygous (2 extremes of trait)</li><li>Take parents, cross to make F1—that are heterozygous</li><li>Take F1s, and do a series of cross to produce F2</li><li>The chromosomes that vary in the ancestry of different blocks(chromosome)</li><li>The QTL or QTN is unknown (trait based forward genetics approach)</li><li>Markers (microsats) and unlinks them (because of the crossing) </li><li>By multiple rounds of generations, then you get variation in loci size</li></ul></li><li><p><strong>GWAS (Genome-wide association studies) (forward genetics)</strong></p><ul><li><p>Let nature do the crosses, and sample a bunch of individuals</p></li><li><p>genotype them, </p></li><li><p>Nature has been doing a similar experiment as in the QTL mapping but there are many more parents</p></li><li><p>Model: Y = u + Bi x SNPi + Covariates</p><ul><li>Y = Trait </li><li>u = intercept</li><li>Bi = effect size</li><li>SNPi= a particular SNP</li><li>Covariates = population structure </li></ul></li><li><p>You get a manhattan plot, plotting -log(pvalue) against position</p><ul><li>High values indicate significant SNPs </li><li>usually includes several genes although it will involve many genes</li></ul></li></ul></li><li><p><strong>Selection Scans (reverse genetics)</strong></p><ul><li><p>don&#39;t know trait, just zoom in on parts of the genome that have history of selection</p></li><li><p>Sample a bunch of individuals, with a bunch of SNPs<br/></p></li><li><p>Selective sweep, some rise in frequency and others are not<br/></p></li><li><p>Over T generations, it could fix—DECREASING diversity </p><ul><li>iF this varies among populations, then this could indicate selection and divergence (FST)</li><li>data shows us what genes are important when we did not know <em>a priori</em></li></ul></li></ul></li></ol><p>Additional notes: Mutations before selection are equal across loci and their effect sizes. So after selection, the frequency of effect sizes are concentrated on the small values(small effect sizes) and very little at the large effect size.<br/></p><h3><a name='header-n800' class='md-header-anchor '></a>3. Debate-style discussion (&quot;QTN programme&quot;)</h3><p>2 separate groups discussions first.</p><p><strong>Dr. Brody leading Rockman paper</strong>: </p><ul><li>analogy: panning for gold, large nuggets already found, and now we&#39;re finding needles in a haystack </li><li>large effect QTLs, very few cases found, but that is not most of them<br/></li></ul><p>3 main arguments</p><ol start='' ><li>LARGE effect QTLs are mendelian </li><li>Theory does </li><li>searching for QTNs may act in the same way as large effect QTL </li></ol><p>Speciation example: Doug Schemsky</p><p>They speciated because they got different pollination syndromes (changed from yellow to red). It acted like a mendelian trait. Is it important in knowning speciation?  Is it common? Speciation can be neutral just by building up different genetic interactions that leads to reproductive isolation. </p><p><strong>Whole group Discussion</strong> <br/></p><p></p><h3><a name='header-n832' class='md-header-anchor '></a>5. Student Project development</h3><p>Melissa&#39;s dataset</p><p>Seastar wasting disease: kills stuff, <strong>PATHOGEN UNKNOWN</strong> </p><ul><li><p>Some species are resistant and some rae susceptible</p></li><li><p>it is a generalist </p></li><li><p>A couple days or even hours that a host goes from healthy to sick</p></li><li><p>Sickness comes in the form of losing arms, lose turgor pressure, they become droopy</p><ul><li>start off with legions and/or loss of turgor pressure</li><li>loss of limbs = gravity and/or behavioral (when they move,t heir body doesn&#39;t stay together)</li><li>turn into a puddle of white goo; ghostly shapes of sea stars ; so sad</li></ul></li><li><p><strong>Potentially Densovius is causal</strong> (maybe)</p></li><li><p>What factors affect the tipping point? </p></li><li><p>Sampled epidermal biopsy</p><ul><li>total RNA isolation, polyA tail selection to get mRNA</li><li>300 million paired end bp sequencing</li><li>took out 16s rRNA </li></ul></li></ul><p><strong>Hypotheses</strong> </p><p>Theme: </p><ol start='' ><li><p>What is the genetic difference related to susceptibility</p><ul><li>Int or sub tidal are more susceptible </li><li>int or sub differ in gene expression </li></ul></li><li><p>What is the genetic basis for disease resistance? </p><ul><li>positive selection on genes and how they&#39;re related to health/symptoms</li></ul></li><li><p>How does the microbiome contribute to seastar wasting? </p><ul><li><p>microbiome differ among disease/symptoms</p><ul><li>diversity? abundance?</li></ul></li></ul></li><li><p>Is there heritable genetic variation in the microbiome? </p></li></ol><p>Course notes for next week:</p><ol start='' ><li><p>put picture of all of our hypotheses on the screen</p></li><li><p>4 corners of the room, put interests and groups will assemble</p></li><li><p>give half of class to think through and other half to present ideas </p></li><li><p>Make students formulate questions, hypotheses, predictions, what samples</p></li><li><p>Technical questions: construct assembly together? separate? Give students a canonical reference (transcriptome). let students investigate if they want to separate them. </p></li><li><p>Melissa and Melanie will do the assembly on the side. <br/></p></li><li><p>AN will be added as adminstrator access (permission changes and stuff); software installed, </p><ul><li>add trinity</li><li>AN can install </li></ul></li><li><p>Wednesday, hands on logging in on the server, Unix command lines<br/></p><ul><li>have shell, have putty</li><li>connect remote machine</li><li>set up home directory</li><li>move around in the machine</li><li>regular expression </li><li>moving , cutting, copying files</li><li>transfer between server and client</li></ul></li><li><p>Look at wiki to see what we can add or subtract in terms how what commands to show<br/></p></li></ol><p>Set up new webpage for course: </p><ol start='' ><li>links to tutorials</li><li>scripts</li><li>page with class members with hyperlinks to their github notebooks </li><li>(<strong>Getting set up link</strong>)New tab with resources for markdown and online notebook and typora</li><li>Glossary link to blackboard</li></ol><h3><a name='header-n992' class='md-header-anchor '></a>Changing the course material to construct a website:</h3><pre class='md-fences mock-cm' style='display:block;position:relative'>.
├── 04_tutorials
├── 2017_Ecological_Genomics.Rproj
├── ANBE_notebook.html
├── ANBE_notebook.md
├── Class_members.Rmd
├── Class_members.html
├── Getting_started.Rmd
├── Getting_started.html
├── Instructors.Rmd
├── Instructors.html
├── Online_notebook.md
├── PBIO.BIO381_Spring2017_syllabus.pdf
├── README.md
├── Tutorials.Rmd
├── Tutorials.html
├── index.Rmd
└── index.html

1 directory, 16 files
</pre><p></p><p></p><pre class='md-fences mock-cm' style='display:block;position:relative'></pre><p></p><hr /><p> <div id='id-section5'/><br/></p><h3><a name='header-n1004' class='md-header-anchor '></a>Page 5: 2017-01-30. Week 3, Day 4, class notes , Group presentations of project ideas</h3><p></p><p>Melissa gave handoout for a general formula. This may help articulate and motivate and objectives  of the eco genom project. this approach can be applied to your other research projects, papers, or grants!</p><p></p><ol start='' ><li><strong>The opener:</strong> capture the attention of your audience highlighting an important area of research</li><li><strong>Current knowledge:</strong> What is known about this area and will facilitate your work? </li><li><strong>The Gap statement:</strong> Where is the gap in the understanding? The critical missing bit of knowledge that prevents forward scientific progress? </li><li><strong>Objectives:</strong> Articulate the objectives of your research- &quot;Here we...&quot;</li><li><strong>Central hypothesis:</strong> null and alternative </li><li><strong>How will you test this hypothesis?</strong> How will you distinguish between alternative explanations, will you have the statistical power to do so? </li><li><strong>What are your expected results/outcomes?</strong> </li></ol><p>Steve will be writing up projects on the board: </p><ol start='' ><li><p><strong>Immune-related gene expression</strong> </p><ul><li><p>reverse pathology</p></li><li><p>looking specific classes of genes</p></li><li><p>a priori tests for resistance genes</p><ul><li>compare individuals that stayed healthy vs those that got sick</li><li>looking at S-H transition </li></ul></li><li><p><strong>Group(Name = Sherlock):</strong> Erin, Sam, Alex, Lauren, Dr. Brody, and </p><ul><li><p>Interested in reverse pathology</p></li><li><p>What type of pathogens are causing immune response differences? </p><ul><li><p>Viral specific, fungal specific, or bacterial specific</p></li><li><p>Focus on the transitions: HS, HH, SS</p><ul><li><p>blocking based on time </p></li><li><p>what is the workflow? </p><ul><li>​</li></ul></li></ul></li></ul></li><li><p>Back up Q: comapre responses to another species. </p></li><li><p>Use random group of genes to use as housekeeping and control</p></li><li><p>make heat map or venn diagram of differential gene expression </p></li><li><p>​</p></li></ul></li></ul></li><li><p><strong>Intertidal vs subtidal</strong> </p><ul><li><p>Genetic differences in susceptibility</p></li><li><p>local adaptation? </p></li><li><p>gene expression differences</p></li><li><p><strong>Group discussion(Name=):</strong> Lauren, Laura, Kattia, Dr. K</p><ul><li><p>subtidal more susceptible than intertidal. What differences contribute to that? </p></li><li><p>Focus on GXP expression between the two groups: ID, genes that are more diff expressed and do a functional enrichment analysis (immune vs general stress response; tease apart handling and susceptibility)</p></li><li><p>Focus on community structure of micro biome between the two groups: subtotal = more diversity? Equals more resilient to pathogen? Specific taxa associated with disease? </p><ul><li>Candidate genes in stress response? They want to do  a broad survey of genes (generalized). 3 broad categories: <strong>stress, immune, other</strong>. </li><li>Time point? Use first time point (day3 ); potentially comparing healthy vs sick </li><li>Not looking at SNP level differences </li><li>Potentially associate gxp with micro biome ; mantel test. </li></ul></li></ul></li></ul></li><li><p><strong>Temporal Variation</strong> </p><ul><li><p>chagnes in gene expression through time</p></li><li><p>changes in microbiome through time</p></li><li><p>temporal differences in H vs S</p></li><li><p><strong>Group discussion (Name=):</strong></p><ul><li><p>change of gxp through time: compare HH vs HS (that made it to day 15)</p><ul><li>ignore SS </li><li>for each time point, what genes are differentially expressed between HH and HS</li><li>ID&#39;ing those genes </li></ul></li><li><p>What is causing sickness? and how organisms are affected by it?(this is focus)</p></li><li><p>stability of gxp for HH vs HS; House keeping genes? </p></li><li><p>Which ones are varying over time? </p></li><li><p><strong>functional enrichment for varying or stable through time</strong></p></li><li><p>​</p></li></ul></li></ul></li><li><p><strong>Heritability of microbiome</strong></p><ul><li>compare microbial commuity to host individual relatedness</li></ul></li><li><p><strong>Comparison within the intertidal group</strong></p><ul><li><p>genetic differences ( 3 gropus of individual)</p></li><li><p>delta in microbiome</p></li><li><p><strong>Group discussion (Name=Rising starfish ):</strong></p><ul><li><p>Intertidal group only: control for the handling stress</p></li><li><p>Want to do is access genomic differences (SNP data) between individuals that stayed healthy. HH, vs Sick</p><ul><li>Differences between intertidal and subtidal </li></ul></li><li><p>Genetic basis with respect to susceptibilty </p></li><li><p>Look at micro biome: microbiome composition of Healthy vs Sick</p></li><li><p>Microbiome changes across each time point for all of the individuals </p></li><li><p>one specific group of microbes that are found in healthy that are lacking in the sick(indicator taxa)</p><ul><li>aides in resilience or susceptibility </li></ul></li><li><p>Thinking about all time points(or we can start first day) </p></li><li><p>How many OTUs? &gt; 100s</p></li><li><p>Rare taxa might drop out? </p></li><li><p>Finding diffs</p><ul><li>check allele frequency differences among sick and healthy </li><li>Can do PCA  or discriminant analysis (DAPC)</li><li>FST method</li></ul></li></ul></li></ul></li></ol><h3><a name='header-n1249' class='md-header-anchor '></a>Assignment:</h3><p><em>As a group, write up 1 page proposal following the guideline above. Integrate feedback. Due next Monday. We will give additional and formal feedback in the form of writing.</em> </p><p>Email, MS word</p><p>List all the exact sample names too</p><p></p><p><strong>For wednesday:</strong></p><p><strong>First half-</strong> We&#39;re going to do a bioblitz of library prep types. Learn more about how libraries are made and what data they produced: WGS, RNA-seq, GBS(rad-seq), Amplicon-seq. No discussion, just 4 INFO UPDATES(20 minutes each).</p><p></p><p><strong>Second half</strong>- logging into the server. Unix command line stuff. Bring your computer. Have software installed (Putty in Windows). File transfer client </p><h3><a name='header-n1266' class='md-header-anchor '></a>Admin stuff</h3><p></p><p><strong>Logging into the cluster:</strong> </p><p></p><pre class='md-fences mock-cm' style='display:block;position:relative'>ssh adnguyen@pbio381.uvm.edu</pre><p></p><p>Going in to the root: </p><p></p><pre class='md-fences mock-cm' style='display:block;position:relative'>[adnguyen@pbio381 ~]$ cd /
[adnguyen@pbio381 /]$ ls
autorelabel  boot  dev  home  lib64  mnt  opt   root  sbin  sys  users  var
bin          data  etc  lib   media  nsr  proc  run   srv   tmp  usr</pre><p></p><p>Stuff is stored on the data/ directory.  Let&#39;s look inside</p><pre class='md-fences mock-cm' style='display:block;position:relative'>
[adnguyen@pbio381 /]$ cd data/
[adnguyen@pbio381 data]$ ls
databases     packages  project_data  temp_data
example_data  popgen    scripts       users
</pre><p><strong>Notes</strong>: Stuff that needs to be installed, goes into the popgen folder</p><p></p><h3><a name='header-n1290' class='md-header-anchor '></a>Repeated measures anova or time series of differential gene expression</h3><p>We need to figure out how to do this. </p><p></p><p></p><hr /><p> <div id='id-section6'/><br/></p><h3><a name='header-n1300' class='md-header-anchor '></a>Page 6: 2017-02-01. Week 3, Day 5, info-updates; command line unix tutorial</h3><h2><a name='header-n1301' class='md-header-anchor '></a>Announcements</h2><ol start='' ><li><p><usStarters</u><br/></p><ul><li>Send link to Github notebook to Antdrew (<a href='mailto:adnguyen@uvm.edu' target='_blank' >adnguyen@uvm.edu</a>)</li><li>sign-ups (folkds sitting in , too!)</li><li>Project proposals due by email next <mark>Monday February 6th.</mark></li><li>We start transcriptomics next week!</li></ul></li><li><p><u>Info-update Blitz</u><br/></p></li><li><p><u>Unix tutorial</u><br/></p></li></ol><h3><a name='header-n1325' class='md-header-anchor '></a><u>2. Info Updates</u></h3><ul><li><p><u>Whole genome sequencing</u><br/></p><ul><li><p><strong>applications</strong></p><ul><li><p>high power and high resolution for pop gen, effective pop size, genetic relatedness, inbreeding, admixture events, conservations(monitoring or control breeding)</p></li><li><p>new ones too! screen for adaptive potential, inbreeding depression, impacts of genetic variation, plastic responses</p></li><li><p>Needs $, computational expertise (cluster, command line, python/perl)</p></li><li><p><mark>Limitations</mark></p><ul><li>Polymorphic genes: core genes that are highly conserved</li><li>Paralog </li><li>Rapidly evolving genes (poor representation)</li><li>Large gene families</li><li>Using 1 individual (does not capture the diversity of the species)</li><li>Pool samples is a possibility</li><li><strong>Not really whole genome!!</strong>; some parts cant be sequenced, heterochromatic regions and highly repetitive regions</li></ul></li></ul></li><li><p><strong>Prior expectations</strong></p><ul><li><p>Reference genome? </p><ul><li>No = de novo assembly</li><li>Yes = map reads to it</li></ul></li></ul></li><li><p><strong>Methods</strong></p><ul><li><p>Platforms:</p><ul><li>Short reads: Illumina (150bp), Solid (50bp)</li><li>Longer reads: Pacific bioscience (5kb); Ion torrent (~500 bps)</li></ul></li><li><p><u>Knowing your organism!</u><br/></p><ul><li><p>Genome size! (K-mer approach: unique element of DNA seq length)</p></li><li><p>Know repetitive content and error rates of sequencing</p><ul><li>GC content too</li></ul></li><li><p>Degree of Genome duplication</p></li></ul></li></ul></li></ul></li></ul><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart0" width="395.4000015258789" xmlns="http://www.w3.org/2000/svg" height="172" viewBox="0 0 415.4000015258789 192" style="height: 232px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M105.79574241528537,69.39574165234592L161.9000015258789,40.5L201.4000015258789,40.5" marker-end="url(#arrowhead10)" style="fill:none"></path><defs><marker id="arrowhead10" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M105.79574139915307,103.60425783790745L161.9000015258789,131.5L239.9000015258789,131.5" marker-end="url(#arrowhead11)" style="fill:none"></path><defs><marker id="arrowhead11" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="translate(161.9000015258789,40.5)" style="opacity: 1;"><g transform="translate(-9.5,-10.5)" class="label"><foreignObject width="19" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">No</span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(161.9000015258789,131.5)" style="opacity: 1;"><g transform="translate(-14.5,-10.5)" class="label"><foreignObject width="29" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">Yes</span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="A" transform="translate(71.20000076293945,86)" style="opacity: 1;"><polygon points="51.2,0 102.4,-51.2 51.2,-102.4 0,-51.2" rx="5" ry="5" transform="translate(-51.2,51.2)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-33.5,-10.5)"><foreignObject width="67" height="21"><div style="display: inline-block; white-space: nowrap;">Genome?</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(288.4000015258789,40.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-87" y="-20.5" width="174" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-77,-10.5)"><foreignObject width="154" height="21"><div style="display: inline-block; white-space: nowrap;">De novo assembly</div></foreignObject></g></g></g><g class="node" id="C" transform="translate(288.4000015258789,131.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-48.5" y="-20.5" width="97" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-38.5,-10.5)"><foreignObject width="77" height="21"><div style="display: inline-block; white-space: nowrap;">YOu good</div></foreignObject></g></g></g></g></g></g></svg></div><p><strong>More methods:</strong> </p><ul><li><p>Labby stuff </p><ul><li><p>high quality, avoid energetic active  tissue (mitochondria may mess up depth)</p></li><li><p>avoid gut and skin</p></li><li><p>Quantity: 1 -6 ug(short)</p></li><li><p>Library prep</p><ul><li>single pair, paired end or mate-pair (shotgun sequencing) </li><li>assemble contigs and map to scaffolds</li></ul></li></ul></li><li><p>Computational stuff</p><ul><li>De novo, use algorithms to maximize accuracy in assembly</li><li>Annotate; automate, manually</li></ul></li></ul><ul><li><p><strong>RNA-seq</strong></p><ul><li><p>Advantages: </p><ul><li>Diff gene expression</li><li>Allele-specific expression from environmental response or adaptive significance</li><li>Gives reliable and relevent subset of the genome</li><li>Wider dynamic range than micro-arrays</li><li>Can give info on splicing events</li></ul></li><li><p><mark>Limitations</mark></p><ul><li>some transcripts may not be reflected in the protein abundance</li></ul></li><li><p>Workflow</p><ul><li><p><u>Considerations:</u><br/></p><ul><li><p>prot coding or regulatory non coding?</p></li><li><p>ref genome?</p></li><li><p>alt splicing?</p></li><li><p>tech?</p></li><li><p>population or treatment specific </p></li><li><p>Choice of tissue type</p><ul><li>Small organisms- pool organisms</li></ul></li><li><p>Biological replicates!?</p></li></ul></li><li><p><u>Wet lab</u><br/></p><ul><li>RNASE free; DNASE free (nuclease free)</li><li>Get rid of ribosomal RNA; so enrich for poly-AAA tails</li><li>Single end or paired end </li></ul></li><li><p><u>Sequencing platforms</u><br/></p><ul><li>Roche 454 sequencing - out of date</li><li>Hi-seq by Illumina</li><li>Think about error profiles</li><li>coverage (100 million bps with  &gt; 100 bps per read )</li></ul></li><li><p><u>Computational</u><br/></p><ul><li>Programming: Unix , python, R</li><li>De novo assembly: consruct contigs (stretches of RNA)</li><li>Map to reference genome!</li></ul></li></ul></li></ul></li></ul><p></p><p></p><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart1" width="436" xmlns="http://www.w3.org/2000/svg" height="627" viewBox="0 0 456 647" style="height: 687px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M218,61L218,86L218,111" marker-end="url(#arrowhead43)" style="fill:none"></path><defs><marker id="arrowhead43" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M218,152L218,177L218,202" marker-end="url(#arrowhead44)" style="fill:none"></path><defs><marker id="arrowhead44" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M218,243L218,268L218,293" marker-end="url(#arrowhead45)" style="fill:none"></path><defs><marker id="arrowhead45" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M218,334L218,359L218,384" marker-end="url(#arrowhead46)" style="fill:none"></path><defs><marker id="arrowhead46" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M218,425L218,450L218,475" marker-end="url(#arrowhead47)" style="fill:none"></path><defs><marker id="arrowhead47" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M218,516L218,541L218,566" marker-end="url(#arrowhead48)" style="fill:none"></path><defs><marker id="arrowhead48" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="A" transform="translate(218,40.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-159.5" y="-20.5" width="319" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-149.5,-10.5)"><foreignObject width="299" height="21"><div style="display: inline-block; white-space: nowrap;">Purpose and Experimental Set up</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(218,131.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-111" y="-20.5" width="222" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-101,-10.5)"><foreignObject width="202" height="21"><div style="display: inline-block; white-space: nowrap;">Tissue Prep + Library</div></foreignObject></g></g></g><g class="node" id="C" transform="translate(218,222.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-135" y="-20.5" width="270" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-125,-10.5)"><foreignObject width="250" height="21"><div style="display: inline-block; white-space: nowrap;">High throughput sequencing</div></foreignObject></g></g></g><g class="node" id="D" transform="translate(218,313.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-178.5" y="-20.5" width="357" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-168.5,-10.5)"><foreignObject width="337" height="21"><div style="display: inline-block; white-space: nowrap;">Transcriptome assembly construction</div></foreignObject></g></g></g><g class="node" id="E" transform="translate(218,404.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-68" y="-20.5" width="136" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-58,-10.5)"><foreignObject width="116" height="21"><div style="display: inline-block; white-space: nowrap;">Align reads!</div></foreignObject></g></g></g><g class="node" id="F" transform="translate(218,495.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-198" y="-20.5" width="396" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-188,-10.5)"><foreignObject width="376" height="21"><div style="display: inline-block; white-space: nowrap;">Read quantification, marker development</div></foreignObject></g></g></g><g class="node" id="G" transform="translate(218,586.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-111" y="-20.5" width="222" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-101,-10.5)"><foreignObject width="202" height="21"><div style="display: inline-block; white-space: nowrap;">Biological inferences</div></foreignObject></g></g></g></g></g></g></svg></div><ul><li><p>** <u>Amplicon-seq</u><br/>
** (Hannah): 16s rRNA as example - IDs prokaryotes such as bacteria (18s r RNA, fungi, protozoa)</p><ul><li><p>Methods</p><ul><li><p>Library prep </p><ul><li>extract —&gt; check quality and quantity —&gt; pcr with specific primers</li></ul></li><li><p>sequencing</p><ul><li>200-600 bps</li></ul></li><li><p>data analysis</p><ul><li>Learning activity</li></ul></li></ul></li><li><p>Applications</p></li></ul></li></ul><p><mark>Workflow</mark></p><p></p><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart2" width="611" xmlns="http://www.w3.org/2000/svg" height="956.8000030517578" viewBox="0 0 631 976.8000030517578" style="height: 1016.8000030517578px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M301.25,154.0999969482422L300.75,178.60000610351562L300.75,203.60000610351562" marker-end="url(#arrowhead87)" style="fill:none"></path><defs><marker id="arrowhead87" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M300.75,244.60000610351562L300.75,269.6000061035156L300.75,294.6000061035156" marker-end="url(#arrowhead88)" style="fill:none"></path><defs><marker id="arrowhead88" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M300.75,335.6000061035156L300.75,360.6000061035156L300.75,385.6000061035156" marker-end="url(#arrowhead89)" style="fill:none"></path><defs><marker id="arrowhead89" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M230.80219780219778,426.6000061035156L145.5,451.6000061035156L145.5,476.6000061035156" marker-end="url(#arrowhead90)" style="fill:none"></path><defs><marker id="arrowhead90" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M370.6978021978022,426.6000061035156L456,451.6000061035156L456,476.6000061035156" marker-end="url(#arrowhead91)" style="fill:none"></path><defs><marker id="arrowhead91" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M145.5,517.6000061035156L145.5,542.6000061035156L230.80219780219778,567.6000061035156" marker-end="url(#arrowhead92)" style="fill:none"></path><defs><marker id="arrowhead92" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M456,517.6000061035156L456,542.6000061035156L370.6978021978022,567.6000061035156" marker-end="url(#arrowhead93)" style="fill:none"></path><defs><marker id="arrowhead93" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M300.75,608.6000061035156L300.75,633.6000061035156L300.75,658.6000061035156" marker-end="url(#arrowhead94)" style="fill:none"></path><defs><marker id="arrowhead94" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M300.75,699.6000061035156L300.75,724.6000061035156L301.25,750.1000076293947" marker-end="url(#arrowhead95)" style="fill:none"></path><defs><marker id="arrowhead95" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="A" transform="translate(300.75,86.80000305175781)" style="opacity: 1;"><polygon points="66.8,0 133.6,-66.8 66.8,-133.6 0,-66.8" rx="5" ry="5" transform="translate(-66.8,66.8)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-53,-10.5)"><foreignObject width="106" height="21"><div style="display: inline-block; white-space: nowrap;">Extract DNA</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(300.75,224.10000610351562)" style="opacity: 1;"><rect rx="5" ry="5" x="-135" y="-20.5" width="270" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-125,-10.5)"><foreignObject width="250" height="21"><div style="display: inline-block; white-space: nowrap;">check quality and quantity</div></foreignObject></g></g></g><g class="node" id="C" transform="translate(300.75,315.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-130.5" y="-20.5" width="261" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-120.5,-10.5)"><foreignObject width="241" height="21"><div style="display: inline-block; white-space: nowrap;">PCR with specific primers</div></foreignObject></g></g></g><g class="node" id="D" transform="translate(300.75,406.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-106.5" y="-20.5" width="213" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-96.5,-10.5)"><foreignObject width="193" height="21"><div style="display: inline-block; white-space: nowrap;">Clean up PCR product</div></foreignObject></g></g></g><g class="node" id="Z" transform="translate(145.5,497.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-125.5" y="-20.5" width="251" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-115.5,-10.5)"><foreignObject width="231" height="21"><div style="display: inline-block; white-space: nowrap;">Run gel and cut out band</div></foreignObject></g></g></g><g class="node" id="T" transform="translate(456,497.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-135" y="-20.5" width="270" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-125,-10.5)"><foreignObject width="250" height="21"><div style="display: inline-block; white-space: nowrap;">Or run it through a column</div></foreignObject></g></g></g><g class="node" id="Y" transform="translate(300.75,588.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-183.5" y="-20.5" width="367" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-173.5,-10.5)"><foreignObject width="347" height="21"><div style="display: inline-block; white-space: nowrap;">Second PCR: barcode and add adaptors</div></foreignObject></g></g></g><g class="node" id="N" transform="translate(300.75,679.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-149.5" y="-20.5" width="299" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-139.5,-10.5)"><foreignObject width="279" height="21"><div style="display: inline-block; white-space: nowrap;">Clean again and check quality</div></foreignObject></g></g></g><g class="node" id="L" transform="translate(300.75,843.2000045776367)" style="opacity: 1;"><polygon points="93.60000000000001,0 187.20000000000002,-93.60000000000001 93.60000000000001,-187.20000000000002 0,-93.60000000000001" rx="5" ry="5" transform="translate(-93.60000000000001,93.60000000000001)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-86.5,-10.5)"><foreignObject width="173" height="21"><div style="display: inline-block; white-space: nowrap;">Nex-gen Sequencing</div></foreignObject></g></g></g></g></g></g></svg></div><ul><li>Data analysis ; where you spend most of your time!</li></ul><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart3" width="214" xmlns="http://www.w3.org/2000/svg" height="469.8000030517578" viewBox="0 0 234 489.8000030517578" style="height: 529.8000030517578px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M107.5,177.2999984741211L107,201.8000030517578L107,226.8000030517578" marker-end="url(#arrowhead112)" style="fill:none"></path><defs><marker id="arrowhead112" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M107,267.8000030517578L107,292.8000030517578L107,317.8000030517578" marker-end="url(#arrowhead113)" style="fill:none"></path><defs><marker id="arrowhead113" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M107,358.8000030517578L107,383.8000030517578L107,408.8000030517578" marker-end="url(#arrowhead114)" style="fill:none"></path><defs><marker id="arrowhead114" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="A" transform="translate(107,98.4000015258789)" style="opacity: 1;"><polygon points="78.4,0 156.8,-78.4 78.4,-156.8 0,-78.4" rx="5" ry="5" transform="translate(-78.4,78.4)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-67.5,-10.5)"><foreignObject width="135" height="21"><div style="display: inline-block; white-space: nowrap;">Take raw reads</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(107,247.3000030517578)" style="opacity: 1;"><rect rx="5" ry="5" x="-19.5" y="-20.5" width="39" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-9.5,-10.5)"><foreignObject width="19" height="21"><div style="display: inline-block; white-space: nowrap;">QC</div></foreignObject></g></g></g><g class="node" id="C" transform="translate(107,338.3000030517578)" style="opacity: 1;"><rect rx="0" ry="0" x="-72.5" y="-20.5" width="145" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-62.5,-10.5)"><foreignObject width="125" height="21"><div style="display: inline-block; white-space: nowrap;">Trim adaptors</div></foreignObject></g></g></g><g class="node" id="D" transform="translate(107,429.3000030517578)" style="opacity: 1;"><rect rx="5" ry="5" x="-87" y="-20.5" width="174" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-77,-10.5)"><foreignObject width="154" height="21"><div style="display: inline-block; white-space: nowrap;">Align Sequences!</div></foreignObject></g></g></g></g></g></g></svg></div><p></p><ul><li><p>Applications:</p><ul><li>Known sequences saves you time</li><li>ID species </li></ul></li></ul><p> <u>Glossary</u><br/></p><p>Amplicon-seq: targeted approach for analyzing genetic variation in a specifc genomic region </p><p>Amplicon: Targeted gene (region) to abe amplified via pca with specific primers</p><p></p><ul><li><ul><li><u>GBS or Rad-seq</u><br/>
**</li></ul></li></ul><p>There is a continuum in how much sequence you can get (completeness of samples):</p><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart4" width="435" xmlns="http://www.w3.org/2000/svg" height="112" viewBox="0 0 455 132" style="height: 172px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M92.5,56.5L117,56L142,56" marker-end="url(#arrowhead126)" style="fill:none"></path><defs><marker id="arrowhead126" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M229,56L254,56L279,56" marker-end="url(#arrowhead127)" style="fill:none"></path><defs><marker id="arrowhead127" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="A" transform="translate(56,56)" style="opacity: 1;"><polygon points="36,0 72,-36 36,-72 0,-36" rx="5" ry="5" transform="translate(-36,36)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-14.5,-10.5)"><foreignObject width="29" height="21"><div style="display: inline-block; white-space: nowrap;">WGS</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(185.5,56)" style="opacity: 1;"><rect rx="5" ry="5" x="-43.5" y="-20.5" width="87" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-33.5,-10.5)"><foreignObject width="67" height="21"><div style="display: inline-block; white-space: nowrap;">RNA-seq</div></foreignObject></g></g></g><g class="node" id="Amplicon-seq" transform="translate(347,56)" style="opacity: 1;"><rect rx="0" ry="0" x="-68" y="-20.5" width="136" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-58,-10.5)"><foreignObject width="116" height="21"><div style="display: inline-block; white-space: nowrap;">Amplicon-seq</div></foreignObject></g></g></g></g></g></g></svg></div><ul><li><p>Trade off between sampling and completeness. </p></li><li><p>GBS falls between RNA-seq and Amplicon-Seq. </p></li><li><p>Good for population geneticists</p></li><li><p>Goals:</p><ul><li>Lots of individuals</li><li>Lots of SNPs across the genome</li><li>Don&#39;t care about specific genes</li><li>Dont need complete genome sequence</li></ul></li><li><p>Genotyping-by-sequencing(GBS) or Restriction Assisted DNA -seqencing (RAD-seq)</p></li><li><p>Cut up with restriction enzymes,</p></li><li><p>Sequence with single ends</p></li></ul><h3><a name='header-n1676' class='md-header-anchor '></a>3. Tutorials reference <a href='https://adnguyen.github.io/2017_Ecological_Genomics/Tutorials.html'>page</a></h3><ul><li><p>Cluster (Shared resource): </p><ul><li>24 CPU cores! </li><li>32 GB of RAM</li><li>maintained by IT</li><li>1 TB harddrive </li><li>THink about other people</li></ul></li><li><p>Logging into the server: </p><ul><li>Everybody has a home directory (~/)</li></ul></li></ul><ul><li>Finding out where you are?</li></ul><pre class='md-fences mock-cm' style='display:block;position:relative'></pre><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">[adnguyen@pbio381 ~]$ pwd</span></pre></div><pre class=""><span style="padding-right: 0.1px;">/users/a/d/adnguyen</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 42px;"></div></div></div></pre><p></p><ul><li>copying the data file</li></ul><pre class='md-fences mock-cm' style='display:block;position:relative'>cp /data/project_data/ssw_samples.txt .</pre><hr /><p> <div id='id-section7'/><br/></p><h3><a name='header-n1723' class='md-header-anchor '></a>Page 7: 2017-02-03. Installing trinity into the cluster</h3><p>Get the tar and zip file here: <a href='https://github.com/trinityrnaseq/trinityrnaseq/releases' target='_blank' >https://github.com/trinityrnaseq/trinityrnaseq/releases</a>; </p><p><a href='https://github.com/trinityrnaseq/trinityrnaseq/wiki/Installing%20Trinity'>Installation instructions</a></p><p></p><ol start='' ><li><p>Working directory: /data/popgen</p></li><li><p>Used &quot;wget&quot;</p><ul><li>Downloading the file: </li></ul></li></ol><pre class='md-fences mock-cm' style='display:block;position:relative'>			wget &quot;https://github.com/trinityrnaseq/trinityrnaseq/archive/Trinity-v2.3.2.tar.gz&quot;</pre><ol start='3' ><li>Gunzip and tar commands to get executable.</li></ol><pre class='md-fences mock-cm' style='display:block;position:relative'>gunzip Trinity-v2.3.2.tar.gz
tar -xvf Trinity-v2.3.2.tar</pre><ol start='4' ><li><p>cd&#39;d into triniy directory and use &quot;make&quot;</p></li><li><p>make plugins</p></li><li><p>Check to see if it works</p><pre class='md-fences mock-cm' style='display:block;position:relative'>cd sample_data/test_Trinity_Assembly/
./runMe.sh</pre><p>​</p></li></ol><p></p><hr /><p></p><p> <div id='id-section8'/><br/></p><h3><a name='header-n1767' class='md-header-anchor '></a>Page 8: 2017-02-06. Week 4, Day 6, RNA-seq</h3><h2><a name='header-n1768' class='md-header-anchor '></a>Class outline:</h2><ol start='' ><li>info update with Melissa on RNA-seq</li><li>Paper discussion</li><li>RNA-seq pipeline through the server</li></ol><h2><a name='header-n1780' class='md-header-anchor '></a>1 .info update with Melissa on RNA-seq</h2><p>Pre-overall workflow:</p><ol start='' ><li>Approach</li><li>Experimental design</li><li>Library Prep</li><li>Sequencing facility</li><li>Receive data</li><li>Computer/server setup </li><li>Processing</li></ol><p><mark>Processing Workflow:</mark></p><p></p><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart5" width="1933.75" xmlns="http://www.w3.org/2000/svg" height="1331.6000061035156" viewBox="0 0 1953.75 1351.6000061035156" style="height: 1391.6000061035156px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M784.21875,128.46875L394.75,225L394.75,250" marker-end="url(#arrowhead194)" style="fill:none"></path><defs><marker id="arrowhead194" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M394.75,291L394.75,316L394.75,341" marker-end="url(#arrowhead195)" style="fill:none"></path><defs><marker id="arrowhead195" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M394.75,382L394.75,407L394.75,432" marker-end="url(#arrowhead196)" style="fill:none"></path><defs><marker id="arrowhead196" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M394.75,473L394.75,498L394.75,523" marker-end="url(#arrowhead197)" style="fill:none"></path><defs><marker id="arrowhead197" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M394.75,564L394.75,589L395.24999999999994,614.5000030517579" marker-end="url(#arrowhead198)" style="fill:none"></path><defs><marker id="arrowhead198" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M386.9843906106275,939.8343936623852L384.75,972.6000061035156L384.75,997.6000061035156" marker-end="url(#arrowhead199)" style="fill:none"></path><defs><marker id="arrowhead199" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M298.8076923076923,1038.6000061035156L194,1063.6000061035156L194,1088.6000061035156" marker-end="url(#arrowhead200)" style="fill:none"></path><defs><marker id="arrowhead200" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M410.7692307692308,1038.6000061035156L442.5,1063.6000061035156L442.5,1088.6000061035156" marker-end="url(#arrowhead201)" style="fill:none"></path><defs><marker id="arrowhead201" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M503.13186813186815,1038.6000061035156L647.5,1063.6000061035156L647.5,1088.6000061035156" marker-end="url(#arrowhead202)" style="fill:none"></path><defs><marker id="arrowhead202" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M544.25,1031.6586235719883L920,1063.6000061035156L920,1088.6000061035156" marker-end="url(#arrowhead203)" style="fill:none"></path><defs><marker id="arrowhead203" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M526.4753922843265,816.874604663916L1102.25,972.6000061035156L1192.6346153846155,997.6000061035156" marker-end="url(#arrowhead204)" style="fill:none"></path><defs><marker id="arrowhead204" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M927.2155172413793,129.5344827586207L1284.5,225L1284.5,270.5L1284.5,316L1284.5,361.5L1284.5,407L1284.5,452.5L1284.5,498L1284.5,543.5L1284.5,589L1284.5,780.8000030517578L1284.5,972.6000061035156L1274.7472527472528,997.6000061035156" marker-end="url(#arrowhead205)" style="fill:none"></path><defs><marker id="arrowhead205" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M1265.7362637362637,1038.6000061035156L1264.5,1063.6000061035156L1264.5,1088.6000061035156" marker-end="url(#arrowhead206)" style="fill:none"></path><defs><marker id="arrowhead206" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M1264.5,1129.6000061035156L1264.5,1154.6000061035156L1264.5,1179.6000061035156" marker-end="url(#arrowhead207)" style="fill:none"></path><defs><marker id="arrowhead207" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M1392.25,1035.7924925481323L1589.5,1063.6000061035156L1589.5,1088.6000061035156" marker-end="url(#arrowhead208)" style="fill:none"></path><defs><marker id="arrowhead208" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M1589.5,1129.6000061035156L1589.5,1154.6000061035156L1589.5,1179.6000061035156" marker-end="url(#arrowhead209)" style="fill:none"></path><defs><marker id="arrowhead209" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M1444.5,1215.171393881928L1151.75,1245.6000061035156L1151.75,1270.6000061035156" marker-end="url(#arrowhead210)" style="fill:none"></path><defs><marker id="arrowhead210" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M1517.2994505494505,1220.6000061035156L1429.25,1245.6000061035156L1429.25,1270.6000061035156" marker-end="url(#arrowhead211)" style="fill:none"></path><defs><marker id="arrowhead211" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M1661.7005494505495,1220.6000061035156L1749.75,1245.6000061035156L1749.75,1270.6000061035156" marker-end="url(#arrowhead212)" style="fill:none"></path><defs><marker id="arrowhead212" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="A" transform="translate(855.75,110)" style="opacity: 1;"><polygon points="90,0 180,-90 90,-180 0,-90" rx="5" ry="5" transform="translate(-90,90)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-82,-10.5)"><foreignObject width="164" height="21"><div style="display: inline-block; white-space: nowrap;">Raw reads, fast-q</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(394.75,270.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-164" y="-20.5" width="328" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-154,-10.5)"><foreignObject width="308" height="21"><div style="display: inline-block; white-space: nowrap;">Clean reads: Get rid of adaptors</div></foreignObject></g></g></g><g class="node" id="C" transform="translate(394.75,361.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-188" y="-20.5" width="376" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-178,-10.5)"><foreignObject width="356" height="21"><div style="display: inline-block; white-space: nowrap;">Clean reads: Check nucleotide quality</div></foreignObject></g></g></g><g class="node" id="D" transform="translate(394.75,452.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-106.5" y="-20.5" width="213" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-96.5,-10.5)"><foreignObject width="193" height="21"><div style="display: inline-block; white-space: nowrap;">Clean reads: Length?</div></foreignObject></g></g></g><g class="node" id="E" transform="translate(394.75,543.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-87" y="-20.5" width="174" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-77,-10.5)"><foreignObject width="154" height="21"><div style="display: inline-block; white-space: nowrap;">Evaluate Quality</div></foreignObject></g></g></g><g class="node" id="F" transform="translate(394.75,780.8000030517578)" style="opacity: 1;"><polygon points="166.8,0 333.6,-166.8 166.8,-333.6 0,-166.8" rx="5" ry="5" transform="translate(-166.8,166.8)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-178,-10.5)"><foreignObject width="356" height="21"><div style="display: inline-block; white-space: nowrap;">De novo Transcriptome Assembly, fasta</div></foreignObject></g></g></g><g class="node" id="G" transform="translate(384.75,1018.1000061035156)" style="opacity: 1;"><rect rx="0" ry="0" x="-159.5" y="-20.5" width="319" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-149.5,-10.5)"><foreignObject width="299" height="21"><div style="display: inline-block; white-space: nowrap;">Evaluate Transcriptome assembly</div></foreignObject></g></g></g><g class="node" id="Y" transform="translate(194,1109.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-174" y="-20.5" width="348" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-164,-10.5)"><foreignObject width="328" height="21"><div style="display: inline-block; white-space: nowrap;">compare to closely related species</div></foreignObject></g></g></g><g class="node" id="P" transform="translate(442.5,1109.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-24.5" y="-20.5" width="49" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-14.5,-10.5)"><foreignObject width="29" height="21"><div style="display: inline-block; white-space: nowrap;">CEG</div></foreignObject></g></g></g><g class="node" id="W" transform="translate(647.5,1109.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-130.5" y="-20.5" width="261" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-120.5,-10.5)"><foreignObject width="241" height="21"><div style="display: inline-block; white-space: nowrap;">N50 and number of contigs</div></foreignObject></g></g></g><g class="node" id="y" transform="translate(920,1109.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-92" y="-20.5" width="184" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-82,-10.5)"><foreignObject width="164" height="21"><div style="display: inline-block; white-space: nowrap;">BLAST to annotate</div></foreignObject></g></g></g><g class="node" id="K" transform="translate(1266.75,1018.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-125.5" y="-20.5" width="251" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-115.5,-10.5)"><foreignObject width="231" height="21"><div style="display: inline-block; white-space: nowrap;">Map raw reads, SAM files</div></foreignObject></g></g></g><g class="node" id="H" transform="translate(1264.5,1109.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-202.5" y="-20.5" width="405" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-192.5,-10.5)"><foreignObject width="385" height="21"><div style="display: inline-block; white-space: nowrap;">Extract read count info, gene expression</div></foreignObject></g></g></g><g class="node" id="U" transform="translate(1264.5,1200.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-68" y="-20.5" width="136" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-58,-10.5)"><foreignObject width="116" height="21"><div style="display: inline-block; white-space: nowrap;">DGE analysis</div></foreignObject></g></g></g><g class="node" id="V" transform="translate(1589.5,1109.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-72.5" y="-20.5" width="145" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-62.5,-10.5)"><foreignObject width="125" height="21"><div style="display: inline-block; white-space: nowrap;">Identify SNPs</div></foreignObject></g></g></g><g class="node" id="J" transform="translate(1589.5,1200.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-145" y="-20.5" width="290" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-135,-10.5)"><foreignObject width="270" height="21"><div style="display: inline-block; white-space: nowrap;">Population genomics analyses</div></foreignObject></g></g></g><g class="node" id="a" transform="translate(1151.75,1291.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-121" y="-20.5" width="242" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-111,-10.5)"><foreignObject width="222" height="21"><div style="display: inline-block; white-space: nowrap;">Genetic Differentiation</div></foreignObject></g></g></g><g class="node" id="b" transform="translate(1429.25,1291.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-106.5" y="-20.5" width="213" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-96.5,-10.5)"><foreignObject width="193" height="21"><div style="display: inline-block; white-space: nowrap;">Population Structure</div></foreignObject></g></g></g><g class="node" id="h" transform="translate(1749.75,1291.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-164" y="-20.5" width="328" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-154,-10.5)"><foreignObject width="308" height="21"><div style="display: inline-block; white-space: nowrap;">test for signatures of selection</div></foreignObject></g></g></g></g></g></g></svg></div><p></p><h2><a name='header-n1812' class='md-header-anchor '></a>2. Paper discussion: Dunning et al. 2014</h2><p>Title: Divergent transcriptional responses to low temperature among populations of alpine and lowland species of New Zealand stick insects (Micrarchus).</p><p></p><p>Shared response to cold shock? Stick insects in new zealand. They all responded to cold shock differently through differential gene expression (~ 2000 unique genes). </p><p></p><p><strong>Hypothesis</strong>: We hypothesize that species with poor dispersal ability are likely to have strong phylogeographic structure as a result of genetic drift and, possibly, local adaptation.</p><ul><li>The resulting divergent genetic backgrounds are likely to contribute to variation in the intra- and interspecific transcriptional responses to environmental stress.</li></ul><p>They&#39;re poor dispersers, so that you&#39;d expect for them to be locally adapted, particularly to the environment. </p><p></p><p>Approach: </p><ul><li>They used SNPs and COI(mitochondria) to measure extent of population/species differentiation.</li><li>De novo assembly in transcriptome</li><li>4 taxa; generated transcriptome data from experiment, </li></ul><p>What is phred quality? Quality score on the propbability of the error.  &gt;30  chance it is &quot;right&quot;.</p><p></p><p>What are unigenes? <em>Trinity components containing clusters of ‘contigs’ representing splice variants of the same locus.</em></p><p></p><h2><a name='header-n1851' class='md-header-anchor '></a>RNA-seq tutorial</h2><ol start='' ><li><p>CD to path</p><ul><li><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">cd project_data/fastq/</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre></li></ul><pre class='md-fences mock-cm' style='display:block;position:relative'></pre></li><li><p>Files: </p><ul><li><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">ls</span></pre></div><pre class=""><span style="padding-right: 0.1px;">07_5-08_S_1_R1.fq.gz  27_5-11_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">07_5-08_S_1_R2.fq.gz  27_5-14_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">07_5-11_S_4_R1.fq.gz  27_5-14_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">07_5-11_S_4_R2.fq.gz  27_5-20_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">08_5-11_S_1_R1.fq.gz  27_5-20_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">08_5-11_S_1_R2.fq.gz  28_5-08_S_1_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">08_5-14_S_1_R1.fq.gz  28_5-08_S_1_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">08_5-14_S_1_R2.fq.gz  28_5-11_S_1_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">09_5-08_H_0_R1.fq.gz  28_5-11_S_1_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">09_5-08_H_0_R2.fq.gz  28_5-17_S_2_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">09_5-14_S_2_R1.fq.gz  28_5-17_S_2_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">09_5-14_S_2_R2.fq.gz  29_5-08_S_2_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">10_5-08_H_0_R1.fq.gz  29_5-08_S_2_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">10_5-08_H_0_R2.fq.gz  29_5-11_S_2_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">10_5-11_H_0_R1.fq.gz  29_5-11_S_2_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">10_5-11_H_0_R2.fq.gz  29_5-14_S_2_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">10_5-20_S_2_R1.fq.gz  29_5-14_S_2_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">10_5-20_S_2_R2.fq.gz  31_6-21_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">15_5-17_S_3_R1.fq.gz  31_6-21_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">15_5-17_S_3_R2.fq.gz  32_6-15_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">19_5-11_H_0_R1.fq.gz  32_6-15_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">19_5-11_H_0_R2.fq.gz  32_6-18_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">19_5-17_H_0_R1.fq.gz  32_6-18_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">19_5-17_H_0_R2.fq.gz  32_6-21_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">19_5-20_S_5_R1.fq.gz  32_6-21_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">19_5-20_S_5_R2.fq.gz  33_6-12_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">20_5-14_H_0_R1.fq.gz  33_6-12_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">20_5-14_H_0_R2.fq.gz  34_6-12_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">22_5-08_S_1_R1.fq.gz  34_6-12_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">22_5-08_S_1_R2.fq.gz  34_6-18_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">22_5-11_S_1_R1.fq.gz  34_6-18_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">22_5-11_S_1_R2.fq.gz  35_6-15_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">23_5-17_S_2_R1.fq.gz  35_6-15_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">23_5-17_S_2_R2.fq.gz  35_6-18_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">24_5-08_H_0_R1.fq.gz  35_6-18_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">24_5-08_H_0_R2.fq.gz  36_6-12_S_1_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">24_5-14_H_0_R1.fq.gz  36_6-12_S_1_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">24_5-14_H_0_R2.fq.gz  36_6-15_S_2_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">24_5-17_H_0_R1.fq.gz  36_6-15_S_2_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">24_5-17_H_0_R2.fq.gz  36_6-18_S_3_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">24_5-20_H_0_R1.fq.gz  36_6-18_S_3_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">24_5-20_H_0_R2.fq.gz  38_6-18_S_2_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">26_5-08_S_2_R1.fq.gz  38_6-18_S_2_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">26_5-08_S_2_R2.fq.gz  38_6-21_H_0_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">26_5-11_S_3_R1.fq.gz  38_6-21_H_0_R2.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">26_5-11_S_3_R2.fq.gz  38_6-24_S_5_R1.fq.gz</span></pre><pre class=""><span style="padding-right: 0.1px;">27_5-08_H_0_R1.fq.gz  38_6-24_S_5_R2_fastqc.html</span></pre><pre class=""><span style="padding-right: 0.1px;">27_5-08_H_0_R2.fq.gz  38_6-24_S_5_R2_fastqc.zip</span></pre><pre class=""><span style="padding-right: 0.1px;">27_5-11_H_0_R1.fq.gz  38_6-24_S_5_R2.fq.gz</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 1050px;"></div></div></div></pre></li><li><p>Each class member will do R1 and R2(left and right reads)</p></li><li><p>number relates to scale of symptoms</p></li><li><p>Samples I&#39;m doing!</p></li><li><pre class="md-fences md-end-block" lang="unix "> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">20_5-14_H_0_R1.fq.gz &amp; 205-14_H_0_R2.fq.gz</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre></li></ul></li><li><p>look at our files: </p><ul><li><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">zcat 20_5-14_H_0_R1.fq.gz | head</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre></li><li><p>Output:</p><p>@J00160:63:HHHT2BBXX:1:1101:27823:1244 1:N:0:TCCGGAGA+AGGCTATA</p><p>GNGCGTTATTATATGGTTTTATCTTCATTTNTTAAATGAACTTGATCTTGAATTTTTTTTTTTTTTTTTTTGGGGGATCGGAAGAGCACACGTNTGAACTC</p><p>+</p><p>A#AFFAFJJJJJJJJJJJJJJJJJJJJJJJ#JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAAFFF-A&lt;JFAJF&lt;JJ-FJJJF#JF-J77A</p><p>@J00160:63:HHHT2BBXX:1:1101:28635:1244 1:N:0:TCCGGAGA+AGGCTATA</p><p>ANTGAGTAGAAGGAATCGGTCCACCATAAANAAGTGGAGGTTCCACATGGGCAAAGATGCCGGTACCATTCTTAACACTAGAAGAAGGAGCTTTTTCACTA</p><p>+</p><p>A#AFFJJJJJJJJJJJJJJJJJJJJJJJJJ#JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJAJJJJJJJJJJJJJJJJJJJJJJJJAFJJJJJF</p><p>@J00160:63:HHHT2BBXX:1:1101:29244:1244 1:N:0:TCCGGAGA+AGGCTATA</p><p>GAGGCACTCATACAGGTTACACAGCTGAGANTAATTTATATCATATACTATAATGCATAATACATGTAAGCATCTCTATTGCTACATTGCCTGGTTATACA</p></li></ul></li><li><p>This is a fastq file</p><ul><li>Unique indentifier</li><li>ATGC&#39;s are sequence data for the reads; same length</li><li>3rd line is a plus, just an indicator; before the quality score (PHRED) for each nucleotide bases (ASCII format to have double digit value to be represented by a single character). Makes downstream processing easier</li><li>Letters are good quality, non-letters are not.</li><li>greater than 30 is usually the cutoff</li></ul></li><li><p>move into my directory and save the scripts into the scripts folder</p><ul><li><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">cd scripts/</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre></li><li><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">cp /data/scripts/trim_example.sh .</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre></li><li><p>​</p></li></ul></li></ol><h2><a name='header-n1937' class='md-header-anchor '></a>Actual script to run trimmomatic</h2><pre class='md-fences mock-cm' style='display:block;position:relative'>#!/bin/bash
      java -classpath /data/popgen/Trimmomatic-0.33/trimmomatic-0.33.jar org.usadellab.trimmomatic.TrimmomaticPE \
                -threads 1 \
                -phred33 \
                 /data/project_data/fastq/20_5-14_H_0_R1.fq.gz \
                 /data/project_data/fastq/20_5-14_H_0_R2.fq.gz \
                 /data/project_data/fastq/cleanreads/&quot;20_5-14_H_0_R1_clean_paired.fa&quot; \
                 /data/project_data/fastq/cleanreads/&quot;20_5-14_H_0_R1_clean_unpaired.fa&quot; \
                 /data/project_data/fastq/cleanreads/&quot;20_5-14_H_0_R2_clean_paired.fa&quot; \
                 /data/project_data/fastq/cleanreads/&quot;20_5-14_H_0_R2_clean_unpaired.fa&quot; \
                 ILLUMINACLIP:/data/popgen/Trimmomatic-0.33/adapters/TruSeq3-PE.fa:2:30:10 \
                 LEADING:28 \
             TRAILING:28 \
             SLIDINGWINDOW:6:28 \
             HEADCROP:9 \
             MINLEN:35 \</pre><p></p><p></p><h2><a name='header-n1943' class='md-header-anchor '></a>Running script:</h2><p></p><pre class='md-fences mock-cm' style='display:block;position:relative'>./trim_example.sh &amp;</pre><p></p><p>Output</p><p></p><pre class='md-fences mock-cm' style='display:block;position:relative'>Input Read Pairs: 13876156 Both Surviving: 11211956 (80.80%) Forward Only Surviving: 2046413 (14.75%) Reverse Only Surviving: 257037 (1.85%) Dropped: 360750 (2.60%)
TrimmomaticPE: Completed successfully</pre><p></p><p></p><h3><a name='header-n1958' class='md-header-anchor '></a>Course notes and thoughts:</h3><ul><li><p>Students may benefit from instructor led demonstration of implementing an electronic notebook.</p><ul><li>It is also more engaging when the instructor takes notes or copy and paste code into a notebook</li></ul></li><li><p>Give more clear instructions for how journal club leaders should lead a paper. </p><ul><li>Students should give questions, rational, hypotheses, experimental design,</li></ul></li><li><p>​</p></li></ul><hr /><p> <div id='id-section9'/><br/></p><h3><a name='header-n1980' class='md-header-anchor '></a>Page 9: 2017-02-08. Week 4, Day 7, RNA-seq cont&#39;d + paper discussion DePanis et al. 2016; MolEco</h3><p>Transcriptomics take 2: </p><h2><a name='header-n1983' class='md-header-anchor '></a>Info update by Lisa Chamberland</h2><ol start='' ><li><p>Intro</p><ul><li><p>studying relationship between organism and environment</p></li><li><p>adding genomics you can look at this at a finer scale</p></li><li><p>observe rapid responses to the environment (gene expression)</p></li><li><p>advantages: can study wild systems (non-models; or non-traditional)</p></li><li><p>find silent genes that get expressed under novel stimuli</p></li><li><p>variation between individuals, within and among populations, </p></li><li><p>novel transcripts without homologues in closely related organisms</p></li><li><p>Main methods:</p><ul><li>microarrays</li><li>nexgen sequencing</li></ul></li></ul></li><li><p>Breif overview</p></li><li><p>Main questions</p><ol start='' ><li><p>How much variation is there in gene expression and how is it structured? (288 studies)</p><ul><li><p>Heritable variation</p></li><li><p>Epigenetics</p></li><li><p>Wild populations</p></li><li><p>Qst-Fst comparisons</p></li><li><p>eQTL</p></li><li><p>Macroevolution</p><ul><li>bottlenecks, drift, selection</li></ul></li></ul></li><li><p>How do environmental stimuli affect gene expression? ( 136)</p><ul><li><p>abiotic stress</p></li><li><p>environmental heterogeneity</p></li><li><p>host-parasite interactions</p></li><li><p>selective abiotic and biotic interactions</p></li><li><p>molecular basis of response</p><ul><li>phenotypic plasticity</li><li>among genotypes</li></ul></li><li><p>drawbacks : need to flash freeze and the transcriptome is just a snapshot</p></li><li><p>time course analyses</p></li></ul></li><li><p>how does gene expression affect phenotype? (15 studies)</p><ul><li><p>alternative phenotypes</p></li><li><p>move from correlation to causation</p><ul><li>transgenics, RNAi, crispr-cas9</li></ul></li></ul></li></ol></li><li><p>future directions</p><ul><li><p>combined microarrays and rna-seq </p></li><li><p>Database for proposed ecologicla variations</p></li><li><p>Problems:</p><ul><li>bias in signals </li><li><del>heterologous arrays</del></li><li>polyploidy</li><li>RNA pooling</li><li>statistical anlayses</li></ul></li></ul></li></ol><p>Glossary: </p><p></p><h2><a name='header-n2132' class='md-header-anchor '></a>Paper discussion</h2><p>Background: Desert drosophila that grows in arid conditions. It has 2 hosts. The two cactus differ in alkyloids. </p><p>Q: How does environment influence gene expression? </p><p></p><p></p><h2><a name='header-n2141' class='md-header-anchor '></a>Coding session</h2><p>Learning goals (skills):</p><ol start='' ><li><p>scripts</p><ul><li>executing scripts</li></ul></li><li><p>paths</p><ul><li>program</li><li>input</li><li>output</li><li>filenames in and out</li></ul></li><li><p>moving through the directories</p></li><li><p>moving files</p><ul><li>on the server </li><li><mark>from server to our computer (scp)</mark></li></ul></li><li><p>​</p></li></ol><p>Practical:</p><ol start='' ><li>Finish up our cleaning: trimmomatic</li><li>fastqc (visualizing)</li><li>make a table of # reads</li><li>design assembly tests</li><li>Start assemblies</li><li>Evaluate assembly</li></ol><p> <u>Moved the trimmomatic files into this path:</u><br/></p><p></p><pre class='md-fences mock-cm' style='display:block;position:relative'>/users/a/d/adnguyen/mydata/2017-02-08_cleanread</pre><p> <u>Ok now, run fastqc to check files</u><br/></p><p></p><pre class='md-fences mock-cm' style='display:block;position:relative'>fastqc 20*</pre><p></p><p> <u>What do I get?</u><br/></p><pre class='md-fences mock-cm' style='display:block;position:relative'>[adnguyen@pbio381 2017-02-08_cleanreads]$ ls
20_5-14_H_0_R1_clean_paired.fa
20_5-14_H_0_R1_clean_paired.fa_fastqc.html
20_5-14_H_0_R1_clean_paired.fa_fastqc.zip
20_5-14_H_0_R1_clean_unpaired.fa
20_5-14_H_0_R1_clean_unpaired.fa_fastqc.html
20_5-14_H_0_R1_clean_unpaired.fa_fastqc.zip
20_5-14_H_0_R2_clean_paired.fa
20_5-14_H_0_R2_clean_paired.fa_fastqc.html
20_5-14_H_0_R2_clean_paired.fa_fastqc.zip
20_5-14_H_0_R2_clean_unpaired.fa
20_5-14_H_0_R2_clean_unpaired.fa_fastqc.html
20_5-14_H_0_R2_clean_unpaired.fa_fastqc.zip</pre><p></p><p> <u>Now, I&#39;ll move it back to my computer: So in my home computer, this is my working directory.</u><br/></p><p></p><pre class='md-fences mock-cm' style='display:block;position:relative'>/Users/andrewnguyen</pre><p></p><p> <u>Now we can move files from server to my home computer</u><br/></p><pre class='md-fences mock-cm' style='display:block;position:relative'>scp adnguyen@pbio381.uvm.edu:~/mydata/2017-02-08_cleanreads/*.html .
</pre><p> <u>This is what i get</u><br/></p><pre class='md-fences mock-cm' style='display:block;position:relative'>andrewnguyen$ ls
20_5-14_H_0_R1_clean_paired.fa_fastqc.html
20_5-14_H_0_R1_clean_unpaired.fa_fastqc.html
20_5-14_H_0_R2_clean_paired.fa_fastqc.html
20_5-14_H_0_R2_clean_unpaired.fa_fastqc.html
Applications
Desktop
Documents
Downloads
Dropbox
Google Drive
Library
Movies
Music
Pictures
Public
Sites
Teaching
bower_components
node_modules
package.json
zScience
</pre><p></p><p><strong>When doing an assembly. You can concatenate reads and then run fastqc.</strong></p><p></p><h2><a name='header-n2247' class='md-header-anchor '></a>Now we can run an assembly:</h2><p>What can influence it?<br/></p><ul><li>number of individuals? </li></ul><p></p><p>instructor meeting:</p><ul><li>Coming up is assembly and mapping sequence reads. </li></ul><ul><li><p>There is a catch up day. We&#39;re ~ half a day behind. </p></li><li><p>Melissa make better quality assembly</p><ul><li>Evaluate the assembly, </li><li>Start annotation mapping (can use Trinitate; built into Trinity)</li></ul></li><li><p>Melissa (for Monday): finish cleaning all the samples and then make a good reference</p><ul><li><p>Health and sick individuals; 50 million reads</p></li><li><p>BWA-mem; short mapping <mark>(by monday)</mark></p><ul><li>students will map their own files —&gt; sam files</li><li>then concatenate sam files</li></ul></li></ul></li><li><p>for transcriptomics, they can extract read counts ffrom their sam files <mark>(Wednesday)</mark>. (this involves a python script)</p><ul><li>DEseq2 in R for transcriptomic analyses</li></ul></li></ul><hr /><p> <div id='id-section10'/><br/></p><h3><a name='header-n2303' class='md-header-anchor '></a>Page 10: 2017-02-10. Prepping for leading journal club discussion: 2015-02-15; Zhao et al. 2016; <em>MBE</em></h3><p>reference:</p><p>Zhao X, Bergland AO, Behrman EL, Gregory BD, Petrov DA, Schmidt PS. 2016. Global Transcriptional Profiling of Diapause and Climatic Adaptation in Drosophila melanogaster. Mol Biol Evol 33:707–720.</p><p></p><p><strong>Background + Objecives</strong></p><p></p><ul><li>Fruit flies occur across a latitudinal gradient, with northern populations experiencing more seasonality. </li><li>Diapause is a life history strategy characterized by inactivity and metabolic arrest—driven environmental cues including nutrient availability, temperature, photoperiod</li><li>Diapause varies across latitude</li></ul><p><mark>Goal is to understand the mechanisms that allow organisms to adapt to distinct environments</mark></p><ul><li>latitudinal gradients are great because environments differ continuously along this axis. </li><li>But! Not only can environments vary through space…but also time, especially, within a season. </li><li>Fruit flies originated in tropics of Africa, but have expanded world wide. And there are signatures of clinal varatiion in thse popualtions from australia, NA, SA, Africa. </li><li>Not only is there clinal variation in SNPS, but there are also seasonal SNPs too. </li><li>Northern lat SNPs are similar to those from SNPs associated with summer. (and vice versa)</li><li>So you have SNPs resonding to the environment through space and time…..soooo</li></ul><p><strong>Focus: Diapause phenotype</strong></p><ul><li>Induced by photoperiod and moderately low temperatures</li><li>results in halt in reproduction</li><li>increased lipid storage</li><li>increased stress tolerance</li></ul><p>Diapause varies along cline. (90% in temperate, 30% nootropics)</p><p></p><p>Fundamental lief history trade-off: somatic maintenance and reproduction. </p><p></p><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart6" width="849" xmlns="http://www.w3.org/2000/svg" height="363.6000061035156" viewBox="0 0 869 383.6000061035156" style="height: 423.6000061035156px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M240.30769230769232,61L298,86L298,111" marker-end="url(#arrowhead230)" style="fill:none"></path><defs><marker id="arrowhead230" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M145.69230769230768,61L88,86L88,111" marker-end="url(#arrowhead231)" style="fill:none"></path><defs><marker id="arrowhead231" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M582.8021978021978,61L522.5,86L522.5,111" marker-end="url(#arrowhead232)" style="fill:none"></path><defs><marker id="arrowhead232" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M681.6978021978022,61L742,86L742,111" marker-end="url(#arrowhead233)" style="fill:none"></path><defs><marker id="arrowhead233" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M522.5,152L522.5,177L603.2768787277041,247.57312279817447" marker-end="url(#arrowhead234)" style="fill:none"></path><defs><marker id="arrowhead234" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M742,152L742,177L662.2231204575778,247.57312350933606" marker-end="url(#arrowhead235)" style="fill:none"></path><defs><marker id="arrowhead235" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M88,152L88,177L156.47808502048196,239.52191192776024" marker-end="url(#arrowhead236)" style="fill:none"></path><defs><marker id="arrowhead236" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M298,152L298,177L230.5219117879387,239.52191483969654" marker-end="url(#arrowhead237)" style="fill:none"></path><defs><marker id="arrowhead237" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="C" transform="translate(193,40.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-92" y="-20.5" width="184" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-82,-10.5)"><foreignObject width="164" height="21"><div style="display: inline-block; white-space: nowrap;">Benign conditions</div></foreignObject></g></g></g><g class="node" id="I" transform="translate(298,131.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-92" y="-20.5" width="184" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-82,-10.5)"><foreignObject width="164" height="21"><div style="display: inline-block; white-space: nowrap;">High reproduction</div></foreignObject></g></g></g><g class="node" id="T" transform="translate(88,131.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-68" y="-20.5" width="136" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-58,-10.5)"><foreignObject width="116" height="21"><div style="display: inline-block; white-space: nowrap;">Low survival</div></foreignObject></g></g></g><g class="node" id="E" transform="translate(632.25,40.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-164" y="-20.5" width="328" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-154,-10.5)"><foreignObject width="308" height="21"><div style="display: inline-block; white-space: nowrap;">Stressful unfavorable conditions</div></foreignObject></g></g></g><g class="node" id="R" transform="translate(522.5,131.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-82.5" y="-20.5" width="165" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-72.5,-10.5)"><foreignObject width="145" height="21"><div style="display: inline-block; white-space: nowrap;">Higher survival</div></foreignObject></g></g></g><g class="node" id="V" transform="translate(742,131.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-87" y="-20.5" width="174" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-77,-10.5)"><foreignObject width="154" height="21"><div style="display: inline-block; white-space: nowrap;">Low reproduction</div></foreignObject></g></g></g><g class="node" id="A" transform="translate(632.25,272.8000030517578)" style="opacity: 1;"><polygon points="55.2,0 110.4,-55.2 55.2,-110.4 0,-55.2" rx="5" ry="5" transform="translate(-55.2,55.2)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-38.5,-10.5)"><foreignObject width="77" height="21"><div style="display: inline-block; white-space: nowrap;">Diapause</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(193,272.8000030517578)" style="opacity: 1;"><polygon points="70.8,0 141.6,-70.8 70.8,-141.6 0,-70.8" rx="5" ry="5" transform="translate(-70.8,70.8)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-58,-10.5)"><foreignObject width="116" height="21"><div style="display: inline-block; white-space: nowrap;">Non-diapause</div></foreignObject></g></g></g></g></g></g></svg></div><p>Candidates for diapause: </p><ul><li>Dp110</li><li>timeless</li><li>cough potato</li><li>circadian clock</li><li>insulin and insulin like growth factor</li></ul><h2><a name='header-n2388' class='md-header-anchor '></a>Questions</h2><ol start='' ><li>What are the common underlying mechanisms of adaptation to the environment?  (gathered from intro)</li><li>What are the genes and transcripts that are diff expressed as a function of diapause phenotype? </li><li>Are the genes previously id as being associated with variance in diapause, diff regulated in diapause vs non diapause</li><li>Are the DE genes segregating for SNPs that vary in frequency with latitude and season? </li><li>Is there tissue specificity? </li></ol><h2><a name='header-n2406' class='md-header-anchor '></a>Hypotheses</h2><ol start='' ><li>Diapause is one major determinant of adaptation to spatial and temporal environmental heterogeneity in fruit flies</li></ol><p>Prediction:</p><p>If the natural variation of diapause does play critical roles in adaptation to environmental heterogeneity, genes differentially regulated as a function of the diapause phenotype are likely under spatially and/or temporally varying selectively pressures, thus genetic polymorphisms on these genes or in vicinity of these genes are likely to show clinal and seasonal patterns as they may have distinct cis-regulatory functions that regulates gene expression in diapausing nondiapausing individuals</p><p></p><p>Basically: Variation in gene expression associated with diapause should be under selection. One way to visualize this is how these genes vary across space(latitude) and time(season)</p><ol start='2' ><li>Maternal epigenetic effects: They hypothesized that higher level regulatory mechanisms such as local sharing of regulatory elements and chromatin structure may also be involved in the regulation of diapause in D. melanogaster,</li><li>​</li></ol><p>Methods</p><ol start='' ><li><p>Collected flies from 4 popoulations (northern) east coast us: sampled in october with 50 isofemale lines each orchard</p><ul><li>Bowdoin maine,</li><li>shoreham vt</li><li>harvard mass</li><li>middlefield ct</li></ul></li><li><p>Treatment: reared at 25 C 12L :12D; 11 C 9L:15D</p><ul><li><p>dissected, scored oocytes as :</p><ul><li>D = arrest before vitellogenesis (before stage 8)</li><li>ND = vitellogenin was observed (stage 8 or later)</li></ul></li></ul></li><li><p>Measured gxp for heads and ovaries ( 2 tissue types)</p><ul><li>92 flies in D_head or O_ovary; 106 flies pooled into ND_head_ND_ovary</li></ul></li><li><p>Sequencing: 100bps short reads</p></li><li><p>Used EBSeq (empirical baysian approach) to call genes and isoforms for gene expression. Measured RPKM ( Reads Per Kilobase of transcript per Million mapped reads)</p></li><li><p>Analaysis: vitellogenesis and oogenesis excluded from analysis. </p></li><li><p>Location dependent expression: Looked at autocorrelation in the log2 fold changes along each chromosome (used moran&#39;s I as the metric of autocorrelation). This was compared to a randomly generated chromosome arm.</p></li><li><p>Test if DE genes are enriched in season/clinal sets</p></li></ol><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart7" width="772" xmlns="http://www.w3.org/2000/svg" height="1039.6000061035156" viewBox="0 0 792 1059.6000061035156" style="height: 1099.6000061035156px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M97.5,61L97.5,86L282.6118691652274,212.8881277830148" marker-end="url(#arrowhead279)" style="fill:none"></path><defs><marker id="arrowhead279" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 0 0 L 0 0 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M293,61L293,86L330.0097785972648,165.49021835097736" marker-end="url(#arrowhead280)" style="fill:none"></path><defs><marker id="arrowhead280" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 0 0 L 0 0 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M474,61L474,86L437.9902194734018,165.4902225251596" marker-end="url(#arrowhead281)" style="fill:none"></path><defs><marker id="arrowhead281" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 0 0 L 0 0 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M669.5,61L669.5,86L485.38812721168125,212.88813026343902" marker-end="url(#arrowhead282)" style="fill:none"></path><defs><marker id="arrowhead282" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 0 0 L 0 0 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M384,453.09999694824216L383.5,477.6000061035156L383.5,502.6000061035156" marker-end="url(#arrowhead283)" style="fill:none"></path><defs><marker id="arrowhead283" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M383.5,543.6000061035156L383.5,579.1000061035156L383.5,614.6000061035156" marker-end="url(#arrowhead284)" style="fill:none"></path><defs><marker id="arrowhead284" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M383.5,655.6000061035156L383.5,680.6000061035156L383.5,705.6000061035156" marker-end="url(#arrowhead285)" style="fill:none"></path><defs><marker id="arrowhead285" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M383.5,746.6000061035156L383.5,771.6000061035156L383.5,796.6000061035156" marker-end="url(#arrowhead286)" style="fill:none"></path><defs><marker id="arrowhead286" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M340,831.3648709683805L244.75,862.6000061035156L244.75,887.6000061035156" marker-end="url(#arrowhead287)" style="fill:none"></path><defs><marker id="arrowhead287" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M427,831.3648709683805L522.25,862.6000061035156L522.25,887.6000061035156" marker-end="url(#arrowhead288)" style="fill:none"></path><defs><marker id="arrowhead288" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M522.25,928.6000061035156L522.25,953.6000061035156L522.25,978.6000061035156" marker-end="url(#arrowhead289)" style="fill:none"></path><defs><marker id="arrowhead289" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(383.5,579.1000061035156)" style="opacity: 1;"><g transform="translate(-48,-10.5)" class="label"><foreignObject width="96" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">11C,9L:15D</span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="a" transform="translate(97.5,40.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-77.5" y="-20.5" width="155" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-67.5,-10.5)"><foreignObject width="135" height="21"><div style="display: inline-block; white-space: nowrap;">Bowdoin, Maine</div></foreignObject></g></g></g><g class="node" id="A" transform="translate(383.5,281.8000030517578)" style="opacity: 1;"><polygon points="170.8,0 341.6,-170.8 170.8,-341.6 0,-170.8" rx="5" ry="5" transform="translate(-170.8,170.8)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-183,-10.5)"><foreignObject width="366" height="21"><div style="display: inline-block; white-space: nowrap;">50 isofemale lines for each population</div></foreignObject></g></g></g><g class="node" id="b" transform="translate(293,40.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-68" y="-20.5" width="136" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-58,-10.5)"><foreignObject width="116" height="21"><div style="display: inline-block; white-space: nowrap;">Shoreham, VT</div></foreignObject></g></g></g><g class="node" id="c" transform="translate(474,40.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-63" y="-20.5" width="126" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-53,-10.5)"><foreignObject width="106" height="21"><div style="display: inline-block; white-space: nowrap;">Harvard, MA</div></foreignObject></g></g></g><g class="node" id="d" transform="translate(669.5,40.5)" style="opacity: 1;"><rect rx="5" ry="5" x="-82.5" y="-20.5" width="165" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-72.5,-10.5)"><foreignObject width="145" height="21"><div style="display: inline-block; white-space: nowrap;">Middlefield, CT</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(383.5,523.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-53.5" y="-20.5" width="107" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-43.5,-10.5)"><foreignObject width="87" height="21"><div style="display: inline-block; white-space: nowrap;">Treatment</div></foreignObject></g></g></g><g class="node" id="C" transform="translate(383.5,635.1000061035156)" style="opacity: 1;"><rect rx="5" ry="5" x="-72.5" y="-20.5" width="145" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-62.5,-10.5)"><foreignObject width="125" height="21"><div style="display: inline-block; white-space: nowrap;">Check ND or D</div></foreignObject></g></g></g><g class="node" id="D" transform="translate(383.5,726.1000061035156)" style="opacity: 1;"><rect rx="0" ry="0" x="-111" y="-20.5" width="222" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-101,-10.5)"><foreignObject width="202" height="21"><div style="display: inline-block; white-space: nowrap;">Dissect head or ovary</div></foreignObject></g></g></g><g class="node" id="E" transform="translate(383.5,817.1000061035156)" style="opacity: 1;"><rect rx="0" ry="0" x="-43.5" y="-20.5" width="87" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-33.5,-10.5)"><foreignObject width="67" height="21"><div style="display: inline-block; white-space: nowrap;">RNA-seq</div></foreignObject></g></g></g><g class="node" id="G" transform="translate(244.75,908.1000061035156)" style="opacity: 1;"><rect rx="0" ry="0" x="-82.5" y="-20.5" width="165" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-72.5,-10.5)"><foreignObject width="145" height="21"><div style="display: inline-block; white-space: nowrap;">Autocorrelation</div></foreignObject></g></g></g><g class="node" id="N" transform="translate(522.25,908.1000061035156)" style="opacity: 1;"><rect rx="0" ry="0" x="-145" y="-20.5" width="290" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-135,-10.5)"><foreignObject width="270" height="21"><div style="display: inline-block; white-space: nowrap;">Differential gene expression</div></foreignObject></g></g></g><g class="node" id="V" transform="translate(522.25,999.1000061035156)" style="opacity: 1;"><rect rx="0" ry="0" x="-111" y="-20.5" width="222" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-101,-10.5)"><foreignObject width="202" height="21"><div style="display: inline-block; white-space: nowrap;">Functional enrichment</div></foreignObject></g></g></g></g></g></g></svg></div><p></p><p></p><h2><a name='header-n2486' class='md-header-anchor '></a>Results</h2><p><strong>Fig 1.</strong> Shows log2 fold change (D over ND) vs RPKM. So higher values of log2 fold change = diapausing individuals having more gxp. This is a plot to simply show no relationship between how highlly expressed a gene is  with expression differences. </p><p><strong>thoughts</strong></p><p>Bad figure. What happened to the populations? That is the more interesting result.</p><p></p><p><strong>Table 1. Summary of gene expression level differences</strong></p><p>Number of genes that are differentially expressed between D and ND for head or ovary.</p><p></p><p><strong>Table 2. Isoform level differential expression</strong></p><p>Isoforms diff expressed between D and ND for head or ovary. </p><p></p><p><strong>Fig2. Venn diagram of overlap between DE of genes or isoforms for head and ovary.</strong></p><ul><li>Higher number of genes with at least 1 DE isofrom in both head and ovary.</li><li>Moderate amount of overlap</li><li>Lowest amount of just overall DE at the gene level</li></ul><p><strong>Table 3. Enriched KEGG pathways.</strong></p><ul><li><p>Head has more pathways than ovaries</p><ul><li>more genes downregulated</li></ul></li><li><p>​</p></li></ul><p><strong>Table 4. Gene-level expression</strong> </p><p>Candidate genes were not differentially expressed. Why is this? </p><ol start='' ><li>cpo</li><li>tim</li><li>dp110</li></ol><p><strong>Table 5. Transcript level expression</strong></p><p>Some of the cp, tim, and dp110 isoforms are differentially expressed. </p><p></p><p><strong>Fig3. Density as a function of moran&#39;s I (autocorrelation) for each chromosome and each heard or ovary.</strong></p><ul><li>The distributions represent the null expectation in the correlation of gxp due to position on the chromosome</li><li>It looks like all but X head chromosomes have autocorrelated gxp. </li></ul><p><strong>Table 6. Summary of overlapping genes DDE and clinal /seasonal genes</strong></p><ul><li>lots of clinal genes are differentially expressed</li><li>less seasonal genes that are differentially expressed</li></ul><p><strong>Fig 4. Enrichment of DE genes that are clinal are seasonal.</strong></p><p>Only downregulated in the head were enriched under clinal and seasonal. </p><p></p><p>Take home</p><ul><li>Tissue specific and isoform specific expression can vary within an organism. </li><li>Downrgulated genes in the head are the types of genes that are related to latitude and seasonality. </li><li>Gene expression is autocorrelated. </li></ul><hr /><p> <div id='id-section11'/><br/></p><h3><a name='header-n2591' class='md-header-anchor '></a>Page 11: 2017-02-13. Week 5, Day 8, RNA-seq mapping and paper discussion: Johnston et al. 2016, <em>Molecular Ecology</em></h3><h2><a name='header-n2592' class='md-header-anchor '></a>Info update- Transcriptomics : Lauren Ash</h2><p></p><p>Glossary:</p><ol start='' ><li>sequence coverage: the average number of reads that align/cover known reference bases</li><li>read depth: total number of bases sequenced/aligned at a given reference base position</li><li>statistical noise: unexplained variation/randomness</li><li>power: probability of rejecting a false null hypothesis</li><li>Biological variation: natural variation in gene expression measurements due to environmental or genetic differences</li></ol><ol start='' ><li><p><strong>Background</strong></p><ul><li>can measure differential gene expression (within population and among )</li><li>topics: diseases resistance, mating behavior, adaptive signfiicance </li><li>connect molecular mechanisms to phenotypic/behavioral plasticity </li><li>limitations: reference genomc quality, availability of genes, expense per smaple lib prep</li></ul></li><li><p><strong>Issues</strong></p><ul><li><p>under utiilization of biological replicates</p></li><li><p>requiring independent library preparations </p></li><li><p>doesn&#39;t include pooled samples (unless pooled samples that were replicated)</p><ul><li>23/158 studies (15%) have more than 3 biological replicates</li><li>derive broad biological conclusions</li><li>prioritize sequencing depth over replication </li></ul></li><li><p>wide dynamic range can make it noisy</p><ol start='' ><li>poisson counting error (error associated with any counting experiment)</li><li>non-poisson technical variance (processing, quality, different lanes, storage)</li><li>biological variance (usually the largest)</li></ol></li></ul></li><li><p>R exercise</p><ul><li>so cool, </li></ul></li><li><p>Gerenal rules of thumb</p><ol start='' ><li><p>more biological replicates more than increasing depth</p></li><li><p>sequence more than 10 reads of depth per transcript</p><ul><li>10-20 million mapped reads per sample is sufficient</li></ul></li><li><p>Use at least 3 biological replicates per condition</p></li><li><p>conduct a pilot experiment </p><ol start='' ><li><p>answer 2 questions:</p><ul><li>What is the and most powerful experiment that I can afford? </li><li>What is the smallest fold change I can detect?</li><li>7 tools in R to estimate power</li></ul></li></ol></li></ol></li></ol><h2><a name='header-n2709' class='md-header-anchor '></a>Coding:</h2><p>workflow:</p><ol start='' ><li><p>clean adn evaluate reads (fastq)</p></li><li><p>Make and evaluate transcriptome assembly (fasta)</p></li><li><p>Map cleaned reads to transcriptome assembly (.sam files)</p></li><li><p>Extract data:</p><ol start='' ><li>read counts: number of reads that uniquely map to each gene</li><li>get SNPs</li></ol></li></ol><p><strong>Transdecoder</strong> predicts open reading frames.</p><p></p><h2><a name='header-n2736' class='md-header-anchor '></a>Beginning to map reads to reference transcriptome</h2><ul><li>Using BWA to do this: Copying script</li></ul><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">cp /data/scripts/bwaaln.sh .</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre><ul><li>waht does the script look like? </li></ul><pre class='md-fences mock-cm' style='display:block;position:relative'>#!/bin/bash 
 
# To run from present directory and save output: ./bwaaln.sh &gt; output.bwaaln.txt 

myLeft=&#39;20_5-14_H_0_R1_clean_paired.fa&#39;
echo $myLeft

myRight=${myLeft/_R1.fq.gz_left/_R2.fq.gz_right}
echo $myRight

myShort=`echo $myLeft | cut -c1-11`
echo $myShort

# bwa index /data/project_data/assembly/longest_orfs.cds  # This only needs to be done once on the reference

bwa aln /data/project_data/assembly/longest_orfs.cds /data/project_data/fastq/cleanreads/$myLeft &gt; $myLeft&quot;.sai&quot;
bwa aln /data/project_data/assembly/longest_orfs.cds /data/project_data/fastq/cleanreads/$myRight &gt; $myRight&quot;.sai&quot;
bwa sampe -r &#39;@RG\tID:&#39;&quot;$myShort&quot;&#39;\tSM:&#39;&quot;$myShort&quot;&#39;\tPL:Illumina&#39; \
        -P /data/project_data/assembly/longest_orfs.cds $myLeft&quot;.sai&quot; $myRight&quot;.sai&quot; \
        /data/project_data/fastq/cleanreads/$myLeft \
        /data/project_data/fastq/cleanreads/$myRight &gt; $myShort&quot;_bwaaln.sam&quot;
        </pre><ul><li>Options for bwa align</li></ul><pre class='md-fences mock-cm' style='display:block;position:relative'> bwa aln

Usage:   bwa aln [options]  &lt;prefix&gt; &lt;in.fq&gt;  


Options: -n NUM    max #diff (int) or missing prob under 0.02 err rate (float) [0.04]
         -o INT    maximum number or fraction of gap opens [1]
         -e INT    maximum number of gap extensions, -1 for disabling long gaps [-1]
         -i INT    do not put an indel within INT bp towards the ends [5]
         -d INT    maximum occurrences for extending a long deletion [10]
         -l INT    seed length [32]
         -k INT    maximum differences in the seed [2]
         -m INT    maximum entries in the queue [2000000]
         -t INT    number of threads [1]
         -M INT    mismatch penalty [3]
         -O INT    gap open penalty [11]
         -E INT    gap extension penalty [4]
         -R INT    stop searching when there are &gt;INT equally best hits [30]
         -q INT    quality threshold for read trimming down to 35bp [0]
         -f FILE   file to write output to instead of stdout
         -B INT    length of barcode
         -L        log-scaled gap penalty for long deletions
         -N        non-iterative mode: search for all n-difference hits (slooow)
         -I        the input is in the Illumina 1.3+ FASTQ-like format
         -b        the input read file is in the BAM format
         -0        use single-end reads only (effective with -b)
         -1        use the 1st read in a pair (effective with -b)
         -2        use the 2nd read in a pair (effective with -b)
         -Y        filter Casava-filtered sequences
</pre><ul><li><p>Run default parameters </p><ul><li>Only thing that would change is altering the maximum SNP differences  between mapping sequence to reference sequence</li><li>​</li></ul></li><li><p><mark>Should end up with <code>.sai</code> files!!!</mark></p></li></ul><p>output:</p><pre class='md-fences mock-cm' style='display:block;position:relative'>ls
20_5-14_H_0_bwaaln.sam              bwaaln.sh
20_5-14_H_0_R1_clean_paired.fa.sai  trim_example.sh
20_5-14_H_0_R2_clean_paired.fa.sai</pre><p></p><ul><li><mark>What the header should look like:  `Transcript_Contigs::Read_ID</mark>`</li></ul><pre class='md-fences mock-cm' style='display:block;position:relative'>head 20_5-14_H_0_bwaaln.sam 
@SQ	SN:TRINITY_DN37_c0_g1::TRINITY_DN37_c0_g1_i1::g.1::m.1	LN:303
@SQ	SN:TRINITY_DN120_c0_g2::TRINITY_DN120_c0_g2_i1::g.2::m.2	LN:381
@SQ	SN:TRINITY_DN125_c0_g1::TRINITY_DN125_c0_g1_i1::g.3::m.3	LN:642
@SQ	SN:TRINITY_DN125_c0_g2::TRINITY_DN125_c0_g2_i1::g.4::m.4	LN:528
@SQ	SN:TRINITY_DN159_c0_g1::TRINITY_DN159_c0_g1_i1::g.5::m.5	LN:696
@SQ	SN:TRINITY_DN159_c0_g1::TRINITY_DN159_c0_g1_i1::g.6::m.6	LN:396
@SQ	SN:TRINITY_DN191_c0_g1::TRINITY_DN191_c0_g1_i1::g.7::m.7	LN:309
@SQ	SN:TRINITY_DN192_c0_g1::TRINITY_DN192_c0_g1_i1::g.8::m.8	LN:318
@SQ	SN:TRINITY_DN192_c0_g2::TRINITY_DN192_c0_g2_i1::g.9::m.9	LN:321
@SQ	SN:TRINITY_DN293_c0_g1::TRINITY_DN293_c0_g1_i1::g.10::m.10	LN:315</pre><ul><li><mark>What the tail should look like: <code>vim tail_version_sam.txt</code></mark></li></ul><pre class='md-fences mock-cm' style='display:block;position:relative'>J00160:63:HHHT2BBXX:4:2228:14296:49089	77	*	0	0	*	CTCTGCCCCGACGGCCGGGTATAGGCGGCACGCTCAGCGCCATCCATTTTCAGGGCTAGTTGATTCGGCAGGTGAGTTGTTACACACTCCT	JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFFJJJJJJFFJJJJJJJJ7FJJJJJJJJJFJJF	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:14296:49089	141	*	0	0	*	TCGGAATCCGCTAAGGAGTGTGTAACAACTCACCTGCCGAATCAACTAGCCCTGAAAATGGATGGCGCTGAGCGTGCCGCCTATACCCGGC	FJJJAJJJJJJJJFJJFFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJ&lt;FJJJJ-FJJJJJJJJJJAJJJJJJJFJJJJJAJJJJJF	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:15026:49089	77	*	0	0	*	TTTTTCGTCACTACCTCCCCGTGTCGGGAGTGGGTAATTTGCGCGCCTGCTGCCTTCCTTGGATGTGGTAGCCGTTTCTCAAGCTCCCTC	JJJJJJJJFJJJJFJAJJJJFAJJJAJJAJ7FJJ&lt;AJJFJFAJFAFJ&lt;&lt;JFAJJFJJJJJAF&lt;AJFFJ-FJFJJFJFAJJJJFJJJFJJF	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:15026:49089	141	*	0	0	*	GGTTCGATTCCGGAGAGGGAGCTTGAGAAACGGCTACCACATCCAAGGAAGGCAGCAGGCGCGCAAATTACCCACTCCCGACACGGGGAGG	JJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:22008:49089	83	TRINITY_DN30310_c1_g10::TRINITY_DN30310_c1_g10_i1::g.7248::m.7248	168	60	91M	=	70	-189	CCTCGCTCCCCGGGCGAAAGGGAATCGGGTCAATATTCCCGAACCCGGAGGCGGAGCCCTCCGTCTTCGTGGCGGTCCGAGCTGTAAAGCG	JJFJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJ	RG:Z:20_5-14_H_0	XT:A:U	NM:i:SM:i:37	AM:i:37	X0:i:1	X1:i:0	XM:i:0	XO:i:0	XG:i:0	MD:Z:91
J00160:63:HHHT2BBXX:4:2228:22008:49089	163	TRINITY_DN30310_c1_g10::TRINITY_DN30310_c1_g10_i1::g.7248::m.7248	70	60	91M	=	168	189	CCATGTGAACAGCAGTTGTACATGGGTCAGTCGATCCTAAGCCCCAGGGAAGTTCCGCTCCGAGCGGAGGCGGGCGCCCCTCTCCATGTGA	FJJJJJJJJJJJJJJFFJJJJJJJJJFJJJJJJJJJJJFFJJJJJJJJJJFJFJJJJJJFFJJJJJJJJJJJJJJJJJJJFJJJJJJJFJA	RG:Z:20_5-14_H_0	XT:A:U	NM:i:SM:i:37	AM:i:37	X0:i:1	X1:i:0	XM:i:0	XO:i:0	XG:i:0	MD:Z:91
J00160:63:HHHT2BBXX:4:2228:24647:49089	77	*	0	0	*	ACGGGCGATGTGTGCGCATTCTAGGGCTTTGAGTTGTTCATGGGCATTTTCTTTTGCTCATTACTGCTGAATCCTGTTTCAAATGGGGCTA	JFJJJJFFJJJJJJJJJFJJJJJJJJJJFJJAJJJJJJJFJJJJJJJFJJJJJJJJJF&lt;JJJJJJJFAFJJFFFJJJJJJJJJJJJJ&lt;JJA	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:24647:49089	141	*	0	0	*	GGATAAGTGAGCTACAATCATAAATATAAGAATAAAAATATGTATGAATAATGAACTGATAGCCCCATTTGAAACAGGATTCAGCAGTAAT	AAJJJJ&lt;JJJJFFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJFFJJJJJJJJJJ&lt;-FFFFFAFFFAFJJJJJJJJJJJFAJFJJFJJJJ	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:30168:49089	77	*	0	0	*	TCTTGAAATCTGTGGGTTTCTCGTATAGTTCAATTACAACAGGTCCTGGTTTCAACTCGTCCATTTCCATGAAGGCAAAACACTTGGTGCT	JJJFFJJJJJJJJJJJAJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJFJJJFFAFAJFJJJJFFFJJJJFFJJJJFJJJFJJ	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:30168:49089	141	*	0	0	*	GAGTTTAAGCATTTCAAAGTGAAAAAGCGCACTATCAGCACCAAGTGTTTTGCCTTCATGGAAATGGACGAGTTG	JJJJJJJJJJJJJFJJAJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFAF	RG:Z:20_5-14_H_0
</pre><ul><li><p>Melissa code hack suggestion: </p><ol start='' ><li><code>screen</code></li><li><code>bash bwaaln.sh</code></li><li>control + a + d</li></ol></li></ul><hr /><p> <div id='id-section12'/><br/></p><h3><a name='header-n2800' class='md-header-anchor '></a>Page 12: 2017-02-15. Week 5, Day 9, Sam info update, ANBE paper discussion, Zhao et al. 2016; <em>MBE</em></h3><p></p><h2><a name='header-n2803' class='md-header-anchor '></a>Sam Alger; SNPs and pop genomics</h2><p>Drawbacks:</p><ol start='' ><li>lots of money</li><li>complex data</li><li>functionally relevant? to organism?</li></ol><p>Ways to get around this: SNP data from expressed sequences. </p><p>AA<mark>A</mark>TGC</p><p>AA<mark>C</mark>TGC</p><p></p><p><strong>Glossary:</strong></p><p>SNPs- single nucleotide polymorphism</p><p>Indels - insertion deletions.</p><p>Fat: % of genetic material explaiend by differences among populations</p><p></p><p><strong>Workflow:</strong> </p><ol start='' ><li><p>Tissue: breadth of tissue, dev stages</p><ul><li>exon skipping produce splice variants</li></ul></li><li><p>pool and sequence, create sequencel libraries; ~30-100 million paired end long reads</p></li><li><p>process raw sequence data for quality and errors</p><ul><li>important for SNP detection</li></ul></li><li><p>digital normalization of sequence data</p><ul><li>removing high coverage reads + associated errors</li><li>loss of quantitative info</li></ul></li><li><p>assemble your cleaned paired end reads</p></li><li><p>prune assembled transcripts; remove DNA contamination, non coding RNA and gene fragments</p></li><li><p>Use reference genome for assembly; or use COGs (conserved orthologue genomes) <strong>Assembly evaluation</strong></p></li></ol><p><strong>SNP detection:</strong></p><ul><li>Software- constant patterns of sequence variation </li><li>Sequence errors will have low frequency and will be removed</li><li>artifacts caused by indels, filter SNP clusters near indels, check quality scores</li></ul><p><strong>Validate SNPs</strong></p><ul><li>Confirm by designing primers and sanger sequencing</li><li>mass spec</li></ul><p><strong>Practical Application</strong></p><ol start='' ><li>What is the genetic structure of populations?</li><li>How are they related? </li><li>How natural selection is acting on loci?</li></ol><p>General approaches: </p><ol start='' ><li><p>Using outliers</p><ul><li>For a given locus, what is the level of differentiation compared to differentiation across the genome (FST)</li></ul></li><li><p>Non-outlier approach: test high Fst loci for other features associated with selection </p><ol start='' ><li>fitness</li><li>enrichment for certain functional roles</li></ol></li></ol><blockquote><p>The beginning for understanding how selection can act on populations!</p></blockquote><p></p><p></p><h2><a name='header-n2933' class='md-header-anchor '></a>Coding session</h2><ol start='' ><li><p>check your SAM file</p><ul><li>Uniquly mapped reads</li></ul></li></ol><ul><li><pre class='md-fences mock-cm' style='display:block;position:relative'>grep -c XT:A:U 20_5-14_H_0_bwaaln.sam 
1,737,572</pre></li><li><p>​</p><ul><li><pre class='md-fences mock-cm' style='display:block;position:relative'>grep -c X0:i:1 20_5-14_H_0_bwaaln.sam 
1747040
</pre></li><li><p>You can check a number of other elements, total number of reads, search for the various flags…</p></li></ul></li></ul><ol start='2' ><li><p>Count reads from sam file with python script</p><ul><li><p>copy file script:  <code>cp /data/scripts/countxpression_PE.py .</code></p></li><li><p>Running script: <code>python countxpression_PE.py 20 35 countstats.txt ../results/20_5-14_H_0_bwaaln.sam</code> <br/></p></li><li><p>ERROR: </p><ul><li><pre class='md-fences mock-cm' style='display:block;position:relative'>Traceback (most recent call last):
  File &quot;countxpression_PE.py&quot;, line 159, in  &lt;module&gt;  

    countxpression(file, sys.argv[1], sys.argv[2], 0, &#39;text&#39;, file[:-4]+&#39;_counts.txt&#39;, sys.argv[3])
  File &quot;countxpression_PE.py&quot;, line 68, in countxpression
    contigs[cols[2]][1]+=1     #add a count to that contig for the &#39;top&#39; hit for reads that optimally align to multiple contigs (all the other good hits are often not printed with current bwa settings)
KeyError: &#39;TRINITY_DN30437_c4_g9::TRINITY_DN30437_c4_g9_i6::g.7687::m.7687&#39;</pre></li><li><p>the names are too long; need a regular expression to find and replace</p></li><li><pre class='md-fences mock-cm' style='display:block;position:relative'>sed -i &#39;s/::/\_/g&#39; 20_5-14_H_0_bwaaln.sam </pre></li><li><p>The <code>s</code> is search</p></li><li><p>the <code>/::/</code> find the colon</p></li><li><p><code>\_</code> replace</p></li><li><p><code>g</code> = globally</p></li><li><p>output</p><ul><li><pre class='md-fences mock-cm' style='display:block;position:relative'>head 20_5-14_H_0_bwaaln.sam 
@SQ	SN:TRINITY_DN37_c0_g1_TRINITY_DN37_c0_g1_i1_g.1_m.1	LN:303
@SQ	SN:TRINITY_DN120_c0_g2_TRINITY_DN120_c0_g2_i1_g.2_m.2	LN:381
@SQ	SN:TRINITY_DN125_c0_g1_TRINITY_DN125_c0_g1_i1_g.3_m.3	LN:642
@SQ	SN:TRINITY_DN125_c0_g2_TRINITY_DN125_c0_g2_i1_g.4_m.4	LN:528
@SQ	SN:TRINITY_DN159_c0_g1_TRINITY_DN159_c0_g1_i1_g.5_m.5	LN:696
@SQ	SN:TRINITY_DN159_c0_g1_TRINITY_DN159_c0_g1_i1_g.6_m.6	LN:396
@SQ	SN:TRINITY_DN191_c0_g1_TRINITY_DN191_c0_g1_i1_g.7_m.7	LN:309
@SQ	SN:TRINITY_DN192_c0_g1_TRINITY_DN192_c0_g1_i1_g.8_m.8	LN:318
@SQ	SN:TRINITY_DN192_c0_g2_TRINITY_DN192_c0_g2_i1_g.9_m.9	LN:321
@SQ	SN:TRINITY_DN293_c0_g1_TRINITY_DN293_c0_g1_i1_g.10_m.10	LN:315</pre></li><li><pre class='md-fences mock-cm' style='display:block;position:relative'>tail 20_5-14_H_0_bwaaln.sam 
J00160:63:HHHT2BBXX:4:2228:14296:49089	77	*	CTCTGCCCCGACGGCCGGGTATAGGCGGCACGCTCAGCGCCATCCATTTTCAGGGCTAGTTGATTCGGCAGGTGAGTTGTTACACACTCCT	JJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJFFJJJJJJFFJJJJJJJJ7FJJJJJJJJJFJJF	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:14296:49089	141	*	TCGGAATCCGCTAAGGAGTGTGTAACAACTCACCTGCCGAATCAACTAGCCCTGAAAATGGATGGCGCTGAGCGTGCCGCCTATACCCGGC	FJJJAJJJJJJJJFJJFFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJ&lt;FJJJJ-FJJJJJJJJJJAJJJJJJJFJJJJJAJJJJJF	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:15026:49089	77	*	TTTTTCGTCACTACCTCCCCGTGTCGGGAGTGGGTAATTTGCGCGCCTGCTGCCTTCCTTGGATGTGGTAGCCGTTTCTCAAGCTCCCTC	JJJJJJJJFJJJJFJAJJJJFAJJJAJJAJ7FJJ&lt;AJJFJFAJFAFJ&lt;&lt;JFAJJFJJJJJAF&lt;AJFFJ-FJFJJFJFAJJJJFJJJFJJF	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:15026:49089	141	*	GGTTCGATTCCGGAGAGGGAGCTTGAGAAACGGCTACCACATCCAAGGAAGGCAGCAGGCGCGCAAATTACCCACTCCCGACACGGGGAGG	JJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJ	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:22008:49089	83	TRINITY_DN30310_c1_g10_TRINITY_DN30310_c1_g10_i1_g.7248_m.7248	168	60	91M	=	70	-189CCTCGCTCCCCGGGCGAAAGGGAATCGGGTCAATATTCCCGAACCCGGAGGCGGAGCCCTCCGTCTTCGTGGCGGTCCGAGCTGTAAAGCG	JJFJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJ	RG:Z:20_5-14_H_0	XT:A:U	NM:i:0	SM:i:37	AM:i:37	X0:i:1	X1:i:0	XM:i:0	XO:i:0	XG:i:0	MD:Z:91
J00160:63:HHHT2BBXX:4:2228:22008:49089	163	TRINITY_DN30310_c1_g10_TRINITY_DN30310_c1_g10_i1_g.7248_m.7248	70	60	91M	=	168	189	CCATGTGAACAGCAGTTGTACATGGGTCAGTCGATCCTAAGCCCCAGGGAAGTTCCGCTCCGAGCGGAGGCGGGCGCCCCTCTCCATGTGA	FJJJJJJJJJJJJJJFFJJJJJJJJJFJJJJJJJJJJJFFJJJJJJJJJJFJFJJJJJJFFJJJJJJJJJJJJJJJJJJJFJJJJJJJFJA	RG:Z:20_5-14_H_0	XT:A:U	NM:i:0	SM:i:37	AM:i:37	X0:i:1	X1:i:0	XM:i:0	XO:i:0	XG:i:0	MD:Z:91
J00160:63:HHHT2BBXX:4:2228:24647:49089	77	*	ACGGGCGATGTGTGCGCATTCTAGGGCTTTGAGTTGTTCATGGGCATTTTCTTTTGCTCATTACTGCTGAATCCTGTTTCAAATGGGGCTA	JFJJJJFFJJJJJJJJJFJJJJJJJJJJFJJAJJJJJJJFJJJJJJJFJJJJJJJJJF&lt;JJJJJJJFAFJJFFFJJJJJJJJJJJJJ&lt;JJA	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:24647:49089	141	*	GGATAAGTGAGCTACAATCATAAATATAAGAATAAAAATATGTATGAATAATGAACTGATAGCCCCATTTGAAACAGGATTCAGCAGTAAT	AAJJJJ&lt;JJJJFFJJJJJJJJJJJJJJJJJJJFJJJJJJJJJJJFFJJJJJJJJJJ&lt;-FFFFFAFFFAFJJJJJJJJJJJFAJFJJFJJJJ	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:30168:49089	77	*	TCTTGAAATCTGTGGGTTTCTCGTATAGTTCAATTACAACAGGTCCTGGTTTCAACTCGTCCATTTCCATGAAGGCAAAACACTTGGTGCT	JJJFFJJJJJJJJJJJAJJJJJJJJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJFJJJFFAFAJFJJJJFFFJJJJFFJJJJFJJJFJJ	RG:Z:20_5-14_H_0
J00160:63:HHHT2BBXX:4:2228:30168:49089	141	*	GAGTTTAAGCATTTCAAAGTGAAAAAGCGCACTATCAGCACCAAGTGTTTTGCCTTCATGGAAATGGACGAGTTG	JJJJJJJJJJJJJFJJAJJJJJJJJJJJJJJJJFJJJJJJJJFJJJJJJJJJJJJJJJJJJJJJJJJJJJJJFAF	RG:Z:20_5-14_H_0
</pre></li><li><p>rerun <code>python countxpression_PE.py 20 35 countstats.txt ../results/20_5-14_H_0_bwaaln.sam &amp;</code></p></li><li><p>output; </p><ul><li><pre class='md-fences mock-cm' style='display:block;position:relative'>Filename	Total#Reads	NumContigsMatched	NumUnaligned	NumAligned	NumMultiAligned	NumSingleAligned	NumQualSingles	Wrongformat	PropQualAligned
../results/20_5-14_H_0_bwaaln.sam	22423912	1971	19988441	2435471	697899	1737572	1735825	0	0.077</pre></li></ul><p>​</p></li></ul></li></ul></li></ul></li></ol><p></p><p></p><p></p><hr /><p> <div id='id-section13'/><br/></p><h3><a name='header-n3016' class='md-header-anchor '></a>Page 13: 2017-02-22. Week 6, Day 10, RNA-seq and paper discussion; Dixon et al. 2016; Genomic determinants of coral heat tolerance across latitudes.</h3><p><strong>Notes:</strong></p><ul><li>Steve wants me to install DADI, it has dependencies. </li><li>day off from info updates </li></ul><p><strong>Glossary:</strong> </p><p>haplotypes: tightly linked genes</p><p>KOG: </p><p></p><h3><a name='header-n3034' class='md-header-anchor '></a>Coding: analysing gene expression data with DESEQ2 in R</h3><p></p><ol start='' ><li>Copy data files from server</li></ol><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">scp adnguyen@pbio381.uvm.edu:/data/project_data/DGE/* .</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre><ol start='2' ><li>path of new folder</li></ol><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">2017-02-22_DEseq2_tutorial.Rmd<span class="cm-tab">  </span>countsdata_trim.txt</span></pre></div><pre class=""><span style="padding-right: 0.1px;">DESeq2_exploreSSW_trim.R<span class="cm-tab">    </span>countstatsummary.txt</span></pre><pre class=""><span style="padding-right: 0.1px;">cols_data_trim.txt<span class="cm-tab">  </span><span class="cm-tab">    </span>explore_expression_data.R</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 66px;"></div></div></div></pre><p>​	Data and script files</p><ol start='3' ><li>​</li></ol><p></p><p></p><p></p><hr /><p> <div id='id-section14'/><br/></p><h3><a name='header-n3062' class='md-header-anchor '></a>Page 14: 2017-02-27: Week 7, Day 11, RNA-seq and paper discussion (Edwards et al. 2016; <em>PNAS</em>)-phylogeography; Scott  Edwards visit.</h3><p>glossary:</p><p>Coalescent- </p><p>Reticulation</p><p>purifying/background selection-</p><p>gene trees vs species- </p><p>recombination- </p><p>introgression- </p><p></p><p>incomplete lineage sorting. population size and time.</p><p>large population size and short time allows to undergo incomplete lineage sorting to be represented in a divergence event. </p><p></p><p>t/Ne</p><p>large t/Ne = resemble species tree</p><p>small t/Ne = Incomplete lineage sorting</p><p>SKeller: genomes are collections of evolutionary history (think of the problem as a sampling problem)</p><ul><li>How many samples do you have to grab? Is it a finite set of solutions? </li><li>How many histories are there? </li></ul><p>Mpespeni: Is there a saturation  such athat when you sample, the more sample doesn&#39;t improve the species tree? </p><p>Edwards: 5-10 loci is generally &quot;good enough&quot; to resolve species. </p><p>Assumption: we can estimate these gene trees accurately. Rapid radiations might be tough to figure out what the gene trees actually are. </p><p>Do we want to estimate the trees? Other statistics? Edwards likes gene trees. Hard to estimate gene trees from a SNP. How granular do you want to analyze the data? </p><p>Kilpatrick: Radiations deep or recent in time? </p><p>Edwards: Can detect heterogeneity in deep rapid radiations in one of his papers. You can have short branch lengths that happened a long time ago. Can you tell? Yes, with the right type of data. </p><p></p><p>Recombination! Another kind of reticulation: Causes gene boundaries to become blurry. </p><p></p><p><mark>Hudson&#39;s MS</mark> Check this program out for coalescent modeling for SNP data. </p><p></p><h3><a name='header-n3122' class='md-header-anchor '></a>Coding:</h3><ol start='' ><li><p>move new data</p><ul><li><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">scp adnguyen@pbio381.uvm.edu:/data/project_data/DGE/* . </span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre></li></ul></li><li><p>Deseq2</p><ul><li>go over 5 models</li></ul></li><li><p>Your models</p></li></ol><p>Cool way to sample a dataframe (I always forget)</p><pre class="md-fences md-end-block" lang="r"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-variable">dds</span> <span class="cm-arrow">&lt;-</span> <span class="cm-variable">dds</span>[<span class="cm-variable">sample</span>(<span class="cm-variable">nrow</span>(<span class="cm-variable">dds</span>), <span class="cm-number">1200</span>), ]</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-variable">dim</span>(<span class="cm-variable">dds</span>)</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-operator">&gt;</span>[<span class="cm-number">1</span>] <span class="cm-number">1200</span> &nbsp; <span class="cm-number">77</span></span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 63px;"></div></div></div></pre><p></p><p>So we randomly sampled a subset (10%) so we can construct models and run it more quickly. </p><p></p><p>Sort dataset based on p-values</p><pre class="md-fences md-end-block" lang="r"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-variable">res</span> <span class="cm-arrow">&lt;-</span> <span class="cm-variable">res</span>[<span class="cm-variable">order</span>(<span class="cm-variable">res</span><span class="cm-dollar">$</span><span class="cm-variable">padj</span>),]</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre><p></p><p></p><hr /><p> <div id='id-section15'/><br/></p><h3><a name='header-n3159' class='md-header-anchor '></a>Page 15: 2017-03-6. Week 8, Day 13, Population genomics; paper discussion- Gayral et al. 2013.</h3><p><strong>Glossary:</strong> </p><p>Paralog: Gene duplicate</p><p>pie: pairwise nucleotide diversity</p><p>SFS: Site frequency spectrum= histogram of allele frequencies</p><p>Ne = Effective population size</p><p></p><p><strong>Skeller info update: Population genomics</strong>: </p><ul><li>SNPs and lots of them (genome wide)</li><li>Sampling unit is individuals <em>within species</em></li></ul><p>Questions:</p><ol start='' ><li><p>What shapes population structure? </p></li><li><p>Diversity within and among populations</p></li><li><p>What is the effect of selection? : synonymous and nonsynonymous mutations</p><ul><li>positive or negative/purifying</li></ul></li><li><p>How does recombination promote diversity?</p></li></ol><p>Pipeline for analysis</p><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart8" width="708" xmlns="http://www.w3.org/2000/svg" height="711.8000030517578" viewBox="0 0 728 731.8000030517578" style="height: 771.8000030517578px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M345,138.8999992370606L344.5,163.4000015258789L344.5,188.4000015258789" marker-end="url(#arrowhead320)" style="fill:none"></path><defs><marker id="arrowhead320" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M344.5,229.4000015258789L344.5,254.4000015258789L345,279.9000022888184" marker-end="url(#arrowhead321)" style="fill:none"></path><defs><marker id="arrowhead321" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M345,398.30000076293936L344.5,422.8000030517578L344.5,447.8000030517578" marker-end="url(#arrowhead322)" style="fill:none"></path><defs><marker id="arrowhead322" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M278.79017857142856,488.8000030517578L165,524.3000030517578L165,559.8000030517578" marker-end="url(#arrowhead323)" style="fill:none"></path><defs><marker id="arrowhead323" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M165,600.8000030517578L165,625.8000030517578L165,650.8000030517578" marker-end="url(#arrowhead324)" style="fill:none"></path><defs><marker id="arrowhead324" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M410.20982142857144,488.8000030517578L524,524.3000030517578L524,559.8000030517578" marker-end="url(#arrowhead325)" style="fill:none"></path><defs><marker id="arrowhead325" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M524,600.8000030517578L524,625.8000030517578L524,650.8000030517578" marker-end="url(#arrowhead326)" style="fill:none"></path><defs><marker id="arrowhead326" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(165,524.3000030517578)" style="opacity: 1;"><g transform="translate(-72.5,-10.5)" class="label"><foreignObject width="145" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">transcriptomics</span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(524,524.3000030517578)" style="opacity: 1;"><g transform="translate(-91.5,-10.5)" class="label"><foreignObject width="183" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">population genomics</span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="A" transform="translate(344.5,79.20000076293945)" style="opacity: 1;"><polygon points="59.2,0 118.4,-59.2 59.2,-118.4 0,-59.2" rx="5" ry="5" transform="translate(-59.2,59.2)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-43.5,-10.5)"><foreignObject width="87" height="21"><div style="display: inline-block; white-space: nowrap;">Raw Reads</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(344.5,208.9000015258789)" style="opacity: 1;"><rect rx="5" ry="5" x="-43.5" y="-20.5" width="87" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-33.5,-10.5)"><foreignObject width="67" height="21"><div style="display: inline-block; white-space: nowrap;">cleaned</div></foreignObject></g></g></g><g class="node" id="C" transform="translate(344.5,338.60000228881836)" style="opacity: 1;"><polygon points="59.2,0 118.4,-59.2 59.2,-118.4 0,-59.2" rx="5" ry="5" transform="translate(-59.2,59.2)"></polygon><g class="label" transform="translate(0,0)"><g transform="translate(-43.5,-10.5)"><foreignObject width="87" height="21"><div style="display: inline-block; white-space: nowrap;">Assembled</div></foreignObject></g></g></g><g class="node" id="D" transform="translate(344.5,468.3000030517578)" style="opacity: 1;"><rect rx="0" ry="0" x="-68" y="-20.5" width="136" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-58,-10.5)"><foreignObject width="116" height="21"><div style="display: inline-block; white-space: nowrap;">Mapped reads</div></foreignObject></g></g></g><g class="node" id="E" transform="translate(165,580.3000030517578)" style="opacity: 1;"><rect rx="0" ry="0" x="-68" y="-20.5" width="136" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-58,-10.5)"><foreignObject width="116" height="21"><div style="display: inline-block; white-space: nowrap;">Count number</div></foreignObject></g></g></g><g class="node" id="P" transform="translate(165,671.3000030517578)" style="opacity: 1;"><rect rx="0" ry="0" x="-145" y="-20.5" width="290" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-135,-10.5)"><foreignObject width="270" height="21"><div style="display: inline-block; white-space: nowrap;">Differential Gene Expression</div></foreignObject></g></g></g><g class="node" id="G" transform="translate(524,580.3000030517578)" style="opacity: 1;"><rect rx="0" ry="0" x="-121" y="-20.5" width="242" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-111,-10.5)"><foreignObject width="222" height="21"><div style="display: inline-block; white-space: nowrap;">Call SNPs and Genotypes</div></foreignObject></g></g></g><g class="node" id="N" transform="translate(524,671.3000030517578)" style="opacity: 1;"><rect rx="5" ry="5" x="-164" y="-20.5" width="328" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-154,-10.5)"><foreignObject width="308" height="21"><div style="display: inline-block; white-space: nowrap;">Allele Frequencies or SFS or PIE</div></foreignObject></g></g></g></g></g></g></svg></div><h3><a name='header-n3203' class='md-header-anchor '></a>A. What are the challenges of calling SNPs?</h3><p> <u>Sequencing Error (Illumina 1%) ; 1 in  100 there is a sequencing error</u><br/></p><ul><li>Apply filter: minor allele frequency </li><li>Apply filter: Check the depth </li></ul><h3><a name='header-n3214' class='md-header-anchor '></a>B. What are the challenges with calling genotypes?</h3><p> <u>Calling homozygous (AA), heterozygous (AT), homozygous (TT)? It is a multinomial distribution</u><br/></p><ul><li><p>If genotype is AT heterozygous, predict A = T = 0.5</p></li><li><p>use multinomial dsitribution to say how likely it is a heterozygoute. Each genotypic class has a probability</p></li><li><table><thead><tr><th>genotype</th><th>probability (referred to as genotype likelihood or genotype prob)</th></tr></thead><tbody><tr><td>AA</td><td>low</td></tr><tr><td>AT</td><td>high</td></tr><tr><td>TT</td><td>low</td></tr></tbody></table></li></ul><p>Bayesian folks: Use those probabilities and propogate those uncertainties to further analyses. (Not common but field is going that way)</p><p> <u>Another problem: paralogues; duplicated gene on separate genome or in synteny</u><br/></p><ul><li>Snps can differ between paralogues; it would predict heterozygotes, but that isnt true because there are 2 copies!</li></ul><ul><li>HWE = 1 = <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-1-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="13.529ex" height="2.755ex" viewBox="-38.5 -921.5 5825 1186.2" role="img" focusable="false" style="vertical-align: -0.615ex; margin-left: -0.089ex;"><defs><path stroke-width="1" id="E1-MJMATHI-70" d="M23 287Q24 290 25 295T30 317T40 348T55 381T75 411T101 433T134 442Q209 442 230 378L240 387Q302 442 358 442Q423 442 460 395T497 281Q497 173 421 82T249 -10Q227 -10 210 -4Q199 1 187 11T168 28L161 36Q160 35 139 -51T118 -138Q118 -144 126 -145T163 -148H188Q194 -155 194 -157T191 -175Q188 -187 185 -190T172 -194Q170 -194 161 -194T127 -193T65 -192Q-5 -192 -24 -194H-32Q-39 -187 -39 -183Q-37 -156 -26 -148H-6Q28 -147 33 -136Q36 -130 94 103T155 350Q156 355 156 364Q156 405 131 405Q109 405 94 377T71 316T59 280Q57 278 43 278H29Q23 284 23 287ZM178 102Q200 26 252 26Q282 26 310 49T356 107Q374 141 392 215T411 325V331Q411 405 350 405Q339 405 328 402T306 393T286 380T269 365T254 350T243 336T235 326L232 322Q232 321 229 308T218 264T204 212Q178 106 178 102Z"></path><path stroke-width="1" id="E1-MJMAIN-32" d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"></path><path stroke-width="1" id="E1-MJMAIN-2B" d="M56 237T56 250T70 270H369V420L370 570Q380 583 389 583Q402 583 409 568V270H707Q722 262 722 250T707 230H409V-68Q401 -82 391 -82H389H387Q375 -82 369 -68V230H70Q56 237 56 250Z"></path><path stroke-width="1" id="E1-MJMATHI-71" d="M33 157Q33 258 109 349T280 441Q340 441 372 389Q373 390 377 395T388 406T404 418Q438 442 450 442Q454 442 457 439T460 434Q460 425 391 149Q320 -135 320 -139Q320 -147 365 -148H390Q396 -156 396 -157T393 -175Q389 -188 383 -194H370Q339 -192 262 -192Q234 -192 211 -192T174 -192T157 -193Q143 -193 143 -185Q143 -182 145 -170Q149 -154 152 -151T172 -148Q220 -148 230 -141Q238 -136 258 -53T279 32Q279 33 272 29Q224 -10 172 -10Q117 -10 75 30T33 157ZM352 326Q329 405 277 405Q242 405 210 374T160 293Q131 214 119 129Q119 126 119 118T118 106Q118 61 136 44T179 26Q233 26 290 98L298 109L352 326Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E1-MJMATHI-70" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E1-MJMAIN-32" x="712" y="513"></use><use xlink:href="#E1-MJMAIN-2B" x="1179" y="0"></use><use xlink:href="#E1-MJMAIN-32" x="2180" y="0"></use><use xlink:href="#E1-MJMATHI-70" x="2680" y="0"></use><use xlink:href="#E1-MJMATHI-71" x="3184" y="0"></use><use xlink:href="#E1-MJMAIN-2B" x="3867" y="0"></use><g transform="translate(4867,0)"><use xlink:href="#E1-MJMATHI-71" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E1-MJMAIN-32" x="657" y="513"></use></g></g></svg></span><script type="math/tex" id="MathJax-Element-1">p^2 + 2pq + q^2</script></li><li>Look at heterozygous; it will violate HWE and get rid of them.  <strong>NOT ACTUALLY A SNP</strong></li></ul><h3><a name='header-n3255' class='md-header-anchor '></a>C. Diversity metrics</h3><ul><li><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-2-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.332ex" height="1.331ex" viewBox="0 -483.6 573.5 573.2" role="img" focusable="false" style="vertical-align: -0.208ex;"><defs><path stroke-width="1" id="E2-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E2-MJMATHI-3C0" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-2">\pi</script> = expected heterozygosity</p><ul><li><ul><li>for sequences <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-3-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="0.802ex" height="1.941ex" viewBox="0 -746.3 345.5 835.9" role="img" focusable="false" style="vertical-align: -0.208ex;"><defs><path stroke-width="1" id="E3-MJMATHI-69" d="M184 600Q184 624 203 642T247 661Q265 661 277 649T290 619Q290 596 270 577T226 557Q211 557 198 567T184 600ZM21 287Q21 295 30 318T54 369T98 420T158 442Q197 442 223 419T250 357Q250 340 236 301T196 196T154 83Q149 61 149 51Q149 26 166 26Q175 26 185 29T208 43T235 78T260 137Q263 149 265 151T282 153Q302 153 302 143Q302 135 293 112T268 61T223 11T161 -11Q129 -11 102 10T74 74Q74 91 79 106T122 220Q160 321 166 341T173 380Q173 404 156 404H154Q124 404 99 371T61 287Q60 286 59 284T58 281T56 279T53 278T49 278T41 278H27Q21 284 21 287Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E3-MJMATHI-69" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-3">i</script> and <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-4-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="0.985ex" height="2.348ex" viewBox="-11.5 -746.3 424 1011.1" role="img" focusable="false" style="vertical-align: -0.615ex; margin-left: -0.027ex;"><defs><path stroke-width="1" id="E4-MJMATHI-6A" d="M297 596Q297 627 318 644T361 661Q378 661 389 651T403 623Q403 595 384 576T340 557Q322 557 310 567T297 596ZM288 376Q288 405 262 405Q240 405 220 393T185 362T161 325T144 293L137 279Q135 278 121 278H107Q101 284 101 286T105 299Q126 348 164 391T252 441Q253 441 260 441T272 442Q296 441 316 432Q341 418 354 401T367 348V332L318 133Q267 -67 264 -75Q246 -125 194 -164T75 -204Q25 -204 7 -183T-12 -137Q-12 -110 7 -91T53 -71Q70 -71 82 -81T95 -112Q95 -148 63 -167Q69 -168 77 -168Q111 -168 139 -140T182 -74L193 -32Q204 11 219 72T251 197T278 308T289 365Q289 372 288 376Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E4-MJMATHI-6A" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-4">j</script>, <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-5-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="20.021ex" height="2.755ex" viewBox="0 -833.9 8620.1 1186.2" role="img" focusable="false" style="vertical-align: -0.818ex;"><defs><path stroke-width="1" id="E5-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path><path stroke-width="1" id="E5-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E5-MJSZ1-2211" d="M61 748Q64 750 489 750H913L954 640Q965 609 976 579T993 533T999 516H979L959 517Q936 579 886 621T777 682Q724 700 655 705T436 710H319Q183 710 183 709Q186 706 348 484T511 259Q517 250 513 244L490 216Q466 188 420 134T330 27L149 -187Q149 -188 362 -188Q388 -188 436 -188T506 -189Q679 -189 778 -162T936 -43Q946 -27 959 6H999L913 -249L489 -250Q65 -250 62 -248Q56 -246 56 -239Q56 -234 118 -161Q186 -81 245 -11L428 206Q428 207 242 462L57 717L56 728Q56 744 61 748Z"></path><path stroke-width="1" id="E5-MJMATHI-58" d="M42 0H40Q26 0 26 11Q26 15 29 27Q33 41 36 43T55 46Q141 49 190 98Q200 108 306 224T411 342Q302 620 297 625Q288 636 234 637H206Q200 643 200 645T202 664Q206 677 212 683H226Q260 681 347 681Q380 681 408 681T453 682T473 682Q490 682 490 671Q490 670 488 658Q484 643 481 640T465 637Q434 634 411 620L488 426L541 485Q646 598 646 610Q646 628 622 635Q617 635 609 637Q594 637 594 648Q594 650 596 664Q600 677 606 683H618Q619 683 643 683T697 681T738 680Q828 680 837 683H845Q852 676 852 672Q850 647 840 637H824Q790 636 763 628T722 611T698 593L687 584Q687 585 592 480L505 384Q505 383 536 304T601 142T638 56Q648 47 699 46Q734 46 734 37Q734 35 732 23Q728 7 725 4T711 1Q708 1 678 1T589 2Q528 2 496 2T461 1Q444 1 444 10Q444 11 446 25Q448 35 450 39T455 44T464 46T480 47T506 54Q523 62 523 64Q522 64 476 181L429 299Q241 95 236 84Q232 76 232 72Q232 53 261 47Q262 47 267 47T273 46Q276 46 277 46T280 45T283 42T284 35Q284 26 282 19Q279 6 276 4T261 1Q258 1 243 1T201 2T142 2Q64 2 42 0Z"></path><path stroke-width="1" id="E5-MJMATHI-69" d="M184 600Q184 624 203 642T247 661Q265 661 277 649T290 619Q290 596 270 577T226 557Q211 557 198 567T184 600ZM21 287Q21 295 30 318T54 369T98 420T158 442Q197 442 223 419T250 357Q250 340 236 301T196 196T154 83Q149 61 149 51Q149 26 166 26Q175 26 185 29T208 43T235 78T260 137Q263 149 265 151T282 153Q302 153 302 143Q302 135 293 112T268 61T223 11T161 -11Q129 -11 102 10T74 74Q74 91 79 106T122 220Q160 321 166 341T173 380Q173 404 156 404H154Q124 404 99 371T61 287Q60 286 59 284T58 281T56 279T53 278T49 278T41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E5-MJMAIN-2217" d="M229 286Q216 420 216 436Q216 454 240 464Q241 464 245 464T251 465Q263 464 273 456T283 436Q283 419 277 356T270 286L328 328Q384 369 389 372T399 375Q412 375 423 365T435 338Q435 325 425 315Q420 312 357 282T289 250L355 219L425 184Q434 175 434 161Q434 146 425 136T401 125Q393 125 383 131T328 171L270 213Q283 79 283 63Q283 53 276 44T250 35Q231 35 224 44T216 63Q216 80 222 143T229 213L171 171Q115 130 110 127Q106 124 100 124Q87 124 76 134T64 161Q64 166 64 169T67 175T72 181T81 188T94 195T113 204T138 215T170 230T210 250L74 315Q65 324 65 338Q65 353 74 363T98 374Q106 374 116 368T171 328L229 286Z"></path><path stroke-width="1" id="E5-MJMATHI-6A" d="M297 596Q297 627 318 644T361 661Q378 661 389 651T403 623Q403 595 384 576T340 557Q322 557 310 567T297 596ZM288 376Q288 405 262 405Q240 405 220 393T185 362T161 325T144 293L137 279Q135 278 121 278H107Q101 284 101 286T105 299Q126 348 164 391T252 441Q253 441 260 441T272 442Q296 441 316 432Q341 418 354 401T367 348V332L318 133Q267 -67 264 -75Q246 -125 194 -164T75 -204Q25 -204 7 -183T-12 -137Q-12 -110 7 -91T53 -71Q70 -71 82 -81T95 -112Q95 -148 63 -167Q69 -168 77 -168Q111 -168 139 -140T182 -74L193 -32Q204 11 219 72T251 197T278 308T289 365Q289 372 288 376Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E5-MJMATHI-3C0" x="0" y="0"></use><use xlink:href="#E5-MJMAIN-3D" x="851" y="0"></use><use xlink:href="#E5-MJSZ1-2211" x="1907" y="0"></use><g transform="translate(3130,0)"><use xlink:href="#E5-MJMATHI-58" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E5-MJMATHI-69" x="1171" y="-213"></use></g><use xlink:href="#E5-MJMAIN-2217" x="4525" y="0"></use><g transform="translate(5248,0)"><use xlink:href="#E5-MJMATHI-58" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E5-MJMATHI-6A" x="1171" y="-213"></use></g><use xlink:href="#E5-MJMAIN-2217" x="6690" y="0"></use><g transform="translate(7413,0)"><use xlink:href="#E5-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(570,-150)"><use transform="scale(0.707)" xlink:href="#E5-MJMATHI-69" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E5-MJMATHI-6A" x="345" y="0"></use></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-5">\pi = \sum X_i * X_j * \pi_{ij}</script> </li><li>can be scaled by the number of sites</li><li><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-6-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="17.331ex" height="2.552ex" viewBox="0 -746.3 7461.8 1098.6" role="img" focusable="false" style="vertical-align: -0.818ex;"><defs><path stroke-width="1" id="E6-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path><path stroke-width="1" id="E6-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E6-MJMATHI-79" d="M21 287Q21 301 36 335T84 406T158 442Q199 442 224 419T250 355Q248 336 247 334Q247 331 231 288T198 191T182 105Q182 62 196 45T238 27Q261 27 281 38T312 61T339 94Q339 95 344 114T358 173T377 247Q415 397 419 404Q432 431 462 431Q475 431 483 424T494 412T496 403Q496 390 447 193T391 -23Q363 -106 294 -155T156 -205Q111 -205 77 -183T43 -117Q43 -95 50 -80T69 -58T89 -48T106 -45Q150 -45 150 -87Q150 -107 138 -122T115 -142T102 -147L99 -148Q101 -153 118 -160T152 -167H160Q177 -167 186 -165Q219 -156 247 -127T290 -65T313 -9T321 21L315 17Q309 13 296 6T270 -6Q250 -11 231 -11Q185 -11 150 11T104 82Q103 89 103 113Q103 170 138 262T173 379Q173 380 173 381Q173 390 173 393T169 400T158 404H154Q131 404 112 385T82 344T65 302T57 280Q55 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E6-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E6-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E6-MJMAIN-34" d="M462 0Q444 3 333 3Q217 3 199 0H190V46H221Q241 46 248 46T265 48T279 53T286 61Q287 63 287 115V165H28V211L179 442Q332 674 334 675Q336 677 355 677H373L379 671V211H471V165H379V114Q379 73 379 66T385 54Q393 47 442 46H471V0H462ZM293 211V545L74 212L183 211H293Z"></path><path stroke-width="1" id="E6-MJMAIN-2217" d="M229 286Q216 420 216 436Q216 454 240 464Q241 464 245 464T251 465Q263 464 273 456T283 436Q283 419 277 356T270 286L328 328Q384 369 389 372T399 375Q412 375 423 365T435 338Q435 325 425 315Q420 312 357 282T289 250L355 219L425 184Q434 175 434 161Q434 146 425 136T401 125Q393 125 383 131T328 171L270 213Q283 79 283 63Q283 53 276 44T250 35Q231 35 224 44T216 63Q216 80 222 143T229 213L171 171Q115 130 110 127Q106 124 100 124Q87 124 76 134T64 161Q64 166 64 169T67 175T72 181T81 188T94 195T113 204T138 215T170 230T210 250L74 315Q65 324 65 338Q65 353 74 363T98 374Q106 374 116 368T171 328L229 286Z"></path><path stroke-width="1" id="E6-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E6-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path><path stroke-width="1" id="E6-MJMATHI-3BC" d="M58 -216Q44 -216 34 -208T23 -186Q23 -176 96 116T173 414Q186 442 219 442Q231 441 239 435T249 423T251 413Q251 401 220 279T187 142Q185 131 185 107V99Q185 26 252 26Q261 26 270 27T287 31T302 38T315 45T327 55T338 65T348 77T356 88T365 100L372 110L408 253Q444 395 448 404Q461 431 491 431Q504 431 512 424T523 412T525 402L449 84Q448 79 448 68Q448 43 455 35T476 26Q485 27 496 35Q517 55 537 131Q543 151 547 152Q549 153 557 153H561Q580 153 580 144Q580 138 575 117T555 63T523 13Q510 0 491 -8Q483 -10 467 -10Q446 -10 429 -4T402 11T385 29T376 44T374 51L368 45Q362 39 350 30T324 12T288 -4T246 -11Q199 -11 153 12L129 -85Q108 -167 104 -180T92 -202Q76 -216 58 -216Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E6-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(570,-150)"><use transform="scale(0.707)" xlink:href="#E6-MJMATHI-73" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E6-MJMATHI-79" x="469" y="0"></use><use transform="scale(0.707)" xlink:href="#E6-MJMATHI-6E" x="966" y="0"></use></g><use xlink:href="#E6-MJMAIN-3D" x="2056" y="0"></use><use xlink:href="#E6-MJMAIN-34" x="3112" y="0"></use><use xlink:href="#E6-MJMAIN-2217" x="3835" y="0"></use><use xlink:href="#E6-MJMATHI-4E" x="4558" y="0"></use><use xlink:href="#E6-MJMATHI-65" x="5446" y="0"></use><use xlink:href="#E6-MJMAIN-2217" x="6135" y="0"></use><use xlink:href="#E6-MJMATHI-3BC" x="6858" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-6">\pi_{syn} = 4 *Ne * \mu</script>  (neutral genetic change)</li><li><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-28-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="6.901ex" height="1.941ex" viewBox="0 -483.6 2971.4 835.9" role="img" focusable="false" style="vertical-align: -0.818ex;"><defs><path stroke-width="1" id="E28-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path><path stroke-width="1" id="E28-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E28-MJMATHI-6F" d="M201 -11Q126 -11 80 38T34 156Q34 221 64 279T146 380Q222 441 301 441Q333 441 341 440Q354 437 367 433T402 417T438 387T464 338T476 268Q476 161 390 75T201 -11ZM121 120Q121 70 147 48T206 26Q250 26 289 58T351 142Q360 163 374 216T388 308Q388 352 370 375Q346 405 306 405Q243 405 195 347Q158 303 140 230T121 120Z"></path><path stroke-width="1" id="E28-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E28-MJMATHI-79" d="M21 287Q21 301 36 335T84 406T158 442Q199 442 224 419T250 355Q248 336 247 334Q247 331 231 288T198 191T182 105Q182 62 196 45T238 27Q261 27 281 38T312 61T339 94Q339 95 344 114T358 173T377 247Q415 397 419 404Q432 431 462 431Q475 431 483 424T494 412T496 403Q496 390 447 193T391 -23Q363 -106 294 -155T156 -205Q111 -205 77 -183T43 -117Q43 -95 50 -80T69 -58T89 -48T106 -45Q150 -45 150 -87Q150 -107 138 -122T115 -142T102 -147L99 -148Q101 -153 118 -160T152 -167H160Q177 -167 186 -165Q219 -156 247 -127T290 -65T313 -9T321 21L315 17Q309 13 296 6T270 -6Q250 -11 231 -11Q185 -11 150 11T104 82Q103 89 103 113Q103 170 138 262T173 379Q173 380 173 381Q173 390 173 393T169 400T158 404H154Q131 404 112 385T82 344T65 302T57 280Q55 278 41 278H27Q21 284 21 287Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E28-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(570,-150)"><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-6E" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-6F" x="600" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-6E" x="1086" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-73" x="1686" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-79" x="2156" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-6E" x="2653" y="0"></use></g></g></svg></span><script type="math/tex" id="MathJax-Element-28">\pi_{nonsyn}</script></li><li><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-8-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="5.716ex" height="3.67ex" viewBox="0 -965.3 2461.1 1580.3" role="img" focusable="false" style="vertical-align: -1.428ex;"><defs><path stroke-width="1" id="E8-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path><path stroke-width="1" id="E8-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E8-MJMATHI-6F" d="M201 -11Q126 -11 80 38T34 156Q34 221 64 279T146 380Q222 441 301 441Q333 441 341 440Q354 437 367 433T402 417T438 387T464 338T476 268Q476 161 390 75T201 -11ZM121 120Q121 70 147 48T206 26Q250 26 289 58T351 142Q360 163 374 216T388 308Q388 352 370 375Q346 405 306 405Q243 405 195 347Q158 303 140 230T121 120Z"></path><path stroke-width="1" id="E8-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E8-MJMATHI-79" d="M21 287Q21 301 36 335T84 406T158 442Q199 442 224 419T250 355Q248 336 247 334Q247 331 231 288T198 191T182 105Q182 62 196 45T238 27Q261 27 281 38T312 61T339 94Q339 95 344 114T358 173T377 247Q415 397 419 404Q432 431 462 431Q475 431 483 424T494 412T496 403Q496 390 447 193T391 -23Q363 -106 294 -155T156 -205Q111 -205 77 -183T43 -117Q43 -95 50 -80T69 -58T89 -48T106 -45Q150 -45 150 -87Q150 -107 138 -122T115 -142T102 -147L99 -148Q101 -153 118 -160T152 -167H160Q177 -167 186 -165Q219 -156 247 -127T290 -65T313 -9T321 21L315 17Q309 13 296 6T270 -6Q250 -11 231 -11Q185 -11 150 11T104 82Q103 89 103 113Q103 170 138 262T173 379Q173 380 173 381Q173 390 173 393T169 400T158 404H154Q131 404 112 385T82 344T65 302T57 280Q55 278 41 278H27Q21 284 21 287Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g transform="translate(120,0)"><rect stroke="none" width="2221" height="60" x="0" y="220"></rect><g transform="translate(60,601)"><use transform="scale(0.707)" xlink:href="#E8-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(403,-107)"><use transform="scale(0.5)" xlink:href="#E8-MJMATHI-6E" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E8-MJMATHI-6F" x="600" y="0"></use><use transform="scale(0.5)" xlink:href="#E8-MJMATHI-6E" x="1086" y="0"></use><use transform="scale(0.5)" xlink:href="#E8-MJMATHI-73" x="1686" y="0"></use><use transform="scale(0.5)" xlink:href="#E8-MJMATHI-79" x="2156" y="0"></use><use transform="scale(0.5)" xlink:href="#E8-MJMATHI-6E" x="2653" y="0"></use></g></g><g transform="translate(481,-345)"><use transform="scale(0.707)" xlink:href="#E8-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(403,-107)"><use transform="scale(0.5)" xlink:href="#E8-MJMATHI-73" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E8-MJMATHI-79" x="469" y="0"></use><use transform="scale(0.5)" xlink:href="#E8-MJMATHI-6E" x="967" y="0"></use></g></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-8">\frac{\pi_{nonsyn}}{\pi_{syn}}</script> estimates the strength of selection</li><li>What drives Ne up and down? </li></ul></li></ul></li></ul><h3><a name='header-n3282' class='md-header-anchor '></a>Paper discussion: Gayral et al. 2013</h3><p>Reference free population genomics study</p><p>Expectation:<br/></p><blockquote><p>Our expectation is that small-Ne species should show a lower pN, a lower pS, and a higher pN/pS ratio than large-Ne species. This is because genetic drift, which is enhanced in small populations, is expected to reduce the neutral and selected levels of genomic diversity, but to increase the relative probability of slightly deleterious, non-synonymous mutations (relatively to neutral, synonymous mutations) segregating at observable frequency.</p></blockquote><p></p><p>Where do mutations happen? During meiosis. In a big population there is more meosis. Mice in the sewers of NYC compared to a black rhino populations. They are no where close!</p><p></p><p>Paralogue filtering: nothing to map to so it is hard to ID&#39;ing a paralogue.</p><p>Figure 2: SFS differs in test method than standard method (samtools). New method can id more rare alleles. New method can call heterozygotes is better.  Rare alleles occur more often in heterozygote individuals. </p><p></p><h3><a name='header-n3302' class='md-header-anchor '></a>2017-03-06 coding session; Population genomics</h3><p>Mistake: pseudoreplication in calling SNPs because multiple individuals are represented under varying conditions. So we need to analyze and call SNPs at the individual level and not smaple level (93)</p><p>Approach to fix:</p><ol start='' ><li>take same files and merge them for every individual.</li><li>Call SNPs separately and then compare SNPs across replicates. </li></ol><p>use vcftools:</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">$ vcftools</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">VCFtools (0.1.14)</span></pre><pre class=""><span style="padding-right: 0.1px;">© Adam Auton and Anthony Marcketta 2009</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">Process Variant Call Format files</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">For a list of options, please go to:</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>https://vcftools.github.io/man_latest.html</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">Alternatively, a man page is available, type:</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>man vcftools</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">Questions, comments, and suggestions should be emailed to:</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>vcftools-help@lists.sourceforge.net</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 318px;"></div></div></div></pre><p>move into directory with data:</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">cd /data/project_data/snps/reads2snps/</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre><p>check out a portion of the vcf file that has called SNPs</p><pre class='md-fences mock-cm' style='display:block;position:relative'>head head_SSW_bamlist.txt.vcf 

##fileformat=VCFv4.0
##source=reads2snp
##phasing=unphased
##FILTER= &lt;ID=multi,Description=&quot;At least one individual shows more than 2 alleles&quot;&gt;  

##FILTER= &lt;ID=unres,Description=&quot;Insufficient sequencing depth or uncertain genotyping in all individuals&quot;&gt;  

##FILTER= &lt;ID=para,Description=&quot;Suspicion of paralogy&quot;&gt;  

##INFO= &lt;ID=N,Number=1,Type=Integer,Description=&quot;Number of genotyped individuals&quot;&gt;  

##FORMAT= &lt;ID=GT,Number=1,Type=String,Description=&quot;Genotype&quot;&gt;  

#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	03_5-08_S_2	07_5-08_S_1	08_5-08_H_0	09_5-08_H_0	10_5-08_H_0	14_5-08_S_2	15_5-08_H_0	19_5-11_H_0	20_5-08_H_0	22_5-08_S_1	23_5-17_S_2	24_5-08_H_0	26_5-08_S_2	27_5-08_H_0	28_5-08_S_1	29_5-08_S_2	31_6-12_H_0	32_6-12_H_0	33_6-12_H_0	34_6-12_H_0	35_6-12_H_0	36_6-12_S_1	37_6-12_H_0	38_6-12_H_0
TRINITY_DN47185_c0_g1_TRINITY_DN47185_c0_g1_i2_g.24943_m.24943	1	.	unres	N=0	GT	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.	.|.
</pre><p>unres = unresolved. </p><p>depth filter of 10 reads per genotype call and filters out genotype prob calls less than prob of 95%.</p><p></p><p>Summarize data vcf file for us:</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">vcftools --vcf SSW_bamlist.txt.vcf </span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre><p>output</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre></div><pre class=""><span style="padding-right: 0.1px;">VCFtools - 0.1.14</span></pre><pre class=""><span style="padding-right: 0.1px;">(C) Adam Auton and Anthony Marcketta 2009</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">Parameters as interpreted:</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--vcf SSW_bamlist.txt.vcf</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 24 out of 24 Individuals</span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 7472775 out of a possible 7472775 Sites</span></pre><pre class=""><span style="padding-right: 0.1px;">Run Time = 22.00 seconds</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 211px;"></div></div></div></pre><p>Check number of individuals. Some are not real. </p><p>Finding the number of unresolved:</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"> grep "unres" SSW_bamlist.txt.vcf | wc</span></pre></div><pre class=""><span style="padding-right: 0.1px;"> 5631864 185851488 1028494934</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 42px;"></div></div></div></pre><p>5.6 million are unresolved!!</p><p>Check for paralogy</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">grep "para" SSW_bamlist.txt.vcf | wc</span></pre></div><pre class=""><span style="padding-right: 0.1px;"> &nbsp; 4354  143652  795592</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 42px;"></div></div></div></pre><p>4,354 incidence of paralogy</p><p> <u>1.8 million SNPs left!!!</u><br/></p><p></p><p>More filtering: </p><ol start='' ><li>biallelic vs multi-allelic SNPs; filter number of alleles; min number = 2 and max = 2</li></ol><pre class='md-fences mock-cm' style='display:block;position:relative'>$ vcftools --vcf SSW_bamlist.txt.vcf --min-alleles 2 --max-alleles 2

VCFtools - 0.1.14
(C) Adam Auton and Anthony Marcketta 2009

Parameters as interpreted:
	--vcf SSW_bamlist.txt.vcf
	--max-alleles 2
	--min-alleles 2

After filtering, kept 24 out of 24 Individuals
After filtering, kept 20319 out of a possible 7472775 Sites
Run Time = 19.00 seconds</pre><p></p><ol start='2' ><li>Minor allele frequency (MAF):  gets rid of very rare SNPs (based on user-defined threshold)</li></ol><p>1/48 = 0.02</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">$ vcftools --vcf SSW_bamlist.txt.vcf --maf 0.02</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">VCFtools - 0.1.14</span></pre><pre class=""><span style="padding-right: 0.1px;">(C) Adam Auton and Anthony Marcketta 2009</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">Parameters as interpreted:</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--vcf SSW_bamlist.txt.vcf</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--maf 0.02</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 24 out of 24 Individuals</span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 5656584 out of a possible 7472775 Sites</span></pre><pre class=""><span style="padding-right: 0.1px;">Run Time = 70.00 seconds</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 254px;"></div></div></div></pre><p>retained 5.6 million SNPs</p><ol start='3' ><li>Proportion of missing data. Get rid of sites fewer than 80% of our samples have data. Allow 20% of missing data.</li></ol><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">$ vcftools --vcf SSW_bamlist.txt.vcf --max-missing 0.80</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">VCFtools - 0.1.14</span></pre><pre class=""><span style="padding-right: 0.1px;">(C) Adam Auton and Anthony Marcketta 2009</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">Parameters as interpreted:</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--vcf SSW_bamlist.txt.vcf</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--max-missing 0.8</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 24 out of 24 Individuals</span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 100219 out of a possible 7472775 Sites</span></pre><pre class=""><span style="padding-right: 0.1px;">Run Time = 68.00 seconds</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 254px;"></div></div></div></pre><p><strong>OK! Now we can combine filters!</strong> and output it into a file</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">$ vcftools --vcf SSW_bamlist.txt.vcf --min-alleles 2 --max-alleles 2 --maf 0.02 --max-missing 0.8 --recode --out ~/mydata/2017-03-06_SNPcallfilter</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">VCFtools - 0.1.14</span></pre><pre class=""><span style="padding-right: 0.1px;">(C) Adam Auton and Anthony Marcketta 2009</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">Parameters as interpreted:</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--vcf SSW_bamlist.txt.vcf</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--maf 0.02</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--max-alleles 2</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--min-alleles 2</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--max-missing 0.8</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--out /users/a/d/adnguyen/mydata/2017-03-06_SNPcallfilter</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--recode</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 24 out of 24 Individuals</span></pre><pre class=""><span style="padding-right: 0.1px;">Outputting VCF file...</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 364px;"></div></div></div></pre><p>Output of file</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">2017-03-06_SNPcallfilter.log</span></pre></div><pre class=""><span style="padding-right: 0.1px;">2017-03-06_SNPcallfilter.recode.vcf</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 42px;"></div></div></div></pre><p></p><p>What the files look like: </p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">head 2017-03-06_SNPcallfilter.recode.vcf </span></pre></div><pre class=""><span style="padding-right: 0.1px;">##fileformat=VCFv4.0</span></pre><pre class=""><span style="padding-right: 0.1px;">##source=reads2snp</span></pre><pre class=""><span style="padding-right: 0.1px;">##phasing=unphased</span></pre><pre class=""><span style="padding-right: 0.1px;">##FILTER= &lt;ID=multi,Description="At least one individual shows more than 2 alleles"&gt;  </span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">##FILTER= &lt;ID=unres,Description="Insufficient sequencing depth or uncertain genotyping in all individuals"&gt;  </span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">##FILTER= &lt;ID=para,Description="Suspicion of paralogy"&gt;  </span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">##INFO= &lt;ID=N,Number=1,Type=Integer,Description="Number of genotyped individuals"&gt;  </span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">##FORMAT= &lt;ID=GT,Number=1,Type=String,Description="Genotype"&gt;  </span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">#CHROM<span class="cm-tab">  </span>POS<span class="cm-tab"> </span>ID<span class="cm-tab">  </span>REF<span class="cm-tab"> </span>ALT<span class="cm-tab"> </span>QUAL<span class="cm-tab">    </span>FILTER<span class="cm-tab">  </span>INFO<span class="cm-tab">    </span>FORMAT<span class="cm-tab">  </span>03_5-08_S_2<span class="cm-tab"> </span>07_5-08_S_1<span class="cm-tab"> </span>08_5-08_H_0<span class="cm-tab"> </span>09_5-08_H_0<span class="cm-tab"> </span>10_5-08_H_0<span class="cm-tab"> </span>14_5-08_S_2<span class="cm-tab"> </span>15_5-08_H_0<span class="cm-tab"> </span>19_5-11_H_0<span class="cm-tab"> </span>20_5-08_H_0<span class="cm-tab"> </span>22_5-08_S_1<span class="cm-tab"> </span>23_5-17_S_2<span class="cm-tab"> </span>24_5-08_H_0<span class="cm-tab"> </span>26_5-08_S_2<span class="cm-tab"> </span>27_5-08_H_0<span class="cm-tab"> </span>28_5-08_S_1<span class="cm-tab"> </span>29_5-08_S_2<span class="cm-tab"> </span>31_6-12_H_0<span class="cm-tab"> </span>32_6-12_H_0<span class="cm-tab"> </span>33_6-12_H_0<span class="cm-tab"> </span>34_6-12_H_0<span class="cm-tab"> </span>35_6-12_H_0<span class="cm-tab"> </span>36_6-12_S_1<span class="cm-tab"> </span>37_6-12_H_0<span class="cm-tab"> </span>38_6-12_H_0</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680<span class="cm-tab">  </span>2127<span class="cm-tab">    </span>.<span class="cm-tab">   </span>PASS<span class="cm-tab">    </span>.<span class="cm-tab">   </span>GT<span class="cm-tab">  </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>.<span class="cm-tab">   </span>0|0<span class="cm-tab"> </span>0|1<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>.<span class="cm-tab">   </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>0|0<span class="cm-tab"> </span>.<span class="cm-tab">   </span>0|0<span class="cm-tab"> </span>0|0</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 490px;"></div></div></div></pre><p></p><p>Looking at hardy weinburg equilibrium</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">$ vcftools --vcf 2017-03-06_SNPcallfilter.recode.vcf --hardy</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">VCFtools - 0.1.14</span></pre><pre class=""><span style="padding-right: 0.1px;">(C) Adam Auton and Anthony Marcketta 2009</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">Parameters as interpreted:</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--vcf 2017-03-06_SNPcallfilter.recode.vcf</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--hardy</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 24 out of 24 Individuals</span></pre><pre class=""><span style="padding-right: 0.1px;">Outputting HWE statistics (but only for biallelic loci)</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>HWE: Only using fully diploid SNPs.</span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 1180 out of a possible 1180 Sites</span></pre><pre class=""><span style="padding-right: 0.1px;">Run Time = 0.00 seconds</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 297px;"></div></div></div></pre><p>stored as:</p><pre class="md-fences md-end-block" lang="u"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">out.hwe</span></pre></div><pre class=""><span style="padding-right: 0.1px;">out.log</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 42px;"></div></div></div></pre><p></p><p>What out.hwe would look like:</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">head out.hwe </span></pre></div><pre class=""><span style="padding-right: 0.1px;">CHR<span class="cm-tab"> </span>POS<span class="cm-tab"> </span>OBS(HOM1/HET/HOM2)<span class="cm-tab">  </span>E(HOM1/HET/HOM2)<span class="cm-tab">    </span>ChiSq_HWE<span class="cm-tab">   </span>P_HWE<span class="cm-tab">   </span>P_HET_DEFICIT<span class="cm-tab">   </span>P_HET_EXCESS</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680<span class="cm-tab">  </span>4566<span class="cm-tab">    </span>23/1/0<span class="cm-tab">  </span>23.01/0.98/0.011.086464e-02<span class="cm-tab"> </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680<span class="cm-tab">  </span>4665<span class="cm-tab">    </span>21/3/0<span class="cm-tab">  </span>21.09/2.81/0.091.066667e-01<span class="cm-tab"> </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00<span class="cm-tab">    </span>9.361702e-01</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN46874_c5_g2_TRINITY_DN46874_c5_g2_i2_g.23776_m.23776<span class="cm-tab">  </span>978<span class="cm-tab"> </span>23/1/0<span class="cm-tab">  </span>23.01/0.98/0.011.086464e-02<span class="cm-tab"> </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN46874_c5_g2_TRINITY_DN46874_c5_g2_i2_g.23776_m.23776<span class="cm-tab">  </span>1404<span class="cm-tab">    </span>23/1/0<span class="cm-tab">  </span>23.01/0.98/0.011.086464e-02<span class="cm-tab"> </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN46874_c5_g2_TRINITY_DN46874_c5_g2_i2_g.23776_m.23776<span class="cm-tab">  </span>1722<span class="cm-tab">    </span>20/4/0<span class="cm-tab">  </span>20.17/3.67/0.171.983471e-01<span class="cm-tab"> </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00<span class="cm-tab">    </span>8.737589e-01</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN46874_c5_g2_TRINITY_DN46874_c5_g2_i2_g.23776_m.23776<span class="cm-tab">  </span>3426<span class="cm-tab">    </span>23/1/0<span class="cm-tab">  </span>23.01/0.98/0.011.086464e-02<span class="cm-tab"> </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN46874_c5_g2_TRINITY_DN46874_c5_g2_i2_g.23776_m.23776<span class="cm-tab">  </span>3729<span class="cm-tab">    </span>21/3/0<span class="cm-tab">  </span>21.09/2.81/0.091.066667e-01<span class="cm-tab"> </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00<span class="cm-tab">    </span>9.361702e-01</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN46874_c5_g2_TRINITY_DN46874_c5_g2_i2_g.23776_m.23776<span class="cm-tab">  </span>3912<span class="cm-tab">    </span>19/5/0<span class="cm-tab">  </span>19.26/4.48/0.263.244997e-01<span class="cm-tab"> </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00<span class="cm-tab">    </span>7.943262e-01</span></pre><pre class=""><span style="padding-right: 0.1px;">TRINITY_DN46347_c1_g3_TRINITY_DN46347_c1_g3_i1_g.21992_m.21992<span class="cm-tab">  </span>115<span class="cm-tab"> </span>19/5/0<span class="cm-tab">  </span>19.26/4.48/0.263.244997e-01<span class="cm-tab"> </span>1.000000e+00<span class="cm-tab">    </span>1.000000e+00<span class="cm-tab">    </span>7.943262e-01</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 460px;"></div></div></div></pre><p>Can do R in the command line!</p><pre class="md-fences md-end-block" lang="r"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-variable">b</span><span class="cm-arrow">&lt;-</span><span class="cm-variable">read.table</span>(<span class="cm-string">"out.hwe"</span>,<span class="cm-variable">header</span><span class="cm-arg-is">=</span><span class="cm-variable">TRUE</span>)</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre><p>Structure of data:</p><pre class="md-fences md-end-block" lang="r"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-operator">&gt;</span> <span class="cm-variable">str</span>(<span class="cm-variable">b</span>)</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-string">'data.frame'</span><span class="cm-operator">:</span><span class="cm-tab">   </span><span class="cm-number">442</span> <span class="cm-variable">obs.</span> <span class="cm-variable">of</span>  <span class="cm-number">8</span> <span class="cm-variable">variables</span><span class="cm-operator">:</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> <span class="cm-dollar">$</span> <span class="cm-variable">CHR</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <span class="cm-operator">:</span> <span class="cm-variable">Factor</span> <span class="cm-variable">w</span><span class="cm-operator">/</span> <span class="cm-number">111</span> <span class="cm-variable">levels</span> <span class="cm-string">"TRINITY_DN35598_c0_g1_TRINITY_DN35598_c0_g1_i1_g.5802_m.5802"</span>,<span class="cm-variable">..</span><span class="cm-operator">:</span> <span class="cm-number">65</span> <span class="cm-number">65</span> <span class="cm-number">100</span> <span class="cm-number">100</span> <span class="cm-number">100</span> <span class="cm-number">100</span> <span class="cm-number">100</span> <span class="cm-number">100</span> <span class="cm-number">88</span> <span class="cm-number">88</span> <span class="cm-keyword">...</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> <span class="cm-dollar">$</span> <span class="cm-variable">POS</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <span class="cm-operator">:</span> <span class="cm-variable">int</span>  <span class="cm-number">4566</span> <span class="cm-number">4665</span> <span class="cm-number">978</span> <span class="cm-number">1404</span> <span class="cm-number">1722</span> <span class="cm-number">3426</span> <span class="cm-number">3729</span> <span class="cm-number">3912</span> <span class="cm-number">115</span> <span class="cm-number">141</span> <span class="cm-keyword">...</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> <span class="cm-dollar">$</span> <span class="cm-variable">OBS.HOM1.HET.HOM2.</span><span class="cm-operator">:</span> <span class="cm-variable">Factor</span> <span class="cm-variable">w</span><span class="cm-operator">/</span> <span class="cm-number">27</span> <span class="cm-variable">levels</span> <span class="cm-string">"10/11/3"</span>,<span class="cm-string">"11/0/13"</span>,<span class="cm-variable">..</span><span class="cm-operator">:</span> <span class="cm-number">27</span> <span class="cm-number">22</span> <span class="cm-number">27</span> <span class="cm-number">27</span> <span class="cm-number">20</span> <span class="cm-number">27</span> <span class="cm-number">22</span> <span class="cm-number">18</span> <span class="cm-number">18</span> <span class="cm-number">27</span> <span class="cm-keyword">...</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> <span class="cm-dollar">$</span> <span class="cm-variable">E.HOM1.HET.HOM2.</span>  <span class="cm-operator">:</span> <span class="cm-variable">Factor</span> <span class="cm-variable">w</span><span class="cm-operator">/</span> <span class="cm-number">16</span> <span class="cm-variable">levels</span> <span class="cm-string">"10.01/10.98/3.01"</span>,<span class="cm-variable">..</span><span class="cm-operator">:</span> <span class="cm-number">14</span> <span class="cm-number">12</span> <span class="cm-number">14</span> <span class="cm-number">14</span> <span class="cm-number">11</span> <span class="cm-number">14</span> <span class="cm-number">12</span> <span class="cm-number">10</span> <span class="cm-number">10</span> <span class="cm-number">14</span> <span class="cm-keyword">...</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> <span class="cm-dollar">$</span> <span class="cm-variable">ChiSq_HWE</span> &nbsp; &nbsp; &nbsp; &nbsp; <span class="cm-operator">:</span> <span class="cm-variable">num</span>  <span class="cm-number">0.0109</span> <span class="cm-number">0.1067</span> <span class="cm-number">0.0109</span> <span class="cm-number">0.0109</span> <span class="cm-number">0.1983</span> <span class="cm-keyword">...</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> <span class="cm-dollar">$</span> <span class="cm-variable">P_HWE</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <span class="cm-operator">:</span> <span class="cm-variable">num</span>  <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-keyword">...</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> <span class="cm-dollar">$</span> <span class="cm-variable">P_HET_DEFICIT</span> &nbsp; &nbsp; <span class="cm-operator">:</span> <span class="cm-variable">num</span>  <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-keyword">...</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> <span class="cm-dollar">$</span> <span class="cm-variable">P_HET_EXCESS</span> &nbsp; &nbsp;  <span class="cm-operator">:</span> <span class="cm-variable">num</span>  <span class="cm-number">1</span> <span class="cm-number">0.936</span> <span class="cm-number">1</span> <span class="cm-number">1</span> <span class="cm-number">0.874</span> <span class="cm-keyword">...</span></span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 295px;"></div></div></div></pre><p>Which ones have heterozygous excess?</p><pre class="md-fences md-end-block" lang="r"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-operator">&gt;</span> <span class="cm-variable">b</span>[<span class="cm-variable">which</span>(<span class="cm-variable">b</span><span class="cm-dollar">$</span><span class="cm-variable">P_HET_EXCESS</span> <span class="cm-operator">&lt;</span> <span class="cm-number">0.001</span> ),]</span></pre></div><pre class=""><span style="padding-right: 0.1px;">[<span class="cm-number">1</span>] <span class="cm-variable">CHR</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-variable">POS</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-variable">OBS.HOM1.HET.HOM2.</span> <span class="cm-variable">E.HOM1.HET.HOM2.</span>  </span></pre><pre class=""><span style="padding-right: 0.1px;">[<span class="cm-number">5</span>] <span class="cm-variable">ChiSq_HWE</span> &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-variable">P_HWE</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-variable">P_HET_DEFICIT</span> &nbsp; &nbsp;  <span class="cm-variable">P_HET_EXCESS</span> &nbsp; &nbsp;  </span></pre><pre class=""><span style="padding-right: 0.1px;"> <span class="cm-operator">&lt;</span><span class="cm-number">0</span> <span class="cm-variable">rows</span><span class="cm-operator">&gt;</span>  </span></pre><pre class=""><span style="padding-right: 0.1px;"> (<span class="cm-variable">or</span> <span class="cm-number">0</span><span class="cm-operator">-</span><span class="cm-variable">length</span> <span class="cm-variable">row.names</span>)</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 105px;"></div></div></div></pre><p>Go into the rows and tell us the rows that satisfy some criteria</p><p></p><p>Let&#39;s look at deficit</p><pre class="md-fences md-end-block" lang="r"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-operator">&gt;</span> <span class="cm-variable">b</span>[<span class="cm-variable">which</span>(<span class="cm-variable">b</span><span class="cm-dollar">$</span><span class="cm-variable">P_HET_DEFICIT</span> <span class="cm-operator">&lt;</span> <span class="cm-number">0.01</span> ),]</span></pre></div><pre class=""><span style="padding-right: 0.1px;"> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <span class="cm-variable">CHR</span> <span class="cm-variable">POS</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">277</span> <span class="cm-variable">TRINITY_DN45155_c27_g2_TRINITY_DN45155_c27_g2_i2_g.18743_m.18743</span> <span class="cm-number">216</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">291</span> <span class="cm-variable">TRINITY_DN45155_c27_g1_TRINITY_DN45155_c27_g1_i1_g.18742_m.18742</span>  <span class="cm-number">99</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">293</span> <span class="cm-variable">TRINITY_DN45155_c27_g1_TRINITY_DN45155_c27_g1_i1_g.18742_m.18742</span> <span class="cm-number">138</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">401</span> &nbsp; &nbsp; <span class="cm-variable">TRINITY_DN39079_c3_g1_TRINITY_DN39079_c3_g1_i1_g.8354_m.8354</span> <span class="cm-number">244</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">406</span> &nbsp; &nbsp; <span class="cm-variable">TRINITY_DN39696_c4_g1_TRINITY_DN39696_c4_g1_i1_g.8926_m.8926</span> <span class="cm-number">283</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> &nbsp;  <span class="cm-variable">OBS.HOM1.HET.HOM2.</span> <span class="cm-variable">E.HOM1.HET.HOM2.</span> <span class="cm-variable">ChiSq_HWE</span> &nbsp; &nbsp; &nbsp;  <span class="cm-variable">P_HWE</span> <span class="cm-variable">P_HET_DEFICIT</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">277</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <span class="cm-number">22</span><span class="cm-operator">/</span><span class="cm-number">0</span><span class="cm-operator">/</span><span class="cm-number">2</span>  <span class="cm-number">20.17</span><span class="cm-operator">/</span><span class="cm-number">3.67</span><span class="cm-operator">/</span><span class="cm-number">0.17</span> &nbsp; &nbsp; &nbsp;  <span class="cm-number">24</span> <span class="cm-number">1.418440e-03</span>  <span class="cm-number">1.418440e-03</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">291</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-number">11</span><span class="cm-operator">/</span><span class="cm-number">0</span><span class="cm-operator">/</span><span class="cm-number">13</span>  <span class="cm-number">5.04</span><span class="cm-operator">/</span><span class="cm-number">11.92</span><span class="cm-operator">/</span><span class="cm-number">7.04</span> &nbsp; &nbsp; &nbsp;  <span class="cm-number">24</span> <span class="cm-number">9.114786e-08</span>  <span class="cm-number">9.114786e-08</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">293</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; <span class="cm-number">19</span><span class="cm-operator">/</span><span class="cm-number">0</span><span class="cm-operator">/</span><span class="cm-number">5</span>  <span class="cm-number">15.04</span><span class="cm-operator">/</span><span class="cm-number">7.92</span><span class="cm-operator">/</span><span class="cm-number">1.04</span> &nbsp; &nbsp; &nbsp;  <span class="cm-number">24</span> <span class="cm-number">6.498371e-06</span>  <span class="cm-number">6.498371e-06</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">401</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-number">13</span><span class="cm-operator">/</span><span class="cm-number">0</span><span class="cm-operator">/</span><span class="cm-number">11</span>  <span class="cm-number">7.04</span><span class="cm-operator">/</span><span class="cm-number">11.92</span><span class="cm-operator">/</span><span class="cm-number">5.04</span> &nbsp; &nbsp; &nbsp;  <span class="cm-number">24</span> <span class="cm-number">9.114786e-08</span>  <span class="cm-number">9.114786e-08</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">406</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-number">13</span><span class="cm-operator">/</span><span class="cm-number">0</span><span class="cm-operator">/</span><span class="cm-number">11</span>  <span class="cm-number">7.04</span><span class="cm-operator">/</span><span class="cm-number">11.92</span><span class="cm-operator">/</span><span class="cm-number">5.04</span> &nbsp; &nbsp; &nbsp;  <span class="cm-number">24</span> <span class="cm-number">9.114786e-08</span>  <span class="cm-number">9.114786e-08</span></span></pre><pre class=""><span style="padding-right: 0.1px;"> &nbsp;  <span class="cm-variable">P_HET_EXCESS</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">277</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-number">1</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">291</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-number">1</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">293</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-number">1</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">401</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-number">1</span></span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-number">406</span> &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;  <span class="cm-number">1</span></span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 399px;"></div></div></div></pre><p>Evidence of non-random mating; inbreeding</p><p></p><p><strong>Linkage equilibrium</strong></p><p>Association of SNPs at one locus with SNPS at another.</p><p>Too many pairs; so you can set windows. </p><p>--geno-r2</p><p>genotypes of different loci</p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code" style=""><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">$ vcftools --vcf 2017-03-06_SNPcallfilter.recode.vcf --geno-r2</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">VCFtools - 0.1.14</span></pre><pre class=""><span style="padding-right: 0.1px;">(C) Adam Auton and Anthony Marcketta 2009</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">Parameters as interpreted:</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--vcf 2017-03-06_SNPcallfilter.recode.vcf</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--max-alleles 2</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--min-alleles 2</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>--geno-r2</span></pre><pre class=""><span style="padding-right: 0.1px;"><span>​</span></span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 24 out of 24 Individuals</span></pre><pre class=""><span style="padding-right: 0.1px;">Outputting Pairwise LD (bi-allelic only)</span></pre><pre class=""><span style="padding-right: 0.1px;"><span class="cm-tab">    </span>LD: Only using diploid individuals.</span></pre><pre class=""><span style="padding-right: 0.1px;">After filtering, kept 1180 out of a possible 1180 Sites</span></pre><pre class=""><span style="padding-right: 0.1px;">Run Time = 3.00 seconds</span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 341px;"></div></div></div></pre><p>look at file</p><pre class='md-fences mock-cm' style='display:block;position:relative'>head out.geno.ld 
CHR	POS1	POS2	N_INDV	R^2
TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680	2127	2217	19	0.00653595
TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680	2127	2235	19	0.00308642
TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680	2127	2244	19	0.00308642
TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680	2127	2276	19	0.00308642
TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680	2127	2277	19	0.00308642
TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680	2127	2535	20	0.0557276
TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680	2127	2805	19	0.00308642
TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680	2127	2970	20	0.00277008
TRINITY_DN45147_c0_g1_TRINITY_DN45147_c0_g1_i3_g.18680_m.18680	2127	2994	20	0.0430622
</pre><p></p><p>Bring file into R</p><pre class="md-fences md-end-block" lang="r"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-operator">&gt;</span> <span class="cm-variable">LD</span><span class="cm-arrow">&lt;-</span><span class="cm-variable">read.table</span>(<span class="cm-string">"out.geno.ld"</span>,<span class="cm-variable">header</span><span class="cm-arg-is">=</span><span class="cm-variable">TRUE</span>)</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-operator">&gt;</span> <span class="cm-variable">names</span>(<span class="cm-variable">LD</span>)</span></pre><pre class=""><span style="padding-right: 0.1px;">[<span class="cm-number">1</span>] <span class="cm-string">"CHR"</span> &nbsp;  <span class="cm-string">"POS1"</span> &nbsp; <span class="cm-string">"POS2"</span> &nbsp; <span class="cm-string">"N_INDV"</span> <span class="cm-string">"R.2"</span> &nbsp; </span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 63px;"></div></div></div></pre><p>Look at distance between pos 1 and 2</p><pre class="md-fences md-end-block" lang="r"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-operator">&gt;</span> <span class="cm-variable">LD</span><span class="cm-dollar">$</span><span class="cm-variable">dist</span><span class="cm-arrow">&lt;-</span><span class="cm-variable">abs</span>(<span class="cm-variable">LD</span><span class="cm-dollar">$</span><span class="cm-variable">POS1</span> <span class="cm-operator">-</span> <span class="cm-variable">LD</span><span class="cm-dollar">$</span><span class="cm-variable">POS2</span>)</span></pre></div><pre class=""><span style="padding-right: 0.1px;"><span class="cm-operator">&gt;</span> <span class="cm-variable">names</span>(<span class="cm-variable">LD</span>)</span></pre><pre class=""><span style="padding-right: 0.1px;">[<span class="cm-number">1</span>] <span class="cm-string">"CHR"</span> &nbsp;  <span class="cm-string">"POS1"</span> &nbsp; <span class="cm-string">"POS2"</span> &nbsp; <span class="cm-string">"N_INDV"</span> <span class="cm-string">"R.2"</span> &nbsp;  <span class="cm-string">"dist"</span>  </span></pre></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 63px;"></div></div></div></pre><p>Can the distance predict LD? </p><p></p><p></p><hr /><p> <div id='id-section16'/><br/></p><h3><a name='header-n3450' class='md-header-anchor '></a>Page 16: 2017-03-08. Week 8, Day 14, Population genomics part 2; Paper Discussion- <a href='http://www.nature.com/nature/journal/v515/n7526/full/nature13685.html'>Romiguier et al. 2014</a></h3><p></p><p>Kattia info update: Rate of evolution</p><p></p><ol start='' ><li><p>effective pop size</p></li><li><p>mutation</p><ul><li>substitution</li><li>5 types</li><li>variation</li><li>mutation rate</li></ul></li><li><p>effective pop size + linkage</p></li><li><p>NERR + spatiotemporal variation</p></li></ol><p><strong>Effective population size</strong></p><p>Ne= effective population size; number of individuals that exchange genes; estimate inside the genome</p><p>S= selection coefficient</p><p>4 methods to estimate NE:</p><ol start='' ><li>From species life history- <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-9-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="19.898ex" height="3.975ex" viewBox="0 -1096.6 8567 1711.6" role="img" focusable="false" style="vertical-align: -1.428ex;"><defs><path stroke-width="1" id="E9-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E9-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path><path stroke-width="1" id="E9-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E9-MJMAIN-34" d="M462 0Q444 3 333 3Q217 3 199 0H190V46H221Q241 46 248 46T265 48T279 53T286 61Q287 63 287 115V165H28V211L179 442Q332 674 334 675Q336 677 355 677H373L379 671V211H471V165H379V114Q379 73 379 66T385 54Q393 47 442 46H471V0H462ZM293 211V545L74 212L183 211H293Z"></path><path stroke-width="1" id="E9-MJMAIN-2217" d="M229 286Q216 420 216 436Q216 454 240 464Q241 464 245 464T251 465Q263 464 273 456T283 436Q283 419 277 356T270 286L328 328Q384 369 389 372T399 375Q412 375 423 365T435 338Q435 325 425 315Q420 312 357 282T289 250L355 219L425 184Q434 175 434 161Q434 146 425 136T401 125Q393 125 383 131T328 171L270 213Q283 79 283 63Q283 53 276 44T250 35Q231 35 224 44T216 63Q216 80 222 143T229 213L171 171Q115 130 110 127Q106 124 100 124Q87 124 76 134T64 161Q64 166 64 169T67 175T72 181T81 188T94 195T113 204T138 215T170 230T210 250L74 315Q65 324 65 338Q65 353 74 363T98 374Q106 374 116 368T171 328L229 286Z"></path><path stroke-width="1" id="E9-MJMATHI-6D" d="M21 287Q22 293 24 303T36 341T56 388T88 425T132 442T175 435T205 417T221 395T229 376L231 369Q231 367 232 367L243 378Q303 442 384 442Q401 442 415 440T441 433T460 423T475 411T485 398T493 385T497 373T500 364T502 357L510 367Q573 442 659 442Q713 442 746 415T780 336Q780 285 742 178T704 50Q705 36 709 31T724 26Q752 26 776 56T815 138Q818 149 821 151T837 153Q857 153 857 145Q857 144 853 130Q845 101 831 73T785 17T716 -10Q669 -10 648 17T627 73Q627 92 663 193T700 345Q700 404 656 404H651Q565 404 506 303L499 291L466 157Q433 26 428 16Q415 -11 385 -11Q372 -11 364 -4T353 8T350 18Q350 29 384 161L420 307Q423 322 423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 181Q151 335 151 342Q154 357 154 369Q154 405 129 405Q107 405 92 377T69 316T57 280Q55 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E9-MJMATHI-61" d="M33 157Q33 258 109 349T280 441Q331 441 370 392Q386 422 416 422Q429 422 439 414T449 394Q449 381 412 234T374 68Q374 43 381 35T402 26Q411 27 422 35Q443 55 463 131Q469 151 473 152Q475 153 483 153H487Q506 153 506 144Q506 138 501 117T481 63T449 13Q436 0 417 -8Q409 -10 393 -10Q359 -10 336 5T306 36L300 51Q299 52 296 50Q294 48 292 46Q233 -10 172 -10Q117 -10 75 30T33 157ZM351 328Q351 334 346 350T323 385T277 405Q242 405 210 374T160 293Q131 214 119 129Q119 126 119 118T118 106Q118 61 136 44T179 26Q217 26 254 59T298 110Q300 114 325 217T351 328Z"></path><path stroke-width="1" id="E9-MJMATHI-6C" d="M117 59Q117 26 142 26Q179 26 205 131Q211 151 215 152Q217 153 225 153H229Q238 153 241 153T246 151T248 144Q247 138 245 128T234 90T214 43T183 6T137 -11Q101 -11 70 11T38 85Q38 97 39 102L104 360Q167 615 167 623Q167 626 166 628T162 632T157 634T149 635T141 636T132 637T122 637Q112 637 109 637T101 638T95 641T94 647Q94 649 96 661Q101 680 107 682T179 688Q194 689 213 690T243 693T254 694Q266 694 266 686Q266 675 193 386T118 83Q118 81 118 75T117 65V59Z"></path><path stroke-width="1" id="E9-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E9-MJMATHI-66" d="M118 -162Q120 -162 124 -164T135 -167T147 -168Q160 -168 171 -155T187 -126Q197 -99 221 27T267 267T289 382V385H242Q195 385 192 387Q188 390 188 397L195 425Q197 430 203 430T250 431Q298 431 298 432Q298 434 307 482T319 540Q356 705 465 705Q502 703 526 683T550 630Q550 594 529 578T487 561Q443 561 443 603Q443 622 454 636T478 657L487 662Q471 668 457 668Q445 668 434 658T419 630Q412 601 403 552T387 469T380 433Q380 431 435 431Q480 431 487 430T498 424Q499 420 496 407T491 391Q489 386 482 386T428 385H372L349 263Q301 15 282 -47Q255 -132 212 -173Q175 -205 139 -205Q107 -205 81 -186T55 -132Q55 -95 76 -78T118 -61Q162 -61 162 -103Q162 -122 151 -136T127 -157L118 -162Z"></path><path stroke-width="1" id="E9-MJMAIN-2B" d="M56 237T56 250T70 270H369V420L370 570Q380 583 389 583Q402 583 409 568V270H707Q722 262 722 250T707 230H409V-68Q401 -82 391 -82H389H387Q375 -82 369 -68V230H70Q56 237 56 250Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E9-MJMATHI-4E" x="0" y="0"></use><use xlink:href="#E9-MJMATHI-65" x="888" y="0"></use><use xlink:href="#E9-MJMAIN-3D" x="1632" y="0"></use><g transform="translate(2411,0)"><g transform="translate(397,0)"><rect stroke="none" width="5637" height="60" x="0" y="220"></rect><g transform="translate(60,537)"><use transform="scale(0.707)" xlink:href="#E9-MJMAIN-34" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMAIN-2217" x="500" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-6D" x="1001" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-61" x="1879" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-6C" x="2409" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-65" x="2707" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-73" x="3174" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMAIN-2217" x="3643" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-66" x="4144" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-65" x="4694" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-6D" x="5161" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-61" x="6039" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-6C" x="6569" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-65" x="6867" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-73" x="7334" y="0"></use></g><g transform="translate(315,-392)"><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-6D" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-61" x="878" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-6C" x="1408" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-65" x="1706" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-73" x="2173" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMAIN-2B" x="2642" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-66" x="3421" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-65" x="3971" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-6D" x="4438" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-61" x="5316" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-6C" x="5846" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-65" x="6144" y="0"></use><use transform="scale(0.707)" xlink:href="#E9-MJMATHI-73" x="6611" y="0"></use></g></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-9">Ne= \frac{4 * males * females}{males + females}</script></li><li>From variance in allele frequences between genotypes (needs at least 2 generations sampled); look at amount of drift from earlier to later generation; the amount of drift relates to the number of individuals that are reproducing; salman biologists try to use this </li><li>Genetic polymorphism</li><li>Correlated trait  (Body size ~ Ne)</li></ol><p>Ne vary across genomes: depending on genetic hitchhiking, background selection, sex chromosomes and autosomes number ()</p><p>genetic hitchhiking is also known as selecive sweep. </p><p>Males have a higher mutation rate than females. </p><p>Hitchhiking and background selection is in the absence of recombination. </p><p></p><p><strong>Mutation can occur at 2 levels:</strong> </p><ol start='' ><li><p>whole gene or chromosome </p><ul><li>duplication</li><li>inversion</li><li>deletion</li><li>translocation</li></ul></li><li><p>base level = point mutation</p><ul><li><p>substitution rate</p><ul><li>transition - changes between purine to purine or pyrimidine to pyrimidine</li><li>transversion - purine to pyrimidine</li></ul></li><li><p>synonymous - silent sites; amino acid does not change</p></li><li><p>non-synonymous - replacement mutations; results in a change in amino acid</p></li></ul></li><li><p>classes taht fit into syn and non-syn</p><ul><li><p>neutral -  <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-10-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="12.255ex" height="3.162ex" viewBox="0 -921.5 5276.6 1361.3" role="img" focusable="false" style="vertical-align: -1.022ex;"><defs><path stroke-width="1" id="E10-MJMATHI-3C9" d="M495 384Q495 406 514 424T555 443Q574 443 589 425T604 364Q604 334 592 278T555 155T483 38T377 -11Q297 -11 267 66Q266 68 260 61Q201 -11 125 -11Q15 -11 15 139Q15 230 56 325T123 434Q135 441 147 436Q160 429 160 418Q160 406 140 379T94 306T62 208Q61 202 61 187Q61 124 85 100T143 76Q201 76 245 129L253 137V156Q258 297 317 297Q348 297 348 261Q348 243 338 213T318 158L308 135Q309 133 310 129T318 115T334 97T358 83T393 76Q456 76 501 148T546 274Q546 305 533 325T508 357T495 384Z"></path><path stroke-width="1" id="E10-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E10-MJMAIN-30" d="M96 585Q152 666 249 666Q297 666 345 640T423 548Q460 465 460 320Q460 165 417 83Q397 41 362 16T301 -15T250 -22Q224 -22 198 -16T137 16T82 83Q39 165 39 320Q39 494 96 585ZM321 597Q291 629 250 629Q208 629 178 597Q153 571 145 525T137 333Q137 175 145 125T181 46Q209 16 250 16Q290 16 318 46Q347 76 354 130T362 333Q362 478 354 524T321 597Z"></path><path stroke-width="1" id="E10-MJMAIN-2C" d="M78 35T78 60T94 103T137 121Q165 121 187 96T210 8Q210 -27 201 -60T180 -117T154 -158T130 -185T117 -194Q113 -194 104 -185T95 -172Q95 -168 106 -156T131 -126T157 -76T173 -3V9L172 8Q170 7 167 6T161 3T152 1T140 0Q113 0 96 17Z"></path><path stroke-width="1" id="E10-MJMAIN-3C" d="M694 -11T694 -19T688 -33T678 -40Q671 -40 524 29T234 166L90 235Q83 240 83 250Q83 261 91 266Q664 540 678 540Q681 540 687 534T694 519T687 505Q686 504 417 376L151 250L417 124Q686 -4 687 -5Q694 -11 694 -19Z"></path><path stroke-width="1" id="E10-MJMAIN-31" d="M213 578L200 573Q186 568 160 563T102 556H83V602H102Q149 604 189 617T245 641T273 663Q275 666 285 666Q294 666 302 660V361L303 61Q310 54 315 52T339 48T401 46H427V0H416Q395 3 257 3Q121 3 100 0H88V46H114Q136 46 152 46T177 47T193 50T201 52T207 57T213 61V578Z"></path><path stroke-width="1" id="E10-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E10-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E10-MJMATHI-3C9" x="0" y="0"></use><use xlink:href="#E10-MJMAIN-3D" x="900" y="0"></use><use xlink:href="#E10-MJMAIN-30" x="1956" y="0"></use><use xlink:href="#E10-MJMAIN-2C" x="2457" y="0"></use><use xlink:href="#E10-MJMAIN-3C" x="2902" y="0"></use><g transform="translate(3680,0)"><g transform="translate(397,0)"><rect stroke="none" width="1078" height="60" x="0" y="220"></rect><use transform="scale(0.707)" xlink:href="#E10-MJMAIN-31" x="512" y="556"></use><g transform="translate(60,-376)"><use transform="scale(0.707)" xlink:href="#E10-MJMATHI-4E" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E10-MJMATHI-65" x="888" y="0"></use></g></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-10">\omega = 0 , < \frac{1}{Ne}</script>; no natural selection, drift, more common in large genomes</p></li><li><p>slightly deleterious; small effect on fitness <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-14-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.446ex" height="1.433ex" viewBox="0 -527.4 622.5 617" role="img" focusable="false" style="vertical-align: -0.208ex;"><defs><path stroke-width="1" id="E14-MJMATHI-3C9" d="M495 384Q495 406 514 424T555 443Q574 443 589 425T604 364Q604 334 592 278T555 155T483 38T377 -11Q297 -11 267 66Q266 68 260 61Q201 -11 125 -11Q15 -11 15 139Q15 230 56 325T123 434Q135 441 147 436Q160 429 160 418Q160 406 140 379T94 306T62 208Q61 202 61 187Q61 124 85 100T143 76Q201 76 245 129L253 137V156Q258 297 317 297Q348 297 348 261Q348 243 338 213T318 158L308 135Q309 133 310 129T318 115T334 97T358 83T393 76Q456 76 501 148T546 274Q546 305 533 325T508 357T495 384Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E14-MJMATHI-3C9" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-14">\omega</script></p><ul><li>depends on ploidy, ie. <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-12-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="7.555ex" height="3.162ex" viewBox="0 -921.5 3252.7 1361.3" role="img" focusable="false" style="vertical-align: -1.022ex;"><defs><path stroke-width="1" id="E12-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E12-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E12-MJMAIN-31" d="M213 578L200 573Q186 568 160 563T102 556H83V602H102Q149 604 189 617T245 641T273 663Q275 666 285 666Q294 666 302 660V361L303 61Q310 54 315 52T339 48T401 46H427V0H416Q395 3 257 3Q121 3 100 0H88V46H114Q136 46 152 46T177 47T193 50T201 52T207 57T213 61V578Z"></path><path stroke-width="1" id="E12-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E12-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E12-MJMATHI-6E" x="0" y="0"></use><use xlink:href="#E12-MJMAIN-3D" x="878" y="0"></use><g transform="translate(1656,0)"><g transform="translate(397,0)"><rect stroke="none" width="1078" height="60" x="0" y="220"></rect><use transform="scale(0.707)" xlink:href="#E12-MJMAIN-31" x="512" y="556"></use><g transform="translate(60,-376)"><use transform="scale(0.707)" xlink:href="#E12-MJMATHI-4E" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E12-MJMATHI-65" x="888" y="0"></use></g></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-12">n = \frac{1}{Ne}</script></li><li><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-13-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="9.539ex" height="3.162ex" viewBox="0 -921.5 4107.1 1361.3" role="img" focusable="false" style="vertical-align: -1.022ex;"><defs><path stroke-width="1" id="E13-MJMAIN-32" d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"></path><path stroke-width="1" id="E13-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E13-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E13-MJMAIN-31" d="M213 578L200 573Q186 568 160 563T102 556H83V602H102Q149 604 189 617T245 641T273 663Q275 666 285 666Q294 666 302 660V361L303 61Q310 54 315 52T339 48T401 46H427V0H416Q395 3 257 3Q121 3 100 0H88V46H114Q136 46 152 46T177 47T193 50T201 52T207 57T213 61V578Z"></path><path stroke-width="1" id="E13-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E13-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E13-MJMAIN-32" x="0" y="0"></use><use xlink:href="#E13-MJMATHI-6E" x="500" y="0"></use><use xlink:href="#E13-MJMAIN-3D" x="1378" y="0"></use><g transform="translate(2157,0)"><g transform="translate(397,0)"><rect stroke="none" width="1432" height="60" x="0" y="220"></rect><use transform="scale(0.707)" xlink:href="#E13-MJMAIN-31" x="762" y="556"></use><g transform="translate(60,-376)"><use transform="scale(0.707)" xlink:href="#E13-MJMAIN-32" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E13-MJMATHI-4E" x="500" y="0"></use><use transform="scale(0.707)" xlink:href="#E13-MJMATHI-65" x="1389" y="0"></use></g></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-13">2n = \frac{1}{2Ne}</script></li></ul></li><li><p>slightly advantage</p></li><li><p>deleterious; negative on <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-14-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.446ex" height="1.433ex" viewBox="0 -527.4 622.5 617" role="img" focusable="false" style="vertical-align: -0.208ex;"><defs><path stroke-width="1" id="E14-MJMATHI-3C9" d="M495 384Q495 406 514 424T555 443Q574 443 589 425T604 364Q604 334 592 278T555 155T483 38T377 -11Q297 -11 267 66Q266 68 260 61Q201 -11 125 -11Q15 -11 15 139Q15 230 56 325T123 434Q135 441 147 436Q160 429 160 418Q160 406 140 379T94 306T62 208Q61 202 61 187Q61 124 85 100T143 76Q201 76 245 129L253 137V156Q258 297 317 297Q348 297 348 261Q348 243 338 213T318 158L308 135Q309 133 310 129T318 115T334 97T358 83T393 76Q456 76 501 148T546 274Q546 305 533 325T508 357T495 384Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E14-MJMATHI-3C9" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-14">\omega</script>; lower substitution rate</p></li><li><p>advantageous</p></li></ul></li></ol><p><strong>mutation rate <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-15-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.402ex" height="1.84ex" viewBox="0 -527.4 603.5 792.1" role="img" focusable="false" style="vertical-align: -0.615ex;"><defs><path stroke-width="1" id="E15-MJMATHI-3BC" d="M58 -216Q44 -216 34 -208T23 -186Q23 -176 96 116T173 414Q186 442 219 442Q231 441 239 435T249 423T251 413Q251 401 220 279T187 142Q185 131 185 107V99Q185 26 252 26Q261 26 270 27T287 31T302 38T315 45T327 55T338 65T348 77T356 88T365 100L372 110L408 253Q444 395 448 404Q461 431 491 431Q504 431 512 424T523 412T525 402L449 84Q448 79 448 68Q448 43 455 35T476 26Q485 27 496 35Q517 55 537 131Q543 151 547 152Q549 153 557 153H561Q580 153 580 144Q580 138 575 117T555 63T523 13Q510 0 491 -8Q483 -10 467 -10Q446 -10 429 -4T402 11T385 29T376 44T374 51L368 45Q362 39 350 30T324 12T288 -4T246 -11Q199 -11 153 12L129 -85Q108 -167 104 -180T92 -202Q76 -216 58 -216Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E15-MJMATHI-3BC" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-15">\mu</script></strong> influenced by</p><ol start='' ><li>generation time- short generation time = higher mutation</li><li>selection; reduces mutation rate (controversial)</li></ol><p><strong>Effect of linkage?</strong> </p><p>Ne and sub rate could be influenced by:</p><ol start='' ><li>selective sweeps</li><li>clonal interference (2 diff individuals have unique mutations and they compete for increasing in frequency)</li></ol><p><strong>Skeller adding:</strong></p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-16-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="11.65ex" height="2.45ex" viewBox="0 -790.1 5016 1054.8" role="img" focusable="false" style="vertical-align: -0.615ex;"><defs><path stroke-width="1" id="E16-MJMAIN-398" d="M56 340Q56 423 86 494T164 610T270 680T388 705Q521 705 621 601T722 341Q722 260 693 191T617 75T510 4T388 -22T267 3T160 74T85 189T56 340ZM610 339Q610 428 590 495T535 598T463 651T384 668Q332 668 289 638T221 566Q168 485 168 339Q168 274 176 235Q189 158 228 105T324 28Q356 16 388 16Q415 16 442 24T501 54T555 111T594 205T610 339ZM223 263V422H263V388H514V422H554V263H514V297H263V263H223Z"></path><path stroke-width="1" id="E16-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E16-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E16-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path><path stroke-width="1" id="E16-MJMAIN-2217" d="M229 286Q216 420 216 436Q216 454 240 464Q241 464 245 464T251 465Q263 464 273 456T283 436Q283 419 277 356T270 286L328 328Q384 369 389 372T399 375Q412 375 423 365T435 338Q435 325 425 315Q420 312 357 282T289 250L355 219L425 184Q434 175 434 161Q434 146 425 136T401 125Q393 125 383 131T328 171L270 213Q283 79 283 63Q283 53 276 44T250 35Q231 35 224 44T216 63Q216 80 222 143T229 213L171 171Q115 130 110 127Q106 124 100 124Q87 124 76 134T64 161Q64 166 64 169T67 175T72 181T81 188T94 195T113 204T138 215T170 230T210 250L74 315Q65 324 65 338Q65 353 74 363T98 374Q106 374 116 368T171 328L229 286Z"></path><path stroke-width="1" id="E16-MJMATHI-3BC" d="M58 -216Q44 -216 34 -208T23 -186Q23 -176 96 116T173 414Q186 442 219 442Q231 441 239 435T249 423T251 413Q251 401 220 279T187 142Q185 131 185 107V99Q185 26 252 26Q261 26 270 27T287 31T302 38T315 45T327 55T338 65T348 77T356 88T365 100L372 110L408 253Q444 395 448 404Q461 431 491 431Q504 431 512 424T523 412T525 402L449 84Q448 79 448 68Q448 43 455 35T476 26Q485 27 496 35Q517 55 537 131Q543 151 547 152Q549 153 557 153H561Q580 153 580 144Q580 138 575 117T555 63T523 13Q510 0 491 -8Q483 -10 467 -10Q446 -10 429 -4T402 11T385 29T376 44T374 51L368 45Q362 39 350 30T324 12T288 -4T246 -11Q199 -11 153 12L129 -85Q108 -167 104 -180T92 -202Q76 -216 58 -216Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E16-MJMAIN-398" x="0" y="0"></use><use xlink:href="#E16-MJMAIN-3D" x="1056" y="0"></use><use xlink:href="#E16-MJMATHI-4E" x="2112" y="0"></use><use xlink:href="#E16-MJMATHI-65" x="3001" y="0"></use><use xlink:href="#E16-MJMAIN-2217" x="3689" y="0"></use><use xlink:href="#E16-MJMATHI-3BC" x="4412" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-16">\Theta = Ne * \mu </script>  in bacteria</p><p>for diploid hemaphrotidic</p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-17-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="12.813ex" height="2.45ex" viewBox="0 -790.1 5516.5 1054.8" role="img" focusable="false" style="vertical-align: -0.615ex;"><defs><path stroke-width="1" id="E17-MJMAIN-398" d="M56 340Q56 423 86 494T164 610T270 680T388 705Q521 705 621 601T722 341Q722 260 693 191T617 75T510 4T388 -22T267 3T160 74T85 189T56 340ZM610 339Q610 428 590 495T535 598T463 651T384 668Q332 668 289 638T221 566Q168 485 168 339Q168 274 176 235Q189 158 228 105T324 28Q356 16 388 16Q415 16 442 24T501 54T555 111T594 205T610 339ZM223 263V422H263V388H514V422H554V263H514V297H263V263H223Z"></path><path stroke-width="1" id="E17-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E17-MJMAIN-32" d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"></path><path stroke-width="1" id="E17-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E17-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path><path stroke-width="1" id="E17-MJMAIN-2217" d="M229 286Q216 420 216 436Q216 454 240 464Q241 464 245 464T251 465Q263 464 273 456T283 436Q283 419 277 356T270 286L328 328Q384 369 389 372T399 375Q412 375 423 365T435 338Q435 325 425 315Q420 312 357 282T289 250L355 219L425 184Q434 175 434 161Q434 146 425 136T401 125Q393 125 383 131T328 171L270 213Q283 79 283 63Q283 53 276 44T250 35Q231 35 224 44T216 63Q216 80 222 143T229 213L171 171Q115 130 110 127Q106 124 100 124Q87 124 76 134T64 161Q64 166 64 169T67 175T72 181T81 188T94 195T113 204T138 215T170 230T210 250L74 315Q65 324 65 338Q65 353 74 363T98 374Q106 374 116 368T171 328L229 286Z"></path><path stroke-width="1" id="E17-MJMATHI-3BC" d="M58 -216Q44 -216 34 -208T23 -186Q23 -176 96 116T173 414Q186 442 219 442Q231 441 239 435T249 423T251 413Q251 401 220 279T187 142Q185 131 185 107V99Q185 26 252 26Q261 26 270 27T287 31T302 38T315 45T327 55T338 65T348 77T356 88T365 100L372 110L408 253Q444 395 448 404Q461 431 491 431Q504 431 512 424T523 412T525 402L449 84Q448 79 448 68Q448 43 455 35T476 26Q485 27 496 35Q517 55 537 131Q543 151 547 152Q549 153 557 153H561Q580 153 580 144Q580 138 575 117T555 63T523 13Q510 0 491 -8Q483 -10 467 -10Q446 -10 429 -4T402 11T385 29T376 44T374 51L368 45Q362 39 350 30T324 12T288 -4T246 -11Q199 -11 153 12L129 -85Q108 -167 104 -180T92 -202Q76 -216 58 -216Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E17-MJMAIN-398" x="0" y="0"></use><use xlink:href="#E17-MJMAIN-3D" x="1056" y="0"></use><use xlink:href="#E17-MJMAIN-32" x="2112" y="0"></use><use xlink:href="#E17-MJMATHI-4E" x="2613" y="0"></use><use xlink:href="#E17-MJMATHI-65" x="3501" y="0"></use><use xlink:href="#E17-MJMAIN-2217" x="4190" y="0"></use><use xlink:href="#E17-MJMATHI-3BC" x="4913" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-17">\Theta = 2Ne * \mu </script><br/></p><p>for diploid separate sex it is </p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-18-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="12.813ex" height="2.45ex" viewBox="0 -790.1 5516.5 1054.8" role="img" focusable="false" style="vertical-align: -0.615ex;"><defs><path stroke-width="1" id="E18-MJMAIN-398" d="M56 340Q56 423 86 494T164 610T270 680T388 705Q521 705 621 601T722 341Q722 260 693 191T617 75T510 4T388 -22T267 3T160 74T85 189T56 340ZM610 339Q610 428 590 495T535 598T463 651T384 668Q332 668 289 638T221 566Q168 485 168 339Q168 274 176 235Q189 158 228 105T324 28Q356 16 388 16Q415 16 442 24T501 54T555 111T594 205T610 339ZM223 263V422H263V388H514V422H554V263H514V297H263V263H223Z"></path><path stroke-width="1" id="E18-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E18-MJMAIN-34" d="M462 0Q444 3 333 3Q217 3 199 0H190V46H221Q241 46 248 46T265 48T279 53T286 61Q287 63 287 115V165H28V211L179 442Q332 674 334 675Q336 677 355 677H373L379 671V211H471V165H379V114Q379 73 379 66T385 54Q393 47 442 46H471V0H462ZM293 211V545L74 212L183 211H293Z"></path><path stroke-width="1" id="E18-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E18-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path><path stroke-width="1" id="E18-MJMAIN-2217" d="M229 286Q216 420 216 436Q216 454 240 464Q241 464 245 464T251 465Q263 464 273 456T283 436Q283 419 277 356T270 286L328 328Q384 369 389 372T399 375Q412 375 423 365T435 338Q435 325 425 315Q420 312 357 282T289 250L355 219L425 184Q434 175 434 161Q434 146 425 136T401 125Q393 125 383 131T328 171L270 213Q283 79 283 63Q283 53 276 44T250 35Q231 35 224 44T216 63Q216 80 222 143T229 213L171 171Q115 130 110 127Q106 124 100 124Q87 124 76 134T64 161Q64 166 64 169T67 175T72 181T81 188T94 195T113 204T138 215T170 230T210 250L74 315Q65 324 65 338Q65 353 74 363T98 374Q106 374 116 368T171 328L229 286Z"></path><path stroke-width="1" id="E18-MJMATHI-3BC" d="M58 -216Q44 -216 34 -208T23 -186Q23 -176 96 116T173 414Q186 442 219 442Q231 441 239 435T249 423T251 413Q251 401 220 279T187 142Q185 131 185 107V99Q185 26 252 26Q261 26 270 27T287 31T302 38T315 45T327 55T338 65T348 77T356 88T365 100L372 110L408 253Q444 395 448 404Q461 431 491 431Q504 431 512 424T523 412T525 402L449 84Q448 79 448 68Q448 43 455 35T476 26Q485 27 496 35Q517 55 537 131Q543 151 547 152Q549 153 557 153H561Q580 153 580 144Q580 138 575 117T555 63T523 13Q510 0 491 -8Q483 -10 467 -10Q446 -10 429 -4T402 11T385 29T376 44T374 51L368 45Q362 39 350 30T324 12T288 -4T246 -11Q199 -11 153 12L129 -85Q108 -167 104 -180T92 -202Q76 -216 58 -216Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E18-MJMAIN-398" x="0" y="0"></use><use xlink:href="#E18-MJMAIN-3D" x="1056" y="0"></use><use xlink:href="#E18-MJMAIN-34" x="2112" y="0"></use><use xlink:href="#E18-MJMATHI-4E" x="2613" y="0"></use><use xlink:href="#E18-MJMATHI-65" x="3501" y="0"></use><use xlink:href="#E18-MJMAIN-2217" x="4190" y="0"></use><use xlink:href="#E18-MJMATHI-3BC" x="4913" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-18">\Theta = 4Ne * \mu </script><br/></p><p></p><p>How do you estimate <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-19-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.808ex" height="2.043ex" viewBox="0 -790.1 778.5 879.7" role="img" focusable="false" style="vertical-align: -0.208ex;"><defs><path stroke-width="1" id="E19-MJMAIN-398" d="M56 340Q56 423 86 494T164 610T270 680T388 705Q521 705 621 601T722 341Q722 260 693 191T617 75T510 4T388 -22T267 3T160 74T85 189T56 340ZM610 339Q610 428 590 495T535 598T463 651T384 668Q332 668 289 638T221 566Q168 485 168 339Q168 274 176 235Q189 158 228 105T324 28Q356 16 388 16Q415 16 442 24T501 54T555 111T594 205T610 339ZM223 263V422H263V388H514V422H554V263H514V297H263V263H223Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E19-MJMAIN-398" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-19">\Theta</script>?</p><p>Grab neutral sites!</p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-20-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="9.038ex" height="2.653ex" viewBox="0 -790.1 3891.4 1142.4" role="img" focusable="false" style="vertical-align: -0.818ex;"><defs><path stroke-width="1" id="E20-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path><path stroke-width="1" id="E20-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E20-MJMATHI-79" d="M21 287Q21 301 36 335T84 406T158 442Q199 442 224 419T250 355Q248 336 247 334Q247 331 231 288T198 191T182 105Q182 62 196 45T238 27Q261 27 281 38T312 61T339 94Q339 95 344 114T358 173T377 247Q415 397 419 404Q432 431 462 431Q475 431 483 424T494 412T496 403Q496 390 447 193T391 -23Q363 -106 294 -155T156 -205Q111 -205 77 -183T43 -117Q43 -95 50 -80T69 -58T89 -48T106 -45Q150 -45 150 -87Q150 -107 138 -122T115 -142T102 -147L99 -148Q101 -153 118 -160T152 -167H160Q177 -167 186 -165Q219 -156 247 -127T290 -65T313 -9T321 21L315 17Q309 13 296 6T270 -6Q250 -11 231 -11Q185 -11 150 11T104 82Q103 89 103 113Q103 170 138 262T173 379Q173 380 173 381Q173 390 173 393T169 400T158 404H154Q131 404 112 385T82 344T65 302T57 280Q55 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E20-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E20-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E20-MJMAIN-398" d="M56 340Q56 423 86 494T164 610T270 680T388 705Q521 705 621 601T722 341Q722 260 693 191T617 75T510 4T388 -22T267 3T160 74T85 189T56 340ZM610 339Q610 428 590 495T535 598T463 651T384 668Q332 668 289 638T221 566Q168 485 168 339Q168 274 176 235Q189 158 228 105T324 28Q356 16 388 16Q415 16 442 24T501 54T555 111T594 205T610 339ZM223 263V422H263V388H514V422H554V263H514V297H263V263H223Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E20-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(570,-150)"><use transform="scale(0.707)" xlink:href="#E20-MJMATHI-73" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E20-MJMATHI-79" x="469" y="0"></use><use transform="scale(0.707)" xlink:href="#E20-MJMATHI-6E" x="966" y="0"></use></g><use xlink:href="#E20-MJMAIN-3D" x="2056" y="0"></use><use xlink:href="#E20-MJMAIN-398" x="3112" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-20">\pi_{syn} = \Theta</script></p><p>then rearrange:</p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-21-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="17.331ex" height="2.552ex" viewBox="0 -746.3 7461.8 1098.6" role="img" focusable="false" style="vertical-align: -0.818ex;"><defs><path stroke-width="1" id="E21-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path><path stroke-width="1" id="E21-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E21-MJMATHI-79" d="M21 287Q21 301 36 335T84 406T158 442Q199 442 224 419T250 355Q248 336 247 334Q247 331 231 288T198 191T182 105Q182 62 196 45T238 27Q261 27 281 38T312 61T339 94Q339 95 344 114T358 173T377 247Q415 397 419 404Q432 431 462 431Q475 431 483 424T494 412T496 403Q496 390 447 193T391 -23Q363 -106 294 -155T156 -205Q111 -205 77 -183T43 -117Q43 -95 50 -80T69 -58T89 -48T106 -45Q150 -45 150 -87Q150 -107 138 -122T115 -142T102 -147L99 -148Q101 -153 118 -160T152 -167H160Q177 -167 186 -165Q219 -156 247 -127T290 -65T313 -9T321 21L315 17Q309 13 296 6T270 -6Q250 -11 231 -11Q185 -11 150 11T104 82Q103 89 103 113Q103 170 138 262T173 379Q173 380 173 381Q173 390 173 393T169 400T158 404H154Q131 404 112 385T82 344T65 302T57 280Q55 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E21-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E21-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E21-MJMAIN-34" d="M462 0Q444 3 333 3Q217 3 199 0H190V46H221Q241 46 248 46T265 48T279 53T286 61Q287 63 287 115V165H28V211L179 442Q332 674 334 675Q336 677 355 677H373L379 671V211H471V165H379V114Q379 73 379 66T385 54Q393 47 442 46H471V0H462ZM293 211V545L74 212L183 211H293Z"></path><path stroke-width="1" id="E21-MJMAIN-2217" d="M229 286Q216 420 216 436Q216 454 240 464Q241 464 245 464T251 465Q263 464 273 456T283 436Q283 419 277 356T270 286L328 328Q384 369 389 372T399 375Q412 375 423 365T435 338Q435 325 425 315Q420 312 357 282T289 250L355 219L425 184Q434 175 434 161Q434 146 425 136T401 125Q393 125 383 131T328 171L270 213Q283 79 283 63Q283 53 276 44T250 35Q231 35 224 44T216 63Q216 80 222 143T229 213L171 171Q115 130 110 127Q106 124 100 124Q87 124 76 134T64 161Q64 166 64 169T67 175T72 181T81 188T94 195T113 204T138 215T170 230T210 250L74 315Q65 324 65 338Q65 353 74 363T98 374Q106 374 116 368T171 328L229 286Z"></path><path stroke-width="1" id="E21-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E21-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path><path stroke-width="1" id="E21-MJMATHI-3BC" d="M58 -216Q44 -216 34 -208T23 -186Q23 -176 96 116T173 414Q186 442 219 442Q231 441 239 435T249 423T251 413Q251 401 220 279T187 142Q185 131 185 107V99Q185 26 252 26Q261 26 270 27T287 31T302 38T315 45T327 55T338 65T348 77T356 88T365 100L372 110L408 253Q444 395 448 404Q461 431 491 431Q504 431 512 424T523 412T525 402L449 84Q448 79 448 68Q448 43 455 35T476 26Q485 27 496 35Q517 55 537 131Q543 151 547 152Q549 153 557 153H561Q580 153 580 144Q580 138 575 117T555 63T523 13Q510 0 491 -8Q483 -10 467 -10Q446 -10 429 -4T402 11T385 29T376 44T374 51L368 45Q362 39 350 30T324 12T288 -4T246 -11Q199 -11 153 12L129 -85Q108 -167 104 -180T92 -202Q76 -216 58 -216Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E21-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(570,-150)"><use transform="scale(0.707)" xlink:href="#E21-MJMATHI-73" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E21-MJMATHI-79" x="469" y="0"></use><use transform="scale(0.707)" xlink:href="#E21-MJMATHI-6E" x="966" y="0"></use></g><use xlink:href="#E21-MJMAIN-3D" x="2056" y="0"></use><use xlink:href="#E21-MJMAIN-34" x="3112" y="0"></use><use xlink:href="#E21-MJMAIN-2217" x="3835" y="0"></use><use xlink:href="#E21-MJMATHI-4E" x="4558" y="0"></use><use xlink:href="#E21-MJMATHI-65" x="5446" y="0"></use><use xlink:href="#E21-MJMAIN-2217" x="6135" y="0"></use><use xlink:href="#E21-MJMATHI-3BC" x="6858" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-21">\pi_{syn} = 4* Ne * \mu</script></p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-22-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="10.003ex" height="3.569ex" viewBox="0 -965.3 4306.9 1536.5" role="img" focusable="false" style="vertical-align: -1.327ex;"><defs><path stroke-width="1" id="E22-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E22-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path><path stroke-width="1" id="E22-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E22-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path><path stroke-width="1" id="E22-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E22-MJMATHI-79" d="M21 287Q21 301 36 335T84 406T158 442Q199 442 224 419T250 355Q248 336 247 334Q247 331 231 288T198 191T182 105Q182 62 196 45T238 27Q261 27 281 38T312 61T339 94Q339 95 344 114T358 173T377 247Q415 397 419 404Q432 431 462 431Q475 431 483 424T494 412T496 403Q496 390 447 193T391 -23Q363 -106 294 -155T156 -205Q111 -205 77 -183T43 -117Q43 -95 50 -80T69 -58T89 -48T106 -45Q150 -45 150 -87Q150 -107 138 -122T115 -142T102 -147L99 -148Q101 -153 118 -160T152 -167H160Q177 -167 186 -165Q219 -156 247 -127T290 -65T313 -9T321 21L315 17Q309 13 296 6T270 -6Q250 -11 231 -11Q185 -11 150 11T104 82Q103 89 103 113Q103 170 138 262T173 379Q173 380 173 381Q173 390 173 393T169 400T158 404H154Q131 404 112 385T82 344T65 302T57 280Q55 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E22-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E22-MJMAIN-34" d="M462 0Q444 3 333 3Q217 3 199 0H190V46H221Q241 46 248 46T265 48T279 53T286 61Q287 63 287 115V165H28V211L179 442Q332 674 334 675Q336 677 355 677H373L379 671V211H471V165H379V114Q379 73 379 66T385 54Q393 47 442 46H471V0H462ZM293 211V545L74 212L183 211H293Z"></path><path stroke-width="1" id="E22-MJMAIN-2217" d="M229 286Q216 420 216 436Q216 454 240 464Q241 464 245 464T251 465Q263 464 273 456T283 436Q283 419 277 356T270 286L328 328Q384 369 389 372T399 375Q412 375 423 365T435 338Q435 325 425 315Q420 312 357 282T289 250L355 219L425 184Q434 175 434 161Q434 146 425 136T401 125Q393 125 383 131T328 171L270 213Q283 79 283 63Q283 53 276 44T250 35Q231 35 224 44T216 63Q216 80 222 143T229 213L171 171Q115 130 110 127Q106 124 100 124Q87 124 76 134T64 161Q64 166 64 169T67 175T72 181T81 188T94 195T113 204T138 215T170 230T210 250L74 315Q65 324 65 338Q65 353 74 363T98 374Q106 374 116 368T171 328L229 286Z"></path><path stroke-width="1" id="E22-MJMATHI-3BC" d="M58 -216Q44 -216 34 -208T23 -186Q23 -176 96 116T173 414Q186 442 219 442Q231 441 239 435T249 423T251 413Q251 401 220 279T187 142Q185 131 185 107V99Q185 26 252 26Q261 26 270 27T287 31T302 38T315 45T327 55T338 65T348 77T356 88T365 100L372 110L408 253Q444 395 448 404Q461 431 491 431Q504 431 512 424T523 412T525 402L449 84Q448 79 448 68Q448 43 455 35T476 26Q485 27 496 35Q517 55 537 131Q543 151 547 152Q549 153 557 153H561Q580 153 580 144Q580 138 575 117T555 63T523 13Q510 0 491 -8Q483 -10 467 -10Q446 -10 429 -4T402 11T385 29T376 44T374 51L368 45Q362 39 350 30T324 12T288 -4T246 -11Q199 -11 153 12L129 -85Q108 -167 104 -180T92 -202Q76 -216 58 -216Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E22-MJMATHI-4E" x="0" y="0"></use><use xlink:href="#E22-MJMATHI-65" x="888" y="0"></use><use xlink:href="#E22-MJMAIN-3D" x="1632" y="0"></use><g transform="translate(2411,0)"><g transform="translate(397,0)"><rect stroke="none" width="1377" height="60" x="0" y="220"></rect><g transform="translate(60,601)"><use transform="scale(0.707)" xlink:href="#E22-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(403,-107)"><use transform="scale(0.5)" xlink:href="#E22-MJMATHI-73" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E22-MJMATHI-79" x="469" y="0"></use><use transform="scale(0.5)" xlink:href="#E22-MJMATHI-6E" x="967" y="0"></use></g></g><g transform="translate(121,-372)"><use transform="scale(0.707)" xlink:href="#E22-MJMAIN-34" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E22-MJMAIN-2217" x="500" y="0"></use><use transform="scale(0.707)" xlink:href="#E22-MJMATHI-3BC" x="1001" y="0"></use></g></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-22">Ne = \frac{\pi_{syn}}{4* \mu}</script></p><p></p><h3><a name='header-n3627' class='md-header-anchor '></a>Paper discussion</h3><p>Coding session:</p><p>change directory into where data are: </p><pre class="md-fences md-end-block" lang="unix"> <div class="CodeMirror cm-s-inner CodeMirror-wrap"><div style="overflow: hidden; position: relative; width: 3px; height: 0px; top: 0px; left: 4px;"></div><div class="CodeMirror-hscrollbar" style="left: 0px; min-height: 18px;"><div style="height: 100%; min-height: 1px; width: 0px;"></div></div><div class="CodeMirror-vscrollbar" style="min-width: 18px;"><div style="min-width: 1px; height: 0px;"></div></div><div class="CodeMirror-scrollbar-filler"></div><div class="CodeMirror-gutter-filler"></div><div class="CodeMirror-scroll" tabindex="-1"><div class="CodeMirror-sizer" style="margin-left: 0px; min-height: 0px;"><div style="position: relative; top: 0px;"><div class="CodeMirror-lines"><div style="position: relative; outline: none;"><div class="CodeMirror-measure"></div><div class="CodeMirror-measure"></div><div style="position: relative; z-index: 1;"></div><div class="CodeMirror-code"><div class="CodeMirror-activeline" style="position: relative;"><div class="CodeMirror-activeline-background CodeMirror-linebackground"></div><pre class=""><span style="padding-right: 0.1px;">cd /data/project_data/snps/reads2snps/</span></pre></div></div></div></div></div></div><div style="position: absolute; height: 30px; width: 1px; top: 0px;"></div><div class="CodeMirror-gutters" style="display: none; height: 21px;"></div></div></div></pre><p></p><p></p><p></p><hr /><p> <div id='id-section17'/><br/></p><h3><a name='header-n3642' class='md-header-anchor '></a>Page 17: 2017-03-20. Week 9, Day 15, Population genomics part 3; paper discussion Gompert et al. 2014</h3><p></p><p>Info update: Dr. Kilpatrick:</p><p></p><p>Global ancestry</p><p>Local ancestry; chromosome is a mosaic. Which ancestry each of these segments come from? </p><p></p><p>Model based: </p><ol start='' ><li>STRUCTURE (global)- bayesian program; allelic frequency distribution and ; under assumption of HWE and linkage equilibirum. Does not explicitly model linkage diequilibrium. </li><li>ADMIXTURE: ML approach; similar model in model. Assign populations or clusters or ancestors and their allelic frequencies. Difference—optimizes 2 matrices; ancestry matrix and gene flow matrix. </li></ol><p>Waterson&#39;s theta = richness</p><p>expected heterozygosity = evenness</p><p></p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-29-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.09ex" height="2.043ex" viewBox="0 -790.1 469.5 879.7" role="img" focusable="false" style="vertical-align: -0.208ex;"><defs><path stroke-width="1" id="E29-MJMATHI-3B8" d="M35 200Q35 302 74 415T180 610T319 704Q320 704 327 704T339 705Q393 701 423 656Q462 596 462 495Q462 380 417 261T302 66T168 -10H161Q125 -10 99 10T60 63T41 130T35 200ZM383 566Q383 668 330 668Q294 668 260 623T204 521T170 421T157 371Q206 370 254 370L351 371Q352 372 359 404T375 484T383 566ZM113 132Q113 26 166 26Q181 26 198 36T239 74T287 161T335 307L340 324H145Q145 321 136 286T120 208T113 132Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E29-MJMATHI-3B8" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-29">\theta</script> and pi diverge when populations are expanding and bottlenecks</p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-29-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.09ex" height="2.043ex" viewBox="0 -790.1 469.5 879.7" role="img" focusable="false" style="vertical-align: -0.208ex;"><defs><path stroke-width="1" id="E29-MJMATHI-3B8" d="M35 200Q35 302 74 415T180 610T319 704Q320 704 327 704T339 705Q393 701 423 656Q462 596 462 495Q462 380 417 261T302 66T168 -10H161Q125 -10 99 10T60 63T41 130T35 200ZM383 566Q383 668 330 668Q294 668 260 623T204 521T170 421T157 371Q206 370 254 370L351 371Q352 372 359 404T375 484T383 566ZM113 132Q113 26 166 26Q181 26 198 36T239 74T287 161T335 307L340 324H145Q145 321 136 286T120 208T113 132Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E29-MJMATHI-3B8" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-29">\theta</script> is higher when populations are expanding; but smaller when in bottlenecks (double check this)</p><h3><a name='header-n3674' class='md-header-anchor '></a>coding:</h3><ol start='' ><li><p>get final vcf data ; n=24</p></li><li><p>Filtering</p></li><li><p>output to home directory</p></li><li><p>Estimate allele frequencies between healthy and sick individuals (raw differences; <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-26-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.279ex" height="2.45ex" viewBox="0 -790.1 550.5 1054.8" role="img" focusable="false" style="vertical-align: -0.615ex;"><defs><path stroke-width="1" id="E26-MJMATHI-66" d="M118 -162Q120 -162 124 -164T135 -167T147 -168Q160 -168 171 -155T187 -126Q197 -99 221 27T267 267T289 382V385H242Q195 385 192 387Q188 390 188 397L195 425Q197 430 203 430T250 431Q298 431 298 432Q298 434 307 482T319 540Q356 705 465 705Q502 703 526 683T550 630Q550 594 529 578T487 561Q443 561 443 603Q443 622 454 636T478 657L487 662Q471 668 457 668Q445 668 434 658T419 630Q412 601 403 552T387 469T380 433Q380 431 435 431Q480 431 487 430T498 424Q499 420 496 407T491 391Q489 386 482 386T428 385H372L349 263Q301 15 282 -47Q255 -132 212 -173Q175 -205 139 -205Q107 -205 81 -186T55 -132Q55 -95 76 -78T118 -61Q162 -61 162 -103Q162 -122 151 -136T127 -157L118 -162Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E26-MJMATHI-66" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-26">f</script>(H)-<span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-26-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.279ex" height="2.45ex" viewBox="0 -790.1 550.5 1054.8" role="img" focusable="false" style="vertical-align: -0.615ex;"><defs><path stroke-width="1" id="E26-MJMATHI-66" d="M118 -162Q120 -162 124 -164T135 -167T147 -168Q160 -168 171 -155T187 -126Q197 -99 221 27T267 267T289 382V385H242Q195 385 192 387Q188 390 188 397L195 425Q197 430 203 430T250 431Q298 431 298 432Q298 434 307 482T319 540Q356 705 465 705Q502 703 526 683T550 630Q550 594 529 578T487 561Q443 561 443 603Q443 622 454 636T478 657L487 662Q471 668 457 668Q445 668 434 658T419 630Q412 601 403 552T387 469T380 433Q380 431 435 431Q480 431 487 430T498 424Q499 420 496 407T491 391Q489 386 482 386T428 385H372L349 263Q301 15 282 -47Q255 -132 212 -173Q175 -205 139 -205Q107 -205 81 -186T55 -132Q55 -95 76 -78T118 -61Q162 -61 162 -103Q162 -122 151 -136T127 -157L118 -162Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E26-MJMATHI-66" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-26">f</script>(S))</p><ul><li>Find SNPs that deviate from Healthy and sick</li><li>another way to cacluate is FST</li></ul></li><li><p>Output to local machine and analyze in R </p></li><li><p>estimate <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-27-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="4.132ex" height="1.941ex" viewBox="0 -483.6 1778.9 835.9" role="img" focusable="false" style="vertical-align: -0.818ex;"><defs><path stroke-width="1" id="E27-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path><path stroke-width="1" id="E27-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E27-MJMATHI-79" d="M21 287Q21 301 36 335T84 406T158 442Q199 442 224 419T250 355Q248 336 247 334Q247 331 231 288T198 191T182 105Q182 62 196 45T238 27Q261 27 281 38T312 61T339 94Q339 95 344 114T358 173T377 247Q415 397 419 404Q432 431 462 431Q475 431 483 424T494 412T496 403Q496 390 447 193T391 -23Q363 -106 294 -155T156 -205Q111 -205 77 -183T43 -117Q43 -95 50 -80T69 -58T89 -48T106 -45Q150 -45 150 -87Q150 -107 138 -122T115 -142T102 -147L99 -148Q101 -153 118 -160T152 -167H160Q177 -167 186 -165Q219 -156 247 -127T290 -65T313 -9T321 21L315 17Q309 13 296 6T270 -6Q250 -11 231 -11Q185 -11 150 11T104 82Q103 89 103 113Q103 170 138 262T173 379Q173 380 173 381Q173 390 173 393T169 400T158 404H154Q131 404 112 385T82 344T65 302T57 280Q55 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E27-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E27-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(570,-150)"><use transform="scale(0.707)" xlink:href="#E27-MJMATHI-73" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E27-MJMATHI-79" x="469" y="0"></use><use transform="scale(0.707)" xlink:href="#E27-MJMATHI-6E" x="966" y="0"></use></g></g></svg></span><script type="math/tex" id="MathJax-Element-27">\pi_{syn}</script>, <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-28-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="6.901ex" height="1.941ex" viewBox="0 -483.6 2971.4 835.9" role="img" focusable="false" style="vertical-align: -0.818ex;"><defs><path stroke-width="1" id="E28-MJMATHI-3C0" d="M132 -11Q98 -11 98 22V33L111 61Q186 219 220 334L228 358H196Q158 358 142 355T103 336Q92 329 81 318T62 297T53 285Q51 284 38 284Q19 284 19 294Q19 300 38 329T93 391T164 429Q171 431 389 431Q549 431 553 430Q573 423 573 402Q573 371 541 360Q535 358 472 358H408L405 341Q393 269 393 222Q393 170 402 129T421 65T431 37Q431 20 417 5T381 -10Q370 -10 363 -7T347 17T331 77Q330 86 330 121Q330 170 339 226T357 318T367 358H269L268 354Q268 351 249 275T206 114T175 17Q164 -11 132 -11Z"></path><path stroke-width="1" id="E28-MJMATHI-6E" d="M21 287Q22 293 24 303T36 341T56 388T89 425T135 442Q171 442 195 424T225 390T231 369Q231 367 232 367L243 378Q304 442 382 442Q436 442 469 415T503 336T465 179T427 52Q427 26 444 26Q450 26 453 27Q482 32 505 65T540 145Q542 153 560 153Q580 153 580 145Q580 144 576 130Q568 101 554 73T508 17T439 -10Q392 -10 371 17T350 73Q350 92 386 193T423 345Q423 404 379 404H374Q288 404 229 303L222 291L189 157Q156 26 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 112 180T152 343Q153 348 153 366Q153 405 129 405Q91 405 66 305Q60 285 60 284Q58 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E28-MJMATHI-6F" d="M201 -11Q126 -11 80 38T34 156Q34 221 64 279T146 380Q222 441 301 441Q333 441 341 440Q354 437 367 433T402 417T438 387T464 338T476 268Q476 161 390 75T201 -11ZM121 120Q121 70 147 48T206 26Q250 26 289 58T351 142Q360 163 374 216T388 308Q388 352 370 375Q346 405 306 405Q243 405 195 347Q158 303 140 230T121 120Z"></path><path stroke-width="1" id="E28-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E28-MJMATHI-79" d="M21 287Q21 301 36 335T84 406T158 442Q199 442 224 419T250 355Q248 336 247 334Q247 331 231 288T198 191T182 105Q182 62 196 45T238 27Q261 27 281 38T312 61T339 94Q339 95 344 114T358 173T377 247Q415 397 419 404Q432 431 462 431Q475 431 483 424T494 412T496 403Q496 390 447 193T391 -23Q363 -106 294 -155T156 -205Q111 -205 77 -183T43 -117Q43 -95 50 -80T69 -58T89 -48T106 -45Q150 -45 150 -87Q150 -107 138 -122T115 -142T102 -147L99 -148Q101 -153 118 -160T152 -167H160Q177 -167 186 -165Q219 -156 247 -127T290 -65T313 -9T321 21L315 17Q309 13 296 6T270 -6Q250 -11 231 -11Q185 -11 150 11T104 82Q103 89 103 113Q103 170 138 262T173 379Q173 380 173 381Q173 390 173 393T169 400T158 404H154Q131 404 112 385T82 344T65 302T57 280Q55 278 41 278H27Q21 284 21 287Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E28-MJMATHI-3C0" x="0" y="0"></use><g transform="translate(570,-150)"><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-6E" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-6F" x="600" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-6E" x="1086" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-73" x="1686" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-79" x="2156" y="0"></use><use transform="scale(0.707)" xlink:href="#E28-MJMATHI-6E" x="2653" y="0"></use></g></g></svg></span><script type="math/tex" id="MathJax-Element-28">\pi_{nonsyn}</script>,</p></li><li><p>​</p></li></ol><p></p><p></p><hr /><p> <div id='id-section18'/><br/></p><h3><a name='header-n3712' class='md-header-anchor '></a>Page 18: 2017-03-22. Week 9, Day 16, Population genomics part 4; paper discussion</h3><p><strong>Glossary:</strong></p><p>Ascertainment bias- bias introduced by sampling design that induces a nonrandom sample</p><p>Sympatric speciation- presence of gene flow, via diversifying selection; alleles under selection are divergent; neutral alleles appear homogenous</p><p>Allopatric Speciation-absence of gene flow; physical isolation</p><p>Islands of differntiation: genomic regions with increased diff due to natural selection</p><p>Allele frequency spectrum- dist of counts of snps with an observed frequency (histogram of allele frequencies)</p><p>Gene flow- transfer of genes from one population to another</p><p>Diversifying selection- seleciton that favors different alleles in different parts of the species range</p><p><strong>Info update from Erin Keller:</strong> Inferring history of divergence</p><p> <u>Genomic scans:</u><br/></p><ol start='' ><li><p>distribution of differentiation in the genome <br/></p><ul><li>High fst  —&gt; region under selection</li><li>extended to island models and metapopulations</li></ul></li></ol><ol start='2' ><li><p>gene vs population trees</p><ul><li><p>compare assumed pop trees to gene trees</p></li><li><p>compare differnt gene trees</p></li><li><p>measure introgression: D-statistic determines introgression</p><ul><li>ABBA-BABA approach: </li><li>D=0 means no introgression</li><li>D =/= 0</li></ul></li><li><p>Limitations- throws out data, requires many genomes, same value multiple explanations</p></li></ul></li></ol><p> <u>Model-based methods:</u><br/></p><p>Shape of Allele frequency spectrum(AFS) can get a sense of population processes. </p><table><thead><tr><th>Sample</th><th>1</th><th>2</th><th>3</th><th>4</th><th>5</th></tr></thead><tbody><tr><td>1</td><td>0</td><td>1</td><td>1</td><td>0</td><td>1</td></tr></tbody></table><p>etc…to create AFS</p><p>Assumptions:</p><ul><li>All alleles/snps are independent</li><li>free recombination among snps</li><li>mutation rates are equal</li></ul><p>Limitations:</p><ul><li>computationally challenging</li><li>lose lots of data</li><li>expensive for models with more than 3 populations</li></ul><p> <u>Geneology sampling</u><br/></p><ul><li><p>multiple regions sampled to produce 1 tree</p><ul><li>estimates Ne, migration, admixture</li></ul></li></ul><p>Assumptions:</p><ul><li>Free recombination among different genes</li><li>complete linkage within a gene</li><li>mutations can differ among snps</li><li>no major recombination at the most recent common ancestor</li></ul><p> <u>Likelihood-free method</u><br/></p><p>ABC- approximate bayesian comp. Use simulations under models of interest. </p><p></p><p> <u>Historical gene flow and LD patterns</u><br/></p><p>Distribution of haplotype lengths: Assumes a haplotype as a migrant and enters a population of interest. Over time, these genomes recombine and see the slow shortening of sequence lenght. Based on lengths of original mirgrant, you can get a sense of time of when migrant entered populations. Recombination leads to smaller fragments over time. Many demographic events can produce similar pattern. Problems with identifying migrant. </p><p>Skeller: single genome- look at blocks of heterozygosity and homozygosity and gives an estimate population size during a recent history. </p><p></p><p>Approximation of conditional likelihood: Ancestral recombination graphs (ARGs). Gene tree that bifurcates based on recombination. estimate migration and coalescent events. </p><p>Limitations: complex-difficult to estimate recombination rates. Multiple completing recombination trees</p><pre class='md-fences mock-cm' style='display:block;position:relative'>Advantages of whole genome: recombination rates, lots of information, large &quot;area&quot; for genome scans

Limitations of next gen: ascertainment bias, throw out data</pre><p></p><p><strong>Paper discussion:</strong></p><p></p><p><strong>Coding session: Population genomics part 4</strong></p><p>finish part 3 first</p><p>output: </p><blockquote><p>cat SSW_bamlist.txt.sum </p><p>################################################################################</p><p>#                              Biological Summary                              #</p><p>################################################################################</p><p>Selected ingroup species: sp</p><p>Number of analyzed individual: 24 (from 1 population(s))</p><p>Total number of contig used for sequence analysis: 1113</p><p>Total number of SNPs: 5040</p><ul><li><p>Biallelic: 4991</p><ul><li>Triallelic: 49</li><li>Quadriallelic: 0</li></ul></li></ul><p>Fit:</p><p>Average Fit: -0.0507419 [-0.06817; -0.031933]</p><p>(Fit calculated in 902 contigs)</p><p>Weir &amp; Cockerham Fit (Evolution 1984):</p><p>Average Weir &amp; Cockerham Fit: 0.00703754 [-0.017669; 0.032047]</p><p>(Fit calculated in 902 contigs)</p><p>piN/piS ratio:</p><p>Average piS in focal species: 0.00585312 [0.005172; 0.006598]</p><p>Average piN in focal species: 0.00154546 [0.00133; 0.001782]</p><p>Average piN / average piS: 0.264041 [0.223914; 0.310575]</p><p>(piS and piN calculated in 902 contigs of average length 50)</p></blockquote><p>FIT is similar to FST but accounts for inbreeding. It ranges from 0-1, whereby 1 = more homozygosity. This number is low, so it suggests no inbreeding and may hint at little or no population structure. </p><p></p><p></p><hr /><p> <div id='id-section19'/><br/></p><h3><a name='header-n3933' class='md-header-anchor '></a>Page 19: 2017-03-27. Week 10, Day 17, Population genomics-detecting signatures of natural selection; selection scans; paper discussion- Laurent et al. 2016, MolEco</h3><p>Separate out neutral processes(demography,bottlenecks) and selection. Selection acts locally on the genome, whereas neutral processes act across the whole genome. </p><p> <u>Glossary:</u><br/></p><p>Selective sweep: pattern whereby a single adaptive allele sweeps through population(goes to near fixation)</p><p>hard sweeps: single adaptive allele in common genetic background</p><p>soft sweep: &gt; 1 adaptive alleles in diff genetic backgrounds</p><p> <u>Dr. Brody Info Update: Selective Sweeps</u><br/></p><p>Big picture questioN: What maintains variation or produces variation? What reduces genetic variation amongm embers of populations. How do we discover those patterns and what do those patterns mean? </p><p>Why do we care?  Heuristic perspective. Who else might care? Conservation, human population, We care about selection and other poplations evolve. HOw do we interpret those signatures? Predict when they might occur. </p><p></p><h3><a name='header-n3952' class='md-header-anchor '></a>Selective sweeps</h3><p>Not only do adaptive alelles become favored, but the genes linked to it are increasing in the population. Linked genes that may not be adaptive are refereed to as hitchhiking genes. </p><p>Pleuni Pennings: Stanford University. Her phd students have 2 minute youtube videos for each of their papers.</p><p> <u>Sweeps can be hard on a local scale but soft on a global scale!!!</u><br/></p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-29-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="1.09ex" height="2.043ex" viewBox="0 -790.1 469.5 879.7" role="img" focusable="false" style="vertical-align: -0.208ex;"><defs><path stroke-width="1" id="E29-MJMATHI-3B8" d="M35 200Q35 302 74 415T180 610T319 704Q320 704 327 704T339 705Q393 701 423 656Q462 596 462 495Q462 380 417 261T302 66T168 -10H161Q125 -10 99 10T60 63T41 130T35 200ZM383 566Q383 668 330 668Q294 668 260 623T204 521T170 421T157 371Q206 370 254 370L351 371Q352 372 359 404T375 484T383 566ZM113 132Q113 26 166 26Q181 26 198 36T239 74T287 161T335 307L340 324H145Q145 321 136 286T120 208T113 132Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E29-MJMATHI-3B8" x="0" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-29">\theta</script> is a rate in which a mutation enters a population</p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-30-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="14.29ex" height="2.45ex" viewBox="0 -790.1 6152.4 1054.8" role="img" focusable="false" style="vertical-align: -0.615ex;"><defs><path stroke-width="1" id="E30-MJMATHI-3B8" d="M35 200Q35 302 74 415T180 610T319 704Q320 704 327 704T339 705Q393 701 423 656Q462 596 462 495Q462 380 417 261T302 66T168 -10H161Q125 -10 99 10T60 63T41 130T35 200ZM383 566Q383 668 330 668Q294 668 260 623T204 521T170 421T157 371Q206 370 254 370L351 371Q352 372 359 404T375 484T383 566ZM113 132Q113 26 166 26Q181 26 198 36T239 74T287 161T335 307L340 324H145Q145 321 136 286T120 208T113 132Z"></path><path stroke-width="1" id="E30-MJMAIN-3D" d="M56 347Q56 360 70 367H707Q722 359 722 347Q722 336 708 328L390 327H72Q56 332 56 347ZM56 153Q56 168 72 173H708Q722 163 722 153Q722 140 707 133H70Q56 140 56 153Z"></path><path stroke-width="1" id="E30-MJMAIN-32" d="M109 429Q82 429 66 447T50 491Q50 562 103 614T235 666Q326 666 387 610T449 465Q449 422 429 383T381 315T301 241Q265 210 201 149L142 93L218 92Q375 92 385 97Q392 99 409 186V189H449V186Q448 183 436 95T421 3V0H50V19V31Q50 38 56 46T86 81Q115 113 136 137Q145 147 170 174T204 211T233 244T261 278T284 308T305 340T320 369T333 401T340 431T343 464Q343 527 309 573T212 619Q179 619 154 602T119 569T109 550Q109 549 114 549Q132 549 151 535T170 489Q170 464 154 447T109 429Z"></path><path stroke-width="1" id="E30-MJMAIN-2217" d="M229 286Q216 420 216 436Q216 454 240 464Q241 464 245 464T251 465Q263 464 273 456T283 436Q283 419 277 356T270 286L328 328Q384 369 389 372T399 375Q412 375 423 365T435 338Q435 325 425 315Q420 312 357 282T289 250L355 219L425 184Q434 175 434 161Q434 146 425 136T401 125Q393 125 383 131T328 171L270 213Q283 79 283 63Q283 53 276 44T250 35Q231 35 224 44T216 63Q216 80 222 143T229 213L171 171Q115 130 110 127Q106 124 100 124Q87 124 76 134T64 161Q64 166 64 169T67 175T72 181T81 188T94 195T113 204T138 215T170 230T210 250L74 315Q65 324 65 338Q65 353 74 363T98 374Q106 374 116 368T171 328L229 286Z"></path><path stroke-width="1" id="E30-MJMATHI-4E" d="M234 637Q231 637 226 637Q201 637 196 638T191 649Q191 676 202 682Q204 683 299 683Q376 683 387 683T401 677Q612 181 616 168L670 381Q723 592 723 606Q723 633 659 637Q635 637 635 648Q635 650 637 660Q641 676 643 679T653 683Q656 683 684 682T767 680Q817 680 843 681T873 682Q888 682 888 672Q888 650 880 642Q878 637 858 637Q787 633 769 597L620 7Q618 0 599 0Q585 0 582 2Q579 5 453 305L326 604L261 344Q196 88 196 79Q201 46 268 46H278Q284 41 284 38T282 19Q278 6 272 0H259Q228 2 151 2Q123 2 100 2T63 2T46 1Q31 1 31 10Q31 14 34 26T39 40Q41 46 62 46Q130 49 150 85Q154 91 221 362L289 634Q287 635 234 637Z"></path><path stroke-width="1" id="E30-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path><path stroke-width="1" id="E30-MJMATHI-3BC" d="M58 -216Q44 -216 34 -208T23 -186Q23 -176 96 116T173 414Q186 442 219 442Q231 441 239 435T249 423T251 413Q251 401 220 279T187 142Q185 131 185 107V99Q185 26 252 26Q261 26 270 27T287 31T302 38T315 45T327 55T338 65T348 77T356 88T365 100L372 110L408 253Q444 395 448 404Q461 431 491 431Q504 431 512 424T523 412T525 402L449 84Q448 79 448 68Q448 43 455 35T476 26Q485 27 496 35Q517 55 537 131Q543 151 547 152Q549 153 557 153H561Q580 153 580 144Q580 138 575 117T555 63T523 13Q510 0 491 -8Q483 -10 467 -10Q446 -10 429 -4T402 11T385 29T376 44T374 51L368 45Q362 39 350 30T324 12T288 -4T246 -11Q199 -11 153 12L129 -85Q108 -167 104 -180T92 -202Q76 -216 58 -216Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E30-MJMATHI-3B8" x="0" y="0"></use><use xlink:href="#E30-MJMAIN-3D" x="747" y="0"></use><use xlink:href="#E30-MJMAIN-32" x="1803" y="0"></use><use xlink:href="#E30-MJMAIN-2217" x="2526" y="0"></use><use xlink:href="#E30-MJMATHI-4E" x="3249" y="0"></use><use xlink:href="#E30-MJMATHI-65" x="4137" y="0"></use><use xlink:href="#E30-MJMAIN-2217" x="4826" y="0"></use><use xlink:href="#E30-MJMATHI-3BC" x="5548" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-30">\theta = 2*Ne* \mu</script></p><p>What influences the ability for alleles to become fixed? </p><ul><li>Population size</li><li>Fitness effects</li></ul><p>Small populations should fix alleles more rapidly.</p><p></p><p>Suppose we see differences in alleles in gene regions among populations. Populations are habitat or healthy and sick. </p><p> <u>What are alternative hypotheses?</u><br/>
 Limitations?</p><ul><li>Drift!</li><li>Functional differences may be neutral</li><li>Cause and effect</li></ul><p> <u>Steven leading discussion</u><br/></p><p>Think about genetic targets of adaptation (local adaptaiton). How do selective sweeps happen and how are they distinguished?</p><p><em>Tajima&#39;s D</em>: </p><p>D = 0 = neutrally evolving</p><p>D = negative =  all alleles are recent and rare within the population (We expect this under selection)</p><p>D = positive = rare mutations are missing relative to what we expect from a random mutation processs. </p><p></p><p> </p><p></p><p></p><hr /><p> <div id='id-section20'/><br/></p><h3><a name='header-n4014' class='md-header-anchor '></a>Page 20: 2017-03-29. Week 10, Day 18</h3><p>Lauren Ashlock info update</p><p>Admixture coding session:</p><p>For a dataset of <em>i</em> individuals in <em>j</em> SNPs.</p><p><span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-31-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="14.795ex" height="2.653ex" viewBox="0 -833.9 6369.8 1142.4" role="img" focusable="false" style="vertical-align: -0.717ex;"><defs><path stroke-width="1" id="E31-MJMATHI-50" d="M287 628Q287 635 230 637Q206 637 199 638T192 648Q192 649 194 659Q200 679 203 681T397 683Q587 682 600 680Q664 669 707 631T751 530Q751 453 685 389Q616 321 507 303Q500 302 402 301H307L277 182Q247 66 247 59Q247 55 248 54T255 50T272 48T305 46H336Q342 37 342 35Q342 19 335 5Q330 0 319 0Q316 0 282 1T182 2Q120 2 87 2T51 1Q33 1 33 11Q33 13 36 25Q40 41 44 43T67 46Q94 46 127 49Q141 52 146 61Q149 65 218 339T287 628ZM645 554Q645 567 643 575T634 597T609 619T560 635Q553 636 480 637Q463 637 445 637T416 636T404 636Q391 635 386 627Q384 621 367 550T332 412T314 344Q314 342 395 342H407H430Q542 342 590 392Q617 419 631 471T645 554Z"></path><path stroke-width="1" id="E31-MJMATHI-72" d="M21 287Q22 290 23 295T28 317T38 348T53 381T73 411T99 433T132 442Q161 442 183 430T214 408T225 388Q227 382 228 382T236 389Q284 441 347 441H350Q398 441 422 400Q430 381 430 363Q430 333 417 315T391 292T366 288Q346 288 334 299T322 328Q322 376 378 392Q356 405 342 405Q286 405 239 331Q229 315 224 298T190 165Q156 25 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 114 189T154 366Q154 405 128 405Q107 405 92 377T68 316T57 280Q55 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E31-MJMAIN-28" d="M94 250Q94 319 104 381T127 488T164 576T202 643T244 695T277 729T302 750H315H319Q333 750 333 741Q333 738 316 720T275 667T226 581T184 443T167 250T184 58T225 -81T274 -167T316 -220T333 -241Q333 -250 318 -250H315H302L274 -226Q180 -141 137 -14T94 250Z"></path><path stroke-width="1" id="E31-MJMATHI-47" d="M50 252Q50 367 117 473T286 641T490 704Q580 704 633 653Q642 643 648 636T656 626L657 623Q660 623 684 649Q691 655 699 663T715 679T725 690L740 705H746Q760 705 760 698Q760 694 728 561Q692 422 692 421Q690 416 687 415T669 413H653Q647 419 647 422Q647 423 648 429T650 449T651 481Q651 552 619 605T510 659Q492 659 471 656T418 643T357 615T294 567T236 496T189 394T158 260Q156 242 156 221Q156 173 170 136T206 79T256 45T308 28T353 24Q407 24 452 47T514 106Q517 114 529 161T541 214Q541 222 528 224T468 227H431Q425 233 425 235T427 254Q431 267 437 273H454Q494 271 594 271Q634 271 659 271T695 272T707 272Q721 272 721 263Q721 261 719 249Q714 230 709 228Q706 227 694 227Q674 227 653 224Q646 221 643 215T629 164Q620 131 614 108Q589 6 586 3Q584 1 581 1Q571 1 553 21T530 52Q530 53 528 52T522 47Q448 -22 322 -22Q201 -22 126 55T50 252Z"></path><path stroke-width="1" id="E31-MJMAIN-7C" d="M139 -249H137Q125 -249 119 -235V251L120 737Q130 750 139 750Q152 750 159 735V-235Q151 -249 141 -249H139Z"></path><path stroke-width="1" id="E31-MJMATHI-4B" d="M285 628Q285 635 228 637Q205 637 198 638T191 647Q191 649 193 661Q199 681 203 682Q205 683 214 683H219Q260 681 355 681Q389 681 418 681T463 682T483 682Q500 682 500 674Q500 669 497 660Q496 658 496 654T495 648T493 644T490 641T486 639T479 638T470 637T456 637Q416 636 405 634T387 623L306 305Q307 305 490 449T678 597Q692 611 692 620Q692 635 667 637Q651 637 651 648Q651 650 654 662T659 677Q662 682 676 682Q680 682 711 681T791 680Q814 680 839 681T869 682Q889 682 889 672Q889 650 881 642Q878 637 862 637Q787 632 726 586Q710 576 656 534T556 455L509 418L518 396Q527 374 546 329T581 244Q656 67 661 61Q663 59 666 57Q680 47 717 46H738Q744 38 744 37T741 19Q737 6 731 0H720Q680 3 625 3Q503 3 488 0H478Q472 6 472 9T474 27Q478 40 480 43T491 46H494Q544 46 544 71Q544 75 517 141T485 216L427 354L359 301L291 248L268 155Q245 63 245 58Q245 51 253 49T303 46H334Q340 37 340 35Q340 19 333 5Q328 0 317 0Q314 0 280 1T180 2Q118 2 85 2T49 1Q31 1 31 11Q31 13 34 25Q38 41 42 43T65 46Q92 46 125 49Q139 52 144 61Q147 65 216 339T285 628Z"></path><path stroke-width="1" id="E31-MJMAIN-2C" d="M78 35T78 60T94 103T137 121Q165 121 187 96T210 8Q210 -27 201 -60T180 -117T154 -158T130 -185T117 -194Q113 -194 104 -185T95 -172Q95 -168 106 -156T131 -126T157 -76T173 -3V9L172 8Q170 7 167 6T161 3T152 1T140 0Q113 0 96 17Z"></path><path stroke-width="1" id="E31-MJMATHI-51" d="M399 -80Q399 -47 400 -30T402 -11V-7L387 -11Q341 -22 303 -22Q208 -22 138 35T51 201Q50 209 50 244Q50 346 98 438T227 601Q351 704 476 704Q514 704 524 703Q621 689 680 617T740 435Q740 255 592 107Q529 47 461 16L444 8V3Q444 2 449 -24T470 -66T516 -82Q551 -82 583 -60T625 -3Q631 11 638 11Q647 11 649 2Q649 -6 639 -34T611 -100T557 -165T481 -194Q399 -194 399 -87V-80ZM636 468Q636 523 621 564T580 625T530 655T477 665Q429 665 379 640Q277 591 215 464T153 216Q153 110 207 59Q231 38 236 38V46Q236 86 269 120T347 155Q372 155 390 144T417 114T429 82T435 55L448 64Q512 108 557 185T619 334T636 468ZM314 18Q362 18 404 39L403 49Q399 104 366 115Q354 117 347 117Q344 117 341 117T337 118Q317 118 296 98T274 52Q274 18 314 18Z"></path><path stroke-width="1" id="E31-MJMAIN-29" d="M60 749L64 750Q69 750 74 750H86L114 726Q208 641 251 514T294 250Q294 182 284 119T261 12T224 -76T186 -143T145 -194T113 -227T90 -246Q87 -249 86 -250H74Q66 -250 63 -250T58 -247T55 -238Q56 -237 66 -225Q221 -64 221 250T66 725Q56 737 55 738Q55 746 60 749Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><use xlink:href="#E31-MJMATHI-50" x="0" y="0"></use><use xlink:href="#E31-MJMATHI-72" x="751" y="0"></use><use xlink:href="#E31-MJMAIN-28" x="1203" y="0"></use><use xlink:href="#E31-MJMATHI-47" x="1592" y="0"></use><use xlink:href="#E31-MJMAIN-7C" x="2379" y="0"></use><use xlink:href="#E31-MJMATHI-4B" x="2657" y="0"></use><use xlink:href="#E31-MJMAIN-2C" x="3547" y="0"></use><use xlink:href="#E31-MJMATHI-51" x="3992" y="0"></use><use xlink:href="#E31-MJMAIN-2C" x="4783" y="0"></use><use xlink:href="#E31-MJMATHI-50" x="5228" y="0"></use><use xlink:href="#E31-MJMAIN-29" x="5980" y="0"></use></g></svg></span><script type="math/tex" id="MathJax-Element-31">Pr(G|K,Q,P)</script></p><p>G = genotype</p><p>Q = ancestry (proportion of an individual&#39;s genome)</p><p>P = allele frequency</p><p></p><p></p><p></p><hr /><p> <div id='id-section21'/><br/></p><h3><a name='header-n4038' class='md-header-anchor '></a>Page 21: 2017-04-03. Karl&#39;s guest lecture; FST and FST outlier</h3><p>Concepts:</p><ol start='' ><li>Inbreeding produces structured populations</li><li>selective sweeps change allele frequencies in populations</li><li>Empirical p-values created from a distribution of putataively neutral loci are super useful for finding natural selection</li><li>Methods: OutFlank 2015</li></ol><p>Two questions to consider:</p><ol start='' ><li>What challenges do outlier detection methods face? </li><li>How is LD (linkage disequilibrium) our friend or foe? </li></ol><p><strong>Inbreeding coefficient and Structured Populations</strong></p><p>F-statistics: created by Sewell Wright</p><ul><li><p>Considers heterozygosity</p><ul><li>at the individual <em>I</em></li><li>Subpopulation (<em>S</em>)</li><li>Total populations (<em>T</em>)</li></ul></li><li><p>Heterozygosity: 2 alelles sampled from a population are identical by descent. </p></li></ul><p>Fst: <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-32-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="6.139ex" height="3.772ex" viewBox="0 -1052.8 2643.3 1624.1" role="img" focusable="false" style="vertical-align: -1.327ex;"><defs><path stroke-width="1" id="E32-MJMATHI-48" d="M228 637Q194 637 192 641Q191 643 191 649Q191 673 202 682Q204 683 219 683Q260 681 355 681Q389 681 418 681T463 682T483 682Q499 682 499 672Q499 670 497 658Q492 641 487 638H485Q483 638 480 638T473 638T464 637T455 637Q416 636 405 634T387 623Q384 619 355 500Q348 474 340 442T328 395L324 380Q324 378 469 378H614L615 381Q615 384 646 504Q674 619 674 627T617 637Q594 637 587 639T580 648Q580 650 582 660Q586 677 588 679T604 682Q609 682 646 681T740 680Q802 680 835 681T871 682Q888 682 888 672Q888 645 876 638H874Q872 638 869 638T862 638T853 637T844 637Q805 636 794 634T776 623Q773 618 704 340T634 58Q634 51 638 51Q646 48 692 46H723Q729 38 729 37T726 19Q722 6 716 0H701Q664 2 567 2Q533 2 504 2T458 2T437 1Q420 1 420 10Q420 15 423 24Q428 43 433 45Q437 46 448 46H454Q481 46 514 49Q520 50 522 50T528 55T534 64T540 82T547 110T558 153Q565 181 569 198Q602 330 602 331T457 332H312L279 197Q245 63 245 58Q245 51 253 49T303 46H334Q340 38 340 37T337 19Q333 6 327 0H312Q275 2 178 2Q144 2 115 2T69 2T48 1Q31 1 31 10Q31 12 34 24Q39 43 44 45Q48 46 59 46H65Q92 46 125 49Q139 52 144 61Q147 65 216 339T285 628Q285 635 228 637Z"></path><path stroke-width="1" id="E32-MJMATHI-74" d="M26 385Q19 392 19 395Q19 399 22 411T27 425Q29 430 36 430T87 431H140L159 511Q162 522 166 540T173 566T179 586T187 603T197 615T211 624T229 626Q247 625 254 615T261 596Q261 589 252 549T232 470L222 433Q222 431 272 431H323Q330 424 330 420Q330 398 317 385H210L174 240Q135 80 135 68Q135 26 162 26Q197 26 230 60T283 144Q285 150 288 151T303 153H307Q322 153 322 145Q322 142 319 133Q314 117 301 95T267 48T216 6T155 -11Q125 -11 98 4T59 56Q57 64 57 83V101L92 241Q127 382 128 383Q128 385 77 385H26Z"></path><path stroke-width="1" id="E32-MJMAIN-2212" d="M84 237T84 250T98 270H679Q694 262 694 250T679 230H98Q84 237 84 250Z"></path><path stroke-width="1" id="E32-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g transform="translate(120,0)"><rect stroke="none" width="2403" height="60" x="0" y="220"></rect><g transform="translate(60,504)"><use transform="scale(0.707)" xlink:href="#E32-MJMATHI-48" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E32-MJMATHI-74" x="1175" y="-213"></use><use transform="scale(0.707)" xlink:href="#E32-MJMAIN-2212" x="1187" y="0"></use><g transform="translate(1389,0)"><use transform="scale(0.707)" xlink:href="#E32-MJMATHI-48" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E32-MJMATHI-73" x="1175" y="-213"></use></g></g><g transform="translate(781,-376)"><use transform="scale(0.707)" xlink:href="#E32-MJMATHI-48" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E32-MJMATHI-74" x="1175" y="-213"></use></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-32">\frac{H_t - H_s}{H_t}</script></p><p>Fis: <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-33-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="21.439ex" height="4.28ex" viewBox="0 -1184.2 9230.6 1843" role="img" focusable="false" style="vertical-align: -1.53ex;"><defs><path stroke-width="1" id="E33-MJMATHI-45" d="M492 213Q472 213 472 226Q472 230 477 250T482 285Q482 316 461 323T364 330H312Q311 328 277 192T243 52Q243 48 254 48T334 46Q428 46 458 48T518 61Q567 77 599 117T670 248Q680 270 683 272Q690 274 698 274Q718 274 718 261Q613 7 608 2Q605 0 322 0H133Q31 0 31 11Q31 13 34 25Q38 41 42 43T65 46Q92 46 125 49Q139 52 144 61Q146 66 215 342T285 622Q285 629 281 629Q273 632 228 634H197Q191 640 191 642T193 659Q197 676 203 680H757Q764 676 764 669Q764 664 751 557T737 447Q735 440 717 440H705Q698 445 698 453L701 476Q704 500 704 528Q704 558 697 578T678 609T643 625T596 632T532 634H485Q397 633 392 631Q388 629 386 622Q385 619 355 499T324 377Q347 376 372 376H398Q464 376 489 391T534 472Q538 488 540 490T557 493Q562 493 565 493T570 492T572 491T574 487T577 483L544 351Q511 218 508 216Q505 213 492 213Z"></path><path stroke-width="1" id="E33-MJMATHI-78" d="M52 289Q59 331 106 386T222 442Q257 442 286 424T329 379Q371 442 430 442Q467 442 494 420T522 361Q522 332 508 314T481 292T458 288Q439 288 427 299T415 328Q415 374 465 391Q454 404 425 404Q412 404 406 402Q368 386 350 336Q290 115 290 78Q290 50 306 38T341 26Q378 26 414 59T463 140Q466 150 469 151T485 153H489Q504 153 504 145Q504 144 502 134Q486 77 440 33T333 -11Q263 -11 227 52Q186 -10 133 -10H127Q78 -10 57 16T35 71Q35 103 54 123T99 143Q142 143 142 101Q142 81 130 66T107 46T94 41L91 40Q91 39 97 36T113 29T132 26Q168 26 194 71Q203 87 217 139T245 247T261 313Q266 340 266 352Q266 380 251 392T217 404Q177 404 142 372T93 290Q91 281 88 280T72 278H58Q52 284 52 289Z"></path><path stroke-width="1" id="E33-MJMATHI-70" d="M23 287Q24 290 25 295T30 317T40 348T55 381T75 411T101 433T134 442Q209 442 230 378L240 387Q302 442 358 442Q423 442 460 395T497 281Q497 173 421 82T249 -10Q227 -10 210 -4Q199 1 187 11T168 28L161 36Q160 35 139 -51T118 -138Q118 -144 126 -145T163 -148H188Q194 -155 194 -157T191 -175Q188 -187 185 -190T172 -194Q170 -194 161 -194T127 -193T65 -192Q-5 -192 -24 -194H-32Q-39 -187 -39 -183Q-37 -156 -26 -148H-6Q28 -147 33 -136Q36 -130 94 103T155 350Q156 355 156 364Q156 405 131 405Q109 405 94 377T71 316T59 280Q57 278 43 278H29Q23 284 23 287ZM178 102Q200 26 252 26Q282 26 310 49T356 107Q374 141 392 215T411 325V331Q411 405 350 405Q339 405 328 402T306 393T286 380T269 365T254 350T243 336T235 326L232 322Q232 321 229 308T218 264T204 212Q178 106 178 102Z"></path><path stroke-width="1" id="E33-MJMATHI-65" d="M39 168Q39 225 58 272T107 350T174 402T244 433T307 442H310Q355 442 388 420T421 355Q421 265 310 237Q261 224 176 223Q139 223 138 221Q138 219 132 186T125 128Q125 81 146 54T209 26T302 45T394 111Q403 121 406 121Q410 121 419 112T429 98T420 82T390 55T344 24T281 -1T205 -11Q126 -11 83 42T39 168ZM373 353Q367 405 305 405Q272 405 244 391T199 357T170 316T154 280T149 261Q149 260 169 260Q282 260 327 284T373 353Z"></path><path stroke-width="1" id="E33-MJMATHI-63" d="M34 159Q34 268 120 355T306 442Q362 442 394 418T427 355Q427 326 408 306T360 285Q341 285 330 295T319 325T330 359T352 380T366 386H367Q367 388 361 392T340 400T306 404Q276 404 249 390Q228 381 206 359Q162 315 142 235T121 119Q121 73 147 50Q169 26 205 26H209Q321 26 394 111Q403 121 406 121Q410 121 419 112T429 98T420 83T391 55T346 25T282 0T202 -11Q127 -11 81 37T34 159Z"></path><path stroke-width="1" id="E33-MJMATHI-74" d="M26 385Q19 392 19 395Q19 399 22 411T27 425Q29 430 36 430T87 431H140L159 511Q162 522 166 540T173 566T179 586T187 603T197 615T211 624T229 626Q247 625 254 615T261 596Q261 589 252 549T232 470L222 433Q222 431 272 431H323Q330 424 330 420Q330 398 317 385H210L174 240Q135 80 135 68Q135 26 162 26Q197 26 230 60T283 144Q285 150 288 151T303 153H307Q322 153 322 145Q322 142 319 133Q314 117 301 95T267 48T216 6T155 -11Q125 -11 98 4T59 56Q57 64 57 83V101L92 241Q127 382 128 383Q128 385 77 385H26Z"></path><path stroke-width="1" id="E33-MJMATHI-64" d="M366 683Q367 683 438 688T511 694Q523 694 523 686Q523 679 450 384T375 83T374 68Q374 26 402 26Q411 27 422 35Q443 55 463 131Q469 151 473 152Q475 153 483 153H487H491Q506 153 506 145Q506 140 503 129Q490 79 473 48T445 8T417 -8Q409 -10 393 -10Q359 -10 336 5T306 36L300 51Q299 52 296 50Q294 48 292 46Q233 -10 172 -10Q117 -10 75 30T33 157Q33 205 53 255T101 341Q148 398 195 420T280 442Q336 442 364 400Q369 394 369 396Q370 400 396 505T424 616Q424 629 417 632T378 637H357Q351 643 351 645T353 664Q358 683 366 683ZM352 326Q329 405 277 405Q242 405 210 374T160 293Q131 214 119 129Q119 126 119 118T118 106Q118 61 136 44T179 26Q233 26 290 98L298 109L352 326Z"></path><path stroke-width="1" id="E33-MJMAIN-28" d="M94 250Q94 319 104 381T127 488T164 576T202 643T244 695T277 729T302 750H315H319Q333 750 333 741Q333 738 316 720T275 667T226 581T184 443T167 250T184 58T225 -81T274 -167T316 -220T333 -241Q333 -250 318 -250H315H302L274 -226Q180 -141 137 -14T94 250Z"></path><path stroke-width="1" id="E33-MJMATHI-48" d="M228 637Q194 637 192 641Q191 643 191 649Q191 673 202 682Q204 683 219 683Q260 681 355 681Q389 681 418 681T463 682T483 682Q499 682 499 672Q499 670 497 658Q492 641 487 638H485Q483 638 480 638T473 638T464 637T455 637Q416 636 405 634T387 623Q384 619 355 500Q348 474 340 442T328 395L324 380Q324 378 469 378H614L615 381Q615 384 646 504Q674 619 674 627T617 637Q594 637 587 639T580 648Q580 650 582 660Q586 677 588 679T604 682Q609 682 646 681T740 680Q802 680 835 681T871 682Q888 682 888 672Q888 645 876 638H874Q872 638 869 638T862 638T853 637T844 637Q805 636 794 634T776 623Q773 618 704 340T634 58Q634 51 638 51Q646 48 692 46H723Q729 38 729 37T726 19Q722 6 716 0H701Q664 2 567 2Q533 2 504 2T458 2T437 1Q420 1 420 10Q420 15 423 24Q428 43 433 45Q437 46 448 46H454Q481 46 514 49Q520 50 522 50T528 55T534 64T540 82T547 110T558 153Q565 181 569 198Q602 330 602 331T457 332H312L279 197Q245 63 245 58Q245 51 253 49T303 46H334Q340 38 340 37T337 19Q333 6 327 0H312Q275 2 178 2Q144 2 115 2T69 2T48 1Q31 1 31 10Q31 12 34 24Q39 43 44 45Q48 46 59 46H65Q92 46 125 49Q139 52 144 61Q147 65 216 339T285 628Q285 635 228 637Z"></path><path stroke-width="1" id="E33-MJMATHI-73" d="M131 289Q131 321 147 354T203 415T300 442Q362 442 390 415T419 355Q419 323 402 308T364 292Q351 292 340 300T328 326Q328 342 337 354T354 372T367 378Q368 378 368 379Q368 382 361 388T336 399T297 405Q249 405 227 379T204 326Q204 301 223 291T278 274T330 259Q396 230 396 163Q396 135 385 107T352 51T289 7T195 -10Q118 -10 86 19T53 87Q53 126 74 143T118 160Q133 160 146 151T160 120Q160 94 142 76T111 58Q109 57 108 57T107 55Q108 52 115 47T146 34T201 27Q237 27 263 38T301 66T318 97T323 122Q323 150 302 164T254 181T195 196T148 231Q131 256 131 289Z"></path><path stroke-width="1" id="E33-MJMAIN-29" d="M60 749L64 750Q69 750 74 750H86L114 726Q208 641 251 514T294 250Q294 182 284 119T261 12T224 -76T186 -143T145 -194T113 -227T90 -246Q87 -249 86 -250H74Q66 -250 63 -250T58 -247T55 -238Q56 -237 66 -225Q221 -64 221 250T66 725Q56 737 55 738Q55 746 60 749Z"></path><path stroke-width="1" id="E33-MJMAIN-2212" d="M84 237T84 250T98 270H679Q694 262 694 250T679 230H98Q84 237 84 250Z"></path><path stroke-width="1" id="E33-MJMATHI-4F" d="M740 435Q740 320 676 213T511 42T304 -22Q207 -22 138 35T51 201Q50 209 50 244Q50 346 98 438T227 601Q351 704 476 704Q514 704 524 703Q621 689 680 617T740 435ZM637 476Q637 565 591 615T476 665Q396 665 322 605Q242 542 200 428T157 216Q157 126 200 73T314 19Q404 19 485 98T608 313Q637 408 637 476Z"></path><path stroke-width="1" id="E33-MJMATHI-62" d="M73 647Q73 657 77 670T89 683Q90 683 161 688T234 694Q246 694 246 685T212 542Q204 508 195 472T180 418L176 399Q176 396 182 402Q231 442 283 442Q345 442 383 396T422 280Q422 169 343 79T173 -11Q123 -11 82 27T40 150V159Q40 180 48 217T97 414Q147 611 147 623T109 637Q104 637 101 637H96Q86 637 83 637T76 640T73 647ZM336 325V331Q336 405 275 405Q258 405 240 397T207 376T181 352T163 330L157 322L136 236Q114 150 114 114Q114 66 138 42Q154 26 178 26Q211 26 245 58Q270 81 285 114T318 219Q336 291 336 325Z"></path><path stroke-width="1" id="E33-MJMATHI-72" d="M21 287Q22 290 23 295T28 317T38 348T53 381T73 411T99 433T132 442Q161 442 183 430T214 408T225 388Q227 382 228 382T236 389Q284 441 347 441H350Q398 441 422 400Q430 381 430 363Q430 333 417 315T391 292T366 288Q346 288 334 299T322 328Q322 376 378 392Q356 405 342 405Q286 405 239 331Q229 315 224 298T190 165Q156 25 151 16Q138 -11 108 -11Q95 -11 87 -5T76 7T74 17Q74 30 114 189T154 366Q154 405 128 405Q107 405 92 377T68 316T57 280Q55 278 41 278H27Q21 284 21 287Z"></path><path stroke-width="1" id="E33-MJMATHI-76" d="M173 380Q173 405 154 405Q130 405 104 376T61 287Q60 286 59 284T58 281T56 279T53 278T49 278T41 278H27Q21 284 21 287Q21 294 29 316T53 368T97 419T160 441Q202 441 225 417T249 361Q249 344 246 335Q246 329 231 291T200 202T182 113Q182 86 187 69Q200 26 250 26Q287 26 319 60T369 139T398 222T409 277Q409 300 401 317T383 343T365 361T357 383Q357 405 376 424T417 443Q436 443 451 425T467 367Q467 340 455 284T418 159T347 40T241 -11Q177 -11 139 22Q102 54 102 117Q102 148 110 181T151 298Q173 362 173 380Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g transform="translate(120,0)"><rect stroke="none" width="8990" height="60" x="0" y="220"></rect><g transform="translate(60,569)"><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-45" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-78" x="764" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-70" x="1337" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-65" x="1840" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-63" x="2307" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-74" x="2740" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-65" x="3102" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-64" x="3568" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMAIN-28" x="4092" y="0"></use><g transform="translate(3168,0)"><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-48" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E33-MJMATHI-73" x="1175" y="-213"></use></g><use transform="scale(0.707)" xlink:href="#E33-MJMAIN-29" x="5744" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMAIN-2212" x="6134" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-4F" x="6912" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-62" x="7676" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-73" x="8105" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-72" x="8575" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-76" x="9026" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-65" x="9512" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-64" x="9978" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMAIN-28" x="10502" y="0"></use><g transform="translate(7701,0)"><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-48" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E33-MJMATHI-73" x="1175" y="-213"></use></g><use transform="scale(0.707)" xlink:href="#E33-MJMAIN-29" x="12155" y="0"></use></g><g transform="translate(2326,-424)"><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-45" x="0" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-78" x="764" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-70" x="1337" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-65" x="1840" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-63" x="2307" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-74" x="2740" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-65" x="3102" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-64" x="3568" y="0"></use><use transform="scale(0.707)" xlink:href="#E33-MJMAIN-28" x="4092" y="0"></use><g transform="translate(3168,0)"><use transform="scale(0.707)" xlink:href="#E33-MJMATHI-48" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E33-MJMATHI-73" x="1175" y="-213"></use></g><use transform="scale(0.707)" xlink:href="#E33-MJMAIN-29" x="5744" y="0"></use></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-33">\frac{Expected (H_s)-Obsrved(H_s)}{Expected(H_s)}</script></p><p>Fit: <span class="MathJax_Preview"></span><span class="MathJax_SVG" id="MathJax-Element-34-Frame" tabindex="-1" style="font-size: 100%; display: inline-block;"><svg xmlns:xlink="http://www.w3.org/1999/xlink" width="6.18ex" height="3.772ex" viewBox="0 -1052.8 2660.8 1624.1" role="img" focusable="false" style="vertical-align: -1.327ex;"><defs><path stroke-width="1" id="E34-MJMATHI-48" d="M228 637Q194 637 192 641Q191 643 191 649Q191 673 202 682Q204 683 219 683Q260 681 355 681Q389 681 418 681T463 682T483 682Q499 682 499 672Q499 670 497 658Q492 641 487 638H485Q483 638 480 638T473 638T464 637T455 637Q416 636 405 634T387 623Q384 619 355 500Q348 474 340 442T328 395L324 380Q324 378 469 378H614L615 381Q615 384 646 504Q674 619 674 627T617 637Q594 637 587 639T580 648Q580 650 582 660Q586 677 588 679T604 682Q609 682 646 681T740 680Q802 680 835 681T871 682Q888 682 888 672Q888 645 876 638H874Q872 638 869 638T862 638T853 637T844 637Q805 636 794 634T776 623Q773 618 704 340T634 58Q634 51 638 51Q646 48 692 46H723Q729 38 729 37T726 19Q722 6 716 0H701Q664 2 567 2Q533 2 504 2T458 2T437 1Q420 1 420 10Q420 15 423 24Q428 43 433 45Q437 46 448 46H454Q481 46 514 49Q520 50 522 50T528 55T534 64T540 82T547 110T558 153Q565 181 569 198Q602 330 602 331T457 332H312L279 197Q245 63 245 58Q245 51 253 49T303 46H334Q340 38 340 37T337 19Q333 6 327 0H312Q275 2 178 2Q144 2 115 2T69 2T48 1Q31 1 31 10Q31 12 34 24Q39 43 44 45Q48 46 59 46H65Q92 46 125 49Q139 52 144 61Q147 65 216 339T285 628Q285 635 228 637Z"></path><path stroke-width="1" id="E34-MJMATHI-74" d="M26 385Q19 392 19 395Q19 399 22 411T27 425Q29 430 36 430T87 431H140L159 511Q162 522 166 540T173 566T179 586T187 603T197 615T211 624T229 626Q247 625 254 615T261 596Q261 589 252 549T232 470L222 433Q222 431 272 431H323Q330 424 330 420Q330 398 317 385H210L174 240Q135 80 135 68Q135 26 162 26Q197 26 230 60T283 144Q285 150 288 151T303 153H307Q322 153 322 145Q322 142 319 133Q314 117 301 95T267 48T216 6T155 -11Q125 -11 98 4T59 56Q57 64 57 83V101L92 241Q127 382 128 383Q128 385 77 385H26Z"></path><path stroke-width="1" id="E34-MJMAIN-2212" d="M84 237T84 250T98 270H679Q694 262 694 250T679 230H98Q84 237 84 250Z"></path><path stroke-width="1" id="E34-MJMATHI-49" d="M43 1Q26 1 26 10Q26 12 29 24Q34 43 39 45Q42 46 54 46H60Q120 46 136 53Q137 53 138 54Q143 56 149 77T198 273Q210 318 216 344Q286 624 286 626Q284 630 284 631Q274 637 213 637H193Q184 643 189 662Q193 677 195 680T209 683H213Q285 681 359 681Q481 681 487 683H497Q504 676 504 672T501 655T494 639Q491 637 471 637Q440 637 407 634Q393 631 388 623Q381 609 337 432Q326 385 315 341Q245 65 245 59Q245 52 255 50T307 46H339Q345 38 345 37T342 19Q338 6 332 0H316Q279 2 179 2Q143 2 113 2T65 2T43 1Z"></path></defs><g stroke="currentColor" fill="currentColor" stroke-width="0" transform="matrix(1 0 0 -1 0 0)"><g transform="translate(120,0)"><rect stroke="none" width="2420" height="60" x="0" y="220"></rect><g transform="translate(60,504)"><use transform="scale(0.707)" xlink:href="#E34-MJMATHI-48" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E34-MJMATHI-74" x="1175" y="-213"></use><use transform="scale(0.707)" xlink:href="#E34-MJMAIN-2212" x="1187" y="0"></use><g transform="translate(1389,0)"><use transform="scale(0.707)" xlink:href="#E34-MJMATHI-48" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E34-MJMATHI-49" x="1175" y="-213"></use></g></g><g transform="translate(790,-376)"><use transform="scale(0.707)" xlink:href="#E34-MJMATHI-48" x="0" y="0"></use><use transform="scale(0.5)" xlink:href="#E34-MJMATHI-74" x="1175" y="-213"></use></g></g></g></svg></span><script type="math/tex" id="MathJax-Element-34">\frac{H_t - H_I}{H_t}</script></p><p></p><p>Inbreeding influences Fst. </p><p>Inbreeding coefficient goes from 0-1; 0 = no inbreeding, 1 is completely inbreeding. </p><p>Inbreeding at the local within pop scale, is concave. At the regional level/scale, genetic variation is higher. </p><p>Total genetic variation</p><p></p><p></p><hr /><p> <div id='id-section22'/><br/></p><h3><a name='header-n4107' class='md-header-anchor '></a>Page 22: 2017-04-05. Week 11, Day 20; Annotation</h3><p>Melissa info update on annotations:</p><p>Annotations take a long time. Imagine a funnel. Broad to specific</p><ol start='' ><li><strong>Processing raw reads:</strong> started with a fastq file using the program trimmomatic</li><li><strong>Transcriptome assembly:</strong> Used program trinity to get a .fasta file</li><li>a) Map reads: .sam file using BWA</li><li>b) Annotate: .fasta files use BLAST</li></ol><div class="md-diagram-panel" mdtype="math_block"><svg id="mermaidChart9" width="788" xmlns="http://www.w3.org/2000/svg" height="732" viewBox="0 0 808 752" style="height: 792px;"><g><g class="output"><g class="clusters"></g><g class="edgePaths"><g class="edgePath" style="opacity: 1;"><path class="path" d="M396.25,61L396.25,96.5L396.25,132" marker-end="url(#arrowhead371)" style="fill:none"></path><defs><marker id="arrowhead371" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M396.25,173L396.25,208.5L396.25,244" marker-end="url(#arrowhead372)" style="fill:none"></path><defs><marker id="arrowhead372" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M343.71875,285L252.75,320.5L252.75,356" marker-end="url(#arrowhead373)" style="fill:none"></path><defs><marker id="arrowhead373" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M502.1361607142857,285L685.5,320.5L685.5,376.5L685.5,432.5L685.5,468" marker-end="url(#arrowhead374)" style="fill:none"></path><defs><marker id="arrowhead374" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M199.39508928571428,397L107,432.5L107,468" marker-end="url(#arrowhead375)" style="fill:none"></path><defs><marker id="arrowhead375" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M306.1049107142857,397L398.5,432.5L398.5,468" marker-end="url(#arrowhead376)" style="fill:none"></path><defs><marker id="arrowhead376" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M107,509L107,534L267.16483516483515,559" marker-end="url(#arrowhead377)" style="fill:none"></path><defs><marker id="arrowhead377" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M398.5,509L398.5,534L398.5,559" marker-end="url(#arrowhead378)" style="fill:none"></path><defs><marker id="arrowhead378" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M685.5,509L685.5,534L527.8076923076924,559" marker-end="url(#arrowhead379)" style="fill:none"></path><defs><marker id="arrowhead379" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g><g class="edgePath" style="opacity: 1;"><path class="path" d="M398.5,600L398.5,635.5L398.5,671" marker-end="url(#arrowhead380)" style="fill:none"></path><defs><marker id="arrowhead380" viewBox="0 0 10 10" refX="9" refY="5" markerUnits="strokeWidth" markerWidth="8" markerHeight="6" orient="auto"><path d="M 0 0 L 10 5 L 0 10 z" style="fill: #333"></path></marker></defs></g></g><g class="edgeLabels"><g class="edgeLabel" transform="translate(396.25,96.5)" style="opacity: 1;"><g transform="translate(-53,-10.5)" class="label"><foreignObject width="106" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">trimmomatic</span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(396.25,208.5)" style="opacity: 1;"><g transform="translate(-33.5,-10.5)" class="label"><foreignObject width="67" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">Trinity</span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(252.75,320.5)" style="opacity: 1;"><g transform="translate(-14.5,-10.5)" class="label"><foreignObject width="29" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">BWA</span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(685.5,376.5)" style="opacity: 1;"><g transform="translate(-24,-10.5)" class="label"><foreignObject width="48" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">BLAST</span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(107,432.5)" style="opacity: 1;"><g transform="translate(-29,-10.5)" class="label"><foreignObject width="58" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">DESeq2</span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(398.5,432.5)" style="opacity: 1;"><g transform="translate(-91.5,-10.5)" class="label"><foreignObject width="183" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">PCA/DAPCA/Admixture</span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="" style="opacity: 1;"><g transform="translate(0,0)" class="label"><foreignObject width="0" height="0"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel"></span></div></foreignObject></g></g><g class="edgeLabel" transform="translate(398.5,635.5)" style="opacity: 1;"><g transform="translate(-144.5,-10.5)" class="label"><foreignObject width="289" height="21"><div style="display: inline-block; white-space: nowrap;"><span class="edgeLabel">Functional Enrichment Analysis</span></div></foreignObject></g></g></g><g class="nodes"><g class="node" id="A" transform="translate(396.25,40.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-92" y="-20.5" width="184" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-82,-10.5)"><foreignObject width="164" height="21"><div style="display: inline-block; white-space: nowrap;">Process raw reads</div></foreignObject></g></g></g><g class="node" id="B" transform="translate(396.25,152.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-58" y="-20.5" width="116" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-48,-10.5)"><foreignObject width="96" height="21"><div style="display: inline-block; white-space: nowrap;">fastq file</div></foreignObject></g></g></g><g class="node" id="C" transform="translate(396.25,264.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-178.5" y="-20.5" width="357" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-168.5,-10.5)"><foreignObject width="337" height="21"><div style="display: inline-block; white-space: nowrap;">Transcriptome assembly fasta format</div></foreignObject></g></g></g><g class="node" id="D" transform="translate(252.75,376.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-96.5" y="-20.5" width="193" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-86.5,-10.5)"><foreignObject width="173" height="21"><div style="display: inline-block; white-space: nowrap;">Map reads sam file</div></foreignObject></g></g></g><g class="node" id="E" transform="translate(685.5,488.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-82.5" y="-20.5" width="165" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-72.5,-10.5)"><foreignObject width="145" height="21"><div style="display: inline-block; white-space: nowrap;">Gene Annotation</div></foreignObject></g></g></g><g class="node" id="T" transform="translate(107,488.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-87" y="-20.5" width="174" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-77,-10.5)"><foreignObject width="154" height="21"><div style="display: inline-block; white-space: nowrap;">DGE counts table</div></foreignObject></g></g></g><g class="node" id="G" transform="translate(398.5,488.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-154.5" y="-20.5" width="309" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-144.5,-10.5)"><foreignObject width="289" height="21"><div style="display: inline-block; white-space: nowrap;">Population genomics vcf format</div></foreignObject></g></g></g><g class="node" id="H" transform="translate(398.5,579.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-217" y="-20.5" width="434" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-207,-10.5)"><foreignObject width="414" height="21"><div style="display: inline-block; white-space: nowrap;">Genes that are different or under selection</div></foreignObject></g></g></g><g class="node" id="Z" transform="translate(398.5,691.5)" style="opacity: 1;"><rect rx="0" ry="0" x="-130.5" y="-20.5" width="261" height="41"></rect><g class="label" transform="translate(0,0)"><g transform="translate(-120.5,-10.5)"><foreignObject width="241" height="21"><div style="display: inline-block; white-space: nowrap;">Infer biological function</div></foreignObject></g></g></g></g></g></g></svg></div><p></p><p>For the annotation: </p><p>Blast2Go—need to pay</p><p>Brute force</p><p>Pipelines: Trinotate</p><p></p><p>BLAST: taking a fasta file and search a database (NCBI non-redundant)</p><p>Blastp: proteins</p><p>Blastx: translate all 6 reading frames</p><p></p><hr /><p> <div id='id-section23'/><br/>
  <br/></p><h3><a name='header-n4149' class='md-header-anchor '></a>Page 23: 2017-04-10,Week 12, Day 21; Metagenomics, paper discussion Ziegler et al. 2017</h3><p>Announcements:  <br/></p><ol start='' ><li>SSW projects; metagenomics  <br/></li><li>Microbiome vs metagenome  <br/></li><li>Process of generating and analyzing data <br/></li><li>Microbial revolution</li><li>Big questions...</li></ol><p>Info update: SKeller; Metagenomics</p><p>What kind of data do we have? <br/></p><ol start='' ><li>RNA  -&gt; sequencing --&gt; Differential gene expression; what is diversity and structure of the samples? IS there evidence for selection?  <br/></li><li>Metagenomics: Amplify 16s rRNA gene. </li></ol><p>What microbes are present? What are they doing?    <br/></p><p><strong>What is a microbiome?</strong> (Who are they?)  <br/>
Assemblage of microbial taxa associated with host or environment. This includes: pathogens, symbionts, mutualists, and commensals <br/></p><p>Holobiont is host + microbiome</p><p>How do you measure a microbiome?  <br/></p><ol start='' ><li>16s rRNA <br/></li><li>ITS : Internal transcribed spacer  <br/></li></ol><p><strong>Metagenomics</strong>  (What are they doing) <br/></p><p>Genes being expressed by he microbiome, FUNCTION. The focus is more on RNA-seq or shot gun DNA seq.</p><p>How do we process these data?!?! <br/></p><ol start='' ><li>Sample tissue and extract DNA or RNA.  <br/></li><li>PCR target: 16s rRNA.  <br/></li><li>Library prep: barcode samples  <br/></li><li>Cluster sequences based on similiarity  (97%)-- OTUs</li><li>BLAST OTU&#39;s to a database to ID taxonomy.</li></ol><p>REVOLUTION!</p><p>Most microbes are not culturable.   <br/></p><h3><a name='header-n4223' class='md-header-anchor '></a>Big question!!?!?</h3><p></p><p></p><p></p><p>Glossary: <br/></p><ul><li></li></ul><h3><a name='header-n4234' class='md-header-anchor '></a>Paper discussion: Ziegler et al. 2017</h3><hr /><p> <div id='id-section24'/><br/></p><h3><a name='header-n4239' class='md-header-anchor '></a>Page 24: 2017-04-12. Week 12, Day 22, KTyler info update; microbiome; paper discussion Bolnick et al. 2014</h3><p>KTyler info update:</p><p>Microbiome can be beneficial  <br/></p><ul><li>digestive  <br/></li><li>immune <br/></li><li>vitamins <br/></li><li>xenobiotics<br/></li><li>pathogens  <br/></li></ul><p>Heritability: proportion of variance in a host-trait   <br/></p><ul><li>measured across population  <br/></li><li>explained by genetics <br/></li></ul><p>Approaches: <br/></p><ol start='' ><li><p>Can be direct: twin pairs- monozygotic = 100% genes;  <br/></p></li><li><p>GWAS - Humans; </p><ul><li>Microbiome is the trait</li></ul></li><li><p>Cross study: QTL&#39;s   <br/></p><ul><li>MICE! They found immune functions</li><li>plants- species richness and abundance of ind taxa</li><li>flies - immunity ,barrier defense</li></ul></li></ol><p>Limitations: </p><ul><li>MB expensive  <br/></li><li>small sample sizes; lack of power   <br/></li><li>Not all microbial taxa are sequenced, missing taxa  <br/></li><li>Meta analyses</li></ul><p>Future: </p><hr /><p> <div id='id-section25'/><br/></p><h3><a name='header-n4316' class='md-header-anchor '></a>Page 25: 201704-19. Week 13, Day 24, microbiome cont&#39;d.</h3><p>info update Aburnham:  <br/></p><p>Outline:</p><p><strong>Rarefying vs rarefaction</strong></p><p>rarefying = normalization of the data, subsampling depending on lowest N  <br/></p><p>rarefaction = not normalization; estimate species richness (extrapolation process)   <br/></p><p>Is what you find depend on method(sampling) or is it real?   <br/></p><p><strong>Methods:</strong>         <br/></p><ul><li><p>Past-  <br/></p><ul><li>proportions; high rate of false positives</li><li>heteroskedasticity  <br/></li><li>reduced power</li></ul></li><li><p>Rarefying (reads = sampling effort)  <br/></p><ol start='' ><li>Find smallest N (Nlmin)  reads</li><li>discard libraries fewerr than Nlmin</li><li>Subsample the reads from larger N&#39;s to the smallest N. <br/></li></ol></li></ul><p>What does this do? Normalizes the data. Removing the large descrepancy in read numbers.  <br/></p><p>Problems! <br/></p><ul><li>high rate of false positives <br/></li><li>requires ommission of actual data   <br/></li><li>reduces power <br/></li><li>adding another level of uncertainty  <br/></li></ul><p><strong>USE A MIXTURE MODEL</strong>  <br/></p><p>What does this do? </p><p></p><p></p><hr /></div>
</body>
</html>