

# 2017 General Notebook

### Author: Andrew D. Nguyen, [Evolutionary Physiologist](https://adnguyen.github.io)      
### Affiliation: Biology Department, University of Vermont      
### Contact: anbe642@gmail.com     

### Date started: 2017-01-01    
### Date end (last modified): ongoing   

<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.    

### General Lab protocols found [here for heat shocks and RNA related experiments](https://github.com/adnguyen/Notebooks_and_Protocols/blob/master/2016_ANBE_protocols.md) and [here for protein related experiments](https://github.com/adnguyen/2016_Protein_stability_evolution/blob/master/Documents/Protocols/Protocols.md).     


**Introduction:**    
Notebook for 2017 new year. It'll log the rest of my dissertation and potentially new post doc ideas and/or projects

## List of projects and description   
* Hsp rxn norm: Understanding how the local thermal environment shapes thermal tolerance and stress response (using Hsps as a proxy for stress) in forest ants of the genus *Aphaenogaster*. CTmax and rxn norm of Hsp expression measured across forest ants from Fl to Maine.   
* Range limits: Identifying the factors/forces that set range limits in common forest ants (*Aphaenogaster picea*). Modelling + measured their cold physiology in forest ants of Maine and Vt.  
* Multiple stressors: Understanding how progressive desiccation and starvation impacts thermal tolerances in *A. picea*. We pre-treated with desiccation and starvation and then measured KO-time. We also measured the stress response.        
* Thermal niche paper: Collaborative paper understanding how the environment shapes the ability to withstand cold and hot temperatures. In field and in a common garden, we measured upper and lower thermal limits of ants from GA-Maine (2 species).    
* Stress in nature: Are ants stressed under experimental warming that projects climate change? Ants were collected from warming chambers (0-5 C increase from ambient) and we measured their stress response.    
* ​

### Table of Contents (Layout follows Page number: Date. Title of entry)    
* [Page 1: 2017-01-01](#id-section1). SICB meeting
* [Page 2: 2017-01-03](#id-section2). Yearly goals
* [Page 3: 2017-01-10](#id-section3). Status of projects
* [Page 4: 2017-01-17](#id-section4). Displaying hierarchical structure of a repo or set of directories in the terminal with tree command
* [Page 5: 2017-01-17](#id-section5). Project idea: The impact of temperature variation on ant colonly level performance.
* [Page 6: 2017-01-17](#id-section6). Tutorial for making fancy documents(Cover letters) in latex and rendering into pdf!
* [Page 7: 2017-01-19](#id-section7). Making sense of reviewer comment; multiple stressors ms
* [Page 8: 2017-01-19](#id-section8). Notes from SICB
* [Page 9: 2017-01-20](#id-section9). Status of projects
* [Page 10: 2017-01-26](#id-section10). **Hoffmann et al. 2013**, Functional Ecology Paper notes ; Constraints on upper thermal limits
* [Page 11: 2017-01-26](#id-section11). Notes on teaching statements
* [Page 12: 2017-01-26.](#id-section12). SHC lab meeting, organizational
* [Page 13: 2017-01-26](#id-section13). **Bubliy & Loeschke 2005** paper notes; comparisons of selection on stress resistance
* [Page 14: 2017-01-26.](#id-section14). **Mitchell & Hoffmann 2010** paper notes, Functional Ecology; Thermal ramping influences evolutionary potential
* [Page 15: 2017-01-27.](#id-section15) Embedding a calendar into your webpage
* [Page 16: 2017-01-31](#id-section16). **Sørensen et al. 2003**; Ecology letters, The Evolutionary and ecological role of heat shock proteins; paper notes
* [Page 17: 2017-01-31](#id-section17). List of Adaptive Variation in Hsps lit table. 
* [Page 18: 2017-01-31](#id-section18). Statuf of projects: writing
* [Page 19: 2017-01-31](#id-section19). SHC lab meeting reading: Ranga et al. 2017; Evol Ecol: Sibling Drosophila species (Drosophila leontia and Drosophila kikkawai) show divergence for thermotolerance along a latitudinal gradient
* [Page 20: 2017-02-02](#id-section20). SHC lab meeting: reading Ranga et al. 2017; lab discussion
* [Page 21: 2017-02-03](#id-section21).**Todgham et al. 2017**; JEB, The effect of temperature adaptation on the ubiquitin-proteasome pathway in notothenioid fishes
* [Page 22: 2017-02-06](#id-section22). **Tomanek & Somero 2002**; JEB, Paper notes
* [Page 23: 2017-02-06](#id-section23). **Tomanek 2010;** JEB, paper notes
* [Page 24: 2017-02-06](#id-section24). **Parcel & Lindquist 1994;** cold spring harbor , paper notes
* [Page 25: 2017-02-07](#id-section25). How far down do Aphaenogaster colonies go in the winter?
* [Page 26: 2017-02-07](#id-section26).  A definition of a trade-off
* [Page 27: 2017-02-09](#id-section27). SHC lab meeting: Practice talk for biolunch and dissertation defense
* [Page 28: 2017-02-14](#id-section28). Status of projects + PHD Progress
* [Page 29: 2017-02-15.](#id-section29) **Aphaenogaster, Hsp rxn norm paper stats revisited**: regression without open habitat;Sampling info ; PCA effective number of dimensions calculation for Hsps
* [Page 30: 2017-02-20](#id-section30). prep for committee meeting 2017-02-24, 2:00PM
* [Page 31: 2017-02-21](#id-section31). Measuring evolutionary rates in darwins and haldanes  
* [Page 32: 2017-02-22](#id-section32). Resistance vs Tolerance strategies to stress; paper notes-**Núñez-Farfán et al. 2007**; The Evolution of Resistance and Tolerance to Herbivores; Annu. Rev. Ecol. Evol. Syst.
* [Page 33: 2017-02-23](#id-section33).  Multiple regression models testing effect of Hsp parameters on Ctmax
* [Page 34: 2017-02-23](#id-section34). SHC lab meeting: Going over hxp rxn norm paper. 
* [Page 35: 2017-02-23](#id-section35). What are Evolutionary Innovations? 
* [Page 36: 2017-02-24](#id-section36). Last biolunch and committee meeting notes
* [Page 37: 2017-02-27](#id-section37). Mappies in R for Zamira
* [Page 38: 2017-02-28](#id-section38). Questions to think about from SHC
* [Page 39: 2017-02-28](#id-section39). More analyses for hsp rxn norm paper
* [Page 40: 2017-03-04](#id-section40). Visit at UF; feedback on dissertation talk
* [Page 41: 2017-03-07](#id-section41). To do.
* [Page 42: 2017-03-09](#id-section42). General reading notes
* [Page 43: 2017-03-09](#id-section43). Dissertation format check with Sean M.
* [Page 44: 2017-03-10](#id-section44). General phylogenetics workflow
* [Page 45: 2017-03-10](#id-section45). SHC lab schedule 
* [Page 46: 2017-03-13](#id-section46). Meeting with SHC for dissertation timeline
* [Page 47: 2017-03-13](#id-section47).  SHC analysis suggestion hsp rxn norm paper
* [Page 48: 2017-03-14](#id-section48). Reading notes- Armstrong et al. 2011; hsp70 and neurophysiology
* [Page 49: 2017-03-16](#id-section49). Writing notes
* [Page 50: 2017-03-17](#id-section50). Writing notes
* [Page 51: 2017-03-20](#id-section51). Writing notes
* [Page 52: 2017-03-21](#id-section52). Writing notes
* [Page 53: 2017-03-22](#id-section53). Prepping for oral defense, reading notes
* [Page 54: 2017-03-23](#id-section54). To do list
* [Page 55: 2017-3-24](#id-section55). multiple stressors reviewer comments
* [Page 56: 2017-03-28](#id-section56). Definition of phenotypic plasticity
* [Page 57: 2017-03-28](#id-section57). Meeting with new lab tech, KB
* [Page 58: 2017-03-31](#id-section58). Making thesis fixes, notes
* [Page 59: 2017-03-31](#id-section59). To do list
* [Page 60: 2017-04-07](#id-section60). Project updates    
* [Page 61: 2017-04-10](#id-section61). To do and project updates
* [Page 62: 2017-04-11](#id-section62). Hsp rxn norm project: Species efficiency curves  
* [Page 63: 2017-04-13](#id-section63). Hsp rxn norm proj: amplicon quant
* [Page 64: 2017-04-14](#id-section64). Hsp rxn norm project: Species efficiency curves cont'd    
* [Page 65: 2017-04-14](#id-section65). Proteome stability project: Meeting with Wai
* [Page 66: 2017-04-19](#id-section66). Proteome stability project: Data analysis
* [Page 67: 2017-04-20](#id-section67).  Proteome stability project: Re-analysis with normalizing with gapdh     
* [Page 68: 2017-04-21](#id-section68). Proteome Stability project: More data analysis; fitting curves separately to each species    
* [Page 69: 2017-04-24](#id-section69). Post doc project ideas: Diapause timing and circadian clocks
* [Page 70: 2017-04-25](#id-section70). Proteome stability, individual colony variation and other stats   
* [Page 71: 2017-04-26](#id-section71). Climate cascade meeting: Project updates; to do list
* [Page 72: 2017-04-28](#id-section72). Stressed in nature project: data analysis
* [Page 73: 2017-04-30](#id-section73). Stressed in nature project: Re-analysis
* [Page 74: 2017-05-01](#id-section74). Stressed in nature project: re-analysis
* [Page 75: 2017-05-03](#id-section75). Stressed in nature project: Analyses...again    
* [Page 76: 2017-05-04](#id-section76). Stressed in nature project: Analyses, including technical replicates    
* [Page 77: 2017-05-05](#id-section77). Stressed in nature project: Analyses with recoded data and splitting by site    
* [Page 78: 2017-05-08](#id-section78). quantifying gene stability with geNorm      
* [Page 79:](#id-section79).
* [Page 80:](#id-section80).
* [Page 81:](#id-section81).
* [Page 82:](#id-section82).
* [Page 83:](#id-section83).
* [Page 84:](#id-section84).
* [Page 85:](#id-section85).
* [Page 86:](#id-section86).
* [Page 87:](#id-section87).
* [Page 88:](#id-section88).
* [Page 89:](#id-section89).
* [Page 90:](#id-section90).
* [Page 91:](#id-section91).
* [Page 92:](#id-section92).
* [Page 93:](#id-section93).
* [Page 94:](#id-section94).
* [Page 95:](#id-section95).
* [Page 96:](#id-section96).
* [Page 97:](#id-section97).
* [Page 98:](#id-section98).
* [Page 99:](#id-section99).
* [Page 100:](#id-section100).
* [Page 101:](#id-section101).
* [Page 102:](#id-section102).
* [Page 103:](#id-section103).
* [Page 104:](#id-section104).
* [Page 105:](#id-section105).
* [Page 106:](#id-section106).
* [Page 107:](#id-section107).
* [Page 108:](#id-section108).
* [Page 109:](#id-section109).
* [Page 110:](#id-section110).
* [Page 111:](#id-section111).
* [Page 112:](#id-section112).
* [Page 113:](#id-section113).
* [Page 114:](#id-section114).
* [Page 115:](#id-section115).
* [Page 116:](#id-section116).
* [Page 117:](#id-section117).
* [Page 118:](#id-section118).
* [Page 119:](#id-section119).
* [Page 120:](#id-section120).
* [Page 121:](#id-section121).
* [Page 122:](#id-section122).
* [Page 123:](#id-section123).
* [Page 124:](#id-section124).
* [Page 125:](#id-section125).
* [Page 126:](#id-section126).
* [Page 127:](#id-section127).
* [Page 128:](#id-section128).
* [Page 129:](#id-section129).
* [Page 130:](#id-section130).
* [Page 131:](#id-section131).
* [Page 132:](#id-section132).
* [Page 133:](#id-section133).
* [Page 134:](#id-section134).
* [Page 135:](#id-section135).
* [Page 136:](#id-section136).
* [Page 137:](#id-section137).
* [Page 138:](#id-section138).
* [Page 139:](#id-section139).
* [Page 140:](#id-section140).
* [Page 141:](#id-section141).
* [Page 142:](#id-section142).
* [Page 143:](#id-section143).
* [Page 144:](#id-section144).
* [Page 145:](#id-section145).
* [Page 146:](#id-section146).
* [Page 147:](#id-section147).
* [Page 148:](#id-section148).
* [Page 149:](#id-section149).
* [Page 150:](#id-section150).
* [Page 151:](#id-section151).
* [Page 152:](#id-section152).
* [Page 153:](#id-section153).
* [Page 154:](#id-section154).
* [Page 155:](#id-section155).
* [Page 156:](#id-section156).
* [Page 157:](#id-section157).
* [Page 158:](#id-section158).
* [Page 159:](#id-section159).
* [Page 160:](#id-section160).
* [Page 161:](#id-section161).
* [Page 162:](#id-section162).
* [Page 163:](#id-section163).
* [Page 164:](#id-section164).
* [Page 165:](#id-section165).
* [Page 166:](#id-section166).
* [Page 167:](#id-section167).
* [Page 168:](#id-section168).
* [Page 169:](#id-section169).
* [Page 170:](#id-section170).
* [Page 171:](#id-section171).
* [Page 172:](#id-section172).
* [Page 173:](#id-section173).
* [Page 174:](#id-section174).
* [Page 175:](#id-section175).
* [Page 176:](#id-section176).
* [Page 177:](#id-section177).
* [Page 178:](#id-section178).
* [Page 179:](#id-section179).
* [Page 180:](#id-section180).
* [Page 181:](#id-section181).
* [Page 182:](#id-section182).
* [Page 183:](#id-section183).
* [Page 184:](#id-section184).
* [Page 185:](#id-section185).
* [Page 186:](#id-section186).
* [Page 187:](#id-section187).
* [Page 188:](#id-section188).
* [Page 189:](#id-section189).
* [Page 190:](#id-section190).
* [Page 191:](#id-section191).
* [Page 192:](#id-section192).
* [Page 193:](#id-section193).
* [Page 194:](#id-section194).
* [Page 195:](#id-section195).
* [Page 196:](#id-section196).
* [Page 197:](#id-section197).
* [Page 198:](#id-section198).
* [Page 199:](#id-section199).
* [Page 200:](#id-section200).


------

 <div id='id-section1'/> 

### Page 1: 2017-01-01. Society for Integrative and Comparative Physiology (SICB) meeting in New Orleans       


### Table of talks I'd like to attend       


| Date            | Time        | Room                   | Speaker                          | Title                                    |
| :-------------- | :---------- | :--------------------- | :------------------------------- | :--------------------------------------- |
| Wednesday, Jan4 | 19:30       | Hilton Ballroom        | Swalla                           | Follow the Yellow Brick Road: An Odyssey from Myoplasm to Marine Biology to Genomics |
| Thursday, Jan5  | 8:00        | 217                    | Kelly                            | Protein coding and regulatory variation contribute to heat adaptation in the copepod Tigriopus californicus. |
| Thursday, Jan5  | 8:15        | 217                    | Logan                            | The evolutionary potential of a global insect invader in the face of rapid environmental change |
| Thursday, Jan5  | 8:30        | 217                    | Campbell                         | Urban heat islands and temperature-mediated physiological shifts between populations of the Puerto Rican crested anole |
| Thursday, Jan5  | 8:30        | 221                    | Charbonneau                      | Who Are the ‘Lazy’ Ants? Inter-worker Variation Gives Insight into Potential Functions of Inactivity |
| Thursday, Jan5  | 9:00        | 217                    | Ferris                           | The genomics of rapid adaptation to climatic extremes in house mice across the Americas |
| Thursday, Jan5  | 9:15        | 214                    | Geyman                           | Temperature Effects on Parasite Larval Size Over Time and Across Multiple Life Stages |
| Thursday, Jan5  | 9:30        | 214                    | Genovese                         | Plasticity in thermal tolerance of early life history stages of marine invertebrate larvae |
| Thursday, Jan5  | 9:30        | 217                    | Jangjoo                          | Gene expression associated with dispersal ability under different temperature conditions in the alpine butterfly, Parnassius smintheus |
| Thursday, Jan5  | 11:45       | 211-213                | Gough                            | Physical Properties and Anisotropy in the Central Tissue Layer of Cetacean Tail Flukes |
| Thursday, Jan5  | 11:45       | 214                    | Debiasse                         | Testing the effect of ocean acidification on a sponge-coral species interaction |
| Thursday, Jan5  | 13:00       | 215-216                | Flynn                            | Response of Amphibian Gut Microbiome to Coal Combustion Waste |
| Thursday, Jan5  | 13:30       | 224                    | Tangwancharoen                   | Divergence in cis-regulatory elements and HSPB1 gene expression along a temperature cline in the copepod Tigriopus californicus |
| Thursday, Jan5  | 13:45       | 219                    | Kingsolver                       | Inconstancy is informative: Estimating performance curves in fluctuating environments |
| Thursday, Jan5  | 14:00       | 219                    | Howey                            | Effect of Temperature on Snake Locomotion and the Interpretation of Thermal Performance Curves |
| Thursday, Jan5  | 15:15       | 214                    | Tielens                          | Geological age and host polymorphism affect functional diversity and community composition in plant-insect interactions across a space-for-time chronosequence on the Hawaiian Islands. |
| Thursday, Jan5  | 19:00       | 208                    | Sheriff                          | Integrating physiology, behavior, and ecology to understand the mechanisms that regulate and limit animal populations |
| Thursday, Jan5  | 15:30-17:30 | Exhibit Hall (posters) | Brueggemann                      | The effect of cholesterol and _-tocopherol on cold tolerance, post-cold performance, and rapid cold hardening |
| Thursday, Jan5  | 15:30-17:30 | Exhibit Hall (posters) | Chan                             | Taking the heat: High thermal tolerance of larval and adult mangrove snails |
| Friday, Jan6    | 7:50        | 206                    | Ragland; Williams                | Introduction to Evolutionary Impacts of Seasonality symposium |
| Friday, Jan6    | 8:15        | 211-213                | Gilbert                          | Natural Selection on Thermal Preference and Performance over a Rapid Timescale |
| Friday, Jan6    | 8:30        | 215-216                | Nourabadi                        | Fitness consequences of pH adaptation in an experimentally evolved beneficial symbiosis |
| Friday, Jan6    | 9:00        | 210                    | Obrien                           | The metabolic costs of animal weapons    |
| Friday, Jan6    | 9:30        | 206                    | Buckley; Kingsolver              | Insect Development, Thermal Plasticity and Fitness Implications in Changing, Seasonal Environments |
| Friday, Jan6    | 9:30        | 211-213                | Riddell                          | Potential responses to climate change are improved by physiological acclimation of water loss |
| Friday, Jan6    | 10:30       | 221                    | Clay                             | Transcriptomics of salamander tail tips reveal potential biomarkers of stress. |
| Friday, Jan6    | 11:00       | 221                    | Holden                           | Preparing for winter dormancy: Early-life experience affects condition, metabolism, and hormonal response to cold temperatures in the checkered garter snake, Thamnophis marcianus |
| Friday, Jan6    | 11:30       | 215-216                | Tanner; stillmann                | Locally adapted Phyllaplysia taylori populations in Central California show higher thermal plasticity potential |
| Friday, Jan6    | 11:45       | 215-216                | Oyen                             | Common garden experiments reveal local adaptation in critical thermal limits of bumblebees (Apidae, Bombus) over short geographic distances |
| Friday, Jan6    | 13:45       | 214                    | Norin                            | Plasticity, performance, and pace of life: individual differences in physiological and behavioural flexibility towards daily changes in temperature and oxygen availability |
| Friday, Jan6    | 14:00       | 214                    | Smith                            | Assessing the protein and metabolic costs of a trade-off between reproduction and immunity |
| Friday, Jan6    | 14:00       | 220                    | Crickenberger                    | Do temperature and competition interact to set a range limit? |
| Friday, Jan6    | 14:15       | 219                    | Kenny                            | How temperature influences the viscosity of hornworm hemolymph |
| Friday, Jan6    | 14:15       | 223                    | Refsnider                        | Plasticity in Behavioral Thermoregulation by Lizards on an Elevational Gradient: A Reciprocal Transplant Experiment |
| Friday, Jan6    | 14:45       | 214                    | Jaumann                          | Nutritional Stress Decreases Fecundity and Choosiness in a Butterfly |
| Friday, Jan6    | 15:00       | 206                    | Williams; Ragland                | Evolutionary impacts of seasonality: synthesis and directions forward |
| Friday, Jan6    | 15:00       | 214                    | Triedel; Williams                | The effect of diet nutrient composition on development and life history traits of a wing polymorphic cricket, Gryllus lineaticeps |
| Friday, Jan6    | 19:30       | 215-216                | Sosik                            | Life in the Plankton, Stories from automated submersible microscopy and flow cytometry |
| Friday, Jan6    | 15:30-17:30 | Exhibit Hall (posters) | Kornegay                         | Methylation and chromatin remodeling complex from sponges to humans |
| Saturday, Jan7  | 8:45        | 215-216                | Robberts;Rank;Stillman; Williams | The effects of snow cover on overwinter physiology of a montane insect |
| Saturday, Jan7  | 9:30        | 211-213                | Wada                             | A potential link between organismal adrenocortical responses and cellular heat shock responses |
| Saturday, Jan7  | 10:15       | 219                    | Stoehr                           | Temperature, Photoperiod and Nutrients Affect Phenotypically Plastic Wing Patterns in the Cabbage White Butterfly |
| Saturday, Jan7  | 10:45       | 215-216                | Allen                            | Diet-by-temperature interactions on a sexually selected trait and sexual dimorphism |
| Saturday, Jan7  | 10:45       | 220                    | Bryant                           | Uncoupling Proteins and Thermal Acclimation and Adaptation in Atlantic killifish, Fundulus heteroclitus |
| Saturday, Jan7  | 11:00       | 215-216                | Mikucki                          | Seasonal Differences in Diapause Induction in a Vermont Population of Pieris rapae Butterflies |
| Saturday, Jan7  | 11:00       | 220                    | Novarro                          | Geographic patterns of thermal tolerance in a widespread lungless salamander |
| Saturday, Jan7  | 11:30       | 211-213                | Finger                           | The Effect of Heat Shock on Constitutive and Inducible Heat Shock Proteins and Corticosterone in the Zebra Finch |
| Saturday, Jan7  | 11:30       | 215-216                | Lockwood                         | Molecular targets of thermal stress during early development in Drosophila melanogaster |
| Saturday, Jan7  | 11:45       | 215-216                | Niedojadlo                       | Daily Energy Expenditure, but Not Self-Maintenance Costs, Are Related to Hematological Variables in Response to Temperature Acclimation |
| Saturday, Jan7  | 13:30       | 215-216                | Lisovski                         | Biologically Significant Dimensions of Seasonality |
| Saturday, Jan7  | 13:45       | 217                    | Lozier                           | Population Genomics of Color Pattern Variation in a Widespread North American Bumble Bee |
| Saturday, Jan7  | 14:00       | 215-216                | Betini                           | Fitness Trade-off Between Seasons Causes Multigenerational Cycles in Phenotype and Population Size |
| Saturday, Jan7  | 14:00       | 220                    | Mitchell                         | Do covariances between maternal behavior and embryonic physiology drive sex-ratio evolution under environmental sex determination? |
| Saturday, Jan7  | 14:15       | 225-226                | Ceja                             | Ecologically modeling the distribution of an intertidal crab concerning global change |
| Saturday, Jan7  | 15:30-17:30 | Exhibit Hall (posters) | Pigg; Williams                   | Effects of ambient temperature on the organization of lipids of the avian stratum corneum |
| Sunday, Jan8    | 8:00        | 220                    | McGee                            | Machine learning predicts cichlid feeding kinematics from craniofacial morphology |
| Sunday, Jan8    | 8:30        | 217                    | Newman                           | The Influence of the Early-life Environment on Stress Physiology and Fitness in the Wild |
| Sunday, Jan8    | 8:30        | 224                    | Irvine                           | Proteomic Changes Due to Elevated Temperature in Ascidian Ovaries |
| Sunday, Jan8    | 8:45        | 214                    | Matoo; Montooth                  | Role of Genetic Variation on the Ontogeny of Metabolism during Development. |
| Sunday, Jan8    | 8:45        | 224                    | Hurley                           | The Heat is On: Decrease in Avian Sperm Functionality at High Ambient Temperatures |
| Sunday, Jan8    | 9:15        | 215-216                | Soda; Slice                      | Vector autoregressive-moving average models as tools to visualize differences in shape trajectories |
| Sunday, Jan8    | 9:15        | 218                    | Diamond                          | Rapid evolution of ant thermal tolerance within an urban heat island |
| Sunday, Jan8    | 9:30        | 214                    | Neel; Mcbrayer                   | Thermal dependence of sprint performance and critical thermal limits in ecologically distinct populations of a small ectotherm |
| Sunday, Jan8    | 9:30        | 217                    | Graham                           | Evolutionary history matters: Maternal hormonal response to a natural stressor and effects on offspring growth and behavior |
| Sunday, Jan8    | 9:30        | 218                    | Hall; Warner                     | Thermal Spikes Caused by the Urban Heat Island Effect Result in Differential Egg Survival of a Non-native Lizard ( Anolis cristatellus ) |
| Sunday, Jan8    | 10:00       | 221                    | Dormio                           | Staying Active for Life: Investigating the Covariance Between Behavioral and Physiological Trade-offs in Treefrogs |
| Sunday, Jan8    | 10:15       | 217                    | Vitousek                         | Do Brief, Acute Stressors Have Lasting Effects on Phenotype? |
| Sunday, Jan8    | 10:30       | 214                    | Shah;Ghalambor                   | Does Climate Variability Predict Thermal Tolerance? A Comparison of Thermal Breadths in Aquatic Insects Across Elevation & Latitude |
| Sunday, Jan8    | 10:45       | 218                    | Balaban                          | Elastic energy storage and thermal performance in fence lizards |
| Sunday, Jan8    | 10:45       | 224                    | Mccue                            | Repeated exposure to food limitation earlier in life enables rats to spare lipid stores during prolonged starvation |
| Sunday, Jan8    | 11:00       | 224                    | Mcternan                         | Resting metabolism comparisons among populations of a subspecies of lizard differing in climate and vegetation types |
| Sunday, Jan8    | 11:30       | 217                    | Senner                           | The Stress Response of Peromyscus Mice to Experimental High Elevation Conditions |
| Sunday, Jan8    | 11:45       | 214                    | Nguyen                           | Constraints on cold tolerance and hardening ability limit the distribution of forest ants at its northern range boundary. |
| Sunday, Jan8    | 11:45       | 222                    | Crall                            | A Neonicotinoid Pesticide Disrupts Nest Behavior and Social Interactions in Bumblebee Colonies |
| Sunday, Jan8    | 13:30       | 210                    | Harrison                         | Hypometric scaling of metabolic rate arises from size-dependent natural selection on ATP demand |
| Sunday, Jan8    | 13:45       | 214                    | Clark; Williams                  | A Genetic Polymorphism for a Hormonal Circadian Rhythm is Associated With a Shift in Metabolic Fuel Use in Flight-Capable but not Flightless Crickets |
| Sunday, Jan8    | 13:45       | 210                    | Salin                            | How Does Mitochondrial Functioning Constrain Energy Efficiency? |
| Sunday, Jan8    | 14:00       | 222                    | Cirino                           | Effects of male quality and territory quality on female preference of varying condition |
| Sunday, Jan8    | 14:15       | 214                    | Braciszewsi                      | Relatedness and differential disease resistance in eastern Pacific Haliotids |
| Sunday, Jan8    | 14:15       | 224                    | Boothby                          | How do Tardigades Survive Extremes? Disordered Proteins as Mediators of Tardigrade Stress Tolerance |
| Sunday, Jan8    | 14:30       | 221                    | Miles                            | Desert tortoises race against climate change: past, present and future |
| Sunday, Jan8    | 14:45       | 220                    | Schoenle                         | Why Does Malaria Infection Reduce Fitness in Wild Birds?: A Test of Physiological Mechanisms |
| Sunday, Jan8    | 15:00       | 222                    | Leary                            | Acute Stress is a Target of Intra- and Intersexual Selection in the Green Treefrog, Hyla cinerea: Implications for Fitness, Honest Signals, and the Evolution of Endocrine-based Acoustic Armaments |

------

 <div id='id-section2'/> 

### Page 2: 2017-01-03. Yearly Goals  


1. Submit and Publish 3 manuscripts: range limits, Hsp rxn norm, and multiple stressors (also thermal niche paper).    
2. Get a post doc. This'll probably involve learning a new study system. And also sending out tons of applications.      
3. Learn and build a shiny app.    
4. Learn and become more proficient in statistics (Machine learning?, Baysian, predictive modelling, mixed effects modelling, eigentensor analyses).     
   * Quantitative genetics: more statistical genetics   

5. Form new collaborations? It'd be awesome to work with Brent Sinclair, Brent Lockwood, Joel Kingsolver, Caroline Williams, Jon Stillman, Alex Gunderson.     
6. Participate in a meta-analysis? Would be cool. 
7. Learn more physiology: Q10, metabolism related topics, lipid membranes, metabolites.   

------

 <div id='id-section3'/> 

### Page 3: 2017-01-10. Status of Projects      


1. Project updates:    
    * **Hsp gene expression + Ctmax project:**
        * rewrite results, intro and send out to NJG and SHC  (methods done)    
        * Submit to PNAS  
    * **Multiple stressors ms:**   
        * In SHC's hands, due 2017-02-14

    * **Range limits ms**: SHC lab gave verbal edit, still need to incorporate     
        * **Thermal niche ms:** Lacy to check intro, and add refs.    	
    * **Stressed in nature MS: Samples to rerun.**       
        * update: Curtis can no longer work+ write on project   
        * **There are 74 samples: 3 days of RNA isolation + cDNA synthesis. 4 gene targets ran in duplicates is 2 plates per gene = 8 plates total.  2 days for 8 plates.**           
    * **Proteome stability project:**    
        * **~130 proteins for rudis, ~250 proteins for pogos**(we got 500 proteins last time); labelling is ok
          * Rerun mass spec, but loading more proteins (Bethany)

2. **Thesis related**  [FORMS FOUND HERE](http://www.uvm.edu/~gradcoll/?Page=Forms.html)  
    * Formatting:    
      * Introduction (> 3 pages), manuscripts, then synthesis/conclusion (~3 pages) ; SHC and NJG agree
        * started outline   
    * [Deadlines](http://www.uvm.edu/~gradcoll/pdf/Timetable%20for%20defense%202016-17-2.pdf):    
      1. Intent to graduate: February 1st for May.   
      2. Send defense committe form to grad college---now   
      3. Graduate college format check March 4th
      4. Defense notice 3 weeks before defense  (oral defense by March 24th).   
      5. Final thesis April 7th.  


------

 <div id='id-section4'/> 

### Page 4: 2017-01-17. Displaying hierarchical structure of a repo or set of directories in the terminal with tree command  


1. In the terminal, [install the tree command](https://rschu.me/list-a-directory-with-tree-command-on-mac-os-x-3b2d4c4a4827#.wk0u34bpj)
   * You may have to install the [homebrew](http://brew.sh/)    
2. use the tree command in the terminal

```
2017_Ecological_Genomics andrewnguyen$ tree
.
├── 2017_Ecological_Genomics.Rproj
├── Online_notebook.md
├── README.md
├── RasterPCA_demo.Rmd
├── RasterPCA_demo.html
├── index.Rmd
├── index.html
└── index.pdf

0 directories, 8 files
```

------

 <div id='id-section5'/> 

### Page 5: 2017-01-17. Project idea: The impact of temperature variation on ant colonly level performance.       


**Background**   
Ant colonies experience temperature fluctuations throughout the day and season. In response to temperature variation, ants must be able to forage under cool and hot conditions. One way to achieve high performance in the face of temperature variation is for the colony to match individual level performance with shifting temperatures. For example, for colonies with diverse genetic structures, some genotypes may forage better under cool or hot conditions. Multiply mated queens (*Pogonomyrmex*) will produce offspring with more phenotypic variation in traits, compared to singly mated queens (*Aphaenogaster*).     

**Notes:** Need to actually verify that foraging efficiency differs throughout the day in a colony, aka need to know more natural history. Look up Deborah Gordon's work. How will you address the confounding issue of demographic effects? Meaning what if there is genetic clumping of sperm and it corresponds to age polyethism, leading to the covariance between age and genotype. Also, the age structure of the colony itself may covary with genotype.       

**Question**   

1. Is the amount of physiological variation in upper thermal limits reflected in the number of fathers?    
2. Do ant colonies utilize different genotypes to forage throughout the day and season?    

or

Do ants display seasonal adaptation in forager performance?   

**Hypotheses**  

1. The number of fathers increases the amount of phenotypic variatnce in upper thermal limits.    
2. There is genetic structuring in the foragers throughout the day and season.    

**Predictions**    

1. Additive genetic variance in upper thermal limits will be proportional to the number of fathers.     
2. Thermally tolerant genotypes perform during the warmest parts of the day and vice versa for more cool tolerant genotypes.    
3. The more thermally tolerant genotypes start off at low frequency early on in the season, but increase in frequency at peak summer, then decrease again.    

**Experimental Design**   

I should do a power analysis to see how many ants and colonies I'd need to sample. Sam Scarpino has an R package for this I think.   

1. Document the natural history of 10 pogo colonies, to tune for sampling   
2. If there is like 10 foraging bouts, or 10 time slices through the day, sample ~30 ants for each colony at each time slice.   
3. Phenotype each ant: measure morphology (head width, leg lengths, alitrunk length, etc), number of ovarioles?, hydrocarbons, desiccation resistance, measure upper thermal limit   
4. Pool ants at each time slice, isolate DNA, and then pool-seq (capture-seq, but chatting with April, it sounds like biased sampling because only the portion of the first exon and promoter are sequenced.)     

**Critical Results or Alternative outcomes**  

1. Positive relationship between parentage and phenotypic variance.    
2. Structuring of alleles throughout the day or season. Some start off low, increase, then decrease in frequence.  This would indicate seasonal adaptation of foragers!     

It is possible that we won't find anything at all. But maybe because we're pool-sequencing, we can pick up the microbiome? And maybe that is structured diurnally or throughout the season.    

### Another idea: Compare parasite load between ant colonies with high and low number of fathers.    


Quick and dirty: One of the benefits of a diverse genetic background in a colony is disease resistance. So, test this by pool-seq experiment and measuring the parasite loads for ~30 individuals, potentially across a whole phylogeny of ants that vary in number of daddys and number of queens. Parasite load from different taxanomic groups (fungus, nemotodes, bacteria, viruses) can be mapped onto the phylogeny!

------

 <div id='id-section6'/> 

### Page 6: 2017-01-17. Tutorial for making fancy documents(Cover letters) in latex and rendering into pdf!  


I took this [cool template](https://www.sharelatex.com/templates/cover-letters/awesome-cv-cover-letter) for making a cover letter, which was in ".tex" format. Then converted the tex file into pdf using [this tutorial](http://economistry.com/2013/01/installing-and-using-latex-for-mac/), which involve installing mactex, sublime text 2, and skim. 

------

 <div id='id-section7'/> 

### Page 7: 2017-01-19. Making sense of reviewer comment; multiple stressors ms     


**Is there any direct evidence that heat shock proteins prevent heat knockdown?**    

Calabria et al. 2012 find differences in basal hsp70 protein expression but not heat-induced hsp70 expression between 3 genotypes of fruit flies( they differ in chromosomal structure). These differences were reflected in the fast ramp (0.1 C/min ramp), but not slow ramp(0.6 C/min ramping). 

3 genotypes:

1. O(3+4)/O(3+4): higher hsp70 basal expression and thermal tolerance under fast ramp;       
2. O(3+4+8)/O(3+4+8)    
3. O(st)/O(st): lower hsp70 basal expression and thermal tolerance under fast ramp;   

### References:      


1. Calabria G, Dolgova O, Rego C, et al (2012) Hsp70 protein levels and thermotolerance in Drosophila subobscura: a reassessment of the thermal co-adaptation hypothesis. Journal of Evolutionary Biology 25:691–700. doi: 10.1111/j.1420-9101.2012.02463.x


------

 <div id='id-section8'/> 

### Page 8: 2017-01-19. Notes from SICB      


**2017-01-05; Met with Joel Kingsolver**    

Met and talked about range limits project, selection gradient idea, and hsp rxn norm gxp project.      
* For my ancestral trait reconstruction of CTmax, I excluded the outgroup, suggests to include it      
* For multi-panel figure with parameters on x axis and ctmax on y axis; try partial regression      
* If I don't get into PNAS, try Proc B    
* For looking at shifts in trade-offs between multiple stressors look up Nancy Emory, central cali, she is a plant evo ecologist   
* Selection gradient proj idea; JK asks what has been done in well known systems? ie. Arapidopsis (sp?), check Andy Schidth   

**2017-01-06; met with Ray Huey**    

Showed hsp rxn norm paper stuff:     
* Are parameters correlated- he was worrieda bout this.    
* Try fitting splines and extracting parameters     
* Selection on graits; linear or quadratic? non-parametric?       

**2017-01-07 Met with Dan Hahn and Greg Ragland**      

Went over data and Dan liked how Iparsed out all the predictions for the shifts in reaction norms of hsp gxp expression.    
* Finger example highlights the differences between resistance and tolerance    
* Pressing two fingers together without moving = **resistance**       
* When 1 finger moves the other finger but can move back = **tolerance**     

------

 <div id='id-section9'/> 

### Page 9: 2017-01-20. Status of projects  


1. Project updates:    
    * **Hsp gene expression + Ctmax project:**
        * Have intro, discussion mostly complete by today, meet with NJG Monday (2017-01-23)
        * Submit to PNAS in May?   
    * **Multiple stressors ms:**   
        * In SHC's hands, due 2017-02-14
    * **Range limits ms**: SHC lab gave verbal edit, still need to incorporate     
        * **Thermal niche ms:** Lacy to check intro, and add refs.    	
    * **Stressed in nature MS: Samples to rerun.**       
        * **There are 74 samples: 3 days of RNA isolation + cDNA synthesis. 4 gene targets ran in duplicates is 2 plates per gene = 8 plates total.  2 days for 8 plates.**           
    * **Proteome stability project:**    
        * **~130 proteins for rudis, ~250 proteins for pogos**(we got 500 proteins last time); labelling is ok
          * Rerun mass spec, but loading more proteins (Bethany)

2. **Thesis related**   
    * **Defense talk: Have first version by Jan 31st**    
      * Required defense talk in biolunch: Feb 24 (Friday)     
    * Working title: Evolutionary Innovations of Ants to Thermally Stressful Environments     
    * Formatting:    
      * Introduction (> 3 pages), manuscripts, then synthesis/conclusion (~3 pages) ; SHC and NJG agree
        * started outline   
    * [Deadlines](http://www.uvm.edu/~gradcoll/pdf/Timetable%20for%20defense%202016-17-2.pdf):    
      1. Intent to graduate: February 1st for May.   
      2. Send defense committe form to grad college---**done**
      3. Graduate college format check March 4th
      4. Defense notice 3 weeks before defense  (oral defense by March 24th).   
      5. Final thesis April 7th.  


------

 <div id='id-section10'/> 

### Page 10: 2017-01-26. Hoffmann et al. 2013, Functional Ecology Paper notes      


Starting off intro with constraints on upper thermal limits. 



**What can constraint upper thermal limits?**

1. low plasticity (stillmann paper shows this in crabs)
2. low adaptive potential (heritability)   
   * low variation in upper compared to lower thermal limits
3. Phylogenetic constraints inferred from strong phylogenetic signal. 
   * high lambda in PGLS for insects, lizards + snakes  

Some definitions: 

* heritability: degree of phenotypic variation in a trait that is genetic 
* evolvability: extent to which the trait's mean can shift under selection, which depends on the amount of genetic variation and mean values.  

Paper argues that mid-latitude species are most prone to heat stress (presently and to the future). Why? And what is mid-latitude? 

  





------

 <div id='id-section11'/> 

### Page 11: 2017-01-26. Notes on teaching statements     


This science [article](http://www.sciencemag.org/careers/2006/04/writing-teaching-statement) has interesting advice. The teaching statement is usually a writing filter to weed out applicants who have not thought stuff through. Illuminates the character of the writer. It should be 1-2 pages. 



1. **Cater teaching statement to the institution**
2. **Demonstrate a real commitment to teaching** 
   * Low expectations on unique ways of teaching, but just show commitment (usually a sentence or two)
   * cite evidence, but be brief
3. Avoid presenting teaching as a 2nd priority
4. *Show us you care, but make it short and to the point* 
5. echo teachign interests in the cover letter, 1-2 sentences
6. Write about courses you'd like to teach!  For me, it'd be evolutionary physiology, evolution, computational biology?, ecological genomics 
   * cater this to the institution
   * display ideas, but don't be too ambitious
7. Teaching statement also indicates to the search committee how the applicant sees themselves in 
   * don't limit yourself to advanced courses, but also include beginners!
8. Display willingness to learn, pay attention, and change to reflect open mindedness and eagerness; shows you're not limited by what you are now
9. **Draw on your experieences as a student, a scholar, and human being** 
10. Don't promise too much

A good quote: 

`I used to assume that any student who did not get an "A" on an exam wasn't trying, until it occurred to me that no matter how hard I try in dance classes, I rarely excel. Becoming less critical of my students--while still maintaining high standards--has made me a better teacher.`



------

 <div id='id-section12'/> 

### Page 12: 2017-01-26. SHC lab meeting, organizational      


### Write down 3 concrete goals and timeline for the semester:   


1. Publish the multiple stressors manuscript
   * deadline is Feb 12 to submit revisions
2. Get a post doctoral position or real job by end of semester
   * NSF, should know by Feb 9th
   * Hibbitt fellow, no clue, but should be soon
   * prepping application for data scientist position
3. Have submittable version of hsp rxn norm paper to PNAS
   * Part of thesis, so should be accomplished by April 7th

Benchmark goals

1/3- Wait for SHC to give back feedback and take 2 days to submit it, prior to Feb12. Keep writing on PNAS paper, have introduction, discussion done. Results and methods are mostly complete but could be tweaked. Post doc stuff is passive

2/3- get notified from NSF, hibbit. For Hibbitt, if they're interested, they'll interview on site. For PNAS paper, shape up ms.

3/3- Constant revisions on PNAS paper

### Set schedule   


I have a biolunch talk (dissertation requirement talk) February 24th. Give practice talk February 10th. 

Tentative: Thursdays 4-5PM

------

 <div id='id-section13'/> 

### Page 13: 2017-01-26. Bubliy & Loeschke 2005 paper notes  


**Correlated response to selection for stress resistance.** 



Fruit fly paper doing a selection experiment on: 

1. cold (CS) in units of survival rate
   * chilled 5 C, recover 24 hours
2. heat (HS) in units of survival rate
   * hardened 30 min at 36C, recover 20 hours at 25 C, then heat shocked 38C 1 hour
3. desiccation (DS)
4. starvation (SS)
5. Heat knockdown time (KS) -in units of time (minutes)
   * KO at 40C
6. Lifespan (LS)

* And then they checked all of the traits from each selected line relative to controls. 
* 21 generations



1. Cold shock resistance (CS)
   * CS, HS, and DS all increased
2. Heat shock resistance (HS)
   * HS, KS, DS all responded
3. Heat knockdown resistance  (KS)
   * only KS responded

KS and SS increased developmental time. 



Upper and lower thermal limits can equally respond to selection after X generations in fruit flies. IN this experiment, there isn't necessarily a trade off betwween the two (except in cold shock resistance). 





------

 <div id='id-section14'/> 

### Page 14: 2017-01-26. Mitchell & Hoffmann 2010, Functional Ecology; Thermal ramping influences evolutionary potential  


In fruit flies, they measured upper thermal limits as knockdown time in a static and ramping experiment. They did this in  a quantitative genetic design, so they could estimate the narrow-sense heritability (partition out variation to the additive genetic component).  They also measured this for ~11 species.  (looks like a lot of work)



**Methods:**

Slow ramp: 0.06 C/min

Fast, static HS: 38C preset temp



Fast and slow show a postive relationship. 



Set up a model to partition out variation Va (additive genetic variance), Vp (phenotypic variance), and Ve (environmental variance) and presented models for their coefficient of variation.



**Results**: 

* Ramping had low heritability(not sig different than 0), but static had high heritability (sig different than 0)
* Ramping had more environmental variance, but do the data need to be standardized first, such that the mean is 0 and variance i 1? No, the units are the same for both traits
* Ramping had higher phenotypic variance



**THoughts**

This paper contrasts a little bit with van Heraawarden & Sgrò paper, where they focus one 1 species. But they find that ramping, fast HS, and hardening all significantly correlate. But there is an additive genetic axis where



------

 <div id='id-section15'/> 

### Page 15: 2017-01-27. Embedding calender into webpage  


[Followed this tutorial](http://denisecase.github.io/2015/07/10/add-calender-to-site/): 

1. Create a  calender in google (gmail)
2. Go to settings
3. click calendars
4. go down to where it says embed this calendar, it is in html code, so copy that
5. tutorial goes through yml, but I just used the html code to make a new webpage with my calender in it and created a link in the navigation bar

[Here it is](https://adnguyen.github.io/calendar.html), useful for me to access my calender wherever I am now!   



### 2017-01-31 Update    


Calendar doesn't show up on my iphone or ipad (safari), what is the point? Taking the calendar out. 



------

 <div id='id-section16'/> 

### Page 16: 2017-01-31. Sørensen et al. 2003; Ecology letters, The Evolutionary and ecological role of heat shock proteins  


Trying to get a handle on how this paper frames the role of Hsps with upper thermal limits. From the abstract, they mention it as a resistance mechanism. 



1. **Defining stress:**  a condition that disturbs the normal function of the biological system or a condition that decreases fitness (Hoffmann & Parsons 1991; Bijlsma & Loeschcke 1997)
   * I like part of the definition. I'd define stress **as a perturbation to a biological system that decreases fitness.** Or, a force acting on a biological system that decreases fitness. 
   * These forces can be *extrinsic* (environmental) and *intrinsic* (genetics, inbreeding, deleterious mutations, ageing).


2. **Hsp natural history** (cite Ritossa 1962 as first discovering heat shock response through chromosome puffs(which had Hsps on it))
3. **Protein quality control system (PQC)**: importance increases upon exposrue to environmental /genetic stresses
   * function is 2 fold: correct folding and to assist in degradation of denatured or aggregated proteins
   * not very informative
4. Stress as an ecoogical and evo force (Don't agree with the framing)
   * types of responses:
     * move, alter physiological state through hibernation or diapause
     * adapt or acclimate
     * fail and die
   * Adaptive change in Hsp over days (Nguyen et al. 1994; Ferguson et al. 1998) or over seasons (Fader et al. 1994; Hofmann & Somero 1995; Pyza et al. 1997; Miner et al. 2000) and natural populations(no citations, what?)
5. Effects of stress on rates of evolution
   * ​
6. **Costs of Hsp expression**
   * unclear whether reduced fitness is due to cost of acclimation or reared under poor conditions. Why would we tease these apart? 
   * Benefits of acclimation (stress /longevity) can be separated from costs
     * ex: Hercus et al. 2003: repeated mild stress lowered fertility and fecundity in the short term but not long term
     * One way to separte out costs and benefits is by altering acclimation treatments (Scott et al. 1997; Hoffmann & HewaKapuge 2000; Thomson et al. 2001; Wilson & Franklin 2002)
   * lowers fertility fecundity, energy, development, and survival 
   * Direct costs measured by Krebs & Feder 1998: hardened flies at different stages (1-3rd instar) in 4isofemale lines. Multiple heat exposures reduced survival but did not affect development time. Expression of Hsp70 was not correlated with survival, suggesting differences in expression cannot explain survival effects. 
   * said this already but high hsps lower growth and cell division (Feder et al. 1992; Krebs & Feder 1997, reduced reproduction (Krebs & Loeschcke 1994; Silbermann & Tatar 2000))
     * Silbermann & Tatar 2000 showed heat induced hsp expression reduced egg hatching of moms in fruit flies. 
     * Krebs & Loeschcke 1994 found reduced fecundity
7. **The role of Hsps for adaptation** 
   * Improved heat resistance of insects after hsp expression in **insects** ( (Gehring & Wehner 1995; Dahlgaard et al. 1998), **fish**(Basu et al. 2002) , plants (Sun et al. 2002), **mammals** (Ulmasov et al. 1993; Matz
     et al. 1996a). 
   * Some of the first data on the possible ecological relevance of Hsp expression comes from selection studies. In
     different species of Drosophila, it was shown that (contrary to predictions at that time) expression of Hsp70 was lower in lines frequently, or continuously exposed to severe stress (Bettencourt et al. 1999; Sørensen et al. 1999; Lansing et al 2000)
     * interpretation = costs of Hsp in pops exposed to stress outweighs benefits and adaptation is achieved through some other means
   * Density dependence might drive hsp expression 
   * Age will influence hsp expression
   * Life stages may differe in stressors exposed
     * Stage specific Hsp expression and resistance - no citation
     * not much support (Sørensen et al. 1999)
   * In adults, Hsp70 down regulated with decreasing heat stress resistance (Sørensen & Loeschcke 2002a)
8. **Conclusions and future prospectives**
   * ```However, new results show that Hsp expression is highly fine-tuned (not being only an on–off mechanism) and that Hsps are also continuously expressed after mild chronic stress exposure.```
   * Hsps as biomarkers of stress
   * ```However, local adaptation and selection for other kinds of adaptive mechanisms may disturb the evaluation of the results. The results by Sørensen et al. (1999, 2001) and Ko¨hler et al. (2000) showing that there is selection against Hsp expression in populations being exposed to chronic stress clearly demonstrates this problem.```


This is a pretty old study. Looking up papers that cite it that look cool/interesting:

1. T. Esperk, A. Kjærsgaard, R. J. Walters, D. Berger, W. U. Blanckenhorn, **Plastic and evolutionary responses to heat stress in a temperate dung fly: negative correlation between basal and induced heat tolerance?**, Journal of Evolutionary Biology, 2016, 29, 5
2. Ramadoss Dineshram, Kondethimmanahalli Chandramouli, Ginger Wai Kuen Ko, Huoming Zhang, Pei-Yuan Qian, Timothy Ravasi, Vengatesen Thiyagarajan, **Quantitative analysis of oyster larval proteome provides new insights into the effects of multiple climate change stressors**, Global Change Biology, 2016, 22, 6
3. L.Ye. Kozeko, D.B. Rakhmetov, **Variation in dynamics of the heat shock proteins HSP70 synthesis in Malva sylvestris and M. pulchella (Malvaceae) in connection with tolerance to high temperature, flooding and drought,** Ukrainian Botanical Journal, 2016, 73, 2, 194
   * constitutive and inducible; for thermal and drought tolerant plants
4. Delphine Cottin, Natacha Foucreau, Frédéric Hervant, Christophe Piscart, **Differential regulation of hsp70 genes in the freshwater key species Gammarus pulex (Crustacea, Amphipoda) exposed to thermal stress: effects of latitude and ontogeny**, Journal of Comparative Physiology B, 2015, 185, 3, 303
   * good ref, added to my table
5. Jennifer A. Jost, Emily N. Soltis, Marshall R. Moyer, Sarah S. Keshwani, **Linking zebra mussel growth and survival with two cellular stress indicators during chronic temperature stress**, Invertebrate Biology, 2015, 134, 3
6. D Porcelli, R K Butlin, K J Gaston, D Joly, R R Snook, **The environmental genomics of metazoan thermal adaptation,** Heredity, 2015, 114, 5, 502



### Overall thoughts:  


Not very informative in terms of what Hsps are actually doing and how they relate to upper thermal limits. What is the difference between protection and resistance? Tolerance? Some consistent wording would be nice. The paper focused too much on costs and benefits rather than adaptive variation. This is nice to know but what aspect of the thermal ecology do Hsps relate to? I would have hoped there would be a better discussion fo basal vs induced hsps. 

------


 <div id='id-section17'/> 

### Page 17: 2017-01-31. List of Adaptive Variation in Hsps lit table  



| Author           | Year | Journal                                  | Taxa                     | Life.Stage    | Hsp                | Expression.type | Comparison    | Basal.warm.vs.cold.adapted.    | Induction..warm.vs.cold.adapted.       | Trade.off.between.basal.induc | HS_assay     | notes                                    |
| :--------------- | ---: | :--------------------------------------- | :----------------------- | :------------ | :----------------- | :-------------- | :------------ | :----------------------------- | :------------------------------------- | :---------------------------- | :----------- | :--------------------------------------- |
| Dietz & Somero   | 1992 | PNAS                                     | fish                     | adults        | hsp90              | protein         | interspecific | NA                             | induce higher max                      | NA                            | static       | focused on acclimaiton                   |
| Gehring & Wehner | 1995 | PNAS                                     | ants                     | adults        | hsp70              | protein         | interspecific | NA                             | higher                                 | NA                            | static       |                                          |
| Feder            | 1996 | JEB                                      | Fruit flies              | larvae, pupae | hsp70              | protein         | intraspecific | hard to tell                   | higher with copy number                |                               | static       | varried copy number                      |
| Feder            | 1997 | Functional Ecology                       | Fruit flies              | larvae        | hsp70              | protein         | intraspecific |                                |                                        |                               | static       | compare field and lab Hsp 70induciton    |
| Krebs & Feder    | 1997 | Evolution                                | fruit flies              | larvae, adult | hsp70              | protein         | intraspecific |                                |                                        |                               |              | trade off between hsp expression and survival |
| Bosch            | 1998 | PNAS                                     | Hydra                    | Polyps        | hsp60              | protein         | intraspecific | NA                             | higher; low tolerant ones don't induce |                               | static       |                                          |
| Krebs            | 1999 | Cell Stress & Chaperones                 | Fruit flies              | larvae        | hsp70              | protein         | interspecific | NA                             | right shifted; lower max               |                               | static       |                                          |
| Dahlhoff & Rank  | 2000 | PNAS                                     | beetles                  | adults        | hsp70              | gene or protein | intraspecific | lower                          | induce at higher temps, similar max    |                               | static       | higher overall expression in warmer populations in the field |
| Tomanek & Somero | 2000 | Physiological and Biochemical Zoology    | marine snails            | adult         | hsp83              | protein         | interspecific | NA                             | higher max                             |                               | static       | complex results, some paralogues switch over in induction patterns |
| Tomanek & Somero | 2000 | Physiological and Biochemical Zoology    | marine snails            | adult         | hsp70              | protein         | interspecific | NA                             | higher max                             |                               | static       |                                          |
| Boshoff          | 2000 | CMLS                                     | humans                   | blood cells   |                    | protein         | blood cells   |                                |                                        | Yes; expontential decay       | static       |                                          |
| Zatsepina        | 2001 | JEB                                      | Fruit flies              | adult         | hsp70              | protein         | intraspecific | NA                             | lower                                  |                               | static       |                                          |
| Tomanek & Somero | 2002 | JEB                                      | marine snails            | adult         | hsp70              | protein         | interspecific | NA                             | higher                                 |                               | static       |                                          |
| Tomanek & Somero | 2002 | JEB                                      | marine snails            | adult         | hsp90              | protein         | interspecific | NA                             | higher                                 |                               | static       |                                          |
| Hofmann          | 2002 | Integ. & Comp. Biol.                     | urchins                  | tube feet     | hsp70              | protein         |               |                                | higher overall                         | nature                        |              |                                          |
| Dahlgaard        | 2002 | Functional Ecology                       | Fruit flies              | adults        | hsp70              | proteins        | intraspecific |                                |                                        |                               | static       | no correspondance with survival          |
| Garbuz           | 2003 | JEB                                      | Fruit flies              | adults        | hsp70              | protein         | interspecific | NA                             | higher                                 |                               | static       |                                          |
| Place            | 2005 | Polar biology                            | Antarctic fish           | adult         | hsp70              | gene            |               |                                |                                        |                               | static       | figures are almost uninterprettable      |
| Huang            | 2007 | Journal of Insect Physiology             | pea leafminer            | adults        | all                | gene            | intraspecific |                                |                                        |                               |              |                                          |
| Laayouni         | 2007 | BMC evo bio                              | Drosophila subobscura    | larvae        | hsp26              | gene            | intraspecific | NA                             |                                        |                               | static       | experimental evoluiton                   |
| Laayouni         | 2007 | BMC evo bio                              | Drosophila subobscura    | larvae        | hsp68              | gene            | intraspecific | NA                             |                                        |                               | static       | experimental evoluiton                   |
| Bettencourt      | 2008 | BMC Biology                              | Fruit flies              | Larvae        |                    | gene            | intraspecific |                                |                                        |                               | static       |                                          |
| Dong             | 2008 | Biol Bull                                | Limpets                  | adults        | hsp70              | protein         | interspecific | higher                         | higher max in some cases               | No                            | static       |                                          |
| Elekonich        | 2009 | Cell stress and chaperones               | honey bees               | adults        | hsp70              | gene            | tissues       | higher in thorax of older bees |                                        |                               | static       |                                          |
| Jensen           | 2009 | Journal of Experimental Zoology          | Fruit flies              | adult         | hsp70              | protein         | intraspecific | NA                             | no difference                          |                               | static       |                                          |
| Lockwood         | 2010 | JEB                                      | Mytilus                  | gill          | hsp70              | gene            | interspecific | no difference                  | no difference                          |                               | static       |                                          |
| Lockwood         | 2010 | JEB                                      | Mytilus                  | gill          | hsp24              | gene            | interspecific | NA                             | higher                                 |                               | static       |                                          |
| Mizrahi          | 2010 | Cell Stress & Chaperones                 | land snails              | adult         | hsp70              | protein         | interspecific | revisit                        |                                        |                               |              |                                          |
| Tomanek          | 2010 | JEB                                      | mussels                  | adult         | hsp70              | protein         | interspecific | NA                             | right shifted                          |                               | static       |                                          |
| Franssen         | 2011 | PNAS                                     | seagrass                 | grass         | 27 hsps            | gene            | intraspecific |                                | 2/27 differences                       |                               | static       |                                          |
| Carmel           | 2011 | Heredity                                 | Fruit flies              | adults        | hsp40              | protein         | intraspecific | higgher                        | higher                                 | No                            | static       |                                          |
| Carmel           | 2011 | Heredity                                 | Fruit flies              | adults        | sHsps              | protein         | intraspecific | no difference                  | no difference                          | No                            | static       |                                          |
| Calabria         | 2012 | J. Experimental Biology                  | Drosophila subobscura    | adults        | hsp70              | protein         | intraspecific | higher                         | no difference                          | Yes                           | static       |                                          |
| Madeira          | 2012 | Cell Stress & Chaperones                 | crab                     | haemolyph     | hsp70              | protein         | intraspecific | NA                             | higher in coastal                      |                               | static       |                                          |
| Graham           | 2012 | Journal of Heredity                      | Drosophila pseudoobscura | adults        |                    |                 |               |                                |                                        |                               |              |                                          |
| Bedulina         | 2013 | Molecular Ecology                        | Amphipods                | Adults        | hsp70              | gene            | interspecific | higher; hsp70                  | lower                                  | Yes                           | static; LT50 |                                          |
| Cottin           | 2014 | Journal of Comparative Physiology B      | Amphipods                | adult         | hsp70              | gene            | intraspecific | lower                          | lower                                  |                               | static       |                                          |
| Cottin           | 2014 | Journal of Comparative Physiology B      | Amphipods                | adult         | hsc70              | gene            | intraspecific | lower                          | no difference                          |                               | static       |                                          |
| Cottin           | 2014 | Journal of Comparative Physiology B      | Amphipods                | early         | hsp70              | gene            | intraspecific | no difference                  | no difference                          |                               | static       | Southern population(warmer) has lower thermal sensitivity |
| Cottin           | 2014 | Journal of Comparative Physiology B      | Amphipods                | early         | hsc70              | gene            | intraspecific | no difference                  | no difference                          |                               | static       |                                          |
| Franssen         | 2014 | Marine Genomics                          | seagrass                 | shoots        | 28                 | gene            | interspecific | higher                         | no difference; double check            |                               | static       |                                          |
| Madeira          | 2015 | Comparative Biochemistry and Physiology, Part A | shrimp                   | muscle        | hsp70              | protein         | interspecific | no difference                  | no difference                          |                               | static       | no diffeence in CTMAX                    |
| Nguyen           | 2016 | BMC Evo Bio                              | ants                     | adults        | hsp70 (hsc70-4 h1) | gene            | interspecific | higher                         | higher                                 | No                            | static       |                                          |
| Nguyen           | 2016 | BMC Evo Bio                              | ants                     | adults        | hsp70 (hsc70-4 h2) | gene            | interspecific | no difference                  | higher                                 | No                            | static       |                                          |
| Nguyen           | 2016 | BMC Evo Bio                              | ants                     | adults        | hsp83              | gene            | interspecific | no difference                  | higher                                 | No                            | static       |                                          |
| Nguyen           | 2016 | BMC Evo Bio                              | ants                     | adults        | hsp40              | gene            | interspecific | lower                          | higher                                 | No                            | static       |                                          |

------

 <div id='id-section18'/> 

### Page 18: 2017-01-31. Status of projects: writing  


1. Project updates:    
   - **Hsp gene expression + Ctmax project:**
     - Met with NJG yesterday, revised intro, reread, should be ready to review with NJG 2017-02-06; Monday
     - Meeting with NJG 2017-02-03, Friday 1PM to go over results, figure legends, figures. 
     - Submit to PNAS in May?   
   - **Multiple stressors ms:**   
     - ==In SHC's hands, due 2017-02-14==
   - **Range limits ms**: SHC lab gave verbal edit, still need to incorporate     
     - **Thermal niche ms:** Lacy to check intro, and add refs.    	
   - **Stressed in nature MS: Samples to rerun.**       
     - **There are 74 samples: 3 days of RNA isolation + cDNA synthesis. 4 gene targets ran in duplicates is 2 plates per gene = 8 plates total.  2 days for 8 plates.**           
   - **Proteome stability project:**    
     - **~130 proteins for rudis, ~250 proteins for pogos**(we got 500 proteins last time); labelling is ok
       - Rerun mass spec, but loading more proteins (Bethany)
2. **Thesis related**   
   - **Defense talk:** 
     - Giving practice talk ==February 9th== (Thursday in lab meeting, 4PM)
       - Need to write out script by  ==Feb 7th==  
     - Required defense talk in biolunch: ==Feb 24 (Friday)==     
   - Working title: Evolutionary Innovations of Ants to Thermally Stressful Environments     
   - Formatting:    
     - Introduction (> 3 pages), manuscripts, then synthesis/conclusion (~3 pages) ; SHC and NJG agree
       - started filling in introduction, eta? No clue, working on in the background   
   - Deadlines: 
     1. Intent to graduate: February 1st for May.   —**done**
     2. Send defense committe form to grad college—**done**
     3. ==Graduate college format check March 4th==
     4. Defense notice 3 weeks before defense  (oral defense by March 24th—*flexible*).   
     5. Final thesis April 7th.  


------

 <div id='id-section19'/> 

### Page 19: 2017-01-31. SHC lab meeting reading: Ranga et al. 2017; Evol Ecol: Sibling Drosophila species (Drosophila leontia and Drosophila kikkawai) show divergence for thermotolerance along a latitudinal gradient  


This study compares the different aspects of lower and upper thermal limits between two closely related species (*Drosophila*) in India across a wide climate gradient (8 -32 degrees north): 

1. *D. leontia*
2. *D. kikkawai*

==Measurements:==

1. **Upper thermal limits**
   * Heat knockdown time: 39 C treatment in water bath, measured time where they could not stand
   * Survival: 24 hour survey of flies 100% dead when treated at 39C over time (10-80min)
   * Hardening:
2. **Lower thermal limits**
   * Chill coma recovery time
     * 0C treatment for 8 hour then recover at room temp (22C); measured time they could stand upright
   * Cold survival 
     * 0 C treatment for 8 -96 hours, measured survival after 24hours
3. **Absolute hardening capacity (AHC)**
   * AHC = KT - C 
   * KT = thermotolerance after hardening
     * warm: (33, 35 or 37 C)
     * cool: (0, 2 or 4 C)
   * C = basal thermo-tolerance
4. **Relative hardening capacity**
   * RHC = (KT – C)/C
5. **Egg-to-adult viability**
   * Hot: eggs subected to 39 C 1 hour
   * Cold: eggs subjected to 0 C 12 hours
   * transferred to 22C 65% RH

## Results 

(Fig 2)

1. *D. leontia* ==lower plasticity==
   * Clinal variation in Upper thermal limits (KD time, heat survival)
     * No benefit from hardening
   * No clinal variation in lower thermal limits ( CCRT, cold survival)
     * No benefit from hardening
2. *D. kikkawai* —==higher plasticity==
   * Clinal variation in Upper thermal limits (KD time, heat survival)
     - Benefit from hardening
   * Clinal variation in lower thermal limits ( CCRT, cold survival)
     - Benefit from hardening

Table 2: They did anova for each trait, testing the effect of each population

Why not do an ANCOVA:

```R
aov(Thermal trait~ Latitude * Species)
```



**Fig 3: Egg-to-Adult viability**

1. *D. leontia ==lower plasticity==*
   * Southern pop had higher viability under controls, heat stressed, but not cold stress. 
   * **Notes** *makes sense because they have no relationship to cold* 
   * ​
2. *D. kikkawai ==higher plasticity==*
   * Southern pop had higher viability under heat stress but lower under controls and cold stress.

**Fig 4: Mortality**

1. *D. leontia* ==lower plasticity==
   * **HS:** No diff in mortality between hardening and controsl (consistent with fig 2)
   * **HS:** Southern had lower mortality  than north
   * **CS:** No diff between hardening and controls and pops
2. *D. kikkawai* ==higher plasticity==
   * **HS** higher mortality in control than hardened, makes sense; southern has lower mortality
   * **CS:**  Northern have lower mortality than south ; hardening has lower mortality

**Fig 5: AHC (Absolute hardening capacity) and RHC relative hardening capacity**: Same direction

1. *D. leontia* ==lower plasticity==
   * no sig hardening between northern and southern pops
2. *D. kikkawai* ==higher plasticity==
   * Upper thermal limits: positive plasticity
     * Northern pops have higher upper thermal limits
   * Lower thermal limits: 
     * CCRT: northern have faster recovery time
     * survival: northern have higher survival



**Fig 6: Hardening vs pretreatment temperatures**

1. Upper thermal limits:
   * *D. kikkawai* ==higher plasticity== had higher hardening than *D. leontia* ==lower plasticity== for both KD time and survival
2. Lower thermal limits: 
   * *D. kikkawai* ==higher plasticity==  has higher cold tolerance than *D. leontia* ==lower plasticity==

**Fig 7 looks the same, wtf**



**Figure 8** Focus on the climate variables each species experiences 

- North is more variable, south less variable (A)
- Tave and RH negatively related to latitude

1. *D. leontia* ==lower plasticity==
   * higher abundance at the warm end
2. *D. kikkawai* ==higher plasticity== 
   * higher abundance at the cold end 

## Thoughts 

* What is the demographic history? I ask because it can explain some of the results. 


* Optimal temp for D. leontia is the south and D. kikkawai is in the north. So D. leontia could be moving south to north and D. kikkawai could be moving north to south. 
* Painful read, could have analyzed the data differently. 




------

------




 <div id='id-section20'/> 

### Page 20: 2017-02-02. SHC lab meeting: reading Ranga et al. 2017; lab discussion  




Preliminary stuffY:

* SHC needs to quickly read multiple stressors ms



Katie miller wants to draw out stuff



```mermaid
graph TD
A{Heat tolerance}
B{Cold tolerance}
A --> C(33 C)
A --> D(35 C)
A --> E(37 C)
C --> F{Heat shock 37C}
D --> F{Heat shock 37C}
E --> F{Heat shock 37C}
F --> G(% Survival)
F --> H(Knock down time)
F --> J(Egg-adult viability)
```

Not exactly the experimental design….



Model construction for testing the effects of lines and populations on thermal traits: Nested ANOVA



```mermaid
graph TD
A[Thermal Trait] --> B(Populations)
B --> C(Isofemale line 1)
B --> D(Isofemale line 2)
B --> E(Isofemale line 3)
```









------

 <div id='id-section21'/> 

### Page 21: 2017-02-03. Todgham et al. 2017; JEB, The effect of temperature adaptation on the ubiquitin-proteasome pathway in notothenioid fishes  


Ref: Todgham AE, Crombie TA, Hofmann GE. 2016. The effect of temperature adaptation on the ubiquitin-proteasome pathway in notothenioid fishes. The Journal of Experimental Biology:jeb.145946.



**Background**

Ubiquitin tags proteins for degradation through the proteasome(protein chomper). In the face of heat stress, different species can adaptively modulate this pathway to cope with the deleterious effects of protein damage. 
 <u>Antarctic species live in -1.9C while new zealand species live in 10C .</u> 




**Objective:** Understand the thermal compensation of Ub-proteasome pathway in antarctic fish vs ones form warmer climates.

## Approach and workflow

**Compared ubiquitin ,proteasome activity and expression between 2 tissues (gill and liver) for 4 different fish species.** 

```mermaid

graph TD
A(Trematomus bemacchii) --> D{Antarctic fish}
B(Pagothenia borchgrevinki) --> D

R(Notothenia angustata) --> P{New Zealand}
G(Bovichtus variegatus) --> P

D --> M{Ub-proteasome pathway}
P --> M{Ub-proteasome pathway}
M --> K(Gills)
M --> I(Liver)


K --> W{UB tagging proteins}
K --> E{Proteasome 26s complex}

K --> U[Ub-congugated protein]
I --> U

K --> O[Proteasome activity]
I --> O[Proteasome activity]

W --> T(CHIP)
W --> UbE2D1
W --> Ubiquitin

E --> PSMA2
E --> PSMB7
E --> PSMC1


```

**Figure 1: UB conjugated proteins**

Antarctic and N. angustata have higher ubiquitin tagged proteins than B. variegatus for both liver and gill. 



**FIgure 2. Protasome activity** 

Antactic have higher proteasome activity at 0 and 10 C for both gill and liver. The magnitidue of differences were higher in the gills. This may be why they focused only on the gills for gene expression 



**Figure 3: Gill gxp for UB related genes**

  General pattern as Fig 1. where antarctic fish and N. angustata have higher expression of UB realted gene expression than B. variegatus. But N. angustata had higher expression than the antarctic fish. There could be different dynamics that would be captured with a reaction norm approach. 



**Figure 4: Gil gxn for proteasome complex related genes.** 

Similar result as figure 1. (exxcept top panel) . But here we see categorical differences between Antarctic and new zealand. Antarctic fish have higher expression. 



## Thoughts:

 It looks like cold tolerant species up-regulate their protein degradation pathway. It fits the model that if proteins are too damaged, they're clear those proteins. 



Cool method to test for proteasome activity: (Coux et al., 1996)

1.  chymotrypsin-like activities, which cleaves after large hydrophobic residues
2.  trypsin-like activities, which cleaves after basic residues
3.  peptidylglutamylpeptide hydrolyzing activities, which cleaves after acidic residues .

------

 <div id='id-section22'/> 

### Page 22: 2017-02-06. **Tomanek & Somero 2002**; JEB Paper notes  


Reference:   

Tomanek L, Somero GN. 2002. Interspecific- and acclimation-induced variation in levels of heat-shock proteins 70 (hsp70) and 90 (hsp90) and heat-shock transcription factor-1 (HSF1) in congeneric marine snails (genus Tegula): implications for regulation of hsp gene expression. J Exp Biol 205:677–685.



## Rational/Objectives/Questions

How do species adaptively vary in their stress response?



Species differ in vertical distribution

1. *Tegula brunnea* = subtidal 
2. *T. montereyi* = subtidal to low intertidal zones
3. *T. funerals* = low to mid intertidal zone (with wider latitudinal distribution) 
   * ==experiences the highest heat stress==



## Methods

1. Collected mid july and treated 13, 18 and 23 C for 30-34 days
2. Dissected tissues and isolated proteins
3. They measured protein levels of Hsp70/90/40, HSF with western blots.



## Take homes

*Our results indicate that the cellular thermometer model can account for intraspecific but not interspecific variation in Ton  and that the relative levels of expression of different isoforms of hsp70 may provide a better measure of heat stress than the total expression of all hsp70 isoforms.*



Figures are uninterpretable.

------





 <div id='id-section23'/> 


### Page 23: 2017-02-06. Tomanek 2010; JEB  


Reference:

Tomanek L. 2010. Variation in the heat shock response and its implication for predicting the effect of global climate change on species’ biogeographical distribution ranges and metabolic costs. J Exp Biol 213:971–979.



This is one of the few papers that compare the thermal reaction norms of Hsp  protein expression among 3 different species. These species vary in the thermal environment



Species: *Chlorostoma* formerly *Tegula*

1. C. funebralis = warm tolerance
2. C. brunnea = intermediate tolerance
3. C. montereyi = low to intermediate tolerance



Figure 1.

These species shift the reaction norm of Hsp protein expression to the right, in support for the tolerance mechanism. And C. funebralis shifts their max expression upwards at the higher temperature. 



Some species lack HSR:

1. Hydra lack a HSR (Bosch et al. 1988)
   * mRNA gets rapidly degraded
2. antarctic marine organisms have little HSR
   * lacks inducibility of HSPs



**Argues that the variation in the thermal reaction norms of Hsp expression is related to the temperature variation different taxanomic groups experience.**

------

 <div id='id-section24'/> 

### Page 24: 2017-02-06. **Parcel & Lindquist 1994;** cold spring harbor , paper notes   



 <u>Title: Heat Shock proteins and stress tolerance</u> 


* ​
* Protection (resistance) and tolerance are assumed to be the same (conflated). 
* Induced thermotolerance = heat hardening; when exposed to sublethal stress and Hsps increase , it can confer resistance to future temperature threats
* *Nutrient availability, o x y g e n tension, diurnal rhythms, and a host of other variables exert highly reproducible effects on thermotolerance.*
* *In general terms, hsps function by preventing the accumulation of stress-damaged proteins.*
* ​

How does heat stress damage cells?

* Heat damages a wide variety of cellular structures and metabolic proc e s s e s (for detailed reviews, see Nover 1991; Laszlo 1992).*
* *In higher eukaryotes, one of the most immediate effects of heat shock is extensive disruption of the cytoskeleton (Falkner et al. 1981; Coss et al. 1982; Glass et al. 1985; Welch and Suhan 1985; Iida et al. 1 9 8 6 ) .*
* The golgi gets fragmented and number  of lysosomes increase.
* Mitochondria swell and decrease in number— decrease in oxidative phosphorylation
* influence gene expression; reduced
* DNA synthesis is slowed; inhibit chromatn assembly; DNA becomes unstable
* *The total protein content of the nucleus also increases with heat shock (Laszlo 1992).*
* changes lipid membranes; increased fluidity of bilayer and aggregation of integral membrane proteins

What is  the response?

* *In many cases, mild heat pretreatments, which induce hsp synthesis, either reduce the extent of these perturbations or speed their repair.*
* *Unfortunately, although we know that hsps have a central role in thermotolerance and have learned a great deal about the general biochemical functions of these proteins, w e still do not k n ow which heatinduced cellular perturbations are responsible for lethality or w h i c h lethal lesions are the most susceptible to repair by hsps.*

## Hsp70 function in stress tolerance (so this paper frames in term of tolerance)

* *early experiments showed a c l o s e correlation between their induction and the induction of tolerance to high temperatures (Li and Werb 1982; Li and Laszlo 1985; Subjeck and Shyy 1986).*
  * Li & Werb 1982 compare thermal tolerance with Hsp protein expression

Where is hsp70 localized during HS?

* concentrated in membrane, nuclei, and nucleoli (*Pelham 1984; Pelham et al. 1984; Velazquez and Lindquist 1984; Welch and Feramisco 1984*)



**In a f ew experiments, differences in thermotolerance among cultivars have been correlated with differences in hsp synthesis (Ougham and Stoddart 1986; Howarth 1989; Krishnan et al. 1989). **



**In o n e case, however, where the cosegregation of hsp polymorphisms and thermotolerance polymorphisms was examined, no simple correlation was observed (Fender and O'Connell 1989). **



------

 <div id='id-section25'/> 

### Page 25: 2017-02-07. How far down do Aphaenogaster colonies go in the winter?  


Lubertazzi 2012 has the answer. 



```
4.1.4.Winter. In winter months colonies avoid freezing temperatures
by maintaining their nests below ground. Talbot
[24] found the average depth of 5 winter colonies inMissouri
to be 25 cm. Colonies in Connecticut appear to prefer deeper
nests, to a depth of at least 50 cm. Developmental processes
enter a diapause and worker activity within the nest is
minimal.

```

==Depending on where they are, 25-50 cm!!!==



reference: 

Lubertazzi D. 2012. The Biology and Natural History of *Aphaenogaster rudis*. Psyche: A Journal of Entomology [Internet] 2012. Available from: http://www.hindawi.com/journals/psyche/2012/752815/abs

------

 <div id='id-section26'/> 

### Page 26: 2017-02-07. A definition of a trade-off.  


Reading Saltz et al. 2017, Trait Correlations in the Genomics Era; published in *TREE*



I like this: 



```
However, predictions about the evolutionary dynamics of trait correlations go beyond heritability: often we
are interested in why traits are correlated, whether the correlation evolved under selection, and
whether it is possible for selection or drift to change the magnitude or direction of trait
correlations.
```



Trade-off definition:

==Trade-offs occur when functional relationships among traits prevent evolution of optimum values for all traits simultaneously.==



or…when the relationships among traits prevent the evolution of optimal values for all traits simultaneously. 



or

when the correlational structure of functional traits are at odds with selection such that the suite of traits are not optimized.



## Thoughts

Should there be a distinction between a negative correlation among traits as a ==functional trade off==, vs how selection operates on traits to limit trait values in the next generation as an ==evolutionary trade off==?  Because if seleciton operates in the same direction of traits that are negatively correlated, then trait values are optimized. If seleciton operates perpendicular, then that'd be a trade off. 



I like this too:



```
Quantitative genetics theory suggests that identifying genetic loci underlying trait correlations can contribute to answering these questions. Specifically, quantitative trait loci can produce trait correlations through pleiotropy, in which a single locus causally affects two or more traits,
or through linkage, in which two or more loci each affect different traits, but are in linkage disequilibrium (LD) and therefore are inherited together (Box 1). This distinction is important because trait correlations caused by pleiotropy are expected to evolve (adaptively and by drift) differently than trait correlations caused by LD. In general, because LD is expected to erode
through recombination, trait correlations generated by LD are expected to be transient and therefore have a limited scope for contributing to evolutionary change. Similarly, if a trait correlation is caused by LD, each trait might be produced by distinct functional mechanisms
```

reference

  Saltz JB, Hessel FC, Kelly MW. 2017. Trait Correlations in the Genomics Era. Trends in Ecology & Evolution [Internet] 0. Available from: http://www.cell.com/trends/ecology-evolution/abstract/S0169-5347(16)30242-7





------

 <div id='id-section27'/> 

### Page 27: 2017-02-09. SHC lab meeting: Practice talk for biolunch and dissertation defense  




SHC : overall intro is fine, you'll give it better every time you do it. you'll get smoother. 



==SHC has problem with question:==

What types of adaptations are needed to evolve into different environments?

What types of adaptations are needed to colonize different environments? (might want to be more specific, "thermally extreme environments")



SHC wants me to avoid, performance is physiology/ behavior. performance relates to fitness. They are not the same thing. 

In slide 8; I'm showing TPC's across a temperature gradient. The whole thing is shifting, but what im saying is that only ctmax is changing. thinka bout this. display diff



slide 9: there is an issue with me only presenting 1 option. There are other ways!!! don't necessarily talk about other ways. 

Slide 11: Engage with the figure: Here are the 3 heat shock proteins, but if i have a pathway, i better show how it works. Rethink. 



Slide 17: this is vague. I mean that the "dynamics of how they are used". This is what i want to stress. "Temperature selection" is too vague and I can say something more specific. 



Slide 18: this is confusing: too fast, both slide 18 and 19 have positive slopes and how ar they different? 

SHC suggestion: I want to get across, how do i use hsps to be better protected? Have them there all the time! Get across in plain english

Slide 19: Tolerance: when you do get damage, you repair it , turn on when needed and turn on stronger.

==Went too fast; and with graphs, it is hard for audience to get it==. Bar graph instead to give people the words (SHC is not sure)



If HSPs are contributing to thermal limits, how are they doing it? SHC confused as to why upper thermal limits is on y axis

==get rid of slide 20==



Slide 21: title is bad, instead: ants live in different environments. colonized environments from extremely cold to extremely hot!

Slide 22: Question 2 needs adjustments. How implies that it assumed they already adaptively vary. "relate" is vague, be more precise. 

Question 3 is awkward. SHC is not sure, think about how to frame it . How do upper thermal limits interact with other elements of physiology that relates to or influence species survival. or where species occur or performance 



slide 24: NOt clear that i used drosophila for blast search. Which ones in fruit flies are heat inducible? 

SHC suggestion: Save housekeeping business for this slide. Some are for heat response and some are for housekeeping function. Use fruit flies as a reference. Adds to story about shifts taht have gone on in these proteins used in ants as opposed to other insects. it is dynamic. 

==KM suggestions==

conclusions for each chapter was unclear:

multiple stressors: symbols for desiccation and starved were the same. change colors!

----

Slide 25: give a road map for the first time when i present hsp83 gene trees and regulatory elements. diverse group….some might not understand how you can infer duplication on a gene tree. 

**Slide 27: call it hsc70-4  instead of hsp70.**

What is going on with bombus? 



==Slide 30: have contrasting slide, meaning show hsp orthologues not heat inducible!==

Slide 31: Flat title. Conservation and innovation——say something about the evolutionary dynamics of hsps….they are conserved in fucntion, but there is alot of opportunity for innovation. Switch between housekeeping to inducible. There is more to be explored….i.e. subfunctionalization. Not a simple answer, there is a complex answer. Conserved function but the individual players is not static!



One time to make general statements about the implications of the work Im doing. Generalize so you can make general comments about biology. dont overstate but think critically!

Slide 33: both hypotheses don't belong on the same slide

Ask…whether there are adpative shifts in upper thermal limits when ants colonize novel environments. 

==This is what i want to know….==

Extension or enhancement of thermal limits. If you live in a warmer place, can you withstand higher temperatures.  

==if so, how do you do it?== Are hsps involved, how!?

**important**

Then go to the clade. species are structured along the cline. colonieze east coast over X million years. and they have colonized open environments. 

==Bonnie comments==

When i go into the predictions, make it explicit that they are predictions. 

Four main questions was confusing. I only have 3 points, but bonnie was expecting 4. 

Make sure im' clear on DW and FW

**MIstake** : I'm assuming that everybody will understand a phylogenetically informed appraoch. Give overview of approach! What is the phylogeny doing? Set up the audience to understand. More important to be understandable than lenght. 

Slide 35-40 I need to use titles!1



==Conclusion to CTmax part: I set up conflict, but my conclusion does not resolve the conflict.== Needs to be in plain english. 

Are the differences in CTmax explained by differences in how Hsps are used? 

**How does graded protection help/work?**



Tolerance mechanism is not a driver, it's a response to something else that is reducing the incidence of protein denaturing. 

Max is more about repair. 

three possibilities: 

1. protection - active
2. repair - active
3. tolerance - passive

Frame your question! Think about this to tell a precise story. 



RXN norm slides: connect it to something. Issue with how people study this. ANOVA approaches misses elements of adaptive changes in responses 



DW buffer thermal environment compared to FW.  Use data to decide whether i focus on habitat differences or to include cline in ctmax. 



For cahpter 2 conclusion: wrap up the whole chapter…thermal limits and hsp. Evolutionary lability or flexibilty in how hsps are used can lead or ability to colonize new habitats. LInk hsps back to habitat. integrate the data better

Natural shift to last question: The data i have might suggest, in the framework of climat change….DF species might be more protected because they dont get exposed to extremes. HOWEVER, climate change is not about change in temprature….but the constellation of chagnes associated with climate change . INcreases in temperature, other things are changing….other stressors can lead species to become more vulnerable. 

**Goals instead of topics.** Last part could be integrating my results with current climate chagne patterns. 



Slide 68: hypothesis slide for multiple stressors section

I have "or", split up the hypothesis

Slide 70: x axis in predictions could be severity. Overall, predictions are abstract. Explain cross protection or susceptibility mechanisms. ie extra stressor improves or reduces them. NO need for complex prediction slides. Why would another stressor can ever help you? (link back to HSPs…but don't need to add the data;; but you need the explanation)

You need methods: treatments, knockdown time explanation. All results need statistics.  Play with overall conclusion. 



```mermaid
graph TD
A{HSPs} --> B[Sense]
A --> C[Repair]
B --> L(Turn on when proteins unfold)
L --> H{Increase Inflection point}
C --> E(Pre-emptively turn on)
C --> R(Turn on higher)
R --> V{Increase Max expression}
E --> F{Increase Constitutive Expression}
E --> Y{Slow Expression rate}
F --> P[Protect]
Y --> P
V --> T[Tolerate]
```







### 2017-02-15: Alternative way of displaying to get at passive vs active for tolerance  




```mermaid
graph TD
A{HSPs} --> C[Sense and Repair]
C --> L(Turn on when proteins unfold)
C --> R(Turn on higher)
L --> H{Increase Inflection point}
C --> E(Pre-emptively turn on)
R --> V{Increase Max expression}
E --> F{Increase Constitutive Expression}
E --> Y{Slow Expression rate}
F --> P[Protect]
Y --> P
V --active--> T[Tolerate]
H --passive--> T
```





------

 <div id='id-section28'/> 

### Page 28: 2017-02-14. Status of projects + PHD Progress  


Updated again: 2017-02-16

1. Project updates:    
   - **Hsp gene expression + Ctmax project:**
     - NJG made comments 2017-02-13, and I fixed them today 
     - Send out for SHC lab meeting 2017-02-16 to go over next week (2017-02-23)
     - Submit to PNAS in May?   
   - **Multiple stressors ms:**   
     - Resubmitted 2017-02-13; in review (today)
   - **Range limits ms**: SHC lab gave verbal edit, still need to incorporate     
     - **Thermal niche ms:** Lacy to check intro, and add refs.    
   - **Stressed in nature MS: Samples to rerun.**       
     - There are 74 samples: 3 days of RNA isolation + cDNA synthesis. 4 gene targets ran in duplicates is 2 plates per gene = 8 plates total.  2 days for 8 plates.           
   - **Proteome stability project:**    
     - **~130 proteins for rudis, ~250 proteins for pogos**(we got 500 proteins last time); labelling is ok
       - Rerun mass spec, but loading more proteins (Bethany)
2. **Thesis related**   
   - **Defense talk:** 
     - Required defense talk in biolunch: ==Feb 24 (Friday)==     
     - Working title: Evolutionary Innovations of Ants to Thermally Stressful Environments     
   - Formatting:    
     - Introduction (> 3 pages), manuscripts, then synthesis/conclusion (~3 pages) ; SHC and NJG agree
       - CH1. Introduction — started filling in outline
       - **CH2. Hsp functional diversity paper — done (published)**; in thesis now
       - CH3. Hsp rxn norm paper - 
       - **CH4. Multiple stressors paper — done (In review)**; in thesis now
       - CH5. Conclusion — have outline
   - Deadlines: 
     1. Intent to graduate: February 1st for May.   —**done**
     2. Send defense committe form to grad college—**done**
     3. ==Graduate college format check March 4th==
     4. Defense notice 3 weeks before defense  (oral defense by March 24th—*flexible*).   
     5. Final thesis April 7th.  

------

 <div id='id-section29'/> 

### Page 29: 2017-02-15. Aphaenogaster, Hsp rxn norm paper stats revisited:   


From previous analysis, I've found that there is a main effect of Tmax and Habitat type on CTmax in Aphaenogaster. 

```R
library(MASS)
#constructing model
umod<-lm(KO_temp_worker~bio5*habitat_v2+I(bio5^2),data=Aph.dat)
#model selection
summary(stepAIC(umod,direction="both"))

Coefficients:
                       Estimate Std. Error t value
(Intercept)          -4.4102626 12.5885230  -0.350
bio5                  0.2990131  0.0862737   3.466
habitat_v2flat woods  1.5151487  0.2472431   6.128
I(bio5^2)            -0.0004877  0.0001469  -3.320
                     Pr(>|t|)    
(Intercept)          0.726851    
bio5                 0.000792 ***
habitat_v2flat woods 1.96e-08 ***
I(bio5^2)            0.001275 ** 
---
Signif. codes:  
0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.8192 on 96 degrees of freedom
Multiple R-squared:   0.47,	Adjusted R-squared:  0.4534 
F-statistic: 28.37 on 3 and 96 DF,  p-value: 3.191e-13
```



Ok, what if I take out flat woods species, do I still get clinal variation? 

```R
#subset data
aphsub<-subset(Aph.dat,Aph.dat$habitat_v2!="flat woods")
#construct model without habitat, sinece only focusing on deciduous forest species
umod2<-lm(KO_temp_worker~bio5+I(bio5^2),data=aphsub)
#model selection
summary(stepAIC(umod2),direction="both")

Coefficients:
              Estimate Std. Error t value Pr(>|t|)   
(Intercept) -3.1939844 13.5824985  -0.235  0.81468   
bio5         0.2905107  0.0931214   3.120  0.00251 **
I(bio5^2)   -0.0004730  0.0001586  -2.982  0.00379 **
---
Signif. codes:  
0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.8754 on 81 degrees of freedom
Multiple R-squared:  0.208,	Adjusted R-squared:  0.1885 
F-statistic: 10.64 on 2 and 81 DF,  p-value: 7.893e-05
```



## General info on sampling 

**Species numbers**

```R
table(droplevels(Aph.dat$rad_seq_species))

     ashmeadi     floridana         fulva 
            9             9             9 
  lamellidens      miamiana         picea 
            4            13            26 
        rudis tennesseensis 
           28             2 
```

**Habitat numbers**

```R
table(droplevels(Aph.dat$habitat_v2))

deciduous forest       flat woods 
              84               16 
```

**Number of sites:**

```R
 length(unique(Aph.dat$site))
[1] 34
```

**Range of Tmax * 10**

```R
 range(Aph.dat$bio5)
[1] 249 333
```



### For pca of hsp parameters:  


```R
> pchsp$sdev
   Comp.1    Comp.2    Comp.3    Comp.4    Comp.5 
2.1385967 1.3517804 1.0759241 1.0023266 0.8465922 
   Comp.6    Comp.7    Comp.8    Comp.9   Comp.10 
0.8464922 0.7749836 0.6587774 0.5320244 0.4468818 
  Comp.11   Comp.12 
0.3442608 0.2738906 
```

Here are the eigenvalues, and I want to get the number of dimensions $\frac{\lambda_{total}}{\lambda_{1}}$

```R
sum(pchsp$sdev)/pchsp$sdev[1]
4.81275 
```

Closer to 1 means that most of the variation mong these traits is captured by first pc. The larger the number, more PCs capture most of the variation among these traits. 

My session:

```R
> sessionInfo()
R version 3.3.1 (2016-06-21)
Platform: x86_64-apple-darwin13.4.0 (64-bit)
Running under: OS X 10.12.2 (Sierra)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets 
[6] methods   base     

other attached packages:
[1] MASS_7.3-45   ggplot2_2.2.1 raster_2.5-8 
[4] sp_1.2-4     

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.9      lattice_0.20-34  assertthat_0.1  
 [4] grid_3.3.1       plyr_1.8.4       nlme_3.1-131    
 [7] gtable_0.2.0     scales_0.4.1     lazyeval_0.2.0  
[10] tools_3.3.1      munsell_0.4.3    colorspace_1.3-2
[13] knitr_1.15.1     tibble_1.2   
```



------

 <div id='id-section30'/> 

### Page 30: 2017-02-20. prep for committee meeting 2017-02-24, 2:00PM     


### Layout and progress of my chapters  


* 
   <u>Abstract:</u> 
    (3/4 page)




* 
   <u>CH1:  Introduction</u> 
    (3-6 pages)

  ​

* 
   <u>CH2: The evolution of heat shock protein sequences, cis-regulatory elements, and expression profiles in the eusocial Hymenoptera</u> 
    (==published BMC Evolutionary Biology==)

  ​

* 
   <u>Ch3: Molecular adaptations of protection and tolerance predict upper thermal limits in eastern forest ants</u> 
    (==aiming to submit to PNAS==)

* writing in pnas format but will need to reformat for thesis layout?





* 
   <u>CH4: Effects of desiccation and starvation on thermal tolerance and the heat shock response in forest ants</u> 
    (==In 2nd round of reviews, Journal of Comparative Physiology B==)





* 
   <u>CH5: Conclusions and future directions</u> 
    (3 pages)



### Timeline for completing my dissertation this term.    


Important dates: 

* Oral defense: March 24th is latest, but can be later, ==March 27th-31st.== 
  * Send out abstract and notification of oral defense 3 weeks prior
* ==Thesis due April 7th.==

### Set a date for my oral defense. Please bring your calendars.  




### Future post doc plans  


```mermaid
graph TD
M{Future} --> A
M --> D

A[NSF plant genomics postdoctoral fellow] --> B(Finish Dissertation Fall 2017)
B --> C(Start NSF post doc Jan 1st 2018)
D[UFlorida] --> N(Finish dissertation April 2017)
N --> P[Start post doc June 1st 2017]
```





------

 <div id='id-section31'/> 

### Page 31: 2017-02-21. Measuring evolutionary rates in darwins and haldanes    


I thought this was [cool](http://www.biology.ualberta.ca/courses.hp/biol606/OldLecs/rateunits.html):

a darwin ($d$): a change by a factor $e$ per million years



$$rate (d) =  \frac{ln(x_2)- ln(x_1)}{\Delta t}$$



* $ln(x_1)$ and $ln(x_2)$ are two sample means of natural logarthms of measurements
* $\Delta t$ is the time difference between the two samples in millions of years



a haldane (h): a change by a factor of one standard deviation per generation

$$rate(h) = \frac{\frac{lnx_2}{Sln_x}-\frac{lnx_1}{Slnx}}{t_2 - t_1}$$

* $ln(x_1)$ and $ln(x_2)$ are two sample means of natural logarthms of measurements
* $Slnx$ is the pooled standard deviation of $lnx_1$ and $lnx_2$ values
* $t_2 - t_1$ is the time difference between two samples, measured in generations




------

 <div id='id-section32'/> 

### Page 32: 2017-02-22. Resistance vs Tolerance strategies to stress; paper notes-**Núñez-Farfán et al. 2007**; The Evolution of Resistance and Tolerance to Herbivores; Annu. Rev. Ecol. Evol. Syst.  


I want to get a sense of how the herbivore-plant literature uses certain words for responding to stress and potentially adopt that framework in temperature stress. 

1. **Resistance:** Constitutive or induced response of plants against herbivory to avoid or reduce the amount of damage
2. **Tolerance:**  Response of plants induced after consumption to buffer the negative fitness effect of damage. 

These definitions seem inter-related. For example, an induced response that can avoid or reduce damage could also buffer the negative effects of damage. So, a molecule (secondaryy compound) can participate in both resistance and tolerance. I think the difference is the timing of herbivory. Meaning, if a molecule turns on early, it is acting to resist herbivory damage. But if a molecule turns on after herbivory, then it's action will tolerate the damage already done. 

In the coral literature, they like to use ["frontloading"](http://www.pnas.org/content/110/4/1387.full) as another term for resistance. 

```
ref: Barshis DJ, Ladner JT, Oliver TA, et al (2013) Genomic basis for coral resilience to climate change. PNAS 110:1387–1392. doi: 10.1073/pnas.1210224110
```

**In Fineblum and Rausher 1995;** they explain tolerance as the amount of reduced fitness for a given amount of damage, but resistance is the amount of damage a plant experiences for a given abundance of herbivores. 

```
ref:   Fineblum WL, Rausher MD (1995) Tradeoff between resistance and tolerance to herbivore damage in a morning glory. Nature 377:517–520. doi: 10.1038/377517a0
```

**A diff expalantion of tolerance and resistance by Mauricio et al. 1997:**


 <u>Resistance:</u> 
 Traits that reduce the amount of damage a plant experienced. 


 <u>Tolerance:</u> 
 Ability of plant to sustain a fixed amount of herbivore damage without reduction in fitness.

A gem in this paper for the definition of a trade off:

> For example, a negative correlation between two characters, both within and across species, is often taken to indicate the existence of tradeoffs between those characters (Cheverud 1984, Tilman 1990).

```
ref:   Mauricio R, Rausher MD, Burdick DS (1997) Variation in the Defense Strategies of Plants: Are Resistance and Tolerance Mutually Exclusive? Ecology 78:1301–1311. doi: 10.1890/0012-9658(1997)078[1301:VITDSO]2.0.CO;2
```



In a diff system, but another [paper's definition](http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1000150): 

>  Two routes to decreasing susceptibility to infection are resistance (the ability to clear pathogens) and tolerance (the ability to limit damage in response to pathogens).

```
ref:   Ayres JS, Schneider DS (2009) The Role of Anorexia in Resistance and Tolerance to Infections in Drosophila. PLOS Biology 7:e1000150. doi: 10.1371/journal.pbio.1000150
```

Makes no sense….clearing pathogens limits the damage…And clearing a pathogen involves sensing and then responding to pathogens. 

> The first, resistance, is the ability of the host to reduce pathogen levels. The second, tolerance, is the ability to limit the impact of infections.

This makes more sense. 

**Thoughts:** 

I think there are two distinct parts of stress:

1. Whether you encounter it at all. This should involve resistance because there can be ways for organisms to avoid or reduce the amount of stress
2. What do you do when you encounter stress? You can limit the damage caused by it. 

But there is a continuum right? Especially for temperature. 

### How would I define resistance or tolerance in terms of thermal stress biology? Or more generally?    


Resistance is a response (either induced or already present) that impedes disruption of a complex trait/biological system from a perturbation. 

Tolerance is a response that mitigates the disruption of a complex trait/biological system from a perturbation. 

Not sure about these definitions. In real biology, how would this work?

```
		 resistance
		|-----
		|	  \
		|	   \
fitness  |		 \
		|		   -------
		|           tolerance
		______________________
		      leaf damage
```

The amount of leaf damage that does not decrease fitness is resistance. The part where the curve begins to change is tolerance. Can use a breakpoint analysis. 

```
		 
		|		--------
		|	   /  		 \
		|	 /	   		  \
fitness  |	/	 			\
		|/
		|           
		______________________
		      Temperature
```

The slope at the ends would indicate tolerance. But the width of topt may be resistance. 

Not sure this works. 



### Update: 2017-03-09  




Another paper with a definition to think about:

> Plant defence against any type of stress may involve resistance (traits that reduce damage) or tolerance (traits that reduce the negative fitness impacts of damage).

ref:   Agrawal AA, Conner JK, Stinchcombe JR (2004) Evolution of plant resistance and tolerance to frost damage. *Ecology Letters*, **7**, 1199–1208.







------

 <div id='id-section33'/> 

### Page 33: 2017-02-23. Multiple regression models testing effect of Hsp parameters on Ctmax  


Doing a multiple regression as requested by SHC.

> Could you do the straight-up regression?  The PCA does a good job showing a single axis along which expression profiles vary, but the first most basic question is whether hsps are telling you the main story about CTmax.
>
> Sara



Data structure:

```R
'data.frame':	41 obs. of  81 variables:
 $ n                  : int  1 2 3 4 5 6 7 8 9 10 ...
 $ colony.id2         : Factor w/ 41 levels "ALA1","ALA4",..: 1 2 3 4 5 6 7 8 9 10 ...
 $ hsc70              : num  23 22.1 22.5 22.9 23.8 ...
 $ hsp83              : num  22.4 22.1 22.3 22.4 21.6 ...
 $ hsp40              : num  24.4 24 25.1 25.2 25.4 ...
 $ FC_hsc70_1468_max  : num  47.6 29.1 17 30.2 43.7 ...
 $ FC_hsc70_1468_slope: num  1.026 1.071 1.01 0.391 0.909 ...
 $ FC_hsc70_1468_Tm   : num  37.2 36.3 36.5 35.6 36.1 ...
 $ FC_hsp40_541_max   : num  8.5 4.62 21.96 5.79 7.06 ...
 $ FC_hsp40_541_slope : num  2.4311 1.3566 2.6054 0.8119 0.0455 ...
 $ FC_hsp40_541_Tm    : num  37.3 35.3 40.9 35.3 33.1 ...
 $ FC_Hsp83_279_max   : num  5.8 6.46 8.86 7.61 4.3 ...
 $ FC_Hsp83_279_slope : num  2.0964 1.2968 1.757 0.0497 1.0885 ...
 $ FC_Hsp83_279_Tm    : num  37 35 35.1 33.1 34.7 ...
 $ Collection.date    : int  20141011 20141013 20150614 20150622 20140626 20140626 20140626 20140626 20140626 20140626 ...
 $ site               : Factor w/ 16 levels "Alachua Co","Avon",..: 1 1 2 3 4 4 4 4 4 4 ...
 $ state              : Factor w/ 8 levels "FL","MA","MD",..: 1 1 4 4 3 3 3 3 3 3 ...
 $ queen_satus        : Factor w/ 4 levels "","?","QL","QR": 4 3 1 1 4 4 4 4 4 4 ...
 $ location           : Factor w/ 16 levels "ALA","AVON","BING",..: 1 1 2 3 4 4 4 4 4 4 ...
 $ lat                : num  29.7 29.7 44.8 45 39 ...
 $ lon                : num  -82.4 -82.4 -70.3 -69.9 -77.2 ...
 $ genus              : Factor w/ 1 level "Aphaenogaster": 1 1 1 1 1 1 1 1 1 1 ...
 $ species            : Factor w/ 10 levels "?","ashmeadi",..: 7 7 8 8 4 9 9 4 9 4 ...
 $ rad_seq_species    : Factor w/ 8 levels "ashmeadi","floridana",..: 5 5 6 6 3 7 7 3 7 3 ...
 $ colony.id          : Factor w/ 41 levels "Ala1","Ala4",..: 1 2 3 4 5 6 7 8 9 10 ...
 $ Alt_names          : Factor w/ 41 levels "ALA1","ALA4",..: 1 2 3 4 5 6 7 8 9 10 ...
 $ KO_temp_worker     : num  42 41.8 40.4 41.3 40.9 ...
 $ mean_ind_weight_mg : num  0.664 0.509 NA NA 0.936 ...
 $ habitat            : Factor w/ 7 levels "deciduous forest",..: 2 2 4 4 1 1 1 1 1 1 ...
 $ habitat_v2         : Factor w/ 2 levels "deciduous forest",..: 1 1 1 1 1 1 1 1 1 1 ...
 $ nest.location      : Factor w/ 4 levels "log","sand","soil",..: 3 3 1 3 1 1 1 4 4 4 ...
 $ collectors         : Factor w/ 10 levels "ANBE","ANBE, KM, Lloyd Davis",..: 9 10 7 7 6 6 6 6 6 6 ...
 $ bio1               : int  204 204 47 53 125 125 125 125 125 125 ...
 $ bio2               : int  131 131 122 126 119 119 119 119 119 119 ...
 $ bio3               : int  47 47 29 29 33 33 33 33 33 33 ...
 $ bio4               : int  5357 5357 9908 10071 8523 8523 8523 8523 8523 8523 ...
 $ bio5               : int  329 329 249 259 304 304 304 304 304 304 ...
 $ bio6               : int  55 55 -164 -164 -51 -51 -51 -51 -51 -51 ...
 $ bio7               : int  274 274 413 423 355 355 355 355 355 355 ...
 $ bio8               : int  268 268 1 7 212 212 212 212 212 212 ...
 $ bio9               : int  173 173 -74 -70 12 12 12 12 12 12 ...
 $ bio10              : int  268 268 170 178 233 233 233 233 233 233 ...
 $ bio11              : int  132 132 -87 -84 12 12 12 12 12 12 ...
 $ bio12              : int  1336 1336 1102 1052 1038 1038 1038 1038 1038 1038 ...
 $ bio13              : int  202 202 113 106 102 102 102 102 102 102 ...
 $ bio14              : int  56 56 69 64 70 70 70 70 70 70 ...
 $ bio15              : int  41 41 12 13 12 12 12 12 12 12 ...
 $ bio16              : int  547 547 305 291 292 292 292 292 292 292 ...
 $ bio17              : int  189 189 231 211 221 221 221 221 221 221 ...
 $ bio18              : int  547 547 296 286 291 291 291 291 291 291 ...
 $ bio19              : int  277 277 243 229 221 221 221 221 221 221 ...
 $ Axis.1             : num  0.0182 0.0223 0.02 0.0241 -0.2567 ...
 $ Axis.2             : num  0.0222 0.0285 0.0254 0.032 -0.0406 ...
 $ Axis.3             : num  -0.00426 -0.00794 -0.00581 -0.00954 -0.03383 ...
 $ Axis.4             : num  -2.54e-03 -2.59e-03 -6.49e-04 -5.45e-04 -6.64e-05 ...
 $ Axis.5             : num  1.84e-02 2.04e-02 -4.40e-02 -4.99e-02 2.88e-05 ...
 $ Axis.6             : num  0.030309 0.030923 -0.002887 -0.00572 0.000598 ...
 $ Axis.7             : num  -1.59e-02 -2.35e-02 6.21e-05 -4.11e-03 7.34e-04 ...
 $ Axis.8             : num  -4.06e-05 -6.81e-04 4.91e-05 -3.62e-04 6.84e-05 ...
 $ Axis.9             : num  2.55e-04 -2.88e-03 -2.03e-05 -1.81e-03 1.54e-04 ...
 $ Axis.10            : num  9.55e-03 -2.53e-02 2.81e-04 1.53e-03 -1.75e-05 ...
 $ Axis.11            : num  -5.18e-04 4.02e-03 1.38e-03 2.43e-03 9.36e-05 ...
 $ Axis.12            : num  4.51e-04 -5.99e-04 2.49e-03 1.36e-02 -5.69e-06 ...
 $ Axis.13            : num  -2.00e-04 1.58e-04 -3.37e-05 -3.96e-05 -1.12e-06 ...
 $ Axis.14            : num  3.96e-03 -2.98e-03 1.20e-04 3.63e-06 -1.76e-05 ...
 $ Axis.15            : num  1.55e-02 -5.38e-03 2.16e-04 4.29e-04 -1.33e-06 ...
 $ Axis.16            : num  6.07e-03 1.51e-02 -1.57e-04 -1.69e-04 4.88e-06 ...
 $ Axis.17            : num  -6.39e-04 4.58e-04 3.19e-04 -2.48e-04 -2.03e-05 ...
 $ Axis.18            : num  2.72e-03 -1.99e-03 -1.47e-03 1.22e-03 9.44e-05 ...
 $ Axis.19            : num  0.001861 -0.000881 -0.003553 0.006455 0.000168 ...
 $ Axis.20            : num  1.30e-04 -4.88e-05 -5.60e-04 1.14e-03 1.80e-05 ...
 $ Comp.1             : num  -3.711 -3.711 3.836 3.629 0.736 ...
 $ Comp.2             : num  -0.252 -0.252 0.376 0.955 -0.848 ...
 $ rRNA18s            : num  10.87 9.64 9.82 10.18 9.86 ...
 $ n.y                : int  1 2 4 5 7 8 9 10 11 12 ...
 $ hsc70.1            : num  12.1 12.5 12.7 12.7 13.9 ...
 $ hsp83.1            : num  11.6 12.4 12.5 12.2 11.8 ...
 $ hsp40.1            : num  13.6 14.3 15.2 15 15.5 ...
```



Calculating basal expression. Subtracting ct values of hsp83,hsc70-4, and hsp40 from 18srRNA ct values. And then log2 transforming the data to get a measure of basal expression 

```R
basalxp<-log2(jj[,3:5] - jj[,74])
jj<-data.frame(jj,basalxp)
```



Constructing the multiple regression model:

```R
regmod<-lm(KO_temp_worker~hsc70.1+hsp83.1+hsp40.2+FC_hsc70_1468_max+FC_hsc70_1468_slope+FC_hsc70_1468_Tm+FC_hsp40_541_max+FC_hsp40_541_slope+FC_hsp40_541_Tm+FC_Hsp83_279_max+FC_Hsp83_279_slope+FC_Hsp83_279_Tm,data=jj)
```

Model selection based on AIC in both directions (forward and backwards):

```R
summary(stepAIC(regmod,direction="both"))
#output
Call:
lm(formula = KO_temp_worker ~ hsp40.2 + FC_hsc70_1468_max + FC_hsp40_541_slope + 
    FC_hsp40_541_Tm + FC_Hsp83_279_max + FC_Hsp83_279_slope, 
    data = jj)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.92378 -0.28655  0.00972  0.25208  0.97126 

Coefficients:
                    Estimate Std. Error t value Pr(>|t|)    
(Intercept)        37.809870   2.777111  13.615 2.53e-15 ***
hsp40.2            -1.429154   0.673488  -2.122  0.04120 *  
FC_hsc70_1468_max   0.042233   0.006797   6.213 4.56e-07 ***
FC_hsp40_541_slope -0.336009   0.127642  -2.632  0.01266 *  
FC_hsp40_541_Tm     0.224272   0.049786   4.505 7.46e-05 ***
FC_Hsp83_279_max   -0.063139   0.022459  -2.811  0.00813 ** 
FC_Hsp83_279_slope  0.251070   0.109254   2.298  0.02784 *  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.4913 on 34 degrees of freedom
Multiple R-squared:   0.69,	Adjusted R-squared:  0.6353 
F-statistic: 12.61 on 6 and 34 DF,  p-value: 1.926e-07
```



### Model averaging with mumin package  


```R
selmod<-dredge(regmod)
selmod
```

* top 2 models

```R
summary(model.avg(selmod[1:2]))
#output
(conditional average) 
                    Estimate Std. Error Adjusted SE z value Pr(>|z|)    
(Intercept)        36.527394   3.299388    3.371523  10.834  < 2e-16 ***
FC_hsc70_1468_max   0.041943   0.006899    0.007150   5.866  < 2e-16 ***
FC_hsp40_541_slope -0.329519   0.129646    0.134355   2.453  0.01418 *  
FC_hsp40_541_Tm     0.218131   0.051123    0.052921   4.122 3.76e-05 ***
FC_Hsp83_279_max   -0.062754   0.022766    0.023598   2.659  0.00783 ** 
FC_Hsp83_279_slope  0.252757   0.110742    0.114788   2.202  0.02767 *  
hsp40.2            -1.429154   0.673488    0.698325   2.047  0.04070 *  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Relative variable importance: 
                     FC_hsc70_1468_max FC_hsp40_541_slope
Importance:          1.00              1.00              
N containing models:    2                 2              
                     FC_hsp40_541_Tm FC_Hsp83_279_max FC_Hsp83_279_slope
Importance:          1.00            1.00             1.00              
N containing models:    2               2                2              
                     hsp40.2
Importance:          0.73   
N containing models:    1   
```



* Top 10 models

```R
summary(model.avg(selmod[1:10]))
#output
(conditional average) 
                     Estimate Std. Error Adjusted SE z value Pr(>|z|)    
(Intercept)         36.694206   3.447945    3.544068  10.354  < 2e-16 ***
FC_hsc70_1468_max    0.040852   0.007579    0.007834   5.215    2e-07 ***
FC_hsp40_541_slope  -0.323397   0.136255    0.141024   2.293 0.021836 *  
FC_hsp40_541_Tm      0.217875   0.055173    0.057078   3.817 0.000135 ***
FC_Hsp83_279_max    -0.059538   0.023868    0.024679   2.413 0.015842 *  
FC_Hsp83_279_slope   0.252246   0.113812    0.117985   2.138 0.032522 *  
hsp40.2             -1.446654   0.685537    0.711032   2.035 0.041893 *  
FC_hsc70_1468_Tm     0.139557   0.128507    0.132953   1.050 0.293870    
hsc70.1              0.071127   0.087273    0.090593   0.785 0.432381    
hsp83.1              0.056570   0.095056    0.098671   0.573 0.566426    
FC_hsc70_1468_slope  0.103600   0.201892    0.209571   0.494 0.621065    
FC_Hsp83_279_Tm     -0.030072   0.072190    0.074936   0.401 0.688195    
FC_hsp40_541_max    -0.008143   0.019715    0.020465   0.398 0.690716    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Relative variable importance: 
                     FC_hsc70_1468_max FC_hsp40_541_slope FC_hsp40_541_Tm FC_Hsp83_279_max hsp40.2
Importance:          1.00              1.00               1.00            1.00             0.89   
N containing models:   10                10                 10              10                9   
                     FC_Hsp83_279_slope FC_hsc70_1468_Tm hsc70.1 hsp83.1 FC_hsc70_1468_slope
Importance:          0.87               0.15             0.09    0.07    0.07               
N containing models:    8                  2                1       1       1               
                     FC_Hsp83_279_Tm FC_hsp40_541_max
Importance:          0.06            0.06            
N containing models:    1               1                 
```



* Delta 4 AICc criterion

```R
summary(model.avg(selmod, subset = delta < 4))

(conditional average) 
                     Estimate Std. Error Adjusted SE z value Pr(>|z|)    
(Intercept)         36.694206   3.447945    3.544068  10.354  < 2e-16 ***
FC_hsc70_1468_max    0.040852   0.007579    0.007834   5.215    2e-07 ***
FC_hsp40_541_slope  -0.323397   0.136255    0.141024   2.293 0.021836 *  
FC_hsp40_541_Tm      0.217875   0.055173    0.057078   3.817 0.000135 ***
FC_Hsp83_279_max    -0.059538   0.023868    0.024679   2.413 0.015842 *  
FC_Hsp83_279_slope   0.252246   0.113812    0.117985   2.138 0.032522 *  
hsp40.2             -1.446654   0.685537    0.711032   2.035 0.041893 *  
FC_hsc70_1468_Tm     0.139557   0.128507    0.132953   1.050 0.293870    
hsc70.1              0.071127   0.087273    0.090593   0.785 0.432381    
hsp83.1              0.056570   0.095056    0.098671   0.573 0.566426    
FC_hsc70_1468_slope  0.103600   0.201892    0.209571   0.494 0.621065    
FC_Hsp83_279_Tm     -0.030072   0.072190    0.074936   0.401 0.688195    
FC_hsp40_541_max    -0.008143   0.019715    0.020465   0.398 0.690716    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Relative variable importance: 
                     FC_hsc70_1468_max FC_hsp40_541_slope FC_hsp40_541_Tm FC_Hsp83_279_max hsp40.2
Importance:          1.00              1.00               1.00            1.00             0.89   
N containing models:   10                10                 10              10                9   
                     FC_Hsp83_279_slope FC_hsc70_1468_Tm hsc70.1 hsp83.1 FC_hsc70_1468_slope
Importance:          0.87               0.15             0.09    0.07    0.07               
N containing models:    8                  2                1       1       1               
                     FC_Hsp83_279_Tm FC_hsp40_541_max
Importance:          0.06            0.06            
N containing models:    1               1            
```



### Correlation between predictors:   


```R
knitr::kable(round(cor(data.frame(jj[,6:14],basalxp)),3))
```

|                     | FC_hsc70_1468_max | FC_hsc70_1468_slope | FC_hsc70_1468_Tm | FC_hsp40_541_max | FC_hsp40_541_slope | FC_hsp40_541_Tm | FC_Hsp83_279_max | FC_Hsp83_279_slope | FC_Hsp83_279_Tm |  hsc70 |  hsp83 | hsp40 |
| :------------------ | ----------------: | ------------------: | ---------------: | ---------------: | -----------------: | --------------: | ---------------: | -----------------: | --------------: | -----: | -----: | ----: |
| FC_hsc70_1468_max   |             1.000 |               0.522 |            0.599 |            0.425 |              0.289 |           0.295 |            0.645 |              0.216 |           0.453 |  0.271 |  0.165 | 0.117 |
| FC_hsc70_1468_slope |             0.522 |               1.000 |            0.507 |            0.331 |              0.378 |           0.368 |            0.310 |              0.363 |           0.195 |  0.081 |  0.126 | 0.069 |
| FC_hsc70_1468_Tm    |             0.599 |               0.507 |            1.000 |            0.416 |              0.340 |           0.617 |            0.500 |              0.425 |           0.783 |  0.262 |  0.119 | 0.233 |
| FC_hsp40_541_max    |             0.425 |               0.331 |            0.416 |            1.000 |              0.648 |           0.665 |            0.543 |              0.442 |           0.396 |  0.291 |  0.261 | 0.199 |
| FC_hsp40_541_slope  |             0.289 |               0.378 |            0.340 |            0.648 |              1.000 |           0.609 |            0.280 |              0.380 |           0.223 |  0.286 |  0.312 | 0.070 |
| FC_hsp40_541_Tm     |             0.295 |               0.368 |            0.617 |            0.665 |              0.609 |           1.000 |            0.477 |              0.401 |           0.661 |  0.225 |  0.265 | 0.222 |
| FC_Hsp83_279_max    |             0.645 |               0.310 |            0.500 |            0.543 |              0.280 |           0.477 |            1.000 |              0.474 |           0.630 |  0.170 |  0.216 | 0.113 |
| FC_Hsp83_279_slope  |             0.216 |               0.363 |            0.425 |            0.442 |              0.380 |           0.401 |            0.474 |              1.000 |           0.491 | -0.062 | -0.031 | 0.044 |
| FC_Hsp83_279_Tm     |             0.453 |               0.195 |            0.783 |            0.396 |              0.223 |           0.661 |            0.630 |              0.491 |           1.000 |  0.218 |  0.086 | 0.282 |
| hsc70               |             0.271 |               0.081 |            0.262 |            0.291 |              0.286 |           0.225 |            0.170 |             -0.062 |           0.218 |  1.000 |  0.728 | 0.146 |
| hsp83               |             0.165 |               0.126 |            0.119 |            0.261 |              0.312 |           0.265 |            0.216 |             -0.031 |           0.086 |  0.728 |  1.000 | 0.059 |
| hsp40               |             0.117 |               0.069 |            0.233 |            0.199 |              0.070 |           0.222 |            0.113 |              0.044 |           0.282 |  0.146 |  0.059 | 1.000 |



### Another set of analyses:   


> Also, can you test the relationship between the PCs and CTmax without the open habitat species?  This is to see if the same axis of variation also underlies CTmax variation without the big jump associated with the habitat shift.



Assembling dataset with pcs of hsp params with original data:

```R
jj3<-data.frame(jj,hspparams)

summary(jj3$habitat_v2)
#deciduous forest       flat woods 
 #             32                9 

noop<-subset(jj3,jj3$habitat_v2!="flat woods")
#summary(lm(KO_temp_worker~pc1+pc2+pc3,data=noop))
summary(lm(KO_temp_worker~pc1,data=noop))

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept) 41.36600    0.09696 426.630  < 2e-16 ***
pc1         -0.16944    0.06017  -2.816  0.00852 ** 
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.4823 on 30 degrees of freedom
Multiple R-squared:  0.2091,	Adjusted R-squared:  0.1827 
F-statistic: 7.929 on 1 and 30 DF,  p-value: 0.008515

```



### Rational for determining how many PCs to include in analysis  


Found a source [here](http://strata.uga.edu/8370/lecturenotes/principalComponents.html):

> Because reduction of dimensionality is a goal of principal components analysis, several criteria have been proposed for determining how many PCs should be examined and how many should be ignored. One common criteria is to ignore principal components at the point at which the next PC offers little increase in the total explained variation. A second criteria is to include all those PCs up to a predetermined total percent explained variation, such as 90%. A third standard is to ignore components whose explained variation is less than 1 when a correlation matrix is used or less than the average variation explained when a covariance matrix is used, with the idea being that such a PC offers less than one variable’s worth of information. A fourth standard is to ignore the last PCs whose explained variation is all roughly equal.



------

 <div id='id-section34'/> 

### Page 34: 2017-02-23. SHC lab meeting: Going over hxp rxn norm paper.   


Announcements

2017 lab refresher

==ssh in into the gotelli lab server==

Overall feedback:

* Add Katie Miller in the acknowledgements
* ​



Introduction



Results



Discussion



Methods



------

 <div id='id-section35'/> 

### Page 35: 2017-02-23. What are Evolutionary Innovations?   




Reading this paper:   

Hunter JP. 1998. Key innovations and the ecology of macroevolution. Trends in Ecology & Evolution 13:31–36.

Different definitions: 

Miller 1949: 

> key adjustments in the morphological and physiological mechanism which are essential to the origin of new major groups

I like this definition because it describes the emergence of traits with a diversification of a clade. 



**Van Valen 1981:** 

> A key character, in the adaptive sense, is a structure or element of physiology that makes a taxon more or less committed to a way of life different from, or appreciably more efficient  than, that of its ancestors. 

I like how this definition highlights the traits in a new species do better than ancestral ones. So there is a time aspect. 

**Levinton 1988:**

> key innovation is necessary,but not sufficient for a subsequent radiation

**Baum & Larson 1991:**

>  a trait that greatly modifies the selective regime of the linearge in which it evolves

Don't think this is exactly right. Modifies does not have a direction of effect. 

**Rosenzweig & McCord 1991:**

> a key adaptation is a change in the mathematical rule governing a trade-off constraint so taht after the change, the trade-off is less severe

I wouldnt call this a key innovation. Instead, I'd call it breaking a trade off.

**Erwin 1992:** 

> Key Innovations characterize particular clades and are both necessary and sufficient to explain diversification within the clade

What the...

**Heard and Hauser 1995:**

> An evolutionary change in individual trats that causally linked to an increased diversification rate in the resulting clade (for which it is a synapomorphy)

But this definition excludes new species(forms) that have been able to persist in new environments, but have low diversification rates. 

This paper also proposes 3 mechanisms:

1. allowing escape from competition via invasion into a new adaptive zone
2. decreasing probability of extinction by increasing population density via increased individual fitness
3. favoring reproduction or ecological specialization

I think point 3 captures point 1 and 2 simultaneously. 



My own words:

Evolutionary Innovations: 

1. Critical adaptations that facilitate the emergence of new species. 
2. The evolution of critical traits that have enabled species to diversify
3. Adaptations that have enabled species to diversify
4. Evolutionary change in a characteristic that enhances persistance in a new species compared to ancestors


------

 <div id='id-section36'/> 

### Page 36: 2017-02-24. Last biolunch and committee meeting notes.  




### Last biolunch  


Nate:

* Ants are still here…problem with yesterday's adaptation is today's constraint
  * go through argument: yesterday's adaptation is today's constraint—generalizable to the whole clade? ; cross susceptibility, 
* Ants should be resilient

Aimee: 

* Range limits question; can't they move? Then, they'd be resilient.

Melanie Lloyd: 

* I should change fold induction as delta delta CT

Brent's comments:

* semantics; evolutionary innovations makes it sound like something is novel, but it is tweaking an already established molecular mechanism. 
  * careful if you're using evo inn for a paper
* fitting lines to HSR; what I'm looking at is the kinetics of the heat sohck response. How does that change across evo time and diff time
* describing HSR; summarize it a little quicker(i had 3 slides) it could be more succinct so you can get into your experiments. 
* motivation for importance: showing some data that actually substantiates that Hsps are important for thermal tolerance. (good example in drosophila). Hsps are not just a biomarker of stress. Higher hsps = confers greater heat tolerance.  Link Hsp with upper thrmal limits better.
* Setting up predictions for protective vs tolerance: show previously published data showing each protection /tolerance….but tell everybody that people dont systematically look at Hsps and it's role in upper thermal limits. Protection (George's papers; Dong Millers in limpets 2008 or 2009 ish).
* Presenting Hsps gene phylogneies: make it clear which paralogues and orthologues correspond with each species 
* 1st chapter is less well rounded; I went too fast. 
* Why use thermal performance curves? Take it out? 
* At the very end; safety margins but I never plot safety margins. I have my own data, plot it out! And describe it
* "I don't believe it". Small heat shock proteins , ants have small hsps.  A matter of miscommunication



NJG:

* Change his picture 
* Too many "um"s
* I can start with climate change and ants have adaptations to buffer it's effects. 

Melissa's comments:

* Ants are everywhere and can adapt to anything; but it doesn't show finer scale phylogenetic relationshps on that 
* There is constraint based on evo history and that may limit future adaptive potential 
* Have broad map; and then have differences in species ranges and some species dont have a large range; some are secluded. 
  * show map of ant species distributions (show number); they're not all everywhere
* set audiance up for conclusion;by showing in differences in how species inhabit the world ; there is potential for specialization and diversification 
* Spent a long time on Hsps and HSEs without us knowing where it is going. It is nice to have outline for 2 stories. Suggest brief outline slide
* Tighten up what I say about HSE and Hsps. Make it shorter.
* General comment: Innovations. Not the right word. Morphological trait based thing. 
* =="I'd like to tackle" in my dissertation: Say , the question that "I have addressed"==
* Don't call them chapters: talk for 3 questions. "Call it my research" or refer to it that way. 
* for multiple stressors part: there is a biotic and abiotic effect on species
* in multiple stressors part: say sequentially so that people know. 
* When people interrupt me: Say "Great" instead of, where was I? 
* Conclusion: These results suggest that if species didnt adapt in a warm environment they may face exctintion. 
* ==Set up why this idea of yesterday's adaptation is today's constraint is a novel conclusion==
* Make habitat pictures up front. Make pictures the same size. 
* describe the word passive in slide 63; be more explicit in what the Tm shift means
* Show the people what journey they're about to go on. 
  * Tell them what you're about to tell them
  * Tell them
  * Tell them what you told them
* ​



Committee meeting:

* SHC wants me to add proteome stability work if I get NSF. 
* Send out multiple stressors manuscript
* Date set as March 29th








------

 <div id='id-section37'/> 

### Page 37: 2017-02-27. MapPies in R for Zamira  




[mapPies syntax](http://rgraphgallery.blogspot.com/2013/04/rg-plot-bar-or-pie-chart-over-world-map.html)   

Libraries:

```R
#for data parsing
library(dplyr)
library(tidyr)
library(reshape2)
library(plyr)
#phylogenetics packages
library(ape) 
library(geiger) 
library(ade4) 
library(adephylo)
library(phytools)
library(MPSEM)
#for grabbing data from worldclim
library(raster) 
libaray(maps)
library(sp) 
library(dismo) 

#Statistics
library(vegan) # variance partitioning function 
library(MASS)
library(MuMIn)
options(na.action = "na.fail")
#Dealing with matrices
#library(spaa)
```





```R
w <- getData('worldclim', var='bio', res=2.5)
dbio1 <- extract(w, ant_dat[,c("lon","lat")])
ant_dat_clim <- cbind(ant_dat, dbio1[,1:19])
str(ant_dat_clim) #data structure
>'data.frame':	115 obs. of  39 variables:
 $ n                 : int  1 2 3 4 5 6 7 8 9 10 ...
 $ Collection.date   : int  20140520 20140520 20140520 20140616 20140616 20140616 20140616 20140616 20140616 20140617 ...
 $ site              : Factor w/ 45 levels "Alachua Co","Avon",..: 36 36 36 22 22 22 22 22 22 25 ...
 $ state             : Factor w/ 17 levels "","AZ","CA","FL",..: 12 12 12 4 4 4 4 4 4 4 ...
 $ queen_satus       : Factor w/ 4 levels "","?","QL","QR": 4 4 4 3 3 4 3 4 4 4 ...
 $ location          : Factor w/ 42 levels "ALA","AVON","BF",..: 34 34 34 20 20 20 20 20 20 23 ...
 $ lat               : num  40.4 40.4 40.4 29.8 29.8 ...
 $ lon               : num  -75.2 -75.2 -75.2 -82 -82 ...
 $ genus             : Factor w/ 7 levels "Aphaenogaster",..: 1 1 1 1 1 1 1 1 1 1 ...
 $ species           : Factor w/ 16 levels "","?","ashmeadi",..: 14 14 15 7 7 7 3 3 3 7 ...
 $ rad_seq_species   : Factor w/ 15 levels "","ashmeadi",..: 14 14 14 7 7 7 2 2 2 7 ...
 $ colony.id         : Factor w/ 115 levels "Ala1","Ala2",..: 101 102 103 57 58 59 60 61 62 86 ...
 $ colony.id2        : Factor w/ 115 levels "09-A","10-A",..: 92 93 94 58 59 60 61 62 63 77 ...
 $ Alt_names         : Factor w/ 115 levels "09-A","10-A",..: 103 104 105 61 62 63 64 65 66 90 ...
 $ KO_temp_worker    : num  42 41.9 41.6 42.5 42.4 ...
 $ mean_ind_weight_mg: num  1.369 0.969 0.784 1.289 1.206 ...
 $ habitat           : Factor w/ 10 levels "","city","deciduous forest",..: 3 3 3 9 9 9 8 8 8 6 ...
 $ habitat_v2        : Factor w/ 4 levels "city","deciduous forest",..: 2 2 2 4 4 4 4 4 4 4 ...
 $ nest.location     : Factor w/ 5 levels "","log","sand",..: 2 2 2 3 3 3 3 3 3 3 ...
 $ collectors        : Factor w/ 18 levels "ANBE","ANBE, KM, Lloyd Davis",..: 1 1 1 2 2 2 2 2 2 2 ...
 $ bio1              : num  108 108 108 202 202 202 202 202 202 205 ...
 $ bio2              : num  114 114 114 125 125 125 125 125 125 127 ...
 $ bio3              : num  31 31 31 46 46 46 46 46 46 47 ...
 $ bio4              : num  8758 8758 8758 5361 5361 ...
 $ bio5              : num  293 293 293 326 326 326 326 326 326 327 ...
 $ bio6              : num  -67 -67 -67 56 56 56 56 56 56 61 ...
 $ bio7              : num  360 360 360 270 270 270 270 270 270 266 ...
 $ bio8              : num  197 197 197 266 266 266 266 266 266 268 ...
 $ bio9              : num  -8 -8 -8 171 171 171 171 171 171 174 ...
 $ bio10             : num  219 219 219 266 266 266 266 266 266 268 ...
 $ bio11             : num  -8 -8 -8 129 129 129 129 129 129 134 ...
 $ bio12             : num  1162 1162 1162 1319 1319 ...
 $ bio13             : num  116 116 116 196 196 196 196 196 196 191 ...
 $ bio14             : num  74 74 74 58 58 58 58 58 58 57 ...
 $ bio15             : num  12 12 12 42 42 42 42 42 42 41 ...
 $ bio16             : num  327 327 327 533 533 533 533 533 533 537 ...
 $ bio17             : num  250 250 250 197 197 197 197 197 197 195 ...
 $ bio18             : num  323 323 323 533 533 533 533 533 533 537 ...
 $ bio19             : num  250 250 250 259 259 259 259 259 259 262 ...
dim(ant_dat_clim) # dimensions
>[1] 115  39
#grabbing summary of sampling
samples<-as.data.frame(summary(ant_dat_clim$rad_seq_species),rownames=T);samples$species<-rownames(samples)
names(samples)<-c("number","species")
knitr::kable(samples)

```

|               | number | species       |
| :------------ | -----: | :------------ |
|               |      2 |               |
| ashmeadi      |      9 | ashmeadi      |
| barbatus      |      3 | barbatus      |
| caespetum     |      1 | caespetum     |
| Camponotus    |      1 | Camponotus    |
| Crematogaster |      1 | Crematogaster |
| floridana     |      9 | floridana     |
| Formica       |      1 | Formica       |
| fulva         |     10 | fulva         |
| lamellidens   |      4 | lamellidens   |
| miamiana      |     13 | miamiana      |
| pergandei     |      4 | pergandei     |
| picea         |     26 | picea         |
| rudis         |     29 | rudis         |
| tennesseensis |      2 | tennesseensis |



MapPies

```R
dpar<-ddply(ant_dat_clim,.(rad_seq_species,site),summarize,total=length(rad_seq_species))

widepar<-dcast(dpar,site~rad_seq_species)[,-2]
gh<-ant_dat_clim[!duplicated(ant_dat_clim$site),]
gh1<-gh[order(gh$site),]

widepar$lat<-gh1$lat
widepar$lon<-gh1$lon
widepar[is.na(widepar)]<-0
head(widepar)
widepar<-widepar[,-3:-6]
widepar<-widepar[,-4]
widepar<-widepar[,-7]

library(rworldmap)
plot(w, 5, xlim=c(-85,-67), ylim=c(25,47.5),col="white", axes=TRUE, legend=F,main="",box=FALSE)
map("state", c('florida', 'south carolina', 'north carolina', 'georgia', 'virginia', 'west virginia', 'maryland', 'delaware', 'new jersey', 'rhode island', 'new york', 'connecticut', 'massachusetts', 'pennyslvania', 'vermont', 'new hampshire', 'maine', 'alabama', 'tennessee', 'kentucky', 'ohio','iowa','illinois','arkansas','missouri','minnesota','wisconsin','michigan','louisiana','mississippi',"texas","arizona","illinois","california","oregon","utah","washington","kansas","new mexico","montana","idaho","wyoming","north dakota","south dakota","nebraska","oklahoma"), add = TRUE)

mapPies(widepar,nameX="lon",nameY="lat",nameZs=c("picea","fulva","rudis","floridana","ashmeadi","tennesseensis","miamiana","lamellidens"),zColours=c("blue","purple","orange","red","darkred","darkorchid1","darksalmon","goldenrod"),xlim=c(-85,-67), ylim=c(25,47.5),landCol="gray50",addCatLegend = TRUE,add=TRUE,font=3)
#mapBars(widepar,nameX="lon",nameY="lat",nameZs=c("picea","fulva","rudis","floridana","ashmeadi","tennesseensis","miamiana","lamellidens"),zColours=c("blue","purple","orange","red","darkred","darkorchid1","darksalmon","goldenrod"),xlim=c(-85,-67), ylim=c(25,47.5),landCol="gray50",addCatLegend = TRUE,add=TRUE)
```



![](https://cloud.githubusercontent.com/assets/4654474/23382676/2a9fd94c-fd11-11e6-822e-075f160d10ca.jpeg)



------

 <div id='id-section38'/> 

### Page 38: 2017-02-28. Questions to think about from SHC  


In regards to a new paper in JEB finding that cataglyphis ants have higher constitutive protection than induced response. 



SHC

> A good opportunity for you to think about why you found the results that you did – desert Cataglyphis show a shift from induced to constitutive protection, while the Aphaenogaster you surveyed do not.  What might be going on? 

 me:

> One possibility is genetic assimilation where an inducible response evolved into a constitutive one. In other words, evolving into an extremely hot environment requires constant protection from protein damage, while Aphaenogaster turns on Hsps when they encounter protein damage.

SHC

> That’s a mechanistic process, not an explanation for *why* Cataglyphis would make this change while Aphaenogaster would not. 

me:

> Disagree. I explained the why. Maybe you’re looking for this: Temperature variation shapes the plasticity in the heat shock response. If Aphaenogaster live in a more variable environment, then the HSR should be more plastic, while Cataglyphis lives in constant, extreme heat has a less plastic HSR and is constantly on to cope with heat stress. 

SHC:

> Yes, that is in fact better – your first answer was genetic assimilation, with the why partially implied in a follow-up sentence.  But genetic assimilation is a how answer, not a why.  Your PNAS discussion paragraphs will need to have the topics of discussion clearly separated out, not combined into one. ==Do you think you might have seen constitutively higher levels of Hsps if you had included the western members of the genus, or members of the outgroup?  What is the value of looking at the set of species you’ve included instead of (or in addition to) looking at species inhabiting environmental extremes?==

 

 



------

 <div id='id-section39'/> 

### Page 39: 2017-02-28. More analyses for hsp rxn norm paper  




From SHC:

> Another question for you regarding the PNAS analyses.  I am wondering whether the significant effect of “phylogeny” you are picking up is really just a significant effect of species (ie conspecifics are really genetically similar, much more so than species are to one another, and really phenotypically similar and live in really similar environments).  I know the independent contrasts was pretty sparse, but perhaps you could color the points on your scatterplot by species and add best-fit lines to show 1) whether the species differ, and 2) whether there are relationships between environment and CTmax within species.  Just to have a look at it. 



Constructed a model to test the interaction of Tmax and species id(factor), Tmax and habitat type:

```R
umod2<-lm(KO_temp_worker~bio5*habitat_v2+bio5*rad_seq_species,data=Aph.dat)
summary(stepAIC(umod2,direction="both"))
>Coefficients:
                             Estimate Std. Error t value Pr(>|t|)    
(Intercept)                  42.80833    0.26831 159.546  < 2e-16 ***
rad_seq_speciesfloridana     -0.03981    0.37945  -0.105  0.91666    
rad_seq_speciesfulva         -1.79611    0.37945  -4.733 7.97e-06 ***
rad_seq_specieslamellidens   -0.71458    0.48371  -1.477  0.14301    
rad_seq_speciesmiamiana      -1.85706    0.34904  -5.320 7.25e-07 ***
rad_seq_speciespicea         -2.30737    0.31131  -7.412 5.91e-11 ***
rad_seq_speciesrudis         -1.47025    0.30843  -4.767 6.98e-06 ***
rad_seq_speciestennesseensis -2.05833    0.62925  -3.271  0.00151 ** 
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.8049 on 92 degrees of freedom
Multiple R-squared:  0.5096,	Adjusted R-squared:  0.4723 
F-statistic: 13.66 on 7 and 92 DF,  p-value: 5.403e-12
```

Results: It looks like ashmeadi is the reference and all species differ except lamellidens and floridana. I should just do an ANOVA and do a post hoc test to see which ones are diff. 

```R
summary(aov(KO_temp_worker~rad_seq_species,data=Aph.dat))
                Df Sum Sq Mean Sq F value  Pr(>F)    
rad_seq_species  7  61.94   8.848   13.66 5.4e-12 ***
Residuals       92  59.61   0.648                    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```



### OK, let's see the pairwise comparisons:  


```R
TukeyHSD(aov(KO_temp_worker~rad_seq_species,data=Aph.dat))
  Tukey multiple comparisons of means
    95% family-wise confidence level

Fit: aov(formula = KO_temp_worker ~ rad_seq_species, data = Aph.dat)

$rad_seq_species
                                 diff        lwr         upr     p adj
floridana-ashmeadi        -0.03981481 -1.2168021  1.13717250 1.0000000
picea-ashmeadi            -2.30737179 -3.2729871 -1.34175651 0.0000000
rudis-ashmeadi            -1.47025119 -2.4269575 -0.51354490 0.0001832
miamiana-ashmeadi         -1.85705641 -2.9397273 -0.77438555 0.0000196
lamellidens-ashmeadi      -0.71458333 -2.2149536  0.78578698 0.8174237
fulva-ashmeadi            -1.79611111 -2.9730984 -0.61912380 0.0002086
tennesseensis-ashmeadi    -2.05833333 -4.0101460 -0.10652069 0.0312332
picea-floridana           -2.26755698 -3.2331723 -1.30194170 0.0000000
rudis-floridana           -1.43043638 -2.3871427 -0.47373009 0.0003020
miamiana-floridana        -1.81724160 -2.8999125 -0.73457074 0.0000314
lamellidens-floridana     -0.67476852 -2.1751388  0.82560180 0.8573138
fulva-floridana           -1.75629630 -2.9332836 -0.57930899 0.0003129
tennesseensis-floridana   -2.01851852 -3.9703312 -0.06670587 0.0373589
rudis-picea                0.83712060  0.1571201  1.51712110 0.0057326
miamiana-picea             0.45031538 -0.3977942  1.29842501 0.7206279
lamellidens-picea          1.59278846  0.2518094  2.93376752 0.0088873
fulva-picea                0.51126068 -0.4543546  1.47687597 0.7234944
tennesseensis-picea        0.24903846 -1.5830877  2.08116465 0.9998835
miamiana-rudis            -0.38680522 -1.2247575  0.45114704 0.8401083
lamellidens-rudis          0.75566786 -0.5789103  2.09024602 0.6506387
fulva-rudis               -0.32585992 -1.2825662  0.63084637 0.9639642
tennesseensis-rudis       -0.58808214 -2.4155286  1.23936427 0.9736163
lamellidens-miamiana       1.14247308 -0.2851081  2.57005430 0.2161909
fulva-miamiana             0.06094530 -1.0217256  1.14361616 0.9999997
tennesseensis-miamiana    -0.20127692 -2.0977077  1.69515385 0.9999782
fulva-lamellidens         -1.08152778 -2.5818981  0.41884254 0.3406000
tennesseensis-lamellidens -1.34375000 -3.5060138  0.81851376 0.5358888
tennesseensis-fulva       -0.26222222 -2.2140349  1.68959042 0.9998923
```





### Figures:  


Points and linear model fits are colored by species

![](https://cloud.githubusercontent.com/assets/4654474/23439757/3e476e08-fde6-11e6-9686-2763797cbeea.jpeg)

Figure with points colored gray but lines are by species

![](https://cloud.githubusercontent.com/assets/4654474/23439758/3e48fdae-fde6-11e6-82d5-d79eb77a289d.jpeg)



Barplot option:

![](https://cloud.githubusercontent.com/assets/4654474/23440067/eb3fbbfa-fde7-11e6-809a-1ef30e5305a6.jpeg)



**SDiamond suggested I do include species as a random effect:** 

umod3  = additive model between tmax and habitat

umod4= interaction between tmax and habitat

Used anova to compare models

```R
library(lmerTest) # gives p-values for lmer model
umod3<-lmer(KO_temp_worker~bio5+habitat_v2+(1|rad_seq_species),data=Aph.dat)
umod4<-lmer(KO_temp_worker~bio5*habitat_v2+(1|rad_seq_species),data=Aph.dat)

anova(umod3,umod4)
>refitting model(s) with ML (instead of REML)
Data: Aph.dat
Models:
object: KO_temp_worker ~ bio5 + habitat_v2 + (1 | rad_seq_species)
..1: KO_temp_worker ~ bio5 * habitat_v2 + (1 | rad_seq_species)
       Df    AIC    BIC  logLik deviance  Chisq Chi Df Pr(>Chisq)
object  5 259.29 272.31 -124.64   249.28                         
..1     6 259.85 275.48 -123.92   247.85 1.4396      1     0.2302
```



**Results: models dont differ so go with simpler one (umod3)**

```R
summary(umod3)
>Linear mixed model fit by REML t-tests use Satterthwaite approximations to degrees
  of freedom [lmerMod]
Formula: KO_temp_worker ~ bio5 + habitat_v2 + (1 | rad_seq_species)
   Data: Aph.dat

REML criterion at convergence: 260.2

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-4.5651 -0.3550  0.1815  0.5893  1.2790 

Random effects:
 Groups          Name        Variance Std.Dev.
 rad_seq_species (Intercept) 0.1556   0.3944  
 Residual                    0.6672   0.8168  
Number of obs: 100, groups:  rad_seq_species, 8

Fixed effects:
                      Estimate Std. Error        df t value Pr(>|t|)    
(Intercept)          36.942002   1.979357  7.224000  18.664 2.23e-07 ***
bio5                  0.014058   0.006477  7.811000   2.171   0.0626 .  
habitat_v2flat woods  1.050133   0.379272  9.094000   2.769   0.0216 *  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Correlation of Fixed Effects:
            (Intr) bio5  
bio5        -0.995       
hbtt_v2fltw  0.243 -0.284
```

Results: We find a significnat effect of habitat type and Tmax is marginally significant. I also included a quadratic term which made no difference.       







------

 <div id='id-section40'/> 


### Page 40: 2017-03-04. Visit at UF; feedback on dissertation talk.      


Reflecting on visit at UF. 

**Dissertation talk:**

Clancy- part of habitat shifts I see may be due to fire disturbance in flat woods.

Leigh- "I don't believe that the CTmax shifts are related to temperature differences"

In the same vain…some PI named Rob that I met at "zoocial" suggested that I just use land cover. I think I'll use this: https://www.mrlc.gov/nlcd2011.php (this is land cover)

[tree cover data](https://catalog.data.gov/dataset/nlcd-2011-percent-tree-canopy-cartographic)



I got a ton of questions on migration…what the…I didnt focus on this or research this at all. Hmm maybe it would be nice to include pop gen data? Is there viability selection on colony founding among different habitats? 

**Individual meetings:**

Brett: Thought my data were interesting. Suggested to follow up with a biology letters paper by including range size for each species. 

Dan Holt: Consider the effects of both abiotic and biotic interactions in setting range limits. You should read one of his old papers. Also, comparing regression tree with maxent models would be a small paper. He tries to publish in American Naturalist every year. It is his favorite journal too.

Bryony: Being more computational will help you. 

Michael: BAMM still up in the air. 

Christine Miller: Quant gen of sexual selection. males have fancy legs and compete for mates. Doing a phylogenetic approach- reconstructing ancestral states of host plants and morphological innovations.

Baer: mutation accumulation in C. elegans. Has done some networks on metabolites. Odd way to plot things, should try fitness on y and metabolite on x. 



### Getting tree canopy cover data  


NLCD 2011 Percent Tree Canopy (Cartographic) [data download](https://catalog.data.gov/dataset/nlcd-2011-percent-tree-canopy-cartographic)

> The National Land Cover Database 2011 (NLCD2011) USFS percent tree canopy product was produced through a cooperative project conducted by the Multi-Resolution Land Characteristics (MRLC) Consortium (www.mrlc.gov). The MRLC Consortium is a partnership of federal agencies, consisting of the U.S. Geological Survey, the National Oceanic and Atmospheric Administration, the U.S. Environmental Protection Agency, the U.S. Department of Agriculture (USDA) National Agricultural Statistics Service, the U.S. Forest Service, the National Park Service, the U.S. Fish and Wildlife Service, the Bureau of Land Management, NASA, and the U.S. Army Corps of Engineers. One of the primary goals of the project was to generate current, consistent, and seamless national land cover, percent tree canopy, and percent impervious cover at medium spatial resolution. This product is the cartographic version of the NLCD2011 percent tree canopy cover dataset for CONUS at medium spatial resolution (30 m). It was produced by the USDA Forest Service Remote Sensing Applications Center (RSAC). Tree canopy values range from 0 to 100 percent. The analytic tree canopy layer was produced using a Random Forests? regression algorithm. The cartographic product is a filtered version of the regression algorithm output.

Sessioninfo

```R
attached base packages:
[1] stats     graphics  grDevices utils     datasets 
[6] methods   base     

other attached packages:
[1] dismo_1.1-4  raster_2.5-8 sp_1.2-4    

loaded via a namespace (and not attached):
 [1] Rcpp_0.12.9      lattice_0.20-34  assertthat_0.1  
 [4] grid_3.3.1       plyr_1.8.4       gtable_0.2.0    
 [7] scales_0.4.1     ggplot2_2.2.1    lazyeval_0.2.0  
[10] rpart_4.1-10     rgdal_1.2-5      tools_3.3.1     
[13] munsell_0.4.3    yaml_2.1.14      colorspace_1.3-2
[16] knitr_1.15.1     tibble_1.2      
```





```R
x<-raster("nlcd_2011_USFS_tree_canopy_2011_edition_2014_03_31/cartographic_product/nlcd2011_usfs_treecanopy_cartographic_3-31-2014.img")

plot(x)
b<-read.csv("20160517_ANBE_ant_sampling.csv",skip=6) # original dataset
b<-b[-87,] #getting rid of only NA
```

![](https://cloud.githubusercontent.com/assets/4654474/23591907/6f93cefa-01c6-11e7-8d4b-4be62732b1e2.jpeg)

Linking coords with image projections

```R

coords <- b[, c("lon", "lat")]

names(coords) <- c("x", "y")
coordinates(coords) <- ~x + y
proj4string(coords) <- CRS("+proj=longlat +ellps=WGS84 +datum=WGS84")
crs_args <- x@crs@projargs
sites_transformed <- spTransform(coords, CRS(crs_args))


  #extract land cover data for each point, given buffer size
#Landcover <- extract(x, sites_transformed, buffer=30)
Landcover <- extract(x, sites_transformed, buffer=20)

Landcover
b$tree_canopy_20<-unlist(lapply(Landcover, `[[`, 1))

```









------

 <div id='id-section41'/> 

### Page 41: 2017-03-07. To do.  




1. Hsp rxn norm analyses: Is there a way to include intraspecific variation? 
   * I should include species as a random effect in my regression models with CTmax and Hsp parameters
   * Try the jive model: [dryad](http://datadryad.org/resource/doi:10.5061/dryad.n4vp0/1) and [paper](https://academic.oup.com/sysbio/article-lookup/doi/10.1093/sysbio/syw010)
   * I could try [phylocurve](https://cran.r-project.org/web/packages/phylocurve/phylocurve.pdf) for my analyses of CTmax and Hsp rxn norms. 

- [x] KGribble itinerary march 20th
- [ ] format check for thesis
- [x] Thermal niche paper edits
- [ ] ​



Formatting dissertation:

> **Word did not find any entries for your table of
> contents.**
>
> In your document, select the words to include in the table of contents, and
> then on the Home tab, under Styles, click a heading style. Repeat for each
> heading that you want to include, and then insert the table of contents in your
> document. To manually create a table of contents, on the Document Elements tab,
> under Table of Contents, point to a style and then click the down arrow button.
> Click one of the styles under Manual Table of Contents, and then type the
> entries manually.The caption here should match
> that in the body of the document exactly.



Questions to ask and things to go over with Sean for thesis format check.

1. What do I need? Print out of thesis?
2. Double check format for papers in review and already published one
3. Go over defense notice
4. Go over formatting for 3rd chapter, hsp rxn norms
5. ​


------



 <div id='id-section42'/> 


### Page 42: 2017-03-09. General reading notes:       






ref:   Huey RB, Kearney MR, Krockenberger A *et al.* (2012) Predicting organismal vulnerability to climate warming: roles of behaviour, physiology and adaptation. *Philosophical Transactions of the Royal Society B: Biological Sciences*, **367**, 1665–1679.

> Williams et al. [5] developed an integrative framework for assessing traits that promote vulnerability. They proposed that the vulnerability of a species depends on its sensitivity to environmental change, its exposure to that change, its resilience or ability to recover from perturbations and its potential to adapt to change.



A general framework to think about organismal response to climate warming: 

1. thermal sensitivity
2. exposure
3. recovery  or resiliency
4. adaptive potential

How do yo measure these things? 

1. thermal performance curves
2. location, properties of TPCs, behavioral reg, look at life stages
3. ?
4. adaptive potential, additive genetic variance



**Looking for good ref for responding to climate change by acclimation, adaptation, and/or migration.**

refs:

  Moritz C, Agudo R (2013) The Future of Species Under Climate Change: Resilience or Decline? *Science*, **341**, 504–508.

  Wiens JJ (2016) Climate-Related Local Extinctions Are Already Widespread among Plant and Animal Species. *PLOS Biology*, **14**, e2001104.

  Chen I-C, Hill JK, Ohlemüller R, Roy DB, Thomas CD (2011) Rapid Range Shifts of Species Associated with High Levels of Climate Warming. *Science*, **333**, 1024–1026. (range shifts)



**Papers on action potentials and CTmax**

refs: 

  Miller NA, Stillman JH (2012) Neural Thermal Performance in Porcelain Crabs, Genus Petrolisthes. *Physiological and Biochemical Zoology*, **85**, 29–39.



What is K+ conductance doing? 

>For insects, it has been demonstrated that increasing temperature in Locusta migratoria increases the K conductance.of neurons, resulting in nerve transmission failures; that similar failures can be induced by adding K to the extracellular fluid (simulating increased K conductance; Money et al. 2009); and that such failures can be prevented by pharmacologically blocking K channels (Wu et al. 2001).



Hhigher survival to heat stress is associated with decrease nerve membrane K+ conductance

> Additionally, increased survival after sublethal heat stress has also been associated with a decrease in nerve membrane K conductance (Ramirez et al.1999; Money et al. 2009).



Review of action potentials:

> High permeability indicates that particle *mass* moves easily through a membrane. High conductance indicates that electrical *charge* moves easily through a membrane.

Conductance is the inverse of electrical resistance. If conductance is low, then the resistance of that ion to move across the membrane is high. 

K+ more concentrated inside cells. 



------



 <div id='id-section43'/> 


### Page 43: 2017-03-09. Dissertation format check with Sean M.      




1. fix citation page
   * Citation page needs page number "ii"
   * need periods between names
2. some of the pages looked messed up at the bottom, can lower page margin at the bottom of page
3. Upon defense, sign defense form, scan and upload alongside thesis online. 
   * be vigilante for emails because you'll get follow up emails to verify that the formatting is ok




------



 <div id='id-section44'/> 


### Page 44: 2017-03-09. SHC Lab agenda.       




| Date       | Agenda                                   |
| :--------- | :--------------------------------------- |
| Jan. 26th  | Organizational meeting                   |
| Feb. 2nd   | Paper or Data                            |
| Feb. 9th   | Andrew practice talk (actual talk on Feb. 24th) |
| Feb. 16th  | Katie practice talk (actual talk on Mar. 3rd) |
| Feb. 23rd  | Andrews PNAS feedback                    |
| Mar. 2nd   | Lab field trip                           |
| Mar. 9th   | Julias presentation on thesis research   |
| Mar. 16th  | Spring Break!!                           |
| Mar. 23rd  | Caleb Research update                    |
| Mar. 29th  | Andrews Dissertation Defense _ 3pm Jeffords 234 |
| Mar. 30th  | Julia feedback on honors thesis proposal writing review |
| April 6th  | Katies first thesis chapter              |
| April 13th | Paper or Data                            |
| April 20th | Student Research Conference practice poster/talks |
| April 27th | Paper or Data                            |



------



 <div id='id-section45'/> 


### Page 45: 2017-03-10. General phylogenetics workflow      




Drew this up for Emily M. a while ago and just sticking it here for future reference. 

```mermaid
graph TD
A{Phylogeny}
C((Species list)) --> D(Get Sequences)
D --> E{Character Matrix}
E --> K(Model Substitution/Partition finder)
K --> L
E --> L(Analyses)
L --> T(Maximum Likelihood)
T --> A
L --> W[Bayesian]
W --> A
L --> H[BEAST]
H --> B{Ultrametricized Phylogeny}
```





------



 <div id='id-section46'/> 


### Page 46: 2017-03-13: Meeting with SHC for dissertation timeline.      






Does SHC need to read over every piece before I send out? 

Make sure committe members ar ok with me sending stuff out a week out of defene dat. 

I should send out my thesis 2 weeks ahead of time. Send out March 15th? 17th? 20th? 


 <u>Layout:</u> 


* Abstract: (3/4 page)
  * SHC needs to read
* CH1: Introduction (3-6 pages)
  * 5 pages with references
  * SHC needs to read
* CH2: The evolution of heat shock protein sequences, cis-regulatory elements, and expression profiles in the eusocial Hymenoptera (published BMC Evolutionary Biology)
* Ch3: Molecular adaptations of protection and tolerance predict upper thermal limits in eastern forest ants (aiming to submit to PNAS)
  * go over title, 
  * SHC needs to read the whole thing (abstract, intro, results, discussion, methods)
  * I still need to tweak the introduction and discussion
* CH4: Effects of desiccation and starvation on thermal tolerance and the heat shock response in forest ants (In 2nd round of reviews, Journal of Comparative Physiology B)
* CH5: Conclusions and future directions (3 pages)
  * ~ 1 page, have outline, need to fill out



**Go over dissertation talk.**

Are there any glaring things I need to fix? 

Biggest worry: PHylogenetic, PNAS chapter. ==Conclusions felt short, stilted and peices didnt stitch together.== 

Treat phylogenetic part of statistics ; using species as a random effect—not over-ascribing effects of phylogeny. Random effect- there is an independent effect of habitat and temperature. Change in stats might change how I told the story. 

In the ms itself, for reference:

> Colonies within different lineages differed in CTmax between forest habitats (β=1.37 ± 0.253; t=5.38, p <0.001) and with the local thermal extremes (Tmax, β=0.013 ± 0.003; t=3.52, p <0.001; Full model: F2,97=33.58; R2=0.40, p<0.001; Fig 2B). 



Need to change to random effects model

Changed sentence to: 

> When including species as a random effect, colonies within different lineages differed in CTmax between forest habitats (β=1.05 ± 0.379; t=2.76, p <0.05) and with the local thermal extremes (Tmax, β=0.014 ± 0.006; t=2.17, p <0.001;Fig 2B). 





Amanda suggestion: Broader implications? What types of changes we'd expect to see? turnover? certain populations within a species go extinct. Focus on turnover, different areas where ants are locally adapted to climate. Stick with a consistent flow. Specific—>meaning for ants on a large scale. What else do we need to understand? Expecting to hear a broad statement, but didnt hear it and came to a conclusion on it's own. 



My own notes: 

* incorporate ch1 and2 better…dynamic shifts at the gene copy level, and also how they are regulated. 
* Hsps are absoltuely critical for coping with heat stress. 
* hsps can be adaptively tuned over relatively short to broad evolutionary time scales. 
* Evolution of tolerance may make sense because Hsps are costly, and always activating them slows growth, so they should be only turned on when stress is encountered. 
* To respond to climate change, species will need to turn on Hsps all the time? 


------



 <div id='id-section47'/> 


### Page 47: 2017-03-13. SHC analysis suggestion hsp rxn norm paper      




SHC:

> Can you try separating out the deciduous forest species and graph CTmax against PC1, and in a second graph make a box plot comparison of PC1 for deciduous versus pine forest) to show that variation along PC1 significantly explains both patterns of CTmax variation?

Subset, plot, and construct model:

```R
decj<-subset(jj3,jj3$habitat_v2=="deciduous forest") #subset
ggplot(decj,aes(x=pc1,y=KO_temp_worker))+geom_point(size=5,col="darkgray")+T+stat_smooth(method="lm",colour="black",size=2.5)+xlab("PC1 scores")+ylab("CTmax (°C)") # figure
summary(lm(KO_temp_worker~pc1,data=decj)) # model

>Call:
lm(formula = KO_temp_worker ~ pc1, data = decj)

Residuals:
     Min       1Q   Median       3Q      Max 
-1.17476 -0.33930  0.08233  0.34954  0.80763 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept) 41.36600    0.09696 426.630  < 2e-16 ***
pc1         -0.16944    0.06017  -2.816  0.00852 ** 
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.4823 on 30 degrees of freedom
Multiple R-squared:  0.2091,	Adjusted R-squared:  0.1827 
F-statistic: 7.929 on 1 and 30 DF,  p-value: 0.008515
```



![](https://cloud.githubusercontent.com/assets/4654474/23876306/3a46bcf0-0813-11e7-9462-0cd5c87b9d4c.jpeg)



Figure with CTmax v pc1 for deciduous species only.

Looks funky, let's double check:

```R
> decj$rad_seq_species
 [1] miamiana      miamiana      picea         picea        
 [5] fulva         rudis         rudis         fulva        
 [9] rudis         fulva         rudis         rudis        
[13] lamellidens   fulva         lamellidens   picea        
[17] picea         miamiana      rudis         tennesseensis
[21] picea         rudis         rudis         rudis        
[25] rudis         rudis         rudis         rudis        
[29] rudis         rudis         tennesseensis rudis  
```



Ok, no floridana and ashmeadi. 



Species as a random effect (deciduous species)

```R
summary(lmer(KO_temp_worker~pc1+(1|rad_seq_species),data=decj))
Linear mixed model fit by REML t-tests use Satterthwaite
  approximations to degrees of freedom [lmerMod]
Formula: KO_temp_worker ~ pc1 + (1 | rad_seq_species)
   Data: decj

REML criterion at convergence: 48.1

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-2.3133 -0.5270  0.1981  0.5917  1.1821 

Random effects:
 Groups          Name        Variance Std.Dev.
 rad_seq_species (Intercept) 0.06714  0.2591  
 Residual                    0.19680  0.4436  
Number of obs: 32, groups:  rad_seq_species, 6

Fixed effects:
            Estimate Std. Error       df t value Pr(>|t|)    
(Intercept) 41.41169    0.15293  3.72800 270.791 3.84e-09 ***
pc1         -0.16222    0.05958 29.70000  -2.723   0.0107 *  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Correlation of Fixed Effects:
    (Intr)
pc1 -0.385
```





### **Second part of email:** Ok I need to show from the original dataset, the difference in pc1 between forest types:  


```R
ggplot(jj3,aes(x=habitat_v2,y=pc1,fill=habitat_v2))+T+geom_boxplot()+xlab("Forest Type")+ylab("PC1")+scale_fill_manual(name = "", values = c("royalblue3","red"))

summary(aov(pc1~habitat_v2,data=jj3))
            Df Sum Sq Mean Sq F value   Pr(>F)    
habitat_v2   1   85.9   85.90   32.96 1.18e-06 ***
Residuals   39  101.6    2.61                     
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```





![](https://cloud.githubusercontent.com/assets/4654474/23876461/00b8d0e4-0814-11e7-80d2-9c043529a067.jpeg)



Species as a random effect: testing diff in pc1 between habitats (spmo1)

```R
summary(lmer(pc1~habitat_v2+(1|rad_seq_species),data=jj3))
Linear mixed model fit by REML t-tests use Satterthwaite
  approximations to degrees of freedom [lmerMod]
Formula: pc1 ~ habitat_v2 + (1 | rad_seq_species)
   Data: jj3

REML criterion at convergence: 153.7

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-1.5600 -0.7065 -0.2086  0.8903  1.7499 

Random effects:
 Groups          Name        Variance Std.Dev.
 rad_seq_species (Intercept) 0.06065  0.2463  
 Residual                    2.56500  1.6016  
Number of obs: 41, groups:  rad_seq_species, 8

Fixed effects:
                     Estimate Std. Error      df t value
(Intercept)            0.8154     0.3120  2.2230   2.614
habitat_v2flat woods  -3.5297     0.6448  4.3620  -5.474
                     Pr(>|t|)   
(Intercept)            0.1082   
habitat_v2flat woods   0.0042 **
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Correlation of Fixed Effects:
            (Intr)
hbtt_v2fltw -0.484

```

Species as a random effect: testing effect of habitats and Tmax on pc1 (spmo2)

```R
summary(spmo2)
Linear mixed model fit by REML t-tests use Satterthwaite
  approximations to degrees of freedom [lmerMod]
Formula: pc1 ~ habitat_v2 + bio5 + (1 | rad_seq_species)
   Data: jj3

REML criterion at convergence: 160.3

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-1.5469 -0.6572 -0.1900  0.8795  1.6959 

Random effects:
 Groups          Name        Variance Std.Dev.
 rad_seq_species (Intercept) 0.1626   0.4032  
 Residual                    2.5805   1.6064  
Number of obs: 41, groups:  rad_seq_species, 8

Fixed effects:
                       Estimate Std. Error         df t value
(Intercept)           0.9865287  4.3710242  6.3260000   0.226
habitat_v2flat woods -3.5514575  0.7964546  4.1000000  -4.459
bio5                 -0.0004003  0.0145931  6.6500000  -0.027
                     Pr(>|t|)  
(Intercept)            0.8286  
habitat_v2flat woods   0.0106 *
bio5                   0.9789  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Correlation of Fixed Effects:
            (Intr) hbt_2w
hbtt_v2fltw  0.427       
bio5        -0.997 -0.464
```



Ok, let's compare models: 1 with habitat, 1 with habitat and tmax

```R


anova(spmo1,spmo2)
refitting model(s) with ML (instead of REML)
Data: jj3
Models:
object: pc1 ~ habitat_v2 + (1 | rad_seq_species)
..1: pc1 ~ habitat_v2 + bio5 + (1 | rad_seq_species)
       Df    AIC    BIC  logLik deviance Chisq Chi Df
object  4 161.57 168.42 -76.784   153.57             
..1     5 163.57 172.14 -76.784   153.57     0      1
       Pr(>Chisq)
object           
..1        0.9955
```



Temperature range:

```R
range(Aph.dat$bio5)/10
[1] 24.9 33.3
```



Notes from SHC for the discussion:



> - Samepattern of change associated with change in thermal stress within deciduousforest as with shift to more open habitat – common regulatory control such that response can be tuned up or down in a coordinated fashion  in response to selection?  
> - Studyof extreme thermophiles suggests a shift from induced to constitutiveprotection, but this may not be representative of thermal diversification in seasonalenvironments (or less extreme? Or non-desert?). Why not?
> - Protectionis usually thought of in terms of baseline levels, but important to rememberthat this can also be achieved through the induction response via graded orearly-onset protection.  Shift to a moregraded response suggests a change in trigger of Hsp gene expression (fromdirect feedback from denatured proteins to some other temperature sensor –something like HSR1 non-coding RNA?)



Taking out a part of my intro:

> Therefore, the patterns of Hsp gene expression may reveal evolutionary adaptations for
> coping with high temperature (Somero 2011). Evidence for the protection
> mechanism would be elevated levels of Hsp gene expression at relatively benign
> temperatures such that organisms are prepared for future lethal temperatures.
> For example, experimentally elevating Hsps at sublethal levels in fruit flies increased
> survival to extreme heat shock (Cavicchi et al. 1995). Evidence for the
> tolerance mechanism would be the increase of Hsp gene expression in concert
> with temperature increases. For example, the onset of Hsp induction positively
> corresponds with thermal limits in marine snails species (Lars and Somero 2002).

This paragrpah doesn't make that much sense but it has good citations:

> Among studies, evidence supporting the protection and
> tolerance mechanism by is mixed. For example, in controlled heat-shock
> experiments warm-climate populations or species may express Hsps at lower (Dahlhoff
> and Rank 2000; Bedulina et al. 2013) or higher levels (Dietz and Somero 1992;
> Tomanek and Somero 2000). In some cases, warm- and -cool adapted populations or
> species lack adaptive variation in Hsp expression altogether (Madeira et al.
> 2015; Lockwood et al. 2010; Jensen et al. 2010), which might imply no role for Hsps
> in extending upper thermal limits. Moreover, Hsps participate in many other
> biological processes unrelated to heat (Sørensen et al. 2003). For example,
> Hsps can induce in response to oxidative, desiccation, and osmotic stress (Morris
> et al. 2013). However, detailed knockout studies definitively show that thermally
> responsive Hsps are required for mounting a heat shock response (Bettencourt et
> al. 2008). Perhaps adaptive tuning of Hsp gene induction is not possible with rapid
> temperature selection (Sikkink et al. 2014) or in quickly diversifying clades. 



2017-03-14: Plopping my discussion here for reference. A lot of it has to be re-written.  

> Thetransition from closed canopy forests (deciduous) to open canopy pine/oaksavanna, supported by phylogenetic reconstruction and ancestral trait analyses(Fig ), was accompanied by increased upper thermal limits (CTmax; Fig ). The extensionof CTmax was associated with differences in the reaction norms of Hspexpression that reflected both protection and tolerance mechanisms. In supportof the protection mechanism, colonies with higher CTmax had higher expressionat the low end of the temperature gradient simply by slowing expression ratesfor *hsp70* and *hsp40*. In support of the tolerance mechanism, colonies with higher CTmaxexpressed all three Hsps at a higher temperature (Tm) and also to a highermagnitude (maximum expression). Out of the parameters that significantlyexplained variation in CTmax, six out of twelve parameters diverged betweenforest type (Fig). The adaptive modification of Hsps altogether highlightscomplex and non-linear patterns of gene expression.  
>
> ​            The structure of habitats is known alterthe thermal experiences of ectotherms (Brett et al. 2014). For temperate forestants, local environmental temperatures and habitat type have shaped thevariation in thermal traits (Fig ). Deciduous forests species are significantlybuffered from thermal stress than flat woods species due to higher canopy cover(Supplemental), especially over the growing season where foraging activity ishighest. These species are known to utilize behavioral thermal regulation suchas migrating to new nesting locations when they experience unsuitableenvironments and workers may maintain optimal body temperatures by avoidingsolar radiation in the leaf litter, also known as the Bogert effect (Bogert1945). Despite these behavioral activities, forest ants still experienceselection from the thermal environment because there was still clinal variationin Ctmax (Fig 2). In contrast to deciduous forest species, flat woods speciesnest near the surface (13-92 cm in depth; Tschinkel 2011) and are not known tomigrate whole at all (), which subjects them to thermal stress because surfacesoil can become superheated (Porter and Gates 1969; Angilletta 2009). Ifforagers navigate through superheated surface boundary layers, they must rely physiologicalmechanisms to withstand heat stress (). Thermal extremes are known to structurethe diversity of upper thermal limits in ant communities even within the samelocation (Kaspari 2015). For example, in the tropics, ants possess greaterCTmax in the canopy and experience more heat bouts than ants in the leaf litter(Kaspari 2015).
>
> ​            The phenotypic divergence of CTmax inforest ants were explained by 68% of the variation in parameters that representthe sigmoidal shape of Hsp expression. The parameters themselves are correlatedshow large amounts of independent axes of variation for which selection canoperate upon and in fact, we find roughly five effective numbers of dimensionsfrom a PCA analysis (λtotal/λPC1; supplemental). Perhaps high dimensionality is reflected bythe fact that Hsps play other critical roles in the cell (Morris et al. 2013)and are highly pleiotropic. However, only the first PC, which best captured thecorrelation between slope, Tm, max, but not basal gene expression, related themost to CTmax. Each parameter of the Hsp reaction norm captures the non-linearaspect of Hsp gene expression where detection of adaptive variation or lack thereofdepends on the sampled temperature. For example, when sampled at a singletemperature slice along the temperature gradient, differences in Hsp reactionnorms between open and closed forest habitats showed lower, higher, or nodifference at all in Hsp expression. Therefore, sampling dictates where onefinds adaptive variation of a trait without a function-valued approach. This isone of the first studies to our knowledge, to empirically capture thenon-linear process of Hsp gene induction as a function-valued trait(Stinchecombe).
>
> ​            The net effect of shifts in Hsp expressionparameters suggests that colonies with higher CTmax can better maintain proteinhomeostasis in subtle ways that are highly dynamic. For 2/3 Hsps, we foundevidence for protection through graded, but not constitutive means.Constitutive protection may strictly be modulated under different acclimationtemperatures (Helms Cahan et al. 2017), which was kept constant (25 °C) in our study, but greater thermal extremes may be neededfor species to evolve elevated Hsp expression (Willot et al. 2017). On theother hand, graded protection allows for pre-emptive Hsp expression byelevating expression at the lower end of the temperature gradient when the expressionrate is slower. Interestingly, slower expression rate also results in lowerexpression values before reaching maximum expression levels (Fig 1). However,colonies with higher Ctmax compensate by simultaneously elevating maximumexpression to add greater repair at lethal temperatures. Because Hsps act as asensor for protein denaturation (Richter et al. 2010; Craig and Gross 1991), enhancedtolerance (Tm) suggests that the whole proteome is robust to denaturing forspecies in open canopy forests. Very few amino acid changes are needed toevolve greater protein stability (Somero 2004; Lockwood and Somero 2012),however, these changes need to be distributed across many proteins across thewhole proteome (Field 2001). Another route to greater protein stabilityinvolves changing osmolytes and their concentrations to enable higher meltingtemperatures across all proteins (Stillman and Somero 2001). 
>
> ​            Protein homeostasis



------



 <div id='id-section48'/> 


### Page 48: 2017-03-14. Reading notes- Armstrong et al. 2011; hsp70 and neurophysiology        




reference:

  Armstrong GAB, Xiao C, Krill JL *et al.* (2011) Glial Hsp70 Protects K+ Homeostasis in the Drosophila Brain during Repetitive Anoxic Depolarization. *PLoS ONE*, **6**, e28994.



**Objective:** Interesting paper where these authors wanted to understand the role of hsp70 in the ion homeostasis of neurons. 

**Hypothesis:** Hsp70 help maintain K+ ion balance in the brain

**Methods:** To test for the role of hsp70, they began by manipulating fruit flies by treating them with ouabain, which caused loss of electrical activity in neurons. Ouabain is an inhibitor of Na/K ATPase pump.K gets pumped into cells, and NA gets pumped out. So if it is inhibited, K+ will accumulate outside of cells.  Another way they disrupted ion balance was that they put the flies under anoxic conditions (by $N_2$ gas). Their main read out was K+ outside of the cells. Then they measured hsp70 expression in the brain. 

**Results:** Their oubain treatment worked. In hsp70 null background, there were more surges (K+ spikes). The surges were less when Hsp70 was induced in the brain relative to controls. 











------



 <div id='id-section49'/> 


### Page 49: 2017-02-16. Writing notes        




A little tidbit I wrote up that ended up not fitting in with the paragraph:

> To diminish the cost of protein denaturation, irreversibly damaged proteins are degraded by the 26S proteasome when marked with ubiquitin (Ronai
> 2016). Ubiquitination and 26S proteasome activity is higher in cold tolerant compared
> to warm tolerant species under temperature increases (Hofmann and Somero 1996;
> Todgham et al. 2016). In cold tolerant species Higher temperature sensitivity
> to protein damage matches with Hsp expression occurring at lower temperatures
> and further supports that they have less stable proteomes. Therefore clearance
> of denatured proteins is not likely a source of stress resistance. Instead, the stability of the proteome itself
> most likely provides stress resistance because unfolding at higher temperatures
> does not require any additional repair (Hsp) and breakdown (26S proteasome)
> responses that are costly (Hoekstra and Montooth 2013).

I couldn't fix it so I took it out, but it has good ideas and refs.




 <u>Interesting paper that compares the metabolome under different stressors:</u> 


ref:   Robert Michaud M, Benoit JB, Lopez-Martinez G *et al.* (2008) Metabolomics reveals unique and shared metabolic changes in response to heat shock, freezing and desiccation in the Antarctic midge, Belgica antarctica. *Journal of Insect Physiology*, **54**, 645–655.

They measured the metabolome of antarctic midge under desiccation, heat shock, cold shock, and controls. The clustering looks like (newick format):

(desiccation,(cold,(control,heat)))




 <u>last paragraph of dissertation conclusion:</u> 


oberg or pelini paper on ctmax and water weight figure suggests that there is a trade off in 1 subfamily but not another.

cool idea to measure adaptive potential or trade offs between stressors across a whole clade. 

also need to understand how the environment influences thermal experiences





------



 <div id='id-section50'/> 


### Page 50: 2017-03-17. Writing notes  




For the conclusion of my thesis, my thoughts are a little scattered. I took this part out:

> Not only can the HSF-HSE cascade participate in responding to heat stress, but also it appears to be important for development (Gonsalves et al. 2011; Li et al. 2016) and may be critical for otherbiological processes. HSF-HSE binding is associated with transcriptionally active regions that are independent of bound chromatin locations under heat shock (Li et al. 2016), suggesting that the HSF has additional function outside the HSR. In fact, HSF binding can co-localize with the E2F transcription factor, which is known to permit development in insects and is influenced by nutritional stress (Britton and Edgar 1998). It is possible for social insects to utilize the cooperation between HSF and E2F to produce alternative reproductive phenotypes, which are in large part driven by environmental cues (Smith et al. 2008). If stressors besides temperature alter the chromatin structure to enforce a closed (heterochromatin) state to deny HSF-HSE binding. Therefore, even if proteins become unfolded under desiccation or starvation,Hsp may not be able to become induced which may explain the cross-susceptibility to coping with heat stress. 

Hard to fit this in. 



For my hsp rxn discussion, I need to rewrite it, starting with a re-outline for each paragraph:

First part: Focus on CTmax

1. Summary of findings for CTmax and forest types, evolutionary transitions
2. How forest types alter the thermal experiences and in turn, selective regimes. But I need to tie this back to clinal vaariation that I found too
3. Flat woods forests—SHC had problems with examples at the end. Hmmm tie back to selective regimes?



Second part: Focus on hsp rxn norms

4. The transition into flat woods and increase in CTmax were accompanied by dynamic changes in Hsp expression profiles that were captured with a function-valued approach. 

   * function valued approach allowed for novel approach to quantify different strategies for hsp expression that correlate with upper thermal limits 
   * THis is a novel approach over traditional comparisons of typical constitutive and induced expression because
   * Hsp parameters can evolve separately, but the first pc explained 40% of the variation in CTmax

   - give overall relationship, slower expression rate, higher Tm and higher maximum values
   - hsp paramters explained 40% of the variation
   - Furthermore, these parameters can evolve independently

5. Active part of Hsps

   * induced instead of constitutive proteciton, simply by altering expression rate
     * may not be the most important mechanism
     * interestingly, slower expression rate causes higher expression at the low end of the gradient, but lower expression at the higher end of the gradient before reaching max expression values.
   * Instead higher expression or enhanced response shows investment into active repair for flat woods forests species

6. Passive parts of Hsps: 

   * hsps are sensors and the higher onset of induction suggests they can resist protein unfolding
   * ​

7. Conclusion: 



Taking out a part, has good ideas, but poorly organized:



> Hsps expression responds to the thermal sensitivities of enzyme kinetics (Dahlhoff
> and Rank 2000) and protein aggregation (Willot et al. 2016). Very few amino
> acid changes are needed to evolve greater protein stability (Somero
> 2004;Lockwood and Somero 2012) and are enriched in charged amino acids (Leuenberger
> et al. 2017). However, amino acid changes need to be distributed across many
> proteins across the whole proteome (Field 2001), which is extremely difficult
> because many mutations are deleterious and requires high mutation rates. A more
> attainable route to greater proteome stability involves increasing osmolytes
> that which can non-specifically extend melting temperatures across all proteins
> (Stillman and Somero 2001). For example, trehalose is known to act as a
> chaperone and can stabilize proteins (Richter et al. 2010). 





------



 <div id='id-section51'/> 


### Page 51: 2017-03-20. Writing notes.      


Some more writing notes: basically stuff I'm taking out of my discussion. 



> Traditional approaches with comparisons between constitutive and induced responses may miss the subtle responses found in our study because of unsampled variation. For example, a single measurement of Hsp expression at the middle of the temperature gradient reveals no evidence for adaptive variation between forest types. 



------




 <div id='id-section52'/> 


### Page 52: 2017-03-21. Writing notes       


The part that the reviewer wanted me to take out in my discussion because it was too fruit fly focused in my first paper (Nguyen et al. 2016; *BMC Evo Biol*)

>Among different species of *Drosophila*, the primacy of *hsp70*'s role in the heat shock response is
>achieved through an evolutionary increase in copy number from 2 to 5 [16,20]. We
>detected minimal copy number variation in either *hsp83* or *hsc70*, although
>it is important to note that lack of detailed manual annotation of non-*Drosophila* genomes may make it difficult
>to detect highly similar copies should they be present.  Nevertheless, this suggests that Hymenopteran
>(and other insect) Hsps may be recruited to the heat shock response primarily through
>expansions in cis-regulatory HSEs across the entire promoter region, along with
>transcriptional enhancement associated with introns that are lacking in *D. melanogaster *(Additional files 7-8: Figures
>S4-5)[23-25,54-55]. 

**Hsp rxn norm paper**

Took this out of my discussion, couldn't fit it in...

> Broadly in ants, the divergence in upper thermal limits is associated with open to forested habitats and nesting locations (Diamond et al. 2012). However, within our study, we were able to recover divergence in upper thermal limits within forest types that differ in canopy cover. 

Part of discussion I'm saving:

> Acclimation temperature in natural populations change within a season and Hsps are known to track seasonal fluctuations (Dietz and Somero 2004; Banerjee et al. 2014).             



Another part: This is at the end of the proteome stability part

> Therelative contribution of all of these mechanisms are unknown and typically
> studied in isolation. Nonetheless, resisting heat stress as inferred from the
> onset of Hsp induction appears to be a critical feature of Hsp expression
> profiles that is associated with shifts in upper thermal limits.

Maybe I don't even want to pop off on how proteomes can be stable:

> Higher proteome stability can be achieved by expressing more thermally stable isoforms (Mastrangelo et al. 2012), protein-protein interactions (Storey and Wu 2013), accumulation of protective osmolytes (Hottiger et al. 1994; Somero 2003), and evolved differences in amino acid sequence (Lockwood[ and Somero 2012; Fields et al. 2015). 



SHC comments and outline for discussion: 

> - Primarymode of action of Hsps is enhanced response to damage – how much more?  
>
> - Butalso evidence that more heat-tolerant species are better able to toleratethermal stress through other means: protein sequence evolution, osmolyteconcentrations, metabolic depression and gene expression downregulation(Stanton-Geddes).  Because Hsps areexpensive, the more frequent the heat stress, the more costly active responseis.  
>
> - Protectionless important contributor to higher thermal tolerance.  Protection is usually thought of in terms ofbaseline levels, but important to remember that this can also be achievedthrough the induction response via graded or early-onset protection.  Shift to a more graded response suggests achange in trigger of Hsp gene expression (from direct feedback from denaturedproteins to some other temperature sensor – something like HSR1 non-codingRNA?)
>
> - Studyof extreme thermophiles suggests a shift from induced to constitutiveprotection, but this may not be representative of evolved thermal strategies inseasonal environments (or less extreme? Or non-desert?).  Probably not the case that the elevation isoccurring in genes other than the ones sampled here – positive associationbetween inducibility and baseline elevation in hot—desert vs. temperate antcomparison (other literature on the likelihood that non-inducible forms areinvolved in heat-associated chaperoning?) . Why not?  Energetically costly andretards enzyme activity, slowing growth. Most useful when stress is chronic or predictable – typical of speciesliving in predictably extreme environments such as hot deserts (in ants:  Pogonomyrmex and Cataglyphis, desert flies,other taxa).  For temperate species,stress is predictable on timescales that are shorter than the individual’slifetime, making baseline shifts more profitable as a plastic acclimationstrategy.  Evidence that this is whatoccurs in Aphaenogaster (JCB paper), loads of other literature on seasonalacclimation.  
>
> - Samepattern of change associated with change in thermal stress within deciduousforest as with shift to more open habitat – common regulatory control such thatresponse can be tuned up or down in a coordinated fashion  in response to selection?  Any idea of a model that would explainpattern?  HSF binding affinity, etc.?



SHC comment on my conclusion:

> Do you have anything to say about this?  Is there evidence from your work of local
> adaptation to environmental conditions *within*
> species?  

in response to this sentence:

> Further adaptive modifications in the heat shock response will depend on the adaptive
> potential of Hsp expression (Huey et al. 2012; Tedeschi et al. 2016; Diamond
> 2016). 



2017-03-22: taking the significance out before I submit thesis.

> The types evolutionary changes expected for species to persist in a warming world
> are poorly understood. We found that surviving upper thermal extremes involves
> coping with the deleterious effects of heat stress by protecting against and
> tolerating it. 



------



 <div id='id-section53'/> 

### Page 53: 2017-03-22. Prepping for oral defense, reading notes 

Submitted thesis to committee. What types of questions will I get for my defense? 



1. What types of mechanisms increase proteome stability? Why does a proteome need to be stable? 

ref:   Fields PA (2001) Review: Protein function at thermal extremes: balancing stability and flexibility. *Comparative Biochemistry and Physiology Part A: Molecular & Integrative Physiology*, **129**, 417–431.

> Stability is needed to ensure the appropriate geometry for ligand binding, as well as to avoid denaturation, while flexibility is necessary to allow catalysis at a metabolically appropriate rate

a. A stable protein is needed to bind a substrate but also not unfold and aggregate. 

b. Flexibilty is needed for catalysis because it involves domains in the protein to move. 

Stability is defined as the distribution of microstates considered to be in the native form—not sure I like this definition



2. What facilitates protein folding?
   * Hsps 
   * protein-solvant interactions
   * it is a challenge to refold proteins because the number of microstates outnumbers the atoms in the universe
   * hydrophobic effect—non-polar residues are usually in the interior of globular proteins

3. How far do Aphaenogaster migrate? 
   * deciduous forest ones move around a lot, but less than 1 meter
   * ​

4. Across the whole clade, is there evidence for phylogenetic niche conservatism?
   * potentially, the western clade is a bit reversed. There is more diversity in open canopy species and 1 or 2 closed canopy species. 

5. What is the geographical history of NA Aphaenogaster?
   * There is evidence for a diversification event from asia to na. A. japonica is loacted in japan and is sister to all of NA aphaenogaster species and this split occurred ~ 16 may, roughly the same timing as the bering strait. 
   * Part of the Stenamini tribe, and Myrmecine subfamily
   * Genus is polyphyletic and needs to include Messor.

6. What about membrane stability? 
   * Adaptive to shift phospholipid head groups, enrich for PC and lipid saturation to confer stability
   * Under heat stress, lipids melt and become more fluid like, so organisms respond by becoming more rigid

   ​

   ​

   2017-3-24 cont'd:

7. What can pass through a lipid bilayer(pure)? 

   * Things that can pass
     * hydrophobic molecules such as gas, steroids
     * small uncharged polar molecules: urea, water, glycerol, ethanol
   * Things that CANT pass
     * Large uncharged polar molecules: sugars
     * Ions cannot pass (NA+, K+, H+, CA2+)
       * In active transport: 3 NA+ goes out, 2 K+ goes in and produces a negative membrane potential
       * ​


------

------

 <div id='id-section54'/> 

### Page 54: 2017-03-23. To do list  

1. Dissertation defense talk:
   * write out script for parts where I changed the slides (done, need to practice)
2. Multiple stressor ms reivisions
   * SHC-do waht the reveiwers say; so they want relative expression to the initial timepoint
3. Ecological genomics overall hw feedback (sent to MP and SK 2017-03-24)
4. ​


------

 <div id='id-section55'/> 

### Page 55: 2017-03-24. multiple stressors reviewer comments  

On putting stuff online, NJG:

> Well, the reviews are certainly confidential at all times (even after publication) when you are a reviewer. But the reviews are not confidential once the journal gives them to you as an author, so I don't see any harm in having the on your webpage. However, if any of the reviews were signed (the authors revealed their identity), I would not post those because the authors might not appreciate it.



Reviewer comments:

> Reviewer #2: The authors have thoroughly addressed the
> review comments, and in my opinion the manuscript reads much better. The water
> content results and effects of starvation on dry weight are nice additions. I
> only have a couple of minor comments to consider before revision:
>
> 
>
> 1.      Be careful with tense. When discussing the current
>       results, past tense is typically used, when discussing published results, the
>       present tense is typically used. For example, in line 39, "impairs"
>       should read "impaired," and in line 86 "reduced" should be
>       "reduces." Those were the only two that stuck out to me, but make
>       sure tense is consistent.
>
>
>
> 2.      Line 143: Change "2" to "two"
>
>
>
> 3.      Line 148: Space needed after "used"
>
>
>
> 4.      Line 167: Comma needed after "status"
>
>
>
> 5.      In Figures 2 and 4, it would be helpful to clearly
>       indicate in the caption which group is serving as the reference for the 2-ΔΔCt
>       calculations. Also, the authors might want to consider plotting the control
>       groups on the gene expression figures. That is, the control group can be
>       included by setting the mean = 1 scaling the variance accordingly. It is also
>       unclear whether there was a single reference/control group for the induction
>       experiments, or whether each time point had its own room temperature control
>       that served as a reference. If it was the latter, connecting the points with
>       lines is somewhat misleading, because each point is relative to a different
>       control group. Personally, I think it makes sense to use a single reference
>       group (for example, room temperature controls after the first time point), then
>       express everything relative to that group.



Basically I see two approaches to addressing #5, which was the most important suggestion: 



```mermaid
graph TD
. --> A[Explain controls better]
A --> C[and display barplots]
. --> B[Re-calculate gxp from controls and first time point]
```







------

 <div id='id-section56'/> 

### Page 56: 2017-03-28.  Definition of phenotypic plasticity  and general paper reading   

Scheiner 1993; *Annual Review of Ecology and Systematics*

  Scheiner SM (1993) Genetics and Evolution of Phenotypic Plasticity. *Annual Review of Ecology and Systematics*, **24**, 35–68.

Scheiner defines phenotypic plasticity as the general effect of the environment on phenptypes, including when the effect is 0. 

One way to represent phenotypic plasticity is a *norm of reaction* or *reaction norm*. He refers to the reaction norm as the specific form of that(environmental) effect. These terms can be interchangable he argues. Plasticity itself is the specific change in a trait with respect to the environment, regardless of the genotype. 

Angilletta 2009 defines plasticity as the derivative of a reaction norm. He cites a good reference for how plasticity is defined: Debat and David 2001. 

ref: Debat V, David P (2001) Mapping phenotypes: canalization, plasticity and developmental stability. Trends in Ecology & Evolution, 16, 555–561.


I like this quote:    

> The relationship between genotype and phenotype is not one to one.

Hard to understand:   
> This duality is the basis of ‘genetic assimilation’, the process by which a response to unusual environments can be converted by selection into a permanent, genetically determined, phenotypic change.

The response can be genetic itself. Poor definition. I'd define genetic assimilation as the evolution of a plastic response into a fixed one.   

Wow, this dude named Woltereck made up the term "reaction norm" in 1909.     

Definitions of plasticity from this paper:

1. Bradshaw 1965: Plasticity is shown by a genotype when its expression is able to be atlered by environmental influences. This may or many not be adaptive.    

other definitions are junk. They don't include Scheiner into this paper...what the....


### Making sense of JSG's paper:   

JSG discusses 3 potential mechanisms of temperature adaptation that is mediated by the transcriptome:   

1. Enhanced response hypothesis    
* Inducing a stronger transcriptional response to provide protection.    
  * Prediction: cool adapted species strongly upregulate genes at low temperature and down regulate at higher temps

2. Tolerance hypothesis     
* Not really framed as a hypothesis....

> proposes that existing inducible stress responses become insufficient or prohibitively costly as environmental stressors increase in frequency, resulting in a shift away from induced response in favor of structural changes    

What the...does this mean? 

	* I think the hypothesis is this: A shift away from induced response in favor of structural changes.  Makes no sense to me...    
	* This hypothesis predicts adaptation to stress to be associated with lower transcriptional responsiveness and less sensitivity to temperature perturbations. 
	*  I'd describe this as resistance.   

3. Genetic Assimilation hypothesis 
* Selection shifts inducible expression to constitutive expression of stress related genes.   


------

 <div id='id-section57'/> 

### Page 57: 2017-03-28. Meeting with new lab tech, KB.   

Curtis' project needs to be finished. 

Status: 

- **There are 74 samples: 3 days of RNA isolation + cDNA synthesis. 4 gene targets ran in duplicates is 2 plates per gene = 8 plates total.  2 days for 8 plates.**  



Workflow



```mermaid
graph TD
A[Isolate RNA] --> B[Dilute to single concentration]
A --> C(Check quality, RIN)
B --> D[qPCR]

```

1. ==Stressed in nature project==
   * RNA isolations
   * qPCR , measuring gene expression
   * data analysis and reproducible science
     * Github, tracks different versions and facilitates online sharing
     * R, main software for data analysis
2. Maintenance of website
3. Protein related assays
   * protein isolations
   * unfolding curves troubleshooting
   * ​
4. Proteome stability data analysis 
5. ==Lab orientation and lab safety.== 



Monday, Tuesday, Wednesdays. Wednesday leave at 3:45PM. Maybe Fridays. 

Plan for  April 3rd-5th: 

1. Lab orientation and safety
2. Set up github account
3. Set up R and R studio
4. Organize samples. Find them and place them all in a known location. Prepare for RNA isolations. 
   * Let's print out the protocol

April 10th

Begin isolations Monday and Tuesday 

------

 <div id='id-section58'/> 

### Page 58: 2017-03-31. Making thesis fixes, notes  

Adding notes from JPreston and Blockwood for Hsp rxn norm chapter.

<u>JP:</u>

*In contrast, increases in the inflection point indicates later onset of protein damage (stress resistance; Fig. 1C*

> Does this mean there is another mechanism that stops protein damage at moderate temperatures OR that Hsps can evolve to bemore potent at lower levels?

*If thermal tolerance is achieved via greater responsiveness to protein damage, this should be reflected in higher peak expression levels….*

> **To test this don't you need to know how muchprotein damage there is? What if protein damage is the same between two speciesat a moderate temperature, but one of the species (the more tolerant species) takes longer to ramp up Hsps? Doesn't this mean that the tolerant species is less responsive to protein damage at least at moderate temperatures?

I'll add Brent's later.

<u>Blockwood:</u>

For the intro

2nd paragraph in intro

> It would be good to include a statement and reference to studies that justify the protective effect of the HSR *in vivo*, i.e. Welte, et al. 1993, Feder et al. 1995, etc. wherein Hsp over expression was engineered with transgenic constructs and increases in whole-organism heat tolerance were measured. This is important because many in the field are confused by thinking of the HSR as a “stress” response and thus see Hsps as bioindicators of stress rather than the protective molecular “heroes” that they are. 

3rd paragraph in intro:

> You need to cite Brian Bettencourt’s work. He tracked alleles of Hsp70 that corresponded to high and low expression in the Australian Drosophila. So, you could probably say something more than just “currently unclear” because we do know something about natural populations. There’s also the copepod work out of Ron Burton’s lab that has done quite a lot to address this question in recent years.

In response to:

*Hsp gene expression usuallyfollows a non-linear, logistic function (Kingsolver & Woods 2016);...*

> Is this really the only reference for this statement? Is this based off of insects only? Or do they cite more broadly into the marine literature? One of the best references for timecourse of hsp expression is Buckley and Somero 2006

For the discussion

> Two major considerations: First, I think you need a set of statements about other mechanisms of thermal tolerance. Your hsp expression explains 40% of the variation, but what about the other 60%? It’s not all about proteins. This is particularly an important consideration for anyone who appreciates the physiology that underlies CTmax, which likely has a neural (membrane) basis. Further, for aerobic eukaryotes, the membranes of the mitochondria are an important factor. This sort of relates to the OCCLT hypothesis by Portner too. Check out Dahlhoff and Somero 1991 (abalone mitochondria) as well as Stillman and Somero 1999 (porcelain crab heat tolerance). Second, you should briefly discuss the obvious alternative
> hypothesis that the evolution of CTmax may not have accompanied or been driven
> by the colonization of different habitats, but rather the species had different
> CTmax prior to sorting themselves based on their thermal preferences.



------

 <div id='id-section59'/> 

### Page 59: 2017-03-31. To do list; project update    

Project updates:    

- **Hsp gene expression + Ctmax project:**
  - Lot of it has to be reviewed and rewritten
    - For intro and discussion: take Brent's comments and rewrite adding refs
    - Methods need to be shrunk for PNAS format
    - Need to do an analysis that can account for intra and interspecific variation
    - ​
  - Submit to PNAS  
- ==**Multiple stressors ms:**==   
  - In SHC's hands
- **Range limits ms**: SHC lab gave verbal edit, still need to incorporate     
  - focus on 1 end of thermal niche breadth(although it is nice to mention it because CTmin decreases across lat)--CTmin. 
  - Dicussion needs to talk about cold adaptation; why trade-offs?
  - Walk through results better
- **Thermal niche ms:** 
  - ?? Lacy's hands	
- ==**Stressed in nature MS: Samples to rerun.**==       
  - <u>There are 74 samples: 3 days of RNA isolation + cDNA synthesis. 4 gene targets ran in duplicates is 2 plates per gene = 8 plates total.  2 days for 8 plates.</u>        
    - I need to find the samples and organize them! Half of the samples are in the chest freezer, half are in our own -80C. 
    - qPCR
    - Analyze the data   
- **Proteome stability project:**    
  - **~130 proteins for rudis, ~250 proteins for pogos**(we got 500 proteins last time); labelling is ok
    - Rerun mass spec, but loading more proteins (Bethany)


------

 <div id='id-section60'/> 

### Page 60: 2017-04-07. Project updates      


Project updates:    

- **Hsp gene expression + Ctmax project:**
  - Lot of it has to be reviewed and rewritten
    - For intro and discussion: take Brent's comments and rewrite adding refs
    - Methods need to be shrunk for PNAS format
    - Need to do an analysis that can account for intra and interspecific variation
  - Submit to PNAS  
  
- **Multiple stressors ms:**   
  - In SHC's hands

- **Range limits ms**: SHC lab gave verbal edit, still need to incorporate     
  - focus on 1 end of thermal niche breadth(although it is nice to mention it because CTmin decreases across lat)--CTmin. 
  - Dicussion needs to talk about cold adaptation; why trade-offs?
  - Walk through results better

- **Thermal niche ms:** 
  - ?? Lacy's hands	
  
- ==**Stressed in nature MS: Samples to rerun.**==       
  - <u>There are 74 samples: 3 days of RNA isolation + cDNA synthesis. 6 gene targets ran in duplicates is 2 plates per gene = 12 plates total</u>        

    
- **Proteome stability project:**    
  - **~130 proteins for rudis, ~250 proteins for pogos**(we got 500 proteins last time); labelling is ok
    - Rerun mass spec, but loading more proteins
    - Need gapdh values




------

 <div id='id-section61'/> 

### Page 61: 2017-04-10. To do and project updates    

I'm meeting with Wai today at proteomics facility. Ask about:

1. Gapdh values   
2. redoing runs: We only found 120-200 proteins, but last time we found 500 proteins.    

**Organize stressed in nature project.** 

* Input RNA values into spreadsheet.   
* Curtis diluted cDNA 1/10 before qpcr; threshold on the machine is 0.5 I think; 
	* So I need to dilute cDNA to prep for qpcrs. 
 

------

 <div id='id-section62'/> 

### Page 62: 2017-04-11. Hsp rxn norm project: Species efficiency curves    

### Workflow

```mermaid
graph TD
A[qPCR Amplicon] --> B[PCR Purify]
B --> C(Quantify with nanodrop or qubit)
C --> D[Dilute to set amount]
D --> F(Serial dilute 1:10)
F --> K[qPCR]
K --> G(ANCOVA analysis)

```


### Species list   

|species       |hsp70   |hsp40   |hsp83_279 |
|:-------------|:-------|:-------|:---------|
|ashmeadi      |FB2     |LPR4    |LPR4      |
|barbatus      |PB17-10 |PB17-10 |PB17-10   |
|crematogaster |x       |x       |X         |
|floridana     |FB1     |KH7     |KH7       |
|fulva         |CJ6     |CJ6     |DUKE7     |
|lamellidens   |DUKE8   |DUKE9   |DUKE9     |
|miamiana      |ALA1    |ALA4    |ALA4      |
|pergandei     |EXIT    |        |EXIT      |
|picea         |AVON    |TED4    |PHIL      |
|rudis         |POP1    |LEX13   |POP2      |
|tenn.         |        |SR45    |LEX9      |   



### PCR purify amplicons    

Used Qiagen pcr purify kit according to manufacturer's instructions:

1. took 20 uL of amplicon and added 100 uL (5 vol) of PB buffer    
2. Transferred to column, spun top speed 1 min   
3. wash with PE buffer 750 uL, spin top speed 1 min
4. Spin again, top speed 1 min
5. Place column in new tubes (with labels) and elute
6. Eluted by adding 30 uL h20 into center of column and waiting 1 min; spun top speed 1 min


| ISOLATION_ORDER|HSP70   |HSP40   |HSP83         |
|---------------:|:-------|:-------|:-------------|
|               1|FB2     |LPR4    |DUKE9         |
|               2|PB17-10 |PB17-10 |LPR4          |
|               3|x       |x       |PHIL          |
|               4|FB1     |KH7     |KH7           |
|               5|CJ6     |CJ6     |ALA4          |
|               6|DUKE8   |DUKE9   |CREMATOGASTER |
|               7|ALA1    |ALA4    |LEX9          |
|               8|EXIT    |        |DUKE7         |
|               9|AVON    |TED4    |EXIT          |
|              10|POP1    |LEX13   |PB17-10       |
|              11|        |SR45    |POP2          |    


Next I should quantify and then load set amount for first dilution step. THen dilute 1:10 8 times. 

------

 <div id='id-section63'/> 

### Page 63: 2017-4-13. Hsp rxn norm proj: amplicon quant  

RNA quant and 1:10 dilution calculations    

|Species      |Colony        |Gene  | ampliconDNAconc| X10ngcDNA| h20cDNA|
|:------------|:-------------|:-----|---------------:|---------:|-------:|
|ashmeadi     |FB2           |HSP70 |            8.34|      1.20|   98.80|
|barbatus     |PB17-10       |HSP70 |            6.12|      1.63|   98.37|
|crematogster |CREMATOGASTER |HSP70 |            4.65|      2.15|   97.85|
|floridana    |FB1           |HSP70 |            8.50|      1.18|   98.82|
|fulva        |CJ6           |HSP70 |            6.71|      1.49|   98.51|
|lamellidens  |DUKE8         |HSP70 |            2.64|      3.79|   96.21|
|miamiana     |ALA1          |HSP70 |            6.37|      1.57|   98.43|
|pergandei    |EXIT          |HSP70 |            6.37|      1.57|   98.43|
|picea        |AVON          |HSP70 |            5.25|      1.90|   98.10|
|rudis        |POP1          |HSP70 |            5.13|      1.95|   98.05|
|tenn.        |              |HSP70 |              NA|        NA|      NA|
|ashmeadi     |LPR4          |HSP40 |            2.06|      4.85|   95.15|
|rudis        |LEX13         |HSP40 |            2.82|      3.55|   96.45|
|miamiana     |ALA4          |HSP40 |            2.35|      4.26|   95.74|
|tenn.        |SR45          |HSP40 |            2.48|      4.03|   95.97|
|lamellidens  |DUKE9         |HSP40 |            2.45|      4.08|   95.92|
|fulva        |CJ6           |HSP40 |            0.97|     10.33|   89.67|
|barbatus     |PB17-10       |HSP40 |            4.77|      2.10|   97.90|
|pergandei    |              |HSP40 |              NA|        NA|      NA|
|floridana    |KH7           |HSP40 |            2.46|      4.07|   95.93|
|picea        |TED4          |HSP40 |            2.77|      3.61|   96.39|
|crematogster |CREMATOGASTER |HSP40 |            6.26|      1.60|   98.40|
|lamellidens  |DUKE9         |HSP83 |            7.36|      1.36|   98.64|
|ashmeadi     |LPR4          |HSP83 |           12.50|      0.80|   99.20|
|picea        |PHIL          |HSP83 |            9.50|      1.05|   98.95|
|floridana    |KH7           |HSP83 |            9.69|      1.03|   98.97|
|miamiana     |ALA4          |HSP83 |            4.77|      2.10|   97.90|
|crematogster |CREMATOGASTER |HSP83 |            6.12|      1.63|   98.37|
|tenn.        |LEX9          |HSP83 |           11.10|      0.90|   99.10|
|fulva        |DUKE7         |HSP83 |            9.25|      1.08|   98.92|
|pergandei    |EXIT          |HSP83 |            8.14|      1.23|   98.77|
|barbatus     |PB17-10       |HSP83 |            8.07|      1.24|   98.76|
|rudis        |POP2          |HSP83 |            7.64|      1.31|   98.69| 


Dilute 1: 10 = 0.1ng /uL. Then dilute 1:10  (=0.01ng/uL)for first well for efficiency curve.  Dilute by adding 10 uL of sample with 90 uL of water.

### Dilute step for efficiency curve (in concentration of ng/uL)   

1. 0.01
2. 0.001
3. 0.0001
4. 0.00001
5. 0.000001
6. 0.0000001
7. 0.00000001
8. h20

or....

1. 1 E-2
2. 1 E-3
3. 1 E-4
4. 1 E-5
5. 1 E-6
6. 1 E-7
7. 1 E-8
8. h20
 
### qPCR     

Set up hsp70, hsp40, and hsp83 qpcr plates. I made master mix for 100 rxns:   

| Reagent             | Inital conc. | Final conc. | uL for 1 rxn | 100 rxn set up |
| ------------------- | ------------ | ----------- | ------------ | ------------- |
| Power sybr green    | 2x           | 1x          | 5            | 500            |
| Forward Primer      | 10 uM        | 250 nM      | .25          | 25           |
| Reverse Primer      | 10 uM        | 250 nM      | .25          | 25           |
| nuclease free water | na           | na          | .5           | 50             |
| cDNA                | 0.25 ng/ul   | .1 ng/uL    | 4            | na            |
| Total               |              |             | 10           | 600              |    

Loaded 6 uL master mix into plate and then added 4 uL of template (DNA).    

Loaded dilutions for each species H-A. Each species is a row.   


* qpcr steps   
  1. 95 C, 10 minutes     
  2. 40 cycles of: 95 C for 15s, 55 C for 60 seconds and then 70 C for fluorescence acquisition,         

* Melt curve analysis    
  1. Reactions were heated to 95C 15 s    
  2. From 60 C, slowly heat up and measure fluoresence   
  

### Results: Hsp70    

**0.5 fluoresence threshold**

It looks like the dilution didn't work for crematogaster and floridana (Fb1). I didn't mix well for initial 1:10 dilution.

I still need to run hsp40 and hsp83.  

Plate layout: 

Species are loaded as columns:

1. FB2
2. PB17-10
3. CREMATO
4. FB1
5. CJ6
6. DUKE8
7. ALA1
8. EXIT
9. AVON
10. POP11  

Dilution steps are rows: 

H. 1 E-2   

G. 1 E-3   
F. 1 E-4   
E. 1 E-5   
D. 1 E-6    
C. 1 E-7   
B. 1 E-8    
A. h20    

### hsp70 melt curves   

![](https://cloud.githubusercontent.com/assets/4654474/25042674/f7cf2cca-20e7-11e7-97ef-67e73e6899d6.jpg)

Looks like a single product. 

**stats**: Fitting ANCOVA model to determine whether species differ in their  hsp70 primer efficiency (relationship between cycle threshold and dilution of template amplicons)    


```R
summary(mod1)
                    Df Sum Sq Mean Sq   F value   Pr(>F)
log_dil              1 2742.1  2742.1 74884.671  < 2e-16
Sample.Name          7    5.2     0.7    20.392 2.36e-11
log_dil:Sample.Name  7    0.4     0.1     1.684    0.141
Residuals           40    1.5     0.0                   
                       
log_dil             ***
Sample.Name         ***
log_dil:Sample.Name    
Residuals              
---
Signif. codes:  
0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```


------

 <div id='id-section64'/> 

### Page 64: 2017-4-14. Hsp rxn norm project: Species efficiency curves cont'd    

### qPCR: Hsp83 279prim   


Species are loaded as columns:

1. DUKE9   
2. LPR4
3. PHIL
4. KH7
5. ALA4
6. CREMATOGASTER
7. LEX9
8. DUKE7
9. EXIT
10. PB17-10
11. POP2  

Dilution steps are rows: 

H. 1 E-2   
G. 1 E-3   
F. 1 E-4   
E. 1 E-5    
D. 1 E-6    
C. 1 E-7    
B. 1 E-8    
A. h20     

### hsp83 melt curve    

![](https://cloud.githubusercontent.com/assets/4654474/25046504/d307ea2c-20ff-11e7-895d-16daed294d05.jpg)


### Hsp83 effieciency curve

![](https://cloud.githubusercontent.com/assets/4654474/25046795/70f55bba-2101-11e7-91c2-79f793163888.jpeg)

**stats**: Fitting ANCOVA model to determine whether species differ in their  hsp83 primer efficiency (relationship between cycle threshold and dilution of template amplicons)    

```R
str(hsp83)

'data.frame':	77 obs. of  6 variables:
 $ Well_letter: Factor w/ 7 levels "B","C","D","E",..: 7 6 5 4 3 2 1 7 6 5 ...
 $ Well_num   : int  5 5 5 5 5 5 5 6 6 6 ...
 $ Sample.Name: Factor w/ 11 levels "ALA4","CREMATOGASTER",..: 1 1 1 1 1 1 1 2 2 2 ...
 $ Ct         : num  6.77 10.33 13.8 17.46 20.65 ...
 $ Dilution   : num  1e-02 1e-03 1e-04 1e-05 1e-06 1e-07 1e-08 1e-02 1e-03 1e-04 ...
 $ log_dil    : int  -2 -3 -4 -5 -6 -7 -8 -2 -3 -4 ...
 
mod2<-aov(Ct~log_dil*Sample.Name,data=hsp83)
summary(mod2)

Df Sum Sq Mean Sq   F value   Pr(>F)
log_dil              1   3497    3497 36481.640  < 2e-16
Sample.Name         10      5       1     5.730 7.82e-06
log_dil:Sample.Name 10      1       0     0.728    0.695
Residuals           55      5       0                   
                       
log_dil             ***
Sample.Name         ***
log_dil:Sample.Name    
Residuals              
---
Signif. codes:  
0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

```

### Hsp40 data

melt curves:

![](https://cloud.githubusercontent.com/assets/4654474/25056596/ed97c104-2137-11e7-80e7-421366896e9e.jpg)

note: the curves on the left side are hsp70, not hsp40.    

### sp efficiency curves

![](https://cloud.githubusercontent.com/assets/4654474/25056685/6b79dc92-2138-11e7-9659-6174899ad17d.jpeg)

**stats**: Fitting ANCOVA model to determine whether species differ in their  hsp40 primer efficiency (relationship between cycle threshold and dilution of template amplicons)    

```R

                    Df Sum Sq Mean Sq  F value   Pr(>F)
log_dil              1  429.8   429.8 2649.521  < 2e-16
Sample.Name          6   33.7     5.6   34.614 5.83e-08
log_dil:Sample.Name  6    2.0     0.3    2.066    0.119
Residuals           15    2.4     0.2                  
                       
log_dil             ***
Sample.Name         ***
log_dil:Sample.Name    
Residuals              
---
Signif. codes:  
0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

------

 <div id='id-section65'/> 

### Page 65: 2017-04-14. Proteome stability project: Meeting with Wai    

1. gapdh for loading control
2. which samples to analyze? second run or rerun?
3. proteome search for phosphorylated/ubiquitinated database. 

got 1 and3  down, and searched for both second run and rerun. 

next steps:

1. Normalize to gapdh, but I need to find the best peptide that is stable: 
	* vVDLIEYVAkA; this peptide looks the most stable. And it does not align with pogo gapdh   
	* Blastp sequence against pogo genome: 

2. reanalyze unfolding curves with peptide level data

------

 <div id='id-section66'/> 

### Page 66: 2017-04-19. Proteome stability project: Data analysis   

Testing to see if there are differences in the unfolding characteristics of proteins between a warm tolerant (pogos) and cool tolerant species (aphaeno).  

Data related: Need to parse out the ones with enough replicates (ones = peptides)   

```R
countrep<-ddply(glob3,.(Protein.Group.Accessions,Sequence,Modifications),summarize,gene=length(X36))
head(countrep)
[1] 1 5
```

Keep the ones with 5 (2 pogos and 3 aphaeno
```R

genes6colony<-subset(countrep,countrep$gene==5)
genes6colony<-droplevels(genes6colony)
dim(genes6colony)
[1] 202   4

#### Combining dataset based on accessions that have representative colonies for all 5. 
Allcombined<-glob3[glob3$Sequence %in% genes6colony$Sequence,]
dim(Allcombined)
[1] 1052   27
```


Change to wide format:   

```R
comlong<-gather(Allcombined,temperature,unfolding,X30:X65.2)
dim(comlong)
[1] 10520    19

#making species and colony a factor
comlong$species<-as.factor(comlong$species)
comlong$colony<-as.factor(comlong$colony)
```

### Making the temperature actually temperatures:    

```R
comlong$temperature<-as.numeric(sapply(strsplit(comlong$temperature,"X"),'[',2))
```

### Fitting unfolding curves for each colony and for each protein(Accession) and modification 

```R

paramfit<-ddply(comlong,.(species,colony,Protein.Group.Accessions,Sequence),failwith(f=UFfun))

head(paramfit)
summary(paramfit$colony)
species colony Protein.Group.Accessions
1 Aphaenogaster   AR13                365266869
2 Aphaenogaster   AR13                365266869
3 Aphaenogaster   AR13                365266869
4 Aphaenogaster   AR13                769831122
5 Aphaenogaster   AR13                769831122
6 Aphaenogaster   AR13                769831122
          Sequence    Estimate Std. Error
1       AYQEAFDIAK  0.22414403 0.08297519
2       AYQEAFDIAK 41.73987807 1.78323939
3       AYQEAFDIAK  0.04075972 0.09419904
4 SLPFYTEVLGMTLLQK  0.23564075 0.09353631
5 SLPFYTEVLGMTLLQK 42.24986247 1.80218447
6 SLPFYTEVLGMTLLQK  0.05660541 0.09785435
     t value     Pr(>|t|) param
1  2.7013381 3.057976e-02 slope
2 23.4067722 6.595211e-08    Tm
3  0.4326978 6.782526e-01   min
4  2.5192435 3.985459e-02 slope
5 23.4436947 6.523650e-08    Tm
6  0.5784660 5.810753e-01   min

## Count the colonies
summary(paramfit$colony)
AR13  ARY   D1  P45  P53 
 537  384  561  531  573 
```

Only compare the ones with fits for each colony

```R
num<-ddply(paramfit,.(Protein.Group.Accessions,Sequence),summarize,length(colony))
reppep<-subset(num,num$..1==15)

dim(reppep)
[1] 102   3

parparamfit<-paramfit[paramfit$Sequence %in% reppep$Sequence,]
dim(parparamfit)
[1] 1530    9

##original
dim(paramfit)
[1] 2586    9
```

Grabbing the parameters:   

```R
slope<-subset(parparamfit,parparamfit$param=="slope")
Tm<-subset(parparamfit,parparamfit$param=="Tm")
min<-subset(parparamfit,parparamfit$param=="min")
```

### Statistics: Fitting anova for every peptide   

First, wrote an anova function to grab p values
```R
aovfit <- function(dat) {
  aov_fit <- summary(aov(Estimate~species, dat))[[1]][["Pr(>F)"]][1]
}
```

Implement the fit with ddply

```R
li<-ddply(paramfit,.(Sequence),summarize,length=length(colony));dim(li)
IDs4mod<-subset(li,li$length==15)
dim(li)
[1] 200   2
> IDs4mod<-subset(li,li$length==15)
> dim(IDs4mod)
[1] 102   2

### actual fit
totalmod<-ddply(fitdat,.(Sequence,param),failwith(f=aovfit))
```

### Significant differences (Note: peptide level analysis)

No FDR 

```R
totalmod[which(totalmod$V1<0.05),]
           Sequence param         V1
126    INVYYNEASGGK    Tm 0.03304835
132 IQPVLTSGAGVVTNR    Tm 0.03132238
261   TVLIMELINNVAK    Tm 0.04274057
```

With FDR

```R
newp<-p.adjust(totalmod$V1,method="hochberg")
newp[which(newp<0.05)]
numeric(0)
```

### TM stats and plots

```R
Tm<-subset(Tm,Tm$Estimate<60) # subsetting out outliers

Tmav<-ddply(Tm,.(species,Protein.Group.Accessions,Sequence),summarize,Estimate=mean(Estimate))
head(Tmav)

species Protein.Group.Accessions
1 Aphaenogaster                365266869
2 Aphaenogaster                769831122
3 Aphaenogaster                769832256
4 Aphaenogaster                769833958
5 Aphaenogaster                769834659
6 Aphaenogaster                769835104
           Sequence Estimate
1        AYQEAFDIAK 45.18603
2  SLPFYTEVLGMTLLQK 47.02508
3          GIFIVAAK 44.87442
4 AKPVVSFIAGLTAPPGR 42.15132
5         AADTSLYVK 45.31757
6         LAYGTALAK 44.53871
```

Plotting peptide level

```R
ggplot(Tmav, aes(x=Estimate,colour=species, fill=species)) +geom_histogram(binwidth=1, alpha=.5, position="identity")+geom_density(alpha=0.25)+ggcol+ggcol2
```
![](https://cloud.githubusercontent.com/assets/4654474/25201832/2ab2c47a-2521-11e7-9c5b-2919834febae.jpeg)   

### Statistics: Testing differences in overall Tm between aphaeno and pogos.   

```R
summarySE(Tmav,measurevar = "Estimate",groupvars="species")
        species   N Estimate       sd        se
1 Aphaenogaster 102 43.13312 1.488930 0.1474261
2  Pogonomyrmex 101 42.95960 2.996347 0.2981477
         ci
1 0.2924536
2 0.5915165

summary(aov(Estimate~species,data=Tmav))
             Df Sum Sq Mean Sq F value Pr(>F)
species       1    1.5   1.528   0.274  0.601
Residuals   201 1121.7   5.581  
```   

Plotting protein level

```R
ggplot(protlevel, aes(x=Estimate,colour=species, fill=species)) +geom_histogram(binwidth=1, alpha=.5, position="identity")+ggcol+ggcol2
```

![](https://cloud.githubusercontent.com/assets/4654474/25201910/72236d6e-2521-11e7-8fd8-8bf18f72b224.jpeg)     

```R
protlevel<-summarySE(Tmav,measurevar = "Estimate",groupvars=c("species","Protein.Group.Accessions"))

summary(aov(Estimate~species,data=protlevel))
             Df Sum Sq Mean Sq F value Pr(>F)
species       1    0.0   0.024   0.004   0.95
Residuals   100  596.3   5.963   
```


### Slope stats and plots     


Peptide level   

```R
slav<-ddply(slope,.(species,Sequence),summarize,Estimate=mean(Estimate))
>head(slav)
 species          Sequence  Estimate
1 Aphaenogaster         AADTSLYVK 0.3528855
2 Aphaenogaster      ADLVNNLGTIAK 0.3598534
3 Aphaenogaster       AGIPLNDNFVK 0.3501090
4 Aphaenogaster           AIDVAVK 1.0661180
5 Aphaenogaster   AILVDLEPGTMDSVR 0.4839904
6 Aphaenogaster AKPVVSFIAGLTAPPGR 0.7463137

ggplot(slav, aes(x=Estimate,colour=species, fill=species)) +
    geom_histogram(binwidth=.1, alpha=.5, position="identity")+ggcol+ggcol2
```
![](https://cloud.githubusercontent.com/assets/4654474/25202128/407b85ac-2522-11e7-9231-684d369c7ae7.jpeg)

```R
summarySE(slav,measurevar = "Estimate",groupvars="species")
        species   N  Estimate        sd
1 Aphaenogaster 102 0.7693194 0.4620566
2  Pogonomyrmex 102 0.3748694 0.2370752
          se         ci
1 0.04575042 0.09075652
2 0.02347395 0.04656600
```

Slope differences between aphaneo (blue) and pogo (red)
![](https://cloud.githubusercontent.com/assets/4654474/25229939/51da7ea6-25a0-11e7-8b64-fc82a181d21a.jpeg)

### Statistics: Testing overall differences in slope between aphaeno and pogo

```R
summary(aov(Estimate~species,data=slav))
             Df Sum Sq Mean Sq F value  Pr(>F)
species       1  7.935   7.935   58.84 7.1e-13
Residuals   202 27.240   0.135                
               
species     ***
Residuals      
---
Signif. codes:  
0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

### Min value, stats and plots   

```R
ggplot(minav, aes(x=Estimate, colour=species,fill=species)) +
    geom_histogram(binwidth=.05, alpha=.5, position="identity")+ggcol+ggcol2
```

![](https://cloud.githubusercontent.com/assets/4654474/25230246/4aa4c438-25a1-11e7-9374-ecf524d10387.jpeg)

### Statistics: Testing differences in min (overall) between aphaeno and pogo

```R
summary(aov(Estimate~species,data=minav))
             Df Sum Sq Mean Sq F value Pr(>F)   
species       1 0.0392 0.03920   7.949 0.0053 **
Residuals   199 0.9813 0.00493                  
---
Signif. codes:  
0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

summarySE(minav,measurevar = "Estimate",groupvars="species")
        species   N  Estimate         sd
1 Aphaenogaster 102 0.1704008 0.05211795
2  Pogonomyrmex  99 0.1983332 0.08493514
           se         ci
1 0.005160446 0.01023694
2 0.008536302 0.01694001
```

### Slope and Min parameter figure both changing: 

![](https://cloud.githubusercontent.com/assets/4654474/25230732/1c1bd320-25a3-11e7-86a3-289a16a809f6.jpeg)   


------

 <div id='id-section67'/> 

### Page 67: 2017-04-20. Proteome stability project: Re-analysis with normalizing with gapdh   

Previous analysis didn't control for loading (gapdh).   

updated figs:   

![](https://cloud.githubusercontent.com/assets/4654474/25247579/382fda9e-25d9-11e7-96ab-4fddb1558479.jpeg)

### Statistics: Difference in whole proteome unfolding properties between aphaeno and pogo   

**Tm**    

```R
summarySE(Tm,measurevar = "Estimate",groupvars="species")
        species   N Estimate       sd        se        ci
1 Aphaenogaster 187 43.29463 1.789420 0.1308553 0.2581514
2  Pogonomyrmex 171 46.40236 4.081091 0.3120889 0.6160686
> kruskal.test(Estimate~species,data=Tm)

	Kruskal-Wallis rank sum test

data:  Estimate by species
Kruskal-Wallis chi-squared = 57.497, df = 1, p-value = 3.385e-14
```

**Slope**     

```R
summarySE(slope,measurevar = "Estimate",groupvars="species")
        species   N  Estimate        sd         se         ci
1 Aphaenogaster 181 0.5841720 0.4585716 0.03408536 0.06725829
2  Pogonomyrmex 181 0.5394447 0.7583165 0.05636522 0.11122159
> kruskal.test(Estimate~species,data=slope)

	Kruskal-Wallis rank sum test

data:  Estimate by species
Kruskal-Wallis chi-squared = 41.402, df = 1, p-value = 1.239e-10

```

**Min**     

```R
summarySE(min,measurevar = "Estimate",groupvars="species")
        species   N  Estimate        sd          se         ci
1 Aphaenogaster 187 0.2240141 0.1080232 0.007899435 0.01558401
2  Pogonomyrmex 163 0.2633765 0.1235183 0.009674700 0.01910478
kruskal.test(Estimate~species,data=min)

	Kruskal-Wallis rank sum test

data:  Estimate by species
Kruskal-Wallis chi-squared = 15.451, df = 1, p-value = 8.469e-05
```
------

 <div id='id-section68'/> 

### Page 68: 2017-04-21. Proteome Stability project: More data analysis; fitting curves separately to each species    

### The numbers we get:   

|Species     | CommonPeptides_2| Tm_ave| CI_95| Slope| CI_95.1|   Min| CI_95.2|
|:-----------|----------------:|------:|-----:|-----:|-------:|-----:|-------:|
|A. rudis    |              121|  43.14| 0.421| 0.548|  0.0958| 0.173|  0.0166|
|P. barbatus |              729|  47.31| 0.329| 0.455|  0.0315| 0.254|  0.0081|   



### Statistics: differences in overall proteome stability (Tm)

```R
 kruskal.test(Estimate~species,data=combined)

	Kruskal-Wallis rank sum test

data:  Estimate by species
Kruskal-Wallis chi-squared = 97.849, df = 1,
p-value < 2.2e-16

```

figure:   

![](https://cloud.githubusercontent.com/assets/4654474/25279309/fd2f5f96-2673-11e7-8fe2-f3d9e0bde7b4.jpeg)

### Statistics: differences in overall unfolding rate (slope)

```R
 kruskal.test(Estimate~species,data=sl.ave.spec)

	Kruskal-Wallis rank sum test

data:  Estimate by species
Kruskal-Wallis chi-squared = 1, df = 1,
p-value = 0.3173
```

figure: 

![](https://cloud.githubusercontent.com/assets/4654474/25280358/b104b248-2677-11e7-9164-4f9c1df3efe8.jpeg)   

### Statistics: differences in overall min unfolding    

```R
kruskal.test(Estimate~species,data=min.ave)

	Kruskal-Wallis rank sum test

data:  Estimate by species
Kruskal-Wallis chi-squared = 65.718, df = 1,
p-value = 5.203e-16

```

fig:  

![](https://cloud.githubusercontent.com/assets/4654474/25280740/f70a5df0-2678-11e7-9670-a3a6ed550f3e.jpeg)


### Summary and future directions    

1. There were more peptides identified in pogo than aphaenogaster.     

	* Grab fast file with ORFs from Matt Lau. 
	* And he needs to send me an annotation table (with gene IDs, GO and Kegg)
	* I need to get an annotation table for pogos    
	
2. One thing to explore is to do an enrichment analysis for proteins that have high Tms
	* Also, if I have aa sequences, I can determine contribution of    
		* length   
		* aa properties or IDs   
		
3. Searched database for PTMs such as phosphorylation and ubiquitination and found no siginficant modificatoins.    

------

 <div id='id-section69'/> 

### Page 69: 2017-04-24. Post doc project ideas: Diapause timing and circadian clocks.      

Papers I'm reading, refs: 

Levy, R. C., G. M. Kozak, C. B. Wadsworth, B. S. Coates, and E. B. Dopman. 2015. Explaining the sawtooth: latitudinal periodicity in a circadian gene correlates with shifts in generation number. Journal of Evolutionary Biology 28:40–53.




Question? What are the molecular targets of selection in temporally diverging reproductively isolated populations?   

Hypothesis: Circadian clock/rhythm genes mediate shifts in the timing of life history strategies.  

Experimental approach: 

* Knockdown period, timeless, cry1 in Z and E strains with RNAi, inhibitors and compare entry, duration, and exit to diapause.
	* Which inhibitors?  
* To determine if inhibitors perturb protein protein interactions, I will measure the proteome correlates of circ clocks with immunoprecipitation.

Predictions: 

* Z strain (univoltine) will phenocopy E strain. 
* Both strains will delay?    



------

 <div id='id-section70'/> 

### Page 70:  2017-04-25. Proteome stability, individual colony variation and other stats   

No filter for common peptides. So this figure has a mix of unique and overlapping peptides among all colonies.   

![](https://cloud.githubusercontent.com/assets/4654474/25397100/154fd86e-29b5-11e7-8aff-88f5ca5d20b4.jpeg)   


### Revisiting Statistics: ANOVAs for each parameter and each gene: 

function I used
```R
aovfit <- function(dat) {
  aov_fit <- summary(aov(Estimate~species, dat))[[1]][["Pr(>F)"]][1]
}
```


**peptide level :**    


```R
totalmod<-ddply(fitdat,.(Sequence,param),failwith(f=aovfit))
totalmod[which(totalmod$V1<0.05),]
             Sequence param          V1       adj
1           AADTSLYVK   min 0.001916678 0.2970850
7     AILVDLEPGTMDSVR   min 0.037236005 0.9920444
13         ALGLPIERPK   min 0.042182366 0.9920444
26      APFDPPGPPGTPK slope 0.034160311 0.9920444
42       DAGTISGLVVMR    Tm 0.045975707 0.9920444
61       INVYYNEASGGK   min 0.012364019 0.9920444
76         LLEELEQGQK   min 0.015446495 0.9920444
79         LPVVAATIYR   min 0.012437384 0.9920444
93       NPVCTDYFSGQK    Tm 0.019241119 0.9920444
100         SSLPEHVVK   min 0.001634925 0.2550484
108         TIVDITSHK    Tm 0.014540086 0.9920444
129 VIDPFTIKPIDAQTIIK    Tm 0.006483948 0.9920444
142       VVPPILLETGK   min 0.018339626 0.9920444
```

**protein level :**     

```R
protmod1<-ddply(proteinlev,.(Protein.Group.Accessions,param),failwith(f=aovfit))
protmod1[protmod1$V1<0.05,]
  Protein.Group.Accessions param          V1       
7                 769834659   min 0.001916678 
13                769835200   min 0.015446495 
17                769838592 slope 0.034160311 
24                769842741    Tm 0.045975707 
54                769850975    Tm 0.035431063 
55                769851924   min 0.028747970 


protmod1$adj<-p.adjust(protmod1$V1,method="hochberg")
protmod1[protmod1$adj<0.05,]
[1] Protein.Group.Accessions param                   
[3] V1                       adj                     
<0 rows> (or 0-length row.names)
```


It looks like there are whol proteome wide changes in stability(unfolding) that are hard to detect(we have low power maybe bc of sample sizes) at the peptide/protein level. 



------

 <div id='id-section71'/> 

### Page 71: 2017-04-26. Climate cascade meeting: Project updates; to do list     


- **Stressed in nature MS: Samples to rerun.**       
	
	* 2 more plates, 18s rRNA; RIN values in
	* Need to analyze data; show results next week.    

	* Get the deltas too to get at the past historical temperatures they were experiencing. Do regression with deltas. 	

- **Proteome stability project:**

	1. There were more peptides identified in pogo than aphaenogaster.     

		* Grab fast file with ORFs from Matt Lau. 
		* And he needs to send me an annotation table (with gene IDs, GO and Kegg)
		* I need to get an annotation table for pogos    
	
	2. One thing to explore is to do an enrichment analysis for proteins that have high Tms
		* Also, if I have aa sequences, I can determine contribution of    
			* length   
			* aa properties or IDs   
		
	3. Searched database for PTMs such as phosphorylation and ubiquitination and found no siginficant modificatoins.    

Notes from SHC and NGotelli:   

1. Do a randomization test: 
	* Shuffle the labels-- go to the colony level   
	* Compare different medians

2. Do KS test to determine difference in cumulative density function

3. Plot density curves

4. Plot Aph Tm vs Pogo Tm  

5. Create story board for manuscript  ; focus on narrative and then construct figures accordingly   

6. Try to analyze by block which might account for variation intraspecific variation      

7. NGotelli comment: Aph Tms are within the range of pogo Tms     







------

 <div id='id-section72'/> 

### Page 72: 2017-04-28. Stressed in nature project: data analysis    

Quick and dirty: ants were sampled at warming chambers for 2 sites; duke forest and harvard forest.     

Looking at bait temperatures against targeted delta chambers  

![](https://cloud.githubusercontent.com/assets/4654474/25533286/275388ee-2bfe-11e7-832e-00562158866d.jpeg)

```R
summary(lm(baittemp.ave~Delta*Site,data=warm))

Call:
lm(formula = baittemp.ave ~ Delta * Site, data = warm)

Residuals:
    Min      1Q  Median      3Q     Max 
-4.2807 -2.0306 -0.2349  1.5072  5.8855 

Coefficients:
             Estimate Std. Error t value Pr(>|t|)    
(Intercept)  25.94661    0.28228  91.919   <2e-16 ***
Delta         0.21358    0.09977   2.141   0.0333 *  
SiteHF        0.31354    0.53701   0.584   0.5599    
Delta:SiteHF  0.24087    0.16926   1.423   0.1561    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 2.417 on 230 degrees of freedom
Multiple R-squared:  0.107,	Adjusted R-squared:  0.09532 
F-statistic: 9.183 on 3 and 230 DF,  p-value: 9.19e-06

```

The lines are not sig diff. Delta and bait temp are weakly correlated too. 

### Getting gene counts from [MCMC.qpcr](http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0071448)

Tutorial [here](http://journals.plos.org/plosone/article/file?type=supplementary&id=info:doi/10.1371/journal.pone.0071448.s002)

getting gene counts   

```R
gene<-c("CT_18s","CT_40","CT_70","CT_83","CT_actin") # list the genes
amp<-data.frame(gene,efficiency=rep(2,length(gene))) # input efficiencies
names(warm)[6]<-"sample" # rename sample


### getting gene counts  
dd<-cq2counts(data=warm,genecols=c(14:18),condcols=c(13,5:6,2:4,20:21),Cq1=37,effic=amp)
dd$baittemp.ave<-as.numeric(as.character(dd$baittemp.ave))
dd$RIN_Value<-as.numeric(as.character(dd$RIN_Value))
dd$Year_collect<-as.numeric(as.character(dd$Year_collect))
str(dd)

'data.frame':	1135 obs. of  11 variables:
 $ count         : num  713721 1704598 2884720 444863 1145861 ...
 $ gene          : Factor w/ 5 levels "CT_18s","CT_40",..: 1 1 1 1 1 1 1 1 1 1 ...
 $ Isolation.Date: Factor w/ 29 levels "20150624","20150710",..: 1 2 19 7 18 11 3 3 15 14 ...
 $ Cham          : Factor w/ 15 levels "1","2","3","4",..: 1 5 13 1 8 11 14 4 15 2 ...
 $ Sample        : Factor w/ 3 levels "1","2","3": 2 2 2 2 2 3 2 2 3 3 ...
 $ Year_collect  : num  2014 2014 2014 2013 2014 ...
 $ Site          : Factor w/ 2 levels "DF","HF": 1 1 1 1 1 2 1 1 1 2 ...
 $ Vial.me       : Factor w/ 238 levels "","DF 1.1","DF 1.2",..: 3 34 14 54 48 153 18 32 77 159 ...
 $ RIN_Value     : num  7 6 5.5 1 2.9 1 1 1.1 2 1 ...
 $ baittemp.ave  : num  28.6 28.1 28.6 29.8 32.7 ...
 $ Siteyear      : chr  "DF 2014" "DF 2014" "DF 2014" "DF 2013" ...
```

### Statistics: Fitting an ANOVA to test interaction between gene, year, baittemp,year collected, and delta. 


```R
fullmod<-aov(log(count+1)~RIN_Value+gene*Site*Year_collect*baittemp.ave*Delta,data=dd)
summary(stepAIC(fullmod,direction="both"))

Step:  AIC=1508.51
log(count + 1) ~ RIN_Value + gene + Site + Year_collect + baittemp.ave + 
    Delta + gene:Site + gene:Year_collect + Site:Year_collect + 
    gene:baittemp.ave + Site:baittemp.ave + Year_collect:baittemp.ave + 
    Site:Delta + Year_collect:Delta + baittemp.ave:Delta + gene:Site:Year_collect + 
    gene:Year_collect:baittemp.ave + Site:Year_collect:baittemp.ave + 
    Site:Year_collect:Delta + Site:baittemp.ave:Delta

                                  Df Sum of Sq    RSS    AIC
<none>                                         3678.1 1508.5
- Site:Year_collect:baittemp.ave   1     8.487 3686.6 1509.1
- gene:Site:Year_collect           4    31.985 3710.1 1510.3
- Site:Year_collect:Delta          9    74.585 3752.7 1513.3
+ Year_collect:baittemp.ave:Delta  9    42.338 3635.8 1513.4
+ gene:Site:baittemp.ave           4     5.817 3672.3 1514.7
- gene:Year_collect:baittemp.ave   4    57.795 3735.9 1518.2
- RIN_Value                        1    58.863 3737.0 1524.5
- Site:baittemp.ave:Delta          9   213.129 3891.3 1554.4
+ gene:Delta                      36    53.133 3625.0 1564.0
                                 Df Sum Sq Mean Sq  F value   Pr(>F)    
RIN_Value                         1   2509    2509  714.820  < 2e-16 ***
gene                              4  20041    5010 1427.529  < 2e-16 ***
Site                              1     18      18    5.027 0.025162 *  
Year_collect                      1    663     663  188.912  < 2e-16 ***
baittemp.ave                      1      2       2    0.544 0.460803    
Delta                             9    103      11    3.261 0.000637 ***
gene:Site                         4     71      18    5.031 0.000511 ***
gene:Year_collect                 4    221      55   15.734 1.64e-12 ***
Site:Year_collect                 1     64      64   18.251 2.11e-05 ***
gene:baittemp.ave                 4    150      38   10.706 1.66e-08 ***
Site:baittemp.ave                 1    136     136   38.667 7.25e-10 ***
Year_collect:baittemp.ave         1    973     973  277.328  < 2e-16 ***
Site:Delta                        9    184      20    5.819 6.19e-08 ***
Year_collect:Delta                9    273      30    8.630 1.46e-12 ***
baittemp.ave:Delta                9    146      16    4.613 5.32e-06 ***
gene:Site:Year_collect            4     19       5    1.342 0.252392    
gene:Year_collect:baittemp.ave    4     59      15    4.187 0.002279 ** 
Site:Year_collect:baittemp.ave    1     95      95   27.031 2.41e-07 ***
Site:Year_collect:Delta           9     61       7    1.946 0.042393 *  
Site:baittemp.ave:Delta           9    213      24    6.747 1.89e-09 ***
Residuals                      1048   3678       4                      
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```


### model selection with dredge() function in MuMin package

cant do full model: 

```R
fullmod2<-aov(log(count+1)~RIN_Value+gene+Site*Year_collect*baittemp.ave,data=dd)
a<-dredge(fullmod2)
```

```R
get.models(a,subset=delta<4)

summary(aov(formula = log(count + 1) ~ baittemp.ave + gene + RIN_Value + 
    Site + Year_collect + baittemp.ave:Site + baittemp.ave:Year_collect + 
    Site:Year_collect + baittemp.ave:Site:Year_collect + 1, data = dd))
    
  Df Sum Sq Mean Sq  F value   Pr(>F)    
baittemp.ave                      1     62      62   13.439 0.000258 ***
gene                              4  19917    4979 1072.077  < 2e-16 ***
RIN_Value                         1   2579    2579  555.191  < 2e-16 ***
Site                              1     24      24    5.262 0.021984 *  
Year_collect                      1    650     650  139.901  < 2e-16 ***
baittemp.ave:Site                 1    126     126   27.070 2.33e-07 ***
baittemp.ave:Year_collect         1    838     838  180.393  < 2e-16 ***
Site:Year_collect                 1    171     171   36.839 1.75e-09 ***
baittemp.ave:Site:Year_collect    1    100     100   21.439 4.08e-06 ***
Residuals                      1122   5211       5        
```



### Plotting:

Estimated slope between bait temp and gene count, against years; genes are colored, site are different symbols      

![](https://cloud.githubusercontent.com/assets/4654474/25536233/08f65e30-2c08-11e7-8c01-c54a86f3eed0.jpeg)

Estimated slope between bait temp and gene count, against genes

![](https://cloud.githubusercontent.com/assets/4654474/25541321/329f0002-2c1c-11e7-8b97-58085ac81bcf.jpeg)


### Statistics: 18s 

```R
mod5<-aov(log(count+1)~RIN_Value+Site*Year_collect*baittemp.ave*Delta,data=gxp18s)

Step:  AIC=494.96
log(count + 1) ~ Site + Year_collect + baittemp.ave + Delta + 
    Site:Year_collect + Site:baittemp.ave + Year_collect:baittemp.ave + 
    Site:Delta + Year_collect:Delta + baittemp.ave:Delta + Site:Year_collect:baittemp.ave

                                  Df Sum of Sq    RSS    AIC
<none>                                         1782.5 494.96
+ RIN_Value                        1    14.778 1767.7 495.05
- baittemp.ave:Delta               1    19.913 1802.4 495.52
- Site:Year_collect:baittemp.ave   1    20.865 1803.3 495.64
+ Year_collect:baittemp.ave:Delta  1     5.637 1776.8 496.24
+ Site:Year_collect:Delta          1     3.601 1778.9 496.50
- Year_collect:Delta               1    28.524 1811.0 496.62
+ Site:baittemp.ave:Delta          1     0.017 1782.4 496.96
- Site:Delta                       1    64.748 1847.2 501.17
                                Df Sum Sq Mean Sq F value   Pr(>F)    
Site                             1  111.4   111.4  13.620 0.000283 ***
Year_collect                     1  906.3   906.3 110.844  < 2e-16 ***
baittemp.ave                     1   73.4    73.4   8.982 0.003042 ** 
Delta                            1    0.0     0.0   0.001 0.975781    
Site:Year_collect                1    2.1     2.1   0.258 0.612198    
Site:baittemp.ave                1   58.5    58.5   7.152 0.008053 ** 
Year_collect:baittemp.ave        1  212.9   212.9  26.033 7.29e-07 ***
Site:Delta                       1   65.0    65.0   7.944 0.005269 ** 
Year_collect:Delta               1   50.0    50.0   6.116 0.014160 *  
baittemp.ave:Delta               1   19.1    19.1   2.330 0.128354    
Site:Year_collect:baittemp.ave   1   20.9    20.9   2.552 0.111612    
Residuals                      218 1782.5     8.2                     
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

### Statistics: actin

```R
mod4<-aov(log(count+1)~RIN_Value+Site*Year_collect*baittemp.ave*Delta,data=gxpactin)
Start:  AIC=295.09
log(count + 1) ~ RIN_Value + Site * Year_collect * baittemp.ave * 
    Delta

                                       Df Sum of Sq    RSS    AIC
<none>                                              717.50 295.08
- Site:Year_collect:baittemp.ave:Delta  1    8.0973 725.60 295.62
- RIN_Value                             1   25.0229 742.53 300.83
                                      Df Sum Sq Mean Sq F value   Pr(>F)    
RIN_Value                              1  429.7   429.7 125.168  < 2e-16 ***
Site                                   1    9.8     9.8   2.865  0.09203 .  
Year_collect                           1  110.3   110.3  32.137 4.73e-08 ***
baittemp.ave                           1    3.2     3.2   0.936  0.33439    
Delta                                  1    0.0     0.0   0.008  0.92832    
Site:Year_collect                      1   23.4    23.4   6.824  0.00965 ** 
Site:baittemp.ave                      1   19.5    19.5   5.692  0.01793 *  
Year_collect:baittemp.ave              1  224.0   224.0  65.243 5.21e-14 ***
Site:Delta                             1    2.2     2.2   0.649  0.42141    
Year_collect:Delta                     1   27.2    27.2   7.916  0.00537 ** 
baittemp.ave:Delta                     1    1.0     1.0   0.299  0.58478    
Site:Year_collect:baittemp.ave         1   37.2    37.2  10.835  0.00117 ** 
Site:Year_collect:Delta                1    1.6     1.6   0.479  0.48975    
Site:baittemp.ave:Delta                1    0.0     0.0   0.003  0.95749    
Year_collect:baittemp.ave:Delta        1    0.0     0.0   0.011  0.91665    
Site:Year_collect:baittemp.ave:Delta   1    8.1     8.1   2.359  0.12610    
Residuals                            209  717.5     3.4                     
---
```


### Statistics: hsp70     

```R
mod1<-aov(log(count+1)~RIN_Value+Site*Year_collect*baittemp.ave*Delta,data=gxp70)

Step:  AIC=200.41
log(count + 1) ~ RIN_Value + Site + Year_collect + baittemp.ave + 
    Delta + Site:Year_collect + Site:baittemp.ave + Year_collect:baittemp.ave + 
    Site:Delta + Year_collect:Delta + Site:Year_collect:Delta

                                 Df Sum of Sq    RSS    AIC
<none>                                        494.75 200.41
- Site:baittemp.ave               1     4.583 499.33 200.52
+ baittemp.ave:Delta              1     0.689 494.06 202.09
+ Site:Year_collect:baittemp.ave  1     0.002 494.75 202.40
- Site:Year_collect:Delta         1    15.443 510.19 205.44
- RIN_Value                       1    32.820 527.57 213.11
- Year_collect:baittemp.ave       1   152.325 647.07 259.87
                           Df Sum Sq Mean Sq F value   Pr(>F)    
RIN_Value                   1  353.3   353.3 154.963  < 2e-16 ***
Site                        1   12.4    12.4   5.454 0.020431 *  
Year_collect                1   27.1    27.1  11.898 0.000675 ***
baittemp.ave                1    7.9     7.9   3.449 0.064641 .  
Delta                       1    0.1     0.1   0.053 0.818551    
Site:Year_collect           1    0.6     0.6   0.263 0.608702    
Site:baittemp.ave           1   14.8    14.8   6.502 0.011462 *  
Year_collect:baittemp.ave   1  120.5   120.5  52.858 6.41e-12 ***
Site:Delta                  1    1.2     1.2   0.541 0.463016    
Year_collect:Delta          1   26.7    26.7  11.696 0.000748 ***
Site:Year_collect:Delta     1   15.4    15.4   6.773 0.009890 ** 
Residuals                 217  494.7     2.3                     
---
```

dredging: 

```R
mod1<-aov(log(count+1)~RIN_Value+Site*Year_collect*baittemp.ave*Delta,data=gxp70)
h70d<-dredge(mod1)
summary(model.avg(h70d,delta<4))
(conditional average) 
                                   Estimate Std. Error Adjusted SE z value
(Intercept)                      -4.151e+04  5.773e+03   5.800e+03   7.157
baittemp.ave                      1.506e+03  2.155e+02   2.166e+02   6.954
Delta                            -2.422e+02  1.055e+03   1.058e+03   0.229
RIN_Value                         2.611e-01  6.662e-02   6.697e-02   3.899
SiteHF                            2.422e+03  6.509e+03   6.533e+03   0.371
Year_collect                      2.062e+01  2.867e+00   2.881e+00   7.158
baittemp.ave:Year_collect        -7.480e-01  1.071e-01   1.076e-01   6.954
Delta:SiteHF                     -1.244e+03  4.719e+02   4.745e+02   2.622
Delta:Year_collect                1.202e-01  5.238e-01   5.254e-01   0.229
SiteHF:Year_collect              -1.305e+00  3.347e+00   3.359e+00   0.389
Delta:SiteHF:Year_collect         6.179e-01  2.344e-01   2.357e-01   2.622
baittemp.ave:SiteHF              -6.104e+01  3.187e+02   3.198e+02   0.191
baittemp.ave:Delta               -2.946e+01  6.526e+01   6.549e+01   0.450
baittemp.ave:SiteHF:Year_collect  1.456e-01  3.212e-01   3.225e-01   0.451
baittemp.ave:Delta:Year_collect   4.222e-02  4.320e-02   4.344e-02   0.972
                                 Pr(>|z|)    
(Intercept)                       < 2e-16 ***
baittemp.ave                      < 2e-16 ***
Delta                             0.81891    
RIN_Value                        9.66e-05 ***
SiteHF                            0.71089    
Year_collect                      < 2e-16 ***
baittemp.ave:Year_collect         < 2e-16 ***
Delta:SiteHF                      0.00875 ** 
Delta:Year_collect                0.81900    
SiteHF:Year_collect               0.69760    
Delta:SiteHF:Year_collect         0.00875 ** 
baittemp.ave:SiteHF               0.84862    
baittemp.ave:Delta                0.65281    
baittemp.ave:SiteHF:Year_collect  0.65174    
baittemp.ave:Delta:Year_collect   0.33112    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Relative variable importance: 
                     baittemp.ave Delta RIN_Value Site Year_collect
Importance:          1.00         1.00  1.00      1.00 1.00        
N containing models:   10           10    10        10   10        
                     baittemp.ave:Year_collect Delta:Year_collect
Importance:          1.00                      1.00              
N containing models:   10                        10              
                     Site:Year_collect Delta:Site Delta:Site:Year_collect
Importance:          0.92              0.81       0.81                   
N containing models:    9                 7          7                   
                     baittemp.ave:Site baittemp.ave:Delta
Importance:          0.61              0.28              
N containing models:    7                 4              
                     baittemp.ave:Site:Year_collect
Importance:          0.13                          
N containing models:    2                          
                     baittemp.ave:Delta:Year_collect
Importance:          0.10                           
N containing models:    2  
```

fig showing the bait temp by year interaction    

![](https://cloud.githubusercontent.com/assets/4654474/25542976/c37a7ec0-2c22-11e7-9651-839f47b276f1.jpeg)   

fig showing delta by site by year interaction    

![](https://cloud.githubusercontent.com/assets/4654474/25543090/236cb032-2c23-11e7-924e-2e2120af38fa.jpeg)

### Statistics: hsp40

```R
mod2<-aov(log(count+1)~RIN_Value+Site*Year_collect*baittemp.ave*Delta,data=gxp40)

Step:  AIC=117.76
log(count + 1) ~ RIN_Value + Site + Year_collect + baittemp.ave + 
    Delta + Site:Year_collect + Site:baittemp.ave + Year_collect:baittemp.ave + 
    Site:Delta + Year_collect:Delta + baittemp.ave:Delta + Site:Year_collect:Delta + 
    Year_collect:baittemp.ave:Delta

                                  Df Sum of Sq    RSS    AIC
<none>                                         332.63 117.76
- Site:Year_collect:Delta          1    3.1659 335.79 117.87
+ Site:baittemp.ave:Delta          1    1.0630 331.56 119.05
- RIN_Value                        1    5.0316 337.66 119.10
- Year_collect:baittemp.ave:Delta  1    5.3788 338.00 119.33
+ Site:Year_collect:baittemp.ave   1    0.3536 332.27 119.53
- Site:baittemp.ave                1    7.4288 340.05 120.67
                                 Df Sum Sq Mean Sq F value   Pr(>F)    
RIN_Value                         1  262.9  262.92 164.412  < 2e-16 ***
Site                              1    0.6    0.59   0.368 0.544494    
Year_collect                      1   79.0   78.96  49.376 2.97e-11 ***
baittemp.ave                      1    1.1    1.12   0.699 0.404232    
Delta                             1    0.1    0.07   0.046 0.829526    
Site:Year_collect                 1    7.9    7.94   4.962 0.026977 *  
Site:baittemp.ave                 1   17.3   17.28  10.807 0.001187 ** 
Year_collect:baittemp.ave         1  114.3  114.34  71.500 4.87e-15 ***
Site:Delta                        1    9.8    9.81   6.133 0.014068 *  
Year_collect:Delta                1   22.1   22.13  13.841 0.000256 ***
baittemp.ave:Delta                1    5.2    5.16   3.228 0.073855 .  
Site:Year_collect:Delta           1    4.3    4.26   2.664 0.104143    
Year_collect:baittemp.ave:Delta   1    5.4    5.38   3.363 0.068086 .  
Residuals                       208  332.6    1.60                     
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```


### Statistics: hsp83


```R
mod3<-aov(log(count+1)~RIN_Value+Site*Year_collect*baittemp.ave*Delta,data=gxp83)
Step:  AIC=368.17
log(count + 1) ~ RIN_Value + Site + Year_collect + baittemp.ave + 
    Delta + Site:Year_collect + Site:baittemp.ave + Year_collect:baittemp.ave + 
    Year_collect:Delta + baittemp.ave:Delta + Year_collect:baittemp.ave:Delta

                                  Df Sum of Sq    RSS    AIC
<none>                                         1031.6 368.17
+ Site:Delta                       1     8.143 1023.5 368.37
+ Site:Year_collect:baittemp.ave   1     2.224 1029.4 369.68
- Site:baittemp.ave                1    20.209 1051.8 370.60
- Year_collect:baittemp.ave:Delta  1    31.913 1063.5 373.12
- RIN_Value                        1    58.793 1090.4 378.81
- Site:Year_collect                1    71.242 1102.8 381.40
                                 Df Sum Sq Mean Sq F value   Pr(>F)    
RIN_Value                         1 1073.4  1073.4 224.743  < 2e-16 ***
Site                              1    0.0     0.0   0.001  0.97546    
Year_collect                      1  227.3   227.3  47.590 5.73e-11 ***
baittemp.ave                      1   26.7    26.7   5.595  0.01890 *  
Delta                             1    0.2     0.2   0.038  0.84575    
Site:Year_collect                 1   46.4    46.4   9.723  0.00207 ** 
Site:baittemp.ave                 1   38.9    38.9   8.154  0.00472 ** 
Year_collect:baittemp.ave         1  361.6   361.6  75.718 8.44e-16 ***
Year_collect:Delta                1   55.4    55.4  11.591  0.00079 ***
baittemp.ave:Delta                1    0.3     0.3   0.060  0.80745    
Year_collect:baittemp.ave:Delta   1   31.9    31.9   6.682  0.01040 *  
Residuals                       216 1031.6     4.8                     
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```


------

 <div id='id-section73'/> 

### Page 73: 2017-04-30. Stressed in nature project: Re-analysis     

To analyze all genes into 1 model; using the MCMC.qpcr package:   

some data parsing first:      

```R
> warm<-read.csv("../Data/20170427_final_dataset_2013_2014_years_HF_DF.csv",skip=10)
> warm$baittemp.ave<-apply(warm[,8:11],1,mean,na.rm=TRUE)
> warm<-warm[!is.na(warm$RIN_Value),]
> warm$Cham<-as.factor(warm$Cham)
> warm$Sample<-as.factor(warm$Sample)
> warm$Collection.Date<-as.factor(as.character(warm$Collection.Date))
> chd<-read.csv("../Data/2017_chamber_dat.csv")
> chd$Cham<-as.factor(chd$Cham)
> str(chd)
'data.frame':	30 obs. of  3 variables:
 $ Cham : Factor w/ 15 levels "1","2","3","4",..: 1 2 3 4 5 6 7 8 9 10 ...
 $ Site : Factor w/ 2 levels "DF","HF": 2 2 2 2 2 2 2 2 2 2 ...
 $ Delta: num  5.5 4 2 0 5 0 3 1.5 2.5 4.5 ...
> warm<-inner_join(warm,chd,by=c("Cham","Site"))
```

getting gene counts:    

```R
gene<-c("CT_18s","CT_40","CT_70","CT_83","CT_actin") # list the genes
amp<-data.frame(gene,efficiency=rep(2,length(gene))) # input efficiencies
names(warm)[6]<-"sample"

dd<-cq2counts(data=warm,genecols=c(14:18),condcols=c(13,5:6,1:4,20:22),Cq1=34,effic=amp)

### more data mainpulations; changing between factor/class
dd$baittemp.ave<-as.numeric(as.character(dd$baittemp.ave))
dd$RIN_Value<-as.numeric(as.character(dd$RIN_Value))
dd$Year_collect<-as.numeric(as.character(dd$Year_collect))
dd$Siteyear<-paste(dd$Site,dd$Year_collect) # for plotting purposes
dd$Delta<-as.numeric(as.character(dd$Delta))

``` 

Getting Julian days:    

```R
### messing with dates
dd$Collection.Date<-as.character(dd$Collection.Date)
v<-paste(substr(dd$Collection.Date,1,4),"-",substr(dd$Collection.Date,5,6),"-",substr(dd$Collection.Date,7,8))
dd$Date<-gsub(" ", "", v, fixed = TRUE)
dd$JulianDay<-as.numeric(format(as.Date(dd$Date),"%j"))

ddply(dd,.(Year_collect),summarize,max=max(JulianDay)) ## finding the latest julian day

dd$Jdaycont<-ifelse(dd$Year_collect==2013,dd$JulianDay,dd$JulianDay+253) ## adding 
#latest julian day to make it continuous
```

### Statistics: specifying bayesian model to include all genes, no control genes

```R
mm=mcmc.qpcr(
fixed="RIN_Value+Jdaycont*Site*baittemp.ave*Delta",
random="Vial.me",
data=dd)
```

### output

```R
 Iterations = 3001:12991
 Thinning interval  = 10
 Sample size  = 1000 

 DIC: 13001.7 

 G-structure:  ~sample

       post.mean  l-95% CI u-95% CI eff.samp
sample    0.0292 1.741e-43  0.07663    608.2

               ~idh(gene):Vial.me

                     post.mean  l-95% CI u-95% CI eff.samp
geneCT_18s.Vial.me    0.001257 3.491e-16 0.004498    40.48
geneCT_40.Vial.me     0.027512 2.233e-07 0.126707    41.07
geneCT_70.Vial.me     0.001494 1.029e-13 0.009846    36.60
geneCT_83.Vial.me     0.005245 1.123e-09 0.021233    40.93
geneCT_actin.Vial.me  0.003338 4.250e-08 0.020209    25.33

 R-structure:  ~idh(gene):units

                   post.mean l-95% CI u-95% CI eff.samp
geneCT_18s.units       8.187   6.7688    9.743   1000.0
geneCT_40.units        1.162   0.9151    1.435    271.7
geneCT_70.units        2.078   1.7054    2.483   1000.0
geneCT_83.units        4.406   3.6245    5.385    208.8
geneCT_actin.units     3.114   2.5009    3.740    809.4

 Location effects: count ~ 0 + gene + +gene:RIN_Value + gene:Jdaycont * Site * baittemp.ave * Delta 

                                                 post.mean   l-95% CI   u-95% CI eff.samp
geneCT_18s                                      -2.133e+01 -2.668e+01 -1.618e+01   1121.3
geneCT_40                                       -3.130e+01 -3.629e+01 -2.598e+01   1150.3
geneCT_70                                       -2.838e+01 -3.349e+01 -2.302e+01   1139.9
geneCT_83                                       -3.257e+01 -3.746e+01 -2.696e+01   1155.2
geneCT_actin                                    -3.005e+01 -3.514e+01 -2.487e+01   1159.6
SiteHF                                           8.902e+00 -8.067e+00  2.217e+01   1000.0
baittemp.ave                                     1.260e+00  1.054e+00  1.453e+00   1147.9
Delta                                            1.813e+00  2.477e-01  3.300e+00   1000.0
geneCT_18s:RIN_Value                             1.818e-01 -5.029e-02  4.004e-01   1000.0
geneCT_40:RIN_Value                              1.044e-01  1.507e-02  2.070e-01    458.6
geneCT_70:RIN_Value                              2.444e-01  1.194e-01  3.594e-01   1000.0
geneCT_83:RIN_Value                              2.985e-01  1.199e-01  4.684e-01   1000.0
geneCT_actin:RIN_Value                           1.592e-01  1.092e-02  3.006e-01   1000.0
geneCT_18s:Jdaycont                              1.614e-01  1.377e-01  1.902e-01   1000.0
geneCT_40:Jdaycont                               1.152e-01  9.677e-02  1.310e-01   1000.0
geneCT_70:Jdaycont                               1.088e-01  9.180e-02  1.295e-01   1000.0
geneCT_83:Jdaycont                               1.274e-01  1.028e-01  1.507e-01   1000.0
geneCT_actin:Jdaycont                            1.258e-01  1.065e-01  1.466e-01   1000.0
SiteHF:baittemp.ave                             -3.373e-01 -8.651e-01  2.879e-01   1000.0
SiteHF:Delta                                    -1.691e+00 -5.332e+00  2.575e+00    912.4
baittemp.ave:Delta                              -7.949e-02 -1.408e-01 -2.348e-02   1000.0
geneCT_18s:Jdaycont:SiteHF                      -6.293e-02 -1.508e-01  2.361e-02   1000.0
geneCT_40:Jdaycont:SiteHF                       -3.240e-02 -8.934e-02  2.572e-02   1000.0
geneCT_70:Jdaycont:SiteHF                       -4.520e-02 -1.026e-01  1.781e-02   1000.0
geneCT_83:Jdaycont:SiteHF                       -6.173e-02 -1.316e-01  1.224e-02   1000.0
geneCT_actin:Jdaycont:SiteHF                    -5.739e-02 -1.235e-01  1.120e-02   1000.0
geneCT_18s:Jdaycont:baittemp.ave                -5.828e-03 -6.852e-03 -4.893e-03   1000.0
geneCT_40:Jdaycont:baittemp.ave                 -4.200e-03 -4.832e-03 -3.531e-03   1000.0
geneCT_70:Jdaycont:baittemp.ave                 -4.032e-03 -4.785e-03 -3.365e-03   1104.7
geneCT_83:Jdaycont:baittemp.ave                 -4.441e-03 -5.307e-03 -3.522e-03   1000.0
geneCT_actin:Jdaycont:baittemp.ave              -4.509e-03 -5.234e-03 -3.713e-03   1000.0
geneCT_18s:Jdaycont:Delta                       -1.136e-02 -1.954e-02 -3.116e-03   1000.0
geneCT_40:Jdaycont:Delta                        -7.652e-03 -1.244e-02 -2.561e-03   1000.0
geneCT_70:Jdaycont:Delta                        -6.787e-03 -1.287e-02 -1.802e-03   1000.0
geneCT_83:Jdaycont:Delta                        -6.808e-03 -1.368e-02 -3.982e-04    890.1
geneCT_actin:Jdaycont:Delta                     -4.889e-03 -1.087e-02  1.344e-03    747.7
SiteHF:baittemp.ave:Delta                        4.847e-02 -9.596e-02  1.992e-01   1000.0
geneCT_18s:Jdaycont:SiteHF:baittemp.ave          2.659e-03 -6.000e-04  5.992e-03   1000.0
geneCT_40:Jdaycont:SiteHF:baittemp.ave           1.274e-03 -8.384e-04  3.507e-03   1000.0
geneCT_70:Jdaycont:SiteHF:baittemp.ave           1.723e-03 -6.570e-04  3.918e-03   1000.0
geneCT_83:Jdaycont:SiteHF:baittemp.ave           2.440e-03 -1.164e-04  5.299e-03   1000.0
geneCT_actin:Jdaycont:SiteHF:baittemp.ave        2.146e-03 -4.782e-04  4.648e-03   1000.0
geneCT_18s:Jdaycont:SiteHF:Delta                 4.693e-03 -1.550e-02  2.612e-02   1018.7
geneCT_40:Jdaycont:SiteHF:Delta                  1.524e-03 -1.189e-02  1.431e-02    882.5
geneCT_70:Jdaycont:SiteHF:Delta                  7.593e-03 -8.370e-03  2.111e-02   1000.0
geneCT_83:Jdaycont:SiteHF:Delta                  2.915e-03 -1.576e-02  1.984e-02   1000.0
geneCT_actin:Jdaycont:SiteHF:Delta               5.448e-03 -1.136e-02  2.111e-02   1000.0
geneCT_18s:Jdaycont:baittemp.ave:Delta           4.781e-04  1.641e-04  7.779e-04   1000.0
geneCT_40:Jdaycont:baittemp.ave:Delta            3.269e-04  1.358e-04  5.056e-04   1000.0
geneCT_70:Jdaycont:baittemp.ave:Delta            2.900e-04  1.029e-04  5.131e-04   1000.0
geneCT_83:Jdaycont:baittemp.ave:Delta            3.028e-04  5.222e-05  5.459e-04    904.2
geneCT_actin:Jdaycont:baittemp.ave:Delta         2.241e-04  9.050e-06  4.616e-04    749.3
geneCT_18s:Jdaycont:SiteHF:baittemp.ave:Delta   -1.847e-04 -9.327e-04  6.063e-04   1000.0
geneCT_40:Jdaycont:SiteHF:baittemp.ave:Delta    -2.955e-05 -5.167e-04  4.537e-04    881.9
geneCT_70:Jdaycont:SiteHF:baittemp.ave:Delta    -2.312e-04 -7.787e-04  3.257e-04   1000.0
geneCT_83:Jdaycont:SiteHF:baittemp.ave:Delta    -9.858e-05 -6.929e-04  6.199e-04   1000.0
geneCT_actin:Jdaycont:SiteHF:baittemp.ave:Delta -1.592e-04 -7.268e-04  4.714e-04   1000.0
                                                 pMCMC    
geneCT_18s                                      <0.001 ***
geneCT_40                                       <0.001 ***
geneCT_70                                       <0.001 ***
geneCT_83                                       <0.001 ***
geneCT_actin                                    <0.001 ***
SiteHF                                           0.232    
baittemp.ave                                    <0.001 ***
Delta                                            0.024 *  
geneCT_18s:RIN_Value                             0.122    
geneCT_40:RIN_Value                              0.040 *  
geneCT_70:RIN_Value                             <0.001 ***
geneCT_83:RIN_Value                              0.002 ** 
geneCT_actin:RIN_Value                           0.032 *  
geneCT_18s:Jdaycont                             <0.001 ***
geneCT_40:Jdaycont                              <0.001 ***
geneCT_70:Jdaycont                              <0.001 ***
geneCT_83:Jdaycont                              <0.001 ***
geneCT_actin:Jdaycont                           <0.001 ***
SiteHF:baittemp.ave                              0.236    
SiteHF:Delta                                     0.420    
baittemp.ave:Delta                               0.014 *  
geneCT_18s:Jdaycont:SiteHF                       0.142    
geneCT_40:Jdaycont:SiteHF                        0.282    
geneCT_70:Jdaycont:SiteHF                        0.134    
geneCT_83:Jdaycont:SiteHF                        0.100 .  
geneCT_actin:Jdaycont:SiteHF                     0.110    
geneCT_18s:Jdaycont:baittemp.ave                <0.001 ***
geneCT_40:Jdaycont:baittemp.ave                 <0.001 ***
geneCT_70:Jdaycont:baittemp.ave                 <0.001 ***
geneCT_83:Jdaycont:baittemp.ave                 <0.001 ***
geneCT_actin:Jdaycont:baittemp.ave              <0.001 ***
geneCT_18s:Jdaycont:Delta                        0.004 ** 
geneCT_40:Jdaycont:Delta                         0.002 ** 
geneCT_70:Jdaycont:Delta                         0.020 *  
geneCT_83:Jdaycont:Delta                         0.044 *  
geneCT_actin:Jdaycont:Delta                      0.146    
SiteHF:baittemp.ave:Delta                        0.540    
geneCT_18s:Jdaycont:SiteHF:baittemp.ave          0.098 .  
geneCT_40:Jdaycont:SiteHF:baittemp.ave           0.266    
geneCT_70:Jdaycont:SiteHF:baittemp.ave           0.130    
geneCT_83:Jdaycont:SiteHF:baittemp.ave           0.080 .  
geneCT_actin:Jdaycont:SiteHF:baittemp.ave        0.114    
geneCT_18s:Jdaycont:SiteHF:Delta                 0.688    
geneCT_40:Jdaycont:SiteHF:Delta                  0.840    
geneCT_70:Jdaycont:SiteHF:Delta                  0.330    
geneCT_83:Jdaycont:SiteHF:Delta                  0.728    
geneCT_actin:Jdaycont:SiteHF:Delta               0.520    
geneCT_18s:Jdaycont:baittemp.ave:Delta          <0.001 ***
geneCT_40:Jdaycont:baittemp.ave:Delta           <0.001 ***
geneCT_70:Jdaycont:baittemp.ave:Delta            0.010 *  
geneCT_83:Jdaycont:baittemp.ave:Delta            0.016 *  
geneCT_actin:Jdaycont:baittemp.ave:Delta         0.068 .  
geneCT_18s:Jdaycont:SiteHF:baittemp.ave:Delta    0.654    
geneCT_40:Jdaycont:SiteHF:baittemp.ave:Delta     0.902    
geneCT_70:Jdaycont:SiteHF:baittemp.ave:Delta     0.430    
geneCT_83:Jdaycont:SiteHF:baittemp.ave:Delta     0.756    
geneCT_actin:Jdaycont:SiteHF:baittemp.ave:Delta  0.620    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

"control genes" are not stable...or constant across treatments    

model diagnostics    

![](https://cloud.githubusercontent.com/assets/4654474/25568339/9e962e6c-2dce-11e7-8992-f83ba65ba21f.jpeg)
  
  
### Statistics: fitting with control genes 

```R
mm2=mcmc.qpcr(
fixed="RIN_Value+Jdaycont*Site*baittemp.ave*Delta",
random="Vial.me", controls=c("CT_18s","CTactin"),
data=dd)
summary(mm2)
```
  
Output:   

```R
 Iterations = 3001:12991
 Thinning interval  = 10
 Sample size  = 1000 

 DIC: 12982.46 

 G-structure:  ~sample

       post.mean  l-95% CI  u-95% CI eff.samp
sample   0.06628 2.181e-54 6.863e-05     1000

               ~idh(gene):Vial.me

                     post.mean  l-95% CI u-95% CI eff.samp
geneCT_40.Vial.me      0.01051 5.492e-08  0.06105    13.28
geneCT_70.Vial.me      0.22941 1.663e-04  0.78709    37.78
geneCT_83.Vial.me      0.59330 4.545e-07  2.71722    11.88
geneCT_actin.Vial.me   0.27245 1.185e-06  1.28528    13.71
geneCT_18s.Vial.me     0.03488 4.961e-20  0.06262    28.64

 R-structure:  ~idh(gene):units

                   post.mean l-95% CI u-95% CI eff.samp
geneCT_40.units        1.190   0.9469    1.459   587.04
geneCT_70.units        1.871   1.2816    2.486    79.41
geneCT_83.units        3.884   1.7517    5.411    21.65
geneCT_actin.units     2.852   1.7793    3.839    23.67
geneCT_18s.units       8.109   6.6364    9.655   908.67

 Location effects: count ~ 0 + gene + +gene:RIN_Value + gene:Jdaycont * Site * baittemp.ave * Delta 

                                                 post.mean   l-95% CI   u-95% CI eff.samp  pMCMC    
geneCT_40                                       -3.107e+01 -3.559e+01 -2.616e+01   1000.0 <0.001 ***
geneCT_70                                       -2.817e+01 -3.312e+01 -2.354e+01   1022.4 <0.001 ***
geneCT_83                                       -3.231e+01 -3.692e+01 -2.722e+01   1000.0 <0.001 ***
geneCT_actin                                    -2.982e+01 -3.456e+01 -2.496e+01   1057.2 <0.001 ***
geneCT_18s                                      -2.112e+01 -2.561e+01 -1.617e+01   1000.0 <0.001 ***
SiteHF                                           8.508e+00 -6.382e+00  2.209e+01   1000.0  0.250    
baittemp.ave                                     1.251e+00  1.050e+00  1.419e+00   1000.0 <0.001 ***
Delta                                            1.712e+00  2.057e-01  3.224e+00   1000.0  0.034 *  
geneCT_40:RIN_Value                              9.921e-02 -4.455e-03  1.961e-01   1000.0  0.062 .  
geneCT_70:RIN_Value                              2.465e-01  1.478e-01  3.764e-01   1000.0 <0.001 ***
geneCT_83:RIN_Value                              2.911e-01  1.060e-01  4.576e-01   1000.0  0.002 ** 
geneCT_actin:RIN_Value                           1.520e-01  1.671e-03  2.945e-01   1000.0  0.050 .  
geneCT_18s:RIN_Value                             1.747e-01 -4.381e-02  4.105e-01   1000.0  0.134    
geneCT_40:Jdaycont                               1.145e-01  9.705e-02  1.315e-01    979.5 <0.001 ***
geneCT_70:Jdaycont                               1.083e-01  8.998e-02  1.263e-01   1000.0 <0.001 ***
geneCT_83:Jdaycont                               1.272e-01  1.059e-01  1.501e-01    843.5 <0.001 ***
geneCT_actin:Jdaycont                            1.255e-01  1.066e-01  1.468e-01   1103.4 <0.001 ***
geneCT_18s:Jdaycont                              1.609e-01  1.345e-01  1.875e-01    912.4 <0.001 ***
SiteHF:baittemp.ave                             -3.221e-01 -8.854e-01  2.107e-01   1000.0  0.248    
SiteHF:Delta                                    -1.603e+00 -5.566e+00  1.967e+00    890.2  0.404    
baittemp.ave:Delta                              -7.549e-02 -1.310e-01 -1.567e-02   1000.0  0.010 ** 
geneCT_40:Jdaycont:SiteHF                       -2.990e-02 -8.217e-02  2.259e-02    967.4  0.298    
geneCT_70:Jdaycont:SiteHF                       -4.375e-02 -1.042e-01  1.393e-02   1000.0  0.134    
geneCT_83:Jdaycont:SiteHF                       -6.178e-02 -1.275e-01  1.679e-02   1000.0  0.096 .  
geneCT_actin:Jdaycont:SiteHF                    -5.675e-02 -1.209e-01  6.528e-03   1000.0  0.076 .  
geneCT_18s:Jdaycont:SiteHF                      -5.912e-02 -1.412e-01  2.562e-02   1000.0  0.172    
geneCT_40:Jdaycont:baittemp.ave                 -4.167e-03 -4.829e-03 -3.517e-03   1000.0 <0.001 ***
geneCT_70:Jdaycont:baittemp.ave                 -4.012e-03 -4.651e-03 -3.302e-03   1000.0 <0.001 ***
geneCT_83:Jdaycont:baittemp.ave                 -4.431e-03 -5.231e-03 -3.613e-03   1000.0 <0.001 ***
geneCT_actin:Jdaycont:baittemp.ave              -4.495e-03 -5.236e-03 -3.742e-03   1000.0 <0.001 ***
geneCT_18s:Jdaycont:baittemp.ave                -5.801e-03 -6.805e-03 -4.873e-03    874.7 <0.001 ***
geneCT_40:Jdaycont:Delta                        -7.386e-03 -1.297e-02 -2.928e-03   1000.0  0.004 ** 
geneCT_70:Jdaycont:Delta                        -6.513e-03 -1.160e-02 -5.976e-04   1000.0  0.016 *  
geneCT_83:Jdaycont:Delta                        -6.738e-03 -1.344e-02 -1.580e-04   1000.0  0.050 *  
geneCT_actin:Jdaycont:Delta                     -4.677e-03 -1.078e-02  9.420e-04   1000.0  0.126    
geneCT_18s:Jdaycont:Delta                       -1.088e-02 -1.775e-02 -2.091e-03    913.1  0.006 ** 
SiteHF:baittemp.ave:Delta                        4.473e-02 -9.274e-02  1.865e-01    883.6  0.540    
geneCT_40:Jdaycont:SiteHF:baittemp.ave           1.180e-03 -8.438e-04  3.177e-03    953.9  0.268    
geneCT_70:Jdaycont:SiteHF:baittemp.ave           1.667e-03 -4.613e-04  4.046e-03   1000.0  0.146    
geneCT_83:Jdaycont:SiteHF:baittemp.ave           2.443e-03 -5.353e-04  4.971e-03   1000.0  0.080 .  
geneCT_actin:Jdaycont:SiteHF:baittemp.ave        2.120e-03 -2.446e-04  4.658e-03   1000.0  0.080 .  
geneCT_18s:Jdaycont:SiteHF:baittemp.ave          2.511e-03 -7.564e-04  5.644e-03   1000.0  0.138    
geneCT_40:Jdaycont:SiteHF:Delta                  1.144e-03 -1.164e-02  1.390e-02    911.9  0.846    
geneCT_70:Jdaycont:SiteHF:Delta                  7.344e-03 -6.505e-03  2.243e-02    780.1  0.320    
geneCT_83:Jdaycont:SiteHF:Delta                  2.850e-03 -1.492e-02  1.992e-02    706.7  0.758    
geneCT_actin:Jdaycont:SiteHF:Delta               5.225e-03 -1.075e-02  2.028e-02    904.6  0.516    
geneCT_18s:Jdaycont:SiteHF:Delta                 3.949e-03 -1.684e-02  2.472e-02   1118.7  0.712    
geneCT_40:Jdaycont:baittemp.ave:Delta            3.160e-04  1.453e-04  5.166e-04   1000.0  0.002 ** 
geneCT_70:Jdaycont:baittemp.ave:Delta            2.795e-04  7.007e-05  4.703e-04   1000.0  0.006 ** 
geneCT_83:Jdaycont:baittemp.ave:Delta            2.992e-04  6.755e-05  5.555e-04   1000.0  0.020 *  
geneCT_actin:Jdaycont:baittemp.ave:Delta         2.154e-04 -1.270e-05  4.238e-04   1000.0  0.048 *  
geneCT_18s:Jdaycont:baittemp.ave:Delta           4.596e-04  1.451e-04  7.155e-04    905.2  0.004 ** 
geneCT_40:Jdaycont:SiteHF:baittemp.ave:Delta    -1.369e-05 -4.662e-04  4.803e-04    896.1  0.958    
geneCT_70:Jdaycont:SiteHF:baittemp.ave:Delta    -2.208e-04 -7.666e-04  2.911e-04    807.4  0.426    
geneCT_83:Jdaycont:SiteHF:baittemp.ave:Delta    -9.532e-05 -6.881e-04  5.898e-04    696.0  0.790    
geneCT_actin:Jdaycont:SiteHF:baittemp.ave:Delta -1.489e-04 -7.156e-04  4.260e-04    899.7  0.634    
geneCT_18s:Jdaycont:SiteHF:baittemp.ave:Delta   -1.549e-04 -9.061e-04  6.138e-04   1116.7  0.702    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```  
  
### Statistics: "Classic" model (delta delta ct) with control genes 


```R
summary(classicmod)

 Iterations = 3001:12991
 Thinning interval  = 10
 Sample size  = 1000 

 DIC: 4920.499 

 G-structure:  ~idh(gene):Vial.me

                     post.mean  l-95% CI u-95% CI eff.samp
geneCT_40.Vial.me     0.005448 2.373e-14  0.02023    39.20
geneCT_70.Vial.me     0.009244 1.900e-09  0.06938    15.33
geneCT_83.Vial.me     3.759275 2.403e+00  5.27956   795.80
geneCT_actin.Vial.me  1.428124 4.243e-01  2.41822   297.78
geneCT_18s.Vial.me    6.480444 4.572e+00  8.27869   882.87

               ~idh(gene):sample

                    post.mean   l-95% CI  u-95% CI eff.samp
geneCT_40.sample    7.525e-04  5.851e-59 6.254e-08   1000.0
geneCT_70.sample    1.611e-02  2.763e-59 2.056e-02    808.5
geneCT_83.sample    2.277e-40 2.463e-138 1.892e-46      0.0
geneCT_actin.sample 8.960e-04  9.595e-62 3.760e-07    488.2
geneCT_18s.sample   2.551e-02  5.330e-65 1.510e-02    812.1

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units     3.251    2.838    3.664    662.3

 Location effects: count ~ 0 + gene + +gene:RIN_Value + gene:Site * Jdaycont * Delta 

                                    post.mean   l-95% CI   u-95% CI eff.samp  pMCMC    
geneCT_40                          -2.6284204 -3.7176141 -1.4323167    879.9 <0.001 ***
geneCT_70                          -3.2528194 -4.3525418 -2.0888514   1000.0 <0.001 ***
geneCT_83                          -5.3863914 -7.0044499 -3.6856766   1000.0 <0.001 ***
geneCT_actin                       -3.5034426 -4.7385966 -2.0511151   1150.8 <0.001 ***
geneCT_18s                         -4.1354609 -6.0388715 -2.1572579   1000.0 <0.001 ***
Jdaycont                            0.0069301 -0.0022933  0.0161707   1000.0  0.156    
Delta                              -1.0012178 -1.7488433  0.0055000   1000.0  0.028 *  
geneCT_40:RIN_Value                 0.2189257  0.0713273  0.3608404   1000.0  0.004 ** 
geneCT_70:RIN_Value                 0.3380169  0.1844881  0.4760341   1000.0 <0.001 ***
geneCT_83:RIN_Value                 0.4467853  0.2559811  0.6592500   1000.0 <0.001 ***
geneCT_actin:RIN_Value              0.3079191  0.1534536  0.4931644   1000.0 <0.001 ***
geneCT_18s:RIN_Value                0.3417462  0.1006646  0.5934699   1000.0  0.008 ** 
geneCT_40:SiteHF                   -0.1231226 -1.9166395  2.0395052   1000.0  0.918    
geneCT_70:SiteHF                    1.5101947 -0.4924095  3.4462324    892.6  0.126    
geneCT_83:SiteHF                   -1.1804353 -4.2953270  1.6625582   1000.0  0.496    
geneCT_actin:SiteHF                -0.1028772 -2.7037113  2.3326229    870.1  0.922    
geneCT_18s:SiteHF                   2.1563027 -1.3683731  5.5980660   1000.0  0.244    
Jdaycont:Delta                      0.0020503 -0.0008895  0.0044592   1000.0  0.132    
geneCT_40:SiteDF:Jdaycont          -0.0028295 -0.0141627  0.0061651   1000.0  0.592    
geneCT_70:SiteDF:Jdaycont          -0.0029284 -0.0130408  0.0068426   1000.0  0.584    
geneCT_83:SiteDF:Jdaycont           0.0019250 -0.0073718  0.0139120   1000.0  0.722    
geneCT_actin:SiteDF:Jdaycont       -0.0004737 -0.0103707  0.0100169   1000.0  0.944    
geneCT_18s:SiteDF:Jdaycont         -0.0012918 -0.0123587  0.0089910   1000.0  0.822    
geneCT_40:SiteHF:Jdaycont          -0.0006130 -0.0113985  0.0103346   1000.0  0.894    
geneCT_70:SiteHF:Jdaycont          -0.0062379 -0.0176881  0.0048022   1000.0  0.280    
geneCT_83:SiteHF:Jdaycont           0.0080759 -0.0042720  0.0206776   1000.0  0.214    
geneCT_actin:SiteHF:Jdaycont       -0.0006060 -0.0120543  0.0114044   1000.0  0.964    
geneCT_40:SiteDF:Delta              0.8353265 -0.0849807  1.7891522   1000.0  0.090 .  
geneCT_70:SiteDF:Delta              0.9272206 -0.1068245  1.8419725   1000.0  0.066 .  
geneCT_83:SiteDF:Delta              0.8385269 -0.2175849  1.8485020   1000.0  0.118    
geneCT_actin:SiteDF:Delta           1.0904933 -0.0078081  1.9762069   1000.0  0.032 *  
geneCT_18s:SiteDF:Delta             0.7533491 -0.3674505  1.8339046   1000.0  0.188    
geneCT_40:SiteHF:Delta              0.6427846 -0.3309434  1.6253035   1000.0  0.222    
geneCT_70:SiteHF:Delta              0.5351106 -0.4302433  1.5825929   1000.0  0.298    
geneCT_83:SiteHF:Delta              0.7188371 -0.3859249  1.9097256   1000.0  0.230    
geneCT_actin:SiteHF:Delta           0.5040863 -0.6524067  1.4469885   1000.0  0.362    
geneCT_40:SiteDF:Jdaycont:Delta    -0.0014557 -0.0041871  0.0013708   1000.0  0.284    
geneCT_70:SiteDF:Jdaycont:Delta    -0.0018350 -0.0043511  0.0014005   1000.0  0.214    
geneCT_83:SiteDF:Jdaycont:Delta    -0.0013770 -0.0046490  0.0014063   1000.0  0.378    
geneCT_actin:SiteDF:Jdaycont:Delta -0.0023920 -0.0055308  0.0004937   1000.0  0.126    
geneCT_18s:SiteDF:Jdaycont:Delta   -0.0010735 -0.0043216  0.0021393   1000.0  0.494    
geneCT_40:SiteHF:Jdaycont:Delta    -0.0010914 -0.0040039  0.0019467   1000.0  0.496    
geneCT_70:SiteHF:Jdaycont:Delta    -0.0003770 -0.0032484  0.0029631   1000.0  0.820    
geneCT_83:SiteHF:Jdaycont:Delta    -0.0012333 -0.0048577  0.0022097   1000.0  0.498    
geneCT_actin:SiteHF:Jdaycont:Delta -0.0004587 -0.0036330  0.0028876   1000.0  0.764    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

```


Yeah non of this is right...vial.me should not be a random effect. Vial.me should be labeled as the "sample" .       

Also, how do I deal with the fact that the chambers are repeatedly sampled? I have to include a repeated measures aspect into model.  


Also the model looks so complicated. What am I testing?  

Main effects: Rin_VAlue - to control for quality 

Interactions: 

3 way between, Site*Julianday* Delta    

3 way between, Site*Julianday* bait temperature     

Other varibles: 
1. collection date     
2. Year collected    


------

 <div id='id-section74'/> 

### Page 74: 2017-05-01 & 2017-05-02. Stressed in nature project: re-analysis   

**I dont have enough samples per chamber to include it as a random effect.**    

```R
sumda<-ddply(dd,.(Site,Collection.Date,Delta,Cham,gene),summarize,num=length(Date))
> range(sumda$num)
[1] 1 3
```
I'll need to average the genes.   

```R
ddave<-ddply(dd,.(Site,Year_collect,Collection.Date,Delta,gene,Cham),summarize,count=mean(count),RIN_Value=mean(RIN_Value),baittemp.ave=mean(baittemp.ave))
'data.frame':	454 obs. of  9 variables:
 $ Site           : Factor w/ 2 levels "DF","HF": 1 1 1 1 1 1 1 1 1 1 ...
 $ Year_collect   : num  2013 2013 2013 2013 2013 ...
 $ Collection.Date: Factor w/ 7 levels "20130626","20130702",..: 2 2 2 2 2 2 2 2 2 2 ...
 $ Delta          : num  0 0 0 0 0 0 0 0 0 0 ...
 $ gene           : Factor w/ 5 levels "CT_18s","CT_40",..: 1 1 1 1 1 1 2 2 2 2 ...
 $ Cham           : Factor w/ 15 levels "1","2","3","4",..: 2 5 11 13 14 15 2 5 11 13 ...
 $ count          : num  1172618 579994 1640821 11910265 737550 ...
 $ RIN_Value      : num  4.9 2.87 2.7 2.67 1.87 ...
 $ baittemp.ave   : num  23.2 23.8 23.3 23.4 23.8 ...
```

### Figures for environmental conditions   

bait temp vs julian day

![](https://cloud.githubusercontent.com/assets/4654474/25621287/fb2ee67c-2f1e-11e7-9262-656fbe5689ba.jpeg)

plot with bait temp vs delta; colored by year, collection date, site

![](https://cloud.githubusercontent.com/assets/4654474/25621567/ba89ccee-2f1f-11e7-894a-a975a97d7b46.jpeg)   

![](https://cloud.githubusercontent.com/assets/4654474/25621926/dd38a1b0-2f20-11e7-9db7-39aac71568ec.jpeg)

Models don't converge when testing for ...  

```R
mod22<-glm(log(count+1)~RIN_Value+Site*baittemp.ave*Year_collect*Delta,family=poisson,data=gxp70)
```


Parsing out hsp70 to visualize:   

###  counts vs bait temp colored by year

![](https://cloud.githubusercontent.com/assets/4654474/25594598/44816218-2e8f-11e7-9717-b269d58f9c7a.jpeg)  

### counts vs bait temp colored by site and colleciton

![](https://cloud.githubusercontent.com/assets/4654474/25594657/86396e44-2e8f-11e7-8fe9-cfed62bc6c13.jpeg)

### coutns vs julian day colored by site

![](https://cloud.githubusercontent.com/assets/4654474/25595035/4c25a630-2e91-11e7-9a9f-ac22670cb80a.jpeg)


# OK SHOWING ALL GENES   

### counts vs bait   

```R
ggplot(dd,aes(x=baittemp.ave,y=log(count+1),colour=paste(Year_collect,Collection.Date,Site)))+geom_point(size=0.5)+stat_smooth(size=3,method="lm",se=FALSE)+facet_grid(.~gene) 
```

![](https://cloud.githubusercontent.com/assets/4654474/25622159/8c3fccce-2f21-11e7-8606-677623552d5a.jpeg)


### counts vs delta    

```R
ggplot(dd,aes(x=Delta,y=log(count+1),colour=paste(Year_collect,Collection.Date,Site)))+geom_point(size=0.5)+stat_smooth(size=3,method="lm",se=FALSE)+facet_grid(.~gene)
```

![](https://cloud.githubusercontent.com/assets/4654474/25622267/e1302436-2f21-11e7-977d-60c4fa7db45c.jpeg)    


### coutns vs julian day    

numbers for each collection day
```R
summary(factor(dd$Jdaycont))
177 183 233 253 563 589 619 
143 251 192 230 240 164 136 

ddply(dd,.(Site,JulianDay),summarize,length(Delta))
  Site JulianDay length(Delta)
1   DF       183           251
2   DF       198           240
3   DF       253           230
4   DF       254           136
5   HF       177           143
6   HF       224           164
7   HF       233           192

ddply(dd,.(Site,Year_collect,JulianDay),summarize,length(Delta))
  Site Year_collect JulianDay length(Delta)
1   DF         2013       183           251
2   DF         2013       253           230
3   DF         2014       198           240
4   DF         2014       254           136
5   HF         2013       177           143
6   HF         2013       233           192
7   HF         2014       224           164
```

### plots by continuous days

```R
ggplot(dd,aes(x=factor(Jdaycont),y=log(count+1),color=Site))+
geom_boxplot()+facet_grid(.~gene)
```

![](https://cloud.githubusercontent.com/assets/4654474/25624014/31c2b374-2f26-11e7-9d0d-0d19580cc94e.jpeg)

```R
ggplot(dd,aes(x=Jdaycont,y=log(count+1),color=Site))+geom_point()+stat_smooth()+facet_grid(.~gene)
```

![](https://cloud.githubusercontent.com/assets/4654474/25624134/81fa07de-2f26-11e7-8a04-8a67f344614a.jpeg)

### plot just by day

```R
ggplot(dd,aes(x=JulianDay,y=log(count+1),color=Site))+geom_point()+geom_smooth(se=FALSE)+facet_grid(.~gene)
```

![](https://cloud.githubusercontent.com/assets/4654474/25625515/b37f8c44-2f2a-11e7-8f96-6b90ab7fcce1.jpeg)


### Site level differences    

```R
ggplot(dd,aes(x=baittemp.ave,y=log(count+1),color=gene))+geom_point()+geom_smooth(method="lm",se=FALSE)+facet_grid(.~Site)
```

![](https://cloud.githubusercontent.com/assets/4654474/25626869/dc0f44fc-2f2e-11e7-95ee-dcd8657acb84.jpeg)
------

 <div id='id-section75'/> 

### Page 75: 2017-05-03. Stressed in nature project: Analyses...again      

Using mcmc.qpcr package. 

### Model 1: Naive - Testing for site by temperature interaction while controlling for season     

Model testing for the effect of Site by bait temp and site by delta, controlling for RIN and Julian Day (globla fixed effects); including chamber as a random effect. 

* Fixed effects: Site * bait + Site * Delta
* Global fixed effects: RIN value (rna quality) and Julian Day continuous
* Random factor: Chamber

```R
mm=mcmc.qpcr(
fixed="Site*baittemp.ave+Site*Delta",
random=c("Cham"),globalFixed=c("RIN_Value","Jdaycont"),
data=dd,pr=TRUE,vprior="iw",nitt=50000,geneSpecRes = TRUE,pl=TRUE)
summary(mm)
```  

### Model 1 output: Naive - Testing for site by temperature interaction while controlling for season    

```R
Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 15013.3 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     2.971    2.338    3.624     3132

               ~idh(gene):Cham

                  post.mean l-95% CI u-95% CI eff.samp
geneCT_18s.Cham      0.5831   0.1978   1.0303     4700
geneCT_40.Cham       0.3985   0.1705   0.7036     4811
geneCT_70.Cham       0.3942   0.1666   0.6862     4700
geneCT_83.Cham       0.4886   0.2011   0.8666     4700
geneCT_actin.Cham    0.4596   0.1884   0.8179     4996
geneCT_gapdh.Cham    0.5304   0.1953   0.9395     4700

 R-structure:  ~idh(gene):units

                   post.mean l-95% CI u-95% CI eff.samp
geneCT_18s.units      4.7506   3.7644   5.6788     4700
geneCT_40.units       0.4917   0.3377   0.6555     2027
geneCT_70.units       0.7914   0.6072   0.9920     3157
geneCT_83.units       2.1489   1.6681   2.6629     1060
geneCT_actin.units    1.2714   0.9745   1.5733     2346
geneCT_gapdh.units    1.7054   1.3214   2.1022     2197

 Location effects: count ~ 0 + gene + RIN_Value + Jdaycont + +gene:Site * baittemp.ave + gene:Site * Delta 

                                  post.mean   l-95% CI   u-95% CI eff.samp   pMCMC    
geneCT_18s                        16.110765  12.281529  19.996436     4700 < 2e-04 ***
geneCT_40                         -2.531363  -4.724173  -0.348647     3418 0.02170 *  
geneCT_70                         -1.967982  -4.259055   0.126263     3489 0.07404 .  
geneCT_83                         -4.654332  -7.732704  -1.898907     3394 0.00213 ** 
geneCT_actin                       1.126526  -1.291423   3.750314     3925 0.36681    
geneCT_gapdh                       0.552040  -2.374267   3.209835     4382 0.68936    
RIN_Value                          0.199144   0.079509   0.314567     4253 < 2e-04 ***
Jdaycont                           0.006478   0.004881   0.008213     4700 < 2e-04 ***
baittemp.ave                       0.040939  -0.247876   0.344168     4700 0.80255    
Delta                             -0.112723  -0.400060   0.198869     4700 0.47489    
geneCT_18s:SiteHF                 -8.646499 -19.103973   2.627596     4932 0.11277    
geneCT_40:SiteHF                  -5.705043 -13.030149   1.312838     4172 0.12511    
geneCT_70:SiteHF                  -3.986178 -11.408322   3.139362     4700 0.28638    
geneCT_83:SiteHF                 -11.059021 -19.950986  -2.446486     2618 0.01447 *  
geneCT_actin:SiteHF               -5.294406 -13.214177   2.828587     4700 0.18426    
geneCT_gapdh:SiteHF                1.198255  -6.731170   9.986804     4021 0.78298    
geneCT_18s:SiteDF:baittemp.ave    -0.215198  -0.549081   0.125987     4700 0.20511    
geneCT_40:SiteDF:baittemp.ave      0.052836  -0.250665   0.373404     4700 0.74894    
geneCT_70:SiteDF:baittemp.ave      0.151878  -0.162515   0.459935     4700 0.35064    
geneCT_83:SiteDF:baittemp.ave      0.202728  -0.118489   0.515906     4067 0.20809    
geneCT_actin:SiteDF:baittemp.ave   0.007118  -0.314216   0.317920     4700 0.95447    
geneCT_gapdh:SiteDF:baittemp.ave   0.027289  -0.276939   0.361853     4102 0.87106    
geneCT_18s:SiteHF:baittemp.ave     0.213395  -0.135902   0.543047     4418 0.21830    
geneCT_40:SiteHF:baittemp.ave      0.302935   0.085426   0.506891     3002 0.01021 *  
geneCT_70:SiteHF:baittemp.ave      0.330920   0.091069   0.539425     3979 0.00511 ** 
geneCT_83:SiteHF:baittemp.ave      0.648529   0.382448   0.926608     2402 < 2e-04 ***
geneCT_actin:SiteHF:baittemp.ave   0.201615  -0.048443   0.451531     3650 0.11660    
geneCT_18s:SiteDF:Delta            0.232695  -0.186446   0.635062     4700 0.26596    
geneCT_40:SiteDF:Delta             0.116132  -0.245279   0.480588     4700 0.53617    
geneCT_70:SiteDF:Delta             0.047066  -0.308663   0.421765     4700 0.79745    
geneCT_83:SiteDF:Delta             0.204531  -0.174726   0.591889     4700 0.28723    
geneCT_actin:SiteDF:Delta          0.070546  -0.317046   0.432616     4700 0.71787    
geneCT_gapdh:SiteDF:Delta          0.248740  -0.106857   0.664952     4700 0.20851    
geneCT_18s:SiteHF:Delta           -0.298719  -0.669543   0.080339     4700 0.11872    
geneCT_40:SiteHF:Delta            -0.140106  -0.379562   0.099732     3397 0.26298    
geneCT_70:SiteHF:Delta            -0.062122  -0.301972   0.191591     4311 0.63106    
geneCT_83:SiteHF:Delta            -0.155097  -0.457374   0.136096     4052 0.31745    
geneCT_actin:SiteHF:Delta          0.031560  -0.229501   0.318134     4183 0.80255    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

diagnostics:  

![](https://cloud.githubusercontent.com/assets/4654474/25673382/327b2faa-3005-11e7-85b5-b377bea46a0c.jpeg)

### Model 2: Naive-Testing for site by temperature by season (julian day) interaction 

Model testing for effect of Site by bait temp by julian day continuous; controlling for RIN and including chamber as a random effect
* Fixed effects: Site * bait + Site * Delta
* Global fixed effects: RIN value (rna quality) and Julian Day continuous
* Random factor: Chamber

```R
mm2=mcmc.qpcr(
fixed="Site*baittemp.ave*Jdaycont",
random=c("Cham"),globalFixed=c("RIN_Value"),
data=dd,pr=TRUE,vprior="iw",nitt=50000,include=0,geneSpecRes = TRUE,pl=TRUE)
summary(mm2)

```

### MOdel 2 output: Naive-Testing for site by temperature by season (julian day) interaction      

```R

 Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 15023.12 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     1.917    1.514    2.347     3390

               ~idh(gene):Cham

                  post.mean l-95% CI u-95% CI eff.samp
geneCT_18s.Cham      0.6301   0.2387   1.1668     4700
geneCT_40.Cham       0.3843   0.1727   0.6856     4700
geneCT_70.Cham       0.3639   0.1597   0.6361     4700
geneCT_83.Cham       0.4681   0.1787   0.8216     4700
geneCT_actin.Cham    0.4103   0.1743   0.7152     5230
geneCT_gapdh.Cham    0.4443   0.1841   0.7888     5332

 R-structure:  ~idh(gene):units

                   post.mean l-95% CI u-95% CI eff.samp
geneCT_18s.units      4.4459   3.6026   5.2871   4700.0
geneCT_40.units       0.3735   0.2619   0.4933   2511.4
geneCT_70.units       0.6003   0.4567   0.7375   3188.9
geneCT_83.units       1.1112   0.8506   1.3815    905.5
geneCT_actin.units    1.1518   0.9038   1.4147   2207.6
geneCT_gapdh.units    1.0840   0.8476   1.3569   1719.8

 Location effects: count ~ 0 + gene + RIN_Value + +gene:Site * baittemp.ave * Jdaycont 

                                           post.mean   l-95% CI   u-95% CI eff.samp    pMCMC    
geneCT_18s                                -2.697e+00 -1.060e+01  5.214e+00     4700 0.518723    
geneCT_40                                 -1.215e+01 -1.585e+01 -7.936e+00     2962  < 2e-04 ***
geneCT_70                                 -1.587e+01 -1.993e+01 -1.200e+01     4076  < 2e-04 ***
geneCT_83                                 -3.134e+01 -3.626e+01 -2.646e+01     2357  < 2e-04 ***
geneCT_actin                              -1.958e+01 -2.451e+01 -1.468e+01     4107  < 2e-04 ***
geneCT_gapdh                              -2.243e+01 -2.731e+01 -1.775e+01     3686  < 2e-04 ***
RIN_Value                                  2.047e-01  1.102e-01  3.090e-01     4147 0.000426 ***
baittemp.ave                              -1.958e-01 -6.238e-01  2.510e-01     4700 0.382979    
Jdaycont                                  -5.680e-04 -3.427e-02  3.148e-02     4700 0.972340    
geneCT_18s:SiteHF                          1.539e+01 -3.751e+00  3.350e+01     4700 0.108936    
geneCT_40:SiteHF                           1.293e+00 -1.058e+01  1.215e+01     3853 0.809362    
geneCT_70:SiteHF                           6.747e+00 -5.146e+00  1.809e+01     4700 0.250638    
geneCT_83:SiteHF                           7.440e+00 -6.279e+00  2.034e+01     2260 0.265957    
geneCT_actin:SiteHF                        1.731e+01  4.324e+00  3.092e+01     3714 0.011064 *  
geneCT_gapdh:SiteHF                        2.841e+01  1.519e+01  4.106e+01     4490  < 2e-04 ***
baittemp.ave:Jdaycont                      4.655e-04 -7.468e-04  1.643e-03     4700 0.425532    
geneCT_18s:SiteDF:baittemp.ave             7.159e-01  1.879e-01  1.248e+00     4700 0.007234 ** 
geneCT_40:SiteDF:baittemp.ave              6.869e-01  1.890e-01  1.132e+00     4101 0.002979 ** 
geneCT_70:SiteDF:baittemp.ave              9.480e-01  4.694e-01  1.406e+00     4497  < 2e-04 ***
geneCT_83:SiteDF:baittemp.ave              1.435e+00  9.523e-01  1.918e+00     4700  < 2e-04 ***
geneCT_actin:SiteDF:baittemp.ave           1.057e+00  5.651e-01  1.514e+00     4700  < 2e-04 ***
geneCT_gapdh:SiteDF:baittemp.ave           1.147e+00  6.529e-01  1.612e+00     4506  < 2e-04 ***
geneCT_18s:SiteHF:baittemp.ave             1.760e-01 -4.193e-01  7.441e-01     4700 0.566383    
geneCT_40:SiteHF:baittemp.ave              6.176e-01  3.030e-01  9.288e-01     3589  < 2e-04 ***
geneCT_70:SiteHF:baittemp.ave              6.932e-01  3.741e-01  1.028e+00     4700  < 2e-04 ***
geneCT_83:SiteHF:baittemp.ave              1.085e+00  7.130e-01  1.493e+00     2729  < 2e-04 ***
geneCT_actin:SiteHF:baittemp.ave           3.454e-01 -6.321e-02  7.128e-01     3574 0.085532 .  
geneCT_18s:SiteDF:Jdaycont                 7.762e-02  3.948e-02  1.162e-01     4700  < 2e-04 ***
geneCT_40:SiteDF:Jdaycont                  5.000e-02  1.422e-02  8.342e-02     4497 0.002553 ** 
geneCT_70:SiteDF:Jdaycont                  6.005e-02  2.730e-02  9.631e-02     4472 0.000851 ***
geneCT_83:SiteDF:Jdaycont                  9.879e-02  6.440e-02  1.345e-01     4700  < 2e-04 ***
geneCT_actin:SiteDF:Jdaycont               7.954e-02  4.401e-02  1.142e-01     4700  < 2e-04 ***
geneCT_gapdh:SiteDF:Jdaycont               8.735e-02  5.198e-02  1.232e-01     4700  < 2e-04 ***
geneCT_18s:SiteHF:Jdaycont                 4.785e-03 -3.861e-02  5.004e-02     4700 0.826383    
geneCT_40:SiteHF:Jdaycont                  2.291e-02  1.350e-04  4.613e-02     4243 0.051064 .  
geneCT_70:SiteHF:Jdaycont                  2.117e-02 -3.446e-03  4.640e-02     4700 0.092766 .  
geneCT_83:SiteHF:Jdaycont                  4.531e-02  1.793e-02  7.605e-02     3481 0.002128 ** 
geneCT_actin:SiteHF:Jdaycont               5.129e-03 -2.447e-02  3.241e-02     4202 0.741702    
geneCT_18s:SiteDF:baittemp.ave:Jdaycont   -3.044e-03 -4.453e-03 -1.623e-03     4700  < 2e-04 ***
geneCT_40:SiteDF:baittemp.ave:Jdaycont    -2.162e-03 -3.423e-03 -8.991e-04     4509 0.000851 ***
geneCT_70:SiteDF:baittemp.ave:Jdaycont    -2.556e-03 -3.897e-03 -1.369e-03     4493  < 2e-04 ***
geneCT_83:SiteDF:baittemp.ave:Jdaycont    -3.850e-03 -5.103e-03 -2.532e-03     4700  < 2e-04 ***
geneCT_actin:SiteDF:baittemp.ave:Jdaycont -3.274e-03 -4.538e-03 -1.981e-03     4700  < 2e-04 ***
geneCT_gapdh:SiteDF:baittemp.ave:Jdaycont -3.486e-03 -4.756e-03 -2.131e-03     4700  < 2e-04 ***
geneCT_18s:SiteHF:baittemp.ave:Jdaycont   -2.595e-04 -1.828e-03  1.393e-03     4700 0.745106    
geneCT_40:SiteHF:baittemp.ave:Jdaycont    -1.062e-03 -1.879e-03 -2.114e-04     4265 0.014894 *  
geneCT_70:SiteHF:baittemp.ave:Jdaycont    -1.059e-03 -1.937e-03 -1.224e-04     4700 0.018723 *  
geneCT_83:SiteHF:baittemp.ave:Jdaycont    -1.640e-03 -2.709e-03 -6.076e-04     3510 0.003404 ** 
geneCT_actin:SiteHF:baittemp.ave:Jdaycont -3.494e-04 -1.347e-03  7.197e-04     4217 0.517872    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

diagnostics      

![](https://cloud.githubusercontent.com/assets/4654474/25673696/341932b6-3006-11e7-8ff7-ca6ada6d5d17.jpeg)


### Model 3: Soft normalization- Including 18s, actin, and gapdh to normalize   

Model testing for effect of Site by bait temp; controlling for RIN and julian day continuous and including chamber as a random effect
* Fixed effects: Site * bait + Site * Delta
* Global fixed effects: RIN value (rna quality) and Julian Day continuous
* Random factor: Chamber   

```R
softnorm=mcmc.qpcr(fixed="Site*baittemp.ave+Site*Delta",random="Cham",data=dd,pr=TRUE,vprior="iw",nitt=10000,geneSpecRes = TRUE,pl=TRUE,normalize=TRUE,controls=c("CT_actin","CT_gapdh","CT_18s"),globalFixed=c("RIN_Value","Jdaycont"))
summary(softnorm)

```

### Model 3 output: Soft normalization- Including 18s, actin, and gapdh to normalize   

```R

 Iterations = 3001:9991
 Thinning interval  = 10
 Sample size  = 700 

 DIC: 9005.386 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     2.517    1.983    3.093      700

               ~idh(gene):Cham

               post.mean l-95% CI u-95% CI eff.samp
geneNORM.Cham     0.4293   0.1489   0.7762    518.2
geneCT_40.Cham    0.3077   0.1190   0.5434    700.0
geneCT_70.Cham    0.3022   0.1199   0.5436    700.0
geneCT_83.Cham    0.4124   0.1460   0.7656    700.0

 R-structure:  ~idh(gene):units

                post.mean l-95% CI u-95% CI eff.samp
geneNORM.units     1.1929   0.9095   1.4661    474.6
geneCT_40.units    0.3536   0.2313   0.4828    304.2
geneCT_70.units    0.7093   0.5239   0.8606    485.2
geneCT_83.units    2.5393   1.9813   3.1185    115.3

 Location effects: count ~ gene + RIN_Value + Jdaycont + Site * baittemp.ave + Site * Delta + gene:Site * baittemp.ave + gene:Site * Delta 

                               post.mean   l-95% CI   u-95% CI eff.samp   pMCMC   
(Intercept)                     8.684991   6.169046  11.086840    577.4 < 0.001 **
geneCT_40                      -9.671732 -12.106692  -7.552922    468.7 < 0.001 **
geneCT_70                      -9.182214 -11.530024  -7.134289    514.8 < 0.001 **
geneCT_83                     -11.866716 -14.977469  -8.537719    535.9 < 0.001 **
RIN_Value                       0.211073   0.107941   0.331795    700.0 < 0.001 **
Jdaycont                        0.005798   0.004379   0.007320    700.0 < 0.001 **
SiteHF                         -4.721952 -11.937967   3.212892    411.7 0.20286   
baittemp.ave                   -0.113720  -0.214446  -0.025546    438.3 0.01429 * 
Delta                           0.084153  -0.108549   0.293981    700.0 0.41143   
SiteHF:baittemp.ave             0.209194  -0.101481   0.482602    700.0 0.14857   
SiteHF:Delta                   -0.203374  -0.514559   0.153917    700.0 0.23429   
geneCT_40:SiteHF               -1.897398  -6.878381   3.333412    700.0 0.48000   
geneCT_70:SiteHF               -0.323395  -6.073259   5.043501    610.0 0.92000   
geneCT_83:SiteHF               -7.705725 -14.989465   0.375533    601.6 0.06857 . 
geneCT_40:SiteDF:baittemp.ave   0.157087   0.069875   0.242143    505.6 < 0.001 **
geneCT_70:SiteDF:baittemp.ave   0.256887   0.166474   0.339240    515.3 < 0.001 **
geneCT_83:SiteDF:baittemp.ave   0.308844   0.178066   0.427849    599.9 < 0.001 **
geneCT_40:SiteHF:baittemp.ave   0.230984   0.060239   0.419166    700.0 0.01429 * 
geneCT_70:SiteHF:baittemp.ave   0.266757   0.080296   0.462425    542.0 0.00857 **
geneCT_83:SiteHF:baittemp.ave   0.597438   0.342265   0.859273    494.2 < 0.001 **
geneCT_40:SiteDF:Delta         -0.065660  -0.240177   0.106559    700.0 0.42286   
geneCT_70:SiteDF:Delta         -0.127074  -0.322574   0.049327    882.0 0.15429   
geneCT_83:SiteDF:Delta          0.015443  -0.224270   0.231865    700.0 0.90857   
geneCT_40:SiteHF:Delta         -0.124124  -0.320080   0.061610    700.0 0.22571   
geneCT_70:SiteHF:Delta         -0.050784  -0.270149   0.156429    700.0 0.66571   
geneCT_83:SiteHF:Delta         -0.166027  -0.435081   0.113876    607.6 0.26000   
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

diagnostics   

![](https://cloud.githubusercontent.com/assets/4654474/25673956/3df7b8e2-3007-11e7-89f0-0672a00dafbb.jpeg)    


### Model 4: Soft normalization- Including 18s, actin, and gapdh to normalize; include interaction with season    

Model testing for effect of Site by bait temp by julian day continuous; controlling for RIN and including chamber as a random effect
* Fixed effects: Site * bait*Jdaycont
* Global fixed effects: RIN value (rna quality) 
* Random factor: Chamber


```R
softnorm2=mcmc.qpcr(fixed="Site*baittemp.ave*Jdaycont",random="Cham",data=dd,pr=TRUE,vprior="iw",nitt=10000,geneSpecRes = TRUE,pl=TRUE,normalize=TRUE,controls=c("CT_actin","CT_gapdh","CT_18s"),globalFixed=c("RIN_Value"))
summary(softnorm2)
```

### Model 4 output: Soft normalization- Including 18s, actin, and gapdh to normalize; include interaction with season     

```R
Iterations = 3001:9991
 Thinning interval  = 10
 Sample size  = 700 

 DIC: 9005.803 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     1.818    1.433    2.249    610.9

               ~idh(gene):Cham

               post.mean l-95% CI u-95% CI eff.samp
geneNORM.Cham     0.3805   0.1372   0.6941    790.7
geneCT_40.Cham    0.2881   0.1069   0.5029    700.0
geneCT_70.Cham    0.2813   0.1253   0.4923    700.0
geneCT_83.Cham    0.3909   0.1561   0.6891    700.0

 R-structure:  ~idh(gene):units

                post.mean l-95% CI u-95% CI eff.samp
geneNORM.units     0.8994   0.7000   1.0903    700.0
geneCT_40.units    0.3020   0.2036   0.3963    344.5
geneCT_70.units    0.5810   0.4347   0.7147    576.4
geneCT_83.units    1.1998   0.9383   1.4820    237.8

 Location effects: count ~ gene + RIN_Value + Site * baittemp.ave * Jdaycont + gene:Site * baittemp.ave * Jdaycont 

                                        post.mean   l-95% CI   u-95% CI eff.samp   pMCMC   
(Intercept)                            -8.661e+00 -1.331e+01 -4.130e+00    700.0 < 0.001 **
geneCT_40                              -1.903e+00 -6.363e+00  2.411e+00    700.0 0.39143   
geneCT_70                              -5.688e+00 -9.774e+00 -1.181e+00    700.0 0.00857 **
geneCT_83                              -2.090e+01 -2.622e+01 -1.549e+01    700.0 < 0.001 **
RIN_Value                               1.981e-01  9.955e-02  3.036e-01    575.0 < 0.001 **
SiteHF                                  1.648e+01  4.298e+00  2.825e+01    700.0 0.00286 **
baittemp.ave                            5.477e-01  3.813e-01  7.313e-01    700.0 < 0.001 **
Jdaycont                                6.788e-02  5.544e-02  8.040e-02    700.0 < 0.001 **
SiteHF:baittemp.ave                    -6.478e-01 -1.100e+00 -1.991e-01    700.0 < 0.001 **
SiteHF:Jdaycont                        -7.108e-02 -1.052e-01 -4.008e-02    700.0 < 0.001 **
baittemp.ave:Jdaycont                  -2.314e-03 -2.771e-03 -1.835e-03    700.0 < 0.001 **
geneCT_40:SiteHF                       -1.646e+01 -2.491e+01 -6.981e+00    485.8 < 0.001 **
geneCT_70:SiteHF                       -1.118e+01 -2.052e+01 -2.765e+00    700.0 0.02571 * 
geneCT_83:SiteHF                       -1.103e+01 -2.284e+01 -5.886e-01    700.0 0.04571 * 
SiteHF:baittemp.ave:Jdaycont            2.783e-03  1.619e-03  4.034e-03    700.0 < 0.001 **
geneCT_40:SiteDF:baittemp.ave          -1.120e-01 -2.825e-01  5.198e-02    700.0 0.19429   
geneCT_70:SiteDF:baittemp.ave           1.504e-01 -2.890e-02  3.083e-01    700.0 0.08571 . 
geneCT_83:SiteDF:baittemp.ave           6.275e-01  4.305e-01  8.497e-01    700.0 < 0.001 **
geneCT_40:SiteHF:baittemp.ave           5.121e-01  2.556e-01  7.882e-01    373.2 < 0.001 **
geneCT_70:SiteHF:baittemp.ave           5.965e-01  3.218e-01  9.269e-01    700.0 < 0.001 **
geneCT_83:SiteHF:baittemp.ave           9.978e-01  6.902e-01  1.447e+00    618.8 < 0.001 **
geneCT_40:SiteDF:Jdaycont              -2.229e-02 -3.189e-02 -1.169e-02    700.0 < 0.001 **
geneCT_70:SiteDF:Jdaycont              -1.194e-02 -2.321e-02 -1.849e-03    700.0 0.02571 * 
geneCT_83:SiteDF:Jdaycont               2.635e-02  1.412e-02  4.098e-02    700.0 < 0.001 **
geneCT_40:SiteHF:Jdaycont               2.447e-02  5.219e-03  4.679e-02    700.0 0.01714 * 
geneCT_70:SiteHF:Jdaycont               2.317e-02  1.383e-03  4.789e-02    700.0 0.05143 . 
geneCT_83:SiteHF:Jdaycont               4.817e-02  1.820e-02  7.441e-02    700.0 0.00286 **
geneCT_40:SiteDF:baittemp.ave:Jdaycont  7.573e-04  3.373e-04  1.118e-03    700.0 < 0.001 **
geneCT_70:SiteDF:baittemp.ave:Jdaycont  3.523e-04 -4.856e-05  7.660e-04    700.0 0.10857   
geneCT_83:SiteDF:baittemp.ave:Jdaycont -9.246e-04 -1.390e-03 -3.810e-04    700.0 < 0.001 **
geneCT_40:SiteHF:baittemp.ave:Jdaycont -1.025e-03 -1.816e-03 -3.201e-04    700.0 0.00857 **
geneCT_70:SiteHF:baittemp.ave:Jdaycont -1.041e-03 -1.925e-03 -2.063e-04    700.0 0.02571 * 
geneCT_83:SiteHF:baittemp.ave:Jdaycont -1.651e-03 -2.635e-03 -5.883e-04    700.0 0.00571 **
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

diagnostics    

![](https://cloud.githubusercontent.com/assets/4654474/25674145/d8256856-3007-11e7-9b0c-111571c8c3b1.jpeg)

### Model 5: Naive- most complex model- 3 way interaction between delta, site, Jdaycont and 3 way interaction between bait, site, Jdaycont        

Model testing for effect of Site by bait temp by julian day continuous; controlling for RIN and including chamber as a random effect
* Fixed effects: Site * bait* Jdaycont + Site * Delta* Jdaycont
* Global fixed effects: RIN value (rna quality) 
* Random factor: Chamber

```R
com1=mcmc.qpcr(
fixed="Site*baittemp.ave*Jdaycont+Site*Delta*Jdaycont",
random=c("Cham"),globalFixed=c("RIN_Value"),
data=dd,pr=TRUE,vprior="iw",nitt=25000,geneSpecRes = TRUE,pl=TRUE)
summary(com1)
diagnostic.mcmc(model=com1, col="grey50",cex=0.8)
```

### Model 5 output: Naive- most complex model- 3 way interaction between delta, site, Jdaycont and 3 way interaction between bait, site, Jdaycont      


```R
Iterations = 3001:24991
 Thinning interval  = 10
 Sample size  = 2200 

 DIC: 15031.14 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     1.837    1.466    2.245     2200

               ~idh(gene):Cham

                  post.mean l-95% CI u-95% CI eff.samp
geneCT_18s.Cham      0.5911   0.2259   1.0935     2200
geneCT_40.Cham       0.3927   0.1621   0.6771     2200
geneCT_70.Cham       0.3983   0.1698   0.7059     2200
geneCT_83.Cham       0.4718   0.1777   0.8140     2200
geneCT_actin.Cham    0.4319   0.1643   0.7832     2200
geneCT_gapdh.Cham    0.5181   0.2231   0.9287     2200

 R-structure:  ~idh(gene):units

                   post.mean l-95% CI u-95% CI eff.samp
geneCT_18s.units      4.4093   3.5220   5.2938   2027.8
geneCT_40.units       0.3753   0.2659   0.4949   1152.1
geneCT_70.units       0.5522   0.4221   0.6899   1626.8
geneCT_83.units       1.0948   0.8301   1.3632    528.7
geneCT_actin.units    1.1718   0.8997   1.4371   1410.1
geneCT_gapdh.units    1.0445   0.8184   1.3015   1325.4

 Location effects: count ~ 0 + gene + RIN_Value + +gene:Site * baittemp.ave * Jdaycont + gene:Site * Delta * Jdaycont 

                                           post.mean   l-95% CI   u-95% CI eff.samp    pMCMC    
geneCT_18s                                -2.823e+00 -1.120e+01  5.159e+00     2200 0.485455    
geneCT_40                                 -1.189e+01 -1.608e+01 -7.850e+00     2010  < 5e-04 ***
geneCT_70                                 -1.565e+01 -1.968e+01 -1.169e+01     2200  < 5e-04 ***
geneCT_83                                 -3.130e+01 -3.643e+01 -2.673e+01     1330  < 5e-04 ***
geneCT_actin                              -1.952e+01 -2.438e+01 -1.490e+01     2200  < 5e-04 ***
geneCT_gapdh                              -2.262e+01 -2.759e+01 -1.794e+01     1919  < 5e-04 ***
RIN_Value                                  1.995e-01  1.083e-01  2.974e-01     1773  < 5e-04 ***
baittemp.ave                              -1.349e-01 -6.804e-01  3.516e-01     1970 0.606364    
Jdaycont                                  -6.081e-05 -4.225e-02  4.287e-02     2395 0.995455    
Delta                                     -2.896e-01 -7.886e-01  1.875e-01     2200 0.236364    
geneCT_18s:SiteHF                          1.077e+01 -1.013e+01  3.064e+01     2200 0.318182    
geneCT_40:SiteHF                          -3.127e+00 -1.523e+01  9.476e+00     1859 0.630909    
geneCT_70:SiteHF                          -3.848e+00 -1.662e+01  9.315e+00     2200 0.567273    
geneCT_83:SiteHF                           2.845e+00 -1.191e+01  1.736e+01     1948 0.690909    
geneCT_actin:SiteHF                        1.153e+01 -3.734e+00  2.505e+01     2200 0.120909    
geneCT_gapdh:SiteHF                        2.778e+01  1.389e+01  4.227e+01     2015 0.000909 ***
baittemp.ave:Jdaycont                      4.026e-04 -1.357e-03  1.953e-03     2385 0.623636    
Jdaycont:Delta                             4.409e-04 -1.011e-03  1.869e-03     2200 0.561818    
geneCT_18s:SiteDF:baittemp.ave             6.757e-01  1.042e-02  1.254e+00     2145 0.034545 *  
geneCT_40:SiteDF:baittemp.ave              6.270e-01  7.783e-02  1.167e+00     2051 0.027273 *  
geneCT_70:SiteDF:baittemp.ave              8.921e-01  3.226e-01  1.409e+00     2007 0.001818 ** 
geneCT_83:SiteDF:baittemp.ave              1.386e+00  8.521e-01  1.948e+00     1966  < 5e-04 ***
geneCT_actin:SiteDF:baittemp.ave           1.009e+00  4.530e-01  1.552e+00     2200 0.001818 ** 
geneCT_gapdh:SiteDF:baittemp.ave           1.098e+00  5.528e-01  1.628e+00     2027 0.000909 ***
geneCT_18s:SiteHF:baittemp.ave             3.694e-01 -3.419e-01  1.036e+00     2257 0.308182    
geneCT_40:SiteHF:baittemp.ave              7.578e-01  3.526e-01  1.115e+00     1965  < 5e-04 ***
geneCT_70:SiteHF:baittemp.ave              1.087e+00  6.721e-01  1.456e+00     2200  < 5e-04 ***
geneCT_83:SiteHF:baittemp.ave              1.249e+00  7.937e-01  1.743e+00     1088  < 5e-04 ***
geneCT_actin:SiteHF:baittemp.ave           5.374e-01  6.048e-02  1.010e+00     1737 0.028182 *  
geneCT_18s:SiteDF:Jdaycont                 7.880e-02  3.089e-02  1.248e-01     2298 0.005455 ** 
geneCT_40:SiteDF:Jdaycont                  4.986e-02  9.422e-03  9.765e-02     2473 0.030000 *  
geneCT_70:SiteDF:Jdaycont                  5.994e-02  1.345e-02  1.040e-01     2385 0.012727 *  
geneCT_83:SiteDF:Jdaycont                  9.934e-02  5.596e-02  1.459e-01     2284  < 5e-04 ***
geneCT_actin:SiteDF:Jdaycont               7.985e-02  3.446e-02  1.240e-01     2393 0.002727 ** 
geneCT_gapdh:SiteDF:Jdaycont               8.841e-02  3.979e-02  1.303e-01     2200  < 5e-04 ***
geneCT_18s:SiteHF:Jdaycont                 1.270e-02 -4.679e-02  7.036e-02     2518 0.690909    
geneCT_40:SiteHF:Jdaycont                  3.339e-02  8.214e-04  6.623e-02     1801 0.047273 *  
geneCT_70:SiteHF:Jdaycont                  5.750e-02  2.394e-02  9.013e-02     2200 0.000909 ***
geneCT_83:SiteHF:Jdaycont                  5.783e-02  1.919e-02  9.634e-02     1586 0.003636 ** 
geneCT_actin:SiteHF:Jdaycont               2.431e-02 -1.361e-02  6.343e-02     1747 0.221818    
geneCT_18s:SiteDF:Delta                    8.948e-02 -6.011e-01  7.888e-01     2200 0.804545    
geneCT_40:SiteDF:Delta                     1.564e-01 -4.777e-01  6.987e-01     2200 0.602727    
geneCT_70:SiteDF:Delta                     1.317e-01 -4.143e-01  7.291e-01     2200 0.644545    
geneCT_83:SiteDF:Delta                     1.091e-01 -4.431e-01  7.837e-01     2200 0.714545    
geneCT_actin:SiteDF:Delta                  1.013e-01 -5.002e-01  7.231e-01     2200 0.749091    
geneCT_gapdh:SiteDF:Delta                  2.241e-01 -3.804e-01  8.227e-01     2200 0.488182    
geneCT_18s:SiteHF:Delta                   -5.227e-01 -1.201e+00  8.997e-02     2200 0.110909    
geneCT_40:SiteHF:Delta                    -2.061e-01 -5.670e-01  1.574e-01     1595 0.267273    
geneCT_70:SiteHF:Delta                    -4.682e-01 -8.370e-01 -8.751e-02     2200 0.013636 *  
geneCT_83:SiteHF:Delta                    -2.866e-01 -7.655e-01  1.374e-01     1535 0.212727    
geneCT_actin:SiteHF:Delta                 -1.495e-01 -6.054e-01  3.167e-01     1826 0.530000    
geneCT_18s:SiteDF:baittemp.ave:Jdaycont   -3.128e-03 -4.892e-03 -1.247e-03     2289 0.003636 ** 
geneCT_40:SiteDF:baittemp.ave:Jdaycont    -2.150e-03 -3.735e-03 -3.373e-04     2456 0.016364 *  
geneCT_70:SiteDF:baittemp.ave:Jdaycont    -2.535e-03 -4.252e-03 -7.815e-04     2376 0.006364 ** 
geneCT_83:SiteDF:baittemp.ave:Jdaycont    -3.902e-03 -5.534e-03 -2.096e-03     2286  < 5e-04 ***
geneCT_actin:SiteDF:baittemp.ave:Jdaycont -3.288e-03 -4.920e-03 -1.450e-03     2407 0.000909 ***
geneCT_gapdh:SiteDF:baittemp.ave:Jdaycont -3.545e-03 -5.112e-03 -1.627e-03     2200  < 5e-04 ***
geneCT_18s:SiteHF:baittemp.ave:Jdaycont   -6.323e-04 -2.889e-03  1.635e-03     2519 0.590000    
geneCT_40:SiteHF:baittemp.ave:Jdaycont    -1.495e-03 -2.860e-03 -3.344e-04     1816 0.022727 *  
geneCT_70:SiteHF:baittemp.ave:Jdaycont    -2.562e-03 -3.815e-03 -1.243e-03     2200  < 5e-04 ***
geneCT_83:SiteHF:baittemp.ave:Jdaycont    -2.173e-03 -3.659e-03 -6.606e-04     1935 0.003636 ** 
geneCT_actin:SiteHF:baittemp.ave:Jdaycont -1.130e-03 -2.636e-03  3.316e-04     1762 0.150909    
geneCT_18s:SiteDF:Jdaycont:Delta           6.662e-04 -1.029e-03  2.752e-03     2200 0.480909    
geneCT_40:SiteDF:Jdaycont:Delta            8.687e-05 -1.483e-03  1.754e-03     2200 0.916364    
geneCT_70:SiteDF:Jdaycont:Delta           -8.150e-05 -1.645e-03  1.546e-03     2200 0.919091    
geneCT_83:SiteDF:Jdaycont:Delta            5.699e-04 -1.078e-03  2.176e-03     2200 0.487273    
geneCT_actin:SiteDF:Jdaycont:Delta         1.675e-04 -1.521e-03  1.799e-03     2200 0.839091    
geneCT_gapdh:SiteDF:Jdaycont:Delta         4.072e-04 -1.305e-03  2.010e-03     2200 0.650909    
geneCT_18s:SiteHF:Jdaycont:Delta           8.713e-04 -9.942e-04  2.950e-03     2347 0.393636    
geneCT_40:SiteHF:Jdaycont:Delta            5.417e-04 -6.536e-04  1.532e-03     1787 0.334545    
geneCT_70:SiteHF:Jdaycont:Delta            1.767e-03  6.850e-04  2.950e-03     2200 0.002727 ** 
geneCT_83:SiteHF:Jdaycont:Delta            7.647e-04 -6.120e-04  2.075e-03     1867 0.241818    
geneCT_actin:SiteHF:Jdaycont:Delta         8.130e-04 -5.026e-04  2.161e-03     1832 0.237273    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

diagnostics   

![](https://cloud.githubusercontent.com/assets/4654474/25674641/98379636-3009-11e7-81ab-5b9a99186108.jpeg)    


### Model 6: Soft normalization- most complex model- 3 way interaction between delta, site, Jdaycont and 3 way interaction between bait, site, Jdaycont        

Model testing for effect of Site by bait temp by julian day continuous and Site by delta by julian day continuous; controlling for RIN and including chamber as a random effect
* Fixed effects: Site*baittemp.ave*Jdaycont+Site*Delta*Jdaycont
* Global fixed effects: RIN value (rna quality) 
* Random factor: Chamber    

```R
comp2=mcmc.qpcr(fixed="Site*baittemp.ave*Jdaycont+Site*Delta*Jdaycont",random="Cham",data=dd,pr=TRUE,vprior="iw",nitt=25000,geneSpecRes = TRUE,pl=TRUE,normalize=TRUE,controls=c("CT_actin","CT_gapdh","CT_18s"),globalFixed=c("RIN_Value"))
summary(comp2)
diagnostic.mcmc(model=comp2, col="grey50",cex=0.8)
```

### Model 6 OUTPUT: Soft normalization- most complex model- 3 way interaction between delta, site, Jdaycont and 3 way interaction between bait, site, Jdaycont  

```R
Iterations = 3001:24991
 Thinning interval  = 10
 Sample size  = 2200 

 DIC: 9009.713 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     1.732    1.385    2.184     1501

               ~idh(gene):Cham

               post.mean l-95% CI u-95% CI eff.samp
geneNORM.Cham     0.4025   0.1483   0.7419     2503
geneCT_40.Cham    0.2984   0.1205   0.5389     2200
geneCT_70.Cham    0.3028   0.1228   0.5676     2200
geneCT_83.Cham    0.3957   0.1406   0.7272     2001

 R-structure:  ~idh(gene):units

                post.mean l-95% CI u-95% CI eff.samp
geneNORM.units     0.8910   0.6979   1.1078   2007.8
geneCT_40.units    0.2969   0.2043   0.4117   1134.6
geneCT_70.units    0.5373   0.4173   0.6735   1201.4
geneCT_83.units    1.1912   0.9067   1.5100    621.4

 Location effects: count ~ gene + RIN_Value + Site * baittemp.ave * Jdaycont + Site * Delta * Jdaycont + gene:Site * baittemp.ave * Jdaycont + gene:Site * Delta * Jdaycont 

                                        post.mean   l-95% CI   u-95% CI eff.samp    pMCMC    
(Intercept)                            -8.798e+00 -1.361e+01 -4.331e+00     2015  < 5e-04 ***
geneCT_40                              -1.588e+00 -5.868e+00  2.627e+00     1925 0.484545    
geneCT_70                              -5.515e+00 -1.008e+01 -9.670e-01     1960 0.018182 *  
geneCT_83                              -2.119e+01 -2.675e+01 -1.585e+01     1614  < 5e-04 ***
RIN_Value                               1.910e-01  1.024e-01  3.002e-01     1908  < 5e-04 ***
SiteHF                                  1.349e+01  3.435e-01  2.695e+01     2089 0.045455 *  
baittemp.ave                            5.616e-01  3.747e-01  7.367e-01     1991  < 5e-04 ***
Jdaycont                                6.935e-02  5.715e-02  8.314e-02     2200  < 5e-04 ***
Delta                                  -1.194e-01 -4.827e-01  2.018e-01     2200 0.497273    
SiteHF:baittemp.ave                    -5.084e-01 -1.016e+00  7.365e-04     2089 0.049091 *  
SiteHF:Jdaycont                        -6.249e-02 -1.044e-01 -2.113e-02     2200 0.000909 ***
baittemp.ave:Jdaycont                  -2.424e-03 -2.926e-03 -1.934e-03     2200  < 5e-04 ***
SiteHF:Delta                           -2.491e-01 -7.974e-01  3.488e-01     2200 0.403636    
Jdaycont:Delta                          7.583e-04  1.996e-05  1.463e-03     2200 0.044545 *  
geneCT_40:SiteHF                       -1.740e+01 -2.860e+01 -7.230e+00     1983  < 5e-04 ***
geneCT_70:SiteHF                       -1.846e+01 -3.004e+01 -7.750e+00     1856 0.000909 ***
geneCT_83:SiteHF                       -1.163e+01 -2.530e+01  7.869e-01     1777 0.068182 .  
SiteHF:baittemp.ave:Jdaycont            2.444e-03  7.669e-04  4.034e-03     2200 0.000909 ***
SiteHF:Jdaycont:Delta                   4.717e-05 -1.613e-03  1.689e-03     2200 0.953636    
geneCT_40:SiteDF:baittemp.ave          -1.212e-01 -2.943e-01  3.885e-02     1942 0.154545    
geneCT_70:SiteDF:baittemp.ave           1.481e-01 -3.409e-02  3.180e-01     1932 0.092727 .  
geneCT_83:SiteDF:baittemp.ave           6.444e-01  4.416e-01  8.651e-01     1657  < 5e-04 ***
geneCT_40:SiteHF:baittemp.ave           5.427e-01  1.903e-01  9.062e-01     1798 0.002727 ** 
geneCT_70:SiteHF:baittemp.ave           8.914e-01  4.999e-01  1.267e+00     2087  < 5e-04 ***
geneCT_83:SiteHF:baittemp.ave           1.051e+00  5.834e-01  1.500e+00     1633  < 5e-04 ***
geneCT_40:SiteDF:Jdaycont              -2.335e-02 -3.356e-02 -1.328e-02     1976  < 5e-04 ***
geneCT_70:SiteDF:Jdaycont              -1.270e-02 -2.331e-02 -1.892e-03     2200 0.019091 *  
geneCT_83:SiteDF:Jdaycont               2.683e-02  1.438e-02  4.077e-02     1672  < 5e-04 ***
geneCT_40:SiteHF:Jdaycont               2.504e-02 -3.193e-03  5.420e-02     1867 0.092727 .  
geneCT_70:SiteHF:Jdaycont               5.021e-02  1.877e-02  8.278e-02     1945 0.000909 ***
geneCT_83:SiteHF:Jdaycont               5.046e-02  1.094e-02  8.666e-02     2200 0.006364 ** 
geneCT_40:SiteDF:Delta                 -2.968e-02 -2.735e-01  2.198e-01     2042 0.810000    
geneCT_70:SiteDF:Delta                 -5.073e-02 -3.200e-01  1.949e-01     2200 0.718182    
geneCT_83:SiteDF:Delta                 -8.435e-02 -3.878e-01  2.478e-01     1936 0.610909    
geneCT_40:SiteHF:Delta                 -9.843e-02 -4.475e-01  2.744e-01     1733 0.589091    
geneCT_70:SiteHF:Delta                 -3.744e-01 -7.534e-01 -3.034e-04     1868 0.042727 *  
geneCT_83:SiteHF:Delta                 -2.118e-01 -6.535e-01  2.681e-01     1508 0.342727    
geneCT_40:SiteDF:baittemp.ave:Jdaycont  8.110e-04  4.423e-04  1.225e-03     1973  < 5e-04 ***
geneCT_70:SiteDF:baittemp.ave:Jdaycont  4.062e-04 -1.848e-06  8.277e-04     2200 0.050909 .  
geneCT_83:SiteDF:baittemp.ave:Jdaycont -9.647e-04 -1.500e-03 -4.873e-04     1780  < 5e-04 ***
geneCT_40:SiteHF:baittemp.ave:Jdaycont -1.054e-03 -2.134e-03  1.010e-04     1879 0.069091 .  
geneCT_70:SiteHF:baittemp.ave:Jdaycont -2.162e-03 -3.387e-03 -9.247e-04     1956  < 5e-04 ***
geneCT_83:SiteHF:baittemp.ave:Jdaycont -1.771e-03 -3.105e-03 -1.986e-04     2200 0.016364 *  
geneCT_40:SiteDF:Jdaycont:Delta        -1.827e-04 -6.758e-04  3.058e-04     2200 0.485455    
geneCT_70:SiteDF:Jdaycont:Delta        -3.358e-04 -8.632e-04  2.170e-04     2522 0.224545    
geneCT_83:SiteDF:Jdaycont:Delta         2.994e-04 -3.849e-04  1.003e-03     2117 0.400000    
geneCT_40:SiteHF:Jdaycont:Delta         1.086e-04 -9.713e-04  1.120e-03     2200 0.841818    
geneCT_70:SiteHF:Jdaycont:Delta         1.361e-03  8.136e-05  2.380e-03     2009 0.014545 *  
geneCT_83:SiteHF:Jdaycont:Delta         3.906e-04 -9.783e-04  1.700e-03     2066 0.558182    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

diagnostics     

![](https://cloud.githubusercontent.com/assets/4654474/25675813/c571f124-300d-11e7-9ce0-1ca6f3a01893.jpeg)

### plotting hsp70 for this model 6 against delta and site and collection date

![](https://cloud.githubusercontent.com/assets/4654474/25676777/4e666f70-3011-11e7-8e9b-2f7bdf0d54c5.jpeg)


### Using a more simple approach: ANOVA 

Testing the effect of gene by site by delta by season(julian day) and gene by site by bait temp by season(julian day)

```R
glob2<-aov(log(count+1)~RIN_Value+gene*Site*Jdaycont*Delta+gene*Site*Jdaycont*baittemp.ave,data=ddave)
```

output: 

```R
                                 Df Sum Sq Mean Sq  F value   Pr(>F)    
RIN_Value                         1   1223  1223.4  887.552  < 2e-16 ***
gene                              5   9337  1867.3 1354.741  < 2e-16 ***
Site                              1      5     4.7    3.379 0.066662 .  
Jdaycont                          1     76    75.7   54.889 5.93e-13 ***
Delta                             1      1     1.1    0.828 0.363410    
baittemp.ave                      1      4     4.2    3.054 0.081167 .  
gene:Site                         5     36     7.3    5.287 9.74e-05 ***
gene:Jdaycont                     5     72    14.5   10.512 1.39e-09 ***
Site:Jdaycont                     1     30    29.7   21.582 4.40e-06 ***
gene:Delta                        5      2     0.4    0.301 0.912386    
Site:Delta                        1      1     1.1    0.770 0.380517    
Jdaycont:Delta                    1      4     3.6    2.623 0.105977    
gene:baittemp.ave                 5     55    11.0    7.967 3.19e-07 ***
Site:baittemp.ave                 1     41    40.8   29.587 8.60e-08 ***
Jdaycont:baittemp.ave             1    448   447.8  324.912  < 2e-16 ***
gene:Site:Jdaycont                5     11     2.2    1.596 0.159755    
gene:Site:Delta                   5      3     0.7    0.498 0.778013    
gene:Jdaycont:Delta               5      1     0.1    0.092 0.993462    
Site:Jdaycont:Delta               1     20    19.8   14.387 0.000168 ***
gene:Site:baittemp.ave            5      9     1.9    1.360 0.237899    
gene:Jdaycont:baittemp.ave        5     24     4.8    3.480 0.004242 ** 
Site:Jdaycont:baittemp.ave        1     14    14.2   10.283 0.001434 ** 
gene:Site:Jdaycont:Delta          5      6     1.2    0.870 0.501010    
gene:Site:Jdaycont:baittemp.ave   5      9     1.8    1.339 0.246378    
Residuals                       472    651     1.4                      
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

diagnostics:   

![](https://cloud.githubusercontent.com/assets/4654474/25676158/0949ebf8-300f-11e7-919c-d71ad8c6b3ca.jpeg)


### Meeting with SHC and NJG     

* split by site and do models because control genes are different between sites
* do the traditional delta delta ct method


------

 <div id='id-section76'/> 

### Page 76: 2017-05-04. Stressed in nature project: Analyses including technical replicates    

I have to recode the chamber data. It was coded as 15 level factors, but there is really 30 levels. 

### Model 1: Naive model- testing effect of Site * bait and Site * delta  

* Fixed effects- Site * bait and Site * delta       
* Global fixed - Rin value and Jdaycont
* Random effects - Chamber

```R
mm=mcmc.qpcr(
fixed="Site+baittemp.ave+Delta+Site:baittemp.ave+Site:Delta",
globalRandom=c("Cham2"),globalFixed=c("RIN_Value","Jdaycont"),controls=c("CT_actin","CT_gapdh","CT_18s"),include=0,data=dd,pr=TRUE,vprior="iw",nitt=50000,geneSpecRes = TRUE,pl=TRUE)
summary(mm)
```

diagnostics   

![](https://cloud.githubusercontent.com/assets/4654474/25703770/f452e86c-30a4-11e7-850e-af83f0853b04.jpeg)

### Model 1 output : Naive model- testing effect of Site * bait and Site * delta  

```R

 Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 35139.52 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     3.519    2.798    4.281     4508

               ~Cham2

      post.mean  l-95% CI u-95% CI eff.samp
Cham2      0.04 6.823e-09   0.1558     1192

 R-structure:  ~idh(gene):units

                   post.mean l-95% CI u-95% CI eff.samp
geneCT_40.units       0.4136   0.3315   0.5069     4105
geneCT_70.units       0.4770   0.3988   0.5500     4055
geneCT_83.units       2.4430   2.1154   2.8343     3307
geneCT_18s.units      4.2449   3.6756   4.8468     4700
geneCT_gapdh.units    1.6591   1.4060   1.9019     4377
geneCT_actin.units    1.4672   1.2420   1.6840     4531

 Location effects: count ~ 0 + gene + RIN_Value + Jdaycont + +gene:Site + gene:baittemp.ave + gene:Delta + gene:Site:baittemp.ave + gene:Site:Delta 

                                  post.mean   l-95% CI   u-95% CI eff.samp    pMCMC    
geneCT_40                         12.842132  11.434150  14.428087     4447  < 2e-04 ***
geneCT_70                         15.135790  13.698704  16.594134     4151  < 2e-04 ***
geneCT_83                         10.650385   8.424201  12.787419     4700  < 2e-04 ***
geneCT_18s                        33.007600  30.453198  35.634718     4700  < 2e-04 ***
geneCT_gapdh                      17.997321  16.115197  19.939950     4700  < 2e-04 ***
geneCT_actin                      14.635113  12.791848  16.399952     4700  < 2e-04 ***
RIN_Value                         -0.179934  -0.272161  -0.084120     4700 0.000426 ***
Jdaycont                           0.008706   0.007161   0.010252     4700  < 2e-04 ***
geneCT_40:SiteHF                 -23.440970 -30.821937 -16.978075     4700  < 2e-04 ***
geneCT_70:SiteHF                 -25.312961 -32.432097 -18.447230     4700  < 2e-04 ***
geneCT_83:SiteHF                 -27.395793 -35.308069 -19.350075     4700  < 2e-04 ***
geneCT_18s:SiteHF                -30.831404 -39.288593 -22.145284     4700  < 2e-04 ***
geneCT_gapdh:SiteHF              -22.508387 -29.931357 -14.709748     4700  < 2e-04 ***
geneCT_actin:SiteHF              -17.976511 -25.242206 -10.147747     4700  < 2e-04 ***
geneCT_40:baittemp.ave            -0.383023  -0.439891  -0.326289     4437  < 2e-04 ***
geneCT_70:baittemp.ave            -0.350082  -0.404711  -0.296560     4131  < 2e-04 ***
geneCT_83:baittemp.ave            -0.239388  -0.322902  -0.157701     4700  < 2e-04 ***
geneCT_18s:baittemp.ave           -0.705876  -0.797390  -0.601410     4700  < 2e-04 ***
geneCT_gapdh:baittemp.ave         -0.486578  -0.563606  -0.420150     4470  < 2e-04 ***
geneCT_actin:baittemp.ave         -0.361752  -0.430539  -0.293431     4700  < 2e-04 ***
geneCT_40:Delta                    0.159633  -0.012466   0.340509     4700 0.077872 .  
geneCT_70:Delta                    0.167922  -0.014704   0.336399     4844 0.060426 .  
geneCT_83:Delta                    0.225763   0.036008   0.422831     4700 0.025957 *  
geneCT_18s:Delta                   0.246831   0.044045   0.460552     4619 0.024255 *  
geneCT_gapdh:Delta                 0.177491  -0.012900   0.353596     4700 0.057447 .  
geneCT_actin:Delta                 0.150764  -0.032281   0.331628     4393 0.109787    
geneCT_40:SiteHF:baittemp.ave      0.936244   0.675711   1.195277     4700  < 2e-04 ***
geneCT_70:SiteHF:baittemp.ave      1.016519   0.763996   1.290935     4700  < 2e-04 ***
geneCT_83:SiteHF:baittemp.ave      1.083009   0.771374   1.373619     4700  < 2e-04 ***
geneCT_18s:SiteHF:baittemp.ave     1.278770   0.963448   1.610912     4700  < 2e-04 ***
geneCT_gapdh:SiteHF:baittemp.ave   0.879381   0.588608   1.166238     4700  < 2e-04 ***
geneCT_actin:SiteHF:baittemp.ave   0.695124   0.396252   0.968543     4700  < 2e-04 ***
geneCT_40:SiteHF:Delta            -0.428692  -0.736058  -0.114304     4932 0.006809 ** 
geneCT_70:SiteHF:Delta            -0.456783  -0.794041  -0.167260     5201 0.004681 ** 
geneCT_83:SiteHF:Delta            -0.532939  -0.893603  -0.203274     4700 0.001702 ** 
geneCT_18s:SiteHF:Delta           -0.720123  -1.100263  -0.361775     4736  < 2e-04 ***
geneCT_gapdh:SiteHF:Delta         -0.299594  -0.636480   0.026377     4475 0.075319 .  
geneCT_actin:SiteHF:Delta         -0.271953  -0.622655   0.037738     5139 0.097021 .  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1 
```

### Model 1 plots

Normalized gene counts vs delta for each gene    

![](https://cloud.githubusercontent.com/assets/4654474/25708215/d661a6c2-30b3-11e7-83e7-d22f87c64464.jpeg)

![](https://cloud.githubusercontent.com/assets/4654474/25708404/623dc87e-30b4-11e7-8170-1f18baa9b381.jpeg)    

Normalized gene coutns vs bait temp for each gene    

![](https://cloud.githubusercontent.com/assets/4654474/25708265/fc902738-30b3-11e7-82cc-518d52379664.jpeg)     

![](https://cloud.githubusercontent.com/assets/4654474/25708359/444a1dae-30b4-11e7-879b-b52e503fa304.jpeg)




### MOdel 2: Soft normalization-testing effect of Site * bait and Site * delta  

* Fixed effects- Site * bait and Site * delta       
* Global fixed - Rin value and Jdaycont
* Random effects - Chamber

```R
softnorm=mcmc.qpcr(fixed="Site+baittemp.ave+Delta+Site:baittemp.ave+Site:Delta",globalRandom="Cham",data=dd,pr=TRUE,vprior="iw",nitt=50000,geneSpecRes = TRUE,pl=TRUE,normalize=TRUE,controls=c("CT_actin","CT_gapdh","CT_18s"),globalFixed=c("RIN_Value","Jdaycont"))
summary(softnorm)
```

diagnostics   

![](https://cloud.githubusercontent.com/assets/4654474/25703921/8e9ae6fe-30a5-11e7-9afc-33c111b5dc03.jpeg)


### MOdel 2 output: Soft normalization-testing effect of Site * bait and Site * delta  

```R

 Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 19495.47 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample      2.97    2.349      3.6     3995

               ~Cham

     post.mean  l-95% CI u-95% CI eff.samp
Cham   0.04905 5.908e-09   0.1811     1031

 R-structure:  ~idh(gene):units

                post.mean l-95% CI u-95% CI eff.samp
geneNORM.units     1.2051    1.000   1.4300     4700
geneCT_83.units    2.8181    2.410   3.2174     3450
geneCT_70.units    0.4852    0.408   0.5646     4700
geneCT_40.units    0.2422    0.184   0.3026     3298

 Location effects: count ~ gene + RIN_Value + Jdaycont + Site + baittemp.ave + Delta + Site:baittemp.ave + Site:Delta + gene:Site + gene:baittemp.ave + gene:Delta + gene:Site:baittemp.ave + gene:Site:Delta 

                               post.mean   l-95% CI   u-95% CI eff.samp    pMCMC    
(Intercept)                    19.404647  17.634494  21.200173     4700  < 2e-04 ***
geneCT_83                      -8.516793 -11.106321  -6.249250     4700  < 2e-04 ***
geneCT_70                      -4.025889  -5.613463  -2.454108     4700  < 2e-04 ***
geneCT_40                      -6.284329  -7.796068  -4.786409     4700  < 2e-04 ***
RIN_Value                      -0.127996  -0.216956  -0.036955     4700 0.005532 ** 
Jdaycont                        0.007721   0.006342   0.009334     5202  < 2e-04 ***
SiteHF                        -14.568427 -22.367099  -7.392627     4700  < 2e-04 ***
baittemp.ave                   -0.420104  -0.489386  -0.352497     4700  < 2e-04 ***
Delta                           0.099998  -0.091511   0.267359     4700 0.256596    
SiteHF:baittemp.ave             0.584561   0.307684   0.871625     4700  < 2e-04 ***
SiteHF:Delta                   -0.145545  -0.469774   0.165354     4700 0.377872    
geneCT_83:SiteHF              -10.844994 -16.903600  -4.867328     4700 0.000426 ***
geneCT_70:SiteHF               -8.739658 -12.731594  -4.338587     3724  < 2e-04 ***
geneCT_40:SiteHF               -6.926685 -10.760303  -2.717615     4700 0.001702 ** 
geneCT_83:baittemp.ave          0.177069   0.085766   0.269983     4700  < 2e-04 ***
geneCT_70:baittemp.ave          0.066504   0.005830   0.125771     4700 0.028936 *  
geneCT_40:baittemp.ave          0.032970  -0.022780   0.091430     4700 0.260851    
geneCT_83:Delta                 0.116540  -0.005252   0.241345     4459 0.062128 .  
geneCT_70:Delta                 0.055542  -0.029111   0.140927     4700 0.200851    
geneCT_40:Delta                 0.047400  -0.031901   0.128024     4700 0.248511    
geneCT_83:SiteHF:baittemp.ave   0.420970   0.185736   0.641951     4700 0.000426 ***
geneCT_70:SiteHF:baittemp.ave   0.353379   0.185695   0.501578     3709  < 2e-04 ***
geneCT_40:SiteHF:baittemp.ave   0.274517   0.114185   0.418024     4700 0.000851 ***
geneCT_83:SiteHF:Delta         -0.332825  -0.565332  -0.094994     4462 0.005106 ** 
geneCT_70:SiteHF:Delta         -0.253365  -0.414750  -0.080858     4700 0.002979 ** 
geneCT_40:SiteHF:Delta         -0.225062  -0.388829  -0.072069     4099 0.007660 ** 
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```


### Model 2 figures soft normalization: 

![](https://cloud.githubusercontent.com/assets/4654474/25715805/d8f53db0-30ca-11e7-8d77-a30d491d4a68.jpeg)   

![](https://cloud.githubusercontent.com/assets/4654474/25715964/67045834-30cb-11e7-8991-2d7f5ef9f216.jpeg)   
  
### MOdel 3: Classic model-- delta delta ct method

* Fixed effects- Site * bait and Site * delta         
* Global fixed - Rin value and Jdaycont   
* Random effects - Chamber   

```R
classicmod<-mcmc.qpcr.classic(fixed="Site+baittemp.ave+Delta+Site:baittemp.ave+Site:Delta",
globalRandom="Cham",globalFixed=c("RIN_Value","Jdaycont"),data=ctmeth,controls=c("CT_18s","CT_actin","CT_gapdh"),pr=T,pl=T,nitt=50000,singular.ok=TRUE)
summary(classicmod)
diagnostic.mcmc(model=classicmod,col="grey50",cex=0.8)
```

diagnostics     

![](https://cloud.githubusercontent.com/assets/4654474/25707498/b05329ee-30b1-11e7-9412-e7da29def02a.jpeg)

```R
 Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 0 

 G-structure:  ~Cham

     post.mean  l-95% CI u-95% CI eff.samp
Cham   0.02966 2.118e-40   0.0993     65.2

               ~idh(gene):sample

                    post.mean  l-95% CI  u-95% CI eff.samp
geneCT_40.sample    1.739e-01 2.254e-09 5.603e-01   133.59
geneCT_70.sample    5.918e-02 1.993e-22 2.380e-01    79.10
geneCT_83.sample    1.299e+00 7.980e-13 3.519e+00    12.37
geneCT_gapdh.sample 1.302e+04 8.917e-06 7.305e+04    10.73
geneCT_actin.sample 6.696e-01 1.086e-07 5.680e+00    25.63
geneCT_18s.sample   1.497e-01 7.383e-23 1.012e+00    39.48

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units      1705   0.0905    12211    15.75

 Location effects: count ~ 0 + gene + RIN_Value + Jdaycont + +gene:Site + gene:baittemp.ave + gene:Delta + gene:Site:baittemp.ave + gene:Site:Delta 

                                  post.mean   l-95% CI   u-95% CI eff.samp  pMCMC  
geneCT_40                         1.053e+03 -4.823e+02  7.010e+03    1.389 0.9013  
geneCT_70                         1.047e+03 -3.535e+01  4.417e+03    3.277 0.1221  
geneCT_83                         1.008e+03 -1.311e+03  6.766e+03    2.829 0.6357  
geneCT_gapdh                     -1.128e+03 -5.719e+03  2.849e+02    2.809 0.5119  
geneCT_actin                      5.292e+02 -6.079e+02  6.708e+03    6.344 0.7885  
geneCT_18s                        8.093e+02 -1.768e+02  5.053e+03    5.752 0.5260  
RIN_Value                        -3.818e+01 -1.425e+02  5.092e+00    2.314 0.6038  
Jdaycont                          2.761e-01 -7.798e-02  1.353e+00    4.687 0.7302  
geneCT_40:SiteHF                  8.392e+02 -6.353e+03  5.895e+03    8.269 0.2515  
geneCT_70:SiteHF                 -7.961e+02 -5.570e+03  7.284e+02    9.757 0.8664  
geneCT_83:SiteHF                 -7.636e+03 -3.515e+04  1.954e+03    1.364 0.6060  
geneCT_gapdh:SiteHF               1.018e+03 -2.240e+03  7.038e+03    6.232 0.8770  
geneCT_actin:SiteHF               4.782e+03 -1.220e+02  1.899e+04    2.330 0.0532 .
geneCT_18s:SiteHF                -2.538e+03 -1.348e+04  3.907e+02    3.557 0.6443  
geneCT_40:baittemp.ave           -2.699e+01 -2.396e+02  1.678e+01    1.746 0.8979  
geneCT_70:baittemp.ave           -3.338e+01 -1.550e+02  1.780e+00    4.442 0.1553  
geneCT_83:baittemp.ave           -4.613e+01 -2.689e+02  5.122e+01    2.536 0.6370  
geneCT_gapdh:baittemp.ave         4.419e+01 -1.346e+01  2.116e+02    2.447 0.4494  
geneCT_actin:baittemp.ave        -3.250e+01 -2.685e+02  1.354e+01    5.426 0.6026  
geneCT_18s:baittemp.ave          -3.435e+01 -2.003e+02  4.960e+00    5.850 0.3877  
geneCT_40:Delta                  -1.091e+01 -1.783e+02  6.033e+01    8.267 0.8221  
geneCT_70:Delta                  -7.888e+01 -3.479e+02  4.164e+01    1.740 0.2813  
geneCT_83:Delta                   7.219e+01 -3.586e+01  4.061e+02    2.130 0.6072  
geneCT_gapdh:Delta                9.920e+00 -1.528e+02  1.424e+02   10.051 0.7213  
geneCT_actin:Delta                7.125e+01 -4.334e+00  2.508e+02    3.086 0.5511  
geneCT_18s:Delta                  3.309e+00 -9.722e+01  1.049e+02    8.610 0.9102  
geneCT_40:SiteHF:baittemp.ave    -5.552e+01 -3.086e+02  1.975e+02    6.021 0.2191  
geneCT_70:SiteHF:baittemp.ave     2.943e+01 -2.739e+01  2.042e+02   10.090 0.8438  
geneCT_83:SiteHF:baittemp.ave     3.213e+02 -7.232e+01  1.482e+03    1.311 0.6077  
geneCT_gapdh:SiteHF:baittemp.ave -4.338e+01 -2.892e+02  8.391e+01    5.204 0.7523  
geneCT_actin:SiteHF:baittemp.ave -1.486e+02 -5.989e+02  4.479e+00    2.944 0.0489 *
geneCT_18s:SiteHF:baittemp.ave    1.049e+02 -1.349e+01  5.464e+02    3.254 0.6511  
geneCT_40:SiteHF:Delta           -4.646e+01 -3.431e+02  5.487e+01    7.411 0.5566  
geneCT_70:SiteHF:Delta            9.252e+01 -5.145e+01  4.685e+02    2.925 0.3766  
geneCT_83:SiteHF:Delta           -3.837e+02 -1.855e+03  6.381e+01    1.315 0.7043  
geneCT_gapdh:SiteHF:Delta         1.357e+02 -2.403e+01  7.192e+02    3.095 0.7166  
geneCT_actin:SiteHF:Delta         2.340e+01 -1.008e+02  2.067e+02   29.595 0.5136  
geneCT_18s:SiteHF:Delta          -4.692e+00 -1.318e+02  3.094e+02    8.322 0.6728 
```

### HF specific model

```R
```

### DF specific model

```R
```

------

 <div id='id-section77'/> 

### Page 77:  2017-05-05. tressed in nature project: Analyses with recoded data and splitting by site      

Recoded data such that empty ones are -1 to indicate no amplification.  

Approach: For each site fit a 2 model to test for effect of bait temp and Delta, while controlling for RNA quality and Julia Day. 

Models: 

1. Naive- include all genes and test their effects
	* Identify the ones that change and then do a soft normalization for the ones without a significant effect on log(counts)
2. Soft normalization - include genes that are "stable" 

### Harvad forest naive model      

```R
HFnonnorm=mcmc.qpcr(fixed="baittemp.ave+Delta",random="Cham",data=ddHF,pr=TRUE,
vprior="iw",nitt=50000,geneSpecRes =FALSE,pl=TRUE,include=0,controls=c("CT_18s","CT_actin","CT_gapdh"),
globalFixed=c("RIN_Value","Jdaycont"),m.fix=1.5)
summary(HFsoftnorm)
```

### Harvard forest naive model output and figures   
 
```R

 Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 13813.2 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     3.152     2.12    4.266     3129

               ~idh(gene):Cham

                  post.mean l-95% CI u-95% CI eff.samp
geneCT_40.Cham       0.6428   0.2149    1.219     4700
geneCT_70.Cham       0.6768   0.2146    1.301     5346
geneCT_83.Cham       0.6275   0.2179    1.190     4980
geneCT_gapdh.Cham    0.9583   0.2747    1.876     4913
geneCT_actin.Cham    1.0032   0.2746    1.992     4239
geneCT_18s.Cham      0.8761   0.2524    1.717     4700

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units     3.635    3.274    3.997     2666

 Location effects: count ~ 0 + gene + RIN_Value + Jdaycont + +gene:baittemp.ave + gene:Delta 

                           post.mean   l-95% CI   u-95% CI eff.samp    pMCMC    
geneCT_40                  -7.908805 -15.787958   0.375144     4380 0.056596 .  
geneCT_70                  -9.255603 -17.346088  -0.988940     3824 0.025957 *  
geneCT_83                 -15.547922 -23.769932  -7.254690     3660  < 2e-04 ***
geneCT_gapdh               -6.248468 -14.853084   1.718760     4118 0.140851    
geneCT_actin               -3.414440 -11.069957   5.373458     4181 0.417447    
geneCT_18s                  8.630935   1.012052  17.079760     4700 0.035745 *  
RIN_Value                   0.181456  -0.071108   0.419909     4700 0.144681    
Jdaycont                    0.008945   0.006094   0.011705     4700  < 2e-04 ***
geneCT_40:baittemp.ave      0.377825   0.067289   0.698200     4383 0.014468 *  
geneCT_70:baittemp.ave      0.563959   0.250113   0.891069     3642 0.000851 ***
geneCT_83:baittemp.ave      0.710939   0.396867   1.032394     3646  < 2e-04 ***
geneCT_gapdh:baittemp.ave   0.373791   0.065401   0.698623     4181 0.019574 *  
geneCT_actin:baittemp.ave   0.220159  -0.081049   0.546881     3740 0.174043    
geneCT_18s:baittemp.ave     0.230615  -0.080949   0.542483     4322 0.159574    
geneCT_40:Delta            -0.264703  -0.639966   0.094455     4700 0.159574    
geneCT_70:Delta            -0.402686  -0.778682  -0.026056     4116 0.036170 *  
geneCT_83:Delta            -0.228574  -0.584389   0.162200     4700 0.233191    
geneCT_gapdh:Delta         -0.238826  -0.652636   0.161600     4700 0.249787    
geneCT_actin:Delta          0.088106  -0.327377   0.508461     4700 0.673191    
geneCT_18s:Delta           -0.204585  -0.582160   0.196536     4488 0.321702    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

Relationship between normalized log2 abundance vs bait temp.    

![](https://cloud.githubusercontent.com/assets/4654474/25748634/bbbc506c-3179-11e7-8c06-b8e455487a97.jpeg)

### Harvard forest soft normalization     

**based on naive model, I should normalize based on actin and 18s**   

```R
HFsoftnorm2=mcmc.qpcr(fixed="baittemp.ave+Delta",random="Cham",
data=ddHF,pr=TRUE,vprior="iw",nitt=50000,geneSpecRes = TRUE,pl=TRUE,normalize=TRUE,controls=c("CT_18s","CT_actin","CT_gapdh"),
globalFixed=c("RIN_Value","Jdaycont"),m.fix=1.5)
```

### Harvard forest soft norm based on actin and 18s output and figures    

```R
Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 9572.318 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     2.321    1.606    3.237     1579

               ~idh(gene):Cham

                  post.mean l-95% CI u-95% CI eff.samp
geneNORM.Cham        0.6719   0.2049    1.336     5204
geneCT_gapdh.Cham    0.9482   0.2624    1.921     4484
geneCT_83.Cham       0.5887   0.1860    1.131     3916
geneCT_70.Cham       0.5626   0.1961    1.096     4700
geneCT_40.Cham       0.5433   0.1796    1.033     3812

 R-structure:  ~idh(gene):units

                   post.mean l-95% CI u-95% CI eff.samp
geneNORM.units        2.9320   2.1486   3.7880   4700.0
geneCT_gapdh.units    5.1292   3.8858   6.4769    915.2
geneCT_83.units       3.5455   2.6909   4.4898   2183.6
geneCT_70.units       0.9661   0.7120   1.2359   1840.6
geneCT_40.units       0.3372   0.1898   0.4848   2068.1

 Location effects: count ~ gene + RIN_Value + Jdaycont + baittemp.ave + Delta + gene:baittemp.ave + gene:Delta 

                           post.mean   l-95% CI   u-95% CI eff.samp   pMCMC    
(Intercept)                 6.743917  -0.948957  14.211602     4700 0.07830 .  
geneCT_gapdh              -12.334777 -19.925190  -4.609552     3219 0.00128 ** 
geneCT_83                 -21.397507 -28.127556 -14.221628     4304 < 2e-04 ***
geneCT_70                 -15.016724 -20.784617  -9.630071     4700 < 2e-04 ***
geneCT_40                 -14.328587 -19.830423  -9.416300     3147 < 2e-04 ***
RIN_Value                   0.236854   0.023824   0.431321     4700 0.02128 *  
Jdaycont                    0.006872   0.004391   0.009112     4700 < 2e-04 ***
baittemp.ave                0.076859  -0.221565   0.365524     4700 0.60596    
Delta                      -0.026863  -0.417063   0.321977     4700 0.90000    
geneCT_gapdh:baittemp.ave   0.285499  -0.008747   0.567074     3222 0.04936 *  
geneCT_83:baittemp.ave      0.614563   0.360369   0.879077     4236 < 2e-04 ***
geneCT_70:baittemp.ave      0.461884   0.265655   0.675489     3532 < 2e-04 ***
geneCT_40:baittemp.ave      0.301720   0.119143   0.503916     3176 0.00128 ** 
geneCT_gapdh:Delta         -0.203782  -0.678987   0.270997     4700 0.38809    
geneCT_83:Delta            -0.185188  -0.606977   0.228282     4177 0.38128    
geneCT_70:Delta            -0.331567  -0.752459   0.025757     4069 0.09021 .  
geneCT_40:Delta            -0.218815  -0.588207   0.167011     4486 0.26340 
```

![](https://cloud.githubusercontent.com/assets/4654474/25749190/a8fb8810-317b-11e7-9fe9-05f70d446872.jpeg)

### Harvard forest soft normalization based on 3 ref sets: 18s, actin, gapdh    

```R
HFsoftnorm2=mcmc.qpcr(fixed="baittemp.ave+Delta",random="Cham",data=ddHF,
pr=TRUE,vprior="iw",nitt=50000,geneSpecRes = TRUE,pl=TRUE,normalize=TRUE,controls=c("CT_18s","CT_actin","CT_gapdh"),
globalFixed=c("RIN_Value","Jdaycont"),m.fix=1.5)#+Delta
summary(HFsoftnorm2)
```
### Output hf soft norm 18s actin gapdh      

```R
Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 7485.264 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     2.174    1.458    2.981    820.6

               ~idh(gene):Cham

               post.mean l-95% CI u-95% CI eff.samp
geneNORM.Cham     0.7162   0.1672   1.4609     4700
geneCT_83.Cham    0.5460   0.1449   1.0985     4700
geneCT_70.Cham    0.4833   0.1494   0.9621     4278
geneCT_40.Cham    0.4580   0.1279   0.8886     4700

 R-structure:  ~idh(gene):units

                post.mean l-95% CI u-95% CI eff.samp
geneNORM.units     2.1269   1.4886   2.7597     3656
geneCT_83.units    3.6440   2.7244   4.5392     2250
geneCT_70.units    1.0194   0.7435   1.2882     2187
geneCT_40.units    0.2667   0.1538   0.3879     2073

 Location effects: count ~ gene + RIN_Value + Jdaycont + baittemp.ave + Delta + gene:baittemp.ave + gene:Delta 

                        post.mean   l-95% CI   u-95% CI eff.samp    pMCMC    
(Intercept)              4.218044  -2.788091  11.013168     4386 0.235745    
geneCT_83              -18.818872 -25.510002 -12.596046     3065  < 2e-04 ***
geneCT_70              -12.321787 -17.449208  -7.164157     4153  < 2e-04 ***
geneCT_40              -11.642306 -16.547428  -7.329180     3682  < 2e-04 ***
RIN_Value                0.228410   0.032208   0.424987     4464 0.019574 *  
Jdaycont                 0.006782   0.004683   0.009070     4700  < 2e-04 ***
baittemp.ave             0.112626  -0.162648   0.375277     4382 0.410213    
Delta                   -0.052040  -0.414500   0.287017     4700 0.762553    
geneCT_83:baittemp.ave   0.579150   0.340771   0.814229     3073  < 2e-04 ***
geneCT_70:baittemp.ave   0.422447   0.236734   0.609659     3977  < 2e-04 ***
geneCT_40:baittemp.ave   0.263062   0.093489   0.429304     3806 0.000851 ***
geneCT_83:Delta         -0.155925  -0.577289   0.249848     4700 0.455319    
geneCT_70:Delta         -0.297292  -0.677188   0.070924     4700 0.115745    
geneCT_40:Delta         -0.188711  -0.552140   0.177476     4700 0.294468    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

![](https://cloud.githubusercontent.com/assets/4654474/25749190/a8fb8810-317b-11e7-9fe9-05f70d446872.jpeg)


### DF naive model 

```R
DFnaive=mcmc.qpcr(fixed="baittemp.ave+Delta",random=c("Cham"),
data=ddDF,pr=TRUE,vprior="iw",nitt=50000,pl=TRUE,
include=0,controls=c("CT_18s","CT_actin","CT_gapdh"),
globalFixed=c("RIN_Value","Jdaycont"),m.fix=2,geneSpecRes=FALSE)
```
### DF naive model output   

```R
 Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 23250.17 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     6.548    4.926    8.282     4700

               ~idh(gene):Cham

                  post.mean l-95% CI u-95% CI eff.samp
geneCT_40.Cham       0.5158   0.2058   0.9504     4700
geneCT_70.Cham       0.4734   0.1829   0.8395     4700
geneCT_83.Cham       0.4935   0.1951   0.8691     4481
geneCT_gapdh.Cham    0.5111   0.1877   0.9270     4463
geneCT_actin.Cham    0.4577   0.1761   0.8226     4700
geneCT_18s.Cham      0.6851   0.2522   1.2444     4932

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units     2.511    2.327      2.7     3020

 Location effects: count ~ 0 + gene + RIN_Value + Jdaycont + +gene:baittemp.ave + gene:Delta 

                          post.mean  l-95% CI  u-95% CI eff.samp  pMCMC    
geneCT_40                 12.172657  9.546441 14.869301     3528 <2e-04 ***
geneCT_70                 14.130384 11.554427 16.748759     4700 <2e-04 ***
geneCT_83                  9.021455  6.457620 11.670320     4137 <2e-04 ***
geneCT_gapdh              15.475238 12.901377 18.073151     4700 <2e-04 ***
geneCT_actin              12.853326 10.301988 15.450867     4700 <2e-04 ***
geneCT_18s                30.161942 27.519678 32.654623     4700 <2e-04 ***
RIN_Value                 -0.169156 -0.324264 -0.020386     4435 0.0234 *  
Jdaycont                   0.009466  0.007021  0.012421     4700 <2e-04 ***
geneCT_40:baittemp.ave    -0.382081 -0.483717 -0.285517     3242 <2e-04 ***
geneCT_70:baittemp.ave    -0.327654 -0.425382 -0.231107     4700 <2e-04 ***
geneCT_83:baittemp.ave    -0.195752 -0.296805 -0.100031     3904 <2e-04 ***
geneCT_gapdh:baittemp.ave -0.415716 -0.514251 -0.321747     4700 <2e-04 ***
geneCT_actin:baittemp.ave -0.305161 -0.403253 -0.211346     4478 <2e-04 ***
geneCT_18s:baittemp.ave   -0.620586 -0.722245 -0.529481     4700 <2e-04 ***
geneCT_40:Delta            0.163161 -0.131068  0.462919     4700 0.2749    
geneCT_70:Delta            0.145821 -0.143871  0.433128     4700 0.3421    
geneCT_83:Delta            0.196108 -0.091740  0.497139     4700 0.1915    
geneCT_gapdh:Delta         0.232599 -0.079664  0.522301     4632 0.1238    
geneCT_actin:Delta         0.146889 -0.151989  0.423387     5679 0.3226    
geneCT_18s:Delta           0.257939 -0.047918  0.580536     4700 0.1123    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

All genes are going down...I'll try fitting a soft norm with 3 ref genes   

### Duke forest soft normalization with 3 ref genes; 18s, actin, gapdh    

So here i'd test, whether hsps are changing at a faster/slower rate than the ref set along bait temp. 

```R
DFsoftnorm2=mcmc.qpcr(fixed="baittemp.ave+Delta",random="Cham",data=ddDF,
pr=TRUE,vprior="iw",nitt=50000,geneSpecRes = TRUE,pl=TRUE,normalize=TRUE,controls=c("CT_18s","CT_actin","CT_gapdh"),
globalFixed=c("RIN_Value","Jdaycont"),m.fix=1.5)
```

### Duke forest soft norm with 3 refs output and figs    

```R
Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 12844.79 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     5.534    4.151    7.043     2809

               ~idh(gene):Cham

               post.mean l-95% CI u-95% CI eff.samp
geneNORM.Cham     0.4929   0.1500   0.9190     4700
geneCT_83.Cham    0.4214   0.1431   0.8095     4700
geneCT_70.Cham    0.3526   0.1263   0.6596     4700
geneCT_40.Cham    0.4154   0.1451   0.7777     4700

 R-structure:  ~idh(gene):units

                post.mean l-95% CI u-95% CI eff.samp
geneNORM.units     1.3954   1.1122   1.7147     2642
geneCT_83.units    3.2881   2.6955   3.9084     1686
geneCT_70.units    0.7968   0.6352   0.9718     1786
geneCT_40.units    0.9899   0.7637   1.2331     1357

 Location effects: count ~ gene + RIN_Value + Jdaycont + baittemp.ave + Delta + gene:baittemp.ave + gene:Delta 

                        post.mean   l-95% CI   u-95% CI eff.samp   pMCMC    
(Intercept)             22.140225  19.657121  24.435802     3341 < 2e-04 ***
geneCT_83               -9.647201 -12.324195  -6.867827     4700 < 2e-04 ***
geneCT_70               -4.653489  -6.550787  -2.575684     4700 < 2e-04 ***
geneCT_40               -6.618051  -8.703288  -4.501595     3900 < 2e-04 ***
RIN_Value               -0.124016  -0.266650   0.014639     3039 0.08383 .  
Jdaycont                 0.008426   0.005884   0.010823     4626 < 2e-04 ***
baittemp.ave            -0.548965  -0.636750  -0.462001     3619 < 2e-04 ***
Delta                    0.183909  -0.117180   0.453974     4700 0.19830    
geneCT_83:baittemp.ave   0.226478   0.129376   0.329830     4700 < 2e-04 ***
geneCT_70:baittemp.ave   0.099461   0.025047   0.167024     4700 0.00511 ** 
geneCT_40:baittemp.ave   0.047210  -0.027670   0.120743     3815 0.21787    
geneCT_83:Delta          0.024721  -0.231461   0.319838     4700 0.85106    
geneCT_70:Delta         -0.018846  -0.282361   0.227798     4486 0.88255    
geneCT_40:Delta         -0.001807  -0.260435   0.259638     4989 0.99617    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

### Lets look at combined SOFT Normalized model: test for site * bait temp interaction     

model construction:  

```R
mm=mcmc.qpcr(
fixed="Site+baittemp.ave+Delta+Site:baittemp.ave+Site:Delta",
random=c("Cham2"),globalFixed=c("RIN_Value","Jdaycont"),controls=c("CT_actin","CT_gapdh","CT_18s"),include=0,data=dd,pr=TRUE,vprior="iw",nitt=50000,geneSpecRes = FALSE,pl=TRUE)
summary(mm)
```

```R

 Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 37218.42 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     6.274    5.005    7.636    672.2

               ~idh(gene):Cham2

                   post.mean l-95% CI u-95% CI eff.samp
geneCT_40.Cham2       0.4176   0.1896   0.7132     4700
geneCT_70.Cham2       0.3976   0.1855   0.6655     4700
geneCT_83.Cham2       0.3950   0.1747   0.6653     4700
geneCT_18s.Cham2      0.5834   0.2599   1.0168     4700
geneCT_gapdh.Cham2    0.5542   0.2432   0.9563     4700
geneCT_actin.Cham2    0.4558   0.1924   0.7927     4700

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units     2.898    2.713    3.064     3033

 Location effects: count ~ 0 + gene + RIN_Value + Jdaycont + +gene:Site + gene:baittemp.ave + gene:Delta + gene:Site:baittemp.ave + gene:Site:Delta 

                                  post.mean   l-95% CI   u-95% CI eff.samp    pMCMC    
geneCT_40                         11.822062   9.005032  14.432795     3697  < 2e-04 ***
geneCT_70                         13.794258  11.133172  16.508019     3917  < 2e-04 ***
geneCT_83                          8.620266   5.721065  11.238058     4442  < 2e-04 ***
geneCT_18s                        29.771465  27.161697  32.606825     4700  < 2e-04 ***
geneCT_gapdh                      15.096474  12.192159  17.705313     3620  < 2e-04 ***
geneCT_actin                      12.489472   9.796855  15.269530     4465  < 2e-04 ***
RIN_Value                         -0.113658  -0.265027   0.028872     4482 0.123404    
Jdaycont                           0.010066   0.007892   0.012371     4700  < 2e-04 ***
geneCT_40:SiteHF                 -21.975881 -32.487062 -11.289222     4700  < 2e-04 ***
geneCT_70:SiteHF                 -25.065039 -35.036285 -14.541056     4700  < 2e-04 ***
geneCT_83:SiteHF                 -25.060015 -35.575939 -14.371653     4700  < 2e-04 ***
geneCT_18s:SiteHF                -23.153648 -33.472287 -12.514714     4955  < 2e-04 ***
geneCT_gapdh:SiteHF              -23.363184 -33.619975 -12.762416     4700  < 2e-04 ***
geneCT_actin:SiteHF              -17.705744 -28.052297  -7.287786     4700 0.000426 ***
geneCT_40:baittemp.ave            -0.388470  -0.485090  -0.282392     3288  < 2e-04 ***
geneCT_70:baittemp.ave            -0.335051  -0.433817  -0.235877     4026  < 2e-04 ***
geneCT_83:baittemp.ave            -0.200785  -0.296581  -0.091502     4205  < 2e-04 ***
geneCT_18s:baittemp.ave           -0.625554  -0.726443  -0.527732     4700  < 2e-04 ***
geneCT_gapdh:baittemp.ave         -0.421660  -0.523804  -0.322599     3495  < 2e-04 ***
geneCT_actin:baittemp.ave         -0.311388  -0.411320  -0.209647     4582  < 2e-04 ***
geneCT_40:Delta                    0.149857  -0.141126   0.431712     4700 0.306809    
geneCT_70:Delta                    0.140155  -0.153728   0.429957     4700 0.328511    
geneCT_83:Delta                    0.187249  -0.098200   0.467758     4700 0.196170    
geneCT_18s:Delta                   0.248577  -0.058683   0.554538     4700 0.119574    
geneCT_gapdh:Delta                 0.227091  -0.084731   0.525936     4700 0.148511    
geneCT_actin:Delta                 0.139773  -0.171241   0.414983     4700 0.358298    
geneCT_40:SiteHF:baittemp.ave      0.892454   0.505511   1.308730     4700  < 2e-04 ***
geneCT_70:SiteHF:baittemp.ave      1.016732   0.626500   1.401237     4550  < 2e-04 ***
geneCT_83:SiteHF:baittemp.ave      0.998540   0.591417   1.386713     4700  < 2e-04 ***
geneCT_18s:SiteHF:baittemp.ave     0.970596   0.565018   1.357887     4991  < 2e-04 ***
geneCT_gapdh:SiteHF:baittemp.ave   0.913561   0.529752   1.315875     4700  < 2e-04 ***
geneCT_actin:SiteHF:baittemp.ave   0.654713   0.280649   1.062279     4700 0.000851 ***
geneCT_40:SiteHF:Delta            -0.490708  -0.949807   0.053141     4480 0.058723 .  
geneCT_70:SiteHF:Delta            -0.609615  -1.101972  -0.106584     4700 0.017447 *  
geneCT_83:SiteHF:Delta            -0.542245  -1.046443  -0.056325     4924 0.034043 *  
geneCT_18s:SiteHF:Delta           -0.518337  -1.054346  -0.012797     4700 0.052340 .  
geneCT_gapdh:SiteHF:Delta         -0.536793  -1.043260   0.003550     4700 0.049787 *  
geneCT_actin:SiteHF:Delta         -0.194960  -0.696403   0.310583     4700 0.459574    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```



### Lets look at combined SOFT Normalized model: test for site * bait temp interaction     

...while controlling for 3 reference genes.... actin, 18s, gapdh


Model construction   

```R
softnorm=mcmc.qpcr(fixed="Site+baittemp.ave+Delta+Site:baittemp.ave+Site:Delta",
globalRandom="Cham2",data=dd,pr=TRUE,vprior="iw",nitt=50000,
geneSpecRes =FALSE,pl=TRUE,normalize=TRUE,controls=c("CT_actin","CT_gapdh","CT_18s"),
globalFixed=c("RIN_Value","Jdaycont"),m.fix=2)
```


output:   

```R
 Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 20307.05 

 G-structure:  ~sample

       post.mean l-95% CI u-95% CI eff.samp
sample     4.488    3.603    5.441     4700

               ~Cham2

      post.mean  l-95% CI u-95% CI eff.samp
Cham2   0.05307 2.383e-09   0.2132     1238

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units     1.597    1.482    1.725     2090

 Location effects: count ~ gene + RIN_Value + Jdaycont + Site + baittemp.ave + Delta + Site:baittemp.ave + Site:Delta + gene:Site + gene:baittemp.ave + gene:Delta + gene:Site:baittemp.ave + gene:Site:Delta 

                               post.mean   l-95% CI   u-95% CI eff.samp    pMCMC    
(Intercept)                    20.765616  18.518837  23.010194     3192  < 2e-04 ***
geneCT_83                      -9.101701 -11.154752  -7.040275     4334  < 2e-04 ***
geneCT_70                      -4.481281  -6.436953  -2.265938     4700 0.000426 ***
geneCT_40                      -6.981607  -9.163210  -4.942460     3594  < 2e-04 ***
RIN_Value                      -0.009514  -0.147358   0.121073     4269 0.881702    
Jdaycont                        0.008601   0.006671   0.010532     3976  < 2e-04 ***
SiteHF                        -20.033476 -29.109914 -11.837664     4176  < 2e-04 ***
baittemp.ave                   -0.518049  -0.608928  -0.436984     2785  < 2e-04 ***
Delta                           0.160806  -0.037144   0.380270     4700 0.124681    
SiteHF:baittemp.ave             0.792163   0.465078   1.113011     4215  < 2e-04 ***
SiteHF:Delta                   -0.272132  -0.629893   0.103980     4700 0.155319    
geneCT_83:SiteHF               -8.761300 -13.821726  -3.439695     4046 0.000426 ***
geneCT_70:SiteHF               -6.710382 -11.733099  -1.388099     4244 0.007234 ** 
geneCT_40:SiteHF               -3.267049  -8.625278   1.738584     3598 0.216596    
geneCT_83:baittemp.ave          0.204517   0.127550   0.283634     4319  < 2e-04 ***
geneCT_70:baittemp.ave          0.090717   0.009584   0.167971     4700 0.028511 *  
geneCT_40:baittemp.ave          0.059822  -0.017174   0.143952     3497 0.146383    
geneCT_83:Delta                 0.037275  -0.076766   0.147296     4403 0.512766    
geneCT_70:Delta                -0.010336  -0.125017   0.093081     4700 0.853191    
geneCT_40:Delta                -0.005442  -0.115911   0.108092     4444 0.934894    
geneCT_83:SiteHF:baittemp.ave   0.335731   0.128712   0.518240     4106 0.000426 ***
geneCT_70:SiteHF:baittemp.ave   0.286206   0.090584   0.478491     4299 0.003404 ** 
geneCT_40:SiteHF:baittemp.ave   0.147327  -0.038703   0.354963     3555 0.130638    
geneCT_83:SiteHF:Delta         -0.183220  -0.391793   0.014840     3846 0.081277 .  
geneCT_70:SiteHF:Delta         -0.281034  -0.475357  -0.071999     3531 0.006809 ** 
geneCT_40:SiteHF:Delta         -0.165264  -0.366331   0.039326     3576 0.109787    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```

Showing effect of bait temp on gxp     

LInear fit   
![](https://cloud.githubusercontent.com/assets/4654474/25708359/444a1dae-30b4-11e7-879b-b52e503fa304.jpeg)

Loess fit   
![](https://cloud.githubusercontent.com/assets/4654474/25760911/e146d082-31a6-11e7-87cb-433d99137296.jpeg)


### Refitting classic mod   

Parsing data
```R
ctmeth<-cq2log(data=warm,genecols=c(14:19),condcols=c(13,5:6,1:4,20:27),noamp=37,effic=amp)
ctmeth$RIN_Value<-as.numeric(as.character(ctmeth$RIN_Value))
ctmeth$Delta<-as.numeric(as.character(ctmeth$Delta))
ctmeth$baittemp.ave<-as.numeric(as.character(ctmeth$baittemp.ave))
ctmeth$Jdaycont<-as.numeric(as.character(ctmeth$Jdaycont))
str(ctmeth)
```

Model fitting    

```R
classicmod<-mcmc.qpcr.classic(fixed="Site+baittemp.ave+Delta+Site:baittemp.ave+Site:Delta",globalRandom="Cham",globalFixed=c("RIN_Value","Jdaycont"),data=ctmeth,controls=c("CT_actin","CT_gapdh"),pr=T,pl=T,nitt=50000)
```

diagnostics   

![](https://cloud.githubusercontent.com/assets/4654474/25761246/31ff4328-31a8-11e7-91a5-fdba62590b8d.jpeg)


output:    

```R
Iterations = 3001:49991
 Thinning interval  = 10
 Sample size  = 4700 

 DIC: 10316.62 

 G-structure:  ~idh(gene):Cham

                  post.mean   l-95% CI  u-95% CI eff.samp
geneCT_40.Cham    2.168e-03  2.843e-63 1.743e-08   134.26
geneCT_70.Cham    5.678e-02  5.842e-49 3.045e-01    78.55
geneCT_83.Cham    6.201e-18 6.381e-108 4.589e-33     0.00
geneCT_gapdh.Cham 1.519e-03  1.681e-39 4.673e-03   364.01
geneCT_actin.Cham 7.960e-05  2.641e-49 2.683e-07   615.45
geneCT_18s.Cham   3.017e-05  2.951e-56 1.972e-09   401.62

               ~idh(gene):sample

                    post.mean l-95% CI u-95% CI eff.samp
geneCT_40.sample       1.8769   1.3605    2.436   4700.0
geneCT_70.sample       1.5604   1.0790    2.072    706.6
geneCT_83.sample       2.1509   1.5688    2.778   4700.0
geneCT_gapdh.sample    0.7484   0.4714    1.062   4004.1
geneCT_actin.sample    1.7336   1.2430    2.231   5279.4
geneCT_18s.sample      2.6897   2.0524    3.357   4700.0

 R-structure:  ~units

      post.mean l-95% CI u-95% CI eff.samp
units     1.676     1.55    1.797     4988

 Location effects: count ~ 0 + gene + RIN_Value + Jdaycont + +gene:Site + gene:baittemp.ave + gene:Delta + gene:Site:baittemp.ave + gene:Site:Delta 

                                  post.mean   l-95% CI   u-95% CI eff.samp   pMCMC    
geneCT_40                         1.399e+00 -7.993e-01  3.874e+00   4700.0 0.24511    
geneCT_70                         7.141e-01 -1.508e+00  2.982e+00   4700.0 0.52851    
geneCT_83                        -1.465e+00 -4.054e+00  8.896e-01   4700.0 0.23617    
geneCT_gapdh                      3.971e-02 -1.760e+00  1.890e+00   4955.7 0.97447    
geneCT_actin                     -1.757e+00 -3.932e+00  3.514e-01   4700.0 0.10468    
geneCT_18s                        5.140e+00  2.722e+00  7.495e+00   5052.1 < 2e-04 ***
RIN_Value                        -3.748e-02 -8.486e-02  1.419e-02   4700.0 0.14894    
Jdaycont                         -4.573e-05 -7.047e-04  6.020e-04   4700.0 0.87787    
geneCT_40:SiteHF                 -3.176e+00 -9.216e+00  2.793e+00   4700.0 0.29957    
geneCT_70:SiteHF                 -7.646e+00 -1.329e+01 -1.757e+00   4700.0 0.01064 *  
geneCT_83:SiteHF                 -1.059e+01 -1.697e+01 -4.130e+00   4542.6 0.00128 ** 
geneCT_gapdh:SiteHF              -2.480e+00 -7.021e+00  2.227e+00   4700.0 0.29702    
geneCT_actin:SiteHF               1.344e+00 -4.488e+00  7.702e+00   4924.3 0.65957    
geneCT_18s:SiteHF                -3.452e+00 -1.021e+01  3.342e+00   4700.0 0.32809    
geneCT_40:baittemp.ave           -5.011e-02 -1.392e-01  3.826e-02   4700.0 0.27532    
geneCT_70:baittemp.ave           -2.405e-02 -1.078e-01  6.303e-02   4700.0 0.57532    
geneCT_83:baittemp.ave            5.755e-02 -3.511e-02  1.517e-01   4700.0 0.22085    
geneCT_gapdh:baittemp.ave         1.086e-02 -6.153e-02  7.812e-02   4940.6 0.74426    
geneCT_actin:baittemp.ave         9.179e-02  1.157e-02  1.746e-01   4700.0 0.02766 *  
geneCT_18s:baittemp.ave          -2.113e-01 -3.014e-01 -1.217e-01   4891.3 < 2e-04 ***
geneCT_40:Delta                  -4.476e-02 -1.834e-01  9.682e-02   4700.0 0.52851    
geneCT_70:Delta                  -7.246e-02 -2.236e-01  8.242e-02    587.3 0.33064    
geneCT_83:Delta                   1.839e-02 -1.273e-01  1.684e-01   4700.0 0.79489    
geneCT_gapdh:Delta                5.717e-03 -1.051e-01  1.103e-01   4700.0 0.92681    
geneCT_actin:Delta               -6.490e-02 -2.030e-01  7.087e-02   4700.0 0.34085    
geneCT_18s:Delta                  1.006e-01 -5.750e-02  2.662e-01   4700.0 0.22383    
geneCT_40:SiteHF:baittemp.ave     1.466e-01 -7.800e-02  3.808e-01   4700.0 0.20468    
geneCT_70:SiteHF:baittemp.ave     3.269e-01  1.113e-01  5.461e-01   4700.0 0.00426 ** 
geneCT_83:SiteHF:baittemp.ave     4.085e-01  1.596e-01  6.436e-01   4700.0 0.00170 ** 
geneCT_gapdh:SiteHF:baittemp.ave  8.618e-02 -9.165e-02  2.573e-01   4700.0 0.33617    
geneCT_actin:SiteHF:baittemp.ave -1.129e-01 -3.417e-01  1.177e-01   4924.1 0.31745    
geneCT_18s:SiteHF:baittemp.ave    2.030e-01 -5.703e-02  4.546e-01   4700.0 0.12128    
geneCT_40:SiteHF:Delta           -7.663e-02 -3.230e-01  1.895e-01   5225.8 0.54766    
geneCT_70:SiteHF:Delta           -2.160e-01 -4.589e-01  2.999e-02   4700.0 0.08766 .  
geneCT_83:SiteHF:Delta           -1.278e-01 -3.876e-01  1.310e-01   4700.0 0.33447    
geneCT_gapdh:SiteHF:Delta        -9.679e-02 -2.957e-01  9.375e-02   4700.0 0.31234    
geneCT_actin:SiteHF:Delta         2.710e-01  2.563e-02  5.100e-01   4700.0 0.02979 *  
geneCT_18s:SiteHF:Delta          -2.297e-01 -5.112e-01  6.177e-02   4700.0 0.11957    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```


# Manually calculating delta delta CT method   

### data parsing 

```R
x<-read.csv("../Data/20170427_final_dataset_2013_2014_years_HF_DF_del.csv",skip=10)
str(x)
x$baittemp.ave<-apply(x[,8:11],1,mean,na.rm=TRUE)
x$Cham<-as.factor(as.character(x$Cham))
x2<-inner_join(x,chd,by=c("Cham","Site"))

#julian day
v2<-paste(substr(x2$Collection.Date,1,4),"-",substr(x2$Collection.Date,5,6),"-",substr(x2$Collection.Date,7,8))
x2$Date<-as.character(gsub(" ", "", v2, fixed = TRUE))
x2$JulianDay<-as.numeric(format(as.Date(x2$Date),"%j"))
x2$Jdaycont<-ifelse(x2$Year_collect==2013,x2$JulianDay,x2$JulianDay+365)
x2<-x2[!is.na(x2$RIN_Value),]
x3<-na.omit(x2[,-8:-11])

hkg<-cbind(x3$CT_18s,x3$CT_actin,x3$CT_gapdh)
x3$geomean<-apply(hkg,1,function(x){exp(mean(log(x)))})

globalmean<-apply(x3[,10:15],2,mean)
deltaCTnum<-data.frame(t(apply(x3[,10:15],1,function(x){globalmean-x})))
deltaCTden<-mean(x3$geomean)-x3$geomean

##calculating delta delta ct
gxp<-data.frame(log2(apply(deltaCTnum,2,function(x){2^(x)/2^(deltaCTden)})))
names(gxp)<-c("fc18s","fchsp40","fchsp70","fchsp83","fcactin","fcgapdh")

findat<-data.frame(x3,gxp)
str(findat)

##convert to long format 
findat.long<-gather(findat,gene,FC,fc18s:fcgapdh)

```


### Anovas    

```R
fullmod2<-aov(FC~RIN_Value+Jdaycont+gene*Site*baittemp.ave+gene*Site*Delta,data=findat.long)
tep:  AIC=1611.21
FC ~ RIN_Value + Jdaycont + gene + Site + baittemp.ave + Delta + 
    gene:Site + gene:baittemp.ave + Site:baittemp.ave + Site:Delta

                         Df Sum of Sq    RSS    AIC
<none>                                4385.2 1611.2
- RIN_Value               1     7.679 4392.9 1611.4
- Site:baittemp.ave       1    14.268 4399.4 1613.2
- Site:Delta              1    14.417 4399.6 1613.3
+ gene:Site:baittemp.ave  5    24.236 4360.9 1614.4
+ gene:Delta              5     2.085 4383.1 1620.6
- gene:Site               5   185.321 4570.5 1652.4
- Jdaycont                1   197.915 4583.1 1663.8
- gene:baittemp.ave       5   306.347 4691.5 1684.7
                    Df Sum Sq Mean Sq F value   Pr(>F)    
RIN_Value            1     46    46.0  12.719 0.000376 ***
Jdaycont             1    115   114.8  31.747 2.18e-08 ***
gene                 5      0     0.0   0.000 1.000000    
Site                 1     63    63.3  17.505 3.07e-05 ***
baittemp.ave         1    460   460.4 127.347  < 2e-16 ***
Delta                1      1     0.9   0.262 0.608759    
gene:Site            5    161    32.1   8.889 2.66e-08 ***
gene:baittemp.ave    5    306    61.3  16.948 3.18e-16 ***
Site:baittemp.ave    1      5     5.4   1.505 0.220175    
Site:Delta           1     14    14.4   3.988 0.046046 *  
Residuals         1213   4385     3.6                     
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1
```


### Statistics: linear models   

```R
fullmod3<-lm(FC~RIN_Value+Jdaycont+gene*Site*baittemp.ave+
gene*Site*Delta,data=findat.long)

Call:
lm(formula = FC ~ RIN_Value + Jdaycont + gene * Site * baittemp.ave + 
    gene * Site * Delta, data = findat.long)

Residuals:
    Min      1Q  Median      3Q     Max 
-9.8693 -1.0781 -0.0503  1.0091  9.5413 

Coefficients:
                                  Estimate Std. Error t value Pr(>|t|)    
(Intercept)                      2.453e+00  1.625e+00   1.510 0.131335    
RIN_Value                        4.992e-02  3.426e-02   1.457 0.145368    
Jdaycont                        -3.134e-03  4.237e-04  -7.397 2.61e-13 ***
genefcactin                     -5.834e+00  2.294e+00  -2.543 0.011114 *  
genefcgapdh                     -7.245e+00  2.294e+00  -3.158 0.001627 ** 
genefchsp40                     -9.305e+00  2.294e+00  -4.056 5.31e-05 ***
genefchsp70                     -1.310e+01  2.294e+00  -5.709 1.43e-08 ***
genefchsp83                     -1.592e+01  2.294e+00  -6.939 6.46e-12 ***
SiteHF                           1.644e+00  4.042e+00   0.407 0.684228    
baittemp.ave                    -7.262e-02  6.232e-02  -1.165 0.244167    
Delta                            5.386e-02  8.458e-02   0.637 0.524364    
genefcactin:SiteHF              -1.749e+00  5.672e+00  -0.308 0.757798    
genefcgapdh:SiteHF               8.873e+00  5.672e+00   1.565 0.117957    
genefchsp40:SiteHF               8.031e-01  5.672e+00   0.142 0.887418    
genefchsp70:SiteHF               2.035e+00  5.672e+00   0.359 0.719756    
genefchsp83:SiteHF              -5.688e+00  5.672e+00  -1.003 0.316148    
genefcactin:baittemp.ave         2.601e-01  8.791e-02   2.958 0.003156 ** 
genefcgapdh:baittemp.ave         3.145e-01  8.791e-02   3.578 0.000361 ***
genefchsp40:baittemp.ave         3.783e-01  8.791e-02   4.303 1.82e-05 ***
genefchsp70:baittemp.ave         5.163e-01  8.791e-02   5.873 5.55e-09 ***
genefchsp83:baittemp.ave         6.291e-01  8.791e-02   7.156 1.45e-12 ***
SiteHF:baittemp.ave             -1.865e-02  1.534e-01  -0.122 0.903285    
genefcactin:Delta               -1.346e-01  1.195e-01  -1.126 0.260301    
genefcgapdh:Delta               -1.052e-01  1.195e-01  -0.880 0.378968    
genefchsp40:Delta               -1.222e-01  1.195e-01  -1.022 0.306847    
genefchsp70:Delta               -1.268e-01  1.195e-01  -1.060 0.289178    
genefchsp83:Delta               -1.001e-01  1.195e-01  -0.837 0.402580    
SiteHF:Delta                    -1.694e-01  1.597e-01  -1.061 0.288975    
genefcactin:SiteHF:baittemp.ave -6.741e-02  2.151e-01  -0.313 0.754003    
genefcgapdh:SiteHF:baittemp.ave -4.580e-01  2.151e-01  -2.129 0.033428 *  
genefchsp40:SiteHF:baittemp.ave -1.128e-01  2.151e-01  -0.525 0.599933    
genefchsp70:SiteHF:baittemp.ave -1.472e-01  2.151e-01  -0.684 0.493905    
genefchsp83:SiteHF:baittemp.ave  1.194e-01  2.151e-01   0.555 0.578921    
genefcactin:SiteHF:Delta         5.114e-01  2.255e-01   2.267 0.023544 *  
genefcgapdh:SiteHF:Delta         4.643e-01  2.255e-01   2.058 0.039760 *  
genefchsp40:SiteHF:Delta         3.145e-01  2.255e-01   1.394 0.163497    
genefchsp70:SiteHF:Delta         3.197e-01  2.255e-01   1.418 0.156581    
genefchsp83:SiteHF:Delta         1.927e-01  2.255e-01   0.854 0.393119    
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 1.902 on 1198 degrees of freedom
Multiple R-squared:  0.2203,	Adjusted R-squared:  0.1962 
F-statistic: 9.147 on 37 and 1198 DF,  p-value: < 2.2e-16
```

With model selection    

```R
Call:
lm(formula = FC ~ RIN_Value + Jdaycont + gene + Site + baittemp.ave + 
    Delta + gene:Site + gene:baittemp.ave + Site:baittemp.ave + 
    Site:Delta, data = findat.long)

Residuals:
     Min       1Q   Median       3Q      Max 
-10.2877  -1.0909  -0.0054   1.0415  10.0486 

Coefficients:
                           Estimate Std. Error t value Pr(>|t|)    
(Intercept)               2.403e+00  1.476e+00   1.628 0.103837    
RIN_Value                 4.992e-02  3.425e-02   1.457 0.145257    
Jdaycont                 -3.134e-03  4.236e-04  -7.399 2.55e-13 ***
genefcactin              -6.521e+00  2.039e+00  -3.199 0.001416 ** 
genefcgapdh              -5.735e+00  2.039e+00  -2.813 0.004988 ** 
genefchsp40              -9.209e+00  2.039e+00  -4.517 6.89e-06 ***
genefchsp70              -1.281e+01  2.039e+00  -6.284 4.60e-10 ***
genefchsp83              -1.682e+01  2.039e+00  -8.249 4.11e-16 ***
SiteHF                    3.943e+00  1.732e+00   2.277 0.022962 *  
baittemp.ave             -6.304e-02  5.609e-02  -1.124 0.261332    
Delta                    -4.429e-02  3.462e-02  -1.280 0.200961    
genefcactin:SiteHF       -2.324e+00  4.035e-01  -5.761 1.06e-08 ***
genefcgapdh:SiteHF       -2.496e+00  4.035e-01  -6.185 8.46e-10 ***
genefchsp40:SiteHF       -1.520e+00  4.035e-01  -3.767 0.000173 ***
genefchsp70:SiteHF       -1.215e+00  4.035e-01  -3.012 0.002647 ** 
genefchsp83:SiteHF       -1.964e+00  4.035e-01  -4.868 1.28e-06 ***
genefcactin:baittemp.ave  2.758e-01  7.720e-02   3.572 0.000368 ***
genefcgapdh:baittemp.ave  2.487e-01  7.720e-02   3.222 0.001306 ** 
genefchsp40:baittemp.ave  3.651e-01  7.720e-02   4.730 2.51e-06 ***
genefchsp70:baittemp.ave  4.955e-01  7.720e-02   6.419 1.96e-10 ***
genefchsp83:baittemp.ave  6.556e-01  7.720e-02   8.493  < 2e-16 ***
SiteHF:baittemp.ave      -1.296e-01  6.526e-02  -1.987 0.047184 *  
SiteHF:Delta              1.310e-01  6.560e-02   1.997 0.046046 *  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 1.901 on 1213 degrees of freedom
Multiple R-squared:  0.2109,	Adjusted R-squared:  0.1966 
F-statistic: 14.74 on 22 and 1213 DF,  p-value: < 2.2e-16
```

### Statistics with random effects   

```R
fullmod5<-lme(FC~RIN_Value+Jdaycont+gene*Site*baittemp.ave+gene*Site*Delta,random=~1|Cham2,data=findat.long)
summary(fullmod5)
anova(fullmod5)
                       numDF denDF   F-value p-value
(Intercept)                1  1175   0.03058  0.8612
RIN_Value                  1  1175  13.28986  0.0003
Jdaycont                   1  1175  35.65154  <.0001
gene                       5  1175   0.00000  1.0000
Site                       1    23  12.21883  0.0019
baittemp.ave               1  1175 122.07718  <.0001
Delta                      1    23   0.13595  0.7157
gene:Site                  5  1175   8.95546  <.0001
gene:baittemp.ave          5  1175  17.07517  <.0001
Site:baittemp.ave          1  1175   2.17829  0.1402
gene:Delta                 5  1175   0.11621  0.9888
Site:Delta                 1    23   2.95385  0.0991
gene:Site:baittemp.ave     5  1175   1.39975  0.2216
gene:Site:Delta            5  1175   1.37753  0.2300
```

### Figures   

Differences between hf and df with bait and delta 
![](https://cloud.githubusercontent.com/assets/4654474/25764055/11249c28-31b4-11e7-84c8-3de4520a210f.jpeg)



**Parse out by gene**    

![](https://cloud.githubusercontent.com/assets/4654474/25764085/3174cdc2-31b4-11e7-8730-f33fa32fe9f8.jpeg)


![](https://cloud.githubusercontent.com/assets/4654474/25764790/8beded1c-31b7-11e7-8297-c3c2c8979004.jpeg)

**loess**    
  
![](https://cloud.githubusercontent.com/assets/4654474/25764549/5fce2e28-31b6-11e7-8eef-aee830d24eef.jpeg)    


Difference in FC of different genes between sites

![](https://cloud.githubusercontent.com/assets/4654474/25765217/d78c273c-31b9-11e7-9633-8cc24fa00465.jpeg)

------

 <div id='id-section78'/> 

### Page 78:  2017-05-08. quantifying gene stability with geNorm in the SLqPCR package      

MCMC.qpcr package can convert raw ct values into a table for geNORM. geNORM evaluates gene expression stability. Below is the code....    

* for the 3 reference genes    

```R
stable<-cq2genorm(warm,genes=c("CT_actin","CT_gapdh","CT_18s"),noamp=37,effic=amp)
head(stable)
```

* for all 6 genes    

```R
all<-cq2genorm(warm,genes=c("CT_actin","CT_gapdh","CT_18s","CT_70","CT_83","CT_40"),noamp=37,effic=amp)
```

### The selectHKgenes function: 

```
This function can be used to determine a set of reference/housekeeping (HK) genes for gene expression experiments. The default method "Vandesompele" was proposed by Vandesompele et al. (2002).

If method = "Vandesompele" a list with the following components is returnd
ranking	ranking of genes from best to worst where the two most stable genes cannot be ranked
variation pairwise variation during stepwise selection
meanM average expression stability M
```

What is stability M? How is it measured?   

From their paper (Vandesompele); they determine the pariwise variation for every control gene as the SD of log transformed expression ratios. They define this as gene-stability measure M -- average variation of a particular gene with all other control genes. 

* LOW M = most stable. 


1. Evaluating stability for 3 ref sets:    

```R
selectHKgenes(stable,geneSymbol=c("actin","gapdh","18s"))
###############################################################
Step  1 :
gene expression stability values M:
   gapdh    actin      18s 
4.109859 4.417064 5.242474 
average expression stability M:	 4.589799 
gene with lowest stability (largest M value):	 18s 
Pairwise variation, ( 2 / 3 ):	 1.662695 
###############################################################
Step  2 :
gene expression stability values M:
  actin   gapdh 
3.28445 3.28445 
average expression stability M:	 3.28445 
$ranking
      1       1       3 
"actin" "gapdh"   "18s" 

$variation
     2/3 
1.662695 

$meanM
       3        2 
4.589799 3.284450 
```

2. Eval all 6 genes     

```R
selectHKgenes(all,geneSymbol=c("CT_actin","CT_gapdh","CT_18s","CT_70","CT_83","CT_40"))
###############################################################
Step  1 :
gene expression stability values M:
   CT_70    CT_40    CT_83 CT_gapdh CT_actin   CT_18s 
3.213355 3.274018 3.618803 3.692725 3.713486 4.898314 
average expression stability M:	 3.735117 
gene with lowest stability (largest M value):	 CT_18s 
Pairwise variation, ( 5 / 6 ):	 0.7470254 
###############################################################
Step  2 :
gene expression stability values M:
   CT_70    CT_40 CT_actin    CT_83 CT_gapdh 
2.880148 2.963412 3.254438 3.287504 3.382089 
average expression stability M:	 3.153518 
gene with lowest stability (largest M value):	 CT_gapdh 
Pairwise variation, ( 4 / 5 ):	 0.5661937 
###############################################################
Step  3 :
gene expression stability values M:
   CT_70    CT_40    CT_83 CT_actin 
2.758816 2.838390 3.162911 3.244434 
average expression stability M:	 3.001138 
gene with lowest stability (largest M value):	 CT_actin 
Pairwise variation, ( 3 / 4 ):	 0.702729 
###############################################################
Step  4 :
gene expression stability values M:
   CT_70    CT_40    CT_83 
2.498380 2.659493 3.115653 
average expression stability M:	 2.757842 
gene with lowest stability (largest M value):	 CT_83 
Pairwise variation, ( 2 / 3 ):	 0.9826598 
###############################################################
Step  5 :
gene expression stability values M:
  CT_70   CT_40 
2.04222 2.04222 
average expression stability M:	 2.04222 
$ranking
         1          1          3          4          5          6 
   "CT_70"    "CT_40"    "CT_83" "CT_actin" "CT_gapdh"   "CT_18s" 

$variation
      5/6       4/5       3/4       2/3 
0.7470254 0.5661937 0.7027290 0.9826598 

$meanM
       6        5        4        3        2 
3.735117 3.153518 3.001138 2.757842 2.042220 
```

It looks like the ref set is more variable than the gene set we're interested in. Anyway, when looking at the control set, gapdh and actin are the most stable. I should try to calculate relative expression using the geometric mean of these 2. 


OK, lets look at the model without including 18s as a HKG.   

data parsing: i just need to alter this line of code  

```R
#hkg<-cbind(x3$CT_18s,x3$CT_actin,x3$CT_gapdh)
hkg<-cbind(x3$CT_actin,x3$CT_gapdh)

```

### Statistics: linear mixed effects models   

```R
fullmod5<-lme(FC~RIN_Value+Jdaycont+gene*Site*baittemp.ave+gene*Site*Delta,random=~1|Cham2/Vial.me,data=findat.long)
anova(fullmod5)
                     numDF denDF   F-value p-value
(Intercept)                1  1006  0.097464  0.7550
RIN_Value                  1  1006  0.002386  0.9611
Jdaycont                   1  1006  7.053760  0.0080
gene                       5  1006  0.000000  1.0000
Site                       1    23 18.638167  0.0003
baittemp.ave               1  1006  4.349922  0.0373
Delta                      1    23  0.094197  0.7617
gene:Site                  5  1006  9.628759  <.0001
gene:baittemp.ave          5  1006 18.358946  <.0001
Site:baittemp.ave          1   169  1.410071  0.2367
gene:Delta                 5  1006  0.124950  0.9868
Site:Delta                 1    23  3.457577  0.0758
gene:Site:baittemp.ave     5  1006  1.504986  0.1855
gene:Site:Delta            5  1006  1.481099  0.1933
```

Ok, let's do model selection: forward and backwards:   

```R
anova(summary(stepAIC(fullmod5,direction="both")))
                  numDF denDF   F-value p-value
(Intercept)           1  1022  0.061735  0.8038
Jdaycont              1  1022  4.757535  0.0294
gene                  5  1022  0.000000  1.0000
Site                  1    23 24.925993  <.0001
baittemp.ave          1  1022  4.687937  0.0306
Delta                 1    23  0.080154  0.7796
gene:Site             5  1022  9.591095  <.0001
gene:baittemp.ave     5  1022 18.287133  <.0001
Site:baittemp.ave     1   169  1.137005  0.2878
Site:Delta            1    23  3.889363  0.0607
```

Ok, let's look at the parameter estimates:    

```R
summary(stepAIC(fullmod5,direction="both"))
Fixed effects: FC ~ Jdaycont + gene + Site + baittemp.ave + Delta + gene:Site +      gene:baittemp.ave + Site:baittemp.ave + Site:Delta 
                              Value Std.Error   DF   t-value p-value
(Intercept)                7.079159 1.5735006 1022  4.498987  0.0000
Jdaycont                  -0.000706 0.0004481 1022 -1.575720  0.1154
genefcactin               -6.521205 1.9626275 1022 -3.322691  0.0009
genefcgapdh               -5.734743 1.9626275 1022 -2.921972  0.0036
genefchsp40               -9.208631 1.9626275 1022 -4.691991  0.0000
genefchsp70              -12.810381 1.9626275 1022 -6.527159  0.0000
genefchsp83              -16.817619 1.9626275 1022 -8.568931  0.0000
SiteHF                    -1.669959 2.4053577   23 -0.694266  0.4945
baittemp.ave              -0.289514 0.0600672 1022 -4.819841  0.0000
Delta                      0.033287 0.0527495   23  0.631046  0.5342
genefcactin:SiteHF        -2.324439 0.3884348 1022 -5.984117  0.0000
genefcgapdh:SiteHF        -2.495652 0.3884348 1022 -6.424894  0.0000
genefchsp40:SiteHF        -1.519840 0.3884348 1022 -3.912729  0.0001
genefchsp70:SiteHF        -1.215410 0.3884348 1022 -3.128994  0.0018
genefchsp83:SiteHF        -1.964134 0.3884348 1022 -5.056534  0.0000
genefcactin:baittemp.ave   0.275771 0.0743151 1022  3.710831  0.0002
genefcgapdh:baittemp.ave   0.248739 0.0743151 1022  3.347080  0.0008
genefchsp40:baittemp.ave   0.365116 0.0743151 1022  4.913071  0.0000
genefchsp70:baittemp.ave   0.495528 0.0743151 1022  6.667932  0.0000
genefchsp83:baittemp.ave   0.655617 0.0743151 1022  8.822116  0.0000
SiteHF:baittemp.ave        0.163719 0.0907195  169  1.804667  0.0729
SiteHF:Delta              -0.191388 0.0970456   23 -1.972147  0.0607

```

It looks like there is an overall site effect such that HF has lower expression in all 6 genes than DF. Each gene has a sig slope/relationship with bait temp, but they differ in magnitudie. This relationship does not differ sig between sites. 

### Some plots     

**Plotting FC vs bait temp**     

Loess fits for fold change vs bait temp for all genes colored by sites   

![](https://cloud.githubusercontent.com/assets/4654474/25804223/a4353002-33c8-11e7-9de5-ae3060d2cea6.jpeg)

Linear fits for fold change vs bait temp for all genes colored by sites     

![](https://cloud.githubusercontent.com/assets/4654474/25804251/c9830f50-33c8-11e7-8a4f-3e3b808b6ad6.jpeg)    
The site level diffeerences are not sig....     

Linear fits for fold change vs bait temp for all genes         
![](https://cloud.githubusercontent.com/assets/4654474/25804272/ee7101a0-33c8-11e7-9e08-b0a74e1dd1ea.jpeg)   

**Plotting FC vs gene for each site**    

```R
findat.long$gene<-as.factor(findat.long$gene)
gg<-summarySE(findat.long,measurevar ="FC",groupvars=c("Site","gene"))
ggplot(gg,aes(x=Site,y=FC,colour=gene,group=gene))+geom_errorbar(aes(ymin=FC-se,ymax=FC+se),width=.4,position=position_dodge(.5))+geom_point(size=3,position=position_dodge(.5))+geom_line(position=position_dodge(.5))
```

![](https://cloud.githubusercontent.com/assets/4654474/25804396/abdf9788-33c9-11e7-8dbf-c143c3601930.jpeg)

### Redoing stats with more precise efficiencies this time    

data parsing:    

```R
gxp1<-data.frame(log2(apply(deltaCTnum[,1:2],2,function(x){2^(x)/1.93^(deltaCTden)}))) # 18 eff = 2
gxp2<-data.frame(log2(apply(deltaCTnum[,2:4],2,function(x){1.97^(x)/1.93^(deltaCTden)}))) # hsp effic = 1.97
gxp3<-data.frame(log2(apply(deltaCTnum[,5:6],2,function(x){1.93^(x)/1.93^(deltaCTden)}))) # gapdh and actin eff = 1.93
gxp<-data.frame(CT_18s=gxp1[,1],gxp2,gxp3)
names(gxp)<-c("fc18s","fchsp40","fchsp70","fchsp83","fcactin","fcgapdh")

head(gxp)
        fc18s     fchsp40   fchsp70     fchsp83    fcactin     fcgapdh
1 -3.35598720  0.08869005 -0.196357 -5.10857584 -0.2974979  0.33337142
2 -4.22295350 -1.71440629 -1.573938 -3.08692726 -0.3648462  0.39693007
3  1.15712657  3.13068058  2.927802  0.79659331 -0.5778360  0.60227319
4 -4.22385927  0.17671276  1.997721  1.55194331  1.0106763 -0.98797586
5  0.04264102  3.54676875  2.529053  3.67969085 -0.6733599  0.69267514
6  4.67288683  0.14506857  1.144392  0.03931094  0.1147508 -0.06922802
```

same model construction   

```R
fullmod5<-lme(FC~RIN_Value+Jdaycont+gene*Site*baittemp.ave+gene*Site*Delta,random=~1|Cham2/Vial.me,data=findat.long,method="ML")

anova(summary(stepAIC(fullmod5,direction="both")))
```

model output 

```R
Step:  AIC=5133.09
FC ~ gene + Site + baittemp.ave + Delta + gene:Site + gene:baittemp.ave + 
    Site:baittemp.ave + Site:Delta

                         Df    AIC
<none>                      5133.1
+ Jdaycont                1 5134.4
+ RIN_Value               1 5134.8
- Site:baittemp.ave       1 5135.1
- Site:Delta              1 5135.4
+ gene:Site:baittemp.ave  5 5135.8
+ gene:Delta              5 5142.5
- gene:Site               5 5178.8
- gene:baittemp.ave       5 5213.8
                  numDF denDF   F-value p-value
(Intercept)           1  1023  0.077408  0.7809
gene                  5  1023  0.000000  1.0000
Site                  1    23 27.835461  <.0001
baittemp.ave          1  1023  4.074081  0.0438
Delta                 1    23  0.115459  0.7371
gene:Site             5  1023  9.620273  <.0001
gene:baittemp.ave     5  1023 18.634274  <.0001
Site:baittemp.ave     1   169  1.592226  0.2087
Site:Delta            1    23  4.344712  0.0484
```

Ok lets look at parameter estimates   

```R
Fixed effects: FC ~ gene + Site + baittemp.ave + Delta + gene:Site + gene:baittemp.ave +      Site:baittemp.ave + Site:Delta 
                              Value Std.Error   DF   t-value p-value
(Intercept)                6.958985 1.5282717 1023  4.553500  0.0000
genefcactin               -6.449641 1.9157589 1023 -3.366624  0.0008
genefcgapdh               -5.703602 1.9157589 1023 -2.977202  0.0030
genefchsp40               -9.119674 1.9157589 1023 -4.760346  0.0000
genefchsp70              -12.642891 1.9157589 1023 -6.599417  0.0000
genefchsp83              -16.562754 1.9157589 1023 -8.645531  0.0000
SiteHF                    -1.966949 2.3055972   23 -0.853119  0.4024
baittemp.ave              -0.294889 0.0580370 1023 -5.081056  0.0000
Delta                      0.031595 0.0512302   23  0.616729  0.5435
genefcactin:SiteHF        -2.292505 0.3791587 1023 -6.046293  0.0000
genefcgapdh:SiteHF        -2.454918 0.3791587 1023 -6.474644  0.0000
genefchsp40:SiteHF        -1.523837 0.3791587 1023 -4.018994  0.0001
genefchsp70:SiteHF        -1.226045 0.3791587 1023 -3.233592  0.0013
genefchsp83:SiteHF        -1.958443 0.3791587 1023 -5.165232  0.0000
genefcactin:baittemp.ave   0.272656 0.0725404 1023  3.758675  0.0002
genefcgapdh:baittemp.ave   0.247013 0.0725404 1023  3.405179  0.0007
genefchsp40:baittemp.ave   0.361846 0.0725404 1023  4.988197  0.0000
genefchsp70:baittemp.ave   0.489415 0.0725404 1023  6.746790  0.0000
genefchsp83:baittemp.ave   0.646013 0.0725404 1023  8.905556  0.0000
SiteHF:baittemp.ave        0.175617 0.0868661  169  2.021694  0.0448
SiteHF:Delta              -0.195618 0.0938489   23 -2.084397  0.0484
summary(stepAIC(fullmod5,direction="both"))
```

delta * site interaction significant this time.  

![](https://cloud.githubusercontent.com/assets/4654474/25813504/da113404-33e7-11e7-923b-c87ad7290ab9.jpeg)   

### Plots: FC vs bait, 2 panel sites   

loess 
![](https://cloud.githubusercontent.com/assets/4654474/25814124/4d10ca30-33ea-11e7-9e4b-686df997ac91.jpeg)

linear    

![](https://cloud.githubusercontent.com/assets/4654474/25814165/785d3322-33ea-11e7-8464-087fb0f5bab3.jpeg)
------

 <div id='id-section79'/> 

### Page 79:  

------

 <div id='id-section80'/> 

### Page 80:  

------

 <div id='id-section81'/> 

### Page 81:  

------

 <div id='id-section82'/> 

### Page 82:  

------

 <div id='id-section83'/> 

### Page 83:  

------

 <div id='id-section84'/> 

### Page 84:  

------

 <div id='id-section85'/> 

### Page 85:  

------

 <div id='id-section86'/> 

### Page 86:  

------

 <div id='id-section87'/> 

### Page 87:  

------

 <div id='id-section88'/> 

### Page 88:  

------

 <div id='id-section89'/> 

### Page 89:  

------

 <div id='id-section90'/> 

### Page 90:  

------

 <div id='id-section91'/> 

### Page 91:  

------

 <div id='id-section92'/> 

### Page 92:  

------

 <div id='id-section93'/> 

### Page 93:  

------

 <div id='id-section94'/> 

### Page 94:  

------

 <div id='id-section95'/> 

### Page 95:  

------

 <div id='id-section96'/> 

### Page 96:  

------

 <div id='id-section97'/> 

### Page 97:  

------

 <div id='id-section98'/> 

### Page 98:  

------

 <div id='id-section99'/> 

### Page 99:  

------

 <div id='id-section100'/> 

### Page 100:  

------

 <div id='id-section101'/> 

### Page 101:  

------

 <div id='id-section102'/> 

### Page 102:  

------

 <div id='id-section103'/> 

### Page 103:  

------

 <div id='id-section104'/> 

### Page 104:  

------

 <div id='id-section105'/> 

### Page 105:  

------

 <div id='id-section106'/> 

### Page 106:  

------

 <div id='id-section107'/> 

### Page 107:  

------

 <div id='id-section108'/> 

### Page 108:  

------

 <div id='id-section109'/> 

### Page 109:  

------

 <div id='id-section110'/> 

### Page 110:  

------

 <div id='id-section111'/> 

### Page 111:  

------

 <div id='id-section112'/> 

### Page 112:  

------

 <div id='id-section113'/> 

### Page 113:  

------

 <div id='id-section114'/> 

### Page 114:  

------

 <div id='id-section115'/> 

### Page 115:  

------

 <div id='id-section116'/> 

### Page 116:  

------

 <div id='id-section117'/> 

### Page 117:  

------

 <div id='id-section118'/> 

### Page 118:  

------

 <div id='id-section119'/> 

### Page 119:  

------

 <div id='id-section120'/> 

### Page 120:  

------

 <div id='id-section121'/> 

### Page 121:  

------

 <div id='id-section122'/> 

### Page 122:  

------

 <div id='id-section123'/> 

### Page 123:  

------

 <div id='id-section124'/> 

### Page 124:  

------

 <div id='id-section125'/> 

### Page 125:  

------

 <div id='id-section126'/> 

### Page 126:  

------

 <div id='id-section127'/> 

### Page 127:  

------

 <div id='id-section128'/> 

### Page 128:  

------

 <div id='id-section129'/> 

### Page 129:  

------

 <div id='id-section130'/> 

### Page 130:  

------

 <div id='id-section131'/> 

### Page 131:  

------

 <div id='id-section132'/> 

### Page 132:  

------

 <div id='id-section133'/> 

### Page 133:  

------

 <div id='id-section134'/> 

### Page 134:  

------

 <div id='id-section135'/> 

### Page 135:  

------

 <div id='id-section136'/> 

### Page 136:  

------

 <div id='id-section137'/> 

### Page 137:  

------

 <div id='id-section138'/> 

### Page 138:  

------

 <div id='id-section139'/> 

### Page 139:  

------

 <div id='id-section140'/> 

### Page 140:  

------

 <div id='id-section141'/> 

### Page 141:  

------

 <div id='id-section142'/> 

### Page 142:  

------

 <div id='id-section143'/> 

### Page 143:  

------

 <div id='id-section144'/> 

### Page 144:  

------

 <div id='id-section145'/> 

### Page 145:  

------

 <div id='id-section146'/> 

### Page 146:  

------

 <div id='id-section147'/> 

### Page 147:  

------

 <div id='id-section148'/> 

### Page 148:  

------

 <div id='id-section149'/> 

### Page 149:  

------

 <div id='id-section150'/> 

### Page 150:  

------

 <div id='id-section151'/> 

### Page 151:  

------

 <div id='id-section152'/> 

### Page 152:  

------

 <div id='id-section153'/> 

### Page 153:  

------

 <div id='id-section154'/> 

### Page 154:  

------

 <div id='id-section155'/> 

### Page 155:  

------

 <div id='id-section156'/> 

### Page 156:  

------

 <div id='id-section157'/> 

### Page 157:  

------

 <div id='id-section158'/> 

### Page 158:  

------

 <div id='id-section159'/> 

### Page 159:  

------

 <div id='id-section160'/> 

### Page 160:  

------

 <div id='id-section161'/> 

### Page 161:  

------

 <div id='id-section162'/> 

### Page 162:  

------

 <div id='id-section163'/> 

### Page 163:  

------

 <div id='id-section164'/> 

### Page 164:  

------

 <div id='id-section165'/> 

### Page 165:  

------

 <div id='id-section166'/> 

### Page 166:  

------

 <div id='id-section167'/> 

### Page 167:  

------

 <div id='id-section168'/> 

### Page 168:  

------

 <div id='id-section169'/> 

### Page 169:  

------

 <div id='id-section170'/> 

### Page 170:  

------

 <div id='id-section171'/> 

### Page 171:  

------

 <div id='id-section172'/> 

### Page 172:  

------

 <div id='id-section173'/> 

### Page 173:  

------

 <div id='id-section174'/> 

### Page 174:  

------

 <div id='id-section175'/> 

### Page 175:  

------

 <div id='id-section176'/> 

### Page 176:  

------

 <div id='id-section177'/> 

### Page 177:  

------

 <div id='id-section178'/> 

### Page 178:  

------

 <div id='id-section179'/> 

### Page 179:  

------

 <div id='id-section180'/> 

### Page 180:  

------

 <div id='id-section181'/> 

### Page 181:  

------

 <div id='id-section182'/> 

### Page 182:  

------

 <div id='id-section183'/> 

### Page 183:  

------

 <div id='id-section184'/> 

### Page 184:  

------

 <div id='id-section185'/> 

### Page 185:  

------

 <div id='id-section186'/> 

### Page 186:  

------

 <div id='id-section187'/> 

### Page 187:  

------

 <div id='id-section188'/> 

### Page 188:  

------

 <div id='id-section189'/> 

### Page 189:  

------

 <div id='id-section190'/> 

### Page 190:  

------

 <div id='id-section191'/> 

### Page 191:  

------

 <div id='id-section192'/> 

### Page 192:  

------

 <div id='id-section193'/> 

### Page 193:  

------

 <div id='id-section194'/> 

### Page 194:  

------

 <div id='id-section195'/> 

### Page 195:  

------

 <div id='id-section196'/> 

### Page 196:  

------

 <div id='id-section197'/> 

### Page 197:  

------

 <div id='id-section198'/> 

### Page 198:  

------

 <div id='id-section199'/> 

### Page 199:  

------

 <div id='id-section200'/> 

### Page 200:  



