# 2018 General Notebook

### Author: Andrew D. Nguyen, [Evolutionary Physiologist](https://adnguyen.github.io)      
### Affiliation: University of Florida, Department of Entomology and Nematology      
### Contact: anbe642@gmail.com     

### Date started: 2018-01-01    
### Date end (last modified): ongoing   

<a rel="license" href="http://creativecommons.org/licenses/by/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International License</a>.    

**Introduction:**    
Notebook for 2018 new year. It'll log the rest of my dissertation, post doc projects, meetings, papers I've read, and general project ideas.

## List of projects and description   
* Hsp rxn norm: Understanding how the local thermal environment shapes thermal tolerance and stress response (using Hsps as a proxy for stress) in forest ants of the genus *Aphaenogaster*. CTmax and rxn norm of Hsp expression measured across forest ants from Fl to Maine.   
* Range limits: Identifying the factors/forces that set range limits in common forest ants (*Aphaenogaster picea*). Modelling + measured their cold physiology in forest ants of Maine and Vt.  
* Thermal niche paper: Collaborative paper understanding how the environment shapes the ability to withstand cold and hot temperatures. In field and in a common garden, we measured upper and lower thermal limits of ants from GA-Maine (2 species).    
* Stress in nature: Are ants stressed under experimental warming that projects climate change? Ants were collected from warming chambers (0-5 C increase from ambient) and we measured their stress response.    
* ​Biological rhythms in *Rhagoletis*: Determining the relationship between behavioral rhythms in adult *Rhagoletis* and diapause exit timing + depth(eclosion and mass specific metabolic rate). 

# Table of contents (Layout follows Page number: Date. Title of entry)    
* [Page 1: 2018-01-01 ](#id-section1). Yearly goals: recap from last year and this year's
* [Page 2: 2018-01-01 ](#id-section2). Evolution meeting prep/thoughts 
* [Page 3: 2018-01-02 ](#id-section3). Range limits ms edits 
* [Page 4: 2018-01-02 ](#id-section4). to do list this month, 2018 january
* [Page 5: 2018-01-03 ](#id-section5). range limits ms , stats for single regressions of CCRT on Tmin
* [Page 6: 2018-01-05 ](#id-section6). UPDATE: to do list this month, 2018 january
* [Page 7: 2018-01-08 ](#id-section7). To do list
* [Page 8: 2018-01-08 ](#id-section8). Removing DS_store in github repo
* [Page 9: 2018-01-09 ](#id-section9). reading biological rhythms workshop I: Introduction by **Kuhlman et al. 2007**; Cold spring harbor
* [Page 10: 2018-01-09 ](#id-section10). Updated to do list
* [Page 11: 2018-01-10 ](#id-section11). Reading **Wadsworth et al. 2013**; Journal of Evo Biol. Developmental exit to diapause and how it relates to the evolution of insect seasonality 
* [Page 12: 2018-01-10 ](#id-section12). Meeting AMellison on range limits paper
* [Page 13: 2018-01-12 ](#id-section13). **Stålhandske et al. 2014**; post winter development of a butterfly
* [Page 14: 2018-01-12 ](#id-section14). working on hsp rxn norm paper 
* [Page 15: 2018-01-16 ](#id-section15). meeting with KBora on qpcr troubleshooting
* [Page 16:  ](#id-section16).
* [Page 17:  ](#id-section17).
* [Page 18:  ](#id-section18).
* [Page 19:  ](#id-section19).
* [Page 20:  ](#id-section20).
* [Page 21:  ](#id-section21).
* [Page 22:  ](#id-section22).
* [Page 23:  ](#id-section23).
* [Page 24:  ](#id-section24).
* [Page 25:  ](#id-section25).
* [Page 26:  ](#id-section26).
* [Page 27:  ](#id-section27).
* [Page 28:  ](#id-section28).
* [Page 29:  ](#id-section29).
* [Page 30:  ](#id-section30).
* [Page 31:  ](#id-section31).
* [Page 32:  ](#id-section32).
* [Page 33:  ](#id-section33).
* [Page 34:  ](#id-section34).
* [Page 35:  ](#id-section35).
* [Page 36:  ](#id-section36).
* [Page 37:  ](#id-section37).
* [Page 38:  ](#id-section38).
* [Page 39:  ](#id-section39).
* [Page 40:  ](#id-section40).
* [Page 41:  ](#id-section41).
* [Page 42:  ](#id-section42).
* [Page 43:  ](#id-section43).
* [Page 44:  ](#id-section44).
* [Page 45:  ](#id-section45).
* [Page 46:  ](#id-section46).
* [Page 47:  ](#id-section47).
* [Page 48:  ](#id-section48).
* [Page 49:  ](#id-section49).
* [Page 50:  ](#id-section50).
* [Page 51:  ](#id-section51).
* [Page 52:  ](#id-section52).
* [Page 53:  ](#id-section53).
* [Page 54:  ](#id-section54).
* [Page 55:  ](#id-section55).
* [Page 56:  ](#id-section56).
* [Page 57:  ](#id-section57).
* [Page 58:  ](#id-section58).
* [Page 59:  ](#id-section59).
* [Page 60:  ](#id-section60).
* [Page 61:  ](#id-section61).
* [Page 62:  ](#id-section62).
* [Page 63:  ](#id-section63).
* [Page 64:  ](#id-section64).
* [Page 65:  ](#id-section65).
* [Page 66:  ](#id-section66).
* [Page 67:  ](#id-section67).
* [Page 68:  ](#id-section68).
* [Page 69:  ](#id-section69).
* [Page 70:  ](#id-section70).
* [Page 71:  ](#id-section71).
* [Page 72:  ](#id-section72).
* [Page 73:  ](#id-section73).
* [Page 74:  ](#id-section74).
* [Page 75:  ](#id-section75).
* [Page 76:  ](#id-section76).
* [Page 77:  ](#id-section77).
* [Page 78:  ](#id-section78).
* [Page 79:  ](#id-section79).
* [Page 80:  ](#id-section80).
* [Page 81:  ](#id-section81).
* [Page 82:  ](#id-section82).
* [Page 83:  ](#id-section83).
* [Page 84:  ](#id-section84).
* [Page 85:  ](#id-section85).
* [Page 86:  ](#id-section86).
* [Page 87:  ](#id-section87).
* [Page 88:  ](#id-section88).
* [Page 89:  ](#id-section89).
* [Page 90:  ](#id-section90).
* [Page 91:  ](#id-section91).
* [Page 92:  ](#id-section92).
* [Page 93:  ](#id-section93).
* [Page 94:  ](#id-section94).
* [Page 95:  ](#id-section95).
* [Page 96:  ](#id-section96).
* [Page 97:  ](#id-section97).
* [Page 98:  ](#id-section98).
* [Page 99:  ](#id-section99).
* [Page 100:  ](#id-section100).
* [Page 101:  ](#id-section101).
* [Page 102:  ](#id-section102).
* [Page 103:  ](#id-section103).
* [Page 104:  ](#id-section104).
* [Page 105:  ](#id-section105).
* [Page 106:  ](#id-section106).
* [Page 107:  ](#id-section107).
* [Page 108:  ](#id-section108).
* [Page 109:  ](#id-section109).
* [Page 110:  ](#id-section110).
* [Page 111:  ](#id-section111).
* [Page 112:  ](#id-section112).
* [Page 113:  ](#id-section113).
* [Page 114:  ](#id-section114).
* [Page 115:  ](#id-section115).
* [Page 116:  ](#id-section116).
* [Page 117:  ](#id-section117).
* [Page 118:  ](#id-section118).
* [Page 119:  ](#id-section119).
* [Page 120:  ](#id-section120).
* [Page 121:  ](#id-section121).
* [Page 122:  ](#id-section122).
* [Page 123:  ](#id-section123).
* [Page 124:  ](#id-section124).
* [Page 125:  ](#id-section125).
* [Page 126:  ](#id-section126).
* [Page 127:  ](#id-section127).
* [Page 128:  ](#id-section128).
* [Page 129:  ](#id-section129).
* [Page 130:  ](#id-section130).
* [Page 131:  ](#id-section131).
* [Page 132:  ](#id-section132).
* [Page 133:  ](#id-section133).
* [Page 134:  ](#id-section134).
* [Page 135:  ](#id-section135).
* [Page 136:  ](#id-section136).
* [Page 137:  ](#id-section137).
* [Page 138:  ](#id-section138).
* [Page 139:  ](#id-section139).
* [Page 140:  ](#id-section140).
* [Page 141:  ](#id-section141).
* [Page 142:  ](#id-section142).
* [Page 143:  ](#id-section143).
* [Page 144:  ](#id-section144).
* [Page 145:  ](#id-section145).
* [Page 146:  ](#id-section146).
* [Page 147:  ](#id-section147).
* [Page 148:  ](#id-section148).
* [Page 149:  ](#id-section149).
* [Page 150:  ](#id-section150).
* [Page 151:  ](#id-section151).
* [Page 152:  ](#id-section152).
* [Page 153:  ](#id-section153).
* [Page 154:  ](#id-section154).
* [Page 155:  ](#id-section155).
* [Page 156:  ](#id-section156).
* [Page 157:  ](#id-section157).
* [Page 158:  ](#id-section158).
* [Page 159:  ](#id-section159).
* [Page 160:  ](#id-section160).
* [Page 161:  ](#id-section161).
* [Page 162:  ](#id-section162).
* [Page 163:  ](#id-section163).
* [Page 164:  ](#id-section164).
* [Page 165:  ](#id-section165).
* [Page 166:  ](#id-section166).
* [Page 167:  ](#id-section167).
* [Page 168:  ](#id-section168).
* [Page 169:  ](#id-section169).
* [Page 170:  ](#id-section170).
* [Page 171:  ](#id-section171).
* [Page 172:  ](#id-section172).
* [Page 173:  ](#id-section173).
* [Page 174:  ](#id-section174).
* [Page 175:  ](#id-section175).
* [Page 176:  ](#id-section176).
* [Page 177:  ](#id-section177).
* [Page 178:  ](#id-section178).
* [Page 179:  ](#id-section179).
* [Page 180:  ](#id-section180).
* [Page 181:  ](#id-section181).
* [Page 182:  ](#id-section182).
* [Page 183:  ](#id-section183).
* [Page 184:  ](#id-section184).
* [Page 185:  ](#id-section185).
* [Page 186:  ](#id-section186).
* [Page 187:  ](#id-section187).
* [Page 188:  ](#id-section188).
* [Page 189:  ](#id-section189).
* [Page 190:  ](#id-section190).
* [Page 191:  ](#id-section191).
* [Page 192:  ](#id-section192).
* [Page 193:  ](#id-section193).
* [Page 194:  ](#id-section194).
* [Page 195:  ](#id-section195).
* [Page 196:  ](#id-section196).
* [Page 197:  ](#id-section197).
* [Page 198:  ](#id-section198).
* [Page 199:  ](#id-section199).
* [Page 200:  ](#id-section200).



------

<div id='id-section1'/>    
    
### Page 1: 2017-01-01. Yearly goals: recap from last year and this year's    

**2017 recap of goals**  

1. Submit and Publish 3 manuscripts: range limits, Hsp rxn norm, and multiple stressors (also thermal niche paper).    
	* Published multiple stressors ms
	* Still ned to submit and publish 3-5 other ms:  
		* proteome stability proj
		* range limits
		* hsp rxn norm
		* stressed in nature (SHC is lead) 
		* thermal niche paper with Lacy  
		
2. Get a post doc. This'll probably involve learning a new study system. And also sending out tons of applications.      
	* at UF! Yay!
3. Learn and build a shiny app.    
	* did not do this at all  
4. Learn and become more proficient in statistics (Machine learning?, Baysian, predictive modelling, mixed effects modelling, eigentensor analyses).     
	* Quantitative genetics: more statistical genetics   
		* just dabbled in all of this, but didn't really learn much more 
		* Instead, I learned more about time series analyses, so that is something statistical   
	
5. Form new collaborations? It'd be awesome to work with Brent Sinclair, Brent Lockwood, Joel Kingsolver, Caroline Williams, Jon Stillman, Alex Gunderson.     
	* didn't form any new collabs with any of these people   
	* formed one with European colleague studying temperature adaptation in an alpine ant
	
6. Participate in a meta-analysis? Would be cool. 
	* Dan and I are coming up with a meta analysis related to biological timing or diapause  
	
7. Learn more physiology: Q10, metabolism related topics, lipid membranes, metabolites.         
	* Learned some respirometry!   




**2018 year goals**   

1. Submit and publish range limits, hsp rxn norm, and thermal niche papers.    

2. Build a data science course   

3. Solidify meta analysis ideas and start the project  

4. Learn and become more proficient at analyzing biological rhythm data; analyze Rhagoletis biological rhythm data to a point where I have a cool story to tell.

5. Start working on european cornborers: 
	* Learn rearing, diapause biology (induction, termination) 
	* experiment on behavioral rhythms  
	* do experiments to understand the molecular basis of behavioral rhythms, specifically focusing on period   
	
6. Start constructing a teaching and research statement  

7. Apply for and secure external funding  



------

<div id='id-section2'/>    
    
### Page 2: 2018-01-01. Evolution meeting prep/thoughts  

Presenting poster on Rhagoletis and talk on thermal adaptation in Aphaeno   

thoughts on the talk - I can draw from all of my projects to construct a nice story. I want to give my talk in the evolutionary physiology symposium (s69). The flow of the story will be from field metrics of stress responses due to experimental warming and then dissecting out the possible mechanisms of those stress responses, pitched in terms of physiological strategies organisms can adopt. I want to shift away from this gene does this towards a more complete understanding of what the genes tell us about the whole organism and what they're doing!  
 

Title:  
* Physiological strategies of temperature adaptation in ants  
* Responding to environmental change will require temperature adaptation involving tolerance  
* Adaptive responses to environmental change will require physiological shifts in... 
* Evolutionary innovations to environmental change in the common woodland ant 

1. Stressed in nature - yes! Ants show a stress response to experimental warming. 
	* this'll highlight the field relevance and impact !
2. Are these differences due to adaptations? If so, what kinds? And what does this response mean?  
3. Ants can withstand heat stress by resistance and/or tolerance
4. Common garden experiments: 
	* measuring the stress kinetics show that protection and tolerance , but not basal gxp
	* under acclimation, yes basal gxp can change 
5. punchline, these types of adaptations are unique and can only be detected with a rxn norm approach, because shifts in basal are found in the majority of other studies. 	

Potential opening: 

*I'm broadly interested in how organisms operate. And we can access how organisms operate by perturbing them and observing them under different contexts. For example, one big experiment that is happening right now is climate warming. So the main question in our research group is how will organisms respond to climate warming? ...Chamber stuff...source of selection....what is the actual source of selection? does it match up? Phylo context of ctmax*...mechanisms behind variation in CTmax ...phys strategies (resistance and tolerance)...

reminders from last year:  

**Organize evolution and biological rhtyhms meeting (tell Dan about time line)**   

* Evolution 2018: Montpillier France; 
	* August 19-22
	* Post doc travel grant: http://evolutionmontpellier2018.org/travel-grants   
		* Deadline is January 17th 

* Biological rhythms meeting 2018: https://srbr.org/meetings/upcoming-meeting/registration/
	* May 12-16 2018
	* registration: 
		* early(nov 15-march 1) reg non member - **$725**  ; **$425 member**
		* late ( march 2 to onsite) reg non member - **$825**; **$525 member**
		* early(nov 15-march 1) post doc non member - **$475**  ; **$425 post doc member**
		* late ( march 2 to onsite) post doc non member - **$575**; **$525 post doc member**    
		
		
2017-12-22. actual meeting   

Look into Cold spring harbor protocols 
	* set up trikinetics
	* data analysis 
	* data handling  
	
Basically, look at protocols online, actually check papers that have similar data structure like ours. Look up drosophila work. 

**Analyze existing, publicly available datasets** to see if we can recapitulate results.    

How do we separate signal from noise?      
 

------

<div id='id-section3'/>    
    
### Page 3: 2018-01-02. Range limits ms edits   

Addressing some ms edits here:  

from SHC in results section-*Determining the relationship between cold-tolerance and climate*: 

>You need to test how much of the variation in Tmin is explained by the combination of Tmax and seasonality.   

Not sure if this is right to do, but it seems better to just state how Tmin is correlated with Tmax and seasonality. 


correlation between tmin with Tmax and sd

```R
cor.test(dbio2$SD,dbio2$Tmin)

	Pearson's product-moment correlation

data:  dbio2$SD and dbio2$Tmin
t = -22.766, df = 100, p-value < 2.2e-16
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 -0.9422686 -0.8773033
sample estimates:
       cor 
-0.9155697 
```


```R
cor.test(dbio2$Tmax,dbio2$Tmin)

	Pearson's product-moment correlation

data:  dbio2$Tmax and dbio2$Tmin
t = 2.3546, df = 100, p-value = 0.0205
alternative hypothesis: true correlation is not equal to 0
95 percent confidence interval:
 0.03633252 0.40558545
sample estimates:
     cor 
0.229188 
```

take home: Tmin has high correlation with SD and not Tmax   


but i'll do the regression here to show it: 

```R
summary(lm(Tmin~Tmax+SD,data=dbio2))

Call:
lm(formula = Tmin ~ Tmax + SD, data = dbio2)

Residuals:
    Min      1Q  Median      3Q     Max 
-7.7794 -2.3577 -0.8954  1.5918 16.5544 

Coefficients:
              Estimate Std. Error t value Pr(>|t|)    
(Intercept) 88.3238275 11.3484853   7.783 6.98e-12 ***
Tmax         0.6692606  0.0359986  18.591  < 2e-16 ***
SD          -0.0418123  0.0008353 -50.058  < 2e-16 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 3.935 on 99 degrees of freedom
Multiple R-squared:  0.964,	Adjusted R-squared:  0.9633 
F-statistic:  1325 on 2 and 99 DF,  p-value: < 2.2e-16
```

Dan wants G matrix--- put in supplement:  

|                | pretreat_Temp-5| pretreat_Temp0| pretreat_Temp25| pretreat_Temp5|
|:---------------|---------------:|--------------:|---------------:|--------------:|
|pretreat_Temp-5 |         3328.40|       -2175.66|        -5635.48|         287.85|
|pretreat_Temp0  |        -2175.66|       15856.71|       -16232.99|       -8876.65|
|pretreat_Temp25 |        -5635.48|      -16232.99|        38801.25|       17335.45|
|pretreat_Temp5  |          287.85|       -8876.65|        17335.45|       24394.16|


**2018-01-03 entry**

```
% * <shelmscahan@gmail.com> 2017-12-26T17:27:46.052Z:
% The fact that there is a threshold does not imply that environmental change is steep.  Even if a change is very very gradual, if an organism has a lower limit, at some point they will hit it.  You don;t even have an indirect density indicator of this (ie they stay really numerous right up to the boundary until they suddenly drop out).  To say the gradient is steep would mean actually mapping out what happens to environmental predictors across the transect from present to absent.  You have not done this.  I have removed steep from this paragraph (obviously if you are super opposed you could add it back over my objection).
```  


Makes no sense, this is the whole point of modelling exercise. 

------

<div id='id-section4'/>    
    
### Page 4:  2018-01-02. to do list this month, 2018 january

1. range limits ms - send out by end of this week (jan 5)
2. hsp rxn norm ms (send out Jan 19?) 
3. evolution conference submission of abstract, and also for travel grant jan 15
4. construct evolution talk, quick and dirty just to  have the slides together 
5. read more on time series analyses 
6. biological rhythms conference - register as member and for conference just to get out of the way 

------

<div id='id-section5'/>    
    
### Page 5:  2018-01-03. range limits ms , stats for single regressions of CCRT on Tmin     


**basal data**

```R
ksub<-subset(k.dat,k.dat$pretreat_Temp=="25")
ksub$coldplot<-max(ksub$treatment_recovery_s)-ksub$treatment_recovery_s
 str(ksub)
'data.frame':	21 obs. of  47 variables:
 $ Colony              : Factor w/ 21 levels "Avon","Avon_2",..: 21 5 8 4 14 13 1 2 9 3 ...
 $ Date                : Factor w/ 11 levels "","07-Jul-15",..: 3 4 4 5 6 6 7 7 8 9 ...
 $ Plot                : Factor w/ 17 levels "","19:01","19:03",..: 11 1 15 8 4 5 2 3 16 14 ...
 $ State               : Factor w/ 2 levels "Maine","Vermont": 1 1 1 1 1 1 1 1 1 1 ...
 $ County              : Factor w/ 5 levels "","Franklin ",..: 5 4 3 5 4 2 2 2 3 4 ...
 $ Town                : Factor w/ 17 levels "Avon","Bingham",..: 17 5 7 4 13 12 1 1 8 2 ...
 $ Lat                 : num  44.6 45 45.1 44.7 44.7 ...
 $ Lon                 : num  -69.3 -69.5 -68.6 -69.4 -69.8 ...
 $ Altitude_ft         : int  510 543 315 468 203 411 857 852 250 470 ...
 $ Temp                : num  20.6 22.5 27.7 23.9 23 23.6 23.9 24.6 18.8 20 ...
 $ Wind.Speed          : num  0 0 0.7 0 0 0 0 0 0 0 ...
 $ Humidity            : num  83.5 72.6 56.3 71.6 73.2 67.6 54.6 68.7 81.2 78 ...
 $ Wind.Chill          : num  20.6 23.8 27.6 23.8 22.4 22.9 24 24.6 19.2 20.6 ...
 $ Heat.Stress.Index   : num  20.9 22.2 31.9 25.7 22.8 22.1 24.9 28.6 18.7 20.5 ...
 $ Barometric.Pressure : num  29.4 29.3 29.6 29.4 29.7 ...
 $ soil_temp           : num  14.5 15 16 15 17 15 15 14.5 15 15 ...
 $ canopy_photo        : Factor w/ 17 levels "","19:1 Avon",..: 10 16 14 7 17 4 2 3 15 13 ...
 $ Tree.Species        : Factor w/ 17 levels "","birch, conifers",..: 3 17 5 12 7 8 14 13 2 6 ...
 $ Understory          : Factor w/ 11 levels ""," new growth conifers",..: 1 5 7 2 2 1 1 6 8 1 ...
 $ Habitat.Photo.Number: Factor w/ 18 levels "","19:1 Avon",..: 10 17 14 7 18 4 2 3 15 13 ...
 $ Nest_substrate      : Factor w/ 10 levels "","Branch and leaf litter",..: 6 3 9 9 9 9 8 2 9 5 ...
 $ CTmax               : num  40.9 36.8 40.7 41.2 39.5 ...
 $ MAT                 : num  6.7 5.4 5.7 6.6 6.3 5.9 4.7 4.7 5.7 5.3 ...
 $ MDR                 : num  12.2 12.5 12.2 12.4 12.6 12.5 12.2 12.2 12 12.6 ...
 $ ISO                 : num  0.29 0.29 0.29 0.3 0.3 0.3 0.29 0.29 0.29 0.29 ...
 $ SD                  : num  97.7 101.5 98.4 98.8 99.2 ...
 $ Tmax                : num  26.8 26 26.1 26.9 26.8 26.2 24.9 24.9 25.9 25.9 ...
 $ Tmin                : num  -13.9 -16.3 -15.1 -14.4 -15 -15 -16.4 -16.4 -14.8 -16.4 ...
 $ TAR                 : num  40.7 42.3 41.2 41.3 41.8 41.2 41.3 41.3 40.7 42.3 ...
 $ TWQ                 : num  2.1 0.6 1.2 1.9 1.6 1.3 0.1 0.1 1.3 0.7 ...
 $ TDQ                 : num  -5.1 -6.9 -6.2 -5.4 -5.8 -6 -7.4 -7.4 -6.1 -7 ...
 $ TwarmQ              : num  18.9 18 18 18.9 18.6 18.1 17 17 18 17.8 ...
 $ TminQ               : num  -6.5 -8.4 -7.6 -6.8 -7.1 -7.3 -8.7 -8.7 -7.4 -8.4 ...
 $ AP                  : int  1094 1065 1046 1069 1057 1106 1102 1102 1079 1052 ...
 $ PWM                 : int  116 109 113 112 108 113 113 113 118 106 ...
 $ PDM                 : int  73 67 75 69 68 73 69 69 79 64 ...
 $ PSD                 : int  12 12 12 12 11 10 12 12 12 13 ...
 $ PWQ                 : int  316 298 303 306 297 310 305 305 316 291 ...
 $ PDQ                 : int  238 217 230 226 224 244 231 231 242 211 ...
 $ PwarmQ              : int  262 280 258 265 271 279 296 296 257 286 ...
 $ PminQ               : int  254 237 257 242 237 254 243 243 270 229 ...
 $ pretreat_Temp       : int  25 25 25 25 25 25 25 25 25 25 ...
 $ pre                 : num  0 0 0 0 0 0 0 0 0 0 ...
 $ treatment_recovery_s: num  577 624 745 816 724 ...
 $ diff                : num  -577 -624 -745 -816 -724 ...
 $ coldplot            : num  541 494 373 302 394 ...
 $ hard.zero           : num  -153 -335 -386 -309 -264 ...
```

**basal regression**

```R
summary(lm(coldplot~Tmin,data=ksub))

Call:
lm(formula = coldplot ~ Tmin, data = ksub)

Residuals:
    Min      1Q  Median      3Q     Max 
-333.63  -82.44   -5.63   90.37  266.36 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)   
(Intercept)  -889.43     413.86  -2.149  0.04473 * 
Tmin          -89.93      27.62  -3.256  0.00416 **
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 141.1 on 19 degrees of freedom
Multiple R-squared:  0.3581,	Adjusted R-squared:  0.3244 
F-statistic:  10.6 on 1 and 19 DF,  p-value: 0.004157
```


**hardening data** 

```R
zero<-subset(k.dat,k.dat$pretreat_Temp=="0")
ksub$hard.zero<-ksub$treatment_recovery_s-zero$treatment_recovery_s
nbn3<-ddply(ksub,.(Colony,pretreat_Temp),summarize,CCRT=mean(hard.zero),Tmin=mean(Tmin))
str(nbn3)
'data.frame':	21 obs. of  4 variables:
 $ Colony       : Factor w/ 21 levels "Avon","Avon_2",..: 1 2 3 4 5 6 7 8 9 10 ...
 $ pretreat_Temp: int  25 25 25 25 25 25 25 25 25 25 ...
 $ CCRT         : num  41 -72.3 -346.4 -309.2 -334.8 ...
 $ Tmin         : num  -16.4 -16.4 -16.4 -14.4 -16.3 -15.2 -13.6 -15.1 -14.8 -13.5 ...
```

**hardening regression** 

```R
summary(lm(CCRT~Tmin,data=nbn3))

Call:
lm(formula = CCRT ~ Tmin, data = nbn3)

Residuals:
    Min      1Q  Median      3Q     Max 
-402.13  -96.13   26.98  137.37  236.14 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)  
(Intercept) -1346.45     516.54  -2.607   0.0173 *
Tmin          73.85      34.47  -2.142   0.0453 *
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 176.1 on 19 degrees of freedom
Multiple R-squared:  0.1945,	Adjusted R-squared:  0.1522 
F-statistic: 4.589 on 1 and 19 DF,  p-value: 0.04534

```

### SHC comments to address on % hardening improvement from 25 C pre-treatment temp: 

```
% What % improvement do your hardening results represent? 
This would tell you whether 4.8 minutes is really too short to be able to detect hardening given the inherent measurement error in such experiments.
If it is 10%, that would be almost a minute improvement, which seems quite feasible.  
```

how to calculate this... 

```R
###stitching together 0 and 25 C pre-treatment
nn<-rbind(zero,ksub)
#calculating the difference in the max CCRT (for the data which includes 0) from all other measurements of CCRT

## this is to put everything on the same scale and have higher values as more cold tolerant
nn$delta<-max(nn$treatment_recovery_s)-nn$treatment_recovery_s
```

lets look at the data:  

```R
ggplot(nn,aes(x=Tmin,y=delta,colour=factor(pretreat_Temp)))+geom_point()+stat_smooth(method="lm")
```

![](https://user-images.githubusercontent.com/4654474/34528687-9d2477f2-f077-11e7-80b0-c342e49352f0.png)


```R
##convert long to wide format to get  0 and 25 as diff columns
nn2<-melt(nn,id.vars=c("pretreat_Temp","Colony"),measure.vars=c("delta"))
wide.nn<-spread(nn2,pretreat_Temp,value)[-19,-2]


##dataset
knitr::kable(wide.nn)
```

|   |Colony       |        0|       25|
|:--|:------------|--------:|--------:|
|1  |Avon         | 688.4167| 729.4167|
|2  |Avon_2       | 666.7500| 594.4167|
|3  |Bingham      | 803.4167| 457.0000|
|4  |Burnham      | 610.7500| 301.5000|
|5  |Cambridge    | 828.7500| 494.0000|
|6  |Canaan       | 711.7500| 605.7500|
|7  |cent2        | 503.5000| 298.7500|
|8  |Greenbush    | 758.7500| 373.0000|
|9  |Greenfield   | 778.0833| 532.7500|
|10 |Hampden      | 612.0000| 271.7500|
|11 |Monroe       | 702.7500| 582.0000|
|12 |New Portland | 613.0000| 655.5000|
|13 |New Sharon   | 609.7500| 398.0000|
|14 |Norridgewock | 657.2500| 393.5000|
|15 |Palmyra      | 822.7500| 318.2500|
|16 |Phillips     | 646.4167| 679.5000|
|17 |Saponac      | 723.7500| 564.0833|
|18 |ted1         | 766.2500| 328.0000|
|20 |ted8         | 701.5000| 424.0000|
|21 |Unity        | 694.2500| 541.0000|


### calculating % improvement 

so this is the calculation of the amount of seconds changed, scaled by the total time/value at 25 C to calculate improvement of hardening ability. 

% percent changed 
```R
round(abs(wide.nn$`0`-wide.nn$`25`)/wide.nn$`25`,3)*100
 [1]   5.6  12.2  75.8 102.6  67.8  17.5  68.5 103.4  46.1 125.2  20.7   6.5  53.2  67.0
[15] 158.5   4.9  28.3 133.6  65.4  28.3
```

range of % changed

```R
range(round(abs(wide.nn$`0`-wide.nn$`25`)/wide.nn$`25`,3)*100)

 4.9 158.5

```

### lets look at this in terms of minutes instead of seconds 

```R
##lets look at the amount of time that has changed itself

wide.nn$minutes_change<-abs(wide.nn$`0`-wide.nn$`25`)/60
#range of minutes 
```


|   |Colony       |        0|       25| minutes_change|
|:--|:------------|--------:|--------:|--------------:|
|1  |Avon         | 688.4167| 729.4167|      0.6833333|
|2  |Avon_2       | 666.7500| 594.4167|      1.2055556|
|3  |Bingham      | 803.4167| 457.0000|      5.7736111|
|4  |Burnham      | 610.7500| 301.5000|      5.1541667|
|5  |Cambridge    | 828.7500| 494.0000|      5.5791667|
|6  |Canaan       | 711.7500| 605.7500|      1.7666667|
|7  |cent2        | 503.5000| 298.7500|      3.4125000|
|8  |Greenbush    | 758.7500| 373.0000|      6.4291667|
|9  |Greenfield   | 778.0833| 532.7500|      4.0888889|
|10 |Hampden      | 612.0000| 271.7500|      5.6708333|
|11 |Monroe       | 702.7500| 582.0000|      2.0125000|
|12 |New Portland | 613.0000| 655.5000|      0.7083333|
|13 |New Sharon   | 609.7500| 398.0000|      3.5291667|
|14 |Norridgewock | 657.2500| 393.5000|      4.3958333|
|15 |Palmyra      | 822.7500| 318.2500|      8.4083333|
|16 |Phillips     | 646.4167| 679.5000|      0.5513889|
|17 |Saponac      | 723.7500| 564.0833|      2.6611111|
|18 |ted1         | 766.2500| 328.0000|      7.3041667|
|20 |ted8         | 701.5000| 424.0000|      4.6250000|
|21 |Unity        | 694.2500| 541.0000|      2.5541667|

**ok, the range in minutes?**

```R
range(wide.nn$minutes_change)
[1] 0.5513889 8.4083333
```

### taking this paragraph out of discussion   

```
The patterns of cold hardening observed in this study suggest that \textit{A. picea} are exposed to conditions in the field that are at or near their physiological limits.  \textit{Aphaenogaster picea} colonies were the most cold tolerant at 0  \degree C pre-treatment temperatures, with a substantial decline in hardening ability when exposed to -5 \degree C, yet, soil temperatures in Maine at ant-wintering depths typically range from 0  to -5  \degree C, with even lower temperatures possible on a short-term basis (Schaefer et al. 2009).  Maintaining performance below 0  \degree C pre-treated temperatures may impose significant physiological challenges due to tissue damage from freezing selecting for both constitutive and inducible mechanisms to counter stressful mean temperatures and pulses of more extreme conditions (Bale 2002). 
```

2018-01-04: not sure if I want to expand on this idea, but leaving it here   

```
negative correlation in aphaenogaster; 0 was the optimal performance for many colonies, anything past that might be a constraint because tissues begin to become damaged from freezing.
```
------

<div id='id-section6'/>    
    
### Page 6:  2018-01-05. UPDATE: to do list this month, 2018 january

1. range limits ms - send out by end of this week (jan 5)
	* done, tweaked results little, major tweaks discussion, rewrote abstract  
2. hsp rxn norm ms (send out Jan 19?) 
	* work on next, what needs to be done? 
	* From [2018_nb_page108](https://github.com/adnguyen/Notebooks_and_Protocols/blob/master/2017_notebook.md#page-108-2017-07-31-dissertation-work-progress-list):
		* Rearranged figures; so results need to be rewritten; introduction needs tweaking; discussion needs to be finished
			* Figure layout:
				* Predictions figure: Fold induction vs temperature for each parameter
				* Phylogenetic analysis: ancestral trait reconstruction
				* Non-phylogenetic controlled analysis: CTmax vs environment
				* CTmax vs hsp params
				* hsp params vs habitat type
				* Methods figure: example of fitting boltzmann to hsp gxp
		* SHC has comments on results 
		
3. evolution conference submission of abstract, and also for travel grant jan 15
	* submitted both!  
4. construct evolution talk, quick and dirty just to  have the slides together 
5. read more on time series analyses 
6. biological rhythms conference - register as member and for conference just to get out of the way 

not done anything for 4-6. 

------

<div id='id-section7'/>    
    
### Page 7: 2018-01-08. To do list   

1. Set up weekly meeting with Dan    
	* Fridays 3pm, starting next week
	* Dan says I need to understand the Rhagoletis system better   
		* Dan will send out paper  
2. Set up schedule with undergrad researchers  
	* Cwu - try to set her up with Qinwen working on the ECB system ; can she get paid too ? 
	* KLennon: Monday, Tuesday, Sund: 1-3pm; Thursday 10:30Am-12:30PM
	* Tat: M,W,F,Sat: 7-9PM; Thursday 9AM-11AM  
	* Taariq : working on script to assemble trik data with master datasheet 
3. Start working on Hsp rxn norm paper   



------

<div id='id-section8'/>    
    
### Page 8: 2018-01-08.  Removing DS_store in github repo

plaguing my existence... good solutions [here](https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository)

1. In terminal where project is located: 

```
find . -name .DS_Store -print0 | xargs -0 git rm -f --ignore-unmatch
```

2. then add ".DS_Store" in gitignore, and save  



------

<div id='id-section9'/>    
    
### Page 9:  2018-01-09. reading biological rhythms workshop I: Introduction by Kuhlman et al. 2007; Cold spring harbor    


ref:   
Kuhlman, S. J., Mackey, S. R., & Duffy, J. F. (2007). Biological Rhythms Workshop I: introduction to chronobiology. Cold Spring Harbor Symposia on Quantitative Biology, 72, 1–6. https://doi.org/10.1101/sqb.2007.72.059


This paper gives a timeline of chronobiology 

### a little bit of history or historical context

- An astronomer, Jean-Jacque D'Ortous deMairan (1729) made one of the first observations that organisms exhibit cyclical behavior: leaf movements in heliotrope plants continue in constant darkness. (Free run experimetn ) 

- Franz Halberg 1959 coined the term *circadian*, circa = about, dies= day, to refer to daily rhythms that persist under free run. 

- The most well known circadian rhythms occur in single cells, whereby transcription and translation form feedback loops.  

- Biological rhythms are suppose to be or presumed adaptive.    

- It was Bunning 1935 who first discovered that biological rhythms are heritable and have a genetic basis; daily rhythms could be extended to seasonal rhythms.  

- Ronald Konopka and Seymour Benzer 1971 mutgenized fruit flies to understand the rhythnsm of pupal eclosion and locomotor activity (have to read this paper!).These flies could be non-rhythnmic, or rhythmic over short or longer time scales. All phenotypes were found to be within the Period gene (region/locus). 

- Ok, wow, we can manipulate the timing of organisms, but what are some of the functional consequences? 
	* Karl von Frisch and Beling showed that bees use their clocks to visit flowers at the time of the day when the flower opens 
	* Kramer showed that birds' migration patterns are rhythmic
	* DeCoursey et al. 2000 show that chipmucks modulate rhythms to avoid predation  
	* Steve Reppert showed that monarch butterflies likely use biological rhythms to initiate migration and track seasonal changes  	

### Some terminology   

- "Black-box" experiments are used to probe mechanisms of clock function (Moore-Ede et al. 1982)
	* no clue what this means, wtf
	
- Overt outputs - observable, measurable rhythmic outputs ( leaf movement) 
- a rhythm is considered to be circadian if the oscillation has a 24 hour period(approx) 
	* but why... the main argument is the relative role of env vs genetics on biological rhythms  

**Free running period**	

* process of organism's rhythm syncrhonizes to external cycling cue is known as entrainment  
* Free run is the cyclical behavior in the absence of the cue (so darkness if light entrained)
* Diurnal orgs have shorter free run (faster clock) under high light intensity than low light intensity 
	* citation? 
* Free run in diurnal orgs found by Jurgen Aschoff, known as, "Aschoff's Rule" (Aschoff and Wever 1962)  
* The degree of free run change is influneced by prior experience to entrainment cue (light), known as *aftereffect*. fuck these words, wtf? Keep the terms the same among disciplines ---acclimation ! 

**Two properties of circ rhythms that are often focused in experimental studies**    

1. ability of rhythm to be reset, or phase-shifted, by transient exposure to time cues such as light
2. temperature-compensation

bad writing....very little clue about what they're talking about. Why would this matter? Presenting a question and hypothesis helps.   


### Phase shifting and phase-response curve    

This is a manipulation aimed at resetting the clock such that the rhythms differ between manipulated vs unmanipulated. If you do this with an external cue (light), then it can show that the cue and changes to that cue is important for setting biological rhythms. 

* If you set this manipulation at different times of the day, then it'll show that sensitivity to resetting differs at differnt parts fo the cycle (Hastings and Sweeney 1958)


Possible that the clock doesnt reset at all.   

### Temperature-compensation   

Expectation: clock should not change despite changes in ambient temperature 

What? this is garbage. One could expect this under the "hotter is better" hypothesis whereby biochemical reactions simply speed up as a function of temperature. So the clock should change by getting shorter andf aster.   

uncited claim : freer un period changes little over a temp range  


bad....

They need to parse out these ideas better. In the face of temperature changes, the organism can maintain their biological clock. For example, they can initiate behaviors at the same time of the day. This would be robustness of biological rhythms. But, temperature itself can act as an entrainment cue too. 

### Concept of Clock    

Would have been nice to explain jargon before talking about it. Organizational problem here.  
 
Chandrashekaran 1998 - 

Clock - endogenous nature of rhythms ( innate, rather than learned), a way of measuring time for the organism. 

Bad definition. This should be described like any other trait, which has a genetic and environmental basis. 


### Thoughts   

Aside from citations pointing to some cool studies, historical context, and defining some words (entrainment, free run), this paper doesn't have a good conceptual description of biological clocks. Why would phase shift matter, why would someone want to know that, how is it applicable to some greater concept of how organisms work? smh. 









------

<div id='id-section10'/>    
    
### Page 10: 2018-01-09. Updated to do list


1. Read papers for discussion with Dan 
	* still waiting for dan as to what they are
2. Set up schedule with undergrad researchers  
	* Cwu - try to set her up with Qinwen working on the ECB system ; can she get paid too ? ---sent out coordinating email  
	* KLennon: Monday, Tuesday, Sund: 1-3pm; Thursday 10:30Am-12:30PM -- got keys
	* Tat: M,W,F,Sat: 7-9PM; Thursday 9AM-11AM   -- got keys
	* Taariq : working on script to assemble trik data with master datasheet (meeeting wednesday 10AM)
3. Start working on Hsp rxn norm paper   
	* set up github repo, so data are accessible and can work on it from anywhere 
	* still need to start....
4. thermal niche ms
	* Lchick to send me a version by end of week to edit 
5. range limits ms
	* sent out, 
	* submit soon? February...please!?
6. euro collab ms
	* wait for results of tests of selection, then help write results and discussion 


------

<div id='id-section11'/>    
    
### Page 11:  2018-01-10. Reading Wadsworth et al. 2013; Journal of Evo Biol. Developmental exit to diapause and how it relates to the evolution of insect seasonality   

ref: 
Wadsworth, C. B., Woods, W. A., Hahn, D. A., & Dopman, E. B. (2013). One phase of the dormancy developmental pathway is critical for the evolution of insect seasonality. Journal of Evolutionary Biology, 26(11), 2359–2368. https://doi.org/10.1111/jeb.12227   

A little background: These authors are working on the european cornborer (ECB), where two strains differ in their seasonal emergence.  

1. E strain = earlier emerging strain 
	* this strain is bivoltine, so it has two generations per year
	* ID'd by $PDD^S$ allele
2. Z strain = later emerging strain 
	* this strain is univoltine, so 1 generation per year 
	* ID'd by $PDD^L$ allele

### Main Question: What part of diapause is critical for seasonal timing? In this case, emergence. 

Hypothesis: The timing of diapause termination causes the differences in emergence timing between E and Z strains.  

### Experimental approach: 

* Induce diapause  in 12:12 L:D at constant 23 C. (Glover 1992) for both strains, E and Z
* ECB diapauses as larvae; measured metabolic rates and modeled its trajectory with a function 

I messed around with the function [here](http://adnguyen.github.io/demos/2018-01-10_phases_of_diapause_exit.html). It is basically an exponential model with 3 parameters: 

* a = presistent metabolic increase that defines the start of termination   
* b = rate of metabolic increase during diapause termination   
* c = initial metabolic rate during diapuse maintenance phase   

### Major results: diapause termination corresponds with metrics of emergence timing int he field (# breeding adults and pupae)   

E strain, which has earlier emergence timing, has shorter time of diapause break (diapause termination) than Z strain. It looks like the difference amounts to 22 days (Z-E, 34-12).   


### Discussion points

* similar thing happens in swallowtail butterflies (*Papilio*) (Scriber & Ording 2005) 
	* northern ones are univoltine with obligate pupal diapause and produce earlier summer flights
	* southern ones are bivoltine and more facultative, producing diapause and non-diapausing phenotypes



------

<div id='id-section12'/>    
    
### Page 12: 2018-01-10. Meeting AMellison on range limits paper


* In the CART model, scale the bioclim variables first.

For figure 3.

* For vertical variation, say constant for the loadings (panel b)
* panel a, additive response rather than additive variation ; because the sd doesn't change, the eigenvector remains constant 
* for panel c,d ; more than just horizontal shift, both shift in level and in variance. You want to show a step function in the loadings
* simulate distributions and then compute the variance-covariance matrix and then plot the loadings 
* Set up a new set of panels and set up the null expectations  

2018-01-11 update: [done here](http://adnguyen.github.io/demos/2018-01-10_simulate_perf_curves.html); look identical to my figures

For standardizing variables, seems like the consensus is no, unless you're comparing with other predictive models: [1](https://stats.stackexchange.com/questions/89162/standardizing-inputs-for-cart),[2](https://stats.stackexchange.com/questions/51523/log-transformation-in-cart-analysis/51534#51534)

### Take home: 

think about how to write the patterns of perf curves and how it maps to the loadings. Look back at Kingsolver et al. 2015 and see how he writes it. 


* Citizen science idea--I should write this out , later. But it deals with species distribution modeling. One problem you'll encounter is that there is error from citizens! You have to account for error of detection. 
	* talk to matt fitzpatrick about proj	
	* AMellison said it would be a good idea to apply for grant   
* AMEllison is trying to push out genome paper with MLau   

2018-01-12 update

other projects I'm on that I've forgotten about    

* genome paper  
* phylo paper with Bernice


------

<div id='id-section13'/>    
    
### Page 13: 2018-01-12. Stålhandske et al. 2014; post winter development of a butterfly

Reading to discuss with Dan.   

reference: 

Stålhandske, S., Gotthard, K., Posledovich, D., & Leimar, O. (2014). Variation in two phases of post-winter development of a butterfly. Journal of Evolutionary Biology, 27(12), 2644–2653. https://doi.org/10.1111/jeb.12519    

Wow, awesome sentence: 

> Fine- tuning of the life cycle may be particularly important to match the phenology of potential mates and resources as well as for optimizing abiotic conditions at eclosion.

------  

Counter gradient variation is a term that has always confused me; wiki defines as: 

> Countergradient variation is a type of phenotypic plasticity that occurs when the genetic components of populations cause phenotypic variation that opposes the phenotypic variation caused by an environmental gradient.

Almost not understandable and too much to unpack. It sounds more like a trait that varies along an environmental gradient that is counter to expectations. Anyway, to me, this is not a useful term at all because it muddles how we should think about the environment and how organisms can respond to it. 

One prime example is growth across a lat gradient. Populations in the cooler condition (northern) that grow faster than warmer populations (southern) is considered countergradient variation because warmer temperatures are suppose to increase growth rates. But maybe, the env variation that growth rates are responding to is not solely warm temperatures.

Authors' def: 

> Countergradient variation occurs when the inherent relationship between phenotype and environment given by a reaction norm is reduced or removed through genetic adaption across an environmental gra- dient (Conover & Schultz, 1995)  

Totally diff definition than wikipedia. The concept is so abstract so that it is hard to interpret biologically. Angiletta's book doesn't have a good definiton. 

This paper talks about other env variables as to explain the patterns described in countergradient variation....but if there are other env variables to consider other than temp, then that is cogradient variation ( organisms responding with the environmental ) ....see makes no sense. 

> Populations that experience a shorter growing season or lower temperatures may compensate for this by a faster growth rate at any given temperature (Yamahira & Conover, 2002).  





------

Citations for lat gradients: 

> Intraspecific clines in life cycle traits often follow environmental gradients, such as temperature gradients along latitude (Bradford & Roff, 1995; Masaki, 1999; Gaston et al., 2008).

------

Paper is not really structured in a question-hypothesis framework. But they are interested in how the environment (latitude) shapes the variation in seasonal timing of these butterflies (*Anthocharis cardamines*). 

### Natural history of *Anthocharis cardamines*:    

* one life cycle a year (univoltine); diapause
* lays eggs on buds and flowers of early flowering host plants (phenological specialist)  
	* plants within the Brassicaceae
* Females emerge first on flowers, then males follow
* spring emergence is a function of both winter duration of diapause and the rate of post diapause development (they just want to study post witner development)  
* found in europe, asia, and north africa  

### Experimental approach: common garden of pops collected in 6 populations spanning lat and measuring growth rates   

* reared on garlic mustard
* collected eggs and placed in lab 
* larvae hatched and reared to pupation in common garden 
* simulated overwintering - 2 C 24 dark, 5 months  

pupae then moved to 4 constant temperatures, post wintering after 5 months. 

* pupae weighed every 48 hours and rate of weight loss was used as a proxy for rate of development 

* stats mostly on females ; stats not explained very well.... what are they testing. 

### Results: 

Table 1: Linear models, evaluating predictors  that explain growth patterns. They only tested for local * temp interaction in the proportion variable. What about others? Ther perform model selection ? Also, what about local * initial pupal weight interaction? 

How I'd show figure 2- 

![](https://user-images.githubusercontent.com/4654474/34883079-839c6144-f786-11e7-9870-1590bdcb846b.jpg)   

Northern pop have higher overall mean dev rate than southern populations. Prop in phase 1 didn't differ, it looks like. But in the model form table 1, there is a sig local X temp interaction. The y axis may be too large in units for us to see the interaction... 

Table 2. local comparisons for each phase of growth; overall, most locals are diff 

Fig 3. Weight loss is described as having 2 phases. 

FIg 4. prop initial weight vs days for each time point. They could have displayed this better. Calculate the break point and then you'll have the time for each phase. (would be nice to have metabolic rate with these data, maybe have ot check other refs) Then test the effects of local + env(lat) on the timing of each phase. 

Table 3: variance partitioning of growth for each phase, for each temperature level. First phase always explains more of the variation (> 50%!). 


### Thoughts/future directions

* Would have been cool if they measured life time reproductive success to determine fitness consequences of treatments.  

* There needs to be a full factorial design on winter length and emergence temperatures to understand the relative importance on spring time emergence. Done is Rhagoletis for allele frequencies ; but would be nice to measure fitness   

* Would have produced different plots to make it more understandable. Actually plot stuff across lat or the real env variables themselves.

* Figure 1. give this fig, you'd expect faster dev rates for sweden than uk due to the differences in the slopes of the lines. Sweden is steeper, selecting for a phenotype that chagnes in response to it.  



<div id='id-section14'/>    
    
### Page 14:  2018-01-12. working on **hsp rxn norm paper**   

* Mainly working on results and tidying up sentences. 
* There are 2 abstracts, taking out the one that got voted out!    

**Abstract v2
	Colonization into warm environments may involve mechanisms to cope with protein unfolding that is caused by high temperatures. These mechanisms include protection, tolerance, and resistance by heat shock proteins (Hsps), but the extent that contributes to physiological limits is poorly understood. We evaluated the relative importance of Hsp expression dynamics related to upper thermal limits (CTmax) of Eastern US forest ants in the genus Aphaenogaster whose geographical ranges encompass a climate gradient across two distinct forest types. Within this group, the evolutionary transition from closed canopy deciduous forests into open canopy pine forests occurred once and coincided with an increase in CTmax. Using a function-valued approach, we tested whether species divergence in CTmax between forest types corresponded with changes the Hsp expression kinetics that support the concerted roles of protection, tolerance and resistance. Protection was conferred with pre-emptive higher expression at the cooler end of the temperature gradient by slowing the overall expression rate. Greater tolerance was brought about by higher peak expression. Lastly, colonies with higher CTmax delayed their onset of expression, suggesting greater stress resistance to temperature perturbations. The upward extension of CTmax involved coordinated, adaptive, and dynamic shifts in the stress response that reflects better maintenance of protein homeostasis under temperature challenge and may have facilitated the diversification of forest ants into divergent thermal environments. These results lay out the expectation for the types of evolved mechanisms needed for surviving a warmer world in the future.**   
	
* I still need to tweak the other abstract and incorporate edits from Dan. 

* Need to adjust results of topic sentences with this comment from SHC: 

> It would be nice to have a statement of the objective and what you did to test it at the beginning of each results paragraph (To do X, we did Y).  The methods exist, but they are at the end so your readers will not know what is going on.   

* Took out this sentence in intro, leave it here:  

> The protective effects of Hsps are exemplified by enhanced thermal tolerance in flies with higher experimentally increased copy number (Welte et al. 1993; Feder 1995).   



------

<div id='id-section15'/>    
    
### Page 15: 2018-01-16. meeting with KBora on qpcr troubleshooting     

KBora wants to quantify the *foraging* gene from ant brains of pogos  

She has primers, but her reactions are coming up at around 30-40 CT (cycle thresholds). She wants to test out left over cDNA to see if primers work on my samples. 

But lets check the melt curves: They look "good" , single peak. Primer only control wells don't have any amplification, which is good. Therefore, the good ones can be used for a standard.

Recommendations: 

1. take 50 uL amouont of "good" samples (mixed)
2. pcr purify with qiagen kit
3. quantify with qubit 
4. dilute, 1/100 (generally , you can be more precise), then dilute 1/10 serially (12 times, so 1 row on qpcr plate; leave last sample as water for control). 


Kbora will send me melt curve pictures, amplification curves, and the datasheet. 
The goal is to estimate the efficiency of the primers. Once you have this value, you can use it for delta delta CT calculations of gene expression. You should also sequence the amplicon to make sure the sequence is what you think it is (map abi sequencing files to reference sequence in geneious). 

------



<div id='id-section16'/>    
    
### Page 16:  

------

<div id='id-section17'/>    
    
### Page 17:  

------

<div id='id-section18'/>    
    
### Page 18:  

------

<div id='id-section19'/>    
    
### Page 19:  

------

<div id='id-section20'/>    
    
### Page 20:  

------

<div id='id-section21'/>    
    
### Page 21:  

------

<div id='id-section22'/>    
    
### Page 22:  

------

<div id='id-section23'/>    
    
### Page 23:  

------

<div id='id-section24'/>    
    
### Page 24:  

------

<div id='id-section25'/>    
    
### Page 25:  

------

<div id='id-section26'/>    
    
### Page 26:  

------

<div id='id-section27'/>    
    
### Page 27:  

------

<div id='id-section28'/>    
    
### Page 28:  

------

<div id='id-section29'/>    
    
### Page 29:  

------

<div id='id-section30'/>    
    
### Page 30:  

------

<div id='id-section31'/>    
    
### Page 31:  

------

<div id='id-section32'/>    
    
### Page 32:  

------

<div id='id-section33'/>    
    
### Page 33:  

------

<div id='id-section34'/>    
    
### Page 34:  

------

<div id='id-section35'/>    
    
### Page 35:  

------

<div id='id-section36'/>    
    
### Page 36:  

------

<div id='id-section37'/>    
    
### Page 37:  

------

<div id='id-section38'/>    
    
### Page 38:  

------

<div id='id-section39'/>    
    
### Page 39:  

------

<div id='id-section40'/>    
    
### Page 40:  

------

<div id='id-section41'/>    
    
### Page 41:  

------

<div id='id-section42'/>    
    
### Page 42:  

------

<div id='id-section43'/>    
    
### Page 43:  

------

<div id='id-section44'/>    
    
### Page 44:  

------

<div id='id-section45'/>    
    
### Page 45:  

------

<div id='id-section46'/>    
    
### Page 46:  

------

<div id='id-section47'/>    
    
### Page 47:  

------

<div id='id-section48'/>    
    
### Page 48:  

------

<div id='id-section49'/>    
    
### Page 49:  

------

<div id='id-section50'/>    
    
### Page 50:  

------

<div id='id-section51'/>    
    
### Page 51:  

------

<div id='id-section52'/>    
    
### Page 52:  

------

<div id='id-section53'/>    
    
### Page 53:  

------

<div id='id-section54'/>    
    
### Page 54:  

------

<div id='id-section55'/>    
    
### Page 55:  

------

<div id='id-section56'/>    
    
### Page 56:  

------

<div id='id-section57'/>    
    
### Page 57:  

------

<div id='id-section58'/>    
    
### Page 58:  

------

<div id='id-section59'/>    
    
### Page 59:  

------

<div id='id-section60'/>    
    
### Page 60:  

------

<div id='id-section61'/>    
    
### Page 61:  

------

<div id='id-section62'/>    
    
### Page 62:  

------

<div id='id-section63'/>    
    
### Page 63:  

------

<div id='id-section64'/>    
    
### Page 64:  

------

<div id='id-section65'/>    
    
### Page 65:  

------

<div id='id-section66'/>    
    
### Page 66:  

------

<div id='id-section67'/>    
    
### Page 67:  

------

<div id='id-section68'/>    
    
### Page 68:  

------

<div id='id-section69'/>    
    
### Page 69:  

------

<div id='id-section70'/>    
    
### Page 70:  

------

<div id='id-section71'/>    
    
### Page 71:  

------

<div id='id-section72'/>    
    
### Page 72:  

------

<div id='id-section73'/>    
    
### Page 73:  

------

<div id='id-section74'/>    
    
### Page 74:  

------

<div id='id-section75'/>    
    
### Page 75:  

------

<div id='id-section76'/>    
    
### Page 76:  

------

<div id='id-section77'/>    
    
### Page 77:  

------

<div id='id-section78'/>    
    
### Page 78:  

------

<div id='id-section79'/>    
    
### Page 79:  

------

<div id='id-section80'/>    
    
### Page 80:  

------

<div id='id-section81'/>    
    
### Page 81:  

------

<div id='id-section82'/>    
    
### Page 82:  

------

<div id='id-section83'/>    
    
### Page 83:  

------

<div id='id-section84'/>    
    
### Page 84:  

------

<div id='id-section85'/>    
    
### Page 85:  

------

<div id='id-section86'/>    
    
### Page 86:  

------

<div id='id-section87'/>    
    
### Page 87:  

------

<div id='id-section88'/>    
    
### Page 88:  

------

<div id='id-section89'/>    
    
### Page 89:  

------

<div id='id-section90'/>    
    
### Page 90:  

------

<div id='id-section91'/>    
    
### Page 91:  

------

<div id='id-section92'/>    
    
### Page 92:  

------

<div id='id-section93'/>    
    
### Page 93:  

------

<div id='id-section94'/>    
    
### Page 94:  

------

<div id='id-section95'/>    
    
### Page 95:  

------

<div id='id-section96'/>    
    
### Page 96:  

------

<div id='id-section97'/>    
    
### Page 97:  

------

<div id='id-section98'/>    
    
### Page 98:  

------

<div id='id-section99'/>    
    
### Page 99:  

------

<div id='id-section100'/>    
    
### Page 100:  

------

<div id='id-section101'/>    
    
### Page 101:  

------

<div id='id-section102'/>    
    
### Page 102:  

------

<div id='id-section103'/>    
    
### Page 103:  

------

<div id='id-section104'/>    
    
### Page 104:  

------

<div id='id-section105'/>    
    
### Page 105:  

------

<div id='id-section106'/>    
    
### Page 106:  

------

<div id='id-section107'/>    
    
### Page 107:  

------

<div id='id-section108'/>    
    
### Page 108:  

------

<div id='id-section109'/>    
    
### Page 109:  

------

<div id='id-section110'/>    
    
### Page 110:  

------

<div id='id-section111'/>    
    
### Page 111:  

------

<div id='id-section112'/>    
    
### Page 112:  

------

<div id='id-section113'/>    
    
### Page 113:  

------

<div id='id-section114'/>    
    
### Page 114:  

------

<div id='id-section115'/>    
    
### Page 115:  

------

<div id='id-section116'/>    
    
### Page 116:  

------

<div id='id-section117'/>    
    
### Page 117:  

------

<div id='id-section118'/>    
    
### Page 118:  

------

<div id='id-section119'/>    
    
### Page 119:  

------

<div id='id-section120'/>    
    
### Page 120:  

------

<div id='id-section121'/>    
    
### Page 121:  

------

<div id='id-section122'/>    
    
### Page 122:  

------

<div id='id-section123'/>    
    
### Page 123:  

------

<div id='id-section124'/>    
    
### Page 124:  

------

<div id='id-section125'/>    
    
### Page 125:  

------

<div id='id-section126'/>    
    
### Page 126:  

------

<div id='id-section127'/>    
    
### Page 127:  

------

<div id='id-section128'/>    
    
### Page 128:  

------

<div id='id-section129'/>    
    
### Page 129:  

------

<div id='id-section130'/>    
    
### Page 130:  

------

<div id='id-section131'/>    
    
### Page 131:  

------

<div id='id-section132'/>    
    
### Page 132:  

------

<div id='id-section133'/>    
    
### Page 133:  

------

<div id='id-section134'/>    
    
### Page 134:  

------

<div id='id-section135'/>    
    
### Page 135:  

------

<div id='id-section136'/>    
    
### Page 136:  

------

<div id='id-section137'/>    
    
### Page 137:  

------

<div id='id-section138'/>    
    
### Page 138:  

------

<div id='id-section139'/>    
    
### Page 139:  

------

<div id='id-section140'/>    
    
### Page 140:  

------

<div id='id-section141'/>    
    
### Page 141:  

------

<div id='id-section142'/>    
    
### Page 142:  

------

<div id='id-section143'/>    
    
### Page 143:  

------

<div id='id-section144'/>    
    
### Page 144:  

------

<div id='id-section145'/>    
    
### Page 145:  

------

<div id='id-section146'/>    
    
### Page 146:  

------

<div id='id-section147'/>    
    
### Page 147:  

------

<div id='id-section148'/>    
    
### Page 148:  

------

<div id='id-section149'/>    
    
### Page 149:  

------

<div id='id-section150'/>    
    
### Page 150:  

------

<div id='id-section151'/>    
    
### Page 151:  

------

<div id='id-section152'/>    
    
### Page 152:  

------

<div id='id-section153'/>    
    
### Page 153:  

------

<div id='id-section154'/>    
    
### Page 154:  

------

<div id='id-section155'/>    
    
### Page 155:  

------

<div id='id-section156'/>    
    
### Page 156:  

------

<div id='id-section157'/>    
    
### Page 157:  

------

<div id='id-section158'/>    
    
### Page 158:  

------

<div id='id-section159'/>    
    
### Page 159:  

------

<div id='id-section160'/>    
    
### Page 160:  

------

<div id='id-section161'/>    
    
### Page 161:  

------

<div id='id-section162'/>    
    
### Page 162:  

------

<div id='id-section163'/>    
    
### Page 163:  

------

<div id='id-section164'/>    
    
### Page 164:  

------

<div id='id-section165'/>    
    
### Page 165:  

------

<div id='id-section166'/>    
    
### Page 166:  

------

<div id='id-section167'/>    
    
### Page 167:  

------

<div id='id-section168'/>    
    
### Page 168:  

------

<div id='id-section169'/>    
    
### Page 169:  

------

<div id='id-section170'/>    
    
### Page 170:  

------

<div id='id-section171'/>    
    
### Page 171:  

------

<div id='id-section172'/>    
    
### Page 172:  

------

<div id='id-section173'/>    
    
### Page 173:  

------

<div id='id-section174'/>    
    
### Page 174:  

------

<div id='id-section175'/>    
    
### Page 175:  

------

<div id='id-section176'/>    
    
### Page 176:  

------

<div id='id-section177'/>    
    
### Page 177:  

------

<div id='id-section178'/>    
    
### Page 178:  

------

<div id='id-section179'/>    
    
### Page 179:  

------

<div id='id-section180'/>    
    
### Page 180:  

------

<div id='id-section181'/>    
    
### Page 181:  

------

<div id='id-section182'/>    
    
### Page 182:  

------

<div id='id-section183'/>    
    
### Page 183:  

------

<div id='id-section184'/>    
    
### Page 184:  

------

<div id='id-section185'/>    
    
### Page 185:  

------

<div id='id-section186'/>    
    
### Page 186:  

------

<div id='id-section187'/>    
    
### Page 187:  

------

<div id='id-section188'/>    
    
### Page 188:  

------

<div id='id-section189'/>    
    
### Page 189:  

------

<div id='id-section190'/>    
    
### Page 190:  

------

<div id='id-section191'/>    
    
### Page 191:  

------

<div id='id-section192'/>    
    
### Page 192:  

------

<div id='id-section193'/>    
    
### Page 193:  

------

<div id='id-section194'/>    
    
### Page 194:  

------

<div id='id-section195'/>    
    
### Page 195:  

------

<div id='id-section196'/>    
    
### Page 196:  

------

<div id='id-section197'/>    
    
### Page 197:  

------

<div id='id-section198'/>    
    
### Page 198:  

------

<div id='id-section199'/>    
    
### Page 199:  

------

<div id='id-section200'/>    
    
### Page 200:  

